<div class="recent-news">
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/gocam-32px.png"/>
<h4 id="new-interactive-views-for-exploring-go-cam-pathways"><strong>New Interactive Views for Exploring GO-CAM Pathways</strong></h4>
<p><em>2025-08-11</em></p>
<p>Introducing two new interactive views designed to support the exploration and use of <a routerLink="/gocam/">GO-CAM</a> (Gene Ontology Causal Activity Model) data.</p>
<h5 id="summary-map"><strong>Summary Map</strong></h5>
<p><a href="assets/newsfeed/gocam-summary-map.png"><img src="assets/newsfeed/gocam-summary-map-trim.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Summary Map" /></a></p>
<p>The <strong>Summary Map</strong> provides a high-level, visual overview of how individual GO-CAM pathways are causally connected‚Äîwithout displaying their internal structure. This allows you to quickly see relationships between pathways at a glance:</p>
<ul>
<li><a routerLink="/gocam/summary/all">View all models - connected and unconnected</a></li>
<li><a routerLink="/gocam/summary/connected">View only connected models</a></li>
</ul>
<p>The maps are fully interactive:</p>
<ul>
<li>Click on a box to open the corresponding GO-CAM</li>
<li>Click on a relationship arrow to explore connections between models</li>
<li>Zoom, drag, and rearrange nodes to explore the layout</li>
</ul>
<h5 id="mega-model"><strong>Mega Model</strong></h5>
<p><a href="assets/newsfeed/gocam-mega-model.png"><img src="assets/newsfeed/gocam-mega-model-trim.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Mega Model" /></a></p>
<p>The <strong>Mega Model</strong> view displays GO-CAMs in full detail, showing all intra- and inter-model connections. Initially, chemical inputs/outputs and target genes are hidden to reduce visual complexity‚Äîbut you can toggle them on or off as needed.</p>
<ul>
<li><a routerLink="/gocam/mega-model/all">View all models</a></li>
<li><a routerLink="/gocam/mega-model/connected">View only connected models</a></li>
</ul>
<p>Like the Summary Map, Mega Models are fully interactive:</p>
<ul>
<li>Zoom and pan to navigate the network</li>
<li>Click on gene icons or pathway boxes to access detailed information and links to full GO-CAM pathways</li>
</ul>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="jbrowse-version-2-is-now-available-on-pombase"><strong>JBrowse version 2 is now available on PomBase</strong></h4>
<p><em>2025-06-04</em></p>
<p>Version 2 of the JBrowse genome browser is now available on PomBase.<br />
<a href="https://www.pombase.org/jbrowse2/">PomBase JBrowse V2</a></p>
<p>You can now browse the S. pombe genome, and visualize all of our hosted datasets, in a new, nicer interface. JBrowse V2 possesses several new features, such as the possibility to show regions of different chromosomes in the same view, the ability to toggle on or off the display of the forward, reverse and translation sequences for the reference sequence track, and the support of Hi-C data. You can find the complete list of new features on <a href="https://jbrowse.org/jb2/features/">this page</a></p>
<p>Both the old (V1) and the new versions of JBrowse are linked to from the PomBase front page and at the top of the Genome browser widget on the gene pages.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="pombase-expands-go-cam-pathway-models-for-fission-yeast"><strong>PomBase Expands GO-CAM Pathway Models for Fission Yeast</strong></h4>
<p><em>2025-05-12</em></p>
<p>PomBase has more than doubled its curated <a routerLink="/gocam">GO-Causal Activity Modelling</a> (GO-CAM) pathway models‚Äîfrom 36 to 74‚Äînow covering over <a routerLink="/results/from/id/f99d8133-3206-4941-b44e-9314e7cae3d2">1000 fission yeast proteins</a>. These causal models span key biological pathways, with a focus on mitochondrial processes through collaboration with FlyBase and UniProtKB. New tools for pathway visualization and interaction are in development and will be showcased at Pombe 2025.</p>
<p>Explore the pathways and gaps on the <a routerLink="/gocam">PomBase GO-CAM page</a>.</p>
<p>Example GO-CAM pathway:<br />
<a routerLink="/gocam/pombase-view/docs/678073a900003902">L-proline biosynthetic process (GO:0055129), proline catabolic process to glutamate (GO:0010133), glutamine biosynthetic process (GO:0006542))</a></p>
<p><a href="assets/newsfeed/gocam-update-1000-genes.png"><img src="assets/newsfeed/gocam-update-1000-genes.png" class="screenshot" style="width:100.0%" loading="lazy" alt="L-proline biosynthetic process (GO:0055129), proline catabolic process to glutamate (GO:0010133), glutamine biosynthetic process (GO:0006542)" /></a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="the-transcription-factor-dataset-from-skribbe-soneson-stadler-et-al.-is-now-available-in-pombase"><strong>The Transcription Factor Dataset from Skribbe, Soneson, Stadler <em>et al</em>. is now available in PomBase</strong></h4>
<p><em>2025-03-26</em></p>
<p>This publication is an incredible resource to provide biological insights into known and previously unpublished transcription factors, and masterclass in making data <a href="https://www.go-fair.org/fair-principles/">FAIR compliant</a>.</p>
<p><a routerLink="/reference/PMID:40015273">PomBase publication page PMID:40015273</a></p>
<p>Provides:</p>
<ul>
<li>External links to the key resources associated with this project</li>
<li>GO molecular function annotation for promoter binding (&gt;4000 annotations)</li>
<li>Data from the small-scale studies related to ntu1/ntu2 curated in Canto</li>
<li>Affinity capture mass spec data (&gt;300 annotations)</li>
<li>A chooser for the data tracks hosted in JBrowse (&gt;300 data tracks)</li>
</ul>
<p><a href="/jbrowse/?loc=II%3A4442142..4454370&amp;tracks=Forward%20strand%20features%2CReverse%20strand%20features%2CPho7%20ChIP-seq%20%3B%20repeat%201%20-%20Skribbe%20et%20al.%20(2025)&amp;tracklist=1&amp;nav=1&amp;overview=1&amp;highlight"><img src="/assets/newsfeed/skribbe-pmid-40015273-jbrowse.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Skribbe Transcription Factor Dataset in JBrowse" /></a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/>
<h4 id="interpro-version-104.0"><strong>InterPro Version 104.0</strong></h4>
<p><em>2025-03-24</em></p>
<p>PomBase now uses InterPro Version 104.0</p>
<p>New InterPro features in this release include:</p>
<ul>
<li>The addition of 661 InterPro entries.</li>
<li>An update to Pfam [37.2], NCBIfam [17.0], CDD [3.21].</li>
<li>Integration of 682 new methods from the Pfam (663), NCBIfam (5), PANTHER (5), CDD (9) databases.</li>
</ul>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="the-human-functionome-paper-published"><strong>The human functionome paper published</strong></h4>
<p><em>2025-02-27</em></p>
<p>The Gene Ontology Consortium‚Äôs latest publication, <em>A compendium of human gene functions derived from evolutionary modelling</em>, is <a href="https://nature.com/articles/s41586-025-08592-0">out in Nature</a>! PomBase is proud to have contributed to this extensive effort, which leverages all model organism data to support phylogenetic transfer of annotation. Fission yeast GO data currently support <a href="https://www.ebi.ac.uk/QuickGO/annotations?taxonId=9606&taxonUsage=descendants&withFrom=PomBase">6,461 human gene annotations</a>.<br />
Beyond providing annotation, PomBase played a crucial role in reviewing annotation inferences, ensuring accuracy of functional annotation across species. Over seven years, PomBase submitted nearly 600 update requests to the project.</p>
<ul>
<li>üîó <a href="https://functionome.geneontology.org">Explore the functionome</a><br />
</li>
<li>üìñ <a href="https://nature.com/articles/s41586-025-08592-0">Read the paper</a></li>
</ul>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="new-the-missing-activity-list---a-list-of-potential-pathway-holes"><strong>NEW: The Missing activity list - A list of potential ‚Äúpathway holes‚Äù</strong></h4>
<p><em>2025-02-25</em></p>
<p>During GO-CAM pathway curation, we have sometimes been unable to locate a specific annotated activity.</p>
<p>We have generated a <a routerLink="/status/missing-gocam-proteins">list of these missing activities</a> from the GO-CAM pathways.</p>
<p>We expect this list to continue to grow as we curate more pathways. Please note that in some cases these may not be present, or a reaction may be spontaneous.</p>
<p>The list is currently linked to from the <a routerLink="/status/protein-status-tracker">gene characterisation page</a> and from the <a routerLink="/documentation/gocams">GO-CAM pathway list</a>.</p>
<p>Example of pathway with a missing gene / pathway hole: <a routerLink="/gocam/pombase-view/gene/665912ed00002626/SPAC1F7.09c/dal2">nitrogen cycle metabolic process (GO:0071941)</a></p>
<p><a href="assets/newsfeed/gocam_pathway_hole.png"><img src="/assets/newsfeed/gocam_pathway_hole.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Example GO-CAM pathway hole - nitrogen cycle metabolic process" /></a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="pombase-is-global-core-biodata-resource-of-the-week"><strong>PomBase is Global Core BioData Resource of the week</strong></h4>
<p><em>2025-02-17</em></p>
<p>PomBase is this weeks featured <a href="https://globalbiodata.org/">Global Biodata Coalition</a> ‚Äú<a href="https://www.linkedin.com/pulse/pombase-global-biodata-coalition-yocme">Global Core BioData Resource of the week</a>‚Äù.</p>
<p>The Global Biodata Coalition works for and with research funders to ensure sustained support for biodata resources.</p>
<p><a href="https://www.linkedin.com/pulse/pombase-global-biodata-coalition-yocme">Global Core BioData Resource of the week - LinkedIn</a></p>
<p><a href="assets/newsfeed/pombase-gcbr-roc.png"><img src="/assets/newsfeed/pombase-gcbr-roc.png" class="screenshot" style="width:100.0%" loading="lazy" alt="PomBase - GCBR Resource of the week" /></a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="new-curated-pathways-using-go-cam-pathway-modelling"><strong>NEW: Curated pathways using GO-CAM pathway modelling</strong></h4>
<p><em>2025-01-16</em></p>
<p>We have released the first <a routerLink="/gocam">36 curated biological pathways</a>, covering 466 proteins.</p>
<p>These pathways were curated using GO-Causal Activity Modeling (GO-CAM), which provides a system to represent biological processes as a causal network by connecting GO molecular function terms (activities), encoded by individual gene products.</p>
<p>GO-CAM pathway for <a routerLink="/gocam/pombase-view/docs/66c7d41500001171">vitamin B6 metabolic process</a>:</p>
<p><a href="assets/newsfeed/go-cam_vitamin_B6_metabolic_process.png"><img src="assets/newsfeed/go-cam_vitamin_B6_metabolic_process.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Example GO-CAM pathway - vitamin B6 metabolic process" /></a></p>
<p>Visit the <a routerLink="/gocam">GO-CAM pathways page</a> to see the list of curated pathways.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="discovery-of-new-s.-pombe-orthologs-of-human-and-s.-cerevisiae-proteins-using-pfam-n"><strong>Discovery of new S. pombe orthologs of human and S. cerevisiae proteins using Pfam-N</strong></h4>
<p><em>2024-11-22</em></p>
<p>Our work to using Pfam-N to identify distant orthologs between fission yeast and human or budding yeast was included in the <a href="https://doi.org/10.1093/nar/gkae997">Pfam NAR update</a>. Check out the rest of the paper to see recent improvements to Pfam as they continue to work toward comprehensive coverage of the protein universe.</p>
<p><img src="assets/newsfeed/pfam-nar-article-2024.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Pfam NAR article 2024" /></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="mobidb-disorder-predictions"><strong>MobiDB disorder predictions</strong></h4>
<p><em>2024-11-14</em></p>
<p>We now use <a href="https://doi.org/10.1093/bioinformatics/btx015">MobiDB-lite</a> for identifying disordered regions replacing IUPred. Unlike single-residue level predictions, MobiDB-lite focuses on providing a single, consensus-based prediction optimized for high specificity, particularly for detecting long disordered regions with minimal false positives.</p>
<p><a routerLink="/gene/SPAC19G12.10c">Example from cpy1</a>:</p>
<p><a href="assets/newsfeed/feature-full-view-cpy1-mobidb-big.png"><img src="assets/newsfeed/feature-full-view-cpy1-mobidb.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Example full protein feature view for cpy1 with MobiDB-lite disorder features hightlighted" /></a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="additional-protein-features-imported-from-uniprotkb"><strong>Additional protein features imported from UniProtKB</strong></h4>
<p><em>2024-11-13</em></p>
<p>Additional protein features and modifications inferred from sequence context by the UniProt database have been imported to supplement our large curated experimental datasets. <a routerLink="/reference/PMID:36408920">Imported data</a> includes propeptides, active site binding sites, mitochondrial targeting signal and 212 additional signal peptides. Features are listed in the ‚Äúprotein domains and properties‚Äù section of the gene page and features with coordinates are displayed in the protein feature viewer. Signal peptides and mitochondrial targeting signals from all sources can also be retrieved using the queries in the ‚Äúdomains features and motifs‚Äù section of the ‚Äú<a href="/query">Advanced Search</a>‚Äù. We have also imported an additional 445 inferred modifications from UniProtKB to supplement the ~69,000 PomBase curated modifications.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="changes-to-protein-features-view-on-gene-pages"><strong>Changes to protein features view on gene pages</strong></h4>
<p><em>2024-11-12</em></p>
<p>The protein domains and features section of the gene pages now uses the same protein feature display as the ‚Äúfull view‚Äù linked to from the summary section. This provides a consistent view of protein features across the website. We now also display active and binding sites, signal peptides, transit peptides, TMMs, and other Sequence Ontology features in this view.</p>
<p><a routerLink="/gene/SPAC19G12.10c">Example from cpy1</a>:</p>
<p><a href="assets/newsfeed/feature-full-view-cpy1-big.png"><img src="assets/newsfeed/feature-full-view-cpy1.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Example full protein feature view for cpy1" /></a></p>
<p>Thanks to the team at <a href="https://www.rcsb.org/">RCSB PDB</a> for providing the <a href="https://github.com/rcsb/rcsb-saguaro">Open Source software</a> used to implement the protein feature view in PomBase.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/>
<h4 id="pombetalks-season-5-starting-september-18th"><strong>pombeTalks Season 5 starting September 18th</strong></h4>
<p><em>2024-08-29</em></p>
<p>Season 5 of <a href="https://evonuclab.org/pombetalks">pombeTalks</a> will start in September.</p>
<p>Save the dates:</p>
<ul>
<li>September 18th 2024 5pm CET</li>
<li>October 16th 2024 at 5pm CET</li>
<li>November 20th 2024 at 9am CET</li>
<li>December 18th 2024 at 5pm CET</li>
<li>January 15th 2025 at 5pm CET</li>
<li>February 19th 2025 at 9am CET</li>
</ul>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="new-human-and-s.-cerevisiae-orthologs"><strong>New human and <em>S. cerevisiae</em> orthologs</strong></h4>
<p><em>2024-08-13</em></p>
<p>We have recently assigned new orthologs utilising <a href="https://xfam.wordpress.com/2024/05/31/pfam-n-version-3-enhancing-pfam-coverage-of-uniprot-with-computer-vision-deep-learning-techniques/">Pfam-N</a> protein family extensions, <a href="https://www.ebi.ac.uk/Tools/hmmer/search/jackhmmer">JackHMMER</a> iterative searches, and <a href="https://search.foldseek.com/search">FoldSeek</a> structural alignments.</p>
<p>New orthologs:<br />
We have identified orthologs of human <a routerLink="/gene/SPAP27G11.08c">MEIOB</a>, <a routerLink="/gene/SPCC16C4.04">PEX26</a>, <a routerLink="/gene/SPBC19G7.18c">OCIAD1 and OCIAD2</a>, <a routerLink="/gene/SPAC2C4.10c">SIKE1 and FGFR1OP2</a>, <a routerLink="/gene/SPCC1393.11">NGRN</a>, <a routerLink="/gene/SPAC4G9.11c">TFAM</a>, <a routerLink="/gene/SPAC15A10.17">NDUFA3</a>, <a routerLink="/gene/SPCC16C4.04">PEX26 (Sc PEX15)</a>, <a routerLink="/gene/SPAC1805.10">CCZ1</a>.<br />
Plus orthologs of <em>S. cerevisiae</em> <a routerLink="/gene/SPBC16A3.03c">MNE1</a>, <a routerLink="/gene/SPBC11C11.06c">OM14</a>, <a routerLink="/gene/SPAC57A7.15c">PEX22</a>, and <a routerLink="/gene/SPAC23H4.16c">CAF130</a>.</p>
<p>We have also assigned other sequence orphans to known protein families. See <a href="https://lists.cam.ac.uk/sympa/arc/ucam-pombelist/2024-08/msg00000.html">this pombelist post</a> for details.</p>
<p>These additions bring the number of proteins with human orthologs to <a routerLink="/results/from/id/8335a7b3-5105-4414-9743-bb6422692858">3641 (72%)</a>, and the number with S. cerevisiae orthologs to <a routerLink="/results/from/id/12a88b3a-9bab-4eae-a7e0-bb137b55d851">4006 (79%)</a>.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="unknowns-coverage-update"><strong>‚ÄúUnknowns‚Äù coverage update</strong></h4>
<p><em>2024-07-10</em></p>
<p>In our 2019 work, <a href="https://royalsocietypublishing.org/doi/10.1098/rsob.180241">‚ÄúHidden in plain sight: What Remains to be discovered in the eukaryotic proteome?‚Äù</a>, we demonstrated that <em>S. pombe</em> had a higher GO biological process (BP) annotation coverage (86.7%) than both <em>S. cerevisiae</em> (83.5%) and human (82.8%).</p>
<p>Of the model eukaryotes S. pombe has:</p>
<ol type="1">
<li>The highest percentage known coverage</li>
<li>The fewest phylum-specific proteins</li>
<li>The smallest proteome (despite a higher number of genes conserved in metazoa than S. cerevisiae)</li>
<li>Dedicated efforts to uncover unknown functions</li>
</ol>
<p>and is uniquely positioned to be the first eukaryote for which we know the process or pathway for every protein</p>
<p>The trend we identified continues to hold (<em>S. pombe</em> ‚Äúknowns‚Äù have increased to 86.9% in 2024). The coverage is confirmed by the <a href="https://genomeannotation.rheelab.org/#!/overview">Rhee lab GO overview</a>:</p>
<p><a href="assets/newsfeed/Rhee_lab_unknowns_coverage.png"><img src="assets/newsfeed/Rhee_lab_unknowns_coverage.png" title="Rhee lab unknowns coverage" class="screenshot" style="width:100.0%" alt="Rhee lab unknowns coverage" /></a></p>
<p>Image citation: <a href="https://doi.org/10.1002/pld3.499">Xue, B., &amp; Rhee, S. Y. (2023). Status of genome function annotation in model organisms and crops. Plant Direct, 7(7), e499.</a></p>
<p>Related links:</p>
<ul>
<li><a href="https://doi.org/10.1093/genetics/iyae007">Rutherford et al 2024</a> explains PomBase‚Äôs choice of metric for assessing knowledge breadth</li>
<li><a routerLink="/status/protein-status-tracker">Protein coding gene characterization status at PomBase</a></li>
</ul>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/rna_central_icon.png"/>
<h4 id="rna-secondary-structure-predictions-on-gene-pages"><strong>RNA secondary structure predictions on gene pages</strong></h4>
<p><em>2024-07-03</em></p>
<p>We have added RNA secondary structure predictions from <a href="https://rnacentral.org/r2dt">R2DT (RNAcentral)</a> to the gene pages.</p>
<p>When a prediction is available an option will appear in the summary section of the RNA pages. See <a routerLink="/gene/SPRRNA.03">the SPRRNA.03 gene page</a> for an example.</p>
<p><a href="assets/newsfeed/rna_structure_SPRRNA.03_page.png"><img src="assets/newsfeed/rna_structure_SPRRNA.03_page.png" title="RNA structure - SPRRNA.03 gene page" class="screenshot" style="width:100.0%" alt="RNA structure - SPRRNA.03 gene page" /></a></p>
<p>You can retrieve the list of all RNA genes with secondary structure predictions using the ‚ÄúGenes with RNA secondary structure predictions‚Äù query under ‚ÄúCommonly used queries‚Äù in the <a routerLink="/query">PomBase Advanced Search</a>.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/human_disease.png"/>
<h4 id="human-disease-association-changes---monarch"><strong>Human disease association changes - Monarch</strong></h4>
<p><em>2024-06-21</em></p>
<p>We have changed our major source of human disease associations from <a href="https://www.malacards.org/">MalaCards</a> to <a href="https://monarchinitiative.org/">Monarch</a>.</p>
<p>This change has:</p>
<ol type="1">
<li>Increased the number of fission yeast genes with disease associations from 1463 to 1551, made the associations more accurate and up to date</li>
<li>Allowed us to automate the inclusion of new disease gene imports from Monarch based on our ortholog mappings</li>
<li>Enabled us to add the qualifiers ‚Äúcausal‚Äù or ‚Äúcorrelated with‚Äù (for multi-genic disorders) to the disease association (you can see this on the gene pages under ‚Äúshow details‚Äù)</li>
</ol>
<p>Disease associations in PomBase:</p>
<ul>
<li><a routerLink="/term/MONDO:0700096">Browse the new disease list</a></li>
<li><a routerLink="/browse-curation/disease-slim">Overview of disease associations (disease slim)</a></li>
</ul>
<p>If you have knowledge of a specific disease class or notice any issues with the ontology you can <a href="https://github.com/monarch-initiative/mondo/issues/">report this to Monarch</a></p>
<p>The Monarch initiative also host our fission yeast phenotype associations in their <a href="https://monarchinitiative.org/">knowledge graph</a>.</p>
<p>Related items: <a href="https://monarchinit.medium.com/pombase-chooses-monarch-as-source-of-human-disease-gene-associations-e53c6d218dce">Monarch Initiative blog post</a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/>
<h4 id="pombase-now-includes-data-from-interpro-v100.0"><strong>PomBase now includes data from InterPro v100.0</strong></h4>
<p><em>2024-06-14</em></p>
<p>PomBase now uses <a href="https://www.ebi.ac.uk/interpro/">InterPro</a> <a href="https://www.ebi.ac.uk/interpro/release_notes/">release 100.0</a>.</p>
<p>Features include:</p>
<ul>
<li>The addition of 570 InterPro entries.</li>
<li>An update to Pfam [37.0].</li>
<li>Integration of 582 new methods from the NCBIfam (26), Pfam (545), PANTHER (3), CDD (8) databases.</li>
</ul>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="new-database-update-publication-from-pombase"><strong>New database update publication from PomBase</strong></h4>
<p><em>2024-02-22</em></p>
<p>The fission yeast 2024 database update ‚ÄúPomBase: a Global Core Biodata Resource‚Äîgrowth, collaboration, and sustainability‚Äù has been <a href="https://doi.org/10.1093/genetics/iyae007">published in <em>Genetics</em></a>. This update describes PomBase‚Äôs continuing growth and improvement over the past two years and showcases the depth of existing connections both within the biocuration ecosystem and between PomBase and its user community.</p>
<p>We discuss ways to track knowledge coverage including the tracking of unknown proteins, and the use of slimsets. We describe how we provide value by data and software sharing and how we have reused software provided by others in our new views. We provide author-led curation updates, describe contributions to global standards, future plans to support machine learning with high quality curated data and more.</p>
<p><a href="https://doi.org/10.1093/genetics/iyae007">Rutherford <em>et al.</em>, <em>Genetics</em>, iyae007</a></p>
<p><a href="assets/newsfeed/genetics_2024_paper_fig_5.png"><img src="assets/newsfeed/genetics_2024_paper_fig_5.png" title="Data dissemination from PomBase to other resources" class="screenshot" style="width:100.0%" loading="lazy" alt="‚ÄúFig 5 Data dissemination from PomBase to other resources‚Äù" /></a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="viewing-protein-modifications"><strong>Viewing protein modifications</strong></h4>
<p><em>2024-02-08</em></p>
<p>Modified amino acids in the protein feature viewer are now coloured by modification type. See the <a routerLink="/gene_protein_features/SPAC1834.04">histone H3.1 gene page</a> for an example.</p>
<p><a href="assets/newsfeed/gene-hht1-protein-feature-viewer-page.png"><img src="assets/newsfeed/gene-hht1-protein-feature-viewer-page.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Protein feature view for hht1" /></a></p>
<p><a href="assets/newsfeed/protein-feature-modification-key.png"><img src="assets/newsfeed/protein-feature-modification-key.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Protein feature view modification colour key" /></a></p>
<p>Visit the <a routerLink="/documentation/gene-page-protein-features-widget">protein feature view documentation page</a> for more information.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="access-target-genes-of-individual-gene-products"><strong>Access target genes of individual gene products</strong></h4>
<p><em>2024-02-07</em></p>
<p>It is now possible to send lists of annotated molecular function substrates to the advanced search from gene pages:</p>
<p><a href="assets/newsfeed/cdc2-substrates-to-advanced-search.png"><img src="assets/newsfeed/cdc2-substrates-to-advanced-search.png" class="screenshot" style="width:100.0%" loading="lazy" alt="cdc2 - send target genes to the Advanced Search" /></a></p>
<p>Lists of annotated molecular function substrates of individual genes (i.e protein kinase or transcription factor target genes) can now be sent directly to the Advanced Search. You can send complete lists, or pre-filtered by cell cycle phase.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/elixir_uk_thumb.png"/>
<h4 id="pombase-joins-elixir-core-data-resources"><strong>PomBase joins ELIXIR Core Data Resources</strong></h4>
<p><em>2023-12-14</em></p>
<p>PomBase is the first model organism database (MOD) to be selected as an <a href="https://elixir-europe.org">ELIXIR</a> <a href="https://elixir-europe.org/platforms/data/core-data-resources">Core data resource (CDR)</a>. We were evaluated across a long list of indicators including scientific focus and quality, governance, and quality of service. ELIXIR CDRs are European life science data resources that are important to biological and biomedical research and long-term preservation of biological data. This will provide assurance to our funders and users that PomBase satisfies the many criteria required to considered a key part of the European bioinformatics infrastructure. <a href="https://elixir-europe.org">ELIXIR</a> is an intergovernmental organisation that brings together life science resources from across Europe, it currently includes <a href="https://elixir-europe.org/about-us/who-we-are">22 member countries</a>, bringing together over 240 research organisations. The goal of
ELIXIR is to coordinate these resources so that they form a single infrastructure.</p>
<p><a href="assets/newsfeed/elixir_pombase_cdr_announcement.png"><img src="assets/newsfeed/elixir_pombase_cdr_announcement.png" title="ELIXIR CDR announcement" class="screenshot" style="width:100.0%" loading="lazy" alt="ELIXIR CDR announcement" /></a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="new-documentation-and-videos-for-canto"><strong>New documentation and videos for Canto</strong></h4>
<p><em>2023-10-11</em></p>
<p>We have published <a href="https://pombase.github.io/canto_tutorial/">a new website</a> with videos and documentation for <a href="https://curation.pombase.org/">Canto</a>, our community curation tool. The website includes short step-by-step text and video guides on how to make annotations in Canto.</p>
<ul>
<li><a href="https://pombase.github.io/canto_tutorial/">New Canto documentation front page</a></li>
<li><a href="https://www.youtube.com/@PomBase">PomBase YouTube channel</a></li>
</ul>
<p>The work on this website was supported by the ELIXIR-UK: FAIR Data Stewardship training UKRI award (MR/V038966/1) to Manuel Lera Ram√≠rez.</p>
<p><a href="assets/newsfeed/new_canto_video_docs.png"><img src="assets/newsfeed/new_canto_video_docs.png" title="New Canto tutorials video list" class="screenshot" style="width:100.0%" loading="lazy" alt="New Canto tutorials video list" /></a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="revised-fission-yeast-nomenclature-for-genes-and-alleles"><strong>Revised fission yeast nomenclature for genes and alleles</strong></h4>
<p><em>2023-09-30</em></p>
<p>The improved nomenclature guidelines for fission yeast clade genes and alleles have been <a href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyad143/7283026">published in Genetics GSA</a>. This update introduces nomenclature guidelines for noncoding RNA genes, a significant update to the allele and genotype nomenclature guidelines originally published in 1987. We took several steps to improve machine readability that will benefit the artificial intelligence/machine learning communities, and our own community curation program.</p>
<p><a href="https://doi.org/10.1093/genetics/iyad143">Lera-Ram√≠rez <em>et al.</em> 2023, Revised fission yeast gene and allele nomenclature guidelines for machine readability, Genetics, 2023;, iyad143</a></p>
<p><a href="assets/newsfeed/nomenclature_paper_for_news_big.png"><img src="assets/newsfeed/nomenclature_paper_for_news.png" title="Revised fission yeast gene and allele nomenclature guidelines for machine readability" class="screenshot" style="width:100.0%" loading="lazy" alt="‚ÄúRevised fission yeast gene and allele nomenclature guidelines for machine readability‚Äù" /></a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="accessing-the-list-of-genes-from-a-paper"><strong>Accessing the list of genes from a paper</strong></h4>
<p><em>2023-09-20</em></p>
<p>It‚Äôs now possible to view the list of genes associated with a paper, using a new link on the publication pages. The gene list can then optionally be sent to the Advanced Search.</p>
<p><a href="assets/newsfeed/genes-list-from-ref-page.png"><img src="assets/newsfeed/genes-list-from-ref-page.png" title="Publication page, highlightling the new genes link" class="screenshot" style="width:100.0%" loading="lazy" alt="Publication page, highlightling the new genes link" /></a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="protein-feature-viewer-added-to-gene-pages"><strong>Protein feature viewer added to gene pages</strong></h4>
<p><em>2023-07-27</em></p>
<p>PomBase gene pages now include a protein feature widget. This tool shows protein features in the context of amino acid sequence. It includes:</p>
<ul>
<li>manually curated amino acid substitution positions</li>
<li>Pfam domains</li>
<li>protein modifications</li>
<li>protein properties: low complexity regions, disordered regions, coiled coil regions and predicted trans-membrane domains</li>
</ul>
<p>Visit the <a routerLink="/gene/SPAC20G8.05c">cdc15 gene page</a> to see an example.</p>
<p>Hover over features for more information, such as allele descriptions and Pfam domain IDs.</p>
<p><a href="assets/gene-protein-feature-viewer-widget-cdc15.png"><img src="assets/gene-protein-feature-viewer-widget-cdc15.png" title="Protein feature viewer widget - cdc15 gene page" class="screenshot" style="width:100.0%" alt="Protein feature viewer widget - cdc15 gene page" /></a></p>
<p>Follow the ‚ÄúView all protein features ‚Ä¶‚Äù link for a detailed view on the dedicated protein features page that includes:</p>
<ul>
<li>details of the manually curated amino acid substitution allele changes</li>
<li>manually curated partial amino acid deletions</li>
</ul>
<p><a href="assets/gene-protein-feature-viewer-page-cdc15.png"><img src="assets/gene-protein-feature-viewer-page-cdc15.png" title="Protein feature viewer details page for cdc15" class="screenshot" style="width:100.0%" alt="Protein feature viewer details page for cdc15" /></a></p>
<p>Thanks to the team at <a href="https://www.rcsb.org/">RCSB PDB</a> for providing the <a href="https://github.com/rcsb/rcsb-saguaro">Open Source software</a> used to implement this feature.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="reaction-diagrams-on-go-molecular-function-term-pages"><strong>Reaction diagrams on GO molecular function term pages</strong></h4>
<p><em>2023-06-19</em></p>
<p>Where available, we now show the reaction diagram from <a href="https://www.rhea-db.org/">Rhea</a> on GO molecular function term pages. This feature is possible thanks to the great work of <a href="https://www.rhea-db.org/">Rhea</a>.</p>
<p>See the <a routerLink="/term/GO:0003849">3-deoxy-7-phosphoheptulonate synthase activity (GO:0003849) term page</a> for an example.</p>
<p><a href="assets/newsfeed/rhea-reaction-example.png"><img src="assets/newsfeed/rhea-reaction-example.png" class="screenshot" style="width:100.0%" loading="lazy" /></a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="curation-update---nonsense-mutation-merged-into-partial-amino-acid-deletion"><strong>Curation update - ‚Äúnonsense mutation‚Äù merged into ‚Äúpartial amino acid deletion‚Äù</strong></h4>
<p><em>2023-04-26</em></p>
<p>We have decided to merge the allele type ‚Äúnonsense mutation‚Äù into ‚Äúpartial amino acid deletion‚Äù. This has mainly been driven by the fact that allele types that combine different variants require new types, such as ‚Äúamino_acid_deletion_and_mutation‚Äù, ‚Äúamino_acid_insertion_and_deletion‚Äù, etc. Otherwise, we would have ended up with many more types, and at the gene product level (which is what we describe in PomBase in phenotype interactions), both truncations are equivalent. In the next update, this allele type will not be available in Canto.</p>
<p>In any case, even if two alleles produce the same truncation, such as ase1-D13* or ase1Œî(13-731), they would still have separate entries in PomBase, and they may have different phenotypes. We are only assigning them the same allele type.</p>
<p>If for your analysis you need to make a distinction between the two using our allele dataset <a routerLink="/downloads/phenotype-annotations">allele dataset</a>, you can always check the ‚ÄúAllele description‚Äù field for the presence of the ‚Äú*‚Äù character to tell whether an allele includes a nonsense mutation.</p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/protein-structure.png"/>
<h4 id="experimental-structures-from-pdb-on-gene-pages"><strong>Experimental structures from PDB on gene pages</strong></h4>
<p><em>2023-02-22</em></p>
<p>The experimental protein structures from <a href="https://www.ebi.ac.uk/pdbe/">PDB</a> are now embedded on the PomBase gene pages using <a href="https://molstar.org/">Mol*</a>. For example: <a routerLink="/gene/SPCC285.12">lsm7/SPCC285.12 gene page</a></p>
<p>If you select the ‚ÄúPDB structures‚Äù view on a gene page, experimental structures will set as your default. AlphaFold predictions will be shown for genes where an experimental structure are not available.</p>
<p>We now also display the structures on the associated publication page. For example: <a routerLink="/reference/PMID:31010807">PMID:31010807 Garg <em>et al.</em></a></p>
<p>To help locate proteins with experimental protein structures (currently 375), we have added a new query option to the ‚Äú<a routerLink="/query">Advanced search</a>‚Äù, currently under ‚Äúcommonly used queries‚Äù: ‚Äú<a routerLink="/results/from/id/6e8401aa-bab8-48fc-bbb7-c85d13ea10a2">Proteins with PDB structures</a>‚Äù</p>
<p><a href="assets/newsfeed/pdb_structures_lsm7_small.png"><img src="assets/newsfeed/pdb_structures_lsm7_small.png" class="screenshot" style="width:100.0%" loading="lazy" alt="PDB structures on the lsm7 gene page" /></a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/protein-structure.png"/>
<h4 id="alphafold-protein-structure-on-gene-pages"><strong>AlphaFold protein structure on gene pages</strong></h4>
<p><em>2023-02-02</em></p>
<p>AlphaFold protein structure are now embedded on the PomBase gene pages. We hope to embed the experimental structures from PDB in the near future.</p>
<p><a href="assets/newsfeed/alphafold-news-item-example-1.png"><img src="assets/newsfeed/alphafold-news-item-example-1.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Example from the mvp1 gene page" /></a></p>

</div>
<div class="news-item">
<img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/>
<h4 id="revised-canonical-5-utrs"><strong>Revised canonical 5‚Äô UTRs</strong></h4>
<p><em>2023-01-24</em></p>
<p>We have revised the curated 5‚ÄôUTRs using Transcription Start Sites (TSS) data (in vegetative growth/ minimal media) from the Cap Analysis of Gene Expression (CAGE) data provided by <a routerLink="/reference/PMID:30566651">Thodberg <em>et al</em></a>. All new gene structures were manually reviewed, around ~80 protein features had N-terminal coordinate revisions to align with TSS data.</p>

</div>

<div id="archive-link"><a routerLink="/news/">News archive ...</a></div>
</div>

