<div class="docs">
<app-social-contact></app-social-contact>

<div class="docs-404" *ngIf="pageName == '404'">
  <div>
    <app-not-found [path]="router.url"></app-not-found>
  </div>
</div>
<div *ngIf="section == 'about'">
<app-page-contents-menu title="About" titleRoute="/about">

<div class="left-menu-part left-menu-item"><a routerLink="/about/citing-pombase">Citing PomBase</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/about/pombase-and-collaborative-publications">PomBase and collaborative publications</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/about/terms-of-use">Terms of Use</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/about/pombase-sab">SAB</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/about/pombase-staff">Staff</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/about/privacy-policy">Privacy Policy</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/help">Help</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/about/resource-metrics">Resource metrics</a></div>


</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'index'">
<h2 id="about-pombase">About PomBase</h2>
<p>PomBase is the comprehensive model organism knowledgebase for the fission yeast <em>Schizosaccharomyces pombe</em>, which aims to standardise, integrate, display, and disseminate biological knowledge and datasets to the wider scientific community, making a wide range of data-types from large and small-scale publications <a href="https://pubmed.ncbi.nlm.nih.gov/26978244/">FAIR</a>.</p>
<p>You can use PomBase to:</p>
<ul>
<li>Access and query manually <strong>curated literature</strong> annotations:
<ul>
<li><strong>Gene Ontology (GO)</strong> annotations describing the normal (evolved) roles and locations of gene products.</li>
<li><strong>Phenotype</strong> annotations linked to the alleles and genotypes that cause them.</li>
<li><strong>Genetic and Physical interactions</strong> between genes.</li>
<li><strong>Protein modifications,</strong> the gene products that add/ remove them, when in the cell cycle they occur or under which environmental conditions, .</li>
<li><strong>Phylogenetic</strong> information and <strong>gene complementation.</strong></li>
</ul></li>
<li><strong>Browse the genome</strong> of <em>Schizosaccharomyces pombe</em>, and access <strong>gene sequences and features</strong> (UTRs, exons and introns, protein sequence features).</li>
<li>Download <strong>genome-wide datasets</strong>.</li>
</ul>
<p>PomBase is also a community hub for researchers providing news, events, documentation, mailing lists, genome statistics, and a community curation interface. New to the fission yeast community? Visit our <a routerLink="/documentation/getting-started">getting started page</a>.</p>
<h2 id="more-about-pombase">More about PomBase:</h2>
<ul>
<li>PomBase promotes <strong>community curation</strong>, involving authors in the curation of their published research. This increases curation quality, and allows researchers to become familiar with the data organisation and supported data types. We developed <a href="https://github.com/pombase/canto"><u>Canto</u></a>, a web-based curation tool to support professional and community curation that is also used by <a href="https://flybase.org/"><u>FlyBase</u></a>, <a href="http://www.phi-base.org/"><u>PHI-Base</u></a> and <a href="https://www.japonicusdb.org/"><u>JaponicusDB</u></a>. <a routerLink="/community/fission-yeast-community-curation-project">Read more</a> about community curation.</li>
<li>We provide over <strong>300.000 curated standardised annotations</strong> and our 27.000 experimentally supported GO annotations support more than 687.000 species-wide annotations in other key model species. See our <a routerLink="/metrics"><u>Literature and curation metrics page</u></a> for more.</li>
<li>We actively participate in the <strong>development of several ontologies</strong> to extend and improve standards that represent the knowledge produced by researchers.</li>
<li>See all of our ongoing <a routerLink="/about/collaborations">collaborations</a> and <a routerLink="/about/projects">projects</a>.</li>
</ul>
<hr />
<h3 id="contact-pombase-staff">Contact PomBase staff</h3>
<p>Email: <a href="mailto:helpdesk@pombase.org">helpdesk{{"@"}}pombase.org</a></p>
<p>For more information, see the <a routerLink="/about/pombase-staff">PomBase Staff</a> page.</p>
<h3 id="submit-data">Submit Data</h3>
<p>See the <a routerLink="/submit-data">Submit data</a> page</p>
<h3 id="data-sources">Data sources</h3>
<p>Source publications for external data used in PomBase pages and genome browser.</p>
<p><a routerLink="/about/data-sources">Data sources</a></p>
<hr />
<h3 id="citing-pombase">Citing PomBase</h3>
<p>The <a routerLink="/about/citing-pombase">Citing PomBase</a> page lists papers to cite for PomBase, the <em>S. pombe</em> genome sequence, annotation data, and other resources.</p>
<h3 id="terms-of-use">Terms of Use</h3>
<p><a routerLink="/about/terms-of-use">Terms of Use</a> for PomBase data and Canto.</p>
<h3 id="pombase-publications">PomBase Publications</h3>
<p>Complete list of <a routerLink="/about/pombase-and-collaborative-publications">papers published by PomBase</a>.</p>
<hr />
<h3 id="pombase-sab">PomBase SAB</h3>
<p><a routerLink="/about/pombase-sab">Scientific Advisory Board members</a></p>
<hr />
<h3 id="pombase-staff">PomBase staff</h3>
<p>Current and past <a routerLink="/about/pombase-staff">PomBase Staff</a></p>
<hr />
<h3 id="data-versions">Data versions</h3>
<p>Versions of internal and external data loaded into current and past PomBase releases.</p>
<p><a routerLink="/about/version-history">Version history</a></p>
<h3 id="receive-project-updates">Receive project updates</h3>
<p>Subscribe to <a href="https://lists.cam.ac.uk/sympa/info/ucam-pombelist">Pombelist</a> to receive annotation and data updates</p>
<hr />
<h3 id="privacy">Privacy</h3>
<p>See the PomBase <a routerLink="/about/privacy-policy">privacy policy</a> and <a routerLink="/about/cookie-policy">cookie policy</a>.</p>
<hr />
<h3 id="project-funding">Project funding</h3>
<p>PomBase is funded by the <a href="https://wellcome.ac.uk/">Wellcome Trust</a> [218236/Z/19/Z].</p>

  </div>
  </div>
</div>
<div *ngIf="section == 'browse-curation'">
<app-page-contents-menu title="Browse curation" titleRoute="/browse-curation">

<div class="left-menu-part left-menu-item"><a routerLink="/browse-curation/fission-yeast-phenotype-ontology">Fission Yeast Phenotype Ontology</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/browse-curation/fission-yeast-mf-go-slim-terms">GO function slim terms</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">GO process slim terms</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/browse-curation/fission-yeast-cc-go-slim-terms">GO component slim terms</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/browse-curation/dna-binding-sites">DNA binding sites</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/browse-curation/drugs-known-pombe-targets">Drugs with known targets</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/browse-curation/disease-slim">Mondo Disease Ontology slim</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'index'">
<h2 id="browse-curation">Browse curation</h2>
<ul>
<li><a routerLink="/browse-curation/fission-yeast-phenotype-ontology">Fission Yeast Phenotype Ontology</a>: Summary page and links to additional information</li>
<li>GO slim annotation summaries
<ul>
<li><a routerLink="/browse-curation/fission-yeast-mf-go-slim-terms">Molecular function GO slim</a>: terms and annotation totals</li>
<li><a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">Biological process GO slim</a>: terms and annotation totals</li>
<li><a routerLink="/browse-curation/fission-yeast-cc-go-slim-terms">Cellular component GO slim</a>: terms and annotation totals</li>
</ul></li>
<li><a routerLink="/browse-curation/disease-slim">Disease association overview</a>: Fission yeast Mondo slim terms and annotation totals</li>
<li><a routerLink="/browse-curation/dna-binding-sites">DNA binding sites</a>: Consensus binding sequences found in <em>S. pombe</em></li>
<li><a routerLink="/browse-curation/drugs-known-pombe-targets">Drug targets</a>: List of drugs known to affect <em>S. pombe</em>, and their targets or mechanism of action</li>
</ul>

  </div>
  </div>
</div>
<div *ngIf="section == 'community'">
<app-page-contents-menu title="Community" titleRoute="/community">

<div class="left-menu-part left-menu-item"><a routerLink="/community/fission-yeast-community-curation-project">Community curation project</a></div>
<div class="left-menu-part left-sub-menu-item"><a href="https://lists.cam.ac.uk/sympa/info/ucam-pombelist">Mailing list</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/community/internet-resources">External resources</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'index'">
<h1 id="fission-yeast-community">Fission Yeast Community</h1>
<ul>
<li><a routerLink="/community/fission-yeast-community-curation-project">Community curation project</a>, including the old <em>S. pombe</em> <a routerLink="/community/fission-yeast-community-curation-pilot-project">pilot project</a></li>
<li><a href="https://lists.cam.ac.uk/sympa/info/ucam-pombelist">Mailing list</a></li>
<li><a routerLink="/community/internet-resources">External resources</a></li>
</ul>

  </div>
  </div>
</div>
<div *ngIf="section == 'datasets'">
<app-page-contents-menu title="Datasets" titleRoute="/datasets">

<div class="left-menu-part left-menu-item"><a routerLink="/downloads/genome-datasets">Genome datasets</a></div>
<div class="left-menu-part left-menu-item"><span>Annotation datasets</span></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/downloads/protein-datasets">Protein datasets</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/downloads/go-annotations">GO annotations</a></div>
<div class="left-menu-part left-sub-menu-item"><a href="/latest_release/macromolecular_complexes">Macromolecular complexes</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/downloads/phenotype-annotations">Phenotype annotations</a></div>
<div class="left-menu-part left-sub-menu-item"><a href="/latest_release/human_disease_annotation">Human disease associations</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/downloads/modifications">Modifications</a></div>
<div class="left-menu-part left-sub-menu-item"><a href="/latest_release/curated_orthologs">Orthologs</a></div>
<div class="left-menu-part left-sub-menu-item"><a href="/latest_release/gene_expression">Expression</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/downloads/names-and-identifiers">Names and IDs</a></div>
<div class="left-menu-part left-menu-item"><span>GO slims</span></div>
<div class="left-menu-part left-sub-menu-item"><a href="/latest_release/gene_ontology/bp_go_slim_terms.tsv">GO molecular function slim</a></div>
<div class="left-menu-part left-sub-menu-item"><a href="/latest_release/gene_ontology/mf_go_slim_terms.tsv">GO biological process slim</a></div>
<div class="left-menu-part left-sub-menu-item"><a href="/latest_release/gene_ontology/cc_go_slim_terms.tsv">GO cellular component slim</a></div>
<div class="left-menu-part left-menu-item"><a href="/data/external_datasets/">Genome browser datasets</a></div>
<div class="left-menu-part left-menu-item"><a href="/latest_release/human_disease_annotation/pombe_mondo_disease_slim_terms.tsv">Mondo Disease Ontology slim</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/downloads/chado-database-dumps">Chado database dumps</a></div>
<div class="left-menu-part left-menu-item"><span>Curated inventories</span></div>
<div class="left-menu-part left-sub-menu-item"><a href="/browse-curation/dna-binding-sites">DNA binding sites</a></div>
<div class="left-menu-part left-sub-menu-item"><a href="/browse-curation/drugs-known-pombe-targets">Drug targets</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'index'">
<h2 id="datasets">Datasets</h2>
<p>All of the datasets curated and compiled by PomBase are accessible from this page. These include genome sequence and features, protein features, associations to ontology terms, browser data, manually curated orthologs and gene names and identifiers.</p>
<p>If you cannot locate a dataset of interest please contact the <a href="mailto:helpdesk@pombase.org">helpdesk</a>.</p>
<hr />
<h3 id="latest-pombase-data">Latest PomBase data</h3>
<p>The most recent release of the PomBase datasets is available from the <a href="https://www.pombase.org/latest_release">monthly release directory</a>.</p>
<p>We also have an <a href="https://www.pombase.org/monthly_releases/">archive of previous monthly releases</a>.</p>
<hr />
<h3 id="dataset-and-ontology-versions">Dataset and ontology versions</h3>
<p>The <a routerLink="/internal-details">versions page</a> is updated each night.</p>
<hr />
<h3 id="genome-sequence-and-features">Genome sequence and features</h3>
<p>Visit the <a routerLink="/downloads/genome-datasets">main genome dataset page</a> for:</p>
<ul>
<li>Genome sequence (FASTA or enriched EMBL format for Artemis)</li>
<li>Feature coordinates (coding regions, introns, exons, UTRs, etc.)</li>
<li>Genomic regions (centromeres, telomeres, mating type)</li>
</ul>
<hr />
<h3 id="annotation-datasets">Annotation datasets</h3>
<p>Annotations are all ontology-based, and extracted from the primary literature based on experimental data from both hypothesis-driven low-throughput experiments and large discovery screens (GO associations are supplemented by non-experimental annotations). All annotations are fully provenanced, and supported by appropriate evidence codes.</p>
<table>
<thead>
<tr class="header">
<th>Annotation type</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a routerLink="/downloads/protein-datasets">Protein datasets</a></td>
<td>Protein sequence in FASTA format, peptide features, properties, etc.</td>
</tr>
<tr class="even">
<td><a routerLink="/downloads/go-annotations/">GO annotations</a></td>
<td>Gene Ontology annotation files</td>
</tr>
<tr class="odd">
<td><a href="https://www.pombase.org/latest_release/macromolecular_complexes/">Macromolecular complexes</a></td>
<td>Subunits of protein and ribonucleoprotein complexes (GO cellular component terms and annotated genes)</td>
</tr>
<tr class="even">
<td><a routerLink="/downloads/phenotype-annotations/">Phenotype annotations</a></td>
<td>FYPO phenotype annotation files - complete annotation set (PHAF) or viability summary</td>
</tr>
<tr class="odd">
<td><a href="https://www.pombase.org/latest_release/human_disease_annotation/">Human disease associations</a></td>
<td>Fission yeast orthologs of human disease associated genes with disease descriptions and <a href="https://mondo.monarchinitiative.org/">Mondo</a> ontology terms</td>
</tr>
<tr class="even">
<td><a routerLink="/downloads/modifications">Modifications</a></td>
<td>Protein modifications</td>
</tr>
<tr class="odd">
<td><a href="https://www.pombase.org/latest_release/curated_orthologs/">Orthologs</a></td>
<td>Manually curated ortholog sets for <a routerLink="/faq/how-can-i-find-s.-pombe-ortholog-s-human-gene">human</a> and <a routerLink="/faq/how-can-i-search-s.-cerevisiae-ortholog-s-homolog-s-s.-pombe-gene"><em>S. cerevisiae</em></a>; also see <a routerLink="/documentation/orthologs">documentation</a></td>
</tr>
<tr class="even">
<td><a href="https://www.pombase.org/latest_release/gene_expression/">Expression</a></td>
<td>Quantitative and qualitative protein and RNA expression data</td>
</tr>
</tbody>
</table>
<hr />
<h3 id="gene-names-and-identifiers">Gene names and identifiers</h3>
<p>Mappings between PomBase systematic IDs, gene names, product descriptions, and UniProt accession numbers:</p>
<ul>
<li><a routerLink="/downloads/names-and-identifiers">Names and identifiers</a></li>
</ul>
<h3 id="go-slim">GO slim</h3>
<p>Current GO slim IDs and term names:</p>
<ul>
<li><a href="https://www.pombase.org/latest_release/gene_ontology/bp_go_slim_terms.tsv">GO biological process slim</a></li>
<li><a href="https://www.pombase.org/latest_release/gene_ontology/mf_go_slim_terms.tsv">GO molecular function slim</a></li>
<li><a href="https://www.pombase.org/latest_release/gene_ontology/cc_go_slim_terms.tsv">GO cellular component slim</a></li>
</ul>
<hr />
<h3 id="datasets-displayed-in-the-genome-browser">Datasets displayed in the genome browser</h3>
<p>The data files used for the tracks in the <a routerLink="/documentation/JBrowse_quick_start">JBrowse genome browser</a> are available from the <a href="https://www.pombase.org/data/external_datasets/">external datasets directory</a>. The directory names are either the publication details (PubMed ID) or the GEO/SRA dataset ID.</p>
<p>For detailed information about each genome browser track, please see the <a href="https://github.com/pombase/pombase-config/raw/master/website/jbrowse_track_metadata.csv">JBrowse configuration table</a>. This file is a spreadsheet in comma separated value format. The columns are described on <a routerLink="/documentation/data-submission-form-for-HTP-sequence-linked-data">the dataset submission page</a>.</p>
<hr />
<h3 id="mondo-slim">Mondo slim</h3>
<p><a href="https://www.pombase.org/latest_release/human_disease_annotation/pombe_mondo_disease_slim_terms.tsv">Current fission yeast Mondo Disease Ontology slim IDs and term names</a></p>
<hr />
<h3 id="curated-inventories">Curated inventories</h3>
<ul>
<li><a routerLink="/browse-curation/dna-binding-sites">DNA binding sites</a></li>
<li><a routerLink="/browse-curation/drugs-known-pombe-targets">Drug targets</a></li>
</ul>
<hr />
<p><strong>Note:</strong> <em>S. pombe</em> files are no longer available from the old “pombase” FTP site within the EBI domain. If you have a link that contains <strong>ftp.ebi.ac.uk</strong>, please check these pages for a link using <strong>https://www.pombase.org/</strong>. Please <a href="mailto:helpdesk%7B%7B%22@%22%7D%7Dpombase.org">contact the PomBase curators</a> if you need help finding a file or directory.</p>

  </div>
  </div>
</div>
<div *ngIf="section == 'documentation'">
<app-page-contents-menu title="Documentation" titleRoute="/documentation">

<div class="left-menu-part left-menu-item"><span>General</span></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/getting-started">Getting started</a></div>
<div class="left-menu-part left-menu-item"><span>Gene pages</span></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-basic-information">Basic information</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-gene-ontology">Gene Ontology</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-phenotypes">Phenotypes</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-target">Target of</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-protein-features">Protein features</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-transcript">Transcript</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-modifications">Modifications</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-gene-expression">Gene expression</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/genetic-and-physical-interactions">Genetic and physical interactions</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/orthologs">Ortholog curation</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/taxonomic-conservation">Taxonomic conservation</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/disease-association">Disease association</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-sequence">Sequence</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/controlled-curation">Controlled curation</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-external-references">External references</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-literature">Literature</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/gocams">GO-CAM pathways</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/pombase-go-slim-documentation">GO Slim</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/JBrowse_quick_start">PomBase JBrowse</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/quick-little-tool">QuiLT visualisation</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/ontology-term-page">Ontology term pages</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/publication-page">Publication pages</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/genotype-page">Genotype pages</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/browse-curation/fission-yeast-phenotype-ontology">Fission Yeast Phenotype Ontology</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/simple-search-documentation">Simple search</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/advanced-search">Advanced search</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/id-mapper">Identifier mapper</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/motif-search">Motif search</a></div>
<div class="left-menu-part left-menu-item"><span>Data submission file formats</span></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/phenotype-data-bulk-upload-format">Phenotype data</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/modification-data-bulk-upload-format">Modification data</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/qualitative-gene-expression-data-bulk-upload-format">Qualitative gene expression data</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/quantitative-gene-expression-data-bulk-upload-format">Quantitative gene expression data</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/data-submission-form-for-HTP-sequence-linked-data">HTP data for browser tracks</a></div>
<div class="left-menu-part left-menu-item"><span>Frequently Asked Questions (FAQ)</span></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/faq/community">Community</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/faq/data-submission-and-formats">Data submission and formats</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/faq/finding-data">Finding data</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/faq/gene-page">Gene page</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/faq/genome-browser">Genome browser</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/faq/genome-statistics-and-lists">Genome statistics and lists</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/faq/orthology">Orthology</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/faq/tools-and-resources">Tools and resources</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/faq/using-ontologies">Using ontologies</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'go-cam-pathway-models'">
<h2 id="go-cam-curated-biological-pathways">GO-CAM curated biological pathways</h2>
<p>GO-CAMs, or Gene Ontology Causal Activity Models, are a framework developed within the Gene Ontology (GO) project to represent complex biological pathways in a structured, computable format. Unlike traditional GO annotations, GO-CAM curation incorporates causal relationships between molecular activities and connects them to biological processes and cellular components. This approach provides a more detailed depiction of how gene products interact and function within biological systems. GO-CAMs specify entities like genes and proteins, their associated activities, and the causal links between them. These models are visualized as graphs, with nodes representing molecular activities and edges illustrating causal relationships, enabling a comprehensive understanding of interactions and pathways in biological processes.</p>
<p>At PomBase, we have begun representing biological pathways as GO-CAM models and displaying them on the relevant gene and process pages.</p>
<p>For further information see the <a href="https://geneontology.org/docs/gocam-overview/">Gene Ontology GO-CAM overview</a>.</p>
<h3 id="missing-activities">Missing activities</h3>
<p>We maintain a list of activities known or suspected to occur in fission yeast but not yet associated with a gene product. These “pathway holes” have been identified through GO-CAM causal model curation.</p>
<p><a routerLink="/gocam/missing-activities">Missing <em>S. pombe</em> activities</a></p>
<h3 id="available-go-cam-pathways">Available GO-CAM pathways</h3>
<p><app-go-cam-table></app-go-cam-table></p>

  </div>
  <div *ngIf="pageName == 'gocams'">
<h2 id="pombase-go-cam-pathways">PomBase GO-CAM pathways</h2>
<p>GO-CAMs (Gene Ontology Causal Activity Models) are a framework developed by the Gene Ontology (GO) project to represent complex biological pathways in a structured, computable format. They model how molecular activities, enabled by gene products or complexes, are causally linked to form larger biological processes.</p>
<div class="docs-image-flex">
<p><img src="assets/gocam-docs-example-model.png" class="screenshot docs-left-float" loading="lazy" alt="GO-CAM pathway" /></p>
<div>
<p>Creating a comprehensive GO-CAM representation of a eukaryotic cell is an ambitious and ongoing project. We therefore welcome community input to help improve, expand, or correct the current pathway models. If you have subject area knowledge on a specific pathway also please consider contacting us to provide feedback on accuracy and completeness.</p>
<p>Example pathway: <a routerLink="/gocam/pombase-view/docs/678073a900002931">fatty acid biosynthetic process (GO:0006633)</a></p>
<p>Visit the <a routerLink="/gocam/model-list">GO-CAM pathway list</a> or the <a routerLink="/gocam">main PomBase GO-CAM page</a>.</p>
</div>
</div>
<hr />
<h3 id="what-do-go-cam-pathways-represent">What Do GO-CAM Pathways Represent?</h3>
<p>Each GO-CAM pathway provides a structured view of curated biological knowledge, where: Activities (GO Molecular Functions) performed by enablers (gene products or protein complexes) are connected causally through <a href="https://wiki.geneontology.org/Annotation_Relations#GO-CAM:_Causal_Relations">defined relationships</a>.</p>
<p>Activities also have curated connections to:</p>
<ul>
<li>The GO biological process or pathway they are “part of”.</li>
<li>The GO cellular component where the activity “occurs in”.</li>
<li>Chemical inputs and outputs (with associated locations), or target genes and complexes.</li>
</ul>
<p>All pathway models are manually curated, and due to the extensive GO curation coverage at PomBase, most use pre-existing GO annotations and their associated evidence.</p>
<p>For more information on the GO-CAM concept, visit the <a href="https://geneontology.org/docs/gocam-overview/">Gene Ontology Consortium’s GO-CAM overview</a>.</p>
<hr />
<h3 id="overview-of-available-go-cam-pathways-and-project-scope-intended-and-completed-work">Overview of Available GO-CAM Pathways and Project Scope: Intended and Completed Work</h3>
<p>GO-CAMs, or Gene Ontology Causal Activity Models, are a framework developed within the Gene Ontology (GO) project to represent complex biological pathways in a structured, computable format.</p>
<hr />
<h3 id="summary-map">Summary Map</h3>
<div class="docs-image-flex">
<p><img src="assets/gocam-docs-summary-map.png" class="screenshot docs-left-float" loading="lazy" alt="Summary Map" /></p>
<div>
<p>The <strong>Summary Map</strong> provides a high-level, visual overview of how individual GO-CAM pathways are causally connected, without showing the internal structure of particular pathway.</p>
<p>Clicking on a pathway will display its title in the right-hand panel. Click the title to open a new tab/window to view the pathway in detail.</p>
<p>Multiple pathways can be selected by shift-clicking or shift-click and drag in the map. When multiple pathways are selected, click on “View joined models” to see connections between the models in detail.</p>
<h4 id="the-all-pathways-summary-map">The “All Pathways Summary Map”</h4>
<p>Displays a connected Summary Map, but also includes the set of pathway models that are not yet causally linked to any other pathway.</p>
<p><a routerLink="/gocam/summary/all">View the Summary Map</a></p>
<h4 class="clear-float" id="the-connected-summary-map">The “Connected Summary Map”</h4>
<p>Displays curated causal connections between GO-CAM pathways (inter-pathway connections)</p>
<p><a routerLink="/gocam/summary/connected">View the Connected Summary Map</a></p>
<p>The maps are fully interactive:</p>
<ul>
<li>Click on a pathway to view the title of the GO-CAM in the right-hand panel</li>
<li>Click on the pathway title to open a new tab/window to visit a detailed view</li>
<li>Multiple pathways can be selected by shift-clicking or shift-click and drag in the map
<ul>
<li>When multiple pathways are selected, click on “View joined models” to see connections between the models in detail</li>
</ul></li>
<li>Zoom, drag, and rearrange nodes to explore the layout</li>
</ul>
</div>
</div>
<hr />
<h3 id="mega-model">Mega Model</h3>
<div class="docs-image-flex">
<p><img src="assets/gocam-docs-mega-model.png" class="screenshot docs-left-float" loading="lazy" alt="Mega Model" /></p>
<div>
<p>The <strong>Mega Model</strong> view displays GO-CAMs in full detail, showing all intra- and inter-pathway connections. Initially, chemical inputs/outputs and target genes are hidden to reduce visual complexity — but you can toggle them on or off as needed.</p>
<h4 id="the-all-pathways-mega-model">The “All Pathways Mega Model”</h4>
<p>Displays all curated GO-CAM pathway models in full detail:</p>
<ul>
<li>Each pathway is enclosed by a boundary (by default)</li>
<li>All intra- and inter-pathway connections are shown</li>
<li>Unconnected pathways are also included</li>
</ul>
<p><a routerLink="/gocam/mega-model/all">View the All Pathways Mega-model</a>.</p>
<h4 class="clear-float" id="the-connected-pathways-mega-model">The “Connected Pathways Mega Model”</h4>
<p>Displays only those GO-CAM pathway models that have inter-pathway connections, omitting unlinked pathways.</p>
<p><a routerLink="/gocam/mega-model/connected">View the Connected Pathways Mega-model</a>.</p>
</div>
</div>
<hr />
<h3 id="notes-on-pathway-completeness">Notes on Pathway Completeness</h3>
<p>Not all chemical intermediates are curated into pathways. However, for all catalytic steps, full reaction details can be found via links on the associated gene page, under the GO Molecular Function section (via Rhea reaction references). Currency metabolites (e.g., ATP, H₂O) are excluded to simplify pathways. Some known target genes are not yet included in pathways. Many inter-pathway connections are still being identified.</p>
<hr />
<h3 id="missing-activities">Missing activities</h3>
<p>We maintain a list of activities known or suspected to occur in fission yeast that are not yet associated with any specific gene product. These pathway gaps were identified through GO-CAM curation and highlight areas for further research and annotation.</p>
<p><a routerLink="/gocam/missing-activities">Missing <em>S. pombe</em> activities</a></p>
<hr />
<h3 id="pathway-joining-activities">Pathway joining activities</h3>
<p>A curated list of pathway models that share overlapping activities. This list includes: - The specific activities used to link pathways - The joined pathways - Links to views of the joined pathways</p>
<p><a routerLink="/gocam/connections">View pathway joining activities</a></p>

  </div>
  <div *ngIf="pageName == 'index'">
<h1 id="pombase-web-site-help-documentation">PomBase Web Site Help &amp; Documentation</h1>
<div class="doc-search" [class.doc-search-highlight]="highlightDocSearchBox">
<div class="doc-search-title">
Search documentation, news archive and FAQ:
</div>
<app-faceted-search [scope]="'doc'" [focusSearchBox]="highlightDocSearchBox"></app-faceted-search>
</div>
<h3 id="general">General</h3>
<ul>
<li><a routerLink="/documentation/getting-started">Getting started</a></li>
</ul>
<h3 id="datasets-and-download-links">Datasets and Download Links</h3>
<ul>
<li><a routerLink="/datasets/">Main datasets page</a></li>
</ul>
<h3 id="gene-pages">Gene Pages</h3>
<ul>
<li><a routerLink="/documentation/gene-page-basic-information">Basic information</a></li>
<li><a routerLink="/documentation/gene-page-gene-ontology">Gene Ontology</a></li>
<li><a routerLink="/documentation/gene-page-phenotypes">Phenotypes</a></li>
<li><a routerLink="/documentation/gene-page-target">Target of</a></li>
<li><a routerLink="/documentation/gene-page-transcript">Transcript</a></li>
<li><a routerLink="/documentation/gene-page-protein-features">Protein features</a></li>
<li><a routerLink="/documentation/gene-page-modifications">Modifications</a></li>
<li><a routerLink="/documentation/gene-page-gene-expression">Gene expression</a></li>
<li><a routerLink="/documentation/genetic-and-physical-interactions">Interactions</a></li>
<li><a routerLink="/documentation/orthologs">Orthologs</a> and paralogs</li>
<li><a routerLink="/documentation/gene-page-sequence">Sequence</a></li>
<li><a routerLink="/documentation/taxonomic-conservation">Taxonomic conservation</a></li>
<li><a routerLink="/documentation/disease-association">Disease association</a></li>
<li><a routerLink="/documentation/controlled-curation">Controlled curation</a></li>
<li><a routerLink="/documentation/gene-page-external-references">External references</a></li>
<li><a routerLink="/documentation/gene-page-literature">Literature</a></li>
</ul>
<h3 id="searching">Searching</h3>
<ul>
<li><a routerLink="/documentation/simple-search-documentation">Simple search</a></li>
<li><a routerLink="/documentation/advanced-search">Advanced search</a></li>
<li><a routerLink="/documentation/id-mapper">Identifier mapper</a></li>
<li><a routerLink="/documentation/motif-search">Peptide motif search</a></li>
</ul>
<h3 id="jbrowse-genome-browser">JBrowse Genome Browser</h3>
<ul>
<li><a routerLink="/documentation/JBrowse_quick_start">Getting started with PomBase JBrowse</a> - an introductory guide to browser navigation, track loading, metadata, etc. (by Antonia Lock)</li>
<li>Also see the General entry in the JBrowse Help menu</li>
<li>See “File Formats” below to submit data</li>
</ul>
<h3 id="quilt">QuiLT</h3>
<ul>
<li><a routerLink="/documentation/quick-little-tool">Documentation</a> for the Quick Little Tool for gene list visualisation</li>
</ul>
<h3 id="non-gene-sequence-features">Non-gene sequence features</h3>
<ul>
<li><a routerLink="/status/centromeres">Centromere clone information</a></li>
<li><a routerLink="/documentation/misc-sequence-features">Miscellaneous sequence feature table</a></li>
</ul>
<h3 id="other-pombase-pages">Other PomBase Pages</h3>
<ul>
<li>GO Slim <a routerLink="/documentation/pombase-go-slim-documentation">Documentation</a> and <a routerLink="/browse-curation/fission-yeast-go-slimming-tips">Usage tips</a></li>
<li><a routerLink="/documentation/ontology-term-page">Ontology term pages</a></li>
<li><a routerLink="/documentation/publication-page">Publication pages</a></li>
<li><a routerLink="/documentation/genotype-page">Genotype pages</a></li>
<li><a routerLink="/browse-curation/fission-yeast-phenotype-ontology">Fission Yeast Phenotype Ontology (FYPO) page</a></li>
<li><a routerLink="/documentation/high-confidence-physical-interaction-network">High Confidence Physical Interaction Network (HCPIN)</a></li>
<li><a routerLink="/about/version-history">Data versions</a></li>
<li><a routerLink="/documentation/pombase-references">PomBase references (PB_REFs)</a></li>
</ul>
<h3 id="how-to-cite-pombase">How to Cite PomBase</h3>
<ul>
<li>See the <a routerLink="/about/citing-pombase">Citing PomBase</a> page</li>
</ul>
<h3 id="file-formats-for-data-submission">File Formats for Data Submission</h3>
<p>PomBase accepts batch submissions of certain types of data that appear on PomBase gene pages. For these data types, we use dedicated PomBase-specific formats:</p>
<ul>
<li><a routerLink="/documentation/phenotype-data-bulk-upload-format">Phenotypes</a></li>
<li><a routerLink="/documentation/modification-data-bulk-upload-format">Modifications</a></li>
<li><a routerLink="/documentation/qualitative-gene-expression-data-bulk-upload-format">Qualitative gene expression</a></li>
<li><a routerLink="/documentation/quantitative-gene-expression-data-bulk-upload-format">Quantitative gene expression</a></li>
<li><a routerLink="/documentation/data-submission-form-for-HTP-sequence-linked-data">HTP data for JBrowse tracks</a></li>
</ul>
<p>For data that can be connected with sequence features or coordinates, and displayed as tracks in the genome browser, see the <a href="faq/what-file-formats-can-i-use-submit-high-throughput-data">data format FAQ</a> and further details linked there.</p>
<h3 id="linking-to-and-from-pombase">Linking to and from PomBase</h3>
<ul>
<li><p>Linking to PomBase: To link to any PomBase gene page, use the systematic ID for the gene in a URL with the syntax “https://www.pombase.org/gene/[systematic ID]”. For example, <a routerLink="/gene/SPBC11B10.09">https://www.pombase.org/gene/SPBC11B10.09</a> links to the gene page for <a routerLink="/gene/SPBC11B10.09">cdc2</a>.</p></li>
<li><p>Linking from PomBase to external resources: We can provide links from PomBase gene pages to gene- or gene product-specific <em>S. pombe</em> data for any resource that uses URLs with PomBase systematic IDs. Please contact the PomBase Curators for more information.</p></li>
</ul>
<h3 id="comparative-genomics">Comparative Genomics</h3>
<p>We plan to add specific documentation about comparative genomics using <em>S. pombe</em> here. In the meantime, you can look at the documentation for <a href="http://www.ensembl.org/info/genome/compara/index.html">Ensembl Compara</a>.</p>
<h3 id="more-information">More information …</h3>
<p>For help with anything in PomBase not covered here, you can <a href="mailto:helpdesk@pombase.org">contact the curators</a>.</p>

  </div>
  </div>
</div>
<div *ngIf="section == 'downloads'">
<app-page-contents-menu title="Downloads" >

<div class="left-menu-part left-menu-item"><a routerLink="/datasets">Datasets</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/downloads/genome-datasets">Genome datasets</a></div>
<div class="left-menu-part left-sub-menu-item"><span>Annotation datasets</span></div>
<div class="left-menu-part left-sub-sub-menu-item"><a routerLink="/downloads/go-annotations">GO annotations</a></div>
<div class="left-menu-part left-sub-sub-menu-item"><a routerLink="/downloads/phenotype-annotations">Phenotype annotations</a></div>
<div class="left-menu-part left-sub-sub-menu-item"><a routerLink="/downloads/modifications">Modifications</a></div>
<div class="left-menu-part left-sub-sub-menu-item"><a href="/latest_release/curated_orthologs/">Orthologs</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/downloads/protein-datasets">Protein datasets</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/downloads/names-and-identifiers">Names and IDs</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documents">Documents</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/downloads/chado-database-dumps">Chado database dumps</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'go-annotations'">
<h2 id="go-annotations">GO annotations</h2>
<p><em>S. pombe</em> GO annotations are available as tab-delimited files in <a href="http://geneontology.org/docs/go-annotation-file-gaf-format-2.2/">GAF 2.2 format</a>. The files include annotations made by manual literature curation, annotations inferred from keyword mappings based on curated descriptions, and annotations shared by the UniProt <a href="http://www.ebi.ac.uk/GOA">GOA</a> team.</p>
<p>To cite the fission yeast GO data, please see <a routerLink="/about/citing-pombase">Citing PomBase</a>.</p>
<h3 id="annotation-files">Annotation files</h3>
<p>Files from the <a href="https://www.pombase.org/latest_release/gene_ontology/">latest monthly release</a>:</p>
<h4 id="go-association-file-gaf">GO Association File (GAF)</h4>
<p>All current <em>S. pombe</em> GO annotations in tab-delimted <a href="http://geneontology.org/docs/go-annotation-file-gaf-format-2.2/">GAF 2.2 format</a>:</p>
<ul>
<li><a href="https://www.pombase.org/latest_release/gene_ontology/gene_ontology_annotation.gaf.tsv">gene_ontology_annotation.gaf.tsv</a></li>
</ul>
<h4 id="gpadgpi">GPAD/GPI</h4>
<p><em>S. pombe</em> GO annotation in <a href="https://github.com/geneontology/go-annotation/blob/master/specs/gpad-gpi-2-0.md">GPAD/GPI v2.0 format</a>:</p>
<ul>
<li><a href="https://www.pombase.org/latest_release/gene_ontology/gene_product_information_taxonid_4896.tsv">Gene Product Information (GPI)</a> - details of <em>S. pombe</em> gene products</li>
<li><a href="https://www.pombase.org/latest_release/gene_ontology/gene_product_annotation_data_taxonid_4896.tsv">Gene Product Annotation Data (GPAD)</a> - annotation for the genes in the GPI file</li>
</ul>
<h3 id="protein-complexes">Protein complexes</h3>
<p>Also see the <a href="https://www.pombase.org/latest_release/macromolecular_complexes/">list of protein complexes</a>, which uses GO macromolecular complex terms and IDs.</p>
<h3 id="notes">Notes</h3>
<p>The contents of the files downloadable from PomBase may differ from files available elsewhere (e.g. see <a routerLink="/faq/why-are-go-annotations-different-between-pombase-and-uniprot-goa">this FAQ</a>), and will not include annotations inferred by transitivity (see <a routerLink="/faq/why-are-go-terms-missing-downloadable-annotation-file">this FAQ</a>).</p>
<p>Previous versions of the <em>S. pombe</em> GO annotation file can be retrieved from the <a href="https://www.pombase.org/data/annotations/Gene_ontology/OLD/">archive directory</a>. Note that files produced before March 2021 are only available in GAF 2.1 format.</p>

  </div>
  <div *ngIf="pageName == 'index'">
<h2 id="downloads">Downloads</h2>
<ul>
<li><a routerLink="/datasets">Datasets</a>: Genome sequence and features, annotations, etc.</li>
<li><a routerLink="/downloads/chado-database-dumps">Chado database dumps</a>: Dumps of curated data stored in the PomBase Chado database.</li>
</ul>

  </div>
  </div>
</div>
<div *ngIf="section == 'faq'">
<app-page-contents-menu title="FAQ" titleRoute="/faq">

<div class="left-menu-part left-menu-item"><a routerLink="/faq/finding-data">Finding data</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/faq/genome-statistics-and-lists">Genome statistics and lists</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/faq/tools-and-resources">Tools and resources</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/faq/using-ontologies">Using ontologies</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'finding-data'">
<h2 id="finding-data">Finding data</h2>

  </div>
  <div *ngIf="pageName == 'genome-statistics-and-lists'">
<h2 id="genome-statistics-and-lists">Genome statistics and lists</h2>

  </div>
  <div *ngIf="pageName == 'index'">
<h1 id="frequently-asked-questions">Frequently asked questions</h1>
<p>Welcome to the PomBase FAQ. Choose a link in the left-hand menu to see the questions and answers in that category.</p>
<p>If you can’t find the information you need here or in the PomBase documentation, please contact the <a href="mailto:helpdesk@pombase.org">Helpdesk</a>.</p>

  </div>
  <div *ngIf="pageName == 'tools-and-resources'">
<h2 id="tools-and-resources">Tools and resources</h2>

  </div>
  <div *ngIf="pageName == 'using-ontologies'">
<h2 id="using-ontologies">Using ontologies</h2>

  </div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-significant-shared-go-annotations-genes-list', 'How can I find significant shared GO annotations for genes in a list?')" *ngIf="pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-significant-shared-go-annotations-genes-list' || pageName == 'all-faqs' || section == 'faq' && pageName == 'index'">
<h3 id="how-can-i-find-significant-shared-go-annotations-genes-list">How can I find significant shared GO annotations for genes in a list?</h3>
<!-- pombase_categories: Finding data,Tools and resources,Using ontologies -->
<p>GO term enrichment identifies GO terms that are significantly overrepresented (or underrepresented) among a set of genes.</p>
<p>At present PomBase does not have its own GO enrichment tool. We recommend using the <a href="http://go.princeton.edu/cgi-bin/GOTermFinder">Generic GO Term Finder at Princeton</a>, because it offers a simple interface and up-to-date ontology and annotation data, including the current PomBase GO annotation dataset (you can upload your own background set, GO annotation file, or both). The results are provided in an tabular format. You can also use the GO Term Finder to retrieve all annotations for your gene list by setting the p-value to 1.</p>
<p>Before you perform an enrichment analysis, we recommend that you do a GO slim analysis of your gene list, for a broad overview of the annotation set (for more information, see the <a routerLink="/documentation/pombase-go-slim-documentation">Fission Yeast GO slim documentation</a> page and <a routerLink="/faq/how-can-i-use-go-slims-s.-pombe">FAQ</a>). This will enable you to focus on. the most important terms in your enrichment.</p>
<p>You can slim your gene lists for any GO aspect (MF, BP or CC) using the PomBase “Advanced search”:</p>
<ol type="1">
<li>Upload your list</li>
<li>Click on the number of results in column 1 to access the complete list</li>
<li>Use the “slim with” option in the menu to slim your gene set</li>
</ol>
<p>A few other enrichment tools are described on the GO Consortium’s <a href="https://geneontology.github.io/docs/go-enrichment-analysis/">GO Enrichment Analysis</a> page. If you choose one of these, we recommend that you only use an enrichment tool that allows you to upload a background set representing the genes used in your experiment. Even if you regard the whole genome as the relevant background, it is important to specify the background gene set explicitly to obtain meaningful results, especially if you have data only for protein-coding genes. For example, annotated tRNA genes can obscure enrichment of protein-coding genes annotated to translation.</p>
<p>For any GO analysis, we strongly recommend that you describe your approach fully in methods, and include the release details (number and/or date) for PomBase and the GO terms and annotations you use.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-search-or-browse-go-annotations', 'How can I search or browse GO annotations?')" *ngIf="pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-search-or-browse-go-annotations' || pageName == 'all-faqs' || section == 'faq' && pageName == 'index'">
<h3 id="how-can-i-search-or-browse-go-annotations">How can I search or browse GO annotations?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>You can search for GO terms by name or ID in the PomBase <a href="/query">advanced search</a>, and retrieve a list of all genes annotated to the term and its descendants via the relations <em>is_a</em>, <em>part_of</em>, <em>regulates</em>, <em>positively_regulates,</em> and <em>negatively_regulates</em>. For example, a search for “cytokinesis” will include genes annotated to “regulation of cytokinesis”. (See the GO documentation on <a href="http://geneontology.org/docs/ontology-relations/">Ontology Structure and Ontology Relations</a> for more information.)</p>
<p><em>S. pombe</em> GO annotations are also available in browsers that use the GO repository, notably <a href="http://amigo.geneontology.org/">AmiGO</a> and <a href="http://www.ebi.ac.uk/QuickGO/">QuickGO</a>. Both browsers have extensive documentation available:</p>
<ul>
<li><a href="http://wiki.geneontology.org/index.php/AmiGO_2_Manual:_Overview">AmiGO Manual</a></li>
<li><a href="http://www.ebi.ac.uk/QuickGO/help">QuickGO Help page</a></li>
</ul>
<p>Hint: to find <em>S. pombe</em> annotations, use Organism: Schizosaccharomyces pombe, Taxon: 4896 (<em>Schizosaccharomyces pombe</em>) or Source: PomBase. You can download the results in GAF format.</p>
<p>In PomBase, GO term names and IDs on gene pages link to ontology term pages for GO terms, which in turn offer links to AmiGO, QuickGO and BioPortal.</p>

</div>
<div (click)="faq_navigate($event, '/faq/what-does-characterisation-status-mean-gene', 'What does the &quot;characterisation status&quot; mean for a gene?')" *ngIf="pageName == 'genome-statistics-and-lists' || pageName == 'what-does-characterisation-status-mean-gene' || pageName == 'all-faqs' || section == 'faq' && pageName == 'index'">
<h3 id="what-does-characterisation-status-mean-gene">What does the “characterisation status” mean for a gene?</h3>
<!-- pombase_categories: Genome statistics and lists -->
<p>Each gene is assigned exactly one characterisation status that reflects how much is known about the gene, whether it is conserved, etc. Specific status descriptions:</p>
<ul>
<li><strong>Published</strong>: Completely or partially characterised in a small scale experiment, with some published information about the biological role (corresponding to any of the <a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">fission yeast GO biological process slim</a> biological process terms)</li>
<li><strong>Biological role inferred</strong>: A biological role (as above, a <a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">fission yeast GO biological process slim</a> term) is inferred from homology to an experimentally characterised gene product</li>
<li><strong>Conserved unknown</strong>: Conserved outside the <em>Schizosaccharomyces</em>, but nothing known about the biological role in any organism</li>
<li><strong><em>Schizosaccharomyces</em> specific protein, uncharacterised</strong>: Unpublished and found only in fission yeast (<em>S. pombe</em>, <em>S. octosporus</em>, <em>S. japonicus</em>, <em>S. cryophilus</em>); nothing known about biological role. May be single copy or a member of a multi-member family.</li>
<li><strong><em>S. pombe</em> specific protein, uncharacterised</strong>: Unpublished and found only in <em>S. pombe</em> (not detected in other <em>Schizosaccharomyces</em> species); nothing known about biological role</li>
<li><strong>Transposon</strong>: Known or predicted transposable element</li>
<li><strong>Dubious</strong>: Unlikely to be protein coding</li>
</ul>
<p>A current summary of <a routerLink="/status/gene-characterisation">gene characterisation status for the <em>S. pombe</em> genome</a> is available.</p>
<p>You can also retrieve current lists of genes with each characterisation status using the <a routerLink="/query">advanced search</a>. Select the Characterisation status query, then choose a status from the pulldown menu, and submit.</p>

</div>
<div (click)="faq_navigate($event, '/faq/why-are-go-annotations-different-between-pombase-and-uniprot-goa', 'Why are GO annotations different between PomBase and UniProt/GOA?')" *ngIf="pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'why-are-go-annotations-different-between-pombase-and-uniprot-goa' || pageName == 'all-faqs' || section == 'faq' && pageName == 'index'">
<h3 id="why-are-go-annotations-different-between-pombase-and-uniprot-goa">Why are GO annotations different between PomBase and UniProt/GOA?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>The GO annotations available from PomBase (gene pages, advanced search, etc.) and the GO Consortium site (AmiGO; GO downloads) differ from those available from the UniProt GOA site (including QuickGO) for three main reasons:</p>
<ol type="1">
<li>RNA - PomBase provides GO annotations for functional RNAs (e.g. rRNA, tRNA, snRNA), but at present the UniProt GOA dataset only includes annotations for protein-coding genes.</li>
<li>Time lag - <em>S. pombe</em> GO data are updated at the same time on the PomBase and GO Consortium sites, but the UniProt GOA site may be up to a few weeks behind.</li>
<li>Filtering - PomBase does not include automated annotations that are redundant with manual annotations (contact the <a href="mailto:helpdesk@pombase.org">helpdesk</a> for further details). The GO Consortium site uses the same filtered annotation dataset as PomBase, whereas the UniProt GOA site includes the automated annotations.</li>
</ol>

</div>
  </div>
</div>
<div *ngIf="section == 'help'">
<app-page-contents-menu title="Help" titleRoute="/help">

<div class="left-menu-part left-menu-item"><a routerLink="/faq">Frequently Asked Questions (FAQ)</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation">PomBase Documentation</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/about">Contact Curators</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'index'">
<h1 id="help">Help</h1>
<h3 id="frequently-asked-questions-faq"><a routerLink="/faq">Frequently Asked Questions (FAQ)</a></h3>
<h3 id="pombase-documentation"><a routerLink="/documentation">PomBase Documentation</a></h3>
<h3 id="contact-curators"><a routerLink="/about">Contact Curators</a></h3>

  </div>
  </div>
</div>
<div *ngIf="section == 'news'">
<app-page-contents-menu title="News" titleRoute="/news">

<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-interactive-views-exploring-go-cam-pathways'" [pageScrollAdjustHash]="true" href="/news#new-interactive-views-exploring-go-cam-pathways">New Interactive Views for Exploring GO-CAM Pathways</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#jbrowse-version-2-now-available-on-pombase'" [pageScrollAdjustHash]="true" href="/news#jbrowse-version-2-now-available-on-pombase">JBrowse version 2 is now available on PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-expands-go-cam-pathway-models-fission-yeast'" [pageScrollAdjustHash]="true" href="/news#pombase-expands-go-cam-pathway-models-fission-yeast">PomBase Expands GO-CAM Pathway Models for Fission Yeast</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#the-transcription-factor-dataset-skribbe-soneson-stadler-et-al-.-now-available-pombase'" [pageScrollAdjustHash]="true" href="/news#the-transcription-factor-dataset-skribbe-soneson-stadler-et-al-.-now-available-pombase">The Transcription Factor Dataset from Skribbe, Soneson, Stadler *et al*. is now available in PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#interpro-version-104.0'" [pageScrollAdjustHash]="true" href="/news#interpro-version-104.0">InterPro Version 104.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#the-human-functionome-paper-published'" [pageScrollAdjustHash]="true" href="/news#the-human-functionome-paper-published">The human functionome paper published</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-missing-activity-list-list-potential-pathway-holes'" [pageScrollAdjustHash]="true" href="/news#new-missing-activity-list-list-potential-pathway-holes">NEW: The Missing activity list - A list of potential "pathway holes"</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-global-core-biodata-resource-week'" [pageScrollAdjustHash]="true" href="/news#pombase-global-core-biodata-resource-week">PomBase is Global Core BioData Resource of the week</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-february-19th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-february-19th">Next pombeTalks: Wednesday February 19th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-curated-pathways-using-go-cam-pathway-modelling'" [pageScrollAdjustHash]="true" href="/news#new-curated-pathways-using-go-cam-pathway-modelling">NEW: Curated pathways using GO-CAM pathway modelling</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-january-15th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-january-15th">Next pombeTalks: Wednesday January 15th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-december-11th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-december-11th">Next pombeTalks: Wednesday December 11th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#discovery-new-s.-pombe-orthologs-human-and-s.-cerevisiae-proteins-using-pfam-n'" [pageScrollAdjustHash]="true" href="/news#discovery-new-s.-pombe-orthologs-human-and-s.-cerevisiae-proteins-using-pfam-n">Discovery of new S. pombe orthologs of human and S. cerevisiae proteins using Pfam-N</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-november-20th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-november-20th">Next pombeTalks: Wednesday November 20th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#mobidb-disorder-predictions'" [pageScrollAdjustHash]="true" href="/news#mobidb-disorder-predictions">MobiDB disorder predictions</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#additional-protein-features-imported-uniprotkb'" [pageScrollAdjustHash]="true" href="/news#additional-protein-features-imported-uniprotkb">Additional protein features imported from UniProtKB</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#changes-protein-features-view-on-gene-pages'" [pageScrollAdjustHash]="true" href="/news#changes-protein-features-view-on-gene-pages">Changes to protein features view on gene pages</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-october-16th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-october-16th">Next pombeTalks: Wednesday October 16th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-september-18th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-september-18th">Next pombeTalks: September 18th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-updated-use-interpro-version-101.0'" [pageScrollAdjustHash]="true" href="/news#pombase-updated-use-interpro-version-101.0">PomBase updated to use InterPro Version 101.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-season-5-starting-september-18th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-season-5-starting-september-18th">pombeTalks Season 5 starting September 18th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-human-and-s.-cerevisiae-orthologs'" [pageScrollAdjustHash]="true" href="/news#new-human-and-s.-cerevisiae-orthologs">New human and *S. cerevisiae* orthologs</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#unknowns-coverage-update'" [pageScrollAdjustHash]="true" href="/news#unknowns-coverage-update">"Unknowns" coverage update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#rna-secondary-structure-predictions-on-gene-pages'" [pageScrollAdjustHash]="true" href="/news#rna-secondary-structure-predictions-on-gene-pages">RNA secondary structure predictions on gene pages</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#human-disease-association-changes-monarch'" [pageScrollAdjustHash]="true" href="/news#human-disease-association-changes-monarch">Human disease association changes - Monarch</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-now-includes-data-interpro-v100.0'" [pageScrollAdjustHash]="true" href="/news#pombase-now-includes-data-interpro-v100.0">PomBase now includes data from InterPro v100.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-june-19th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-june-19th">Next pombeTalks: Wednesday, June 19th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-may-15th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-may-15th">Next pombeTalks: Wednesday, May 15th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-updated-use-interpro-version-99.0'" [pageScrollAdjustHash]="true" href="/news#pombase-updated-use-interpro-version-99.0">PomBase updated to use InterPro Version 99.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#news-pombetalks-wednesday-april-17th'" [pageScrollAdjustHash]="true" href="/news#news-pombetalks-wednesday-april-17th">News pombeTalks: Wednesday April 17th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-march-20th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-march-20th">Next pombeTalks: March 20th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-database-update-publication-pombase'" [pageScrollAdjustHash]="true" href="/news#new-database-update-publication-pombase">New database update publication from PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-february-21st'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-february-21st">Next pombeTalks: Wednesday, February 21st</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#viewing-protein-modifications'" [pageScrollAdjustHash]="true" href="/news#viewing-protein-modifications">Viewing protein modifications</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#access-target-genes-individual-gene-products'" [pageScrollAdjustHash]="true" href="/news#access-target-genes-individual-gene-products">Access target genes of individual gene products</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-january-17th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-january-17th">Next pombeTalks: Wednesday, January 17th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-joins-elixir-core-data-resources'" [pageScrollAdjustHash]="true" href="/news#pombase-joins-elixir-core-data-resources">PomBase joins ELIXIR Core Data Resources</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-december-13th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-december-13th">Next pombeTalks: Wednesday, December 13th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-updated-use-interpro-version-97.0'" [pageScrollAdjustHash]="true" href="/news#pombase-updated-use-interpro-version-97.0">PomBase updated to use InterPro Version 97.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-november-15th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-november-15th">Next pombeTalks: Wednesday, November 15th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-documentation-and-videos-canto'" [pageScrollAdjustHash]="true" href="/news#new-documentation-and-videos-canto">New documentation and videos for Canto</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-october-18th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-october-18th">Next pombeTalks: Wednesday, October 18th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#revised-fission-yeast-nomenclature-genes-and-alleles'" [pageScrollAdjustHash]="true" href="/news#revised-fission-yeast-nomenclature-genes-and-alleles">Revised fission yeast nomenclature for genes and alleles</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#accessing-list-genes-paper'" [pageScrollAdjustHash]="true" href="/news#accessing-list-genes-paper">Accessing the list of genes from a paper</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#protein-feature-viewer-added-gene-pages'" [pageScrollAdjustHash]="true" href="/news#protein-feature-viewer-added-gene-pages">Protein feature viewer added to gene pages</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#reaction-diagrams-on-go-molecular-function-term-pages'" [pageScrollAdjustHash]="true" href="/news#reaction-diagrams-on-go-molecular-function-term-pages">Reaction diagrams on GO molecular function term pages</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#curation-update-nonsense-mutation-merged-into-partial-amino-acid-deletion'" [pageScrollAdjustHash]="true" href="/news#curation-update-nonsense-mutation-merged-into-partial-amino-acid-deletion">Curation update - "nonsense mutation" merged into "partial amino acid deletion"</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#experimental-structures-pdb-on-gene-pages'" [pageScrollAdjustHash]="true" href="/news#experimental-structures-pdb-on-gene-pages">Experimental structures from PDB on gene pages</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#alphafold-protein-structure-on-gene-pages'" [pageScrollAdjustHash]="true" href="/news#alphafold-protein-structure-on-gene-pages">AlphaFold protein structure on gene pages</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#revised-canonical-5-utrs'" [pageScrollAdjustHash]="true" href="/news#revised-canonical-5-utrs">Revised canonical 5' UTRs</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#global-core-biodata-resource-status-pombase'" [pageScrollAdjustHash]="true" href="/news#global-core-biodata-resource-status-pombase">Global Core Biodata Resource status for PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-december-14th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-december-14th">Next PombeTalks: Wednesday, December 14th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#querying-by-rna-length-advanced-search-tool'" [pageScrollAdjustHash]="true" href="/news#querying-by-rna-length-advanced-search-tool">Querying by RNA length in the Advanced Search tool</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#genetic-interaction-annotation-model-updated'" [pageScrollAdjustHash]="true" href="/news#genetic-interaction-annotation-model-updated">Genetic Interaction annotation model updated</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-november-16th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-november-16th">Next pombeTalks: Wednesday, November 16th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#using-alphafold-models-discover-distant-human-and-budding-yeast-homologues'" [pageScrollAdjustHash]="true" href="/news#using-alphafold-models-discover-distant-human-and-budding-yeast-homologues">Using AlphaFold models to discover distant human and budding yeast homologues</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-october-19th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-october-19th">Next pombeTalks: Wednesday, October 19th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-fission-yeast-check-out-our-quick-start-guide-new-users'" [pageScrollAdjustHash]="true" href="/news#new-fission-yeast-check-out-our-quick-start-guide-new-users">New to fission yeast? check out our quick start guide for new users</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#id-77-new-disease-gene-associations'" [pageScrollAdjustHash]="true" href="/news#id-77-new-disease-gene-associations">77 new disease gene associations</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombemine-intermine-instance-s.-pombe'" [pageScrollAdjustHash]="true" href="/news#pombemine-intermine-instance-s.-pombe">PombeMine: an InterMine instance for *S. pombe*</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-september-21st'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-september-21st">Next pombeTalks: Wednesday, September  21st</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#protein-sequence-changes'" [pageScrollAdjustHash]="true" href="/news#protein-sequence-changes">Protein sequence changes</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-august-17th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-august-17th">Next pombeTalks: Wednesday, August 17th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#next-pombetalks-wednesday-july-20th'" [pageScrollAdjustHash]="true" href="/news#next-pombetalks-wednesday-july-20th">Next pombeTalks: Wednesday, July 20th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-wednesday-june-15th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-wednesday-june-15th">PombeTalks Wednesday June 15th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-may-18th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-may-18th">PombeTalks May 18th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-phenotype-slim'" [pageScrollAdjustHash]="true" href="/news#new-phenotype-slim">New phenotype slim</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#making-biological-knowledge-useful-humans-and-machines'" [pageScrollAdjustHash]="true" href="/news#making-biological-knowledge-useful-humans-and-machines">Making biological knowledge useful for humans and machines</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#the-fission-yeast-community-have-now-curated-over-1000-publications'" [pageScrollAdjustHash]="true" href="/news#the-fission-yeast-community-have-now-curated-over-1000-publications">The fission yeast community have now curated over 1000 publications</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-human-orthologs-including-3-mrp-complex-subunits'" [pageScrollAdjustHash]="true" href="/news#new-human-orthologs-including-3-mrp-complex-subunits">New human orthologs including 3 MRP complex subunits</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-now-uses-interpro-version-88.0'" [pageScrollAdjustHash]="true" href="/news#pombase-now-uses-interpro-version-88.0">PomBase now uses InterPro Version 88.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#tor-and-nutritional-phosphoproteome-dataset-loaded'" [pageScrollAdjustHash]="true" href="/news#tor-and-nutritional-phosphoproteome-dataset-loaded">TOR and nutritional phosphoproteome dataset loaded</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#an-additional-5775-novel-curated-lncrnas-atkinson-et-al-.'" [pageScrollAdjustHash]="true" href="/news#an-additional-5775-novel-curated-lncrnas-atkinson-et-al-.">An additional 5775 novel curated lncRNAs from Atkinson *et al*.</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-japonicusdb-publications-genetics-mod-reports-special-issue'" [pageScrollAdjustHash]="true" href="/news#pombase-japonicusdb-publications-genetics-mod-reports-special-issue">PomBase & JaponicusDB publications in the GENETICS MOD reports special issue</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#improved-options-filtering-annotations-by-cell-cycle-phase'" [pageScrollAdjustHash]="true" href="/news#improved-options-filtering-annotations-by-cell-cycle-phase">Improved options for filtering annotations by cell cycle phase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-now-uses-interpro-version-87.0'" [pageScrollAdjustHash]="true" href="/news#pombase-now-uses-interpro-version-87.0">PomBase now uses InterPro Version 87.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#coils-disorder-and-more-new-protein-feature-queries'" [pageScrollAdjustHash]="true" href="/news#coils-disorder-and-more-new-protein-feature-queries">Coils, disorder, and more: new protein feature queries</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-japonicusdb-preprints'" [pageScrollAdjustHash]="true" href="/news#pombase-japonicusdb-preprints">PomBase & JaponicusDB preprints</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#japonicusdb-new-fission-yeast-database'" [pageScrollAdjustHash]="true" href="/news#japonicusdb-new-fission-yeast-database">JaponicusDB: a new fission yeast database</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#improved-gene-disease-curation-over-1400-s.-pombe-genes'" [pageScrollAdjustHash]="true" href="/news#improved-gene-disease-curation-over-1400-s.-pombe-genes">Improved gene-disease curation: over 1400 S. pombe genes</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-august-4th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-august-4th">PombeTalks August 4th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#alphafold-links'" [pageScrollAdjustHash]="true" href="/news#alphafold-links">AlphaFold links</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-july-21st'" [pageScrollAdjustHash]="true" href="/news#pombetalks-july-21st">PombeTalks July 21st</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#kegg-pathway-links'" [pageScrollAdjustHash]="true" href="/news#kegg-pathway-links">KEGG pathway links</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-now-uses-interpro-version-86.0'" [pageScrollAdjustHash]="true" href="/news#pombase-now-uses-interpro-version-86.0">PomBase now uses InterPro Version 86.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-july-7th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-july-7th">PombeTalks July 7th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-june-23rd'" [pageScrollAdjustHash]="true" href="/news#pombetalks-june-23rd">PombeTalks June 23rd</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#violin-plots-gene-expression'" [pageScrollAdjustHash]="true" href="/news#violin-plots-gene-expression">Violin plots for gene expression</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-june-9th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-june-9th">PombeTalks June 9th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-may-26th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-may-26th">PombeTalks May 26th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-may-12th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-may-12th">PombeTalks May 12th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-diploid-genotype-phenotype-display'" [pageScrollAdjustHash]="true" href="/news#new-diploid-genotype-phenotype-display">New diploid genotype & phenotype display</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-april-28th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-april-28th">PombeTalks April 28th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-now-uses-interpro-version-85.0'" [pageScrollAdjustHash]="true" href="/news#pombase-now-uses-interpro-version-85.0">PomBase now uses InterPro Version 85.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-april-14th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-april-14th">PombeTalks April 14th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-genome-browser-data-gc-content'" [pageScrollAdjustHash]="true" href="/news#new-genome-browser-data-gc-content">New genome browser data: GC content</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#qualitative-gene-expression-annotation-canto'" [pageScrollAdjustHash]="true" href="/news#qualitative-gene-expression-annotation-canto">Qualitative gene expression annotation in Canto</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-protein-feature-display'" [pageScrollAdjustHash]="true" href="/news#new-protein-feature-display">New protein feature display</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-march-31st'" [pageScrollAdjustHash]="true" href="/news#pombetalks-march-31st">PombeTalks March 31st</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-march-17th-new-time'" [pageScrollAdjustHash]="true" href="/news#pombetalks-march-17th-new-time">PombeTalks March 17th - new time!</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-return-march-3rd'" [pageScrollAdjustHash]="true" href="/news#pombetalks-return-march-3rd">PombeTalks return March 3rd</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-identifier-mapper-now-available'" [pageScrollAdjustHash]="true" href="/news#pombase-identifier-mapper-now-available">PomBase identifier mapper now available</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-now-uses-interpro-version-83.0'" [pageScrollAdjustHash]="true" href="/news#pombase-now-uses-interpro-version-83.0">PomBase now uses InterPro Version 83.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#first-s.-pombe-micropublication-goes-live'" [pageScrollAdjustHash]="true" href="/news#first-s.-pombe-micropublication-goes-live">First S. pombe microPublication goes live</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-november-25th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-november-25th">PombeTalks November 25th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-november-11th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-november-11th">PombeTalks November 11th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#go-annotations-paint'" [pageScrollAdjustHash]="true" href="/news#go-annotations-paint">GO annotations from PAINT</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-october-28th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-october-28th">PombeTalks October 28th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-october-14th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-october-14th">PombeTalks October 14th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-september-30th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-september-30th">PombeTalks September 30th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-genome-browser-datasets-2020-09-17'" [pageScrollAdjustHash]="true" href="/news#new-genome-browser-datasets-2020-09-17">New genome browser datasets 2020-09-17</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-september-16th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-september-16th">PombeTalks September 16th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-dataset-hermes-transposon-insertions'" [pageScrollAdjustHash]="true" href="/news#new-dataset-hermes-transposon-insertions">New dataset: Hermes transposon insertions</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-genome-browser-datasets-2020-09-08'" [pageScrollAdjustHash]="true" href="/news#new-genome-browser-datasets-2020-09-08">New genome browser datasets 2020-09-08</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#submit-your-genome-browser-data'" [pageScrollAdjustHash]="true" href="/news#submit-your-genome-browser-data">Submit your genome browser data</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#published-go-term-matrix-annotation-qc'" [pageScrollAdjustHash]="true" href="/news#published-go-term-matrix-annotation-qc">Published: GO Term Matrix for annotation QC</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-september-2nd'" [pageScrollAdjustHash]="true" href="/news#pombetalks-september-2nd">PombeTalks September 2nd</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#id-11th-pombe-meeting-postponed-2022'" [pageScrollAdjustHash]="true" href="/news#id-11th-pombe-meeting-postponed-2022">11th Pombe meeting postponed to 2022</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-august-19th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-august-19th">PombeTalks August 19th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-august-5th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-august-5th">PombeTalks August 5th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#fission-yeast-micropublications'" [pageScrollAdjustHash]="true" href="/news#fission-yeast-micropublications">Fission yeast microPublications</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-july-22nd'" [pageScrollAdjustHash]="true" href="/news#pombetalks-july-22nd">PombeTalks July 22nd</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-july-8th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-july-8th">PombeTalks July 8th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-june-24th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-june-24th">PombeTalks June 24th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#editable-pombase-query-names'" [pageScrollAdjustHash]="true" href="/news#editable-pombase-query-names">Editable PomBase query names</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-postponed-june-17th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-postponed-june-17th">PombeTalks POSTPONED to June 17th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-may-27th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-may-27th">PombeTalks May 27th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-fission-yeast-go-slims'" [pageScrollAdjustHash]="true" href="/news#new-fission-yeast-go-slims">New fission yeast GO slims</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-ontology-slimming-options-advanced-search-results'" [pageScrollAdjustHash]="true" href="/news#new-ontology-slimming-options-advanced-search-results">New ontology slimming options for advanced search results</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-adopts-mondo-disease-gene-curation'" [pageScrollAdjustHash]="true" href="/news#pombase-adopts-mondo-disease-gene-curation">PomBase adopts MONDO for disease gene curation</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#improved-gene-disease-curation'" [pageScrollAdjustHash]="true" href="/news#improved-gene-disease-curation">Improved gene-disease curation</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#published-community-curation-pombase'" [pageScrollAdjustHash]="true" href="/news#published-community-curation-pombase">Published: Community curation in PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-may-13th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-may-13th">PombeTalks May 13th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#chromatin-silencing-ontology-amp-annotation-overhaul'" [pageScrollAdjustHash]="true" href="/news#chromatin-silencing-ontology-amp-annotation-overhaul">Chromatin silencing ontology &amp; annotation overhaul</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-now-uses-interpro-version-79.0'" [pageScrollAdjustHash]="true" href="/news#pombase-now-uses-interpro-version-79.0">PomBase now uses InterPro Version 79.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#mitochondrial-genome-update'" [pageScrollAdjustHash]="true" href="/news#mitochondrial-genome-update">Mitochondrial genome update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#midori-harris-receives-2020-biocuration-career-award'" [pageScrollAdjustHash]="true" href="/news#midori-harris-receives-2020-biocuration-career-award">Midori Harris receives 2020 Biocuration Career Award</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombetalks-start-on-april-29th'" [pageScrollAdjustHash]="true" href="/news#pombetalks-start-on-april-29th">PombeTalks start on April 29th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#angeli-update'" [pageScrollAdjustHash]="true" href="/news#angeli-update">AnGeLi update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-search-result-download-options'" [pageScrollAdjustHash]="true" href="/news#new-search-result-download-options">New search result download options</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-now-uses-interpro-version-77.0'" [pageScrollAdjustHash]="true" href="/news#pombase-now-uses-interpro-version-77.0">PomBase now uses InterPro Version 77.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#community-curation-response-rate-reaches-50'" [pageScrollAdjustHash]="true" href="/news#community-curation-response-rate-reaches-50">Community curation response rate reaches 50%</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#a-quarter-million-annotations'" [pageScrollAdjustHash]="true" href="/news#a-quarter-million-annotations">A quarter of a million annotations</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#query-phenotypes-conditions'" [pageScrollAdjustHash]="true" href="/news#query-phenotypes-conditions">Query phenotypes for conditions</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#levures-mod-les-et-outils-14th-international-conference'" [pageScrollAdjustHash]="true" href="/news#levures-mod-les-et-outils-14th-international-conference">Levures, Modèles et Outils 14th International Conference</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-now-elixir-node-service'" [pageScrollAdjustHash]="true" href="/news#pombase-now-elixir-node-service">PomBase is now an ELIXIR Node Service</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-vectors-simple-reliable-s.-pombe-molecular-biology'" [pageScrollAdjustHash]="true" href="/news#new-vectors-simple-reliable-s.-pombe-molecular-biology">New vectors for simple, reliable *S. pombe* molecular biology</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#unique-permanent-urls-search-results'" [pageScrollAdjustHash]="true" href="/news#unique-permanent-urls-search-results">Unique permanent URLs for search results</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#browser-tracks-now-loadable-publication-pages'" [pageScrollAdjustHash]="true" href="/news#browser-tracks-now-loadable-publication-pages">Browser tracks now loadable from publication pages</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-pombase-funding-wellcome-trust'" [pageScrollAdjustHash]="true" href="/news#new-pombase-funding-wellcome-trust">New PomBase funding from the Wellcome Trust</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-now-uses-interpro-version-76.0'" [pageScrollAdjustHash]="true" href="/news#pombase-now-uses-interpro-version-76.0">PomBase now uses InterPro Version 76.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#replication-origin-data-loaded-into-jbrowse'" [pageScrollAdjustHash]="true" href="/news#replication-origin-data-loaded-into-jbrowse">Replication origin data loaded into JBrowse</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#join-conversations-on-slack'" [pageScrollAdjustHash]="true" href="/news#join-conversations-on-slack">Join the conversations on Slack</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#fitness-landscape-fission-yeast-genome-dataset-loaded-into-jbrowse'" [pageScrollAdjustHash]="true" href="/news#fitness-landscape-fission-yeast-genome-dataset-loaded-into-jbrowse">"Fitness Landscape of the Fission Yeast Genome" dataset loaded into JBrowse</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#trna-metabolism-go-annotation-update'" [pageScrollAdjustHash]="true" href="/news#trna-metabolism-go-annotation-update">tRNA metabolism GO annotation update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#customisable-fasta-download'" [pageScrollAdjustHash]="true" href="/news#customisable-fasta-download">Customisable FASTA download</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-homology-modelling-and-ortholog-links-on-gene-pages'" [pageScrollAdjustHash]="true" href="/news#new-homology-modelling-and-ortholog-links-on-gene-pages">New homology modelling and ortholog links on gene pages</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-interpro-update'" [pageScrollAdjustHash]="true" href="/news#pombase-interpro-update">PomBase InterPro Update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#s.-pombe-included-gene-info-browser-extension'" [pageScrollAdjustHash]="true" href="/news#s.-pombe-included-gene-info-browser-extension">*S. pombe* included in Gene Info browser extension</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#integrated-motif-search'" [pageScrollAdjustHash]="true" href="/news#integrated-motif-search">Integrated motif search</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#icygmb-2019-registration-open'" [pageScrollAdjustHash]="true" href="/news#icygmb-2019-registration-open">ICYGMB 2019  - registration open</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#go-slim-any-s.-pombe-gene-list'" [pageScrollAdjustHash]="true" href="/news#go-slim-any-s.-pombe-gene-list">GO slim for any *S. pombe* gene list</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#south-eastern-regional-yeast-meeting-serym-registration-open'" [pageScrollAdjustHash]="true" href="/news#south-eastern-regional-yeast-meeting-serym-registration-open">South Eastern Regional Yeast Meeting (SERYM) - registration open</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#international-cell-cycle-meeting-registration-open'" [pageScrollAdjustHash]="true" href="/news#international-cell-cycle-meeting-registration-open">International Cell Cycle meeting - registration open</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#published-hidden-plain-sight-what-remains-be-discovered-eukaryotic-proteome'" [pageScrollAdjustHash]="true" href="/news#published-hidden-plain-sight-what-remains-be-discovered-eukaryotic-proteome">Published: Hidden in plain sight: What remains to be discovered in the eukaryotic proteome?</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombe-2019-registration-open'" [pageScrollAdjustHash]="true" href="/news#pombe-2019-registration-open">Pombe 2019 - registration open</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#val-wood-wins-biocuration-society-award'" [pageScrollAdjustHash]="true" href="/news#val-wood-wins-biocuration-society-award">Val Wood wins Biocuration society award</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#improved-disease-association-dataset-released'" [pageScrollAdjustHash]="true" href="/news#improved-disease-association-dataset-released">Improved disease association dataset released</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#mitochondrial-go-annotation-update'" [pageScrollAdjustHash]="true" href="/news#mitochondrial-go-annotation-update">Mitochondrial GO annotation update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-nucleosome-occupancy-maps-loaded'" [pageScrollAdjustHash]="true" href="/news#new-nucleosome-occupancy-maps-loaded">New nucleosome occupancy maps loaded</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#see-your-genes-quilt'" [pageScrollAdjustHash]="true" href="/news#see-your-genes-quilt">See your genes in a QuiLT</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#fission-yeast-transmembrane-transport-overhaul'" [pageScrollAdjustHash]="true" href="/news#fission-yeast-transmembrane-transport-overhaul">Fission yeast transmembrane transport overhaul</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#hidden-plain-sight-what-remains-be-discovered-eukaryotic-proteome'" [pageScrollAdjustHash]="true" href="/news#hidden-plain-sight-what-remains-be-discovered-eukaryotic-proteome">Hidden in plain sight: What remains to be discovered in the eukaryotic proteome?</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-your-pocket'" [pageScrollAdjustHash]="true" href="/news#pombase-your-pocket">PomBase in your pocket</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#celebrating-20-years-go'" [pageScrollAdjustHash]="true" href="/news#celebrating-20-years-go">Celebrating 20 years of GO</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#rna-central-and-pombase'" [pageScrollAdjustHash]="true" href="/news#rna-central-and-pombase">RNA central and PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-pombase-genomic-region-graphics'" [pageScrollAdjustHash]="true" href="/news#new-pombase-genomic-region-graphics">New PomBase genomic region graphics</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-nar-database-issue'" [pageScrollAdjustHash]="true" href="/news#pombase-nar-database-issue">PomBase NAR Database Issue</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#fungal-pathogen-genomics-course-2019-registration-open'" [pageScrollAdjustHash]="true" href="/news#fungal-pathogen-genomics-course-2019-registration-open">Fungal Pathogen Genomics Course 2019 - registration open</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#transcript-tracks-atkinson-et-al.-2018-loaded'" [pageScrollAdjustHash]="true" href="/news#transcript-tracks-atkinson-et-al.-2018-loaded">Transcript tracks from Atkinson et al. (2018) loaded</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombelist-changes'" [pageScrollAdjustHash]="true" href="/news#pombelist-changes">pombelist changes</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-genome-browser-tracks'" [pageScrollAdjustHash]="true" href="/news#new-genome-browser-tracks">New genome browser tracks</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-book-chapter-about-pombase'" [pageScrollAdjustHash]="true" href="/news#new-book-chapter-about-pombase">New book chapter about PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-releases-jbrowse'" [pageScrollAdjustHash]="true" href="/news#pombase-releases-jbrowse">PomBase releases JBrowse</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#in-memory-andr-eacute-goffeau'" [pageScrollAdjustHash]="true" href="/news#in-memory-andr-eacute-goffeau">In memory of Andr&eacute; Goffeau</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#congratulations-gsa-award-winners'" [pageScrollAdjustHash]="true" href="/news#congratulations-gsa-award-winners">Congratulations to GSA award winners</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-improved-pombase-goes-live'" [pageScrollAdjustHash]="true" href="/news#new-improved-pombase-goes-live">New, improved PomBase goes live</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#id-9th-international-fission-yeast-meeting-early-registration-closes-soon'" [pageScrollAdjustHash]="true" href="/news#id-9th-international-fission-yeast-meeting-early-registration-closes-soon">9th International Fission Yeast Meeting - early registration closes soon</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#id-9th-international-fission-yeast-meeting-registration-open'" [pageScrollAdjustHash]="true" href="/news#id-9th-international-fission-yeast-meeting-registration-open">9th International Fission Yeast Meeting registration open</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2016-10-19'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2016-10-19">PomBase data update 2016-10-19</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#show-your-support-database-funding'" [pageScrollAdjustHash]="true" href="/news#show-your-support-database-funding">Show your support for database funding</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#our-model-organism-database-commentary'" [pageScrollAdjustHash]="true" href="/news#our-model-organism-database-commentary">Our model organism database commentary</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2016-05-31'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2016-05-31">PomBase data update 2016-05-31</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2016-05-09'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2016-05-09">PomBase data update 2016-05-09</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2016-04-11'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2016-04-11">PomBase data update 2016-04-11</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2016-02-11'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2016-02-11">PomBase data update 2016-02-11</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2015-12-02'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2015-12-02">PomBase data update 2015-12-02</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-advanced-search-features'" [pageScrollAdjustHash]="true" href="/news#new-advanced-search-features">New Advanced Search features</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-genetics-primer-fission-yeast'" [pageScrollAdjustHash]="true" href="/news#new-genetics-primer-fission-yeast">New genetics primer for fission yeast</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-viability-summary-correction-alert'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-viability-summary-correction-alert">PomBase data update; viability summary correction alert</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-multi-allele-phenotypes'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-multi-allele-phenotypes">PomBase data update with multi-allele phenotypes</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2015-06-16'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2015-06-16">PomBase data update 2015-06-16</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#canto-downtime-new-version'" [pageScrollAdjustHash]="true" href="/news#canto-downtime-new-version">Canto downtime & new version</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2015-05-26'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2015-05-26">PomBase data update 2015-05-26</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombe-2015-travel-fellowships'" [pageScrollAdjustHash]="true" href="/news#pombe-2015-travel-fellowships">Pombe 2015 travel fellowships</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2015-04-19'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2015-04-19">PomBase data update 2015-04-19</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombe-2015-poster-abstract-deadline-extended'" [pageScrollAdjustHash]="true" href="/news#pombe-2015-poster-abstract-deadline-extended">Pombe 2015 poster abstract deadline extended</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombe-2015-abstract-deadline-approaching'" [pageScrollAdjustHash]="true" href="/news#pombe-2015-abstract-deadline-approaching">Pombe 2015 abstract deadline approaching</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2015-03-23'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2015-03-23">PomBase data update 2015-03-23</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombe-2015-registration-now-open'" [pageScrollAdjustHash]="true" href="/news#pombe-2015-registration-now-open">Pombe 2015 registration now open</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2015-02-16'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2015-02-16">PomBase data update 2015-02-16</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2015-01-26'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2015-01-26">PomBase data update 2015-01-26</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-compact-go-annotation-display'" [pageScrollAdjustHash]="true" href="/news#new-compact-go-annotation-display">New compact GO annotation display</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#esyn-network-visualizations-pombase'" [pageScrollAdjustHash]="true" href="/news#esyn-network-visualizations-pombase">esyN network visualizations in PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2014-11-12'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2014-11-12">PomBase data update 2014-11-12</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2014-09-16'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2014-09-16">PomBase data update 2014-09-16</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2014-08-18'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2014-08-18">PomBase data update 2014-08-18</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2014-07-17'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2014-07-17">PomBase data update 2014-07-17</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2014-07-08'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2014-07-08">PomBase data update 2014-07-08</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#gene-ontology-publication-on-annotation-extensions'" [pageScrollAdjustHash]="true" href="/news#gene-ontology-publication-on-annotation-extensions">Gene Ontology publication on annotation extensions</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2014-05-15'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2014-05-15">PomBase data update 2014-05-15</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2014-03-20'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2014-03-20">PomBase data update 2014-03-20</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2014-02-20'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2014-02-20">PomBase data update 2014-02-20</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#human-ortholog-data-correction-coming-next-month'" [pageScrollAdjustHash]="true" href="/news#human-ortholog-data-correction-coming-next-month">Human ortholog data correction coming next month</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2013-12-08'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2013-12-08">PomBase data update 2013-12-08</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#id-2013-meeting-mini-reviews-published'" [pageScrollAdjustHash]="true" href="/news#id-2013-meeting-mini-reviews-published">2013 meeting mini-reviews published</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-survey-results-available'" [pageScrollAdjustHash]="true" href="/news#pombase-survey-results-available">PomBase survey results available</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-target-gene-page-section'" [pageScrollAdjustHash]="true" href="/news#new-target-gene-page-section">New "Target Of" gene page section</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2013-10-21'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2013-10-21">PomBase data update 2013-10-21</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-user-survey-open'" [pageScrollAdjustHash]="true" href="/news#pombase-user-survey-open">PomBase User Survey open</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2013-09-15'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2013-09-15">PomBase data update 2013-09-15</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#send-htp-data-pombase'" [pageScrollAdjustHash]="true" href="/news#send-htp-data-pombase">Send HTP data to PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#connecting-pombase'" [pageScrollAdjustHash]="true" href="/news#connecting-pombase">Connecting With PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombelist-has-moved'" [pageScrollAdjustHash]="true" href="/news#pombelist-has-moved">pombelist has moved</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-website-update'" [pageScrollAdjustHash]="true" href="/news#pombase-website-update">PomBase website update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-launches-community-curation'" [pageScrollAdjustHash]="true" href="/news#pombase-launches-community-curation">PomBase launches community curation</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update-2013-06-20'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update-2013-06-20">PomBase data update 2013-06-20</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-data-update'" [pageScrollAdjustHash]="true" href="/news#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#genedb-s.-pombe-decommissioned'" [pageScrollAdjustHash]="true" href="/news#genedb-s.-pombe-decommissioned">GeneDB S. pombe decommissioned</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#quantitative-gene-expression-data-available-pombase'" [pageScrollAdjustHash]="true" href="/news#quantitative-gene-expression-data-available-pombase">Quantitative gene expression data available in PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#carl-singer-foundation-established'" [pageScrollAdjustHash]="true" href="/news#carl-singer-foundation-established">Carl Singer Foundation Established</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombe-2013-registration-abstracts-by-mon-8th-april'" [pageScrollAdjustHash]="true" href="/news#pombe-2013-registration-abstracts-by-mon-8th-april">Pombe 2013: registration & abstracts by Mon 8th April</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#data-update-on-pombase-web-site'" [pageScrollAdjustHash]="true" href="/news#data-update-on-pombase-web-site">Data update on PomBase web site</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#speed-improvements-and-new-data-on-pombase-web-site'" [pageScrollAdjustHash]="true" href="/news#speed-improvements-and-new-data-on-pombase-web-site">Speed improvements and new data on PomBase web site</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#new-data-and-new-features-on-pombase-web-site'" [pageScrollAdjustHash]="true" href="/news#new-data-and-new-features-on-pombase-web-site">New data and new features on PomBase web site</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-web-site-fully-live'" [pageScrollAdjustHash]="true" href="/news#pombase-web-site-fully-live">PomBase web site fully live</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-preview-launch'" [pageScrollAdjustHash]="true" href="/news#pombase-preview-launch">PomBase preview launch</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#pombase-nar-paper-published-online'" [pageScrollAdjustHash]="true" href="/news#pombase-nar-paper-published-online">PomBase NAR paper published online</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#genedb-s.-pombe-now-uses-latest-release-pfam-protein-family-database-25.0-.'" [pageScrollAdjustHash]="true" href="/news#genedb-s.-pombe-now-uses-latest-release-pfam-protein-family-database-25.0-.">GeneDB (S. pombe) now uses the latest release of the Pfam protein family database(25.0).</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#schizosaccharomyces-comparative-genome-paper-published'" [pageScrollAdjustHash]="true" href="/news#schizosaccharomyces-comparative-genome-paper-published">Schizosaccharomyces Comparative Genome Paper Published</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#genome-reappraisal-reveals-new-genes-and-revised-gene-structures'" [pageScrollAdjustHash]="true" href="/news#genome-reappraisal-reveals-new-genes-and-revised-gene-structures">Genome reappraisal reveals new genes and revised gene structures</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#annotated-transcription-start-and-termination-sites-fission-yeast'" [pageScrollAdjustHash]="true" href="/news#annotated-transcription-start-and-termination-sites-fission-yeast">Annotated transcription start and termination sites for fission yeast</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#analysis-fission-yeast-deletion-publication'" [pageScrollAdjustHash]="true" href="/news#analysis-fission-yeast-deletion-publication">Analysis of Fission Yeast Deletion Publication</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#funding-pombase'" [pageScrollAdjustHash]="true" href="/news#funding-pombase">Funding for PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#fission-yeast-one-12-key-organisms-reference-genomes-project'" [pageScrollAdjustHash]="true" href="/news#fission-yeast-one-12-key-organisms-reference-genomes-project">Fission yeast is one of the 12 key organisms of the reference genomes project</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#genedb-s.-pombe-now-uses-latest-update-pfam-release-24.0'" [pageScrollAdjustHash]="true" href="/news#genedb-s.-pombe-now-uses-latest-update-pfam-release-24.0">GeneDB (S. pombe) now uses the latest update to Pfam, release 24.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#fission-yeast-ensembl-fungi'" [pageScrollAdjustHash]="true" href="/news#fission-yeast-ensembl-fungi">Fission yeast in Ensembl Fungi</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#genedb-now-using-version-23-pfam-protein-family-database.'" [pageScrollAdjustHash]="true" href="/news#genedb-now-using-version-23-pfam-protein-family-database.">GeneDB is now using Version 23 of the Pfam protein family database.</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#s.-pombe-genedb-now-includes-deep-links-biological-general-repository-interaction-datasets-biogrid'" [pageScrollAdjustHash]="true" href="/news#s.-pombe-genedb-now-includes-deep-links-biological-general-repository-interaction-datasets-biogrid">S. pombe GeneDB now includes "deep links" to the Biological General Repository for Interaction Datasets (BioGRID)</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#global-sequence-and-chip-study-examines-eukaryotic-transcription'" [pageScrollAdjustHash]="true" href="/news#global-sequence-and-chip-study-examines-eukaryotic-transcription">Global sequence and chip study examines eukaryotic transcription</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#the-h-mating-type-region-has-been-provided'" [pageScrollAdjustHash]="true" href="/news#the-h-mating-type-region-has-been-provided">The h- mating type region has been provided</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#baumann-and-zakian-labs-identify-telomerase-rna'" [pageScrollAdjustHash]="true" href="/news#baumann-and-zakian-labs-identify-telomerase-rna">Baumann and Zakian labs identify telomerase RNA</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#wellcome-trust-advanced-course-genome-wide-approaches-fission-yeast'" [pageScrollAdjustHash]="true" href="/news#wellcome-trust-advanced-course-genome-wide-approaches-fission-yeast">Wellcome Trust Advanced Course 'Genome-wide approaches with fission yeast</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#id-4th-international-fission-yeast-meeting'" [pageScrollAdjustHash]="true" href="/news#id-4th-international-fission-yeast-meeting">4th International Fission Yeast Meeting</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#genedb-representation-fission-yeast-data-moved-contigs-chromosomes'" [pageScrollAdjustHash]="true" href="/news#genedb-representation-fission-yeast-data-moved-contigs-chromosomes">GeneDB representation of the fission yeast data moved from contigs to chromosomes</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#yeast-special-issue-2006-european-fission-yeast-meeting'" [pageScrollAdjustHash]="true" href="/news#yeast-special-issue-2006-european-fission-yeast-meeting">Yeast Special Issue from the 2006 European Fission Yeast Meeting</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#the-first-fission-yeast-whole-proteome-localization-study-now-published'" [pageScrollAdjustHash]="true" href="/news#the-first-fission-yeast-whole-proteome-localization-study-now-published">The first fission yeast whole proteome localization study is now published</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#fission-yeast-database-survey'" [pageScrollAdjustHash]="true" href="/news#fission-yeast-database-survey">Fission yeast database survey</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#european-fission-yeast-meeting'" [pageScrollAdjustHash]="true" href="/news#european-fission-yeast-meeting">European Fission Yeast Meeting</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#comparative-genomics-eukaryotic-microorganisms'" [pageScrollAdjustHash]="true" href="/news#comparative-genomics-eukaryotic-microorganisms">Comparative Genomics of Eukaryotic Microorganisms</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#second-east-coast-regional-pombe-meeting'" [pageScrollAdjustHash]="true" href="/news#second-east-coast-regional-pombe-meeting">Second East Coast Regional pombe Meeting</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#general-repository-interaction-datasets'" [pageScrollAdjustHash]="true" href="/news#general-repository-interaction-datasets">General Repository for Interaction Datasets</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#the-third-international-fission-yeast-meeting'" [pageScrollAdjustHash]="true" href="/news#the-third-international-fission-yeast-meeting">The Third International Fission Yeast Meeting</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#methods-volume-33-issue-3'" [pageScrollAdjustHash]="true" href="/news#methods-volume-33-issue-3">Methods Volume 33 Issue 3</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#recent-genome-wide-surveys'" [pageScrollAdjustHash]="true" href="/news#recent-genome-wide-surveys">Recent Genome wide surveys</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#the-new-fission-yeast-book-now-published'" [pageScrollAdjustHash]="true" href="/news#the-new-fission-yeast-book-now-published">The 'new' fission yeast book is now published</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#schizosaccharomyces-pombe-essential-genes-pilot-study'" [pageScrollAdjustHash]="true" href="/news#schizosaccharomyces-pombe-essential-genes-pilot-study">Schizosaccharomyces pombe Essential Genes: A pilot Study</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll [pageScrollTarget]="'#global-transcriptional-responses-fission-yeast-environmental-stress'" [pageScrollAdjustHash]="true" href="/news#global-transcriptional-responses-fission-yeast-environmental-stress">Global transcriptional responses of fission yeast to environmental stress</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'gocams-in-pombase'">
<h3 id="new-curated-pathways-using-go-cam-pathway-modelling">NEW: Curated pathways using GO-CAM pathway modelling</h3>
<!-- pombase_flags: frontpage -->
<!-- newsfeed_thumbnail: pombase-logo-32x32px.png -->
<p>We have released the first <a routerLink="/gocam">36 curated biological pathways</a>, covering 466 proteins.</p>
<p>These pathways were curated using GO-Causal Activity Modeling (GO-CAM), which provides a system to represent biological processes as a causal network by connecting GO molecular function terms (activities), encoded by individual gene products.</p>
<p>GO-CAM pathway for <a routerLink="/gocam/pombase-view/docs/66c7d41500001171">vitamin B6 metabolic process</a>:</p>
<p><a href="assets/newsfeed/go-cam_vitamin_B6_metabolic_process.png"><img src="assets/newsfeed/go-cam_vitamin_B6_metabolic_process.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Example GO-CAM pathway - vitamin B6 metabolic process" /></a></p>
<p>Visit the <a routerLink="/gocam">GO-CAM pathways page</a> to see the list of curated pathways.</p>

  </div>
  <div *ngIf="pageName == 'index'">
<h2 id="news-archive">News archive</h2>
<div class="news-archive">
<div id="2025-08-11-gocam-tools" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/gocam-32px.png"/></p>
<h3 id="new-interactive-views-exploring-go-cam-pathways">New Interactive Views for Exploring GO-CAM Pathways</h3>
<p><em>2025-08-11</em></p>
<p>Introducing two new interactive views designed to support the exploration and use of <a routerLink="/gocam/">GO-CAM</a> (Gene Ontology Causal Activity Model) data.</p>
<h5 id="summary-map"><strong>Summary Map</strong></h5>
<p><a href="assets/newsfeed/gocam-summary-map.png"><img src="assets/newsfeed/gocam-summary-map-trim.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Summary Map" /></a></p>
<p>The <strong>Summary Map</strong> provides a high-level, visual overview of how individual GO-CAM pathways are causally connected—without displaying their internal structure. This allows you to quickly see relationships between pathways at a glance:</p>
<ul>
<li><a routerLink="/gocam/summary/all">View all models - connected and unconnected</a></li>
<li><a routerLink="/gocam/summary/connected">View only connected models</a></li>
</ul>
<p>The maps are fully interactive:</p>
<ul>
<li>Click on a box to open the corresponding GO-CAM</li>
<li>Click on a relationship arrow to explore connections between models</li>
<li>Zoom, drag, and rearrange nodes to explore the layout</li>
</ul>
<h5 id="mega-model"><strong>Mega Model</strong></h5>
<p><a href="assets/newsfeed/gocam-mega-model.png"><img src="assets/newsfeed/gocam-mega-model-trim.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Mega Model" /></a></p>
<p>The <strong>Mega Model</strong> view displays GO-CAMs in full detail, showing all intra- and inter-model connections. Initially, chemical inputs/outputs and target genes are hidden to reduce visual complexity—but you can toggle them on or off as needed.</p>
<ul>
<li><a routerLink="/gocam/mega-model/all">View all models</a></li>
<li><a routerLink="/gocam/mega-model/connected">View only connected models</a></li>
</ul>
<p>Like the Summary Map, Mega Models are fully interactive:</p>
<ul>
<li>Zoom and pan to navigate the network</li>
<li>Click on gene icons or pathway boxes to access detailed information and links to full GO-CAM pathways</li>
</ul>
</div>
<div id="2025-06-04-JBrowse-V2-announcement" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="jbrowse-version-2-now-available-on-pombase">JBrowse version 2 is now available on PomBase</h3>
<p><em>2025-06-04</em></p>
<p>Version 2 of the JBrowse genome browser is now available on PomBase.<br />
<a href="https://www.pombase.org/jbrowse2/">PomBase JBrowse V2</a></p>
<p>You can now browse the S. pombe genome, and visualize all of our hosted datasets, in a new, nicer interface. JBrowse V2 possesses several new features, such as the possibility to show regions of different chromosomes in the same view, the ability to toggle on or off the display of the forward, reverse and translation sequences for the reference sequence track, and the support of Hi-C data. You can find the complete list of new features on <a href="https://jbrowse.org/jb2/features/">this page</a></p>
<p>Both the old (V1) and the new versions of JBrowse are linked to from the PomBase front page and at the top of the Genome browser widget on the gene pages.</p>
</div>
<div id="2025-05-12-gocam-10000" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="pombase-expands-go-cam-pathway-models-fission-yeast">PomBase Expands GO-CAM Pathway Models for Fission Yeast</h3>
<p><em>2025-05-12</em></p>
<p>PomBase has more than doubled its curated <a routerLink="/gocam">GO-Causal Activity Modelling</a> (GO-CAM) pathway models—from 36 to 74—now covering over <a routerLink="/results/from/id/f99d8133-3206-4941-b44e-9314e7cae3d2">1000 fission yeast proteins</a>. These causal models span key biological pathways, with a focus on mitochondrial processes through collaboration with FlyBase and UniProtKB. New tools for pathway visualization and interaction are in development and will be showcased at Pombe 2025.</p>
<p>Explore the pathways and gaps on the <a routerLink="/gocam">PomBase GO-CAM page</a>.</p>
<p>Example GO-CAM pathway:<br />
<a routerLink="/gocam/pombase-view/docs/678073a900003902">L-proline biosynthetic process (GO:0055129), proline catabolic process to glutamate (GO:0010133), glutamine biosynthetic process (GO:0006542))</a></p>
<p><a href="assets/newsfeed/gocam-update-1000-genes.png"><img src="assets/newsfeed/gocam-update-1000-genes.png" class="screenshot" style="width:100.0%" loading="lazy" alt="L-proline biosynthetic process (GO:0055129), proline catabolic process to glutamate (GO:0010133), glutamine biosynthetic process (GO:0006542)" /></a></p>
</div>
<div id="2025-03-26-skribbe-transcription-factor-dataset" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="the-transcription-factor-dataset-skribbe-soneson-stadler-et-al-.-now-available-pombase">The Transcription Factor Dataset from Skribbe, Soneson, Stadler <em>et al</em>. is now available in PomBase</h3>
<p><em>2025-03-26</em></p>
<p>This publication is an incredible resource to provide biological insights into known and previously unpublished transcription factors, and masterclass in making data <a href="https://www.go-fair.org/fair-principles/">FAIR compliant</a>.</p>
<p><a routerLink="/reference/PMID:40015273">PomBase publication page PMID:40015273</a></p>
<p>Provides:</p>
<ul>
<li>External links to the key resources associated with this project</li>
<li>GO molecular function annotation for promoter binding (&gt;4000 annotations)</li>
<li>Data from the small-scale studies related to ntu1/ntu2 curated in Canto</li>
<li>Affinity capture mass spec data (&gt;300 annotations)</li>
<li>A chooser for the data tracks hosted in JBrowse (&gt;300 data tracks)</li>
</ul>
<p><a href="/jbrowse/?loc=II%3A4442142..4454370&amp;tracks=Forward%20strand%20features%2CReverse%20strand%20features%2CPho7%20ChIP-seq%20%3B%20repeat%201%20-%20Skribbe%20et%20al.%20(2025)&amp;tracklist=1&amp;nav=1&amp;overview=1&amp;highlight"><img src="/assets/newsfeed/skribbe-pmid-40015273-jbrowse.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Skribbe Transcription Factor Dataset in JBrowse" /></a></p>
</div>
<div id="2025-03-24-interpro-release-104" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="interpro-version-104.0">InterPro Version 104.0</h3>
<p><em>2025-03-24</em></p>
<p>PomBase now uses InterPro Version 104.0</p>
<p>New InterPro features in this release include:</p>
<ul>
<li>The addition of 661 InterPro entries.</li>
<li>An update to Pfam [37.2], NCBIfam [17.0], CDD [3.21].</li>
<li>Integration of 682 new methods from the Pfam (663), NCBIfam (5), PANTHER (5), CDD (9) databases.</li>
</ul>
</div>
<div id="2025-02-27-human-gene-function-paper" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="the-human-functionome-paper-published">The human functionome paper published</h3>
<p><em>2025-02-27</em></p>
<p>The Gene Ontology Consortium’s latest publication, <em>A compendium of human gene functions derived from evolutionary modelling</em>, is <a href="https://nature.com/articles/s41586-025-08592-0">out in Nature</a>! PomBase is proud to have contributed to this extensive effort, which leverages all model organism data to support phylogenetic transfer of annotation. Fission yeast GO data currently support <a href="https://www.ebi.ac.uk/QuickGO/annotations?taxonId=9606&taxonUsage=descendants&withFrom=PomBase">6,461 human gene annotations</a>.<br />
Beyond providing annotation, PomBase played a crucial role in reviewing annotation inferences, ensuring accuracy of functional annotation across species. Over seven years, PomBase submitted nearly 600 update requests to the project.</p>
<ul>
<li>🔗 <a href="https://functionome.geneontology.org">Explore the functionome</a><br />
</li>
<li>📖 <a href="https://nature.com/articles/s41586-025-08592-0">Read the paper</a></li>
</ul>
</div>
<div id="2025-02-25-table-of-holes-in-pathways" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="new-missing-activity-list-list-potential-pathway-holes">NEW: The Missing activity list - A list of potential “pathway holes”</h3>
<p><em>2025-02-25</em></p>
<p>During GO-CAM pathway curation, we have sometimes been unable to locate a specific annotated activity.</p>
<p>We have generated a <a routerLink="/status/missing-gocam-proteins">list of these missing activities</a> from the GO-CAM pathways.</p>
<p>We expect this list to continue to grow as we curate more pathways. Please note that in some cases these may not be present, or a reaction may be spontaneous.</p>
<p>The list is currently linked to from the <a routerLink="/status/gene-characterisation">gene characterisation page</a> and from the <a routerLink="/documentation/gocams">GO-CAM pathway list</a>.</p>
<p>Example of pathway with a missing gene / pathway hole: <a routerLink="/gocam/pombase-view/gene/665912ed00002626/SPAC1F7.09c/dal2">nitrogen cycle metabolic process (GO:0071941)</a></p>
<p><a href="assets/newsfeed/gocam_pathway_hole.png"><img src="/assets/newsfeed/gocam_pathway_hole.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Example GO-CAM pathway hole - nitrogen cycle metabolic process" /></a></p>
</div>
<div id="2025-02-17-pombase-gcbr-week" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="pombase-global-core-biodata-resource-week">PomBase is Global Core BioData Resource of the week</h3>
<p><em>2025-02-17</em></p>
<p>PomBase is this weeks featured <a href="https://globalbiodata.org/">Global Biodata Coalition</a> “<a href="https://www.linkedin.com/pulse/pombase-global-biodata-coalition-yocme">Global Core BioData Resource of the week</a>”.</p>
<p>The Global Biodata Coalition works for and with research funders to ensure sustained support for biodata resources.</p>
<p><a href="https://www.linkedin.com/pulse/pombase-global-biodata-coalition-yocme">Global Core BioData Resource of the week - LinkedIn</a></p>
<p><a href="assets/newsfeed/pombase-gcbr-roc.png"><img src="/assets/newsfeed/pombase-gcbr-roc.png" class="screenshot" style="width:100.0%" loading="lazy" alt="PomBase - GCBR Resource of the week" /></a></p>
</div>
<div id="2025-02-14-pombetalks-feb" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-february-19th">Next pombeTalks: Wednesday February 19th</h3>
<p><em>2025-02-14</em></p>
<p>The next <a href="https://evonuclab.org/pombetalks">pombeTalks</a> will take place on Wednesday, February 19th, at 00:00 San Francisco / 03:00 New York / 08:00 London / 09:00 Paris / 13:30 Delhi / 16:00 Beijing / 17:00 Tokyo</p>
<p>Speakers this month are:</p>
<ul>
<li><p><strong>Ulysses Lam</strong><br />
Chen Ee Sin’s Lab, National University of Singapore<br />
“Set2 regulates Ccp1 and Swc2 to ensure centromeric stability by retargeting CENP-A”</p></li>
<li><p><strong>Sucharita Sen</strong><br />
Dr. Mridula Nambiar’s Lab, IISER Pune, India<br />
“Centromere-proximal crossovers disrupt proper homologous chromosome disjunction during meiosis”</p></li>
</ul>
<p>Also join us for a session called <strong>“Just One Thing”: A brief demonstration of a PomBase feature</strong></p>
<p>After the first talk, PomBase will present a quick 10-minute demo highlighting a specific PomBase feature. This week, the focus will be on <u>Advanced Visualization of query results</u>.</p>
<p>See <a href="https://lists.cam.ac.uk/sympa/info/ucam-pombelist">pombelist</a> for the Zoom details and password.</p>
</div>
<div id="2025-01-16-gocams-in-pombase" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="new-curated-pathways-using-go-cam-pathway-modelling">NEW: Curated pathways using GO-CAM pathway modelling</h3>
<p><em>2025-01-16</em></p>
<p>We have released the first <a routerLink="/gocam">36 curated biological pathways</a>, covering 466 proteins.</p>
<p>These pathways were curated using GO-Causal Activity Modeling (GO-CAM), which provides a system to represent biological processes as a causal network by connecting GO molecular function terms (activities), encoded by individual gene products.</p>
<p>GO-CAM pathway for <a routerLink="/gocam/pombase-view/docs/66c7d41500001171">vitamin B6 metabolic process</a>:</p>
<p><a href="assets/newsfeed/go-cam_vitamin_B6_metabolic_process.png"><img src="assets/newsfeed/go-cam_vitamin_B6_metabolic_process.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Example GO-CAM pathway - vitamin B6 metabolic process" /></a></p>
<p>Visit the <a routerLink="/gocam">GO-CAM pathways page</a> to see the list of curated pathways.</p>
</div>
<div id="2025-01-09-pombetalks-jan" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-january-15th">Next pombeTalks: Wednesday January 15th</h3>
<p><em>2025-01-09</em></p>
<p>The next <a href="https://evonuclab.org/pombetalks">pombeTalks</a> will take place on Wednesday, January 15th, at <a href="https://everytimezone.com/s/0d8dfbff">08:00</a> San Francisco / 11:00 New York / 16:00 London / 17:00 Paris / 21:30 Delhi / 00:00 Beijing / 01:00 Tokyo.</p>
<p>Speakers this month are:</p>
<ul>
<li><p><strong>Kaushik Ragunathan</strong><br />
Department of Biology, Brandeis University<br />
“Don’t forget to remember: HP1 proteins and the persistence of an epigenetic imprint”</p></li>
<li><p><strong>Ayokunle Oluwaseun Araoyinbo</strong><br />
Vjestica Lab, University of Lausanne “Fertilization triggers cytosolic functions and P-body recruitment of the RNA-binding protein Mei2 to drive fission yeast zygotic development”</p></li>
</ul>
<p>And new this month:<br />
<strong>“Just one thing” - A quick demonstration of a PomBase feature.</strong></p>
<p>At the end of the talks, we’ll feature a quick 10-minute demonstration showcasing a specific PomBase feature. This week, <strong>Pascal Carme</strong> will explore the Advanced Search function.</p>
<p>See <a href="https://lists.cam.ac.uk/sympa/info/ucam-pombelist">pombelist</a> for the Zoom details and password.</p>
</div>
<div id="2024-12-04-pombetalks-december" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-december-11th">Next pombeTalks: Wednesday December 11th</h3>
<p><em>2024-12-04</em></p>
<p>The next <a href="https://evonuclab.org/pombetalks">pombeTalks</a> will take place on Wednesday, December 11th, at <a href="https://everytimezone.com/s/0d8dfbff">08:00</a> San Francisco / 11:00 New York / 16:00 London / 17:00 Paris / 21:30 Delhi / 00:00 Beijing / 01:00 Tokyo.</p>
<p>Speaker this month is:</p>
<ul>
<li><strong>Valentin Flury</strong><br />
Max Planck Institute of Immunobiology and Epigenetics, Freiburg, Germany<br />
“The fork protection complex promotes parental histone recycling and epigenetic memory”</li>
</ul>
<p>See <a href="https://lists.cam.ac.uk/sympa/info/ucam-pombelist">pombelist</a> for the Zoom details and password.</p>
</div>
<div id="2024-11-22-new-pfam-paper" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="discovery-new-s.-pombe-orthologs-human-and-s.-cerevisiae-proteins-using-pfam-n">Discovery of new S. pombe orthologs of human and S. cerevisiae proteins using Pfam-N</h3>
<p><em>2024-11-22</em></p>
<p>Our work to using Pfam-N to identify distant orthologs between fission yeast and human or budding yeast was included in the <a href="https://doi.org/10.1093/nar/gkae997">Pfam NAR update</a>. Check out the rest of the paper to see recent improvements to Pfam as they continue to work toward comprehensive coverage of the protein universe.</p>
<p><img src="assets/newsfeed/pfam-nar-article-2024.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Pfam NAR article 2024" /></p>
</div>
<div id="2024-11-15-pombetalks-november" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-november-20th">Next pombeTalks: Wednesday November 20th</h3>
<p><em>2024-11-15</em></p>
<p>The next <a href="https://evonuclab.org/pombetalks">pombeTalks</a> will take place on Wednesday, November 20th, at <a href="https://everytimezone.com/s/4009cf69?t=66ea1800,1a4">9am CET</a>, 00:00 San Francisco / 03:00 New York / 08:00 London / 09:00 Paris / 13:30 Delhi / 16:00 Beijing / 17:00 Tokyo</p>
<p>Speakers this month are:</p>
<ul>
<li><p><strong>Miguel BASANTE-BEDOYA, PhD (He/His)</strong><br />
Sophie Martin’s Lab, University of Geneva<br />
“Antagonistic Pathways of Lipid Transfer Proteins (LTPs) for de novo Membrane Assembly in Fission Yeast”</p></li>
<li><p><strong>Sugata Chaudhuri</strong><br />
Shravan Kumar Mishra’s Lab, IISER Mohali, India<br />
“Trafficking Roles of the DUB Module Ftp105-Ubp5 at the Golgi”</p></li>
</ul>
<p>See <a href="https://lists.cam.ac.uk/sympa/info/ucam-pombelist">pombelist</a> for the Zoom details and password.</p>
</div>
<div id="2024-11-14-mobidb-lite-disorder" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="mobidb-disorder-predictions">MobiDB disorder predictions</h3>
<p><em>2024-11-14</em></p>
<p>We now use <a href="https://doi.org/10.1093/bioinformatics/btx015">MobiDB-lite</a> for identifying disordered regions replacing IUPred. Unlike single-residue level predictions, MobiDB-lite focuses on providing a single, consensus-based prediction optimized for high specificity, particularly for detecting long disordered regions with minimal false positives.</p>
<p><a routerLink="/gene/SPAC19G12.10c">Example from cpy1</a>:</p>
<p><a href="assets/newsfeed/feature-full-view-cpy1-mobidb-big.png"><img src="assets/newsfeed/feature-full-view-cpy1-mobidb.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Example full protein feature view for cpy1 with MobiDB-lite disorder features hightlighted" /></a></p>
</div>
<div id="2024-11-13-uniprot-protein-features" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="additional-protein-features-imported-uniprotkb">Additional protein features imported from UniProtKB</h3>
<p><em>2024-11-13</em></p>
<p>Additional protein features and modifications inferred from sequence context by the UniProt database have been imported to supplement our large curated experimental datasets. <a routerLink="/reference/PMID:36408920">Imported data</a> includes propeptides, active site binding sites, mitochondrial targeting signal and 212 additional signal peptides. Features are listed in the “protein domains and properties” section of the gene page and features with coordinates are displayed in the protein feature viewer. Signal peptides and mitochondrial targeting signals from all sources can also be retrieved using the queries in the “domains features and motifs” section of the “<a href="/query">Advanced Search</a>”. We have also imported an additional 445 inferred modifications from UniProtKB to supplement the ~69,000 PomBase curated modifications.</p>
</div>
<div id="2024-11-12-protein-feature-view-changes" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="changes-protein-features-view-on-gene-pages">Changes to protein features view on gene pages</h3>
<p><em>2024-11-12</em></p>
<p>The protein domains and features section of the gene pages now uses the same protein feature display as the “full view” linked to from the summary section. This provides a consistent view of protein features across the website. We now also display active and binding sites, signal peptides, transit peptides, TMMs, and other Sequence Ontology features in this view.</p>
<p><a routerLink="/gene/SPAC19G12.10c">Example from cpy1</a>:</p>
<p><a href="assets/newsfeed/feature-full-view-cpy1-big.png"><img src="assets/newsfeed/feature-full-view-cpy1.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Example full protein feature view for cpy1" /></a></p>
<p>Thanks to the team at <a href="https://www.rcsb.org/">RCSB PDB</a> for providing the <a href="https://github.com/rcsb/rcsb-saguaro">Open Source software</a> used to implement the protein feature view in PomBase.</p>
</div>
<div id="2024-10-14-pombetalks-16th-oct" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-october-16th">Next pombeTalks: Wednesday October 16th</h3>
<p><em>2024-10-14</em></p>
<p>The next <a href="https://evonuclab.org/pombetalks">pombeTalks</a> will take place on Wednesday, October 16th, at <a href="https://everytimezone.com/s/4009cf69">08:00</a> San Francisco / 11:00 New York / 16:00 London / 17:00 Paris / 20:30 Delhi / 23:00 Beijing / 00:00 Tokyo</p>
<p>The speaker is:</p>
<ul>
<li><strong>Pritha Chowdhury</strong><br />
Qian Chen’s lab, University of Toledo, USA<br />
“A mechanosensitive ion channel Pkd2 modulates the assembly of myosin II and actin in the fission yeast cytokinetic contractile ring”</li>
</ul>
<p>See <a href="https://lists.cam.ac.uk/sympa/info/ucam-pombelist">pombelist</a> for the meeting details and Zoom passcode</p>
</div>
<div id="2024-09-11-pombetalks-18th-sept" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-september-18th">Next pombeTalks: September 18th</h3>
<p><em>2024-09-11</em></p>
<p>The next <a href="https://evonuclab.org/pombetalks">pombeTalks</a> will take place on Wednesday, September 18th, at <a href="https://everytimezone.com/s/4009cf69">08:00</a> San Francisco / 11:00 New York / 16:00 London / 17:00 Paris / 20:30 Delhi / 23:00 Beijing / 00:00 Tokyo.</p>
<p>Speakers are:</p>
<ul>
<li><p><strong>Ying Gu</strong><br />
Snezhana Oliferenko Lab, King’s College London<br />
“Peroxisomal compartmentalization of amino acid biosynthesis reactions imposes an upper limit on compartment size”</p></li>
<li><p><strong>Yannis Reignier</strong><br />
Nicolas Minc lab, Institut Jacques Monod - CNRS, Paris<br />
“Mechanism of mechanosensation and repair in fission yeast cell wall.”</p></li>
</ul>
<p><a href="https://unige.zoom.us/j/69436207207?pwd=MlAIMX1pq4Kl4liXVcopH4MxXOo8C2.1">Zoom Meeting</a><br />
Meeting ID: 694 3620 7207</p>
<p>See <a href="https://lists.cam.ac.uk/sympa/info/ucam-pombelist">pombelist</a> for the meeting passcode</p>
</div>
<div id="2024-09-03-interpro-release-101" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-updated-use-interpro-version-101.0">PomBase updated to use InterPro Version 101.0</h3>
<p><em>2024-09-03</em></p>
<p>PomBase now uses InterPro Version 101.0</p>
<p>Features include:</p>
<ul>
<li>The addition of 753 InterPro entries.</li>
<li>An update to NCBIfam (15.0).</li>
<li>Integration of 767 new methods from the NCBIfam (318), Pfam (446), PANTHER (2), CDD (1) databases.</li>
</ul>
</div>
<div id="2024-08-29-pombetalks-session-5" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-season-5-starting-september-18th">pombeTalks Season 5 starting September 18th</h3>
<p><em>2024-08-29</em></p>
<p>Season 5 of <a href="https://evonuclab.org/pombetalks">pombeTalks</a> will start in September.</p>
<p>Save the dates:</p>
<ul>
<li>September 18th 2024 5pm CET</li>
<li>October 16th 2024 at 5pm CET</li>
<li>November 20th 2024 at 9am CET</li>
<li>December 18th 2024 at 5pm CET</li>
<li>January 15th 2025 at 5pm CET</li>
<li>February 19th 2025 at 9am CET</li>
</ul>
</div>
<div id="2024-08-13-new-human-and-cerevisiae-orthologs" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="new-human-and-s.-cerevisiae-orthologs">New human and <em>S. cerevisiae</em> orthologs</h3>
<p><em>2024-08-13</em></p>
<p>We have recently assigned new orthologs utilising <a href="https://xfam.wordpress.com/2024/05/31/pfam-n-version-3-enhancing-pfam-coverage-of-uniprot-with-computer-vision-deep-learning-techniques/">Pfam-N</a> protein family extensions, <a href="https://www.ebi.ac.uk/Tools/hmmer/search/jackhmmer">JackHMMER</a> iterative searches, and <a href="https://search.foldseek.com/search">FoldSeek</a> structural alignments.</p>
<p>New orthologs:<br />
We have identified orthologs of human <a routerLink="/gene/SPAP27G11.08c">MEIOB</a>, <a routerLink="/gene/SPCC16C4.04">PEX26</a>, <a routerLink="/gene/SPBC19G7.18c">OCIAD1 and OCIAD2</a>, <a routerLink="/gene/SPAC2C4.10c">SIKE1 and FGFR1OP2</a>, <a routerLink="/gene/SPCC1393.11">NGRN</a>, <a routerLink="/gene/SPAC4G9.11c">TFAM</a>, <a routerLink="/gene/SPAC15A10.17">NDUFA3</a>, <a routerLink="/gene/SPCC16C4.04">PEX26 (Sc PEX15)</a>, <a routerLink="/gene/SPAC1805.10">CCZ1</a>.<br />
Plus orthologs of <em>S. cerevisiae</em> <a routerLink="/gene/SPBC16A3.03c">MNE1</a>, <a routerLink="/gene/SPBC11C11.06c">OM14</a>, <a routerLink="/gene/SPAC57A7.15c">PEX22</a>, and <a routerLink="/gene/SPAC23H4.16c">CAF130</a>.</p>
<p>We have also assigned other sequence orphans to known protein families. See <a href="https://lists.cam.ac.uk/sympa/arc/ucam-pombelist/2024-08/msg00000.html">this pombelist post</a> for details.</p>
<p>These additions bring the number of proteins with human orthologs to <a routerLink="/results/from/id/8335a7b3-5105-4414-9743-bb6422692858">3641 (72%)</a>, and the number with S. cerevisiae orthologs to <a routerLink="/results/from/id/12a88b3a-9bab-4eae-a7e0-bb137b55d851">4006 (79%)</a>.</p>
</div>
<div id="2024-07-10-unknowns-coverage-update" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="unknowns-coverage-update">“Unknowns” coverage update</h3>
<p><em>2024-07-10</em></p>
<p>In our 2019 work, <a href="https://royalsocietypublishing.org/doi/10.1098/rsob.180241">“Hidden in plain sight: What Remains to be discovered in the eukaryotic proteome?”</a>, we demonstrated that <em>S. pombe</em> had a higher GO biological process (BP) annotation coverage (86.7%) than both <em>S. cerevisiae</em> (83.5%) and human (82.8%).</p>
<p>Of the model eukaryotes S. pombe has:</p>
<ol type="1">
<li>The highest percentage known coverage</li>
<li>The fewest phylum-specific proteins</li>
<li>The smallest proteome (despite a higher number of genes conserved in metazoa than S. cerevisiae)</li>
<li>Dedicated efforts to uncover unknown functions</li>
</ol>
<p>and is uniquely positioned to be the first eukaryote for which we know the process or pathway for every protein</p>
<p>The trend we identified continues to hold (<em>S. pombe</em> “knowns” have increased to 86.9% in 2024). The coverage is confirmed by the <a href="https://genomeannotation.rheelab.org/#!/overview">Rhee lab GO overview</a>:</p>
<p><a href="assets/newsfeed/Rhee_lab_unknowns_coverage.png"><img src="assets/newsfeed/Rhee_lab_unknowns_coverage.png" title="Rhee lab unknowns coverage" class="screenshot" style="width:100.0%" alt="Rhee lab unknowns coverage" /></a></p>
<p>Image citation: <a href="https://doi.org/10.1002/pld3.499">Xue, B., &amp; Rhee, S. Y. (2023). Status of genome function annotation in model organisms and crops. Plant Direct, 7(7), e499.</a></p>
<p>Related links:</p>
<ul>
<li><a href="https://doi.org/10.1093/genetics/iyae007">Rutherford et al 2024</a> explains PomBase’s choice of metric for assessing knowledge breadth</li>
<li><a routerLink="/status/gene-characterisation">Protein coding gene characterization status at PomBase</a></li>
</ul>
</div>
<div id="2024-07-03-rna-structures" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/rna_central_icon.png"/></p>
<h3 id="rna-secondary-structure-predictions-on-gene-pages">RNA secondary structure predictions on gene pages</h3>
<p><em>2024-07-03</em></p>
<p>We have added RNA secondary structure predictions from <a href="https://rnacentral.org/r2dt">R2DT (RNAcentral)</a> to the gene pages.</p>
<p>When a prediction is available an option will appear in the summary section of the RNA pages. See <a routerLink="/gene/SPRRNA.03">the SPRRNA.03 gene page</a> for an example.</p>
<p><a href="assets/newsfeed/rna_structure_SPRRNA.03_page.png"><img src="assets/newsfeed/rna_structure_SPRRNA.03_page.png" title="RNA structure - SPRRNA.03 gene page" class="screenshot" style="width:100.0%" alt="RNA structure - SPRRNA.03 gene page" /></a></p>
<p>You can retrieve the list of all RNA genes with secondary structure predictions using the “Genes with RNA secondary structure predictions” query under “Commonly used queries” in the <a routerLink="/query">PomBase Advanced Search</a>.</p>
</div>
<div id="2024-06-21-disease-monarch" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/human_disease.png"/></p>
<h3 id="human-disease-association-changes-monarch">Human disease association changes - Monarch</h3>
<p><em>2024-06-21</em></p>
<p>We have changed our major source of human disease associations from <a href="https://www.malacards.org/">MalaCards</a> to <a href="https://monarchinitiative.org/">Monarch</a>.</p>
<p>This change has:</p>
<ol type="1">
<li>Increased the number of fission yeast genes with disease associations from 1463 to 1551, made the associations more accurate and up to date</li>
<li>Allowed us to automate the inclusion of new disease gene imports from Monarch based on our ortholog mappings</li>
<li>Enabled us to add the qualifiers “causal” or “correlated with” (for multi-genic disorders) to the disease association (you can see this on the gene pages under “show details”)</li>
</ol>
<p>Disease associations in PomBase:</p>
<ul>
<li><a routerLink="/term/MONDO:0700096">Browse the new disease list</a></li>
<li><a routerLink="/browse-curation/disease-slim">Overview of disease associations (disease slim)</a></li>
</ul>
<p>If you have knowledge of a specific disease class or notice any issues with the ontology you can <a href="https://github.com/monarch-initiative/mondo/issues/">report this to Monarch</a></p>
<p>The Monarch initiative also host our fission yeast phenotype associations in their <a href="https://monarchinitiative.org/">knowledge graph</a>.</p>
<p>Related items: <a href="https://monarchinit.medium.com/pombase-chooses-monarch-as-source-of-human-disease-gene-associations-e53c6d218dce">Monarch Initiative blog post</a></p>
</div>
<div id="2024-06-14-interpro-release-100" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-now-includes-data-interpro-v100.0">PomBase now includes data from InterPro v100.0</h3>
<p><em>2024-06-14</em></p>
<p>PomBase now uses <a href="https://www.ebi.ac.uk/interpro/">InterPro</a> <a href="https://www.ebi.ac.uk/interpro/release_notes/">release 100.0</a>.</p>
<p>Features include:</p>
<ul>
<li>The addition of 570 InterPro entries.</li>
<li>An update to Pfam [37.0].</li>
<li>Integration of 582 new methods from the NCBIfam (26), Pfam (545), PANTHER (3), CDD (8) databases.</li>
</ul>
</div>
<div id="2024-06-14-pombetalks-june-19th" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-june-19th">Next pombeTalks: Wednesday, June 19th</h3>
<p><em>2024-06-14</em></p>
<p>The final session of this season of <a href="https://evonuclab.org/pombetalks">pombeTalks</a> will take place on Wednesday, June 19th.</p>
<p>The session will start at:<br />
01:00 San Francisco / 04:00 New York / 09:00 London / 10:00 Paris / 13:30 Delhi / 16:00 Beijing / 17:00 Tokyo.<br />
Please note the adjusted start time.</p>
<p>The speaker is:</p>
<ul>
<li><strong>Manu Lera Ramírez</strong><br />
Bähler Lab, University College London, UK<br />
“ShareYourCloning: towards a FAIR standard for sequence provenance”</li>
</ul>
<p>Zoom meeting ID: 664 3811 8855<br />
See <a href="https://lists.cam.ac.uk/sympa/info/ucam-pombelist">pombelist</a> or {{"@"}}pombeSlack for more details. The passcode will be sent shortly before the talk for security reasons.</p>
</div>
<div id="2024-05-11-pombetalks-may-11" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-may-15th">Next pombeTalks: Wednesday, May 15th</h3>
<p><em>2024-05-11</em></p>
<p>The next <a href="https://evonuclab.org/pombetalks">pombeTalks</a> session S04E08 will take place on Wednesday, May 15th.</p>
<p>The session will start at: 08:00 San Francisco / 11:00 New York / 16:00 London / 17:00 Paris / 20:30 Delhi / 23:00 Beijing / 00:00 Tokyo.</p>
<p>Speakers are:</p>
<ul>
<li><p><strong>Shaimaa Hassan</strong><br />
University College London, UK<br />
“Genome regulation in dormant cells of fission yeast, turquoise killifish and human cancer”</p></li>
<li><p><strong>Guillem Murciano-Julià</strong><br />
Oxidative Stress and Cell Cycle, Universitat Pompeu Fabra, Barcelona, Spain<br />
“Quantifying Cell Cycle Progression and Growth Rate at the Single-Cell Level in Fission Yeast”</p></li>
</ul>
<p>Zoom meeting ID: 982 4112 2806<br />
See <a href="https://lists.cam.ac.uk/sympa/info/ucam-pombelist">pombelist</a> or {{"@"}}pombeSlack for more details. The passcode will be sent shortly before the talk for security reasons.</p>
</div>
<div id="2024-05-04-interpro-release-99" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-updated-use-interpro-version-99.0">PomBase updated to use InterPro Version 99.0</h3>
<p><em>2024-05-04</em></p>
<p>PomBase now uses InterPro Version 99.0.</p>
<p>Features include:</p>
<ul>
<li>The addition of 3,837 InterPro entries</li>
<li>An update to NCBIfam (14.0)</li>
<li>Integration of 3,831 new methods from the NCBIfam (432), Pfam (1), CATH-Gene3D (64), PANTHER (3,328), CDD (6) databases</li>
</ul>
</div>
<div id="2024-04-11-pombetalks-april-2024" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="news-pombetalks-wednesday-april-17th">News pombeTalks: Wednesday April 17th</h3>
<p><em>2024-04-11</em></p>
<p>The next pombeTalk, S04E07 will be on Wednesday, April 17th, at 08:00 San Francisco / 11:00 New York / 16:00 London / 17:00 Paris / 20:30 Delhi / 23:00 Beijing / 00:00 Tokyo.</p>
<p>The speakers are:</p>
<ul>
<li><p><strong>Yanze Jian</strong><br />
Perelman school of Medicine, University of Pennsylvania, US<br />
“Phosphorylation of Bub1 by Mph1 promotes Bub1 signaling at the kinetochore to ensure accurate chromosome segregation”</p></li>
<li><p><strong>Ebru Aydin</strong><br />
Justus Liebig University Giessen, Germany “DEAD-box ATPase Dbp2 – mediates a specific mRNP remodelling checkpoint at the 3’-end of genes”</p></li>
</ul>
<p>See <a href="https://lists.cam.ac.uk/sympa/info/ucam-pombelist">pombelist</a> for Zoom meeting details.</p>
</div>
<div id="2024-03-20-pombetalks-march-20" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-march-20th">Next pombeTalks: March 20th</h3>
<p><em>2024-03-20</em></p>
<p>The next <a href="https://evonuclab.org/pombetalks">pombeTalks</a> session S04E06 will take place on Wednesday, March 20th.</p>
<p>The session will start at: 01:00 San Francisco / 04:00 New York / 08:00 London / 09:00 Paris / 13:30 Delhi / 16:00 Beijing / 17:00 Tokyo</p>
<p>The speaker is:</p>
<ul>
<li><strong>Ambre Noly</strong><br />
Helmlinger’s lab, Centre National de la Recherche Scientifique (CNRS), Montpellier, France<br />
“Exploring the adaptation and ecological role of fission yeasts”</li>
</ul>
<p>Zoom details:<br />
Topic: pombeTalks<br />
Meeting ID: 646 1193 8690<br />
Passcode: See pombelist</p>
</div>
<div id="2024-02-22-new-pombase-genetics-publication" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="new-database-update-publication-pombase">New database update publication from PomBase</h3>
<p><em>2024-02-22</em></p>
<p>The fission yeast 2024 database update “PomBase: a Global Core Biodata Resource—growth, collaboration, and sustainability” has been <a href="https://doi.org/10.1093/genetics/iyae007">published in <em>Genetics</em></a>. This update describes PomBase’s continuing growth and improvement over the past two years and showcases the depth of existing connections both within the biocuration ecosystem and between PomBase and its user community.</p>
<p>We discuss ways to track knowledge coverage including the tracking of unknown proteins, and the use of slimsets. We describe how we provide value by data and software sharing and how we have reused software provided by others in our new views. We provide author-led curation updates, describe contributions to global standards, future plans to support machine learning with high quality curated data and more.</p>
<p><a href="https://doi.org/10.1093/genetics/iyae007">Rutherford <em>et al.</em>, <em>Genetics</em>, iyae007</a></p>
<p><a href="assets/newsfeed/genetics_2024_paper_fig_5.png"><img src="assets/newsfeed/genetics_2024_paper_fig_5.png" title="Data dissemination from PomBase to other resources" class="screenshot" style="width:100.0%" loading="lazy" alt="“Fig 5 Data dissemination from PomBase to other resources”" /></a></p>
</div>
<div id="2024-02-15-pombetalks-feb-21" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-february-21st">Next pombeTalks: Wednesday, February 21st</h3>
<p><em>2024-02-15</em></p>
<p>The next <a href="https://evonuclab.org/pombetalks">pombeTalks</a> session S04E05 will take place on Wednesday, February 21st.</p>
<p>The session will start at: 08:00 San Francisco / 11:00 New York / 16:00 London / 17:00 Paris / 21:30 Delhi / 00:00 Beijing / 01:00 Tokyo.</p>
<p>Speakers are:</p>
<ul>
<li><p><strong>Francisco Prieto Ruiz</strong><br />
“Myosin II regulatory light chain phosphorylation and formin availability modulate cytokinesis upon changes in carbohydrate metabolism”</p></li>
<li><p><strong>Haoran Liu</strong> “The cross-regulation between Set1, Clr4, and Lsd1/2 in fission yeast”</p></li>
</ul>
<p>Zoom details:<br />
pombeTalks S04E05 Zoom Meeting<br />
Zoom meeting ID: 944 9125 6052<br />
Passcode: will be given the day of the meeting (and already available on pombeSlack)</p>
</div>
<div id="2024-02-08-modification-colours" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="viewing-protein-modifications">Viewing protein modifications</h3>
<p><em>2024-02-08</em></p>
<p>Modified amino acids in the protein feature viewer are now coloured by modification type. See the <a routerLink="/gene_protein_features/SPAC1834.04">histone H3.1 gene page</a> for an example.</p>
<p><a href="assets/newsfeed/gene-hht1-protein-feature-viewer-page.png"><img src="assets/newsfeed/gene-hht1-protein-feature-viewer-page.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Protein feature view for hht1" /></a></p>
<p><a href="assets/newsfeed/protein-feature-modification-key.png"><img src="assets/newsfeed/protein-feature-modification-key.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Protein feature view modification colour key" /></a></p>
<p>Visit the <a routerLink="/documentation/gene-page-protein-features-widget">protein feature view documentation page</a> for more information.</p>
</div>
<div id="2024-02-07-sent-substrates-to-advanced-search" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="access-target-genes-individual-gene-products">Access target genes of individual gene products</h3>
<p><em>2024-02-07</em></p>
<p>It is now possible to send lists of annotated molecular function substrates to the advanced search from gene pages:</p>
<p><a href="assets/newsfeed/cdc2-substrates-to-advanced-search.png"><img src="assets/newsfeed/cdc2-substrates-to-advanced-search.png" class="screenshot" style="width:100.0%" loading="lazy" alt="cdc2 - send target genes to the Advanced Search" /></a></p>
<p>Lists of annotated molecular function substrates of individual genes (i.e protein kinase or transcription factor target genes) can now be sent directly to the Advanced Search. You can send complete lists, or pre-filtered by cell cycle phase.</p>
</div>
<div id="2024-01-17-pombetalks-jan-17" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-january-17th">Next pombeTalks: Wednesday, January 17th</h3>
<p><em>2024-01-17</em></p>
<p>The next pombeTalks session will take place on Wednesday, January 17th.</p>
<p>Please note that this session will start at: 08:00 San Francisco / 11:00 New York / 16:00 London / 17:00 Paris / 21:30 Delhi / 00:00 Beijing / 01:00 Tokyo.</p>
<p>Speakers are:</p>
<ul>
<li><p><strong>Melvin Berard</strong>, Sophie Martin’s Lab, University of Geneva, Geneva, Switzerland<br />
“Phosphoproteomics analysis during mating reveals TORC1 re-activation upon pheromone signalling.”</p></li>
<li><p><strong>Priyanka Sasmal</strong>, Phong Tran lab, Institut Curie, Paris, France<br />
“Chimeric Kinesin-6 and Kinesin-14 reveal directional and functional control in fission yeast.”</p></li>
</ul>
<p>Zoom details:<br />
pombeTalks S04E04 Zoom Meeting<br />
Join <a href="https://unige.zoom.us/j/63207162782?pwd=eW1FMWJDSkErelZOMTk3VzNmenpWUT09">Zoom Meeting-Wednesday Jan 17th 2024-5pm CET</a><br />
Meeting ID: 632 0716 2782<br />
Passcode: will be given the day of the meeting (and already available on pombeSlack)</p>
</div>
<div id="2023-12-14-ELIXIR-core-data-resource" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/elixir_uk_thumb.png"/></p>
<h3 id="pombase-joins-elixir-core-data-resources">PomBase joins ELIXIR Core Data Resources</h3>
<p><em>2023-12-14</em></p>
<p>PomBase is the first model organism database (MOD) to be selected as an <a href="https://elixir-europe.org">ELIXIR</a> <a href="https://elixir-europe.org/platforms/data/core-data-resources">Core data resource (CDR)</a>. We were evaluated across a long list of indicators including scientific focus and quality, governance, and quality of service. ELIXIR CDRs are European life science data resources that are important to biological and biomedical research and long-term preservation of biological data. This will provide assurance to our funders and users that PomBase satisfies the many criteria required to considered a key part of the European bioinformatics infrastructure. <a href="https://elixir-europe.org">ELIXIR</a> is an intergovernmental organisation that brings together life science resources from across Europe, it currently includes <a href="https://elixir-europe.org/about-us/who-we-are">22 member countries</a>, bringing together over 240 research organisations. The goal of
ELIXIR is to coordinate these resources so that they form a single infrastructure.</p>
<p><a href="assets/newsfeed/elixir_pombase_cdr_announcement.png"><img src="assets/newsfeed/elixir_pombase_cdr_announcement.png" title="ELIXIR CDR announcement" class="screenshot" style="width:100.0%" loading="lazy" alt="ELIXIR CDR announcement" /></a></p>
</div>
<div id="2023-12-06-pombetalks-dec-13" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-december-13th">Next pombeTalks: Wednesday, December 13th</h3>
<p><em>2023-12-06</em></p>
<p>The next pombeTalks session of season 4 will take place on Zoom on <strong>Wednesday, December 13th</strong></p>
<p>Please note that this session will start at midnight San Francisco / 3:00 New York / 08:00 London / 09:00 Paris / 13:30 Delhi / 16:00 Beijing / 17:00 Tokyo.</p>
<p>Speakers are:</p>
<ul>
<li><p><strong>Xiao-Hui Lyo</strong>, Dr. Li-Lin Du’s Lab, National Institute of Biological Sciences, Beijing, China<br />
“An improved tetracycline-inducible expression system for fission yeast”</p></li>
<li><p><strong>Yuichi Morozumi</strong>, Watanabe lab, Nara Institute of Science and Technology, Nara, Japan<br />
“Identification of mechanisms that restrict cell growth below temperatures detrimental to cellular physiology”</p></li>
</ul>
<p>The Zoom details will be shared on the week of the talk, through pombeList and pombeSlack.</p>
<p>For more fission yeast related topics and recordings of the talks, join <a href="http://spombe.slack.com">pombeSlack</a>, where additional questions can also be posted on the <code>#pombetalks-qna</code> channel.</p>
</div>
<div id="2023-11-12-interpro-release-97" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-updated-use-interpro-version-97.0">PomBase updated to use InterPro Version 97.0</h3>
<p><em>2023-11-12</em></p>
<p>PomBase now uses InterPro Version 97.0.</p>
<p>Features include:</p>
<ul>
<li>The addition of 274 InterPro entries</li>
<li>An update to NCBIfam (13.0), PANTHER (18.0)</li>
<li>Integration of 285 new methods from the NCBIfam (80), Pfam (106), CATH-Gene3D (1), PANTHER (43), CDD (55) databases</li>
</ul>
</div>
<div id="2023-11-06-pombetalks-nov-15th" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-november-15th">Next pombeTalks: Wednesday, November 15th</h3>
<p><em>2023-11-06</em></p>
<p>The second pombeTalks session of season 4 will take place on Zoom on <strong>Wednesday, November 15th</strong></p>
<p>Please note that this session will start at 08:00 San Francisco / 11:00 New York / 16:00 London / 17:00 Paris / 21:30 Delhi / 00:00 Beijing / 01:00 Tokyo.</p>
<p>This round of speakers are:</p>
<ul>
<li><p><strong>Kuang Shen</strong>, University of Massachusetts Medical School, Worcester, USA<br />
“Structural evolution of the Rag-LAMTOR complex in mTORC1 pathway”</p></li>
<li><p><strong>Abhishek Kumar</strong>, Friedman’s Lab, UT Southwestern Medical Center, Dallas, USA<br />
“A DRP-like pseudoenzyme coordinates with MICOS to promote cristae architecture”</p></li>
</ul>
<p>The Zoom details will be shared on the week of the talk, through pombeList and pombeSlack.</p>
<p>For more fission yeast related topics and recordings of the talks, join <a href="http://spombe.slack.com">pombeSlack</a>, where additional questions can also be posted on the <code>#pombetalks-qna</code> channel.</p>
</div>
<div id="2023-10-11-new-docs-and-video-site" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="new-documentation-and-videos-canto">New documentation and videos for Canto</h3>
<p><em>2023-10-11</em></p>
<p>We have published <a href="https://pombase.github.io/canto_tutorial/">a new website</a> with videos and documentation for <a href="https://curation.pombase.org/">Canto</a>, our community curation tool. The website includes short step-by-step text and video guides on how to make annotations in Canto.</p>
<ul>
<li><a href="https://pombase.github.io/canto_tutorial/">New Canto documentation front page</a></li>
<li><a href="https://www.youtube.com/@PomBase">PomBase YouTube channel</a></li>
</ul>
<p>The work on this website was supported by the ELIXIR-UK: FAIR Data Stewardship training UKRI award (MR/V038966/1) to Manuel Lera Ramírez.</p>
<p><a href="assets/newsfeed/new_canto_video_docs.png"><img src="assets/newsfeed/new_canto_video_docs.png" title="New Canto tutorials video list" class="screenshot" style="width:100.0%" loading="lazy" alt="New Canto tutorials video list" /></a></p>
</div>
<div id="2023-10-10-pombetalks-oct-18th" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-october-18th">Next pombeTalks: Wednesday, October 18th</h3>
<p><em>2023-10-10</em></p>
<p>The first <a href="https://evonuclab.org/pombetalks">pombeTalks</a> session of season 4 will take place on <strong>Wednesday, October 18th</strong></p>
<p>This session will start at 8:00 San Francisco / 11:00 New York / 16:00 London / 17:00 Paris / 20:30 Delhi / 23:00 Beijing / 00:00 Tokyo.</p>
<p>Speakers:</p>
<ul>
<li><p><strong>Alaina Willet</strong>, Kathy Gould’s lab, Vanderbilt University, USA<br />
“Membrane binding of endocytic myosin-1s is inhibited by a class of ankyrin repeat proteins”</p></li>
<li><p><strong>Silke Hauf</strong>, Virgina Tech, USA<br />
“Different rules for ultra-low gene expression noise?”</p></li>
</ul>
<p>The Zoom details will be shared on the week of the talk, through pombeList and pombeSlack.</p>
<p>For more fission yeast related topics and recordings of the talks, join <a href="http://spombe.slack.com">pombeSlack</a>, where additional questions can also be posted on the <code>#pombetalks-qna</code> channel.</p>
</div>
<div id="2023-09-30-nomenclature-paper" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="revised-fission-yeast-nomenclature-genes-and-alleles">Revised fission yeast nomenclature for genes and alleles</h3>
<p><em>2023-09-30</em></p>
<p>The improved nomenclature guidelines for fission yeast clade genes and alleles have been <a href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyad143/7283026">published in Genetics GSA</a>. This update introduces nomenclature guidelines for noncoding RNA genes, a significant update to the allele and genotype nomenclature guidelines originally published in 1987. We took several steps to improve machine readability that will benefit the artificial intelligence/machine learning communities, and our own community curation program.</p>
<p><a href="https://doi.org/10.1093/genetics/iyad143">Lera-Ramírez <em>et al.</em> 2023, Revised fission yeast gene and allele nomenclature guidelines for machine readability, Genetics, 2023;, iyad143</a></p>
<p><a href="assets/newsfeed/nomenclature_paper_for_news_big.png"><img src="assets/newsfeed/nomenclature_paper_for_news.png" title="Revised fission yeast gene and allele nomenclature guidelines for machine readability" class="screenshot" style="width:100.0%" loading="lazy" alt="“Revised fission yeast gene and allele nomenclature guidelines for machine readability”" /></a></p>
</div>
<div id="2023-09-20-list-of-publication-genes" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="accessing-list-genes-paper">Accessing the list of genes from a paper</h3>
<p><em>2023-09-20</em></p>
<p>It’s now possible to view the list of genes associated with a paper, using a new link on the publication pages. The gene list can then optionally be sent to the Advanced Search.</p>
<p><a href="assets/newsfeed/genes-list-from-ref-page.png"><img src="assets/newsfeed/genes-list-from-ref-page.png" title="Publication page, highlightling the new genes link" class="screenshot" style="width:100.0%" loading="lazy" alt="Publication page, highlightling the new genes link" /></a></p>
</div>
<div id="2023-07-27-protein-feature-viewer" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="protein-feature-viewer-added-gene-pages">Protein feature viewer added to gene pages</h3>
<p><em>2023-07-27</em></p>
<p>PomBase gene pages now include a protein feature widget. This tool shows protein features in the context of amino acid sequence. It includes:</p>
<ul>
<li>manually curated amino acid substitution positions</li>
<li>Pfam domains</li>
<li>protein modifications</li>
<li>protein properties: low complexity regions, disordered regions, coiled coil regions and predicted trans-membrane domains</li>
</ul>
<p>Visit the <a routerLink="/gene/SPAC20G8.05c">cdc15 gene page</a> to see an example.</p>
<p>Hover over features for more information, such as allele descriptions and Pfam domain IDs.</p>
<p><a href="assets/gene-protein-feature-viewer-widget-cdc15.png"><img src="assets/gene-protein-feature-viewer-widget-cdc15.png" title="Protein feature viewer widget - cdc15 gene page" class="screenshot" style="width:100.0%" alt="Protein feature viewer widget - cdc15 gene page" /></a></p>
<p>Follow the “View all protein features …” link for a detailed view on the dedicated protein features page that includes:</p>
<ul>
<li>details of the manually curated amino acid substitution allele changes</li>
<li>manually curated partial amino acid deletions</li>
</ul>
<p><a href="assets/gene-protein-feature-viewer-page-cdc15.png"><img src="assets/gene-protein-feature-viewer-page-cdc15.png" title="Protein feature viewer details page for cdc15" class="screenshot" style="width:100.0%" alt="Protein feature viewer details page for cdc15" /></a></p>
<p>Thanks to the team at <a href="https://www.rcsb.org/">RCSB PDB</a> for providing the <a href="https://github.com/rcsb/rcsb-saguaro">Open Source software</a> used to implement this feature.</p>
</div>
<div id="2023-06-19-rhea-reaction-diagrams" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="reaction-diagrams-on-go-molecular-function-term-pages">Reaction diagrams on GO molecular function term pages</h3>
<p><em>2023-06-19</em></p>
<p>Where available, we now show the reaction diagram from <a href="https://www.rhea-db.org/">Rhea</a> on GO molecular function term pages. This feature is possible thanks to the great work of <a href="https://www.rhea-db.org/">Rhea</a>.</p>
<p>See the <a routerLink="/term/GO:0003849">3-deoxy-7-phosphoheptulonate synthase activity (GO:0003849) term page</a> for an example.</p>
<p><a href="assets/newsfeed/rhea-reaction-example.png"><img src="assets/newsfeed/rhea-reaction-example.png" class="screenshot" style="width:100.0%" loading="lazy" /></a></p>
</div>
<div id="2023-04-26-merging-nonsense-alleles" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="curation-update-nonsense-mutation-merged-into-partial-amino-acid-deletion">Curation update - “nonsense mutation” merged into “partial amino acid deletion”</h3>
<p><em>2023-04-26</em></p>
<p>We have decided to merge the allele type “nonsense mutation” into “partial amino acid deletion”. This has mainly been driven by the fact that allele types that combine different variants require new types, such as “amino_acid_deletion_and_mutation”, “amino_acid_insertion_and_deletion”, etc. Otherwise, we would have ended up with many more types, and at the gene product level (which is what we describe in PomBase in phenotype interactions), both truncations are equivalent. In the next update, this allele type will not be available in Canto.</p>
<p>In any case, even if two alleles produce the same truncation, such as ase1-D13* or ase1Δ(13-731), they would still have separate entries in PomBase, and they may have different phenotypes. We are only assigning them the same allele type.</p>
<p>If for your analysis you need to make a distinction between the two using our allele dataset <a routerLink="/downloads/phenotype-annotations">allele dataset</a>, you can always check the “Allele description” field for the presence of the “*” character to tell whether an allele includes a nonsense mutation.</p>
</div>
<div id="2023-02-22-pdb-structures-gene-pages" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/protein-structure.png"/></p>
<h3 id="experimental-structures-pdb-on-gene-pages">Experimental structures from PDB on gene pages</h3>
<p><em>2023-02-22</em></p>
<p>The experimental protein structures from <a href="https://www.ebi.ac.uk/pdbe/">PDB</a> are now embedded on the PomBase gene pages using <a href="https://molstar.org/">Mol*</a>. For example: <a routerLink="/gene/SPCC285.12">lsm7/SPCC285.12 gene page</a></p>
<p>If you select the “PDB structures” view on a gene page, experimental structures will set as your default. AlphaFold predictions will be shown for genes where an experimental structure are not available.</p>
<p>We now also display the structures on the associated publication page. For example: <a routerLink="/reference/PMID:31010807">PMID:31010807 Garg <em>et al.</em></a></p>
<p>To help locate proteins with experimental protein structures (currently 375), we have added a new query option to the “<a routerLink="/query">Advanced search</a>”, currently under “commonly used queries”: “<a routerLink="/results/from/id/6e8401aa-bab8-48fc-bbb7-c85d13ea10a2">Proteins with PDB structures</a>”</p>
<p><a href="assets/newsfeed/pdb_structures_lsm7_small.png"><img src="assets/newsfeed/pdb_structures_lsm7_small.png" class="screenshot" style="width:100.0%" loading="lazy" alt="PDB structures on the lsm7 gene page" /></a></p>
</div>
<div id="2023-02-02-alphafold-on-gene-pages" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/protein-structure.png"/></p>
<h3 id="alphafold-protein-structure-on-gene-pages">AlphaFold protein structure on gene pages</h3>
<p><em>2023-02-02</em></p>
<p>AlphaFold protein structure are now embedded on the PomBase gene pages. We hope to embed the experimental structures from PDB in the near future.</p>
<p><a href="assets/newsfeed/alphafold-news-item-example-1.png"><img src="assets/newsfeed/alphafold-news-item-example-1.png" class="screenshot" style="width:100.0%" loading="lazy" alt="Example from the mvp1 gene page" /></a></p>
</div>
<div id="2023-01-24-new-utrs-from-tss" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="revised-canonical-5-utrs">Revised canonical 5’ UTRs</h3>
<p><em>2023-01-24</em></p>
<p>We have revised the curated 5’UTRs using Transcription Start Sites (TSS) data (in vegetative growth/ minimal media) from the Cap Analysis of Gene Expression (CAGE) data provided by <a routerLink="/reference/PMID:30566651">Thodberg <em>et al</em></a>. All new gene structures were manually reviewed, around ~80 protein features had N-terminal coordinate revisions to align with TSS data.</p>
</div>
<div id="2022-12-15-gcbr-announce" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/gbc-news-icon.png"/></p>
<h3 id="global-core-biodata-resource-status-pombase">Global Core Biodata Resource status for PomBase</h3>
<p><em>2022-12-15</em></p>
<p>We are very pleased to announce that PomBase has been selected as one of the first <a href="https://globalbiodata.org/scientific-activities/global-core-biodata-resources">Global Core Biodata Resource (GCBR)</a> — a collection of 37 resources whose long term funding and sustainability is critical to life science and biomedical research worldwide. This accreditation recognizes PomBase as a primary knowledge base (adding value to data through expert curation) and as a crucial component of the research ecosystem. The candidate biodata resources were assessed against a series of rigorous criteria that included their scientific focus, the size and reach of their user communities, their quality of service, their governance, and their impact on global research.</p>
<p>Thank you to the entire community, especially the community curators who contribute regularly to our content, and our <a routerLink="/about/pombase-sab">Scientific Advisory Board</a> for their help and support.</p>
<p>For more information about the <a href="https://globalbiodata.org/">Global Biodata Coalition</a> and PomBase’s new status, see <a href="https://globalbiodata.org/global-biodata-coalition-announces-the-first-set-of-global-core-biodata-resources/">the full press release</a>.</p>
</div>
<div id="2022-12-08-pombetalks-14th-dec" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-december-14th">Next PombeTalks: Wednesday, December 14th</h3>
<p><em>2022-12-08</em></p>
<p>The next online PombeTalks will take place on <strong>Wednesday, December 14th</strong>. These talks are virtual seminars by and for the fission yeast community and friends.</p>
<p>8:00 San Francisco; 11:00 NY; 16:00 London; 17:00 Paris; 20:30 Delhi, 23:00 Beijing; midnight Tokyo</p>
<p>This will be our last PombeTalks of 2022 before taking a winter break. The speaker will be:</p>
<ul>
<li><strong>Kristi E. Miller</strong><br />
“The fission yeast cell size control system integrates pathways measuring cell surface area, volume and time”</li>
</ul>
<p>It will be followed by some sum up/ feedback about PombeTalks from the organizing committee.</p>
<h6 id="survey">Survey</h6>
<p>Please help us improve PombeTalks even more by taking <a href="https://forms.gle/cKuqdkuBRVBA7KPi6">this quick survey</a></p>
<h6 id="zoom-details">Zoom details</h6>
<p>Topic: PombeTalksS308 Zoom Meeting<br />
Date: Dec 14th<br />
Meeting ID: 975 0331 6190<br />
Password: will be sent the day of the meeting</p>
<h6 id="slack">Slack</h6>
<p>For more fission yeast related topics and recordings of the talks, join <a href="http://spombe.slack.com">pombeSlack</a>, where additional questions can also be posted on the <code>#pombetalks-qna</code> channel.</p>
</div>
<div id="2022-12-07-rna-length-advanced-search" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="querying-by-rna-length-advanced-search-tool">Querying by RNA length in the Advanced Search tool</h3>
<p><em>2022-12-07</em></p>
<p>You can now query the RNA length of genes (spliced or unspliced) under the “Transcripts and exons” query grouping in the <a href="https://www.pombase.org/query">Advanced search</a>.</p>
<p>You can also add RNA sequence length as a field in tables downloaded from the query builder.</p>
<p><img src="assets/newsfeed/advanced-search-rna-length-query.png" class="screenshot" style="width:100.0%" loading="lazy" alt="RNA length queries are available under the “Transcripts and exons” tab" /></p>
</div>
<div id="2022-11-29-new-gi-announcement" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="genetic-interaction-annotation-model-updated">Genetic Interaction annotation model updated</h3>
<p><em>2022-11-29</em></p>
<p>We have recently implemented an improved way to <strong>annotate and display genetic interactions so they are linked to phenotype annotations and alleles</strong>.</p>
<p>Previously, our Genetic Interaction annotations only mentioned the interacting genes and the type of genetic interaction. For example, if a strain with genotype <code>asp1-H397A</code> has the phenotype <code>decreased acid phosphatase activity affecting activity of pho1</code>, but this phenotype is suppressed when <code>rhn1</code> is deleted in that strain, we would annotate that the genes <code>asp1</code> and <code>rhn1</code> are part of a <code>Phenotypic Suppression</code> interaction.</p>
<p><img src="assets/newsfeed/gi-update-1.png" class="screenshot" style="width:100.0%" loading="lazy" alt="GI old format" /></p>
<p>We continue to display interactions in this format by default (showing only genes and interaction type), but if you expand the annotation, you can view the associated genotypes and phenotypes.</p>
<p><img src="assets/newsfeed/gi-update-2.png" class="screenshot" style="width:100.0%" loading="lazy" alt="GI new format" /></p>
<p>In the revised Canto interface, you can only annotate a genetic interaction from the double mutant phenotype annotation (by clicking on <code>add..</code>, as shown below).</p>
<p><img src="assets/newsfeed/gi-update-3.png" class="screenshot" style="width:100.0%" loading="lazy" alt="GI annotation in Canto" /></p>
<p>Of course, genetic interactions predating this update are not linked to phenotypes or genotypes, but we are hoping to auto-annotate several of those. We will also prioritise for update any interactions where community curators have provided these details in an annotation comment. Finally, <strong>a big shoutout to Ana Sanchez and Angad Garg from the Shuman lab</strong>, for testing the new interface in numerous recently curated publications. The examples provided here are from <a routerLink="/reference/PMID:31276588">Sanchez et al. 2019</a>. Go read it and see the annotations in PomBase.</p>
<p>Best, The PomBase team</p>
</div>
<div id="2022-11-11-pombetalks-16th-nov" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-november-16th">Next pombeTalks: Wednesday, November 16th</h3>
<p><em>2022-11-11</em></p>
<p>The next online pombeTalks will take place on <strong>Wednesday, November 16th</strong>. These talks are virtual seminars by and for the fission yeast community and friends.</p>
<p>Note that this session will happen earlier at:<br />
midnight San Francisco; 03:00 NY; 08:00 London; 09:00 Paris; 13:30 Delhi, 16:00 Beijing; 17:00 Tokyo</p>
<p>Talks this session:</p>
<ul>
<li><p><strong>Wenfan Wei</strong>, University of Science and Technology of China<br />
“The Cdc42 GAP Rga6 promotes monopolar outgrowth of spores”</p></li>
<li><p><strong>Gaowen Liu</strong>, Shenzhen Institute of Synthetic Biology<br />
“Fusion eciency evolution to the deletion of an essential mating gene Prm1”</p></li>
</ul>
<h6 id="zoom-details-1">Zoom details</h6>
<p>Topic: PombeTalks S03E07 Zoom Meeting<br />
Date: Nov 16, 2022<br />
Time: midnight San Francisco; 03:00 NY; 08:00 London; 09:00 Paris; 13:30 Delhi, 16:00 Beijing; 17:00 Tokyo<br />
Meeting ID: 932 8857 4852<br />
Password: will be sent the day of the meeting</p>
<h6 id="slack-1">Slack</h6>
<p>For more fission yeast related topics and recordings of the talks, join <a href="http://spombe.slack.com">pombeSlack</a>, where additional questions can also be posted on the <code>#pombetalks-qna</code> channel.</p>
</div>
<div id="2022-10-14-Alpha-fold_RBSH" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/protein-structure.png"/></p>
<h3 id="using-alphafold-models-discover-distant-human-and-budding-yeast-homologues">Using AlphaFold models to discover distant human and budding yeast homologues</h3>
<p><em>2022-10-14</em></p>
<p>We collaborated with the Pfam group at the EBI to evaluate predictions generated from AlphaFold reciprocal best structure hits to identify potential distant orthologs. The Reciprocal Best Structure Hits (RBSH) approach provided 11 novel human homologues, including <strong>Pho86 -&gt; NAT8</strong> (ER acetyltransferase), <strong>Mug174 -&gt; COIL</strong> (Coilin), <strong>Ach1 -&gt; OXCT1</strong> (succinyl-CoA:3-ketoacid coenzyme A transferase), <strong>SPAC1952.08c -&gt; CREG1</strong>, <strong>imt1 -&gt; A4GALT</strong> (Lactosylceramide 4-alpha-galactosyltransferase), <strong>Rtc5 -&gt; MEAK7</strong> (MTOR associated protein). A further 41 novel orthologs were predicted between <em>S. pombe</em> and <em>S. cerevisiae</em> which had fallen under the radar for all other methods used at PomBase. Most of the novel connections provided additional functional information, or supported existing knowledge for poorly characterised proteins. See supporting data tables S4 and S5 for the
complete list of predictions included in PomBase. <a href="https://academic.oup.com/bioinformaticsadvances/advance-article/doi/10.1093/bioadv/vbac072/6749558">Article</a>.</p>
</div>
<div id="2022-10-13-pombetalks-oct-19th" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-october-19th">Next pombeTalks: Wednesday, October 19th</h3>
<p><em>2022-10-13</em></p>
<p>The next online pombeTalks will take place on <strong>Wednesday, October 19th</strong>. These talks are virtual seminars by and for the fission yeast community and friends.</p>
<p>8:00 San Francisco / 11:00 NY / 16:00 London / 17:00 Paris / 20:30 Delhi / 23:00 Beijing / midnight Tokyo</p>
<p>Talks this session:</p>
<ul>
<li><p><strong>Cecilia D’Alessio</strong>, University of Buenos Aires and CONICET<br />
“N-Glycosylation and glycoprotein folding in fission yeasts, a model to study human congenital disorders of glycosylation”</p></li>
<li><p><strong>Jason Tanny</strong>, McGill University, Montreal<br />
“A novel transcriptional mechanism regulating the cellular response to replication stress”</p></li>
</ul>
<h6 id="zoom-details-2">Zoom details</h6>
<p>Topic: PombeTalks S03E06 Zoom Meeting<br />
Time: Oct 19, 2022 05:00 PM Paris<br />
Meeting ID: 933 7072 6178<br />
Password: will be sent the day of the meeting</p>
<h6 id="slack-2">Slack</h6>
<p>For more fission yeast related topics and recordings of the talks, join <a href="http://spombe.slack.com">pombeSlack</a>, where additional questions can also be posted on the <code>#pombetalks-qna</code> channel.</p>
</div>
<div id="2022-10-13-new-to-pombase" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="new-fission-yeast-check-out-our-quick-start-guide-new-users">New to fission yeast? check out our quick start guide for new users</h3>
<p><em>2022-10-13</em></p>
<p>We have added a new page providing useful information for new fission yeast researchers:<br />
<a routerLink="/documentation/getting-started">Getting started with <em>S.pombe</em> and PomBase</a></p>
<p>Includes details of how to join the community mailing list and the Slack channel, links to useful tools and resources, information about fission yeast as a model organism, an overview of PomBase and more.</p>
</div>
<div id="2022-09-16-new-disease-associations" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/human_disease.png"/></p>
<h3 id="id-77-new-disease-gene-associations">77 new disease gene associations</h3>
<p><em>2022-09-16</em></p>
<p>We have added 77 new disease-gene associations for 71 fission yeast human gene orthologs. These were identified using “PombeMine” to identify the disease genes curated by OMIM not annotated with an existing MONDO mapping. The number of human disease gene associations is currently 1471. Disease genes can be browsed via the <a routerLink="/browse-curation/disease-slim">disease slim set</a> or from the <a routerLink="/term/MONDO:0000001">MONDO root node term</a>.</p>
</div>
<div id="2022-09-16-pombemine" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/intermine.png"/></p>
<h3 id="pombemine-intermine-instance-s.-pombe">PombeMine: an InterMine instance for <em>S. pombe</em></h3>
<p><em>2022-09-16</em></p>
<p>As part of an Elixir funded collaboration with the <a href="http://intermine.org/">InterMine team</a> we have created <a href="https://www.pombase.org/im/pombemine">PombeMine</a>. Gene lists can be sent directly from PomBase query results pages directly to Intermine (under the “export” tab), providing direct (2 click) access to GO and phenotype enrichment tools.</p>
</div>
<div id="2022-09-15-pombetalks-sep-21" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-september-21st">Next pombeTalks: Wednesday, September 21st</h3>
<p><em>2022-09-15</em></p>
<p>The next online pombeTalks will take place on <strong>Wednesday, September 21st</strong>. These talks are virtual seminars by and for the fission yeast community and friends.</p>
<p>8:00 San Francisco / 11:00 NY / 16:00 London / 17:00 Paris / 20:30 Delhi / 23:00 Beijing / midnight Tokyo</p>
<p>Talks this session:</p>
<ul>
<li><p><strong>Manuel Lera Ramirez</strong>, PomBase / Tran Lab, Institute Curie <em>“Microtubule rescue at midzone edges promotes overlap stability and prevents spindle collapse during anaphase B”</em></p></li>
<li><p><strong>Hannah Opalko</strong>, Moseley lab, Dartmouth College <em>“Mechanisms of spatial patterning of cell cycle regulator Cdr2”</em></p></li>
</ul>
<h6 id="zoom-details-3">Zoom details</h6>
<p>Topic: PombeTalks S03E05 Zoom Meeting<br />
Time: Sep 21, 2022 05:00 PM Paris<br />
Meeting ID: 985 8572 1420<br />
Password: will be sent the day of the meeting</p>
<h6 id="slack-3">Slack</h6>
<p>For more fission yeast related topics and recordings of the talks, join <a href="http://spombe.slack.com">pombeSlack</a>, where additional questions can also be posted on the <code>#pombetalks-qna</code> channel.</p>
</div>
<div id="2022-09-06-protein-sequence-changes" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="protein-sequence-changes">Protein sequence changes</h3>
<p><em>2022-09-06</em></p>
<p>78 protein have been shortened (at the N-term). This set includes 34 published proteins (Apl4, Brc1, Cdc48, Cdt1, Cho1, Cmb1, Cut2, Cut6, Cwf26, Dbr1, Dri1, Elo2, Eri1, Lsd2, Lys2, Med13, Naa38, Nup107, Nup82, Orc2, Pof10, Ppt1, Rec24, Rga2, Rmt3, Rns1, RRpn7, Sap145, Skb1, Snf5, Snt2, Spn3, Tpp2, Trm1, and Tup12). Allele description changes and modification position changes are pending.</p>
</div>
<div id="2022-08-13-pombetalks-aug-13" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-august-17th">Next pombeTalks: Wednesday, August 17th</h3>
<p><em>2022-08-13</em></p>
<p>The next online pombeTalks will take place on <strong>Wednesday, August 17th</strong>. These talks are virtual seminars by and for the fission yeast community and friends.</p>
<p>8:00 San Francisco / 11:00 NY / 16:00 London / 17:00 Paris / 20:30 Delhi / 23:00 Beijing / midnight Tokyo</p>
<p>Talks this session:</p>
<ul>
<li><strong>Pranas Grigaitis</strong>, Vrije Universiteit Amsterdam<br />
<em>“Prediction of metabolic strategies in Schizosaccharomyces pombe based on optimal resource allocation”</em></li>
<li><strong>Abhishek Poddar</strong>, University of Toledo<br />
<em>“Membrane stretching activates calcium-permeability of a putative channel Pkd2 during fission yeast cytokinesis”</em></li>
</ul>
<p>As always, connection details will be sent the day of the talk. For more fission yeast related topics and recordings of the talks, join <a href="http://spombe.slack.com">pombeSlack</a>, where additional questions can also be posted on the #pombetalks-qna channel.</p>
</div>
<div id="2022-07-12-pombetalks-july-20th" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="next-pombetalks-wednesday-july-20th">Next pombeTalks: Wednesday, July 20th</h3>
<p><em>2022-07-12</em></p>
<p>The next online pombeTalks will take place on Wednesday, July 20th. These talks are virtual seminars by and for the fission yeast community and friends.</p>
<p>0:00 San Francisco / 3:00 New York / 8:00 London / 9:00 Paris / 12:30 Delhi / 15:00 Beijing / 16:00 Tokyo.</p>
<p>Talks this session:</p>
<ul>
<li>Leeba Ann Chacko, Ananthanarayanan Lab, University of New South Wales<br />
<em>“Microtubules and mitochondria cooperate to ensure cell division symmetry, polarity and equipartitioning in fission yeast”</em></li>
<li>Dan Zhang, Temasek Life Sciences Laboratory, National University of Singapore<br />
<em>“The cortical ER remodeling for actomyosin ring assembly”</em></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and <a href="http://spombe.slack.com">pombeSlack</a> one day in advance, so make sure you’ve signed up.</p>
</div>
<div id="2022-06-14-pombetalks-june15-2022" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-wednesday-june-15th">PombeTalks Wednesday June 15th</h3>
<p><em>2022-06-14</em></p>
<p>The next online pombeTalks will take place on Wednesday June 15th. These are virtual seminars by and for the fission yeast community and friends.</p>
<p>08:00 San Francisco / 11:00 New York / 16:00 London / 17:00 Paris / 20:30 Delhi / 23:00 Beijing / 00:00 Tokyo</p>
<p>Talks this session:</p>
<ul>
<li>Elliott Chapman, Bayne Lab, University of Edinburgh<br />
<em>“Separable roles for RNAi in regulation of transposable elements and viability in the fission yeast Schizosaccharomyces japonicus”</em></li>
<li>Fei Li, New York University<br />
<em>“Phosphorylation-mediated Ccp1-Ndc80 switch at the N-terminus of CENP-T regulates kinetochore assembly in fission yeast”</em></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and <a href="http://spombe.slack.com">pombeSlack</a> one day in advance, so make sure you’ve signed up.</p>
<p>As before, questions will be posted to <a href="https://spombe.slack.com/archives/C013C1NRY80">#pombetalks-qna</a> on <a href="http://spombe.slack.com">pombeSlack</a> channel and recordings uploaded.</p>
</div>
<div id="2022-05-05-pombetalks-may18-2022" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-may-18th">PombeTalks May 18th</h3>
<p><em>2022-05-05</em></p>
<p>The next online pombeTalks will take place on May 18th. These are virtual seminars by and for the fission yeast community and friends.</p>
<p>08:00 San Francisco / 11:00 New York / 16:00 London / 17:00 Paris / 20:30 Delhi / 23:00 Beijing / 00:00 Tokyo</p>
<p>Talks this session:</p>
<ul>
<li><strong>Jennifer Porat</strong>, Bayfield Lab, York University: The methyl phosphate capping enzyme Bmc1/Bin3 is a stable component of the fission yeast telomerase holoenzyme</li>
<li><strong>Ingrid Billault-Chaumartin</strong>, Martin Lab, UNIL: Fus1, the fusion focus formin</li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
</div>
<div id="2022-04-13-phenotype-slim" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="new-phenotype-slim">New phenotype slim</h3>
<p><em>2022-04-13</em></p>
<p>We have added a <a routerLink="/browse-curation/fypo-slim">phenotype slim overview</a> to complement those provided for <a routerLink="/browse-curation/disease-slim">disease association</a>, <a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">biological process</a>, <a routerLink="/browse-curation/fission-yeast-mf-go-slim-terms">molecular function</a> and <a routerLink="/browse-curation/fission-yeast-cc-go-slim-terms">cellular component</a> annotation. The purpose of the phenotype slim is to provide subsets of commonly used ‘broad’ phenotypic classes or annotation subsets that can provide a useful starting point for accessing phenotype lists. The phenotype slim page provides links to ontology term pages, annotated genes, and to download files containing the slim terms and IDs.</p>
<p>The phenotype slim has also been added to the PomBase <a routerLink="/query">advanced search</a> results panel “Slim with” menu. For example, you can query for all genes involved in a GO process, another phenotype, or any other list, and “slim” the results using the “Phenotype slim” option to view categories of phenotypes assigned to the list.</p>
</div>
<div id="2022-04-04-genetics-publications" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="making-biological-knowledge-useful-humans-and-machines">Making biological knowledge useful for humans and machines</h3>
<p><em>2022-04-04</em></p>
<p>A <a href="https://academic.oup.com/genetics/issue/220/4">GENETICS special issue</a> featuring model organism database updates is published today. This issue features the recent PomBase and <a href="https://www.japonicusdb.org/">JaponicusDB</a> publications and is accompanied by an editorial <a href="https://academic.oup.com/genetics/article-abstract/220/4/iyac001/6563297">“Making biological knowledge useful for humans and machines”</a> co-authored by Val Wood, Paul Sternberg and Howard Lipshitz.</p>
<ul>
<li><a href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyab222/6481557?login=false">Fission stories: using PomBase to understand Schizosaccharomyces pombe biology</a></li>
<li><a href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyab223/6481558?login=false">JaponicusDB: rapid deployment of a model organism database for an emerging model species</a></li>
</ul>
</div>
<div id="2022-03-26-1000-community-curated-publications" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="the-fission-yeast-community-have-now-curated-over-1000-publications">The fission yeast community have now curated over 1000 publications</h3>
<p><em>2022-03-26</em></p>
<p>We would like to extend a huge “thank you” to the fission yeast community for curation contributions. The community have now curated 1008 publications providing 19,156 independent annotations, representing 25% of the curation from small-scale publications. In addition, another 80,000 annotations have been provided via the submission of HTP datasets.</p>
<p>Please contact us <a href="mailto:helpdesk@pombase.org?subject=New%20manuscript%20for%20curation">via the helpdesk</a> if you would like to provide curation for your manuscript but don’t know how.</p>
<p>Links:</p>
<ul>
<li><a href="https://www.pombase.org/reference_list/community">All community curated publications</a></li>
<li><a href="https://www.pombase.org/archive/spotlight">Spotlights</a></li>
<li><a href="https://curation.pombase.org/pombe">Canto curation tool</a></li>
<li><a href="https://www.pombase.org/submit-data">HTP data and browser track submission</a></li>
</ul>
</div>
<div id="2022-03-23-new-human-orthologs" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="new-human-orthologs-including-3-mrp-complex-subunits">New human orthologs including 3 MRP complex subunits</h3>
<p><em>2022-03-23</em></p>
<p>We continue to identify distant human orthologs. Four new 1:1 human ortholog connections have been added to PomBase this week:</p>
<ul>
<li>RNase MRP subunit Rmp1 = human NEPRO (family submitted to Pfam)</li>
<li>RNase P and RNase MRP subunit Pop23 = human RPP38 (members of the same Pfam clan)</li>
<li>RNase P and RNase MRP subunit Pop8 = human RPP14 (family submitted to Pfam)</li>
<li>glutamyl-tRNA amidotransferase complex subunit 3 (Gtf1) = human GATC</li>
</ul>
<p>Human NEPRO is a poorly characterised protein linked to the disease Anauxetic dysplasia 3, and GATC is the causal gene for “combined oxidative phosphorylation deficiency 42”</p>
</div>
<div id="2022-03-12-interprop-v88" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-now-uses-interpro-version-88.0">PomBase now uses InterPro Version 88.0</h3>
<p><em>2022-03-12</em></p>
<p>PomBase now uses InterPro Version 88.0.</p>
<p>Features include:</p>
<ul>
<li>The addition of 39 InterPro entries (40,071 total entries)</li>
<li>Integration of 45 new methods from the PRINTS (1), SMART (1), Pfam (2), SUPERFAMILY (7), CATH-Gene3D (14), PANTHER (13), CDD (7) databases</li>
</ul>
</div>
<div id="2022-03-11-tor-nutritional-phosphoproteome-dataset" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="tor-and-nutritional-phosphoproteome-dataset-loaded">TOR and nutritional phosphoproteome dataset loaded</h3>
<p><em>2022-03-11</em></p>
<p>We have loaded the TOR and nutritional phosphoproteome dataset described in <a href="https://www.pombase.org/reference/PMID:33823663">Halova <em>et al</em>.</a> (9424 annotations). Many thanks to Janni Petersen for preparing the files.</p>
<p><a href="https://www.pombase.org/reference/PMID:33823663">Publication page for PMID:33823663</a></p>
</div>
<div id="2022-03-10-Atkinson-lncRNAs" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="an-additional-5775-novel-curated-lncrnas-atkinson-et-al-.">An additional 5775 novel curated lncRNAs from Atkinson <em>et al</em>.</h3>
<p><em>2022-03-10</em></p>
<p>We have added an additional 5775 novel curated lncRNAs from <a href="https://www.pombase.org/reference/PMID:29914874">Atkinson <em>et al</em>.</a> to PomBase. We will refine the descriptions of these gene products to align with Sequence Ontology (SO) terms describing RNA features in the coming months.</p>
<p>Thanks to María Rodríguez-López for preparing the files.</p>
<p><a href="https://www.pombase.org/reference/PMID:29914874">Publication page for PMID:29914874</a></p>
</div>
<div id="2022-02-02-pombase-and-japonicusdb-papers" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="pombase-japonicusdb-publications-genetics-mod-reports-special-issue">PomBase &amp; JaponicusDB publications in the GENETICS MOD reports special issue</h3>
<p><em>2022-02-02</em></p>
<p>Papers describing PomBase and JaponicusDB are now published (early online). These articles are part of a special issue of GENETICS devoted to model organism database (MOD) reports. The MOD papers will highlight the journal’s new section on Computational Resources, Software &amp; Databases.</p>
<ul>
<li><a href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyab222/6481557">Fission stories: using PomBase to understand <em>Schizosaccharomyces pombe</em> biology</a></li>
<li><a href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyab223/6481558">JaponicusDB: rapid deployment of a model organism database for an emerging model species</a></li>
</ul>
</div>
<div id="2021-12-19-improve-go-filter" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="improved-options-filtering-annotations-by-cell-cycle-phase">Improved options for filtering annotations by cell cycle phase</h3>
<p><em>2021-12-19</em></p>
<p>The Gene Ontology annotation filter for “during” specific cell cycle phases is now included on the “Summary” view in addition to the “Details” view. Available phases have been extended to cover all phases used, and to provide<br />
useful grouping terms. This filter is “ontology aware” (i.e. a search on interphase will also display G1/S/G2 phase annotation). The phase filter is most useful on pages that display increasing volume of phase-specific curation (such as cdc2). The revised phase filter options are also available in the gene expression section.</p>
<p>The phase filters are located at the top right of GO and gene expression annotation sections:</p>
<p><img src="assets/newsfeed/during-filter.png" class="screenshot" style="padding-left:5em" loading="lazy" alt="During filter" /><br />
</p>
</div>
<div id="2021-11-23-interpro-v87" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-now-uses-interpro-version-87.0">PomBase now uses InterPro Version 87.0</h3>
<p><em>2021-11-23</em></p>
<p>PomBase now uses InterPro Version 87.0, which integrates:</p>
<ul>
<li>1,155 new InterPro entries</li>
<li>Update to Pfam 34.0</li>
<li>1,256 new methods</li>
</ul>
</div>
<div id="2021-10-18-new-protein-feature-searches" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/advanced_search.png"/></p>
<h3 id="coils-disorder-and-more-new-protein-feature-queries">Coils, disorder, and more: new protein feature queries</h3>
<p><em>2021-10-18</em></p>
<p>The PomBase <a routerLink="/query">advanced search</a> now allows you to find proteins that have coiled-coil regions, disordered regions, and low-complexity regions. The query-building interface also now organises query options more sensibly, and the documentation has been updated.</p>
</div>
<div id="2021-09-27-Genetics-database-pubs" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombelist.png"/></p>
<h3 id="pombase-japonicusdb-preprints">PomBase &amp; JaponicusDB preprints</h3>
<p><em>2021-09-27</em></p>
<p>Papers describing PomBase and JaponicusDB will appear in an issue of <a href="https://academic.oup.com/genetics/">GENETICS</a> devoted to model organism database (MOD) reports. The MOD papers will highlight the journal’s new section on Computational Resources, Software &amp; Databases.</p>
<p>Follow the links to the <a href="https://www.biorxiv.org/content/10.1101/2021.09.07.459264v1.full">PomBase preprint</a> and <a href="https://www.biorxiv.org/content/10.1101/2021.09.23.461587v1">JaponicusDB preprint</a>, and watch for the full-fledged publications to appear in March 2022.</p>
</div>
<div id="2021-09-01-japonicusdb-release" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/japonicusdb-32x32.png"/></p>
<h3 id="japonicusdb-new-fission-yeast-database">JaponicusDB: a new fission yeast database</h3>
<p><em>2021-09-01</em></p>
<p>We are delighted to announce the official release of PomBase’s new sister: <a href="https://www.japonicusdb.org">JaponicusDB</a> is a new, curated model organism database for the fission yeast <em>Schizosaccharomyces japonicus</em>. JaponicusDB highlights include revised gene structures, distant ortholog detection, improved GO annotation, community literature curation, and reciprocal gene page links with PomBase, providing a familiar environment for all fission yeast researchers.</p>
<p>The <em>S. japonicus</em> community will maintain JaponicusDB from now on. Join the <a href="https://mailman.kcl.ac.uk/mailman/listinfo/japonicus-list">new mailing list</a>, and follow {{"@"}}japonicusdb on Twitter.</p>
</div>
<div id="2021-08-06-increased-disease-annotation-coverage" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/human_disease.png"/></p>
<h3 id="improved-gene-disease-curation-over-1400-s.-pombe-genes">Improved gene-disease curation: over 1400 S. pombe genes</h3>
<p><em>2021-08-06</em></p>
<p>PomBase disease gene curation associates disease descriptors with fission yeast orthologs of human disease-causing genes. We have added new gene–disease term connections, to bring the total to 1401 <em>S. pombe</em> genes. Disease associations now cover 27.3% of all fission yeast protein-coding genes, and almost 40% with human orthologs.</p>
</div>
<div id="2021-07-28-pombetalks-aug4th" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-august-4th">PombeTalks August 4th</h3>
<p><em>2021-07-28</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>4th August</strong> 2021, at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>. Speakers:</p>
<ul>
<li><p><strong>Tiffany Mak</strong>, Nurse lab, The Francis Crick Institute: <em>The TOR-dependant phosphoproteome and regulation of cellular protein synthesis</em></p></li>
<li><p><strong>Weifang Wu</strong>, Allshire lab, University of Edinburgh: <em>Spatial organisation of the nucleus influences centromere identity</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>After these talks, PombeTalks will take a break for the rest of the summer, so watch this space, pombelist, or PombeSlack for updates. In the meantime, you can fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a> at any time if you’re interested in speaking.</p>
</div>
<div id="2021-07-28-alphafold-links" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/Ofd2_structure_32px.png"/></p>
<h3 id="alphafold-links">AlphaFold links</h3>
<p><em>2021-07-28</em></p>
<p>PomBase gene pages now have links to the AlphaFold Protein Structure Database, the collection of structures predicted by AI developed by DeepMind, hosted at EBI. Look in the “External references” section of your favorite gene page, or check out <a href="https://alphafold.ebi.ac.uk/entry/Q01663">this example (pap1)</a>, or read more at the <a href="https://alphafold.ebi.ac.uk/">EBI AlphaFold home</a>.</p>
</div>
<div id="2021-07-14-pombetalks-july-21st" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-july-21st">PombeTalks July 21st</h3>
<p><em>2021-07-14</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>21st July</strong> 2021, at 10:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a> / 17:00 <a href="https://greenwichmeantime.com/time-zone/gmt-plus-9/">Japan &amp; Korea</a>. Speakers:</p>
<ul>
<li><p><strong>Yoko Otsubo</strong>, Yamashita lab, National Institute for Basic Biology: <em>Novel links between TORC1 and traditional non-coding RNA, tRNA</em></p></li>
<li><p><strong>Jie Su</strong>, Nakagawa lab, Osaka University: <em>Rad8-dependent PCNA ubiquitination at lysine 107 causes gross chromosomal rearrangements</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>PombeTalks will continue with its fortnightly schedule, and every third session will take place at the Asia-friendly time. Watch this space, pombelist, or PombeSlack for updates, and please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a> if you’re interested in speaking.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2021-07-06-kegg-pathway-links" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/kegg3.png"/></p>
<h3 id="kegg-pathway-links">KEGG pathway links</h3>
<p><em>2021-07-06</em></p>
<p>PomBase gene pages now have links to pathway entries in the Kyoto Encyclopedia of Genes and Genomes (KEGG) database, as well as links to gene lists for each linked pathway (example: <a href="https://www.pombase.org/term/KEGG_PW:spo01210">2-Oxocarboxylic acid metabolism</a>). The KEGG pathway links are the first entry in a new gene page section, “Molecular pathway”, dedicated to connecting genes in PomBase to depictions of biochemical and signaling pathways.</p>
</div>
<div id="2021-06-30-interpro-v86" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-now-uses-interpro-version-86.0">PomBase now uses InterPro Version 86.0</h3>
<p><em>2021-06-30</em></p>
<p>PomBase now uses InterPro Version 86.0, which integrates:</p>
<ul>
<li>299 new InterPro entries</li>
<li>An update to PROSITE patterns [2021_01] and PROSITE profiles [2021_01]</li>
<li>454 new methods from the PROSITE profiles (39), SMART (2), Pfam (7), SUPERFAMILY (3), CATH-Gene3D (80), PANTHER (295), CDD (27), SFLD (1) databases.</li>
</ul>
<p>InterPro cites 52235 publications in PubMed. See the <a href="https://www.ebi.ac.uk/interpro/release_notes/">InterPro release notes</a> for further information.</p>
</div>
<div id="2021-06-30-pombetalks-july7th" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-july-7th">PombeTalks July 7th</h3>
<p><em>2021-06-30</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>7th July</strong> 2021, at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>. Speakers:</p>
<ul>
<li><p><strong>Debatrayee Sinha</strong>, Qian Chen lab, University of Toledo: <em>Fission yeast polycystin Pkd2p promotes resumption of cell growth after cytokinesis</em></p></li>
<li><p><strong>Joël Lemière</strong>, Fred Chang Lab, UCSF: <em>The role of osmotic forces in nuclear size control</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>PombeTalks will continue with its fortnightly schedule, and every third session will take place at the Asia-friendly time. Watch this space, pombelist, or PombeSlack for updates, and please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a> if you’re interested in speaking.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2021-06-17-pombetalks-june-23rd" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-june-23rd">PombeTalks June 23rd</h3>
<p><em>2021-06-17</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>23rd June</strong> 2021, at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>. Speakers:</p>
<ul>
<li><p><strong>Yi Wei</strong>, Grewal lab, NCI CCR, Bethesda: <em>TOR targets an RNA processing network to regulate cell proliferation and sexual development</em></p></li>
<li><p><strong>Nicholas Ader</strong>, LusKing Lab, Yale School of Medicine: <em>I open at the close(d mitosis): Investigating post-mitotic nuclear envelope sealing in fission yeast</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>PombeTalks will continue with its fortnightly schedule, and every third session will take place at the Asia-friendly time. Watch this space, pombelist, or PombeSlack for updates, and please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a> if you’re interested in speaking.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2021-06-15-gene-expression-violin-plots" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/advanced_search.png"/></p>
<h3 id="violin-plots-gene-expression">Violin plots for gene expression</h3>
<p><em>2021-06-15</em></p>
<p>The “Quantitative gene expression” section of PomBase gene pages now offers a display of <a href="https://en.wikipedia.org/wiki/Violin_plot">violin plots</a> to visualize where the gene appears in available expression datasets.</p>
<p>Violin plots are also available to visualize sets of up to 150 genes in the advanced search results.</p>
<p>At present data from <a href="https://www.pombase.org/reference/PMID:23101633">Marguerat S <em>et al.</em> (2012)</a> and <a href="https://www.pombase.org/reference/PMID:24763107">Carpy A <em>et al.</em> (2014)</a> are included.</p>
</div>
<div id="2021-06-02-pombetalks-june-9" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-june-9th">PombeTalks June 9th</h3>
<p><em>2021-06-02</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>9th June</strong> 2021, at 10:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a> / 17:00 <a href="https://greenwichmeantime.com/time-zone/gmt-plus-9/">Japan &amp; Korea</a>. Speakers:</p>
<ul>
<li><p><strong>Yusuke Toyoda</strong>, Saitoh lab, Kurume University: <em>Nitrogen-dependent persistence of S. pombe Ght5 glucose transporter on the cell surface is effected by TORC2 inhibition of α-arrestin Aly3</em></p></li>
<li><p><strong>Anupa T. Anil</strong>, Mishra lab, IISER Mohali: <em>How does spliceosome capture branchpoint-distant 3’ splice site?</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>PombeTalks will continue with its fortnightly schedule, and every third session will take place at the Asia-friendly time. Watch this space, pombelist, or PombeSlack for updates, and please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a> if you’re interested in speaking.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2021-05-20-pombetalks-may-26" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-may-26th">PombeTalks May 26th</h3>
<p><em>2021-05-20</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>26th May</strong> 2021, at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>. Speakers:</p>
<ul>
<li><p><strong>Mélina Vaurs</strong> (Vincent Géli &amp; Stéphane Coulon labs - Cancer Research Center, Marseille): <em>Shelterin-dependent telomerase regulation differs between quiescent and vegetative cells</em></p></li>
<li><p><strong>Arthur Molines</strong> (Fred Chang lab – UCSF): <em>Physical properties of the cytoplasm modulate microtubule dynamics</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>PombeTalks will continue with its fortnightly schedule, and every third session will take place at the Asia-friendly time. Watch this space, pombelist, or PombeSlack for updates, and please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a> if you’re interested in speaking.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2021-05-05-pombetalks-may-12" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-may-12th">PombeTalks May 12th</h3>
<p><em>2021-05-05</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>12th May</strong> 2021, at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>. Speakers:</p>
<ul>
<li><p><strong>Sierra Cullati</strong>, Gould lab, Vanderbilt University: <em>Autophosphorylation of the CK1 Kinase Domain Regulates Enzyme Activity and Substrate Specificity</em></p></li>
<li><p><strong>Stephen Huisman</strong>, Brunner Lab, University of Zurich: <em>Vip1, a temperature-dependent filament forming protein involved in cell length regulation</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>PombeTalks will continue with its fortnightly schedule, and every third session will take place at the Asia-friendly time. Watch this space, pombelist, or PombeSlack for updates, and please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a> if you’re interested in speaking.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2021-04-29-diploid-genotypes-phenotypes" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombelist.png"/></p>
<h3 id="new-diploid-genotype-phenotype-display">New diploid genotype &amp; phenotype display</h3>
<p><em>2021-04-29</em></p>
<p>PomBase now includes pages for curated diploid genotypes, and displays phenotypes annotated to them on gene and publication pages. For more details see the documentation for <a href="documentation/gene-page-phenotypes">phenotype annotations</a> and <a href="documentation/genotype-page">genotype pages</a>.</p>
</div>
<div id="2021-04-22-pombetalks-april-28" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-april-28th">PombeTalks April 28th</h3>
<p><em>2021-04-22</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>28th April</strong> 2021, at 10:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a> / 17:00 <a href="https://greenwichmeantime.com/time-zone/gmt-plus-9/">Japan &amp; Korea</a>. Speakers:</p>
<ul>
<li><p><strong>Tomoyuki Fukuda</strong>, Niigata University Graduate School of Medical and Dental Sciences: <em>Atg43 serves as a selective autophagy receptor to promote mitophagy</em></p></li>
<li><p><strong>Xiao-Ran Zhang</strong>, NIBS, Beijing, China: <em>An improved auxin-inducible degron system for fission yeast</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>PombeTalks will continue with its fortnightly schedule, and every third session will take place at the Asia-friendly time. Watch this space, pombelist, or PombeSlack for updates, and please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a> if you’re interested in speaking.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2021-04-15-interpro-v85" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-now-uses-interpro-version-85.0">PomBase now uses InterPro Version 85.0</h3>
<p><em>2021-04-15</em></p>
<p>PomBase now uses InterPro Version 85.0, which integrates:</p>
<ul>
<li>157 new InterPro entries</li>
<li>An update to CATH-Gene3D [4.3.0]</li>
<li>333 new methods from the Pfam (3), SUPERFAMILY (11), CATH-Gene3D (168), PANTHER (88), CDD (62), SFLD (1) databases</li>
</ul>
<p>InterPro cites 51539 publications in PubMed. See the <a href="https://www.ebi.ac.uk/interpro/release_notes/">InterPro release notes</a> for further information.</p>
</div>
<div id="2021-04-08-pombetalks-april14" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-april-14th">PombeTalks April 14th</h3>
<p><em>2021-04-08</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>14th April</strong> 2021, at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Pabitra Parua</strong>,�Fisher�lab,�Icahn School of Medicine at Mount Sinai: <em>Control of the RNA polymerase II transcription cycle by CDK-phosphatase switches</em></p></li>
<li><p><strong>Ye Dee Tay</strong>, Sawin Lab,�University of�Edinburgh: <em>Gef1: the first aider of Cdc42 polarity module during stress</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>PombeTalks will continue with its fortnightly schedule, and every third session will take place at the new Asia-friendly time. Watch this space, pombelist, or PombeSlack for updates, and please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a> if you’re interested in speaking.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2021-03-31-GC-content-jbrowse" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/></p>
<h3 id="new-genome-browser-data-gc-content">New genome browser data: GC content</h3>
<p><em>2021-03-31</em></p>
<p>A data track showing the fraction of G/C bases in a region is now available in PomBase JBrowse, listed under “Base composition”. The track is generated using the <a href="https://github.com/elsiklab/gccontent">gccontent</a> plugin, and uses a default window of 100 bp.</p>
</div>
<div id="2021-03-31-canto-qualitative-gene-expression" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/canto.png"/></p>
<h3 id="qualitative-gene-expression-annotation-canto">Qualitative gene expression annotation in Canto</h3>
<p><em>2021-03-31</em></p>
<p>Canto, PomBase’s online curation tool, now supports qualitative gene expression annotation. Two new annotation types are available to represent observations about the levels of RNA or protein observed in wild-type cells, and how they change over the cell cycle or in response to a stimulus. See the <a href="https://curation.pombase.org/pombe/docs/gene_expression">Canto documentation</a> for more information. We have also updated the display of qualitative gene expression on PomBase gene and publication pages.</p>
</div>
<div id="2021-03-30-new-protein-feature-graphic" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombelist.png"/></p>
<h3 id="new-protein-feature-display">New protein feature display</h3>
<p><em>2021-03-30</em></p>
<p>PomBase has released a new, interactive display for protein features on gene pages. The new view is clearer, with details for each feature available via mouseover as well as in the accompanying table.</p>
<p>In addition, PomBase now uses InterPro Version 84.0, which includes 205 new entries and integrates 252 new methods from the Pfam, PANTHER, and CDD databases. See the <a href="https://www.ebi.ac.uk/interpro/release_notes/">InterPro release notes</a> for further information.</p>
</div>
<div id="2021-03-27-pombetalks-march31" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-march-31st">PombeTalks March 31st</h3>
<p><em>2021-03-27</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>31st March</strong> 2021, at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Udo Onwubiko</strong>, Das lab, University of Tennessee: <em>Cdc42 prevents early Rho1 activation during cytokinesis</em></p></li>
<li><p><strong>Chunmin Shan</strong>, Jia lab, Columbia University: <em>The INO80 complex regulates epigenetic inheritance of heterochromatin</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>PombeTalks will continue with its fortnightly schedule, and every third session will take place at the new Asia-friendly time. Watch this space, pombelist, or PombeSlack for updates, and please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a> if you’re interested in speaking.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2021-03-10-pombetalks-march17" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-march-17th-new-time">PombeTalks March 17th - new time!</h3>
<p><em>2021-03-10</em></p>
<p>The new season of online PombeTalks for 2021 will begin on Wednesday <strong>3rd March</strong> 2021, at a different time: 9:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a> / 17:00 <a href="https://greenwichmeantime.com/time-zone/gmt-plus-9/">Japan &amp; Korea</a>. Speakers:</p>
<ul>
<li><p><strong>Yasuto Murayama</strong>, National Institute of Genetics, Shizuoka: <em>Biochemical analysis of the fission yeast structural maintenance of chromosomes complex</em></p></li>
<li><p><strong>Ken Ishikawa</strong>, Kurume University, Kurume: <em>dCas9-mediated CRISPRi for S. pombe</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>PombeTalks will continue with its fortnightly schedule, and every third session will take place at the new Asia-friendly time. Watch this space, pombelist, or PombeSlack for updates, and please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a> if you’re interested in speaking.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2021-02-24-pombetalks-march-3" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-return-march-3rd">PombeTalks return March 3rd</h3>
<p><em>2021-02-24</em></p>
<p>The new season of online PombeTalks for 2021 will begin on Wednesday <strong>3rd March</strong> 2021 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Maria Rodriguez Lopez</strong>, Bähler lab, UCL: <em>Clr6 orchestrates transcriptional switches to regulate metabolism during oxidative stress</em></p></li>
<li><p><strong>Olivia Muriel-Lopez</strong>, Martin lab, University of Lausanne: ’Ultrastructural plasma membrane asymmetries underlie cell-cell fusion in S. pombe*</p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>PombeTalks will resume its fortnightly schedule, so watch this space, pombelist, or PombeSlack for updates. As in the past, you are always welcome to fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a> if you’re interested in speaking.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2021-02-04-id-mapper" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/advanced_search.png"/></p>
<h3 id="pombase-identifier-mapper-now-available">PomBase identifier mapper now available</h3>
<p><em>2021-02-04</em></p>
<p>We have developed an identifier mapper that retrieves <em>S. pombe</em> gene systematic IDs and standard names for a selection of different input ID types. You can now find <em>S. pombe</em> genes using UniProt accessions, and retrieve manually curated orthologs for <em>S. cerevisiae</em> using standard gene names or ORF names, and for human using standard gene names or HGNC identifiers.</p>
<p>Try the <a routerLink="/identifier-mapper">identifier mapper</a> or check out the <a routerLink="/documentation/id-mapper">documentation</a>.</p>
</div>
<div id="2021-01-25-interpro-v83" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-now-uses-interpro-version-83.0">PomBase now uses InterPro Version 83.0</h3>
<p><em>2021-01-25</em></p>
<p>PomBase now uses InterPro Version 83.0, which integrates:</p>
<ul>
<li>376 new InterPro entries</li>
<li>An update to HAMAP [2020_05], CDD [3.18]</li>
<li>462 new methods from the SMART (2), TIGRFAMs (2), Pfam (3), PANTHER (140), HAMAP (19), CDD (286), SFLD (10) databases</li>
</ul>
<p>InterPro cites 50487 publications in PubMed. See the <a href="https://www.ebi.ac.uk/interpro/release_notes/">InterPro release notes</a> for further information.</p>
</div>
<div id="2021-01-07-first-fission-yeast-micropublication-live" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/micropubs32px.png"/></p>
<h3 id="first-s.-pombe-micropublication-goes-live">First S. pombe microPublication goes live</h3>
<p><em>2021-01-07</em></p>
<p>The first fission yeast microPublication has now been published:</p>
<p>Nafees Ahamad, Simmi Anjum, Shakil Ahmed\ <a href="https://www.micropublication.org/journals/biology/micropub-biology-000348/">Pyrogallol induces oxidative stress defects in the fission yeast <em>S. pombe</em>.</a></p>
<p>Congratulations to the authors, and thanks to the microPublication team!</p>
</div>
<div id="2020-11-18-pombetalks-nov-25" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-november-25th">PombeTalks November 25th</h3>
<p><em>2020-11-18</em></p>
<p>The last online PombeTalks for 2020 will take place on Wednesday <strong>25th November</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>I-Ju Lee</strong>, David Pellman’s Lab, Dana-Farber Cancer Institute: <em>Factors promoting nuclear envelope assembly independent of the canonical ESCRT pathway</em></p></li>
<li><p><strong>Ulrike Endesfelder</strong>, Carnegie Mellon University: <em>TBC</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>After these talks, PombeTalks will take a well-earned break, and return in early 2021. The <a href="https://researchseminars.org/seminar/pombeTalks">schedule</a> is available, and you are always welcome to fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a> if you’re interested in speaking.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-11-04-pombetalks-nov-11" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-november-11th">PombeTalks November 11th</h3>
<p><em>2020-11-04</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>11th November</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Farnaz Mansouri</strong>, Mark Bayfield lab (York University, Toronto): <em>The uncharacterized S. pombe La-related protein 1 functions in translation and affects RNA abundance</em></p></li>
<li><p><strong>Saz Basu</strong>, Paul Nurse lab (Francis Crick Institute, London): <em>Unmasking the mitotic potential of G1/S Cyclin-CDK</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>November 25</strong>. PombeTalks will then take a break, and return in early 2021. The <a href="https://researchseminars.org/seminar/pombeTalks">schedule</a> is available, and you are always welcome to fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a> if you’re interested in speaking.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-11-01-PAINT-annotations" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/go-logo-icon.mini.png"/></p>
<h3 id="go-annotations-paint">GO annotations from PAINT</h3>
<p><em>2020-11-01</em></p>
<p>PomBase now includes over 3000 GO annotations made using Phylogenetic Annotation and INference Tool (PAINT), developed by the GO Consortium to infer protein function in a phylogenetic context, supporting precise assertions as to when functions were gained and lost during evolution. PAINT annotations use the evidence code <a href="http://wiki.geneontology.org/index.php/Inferred_from_Biological_aspect_of_Ancestor_(IBA)">“inferred from biological aspect of ancestor” (IBA)</a>. PAINT curation is described in more detail in <a href="https://pubmed.ncbi.nlm.nih.gov/21873635/">Gaudet <em>et al.</em> 2011</a>.</p>
</div>
<div id="2020-10-24-pombetalks-oct-28" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-october-28th">PombeTalks October 28th</h3>
<p><em>2020-10-24</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>28th October</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Omaya Dudin</strong>, EPFL, Switzerland: <em>Cellularization of Ichthyosporean coenocytes</em></p></li>
<li><p><strong>Bassem Al-Sady</strong>, UCSF, USA: <em>Single cell analysis of the heterochromatin spreading reaction</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>November 11</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">schedule</a> is available for the next few weeks. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-10-08-pombetalks-oct-14" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-october-14th">PombeTalks October 14th</h3>
<p><em>2020-10-08</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>14th October</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Dimitrios Vavylonis</strong>, Lehigh University: <em>Modeling fission yeast’s polarization pattern</em></p></li>
<li><p><strong>Chloe Snider</strong>, Gould Lab, Vanderbilt University: <em>Opposite surfaces of the Cdc15 F-BAR domain create a membrane platform that coordinates cytoskeletal and signaling components for cytokinesis</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>October 28</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">schedule</a> is available for the next few weeks. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-09-23-pombetalks-sept-30" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-september-30th">PombeTalks September 30th</h3>
<p><em>2020-09-23</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>16th September</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Alexander Lorenz</strong>, University of Aberdeen, UK: <em>Meiotic recombination outcome in the face of genetic diversity</em></p></li>
<li><p><strong>Veneta Gerganova</strong>, Martin Lab, UNIL, Switzerland: <em>Patterning of membrane-associated proteins through membrane flows</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>October 14</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">schedule</a> is available for the next few weeks. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-09-17-new-jbrowse-datasets" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/></p>
<h3 id="new-genome-browser-datasets-2020-09-17">New genome browser datasets 2020-09-17</h3>
<p><em>2020-09-17</em></p>
<p>Two new datasets in are now available in PomBase JBrowse (links go to PomBase publication pages, which in turn link to the browser with the tracks enabled):</p>
<ul>
<li><a routerLink="/reference/PMID:23395004">Meiotic DSBs</a> from<br />
Fowler KR, Gutiérrez-Velasco S, Martín-Castellanos C, Smith GR. 2013.<br />
Protein determinants of meiotic DNA break hot spots.<br />
PMID:25747261 DOI: 10.1016/j.molcel.2013.01.008</li>
</ul>
<p>and</p>
<ul>
<li><a routerLink="/reference/PMID:32152323">Time-lapse single-cell transcripts for dormancy breaking</a> from<br />
Tsuyuzaki H, Hosokawa M, Arikawa K, Yoda T, Okada N, Takeyama H, Sato. 2020.<br />
Time-lapse single-cell transcriptomics reveals modulation of histone H3 for dormancy breaking in fission yeast.<br />
PMID:32152323 DOI: 10.1016/j.molcel.2013.01.008</li>
</ul>
<p>More datasets are always welcome, so check out our <a routerLink="/documentation/data-submission-form-for-HTP-sequence-linked-data">instructions for submission</a>.</p>
</div>
<div id="2020-09-14-pombetalks-sept-16" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-september-16th">PombeTalks September 16th</h3>
<p><em>2020-09-14</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>16th September</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Susan Forsburg</strong>, University of Southern California: <em>Visualizing replication stress</em></p></li>
<li><p><strong>Sigurd Braun</strong>, Ludwig-Maximilians-Universität, München: <em>Gene repression at the nuclear membrane: Multifaceted roles of Lem2</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>September 30</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">schedule</a> is available for the next few weeks. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-09-11-hermes-transposon-data-long" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/transposon.png"/></p>
<h3 id="new-dataset-hermes-transposon-insertions">New dataset: Hermes transposon insertions</h3>
<p><em>2020-09-11</em></p>
<p>PomBase now hosts transposon integration data from <a routerLink="/reference/PMID:32101745">Lee et al. 2020</a>. Henry Levin explains the background and significance of the work:</p>
<p>“Transposon Integration Sequencing is a genome wide method of mapping sequences that contribute to growth. High throughput sequencing of transposon integration sites in haploid cells with single insertions reveals which genes are dispensable. Once propagated, cultures exhibit a pronounced lack of insertions in genes necessary for growth. This method, originally developed to study bacteria is now used to characterize the genomes of several yeasts including <em>S. pombe</em>. In earlier work we used the transposon Hermes to identify genes of <em>S. pombe</em> required for growth (<a routerLink="/reference/PMID:23893486">Guo et al., 2013, Genetics, PMID:23893486</a>). We have now applied Hermes and Transposon Integration Sequencing to identify genes important for the formation of heterochromatin (<a routerLink="/reference/PMID:32101745">Lee et al., 2020, Cell Reports, PMID:32101745</a>). Insertion sites from eight independent cultures can be visualized from PomBase as custom tracks on
Jbrowse. Four cultures were of cells with ura4 silenced by cen1 heterochromatin. The other four cultures were of a strain without ura4. By passaging the cultures in 5-FOA we selected against cells with defects in heterochromatin. Genes that contributed to the formation of heterochromatin exhibited fewer insertions in cells with the cen1 copy of ura4 relative to the strain lacking ura4. To distinguish genes critical for heterochromatin from genes that contribute to a lesser extent we passaged cultures in 5-FOA for 5 generations and for 80 generations. While viewing these integration sites can indicate whether genes of interest contribute to heterochromatin formation you can also examine insertions in the cultures lacking ura4 to gage whether specific genes or noncoding sequences make significant contributions to growth.”</p>
</div>
<div id="2020-09-08-new-jbrowse-datasets" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/></p>
<h3 id="new-genome-browser-datasets-2020-09-08">New genome browser datasets 2020-09-08</h3>
<p><em>2020-09-08</em></p>
<p>Three new datasets in are now available in PomBase JBrowse (links go to PomBase publication pages, which in turn link to the browser with the tracks enabled):</p>
<ul>
<li><p><a href="https://www.pombase.org/reference/PMID:25747261">Transcription start sites</a> from<br />
Li H, Hou J, Bai L, Hu C, Tong P, Kang Y, Zhao X, Shao Z. 2015.<br />
Genome-wide analysis of core promoter structures in Schizosaccharomyces pombe with DeepCAGE.<br />
PMID:25747261 DOI:10.1080/15476286.2015.1022704</p></li>
<li><p><a href="https://www.pombase.org/reference/PMID:26883383">Transcript data</a> from<br />
Eser P, Wachutka L, Maier KC, Demel C, Boroni M, Iyer S, Cramer P, Gagneur J. 2016<br />
Determinants of RNA metabolism in the Schizosaccharomyces pombe genome.<br />
PMID:26883383 DOI:10.15252/msb.20156526</p></li>
<li><p><a href="https://www.pombase.org/reference/PMID:32101745">Transposon insertion sites</a> from<br />
Lee SY, Hung S, Esnault C, Pathak R, Johnson KR, Bankole O, Yamashita A, Zhang H Levin HL.<br />
Dense Transposon Integration Reveals Essential Cleavage and Polyadenylation Factors Promote Heterochromatin Formation.<br />
PMID:32101745 DOI:10.1016/j.celrep.2020.01.094</p></li>
</ul>
</div>
<div id="2020-09-08-jbrowse-data-submission" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/></p>
<h3 id="submit-your-genome-browser-data">Submit your genome browser data</h3>
<p><em>2020-09-08</em></p>
<p>We have updated our HTP data submission procedure to make it easier for you to contribute your datasets for PomBase JBrowse:</p>
<p>We now provide spreadsheet templates in Excel and Open Document formats that gather the metadata we need to load and display your data. You can download a template from the <a href="https://www.pombase.org/documentation/data-submission-form-for-HTP-sequence-linked-data">documentation page on HTP data submission</a>. Send completed spreadsheets to the <a href="mailto:helpdesk@pombase.org">PomBase helpdesk</a>.</p>
</div>
<div id="2020-09-02-GO-term-matrix-publication" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/go-term-matrix.png"/></p>
<h3 id="published-go-term-matrix-annotation-qc">Published: GO Term Matrix for annotation QC</h3>
<p><em>2020-09-02</em></p>
<p>In collaboration with the GO Consortium, the PomBase team has published a report on the <strong>Term Matrix</strong> approach to GO annotation quality control. The article, out this week in <a href="https://royalsocietypublishing.org/doi/10.1098/rsob.200149">Open Biology</a>, describes biological processes that do, or don’t, share annotated gene products, and how we use co-annotation patterns to build rules to detect, correct, and prevent errors.</p>
</div>
<div id="2020-09-01-pombetalks-sept-2" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-september-2nd">PombeTalks September 2nd</h3>
<p><em>2020-09-01</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>2nd September</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>François Bachand</strong>, USherbrooke, Canada: <em>Proximity-dependent biotinylation assays in fission yeast and a tale about slow RNA polymerase II transcription</em></p></li>
<li><p><strong>Scott Curran</strong>, Nurse Lab, The Crick Institute, UK: <em>A quantitative and spatial analysis of the cell cycle control network</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>September 16</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">schedule</a> is available for the next few weeks. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-08-19-11th-pombe-conference-postponed" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombe2022-32px.png"/></p>
<h3 id="id-11th-pombe-meeting-postponed-2022">11th Pombe meeting postponed to 2022</h3>
<p><em>2020-08-19</em></p>
<p>Due to the ongoing Covid-19 pandemic, the <a href="http://www.fission-yeast2022.org/">11th International Fission Yeast Meeting</a>, due to take place in Hiroshima, Japan, has been postponed.</p>
<p>The new dates will be <strong>12th (Sun -17th (Fri) June, 2022</strong>.</p>
<p>Please see the <a href="http://www.fission-yeast2022.org/">conference website</a> and pombelist for further announcements.</p>
</div>
<div id="2020-08-12-pombetalks-aug-19" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-august-19th">PombeTalks August 19th</h3>
<p><em>2020-08-12</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>19th August</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Joe Magliozzi</strong>, Moseley Lab, Dartmouth: <em>Cell polarity kinases regulate RNA-binding protein Sts5 to control cell shape</em></p></li>
<li><p><strong>Ramakanth Neeli</strong>, Minc Lab, Institute Jacques Monod: <em>Mechanisms and Functions of Cell Wall Mechanosensing in Fission Yeast</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>September 2</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">schedule</a> is available for the next few weeks. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-07-31-pombetalks-aug-5" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-august-5th">PombeTalks August 5th</h3>
<p><em>2020-07-31</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>5th August</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Feng Li</strong>, Levin Lab NICHD/NIH, USA: <em>Identification of an integrase-independent pathway of retrotransposition</em></p></li>
<li><p><strong>Ivan Surovtsev</strong>, King lab, Yale University, USA: <em>Liquid-liquid phase separation, heterochromatin domains and nuclear mechanics</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>August 19</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">summer schedule</a> is available. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-07-22-fission-yeast-micropublications" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/micropubs32px.png"/></p>
<h3 id="fission-yeast-micropublications">Fission yeast microPublications</h3>
<p><em>2020-07-22</em></p>
<p>PomBase has recently joined <a href="https://www.micropublication.org/">microPublication.org</a>, which “publishes brief, novel findings, negative and/or reproduced results, and results which may lack a broader scientific narrative”, as a Partner Database. Fission yeast researchers can thus now make any results available to the community, even those that don’t fit neatly into traditional publications.</p>
<p>Visit the <a href="https://www.micropublication.org/">microPublications website</a> to learn more, to <a href="https://www.micropublication.org/submit-new-article/">register</a> and submit your data, or sign up to review. Send questions to the <a href="mailto:helpdesk@pombase.org">PomBase helpdesk</a>.</p>
</div>
<div id="2020-07-21-pombetalks-july-22" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-july-22nd">PombeTalks July 22nd</h3>
<p><em>2020-07-21</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>22nd July</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Prof. Dr. Ann Ehrenhofer-Murray</strong>, Institut für Biologie, Humboldt-Universität zu Berlin: <em>Queuosine and m5c modification of RNA: Nutritional control of translation in S. pombe homestasis</em></p></li>
<li><p><strong>Dr. Sarah Sabatinos</strong>, Department of Chemistry and Biology, Ryerson University: <em>Long-term effects of surviving replication instability</em></p></li>
<li><p><strong>PomBase</strong> microPublications announcement (Midori Harris)</p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>August 5</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">summer schedule</a> is available. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-07-02-pombetalks-july-8" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-july-8th">PombeTalks July 8th</h3>
<p><em>2020-07-02</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>8th July</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Sahana Holla</strong>, Grewal lab, NIH: <em>Positioning heterochromatin at the nuclear periphery promotes epigenetic inheritance</em></p></li>
<li><p><strong>Nick Rhind</strong>, UMass Medical School: <em>Cell size is controlled by size-dependent expression of mitotic activators</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>July 22nd</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">summer schedule</a> is available. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-06-19-pombetalks-june-24" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-june-24th">PombeTalks June 24th</h3>
<p><em>2020-06-19</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>24th June</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Sito Torres-Garcia</strong>, Allshire lab, University of Edinburgh: <em>Epigenetic gene silencing by heterochromatin primes fungal resistance</em></p></li>
<li><p><strong>Julie Rich-Robinson</strong>, Das lab, University of Tennessee: <em>Cell-cycle-dependent cues temporally regulate Cdc42 activity at growth sites in fission yeast</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up.</p>
<p>A <a href="https://researchseminars.org/seminar/pombeTalks">schedule</a> is now available for the rest of the summer, including the next talks on <strong>July 8th</strong>. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-06-11-name-queries" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/advanced_search.png"/></p>
<h3 id="editable-pombase-query-names">Editable PomBase query names</h3>
<p><em>2020-06-11</em></p>
<p>Entries in the <a routerLink="/query">PomBase advanced search</a> query history now show brief, user-editable query descriptions, and a toggle to show or hide additional details.</p>
</div>
<div id="2020-06-05-pombetalks-june-10" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-postponed-june-17th">PombeTalks POSTPONED to June 17th</h3>
<p><em>2020-06-05</em></p>
<p>Please note that the next online PombeTalks will take place one week later than originally planned, to support the <a href="https://www.shutdownstem.com/">STEM Strike for Black Lives</a> on 10th June.</p>
<p>In the meantime, please complete this brief <a href="https://forms.gle/ANVsCcfaqf2p4Q9fA">survey</a> of the audience.</p>
<p>On Wednesday <strong>17th June</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>, the speakers will be:</p>
<ul>
<li><p><strong>Gautam Dey</strong>, Baum lab, UCL / EMBL Heidelberg: <em>Closed mitosis requires local disassembly of the nuclear envelope</em></p></li>
<li><p><strong>Meredith Betterton</strong>, UC Boulder: <em>Computational modeling of fission yeast mitosis: what we can learn about pombe from computer simulations</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. The next two sessions will b on <strong>June 27</strong> and <strong>July 8</strong>. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-05-21-pombetalks-may-27" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-may-27th">PombeTalks May 27th</h3>
<p><em>2020-05-21</em></p>
<p>The next online PombeTalks will take place on Wednesday 27 May 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>. This time, in addition to the usual pair of research talks, our own Val Wood will show a few of PomBase’s lesser-known features.</p>
<ul>
<li><p><strong>Angad Garg</strong>, Stewart Schuman’s lab, Memorial Sloan Kettering Cancer Center: <em>Long non-coding RNA control of phosphate homeostasis</em></p></li>
<li><p><strong>José López Hernández</strong>, Sarah Zander’s lab, Stowers Institute for Medical Research: <em>Diverse mating strategies in S. pombe affect the spread of wtf meiotic drivers</em></p></li>
<li><p><strong>Val Wood</strong>, PomBase: <em>Hidden corners of PomBase: Ten features you might not have seen</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also mark your calendars for the next two sessions on and <strong>June 10</strong> and <strong>24</strong>, and if you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-05-20-new-fission-yeast-go-slims" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/dropdown_menu.png"/></p>
<h3 id="new-fission-yeast-go-slims">New fission yeast GO slims</h3>
<p><em>2020-05-20</em></p>
<p>To complement the overview provided by the <a href="/browse-curation/fission-yeast-bp-go-slim-terms">fission yeast GO biological process slim</a>, we have created GO slims for the <a href="/browse-curation/fission-yeast-mf-go-slim-terms">molecular function</a> and <a routerLink="/browse-curation/fission-yeast-cc-go-slim-terms">cellular component</a> branches of GO. Each slim page provides links to ontology term pages, annotated genes, and to download files containing the slim terms and IDs.</p>
</div>
<div id="2020-05-20-advanced-search-slim-options" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/dropdown_menu.png"/></p>
<h3 id="new-ontology-slimming-options-advanced-search-results">New ontology slimming options for advanced search results</h3>
<p><em>2020-05-20</em></p>
<p>The PomBase <a routerLink="/query">advanced search</a> results panel now allows you to retrieve annotations to any of the fission yeast <a href="documentation/pombase-go-slim-documentation">GO slims</a> or the <a href="https://www.pombase.org/browse-curation/disease-slim">Mondo disease slim</a> for genes in the results list. For example, you can query for all genes involved in a process and slim the resulting list by molecular function or disease association.</p>
</div>
<div id="2020-05-18-pombase-uses-mondo" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/human_disease.png"/></p>
<h3 id="pombase-adopts-mondo-disease-gene-curation">PomBase adopts MONDO for disease gene curation</h3>
<p><em>2020-05-18</em></p>
<p>PomBase has switched from the <a href="https://disease-ontology.org/">Disease Ontology</a> (DO) to the Monarch Initiative’s <a href="https://mondo.monarchinitiative.org/">Mondo Disease Ontology</a> (Mondo) for disease gene curation. Mondo covers the same set of disease descriptions as DO, but has a richer hierarchical structure that classifies more specific descriptions into broad categories (e.g. anemia, cancer, kidney disease) suitable for a disease “slim” term set.</p>
<p>PomBase curators are collaborating with Mondo to improve its disease classification, especially in areas that will support inferences that improve fission yeast disease annotation coverage in the new <a href="https://www.pombase.org/browse-curation/disease-slim">PomBase Mondo slim</a>. The new disease slim is a work in progress, so if there is a particular disease grouping that you would find useful, please <a href="mailto:helpdesk%7B%7B%22@%22%7D%7Dpombase.org">let us know</a>.</p>
</div>
<div id="2020-05-18-increased-disease-annotation-coverage" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/human_disease.png"/></p>
<h3 id="improved-gene-disease-curation">Improved gene-disease curation</h3>
<p><em>2020-05-18</em></p>
<p>PomBase disease gene curation associates disease descriptors with fission yeast orthologs of human disease-causing genes. We have now increased coverage by adding new gene–disease term connections, with 3954 individual annotations to 1195 genes (up from 2588 and 905 respectively in January 2019). Disease associations now cover 24.5% of all fission yeast protein-coding genes, and over one third of those with human orthologs.</p>
</div>
<div id="2020-05-11-community-curation-publication" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/canto.png"/></p>
<h3 id="published-community-curation-pombase">Published: Community curation in PomBase</h3>
<p><em>2020-05-11</em></p>
<p>The PomBase team has published an overview of our experience with community curation for fission yeast. In the article, out this week in <a href="https://academic.oup.com/database/article/doi/10.1093/database/baaa028/5827230">Database</a>, we reflect on the factors that have made our community’s remarkable, standard-setting achievements possible, and on the benefits we and PomBase users derive from this effort. We highlight the collaboration between authors and professional curators that arises via community curation, and how annotation quality improves as a result.</p>
<p>Watch for invitations to curate your new papers, or see our <a href="https://www.pombase.org/community/fission-yeast-community-curation-project">community curation page</a> for more information.</p>
</div>
<div id="2020-05-07-pombetalks-may-13" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-may-13th">PombeTalks May 13th</h3>
<p><em>2020-05-07</em></p>
<p>The next online PombeTalks will take place on Wednesday 13 May 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>. Speakers:</p>
<ul>
<li><p><strong>Sarah Lambert</strong>, Institut Curie, Paris, France: <em>Resolution of replication stress in space and time for maintaining genome stability</em></p></li>
<li><p><strong>Cornelia Kilchert</strong>, Justus-Liebig-University, Giessen, Germany: <em>RNA-binding proteins in fission yeast - a global perspective</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also mark your calendars for the next two sessions on <strong>May 27</strong> and <strong>June 10</strong>, and if you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-05-07-chromatin-silencing" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/nucleosome.png"/></p>
<h3 id="chromatin-silencing-ontology-amp-annotation-overhaul">Chromatin silencing ontology &amp; annotation overhaul</h3>
<p><em>2020-05-07</em></p>
<p>PomBase curators have collaborated with the GO Consortium to improve the representation of chromatin silencing and the underlying heterochromatin organization processes in the GO biological process ontology and annotations.</p>
<p>Notably, “chromatin silencing” terms have been removed from GO on the grounds that they conflated various heterochromatin assembly, formation, and maintenance pathways with processes that affect chromatin-mediated repression more indirectly (e.g. tethering to the nuclear envelope). Chromatin silencing is a phenotype resulting from the cumulative effects of these processes, and the Fission Yeast Phenotype Ontology (FYPO) accordingly retains a full suite of “chromatin silencing” terms.</p>
<p>Annotations using the GO chromatin silencing terms were reviewed, and either removed or reannotated based on what could be inferred from the phenotypes, resulting in a substantially revised set of <a href="https://www.pombase.org/term/GO:0031507">heterochromatin assembly annotations</a>. Further work is required, so please <a href="mailto:helpdesk@pombase.org">send us</a> any corrections.</p>
</div>
<div id="2020-05-05-interpro-v79" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-now-uses-interpro-version-79.0">PomBase now uses InterPro Version 79.0</h3>
<p><em>2020-05-05</em></p>
<p>PomBase now uses InterPro Version 79.0, which integrates:</p>
<ul>
<li>128 new InterPro entries</li>
<li>An update to PIRSF [3.10]</li>
<li>151 new methods from the SUPERFAMILY (4), CATH-Gene3D (6), PIRSF (9), PANTHER (106), and CDD (26) databases.</li>
</ul>
<p>InterPro cites 48466 publications in PubMed. See the <a href="https://www.ebi.ac.uk/interpro/release_notes/">InterPro release notes</a> for further information.</p>
</div>
<div id="2020-05-01-mitochondrial-genome-update" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/mitogenome.png"/></p>
<h3 id="mitochondrial-genome-update">Mitochondrial genome update</h3>
<p><em>2020-05-01</em></p>
<p>The mitochondrial genome sequence in PomBase has been updated to reflect corrections made in <a href="https://www.ncbi.nlm.nih.gov/pubmed/31364709">Tao et al. (2019) “Intraspecific Diversity of Fission Yeast Mitochondrial Genomes”</a>.</p>
</div>
<div id="2020-04-28-midori-biocuration-career-award" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/isb_logo_32px.png"/></p>
<h3 id="midori-harris-receives-2020-biocuration-career-award">Midori Harris receives 2020 Biocuration Career Award</h3>
<p><em>2020-04-28</em></p>
<p>Midori Harris, ontology developer and curator at PomBase, has been awarded the <a href="https://www.biocuration.org/midori-harris-recipient-of-the-2020-biocuration-award/">2020 Biocuration Career Award</a>.</p>
<p>Congratulations to Midori and a huge thanks for all that you do for PomBase.</p>
</div>
<div id="2020-04-22-pombetalks" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-start-on-april-29th">PombeTalks start on April 29th</h3>
<p><em>2020-04-22</em></p>
<p>The first in the new series of online PombeTalks will take place on Wednesday 29 April 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>. Speakers:</p>
<ul>
<li><p><strong>Aleksandar Vjeṧtica</strong>, Sophie Martin’s lab, University of Lausanne: <em>Cycling for reproductive fidelity: Coupling the cell cycle and re-fertilisation blocks ensures ploidy maintenance during sexual lifecycle</em></p></li>
<li><p><strong>Haitong Hou</strong>, Julia Cooper’s lab, NCI &amp; University of Colorado: <em>Centromeres are dismantled by foundational meiotic proteins Spo11 and Rec8</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also mark your calendars for the next two sessions on <strong>May 13</strong> and <strong>May 27</strong>, and if you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-03-23-angeli" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/angeli.png"/></p>
<h3 id="angeli-update">AnGeLi update</h3>
<p><em>2020-03-23</em></p>
<p>AnGeLi (developed by Danny Bitton) is a tool that allows you to perform enrichments over gene lists.</p>
<p>AnGeLi has recently been updated to provide 9320 lists, including ontology-based annotations from PomBase (as of 2020-03-04) as well as many additional datasets from the Bähler laboratory.</p>
</div>
<div id="2020-03-23-new-search-result-download-options" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/advanced_search.png"/></p>
<h3 id="new-search-result-download-options">New search result download options</h3>
<p><em>2020-03-23</em></p>
<p>The PomBase <a routerLink="/query">Advanced search</a> has added new options to the data you can download for your query results:</p>
<ul>
<li>All physical interactors of a gene product</li>
<li>Deletion viability</li>
<li>Protein length</li>
</ul>
</div>
<div id="2020-03-01-interpro-v77" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-now-uses-interpro-version-77.0">PomBase now uses InterPro Version 77.0</h3>
<p><em>2020-03-01</em></p>
<p>PomBase now uses InterPro Version 77.0, which integrates 145 new methods from the CATH-Gene3D (134), and SUPERFAMILY (11) databases. InterPro cites 59894 publications in PubMed. See the InterPro <a href="https://www.ebi.ac.uk/interpro/release_notes/">release notes</a> for further information.</p>
</div>
<div id="2020-02-28-50_percent" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/canto.png"/></p>
<h3 id="community-curation-response-rate-reaches-50">Community curation response rate reaches 50%</h3>
<p><em>2020-02-28</em></p>
<p>789/1587 publications assigned to community members for curation are finished. A big thank you to everyone who has participated so far. For more details, and all our curation metrics, see https://curation.pombase.org/pombe/stats/annotation</p>
</div>
<div id="2020-02-20-quarter_million" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/quarter_mill.png"/></p>
<h3 id="a-quarter-million-annotations">A quarter of a million annotations</h3>
<p><em>2020-02-20</em></p>
<p>PomBase recently reached 250,000 annotations to controlled vocabularies and ontologies. The majority (over 90%) are assigned manually from fission yeast experimental data derived from 3776 publications, most of which report low-throughput, hypothesis-driven experiments.</p>
<p>You can query and combine any of these data types in the <a href="https://www.pombase.org/query">Advanced search</a>.</p>
<p>Thank you to everyone who contributed to this significant achievement through community curation.</p>
</div>
<div id="2020-02-05-advanced-search-phenotype-conditions" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/advanced_search.png"/></p>
<h3 id="query-phenotypes-conditions">Query phenotypes for conditions</h3>
<p><em>2020-02-05</em></p>
<p>The PomBase advanced search <a routerLink="/query">Advanced search</a> now supports using experimental conditions as search criteria for phenotype annotations. For example, you can now query for genes that show abnormal chromosome segregation mutant phenotypes specifically at high or low temperatures. The search uses the same condition descriptors as Canto and the PomBase web pages.</p>
<p>Note that phenotype queries that have condition constraints can be combined, but pay careful attention to the annotations for the results. Future work will add support for querying for multiple conditions on the same annotation, and for specifying conditions to exclude from results.</p>
</div>
<div id="2020-02-04-lmo14-strasbourg" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/poster_LMO14_32px.png"/></p>
<h3 id="levures-mod-les-et-outils-14th-international-conference">Levures, Modèles et Outils 14th International Conference</h3>
<p><em>2020-02-04</em></p>
<p>The 14th edition of the <a href="https://lmo-14-2020.sciencesconf.org/">“Levures, Modèles et Outils” meeting (LMO14)</a> will be held in July 9-11, 2020, at the University of Strasbourg in France. Registration is open February 3rd to June 30th, and abstracts can be submitted from February 3rd to April 10th. Authors will be notified in early May and the final program will be available in early June.</p>
<p>The sessions will be diverse and present the latest findings using yeast as a model organism on the following topics:</p>
<ol type="1">
<li>Cell biology, cell cycle, cytoskeleton</li>
<li>Gene expression regulation</li>
<li>Population, functional and evolutionary genomics</li>
<li>Replication, repair and recombination</li>
<li>Transport, sensing and signaling</li>
<li>Pathogenic yeast and filamentous fungi</li>
<li>Tools, resources and databases</li>
<li>New technologies, yeast and industry</li>
</ol>
</div>
<div id="2020-01-17-pombase-elixir-node-service" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/elixir_uk_thumb.png"/></p>
<h3 id="pombase-now-elixir-node-service">PomBase is now an ELIXIR Node Service</h3>
<p><em>2020-01-17</em></p>
<p>PomBase has been awarded Node Service status by the <a href="https://elixiruknode.org/">UK node</a> of <a href="https://elixir-europe.org/">ELIXIR</a>. ELIXIR-UK Node Services support the bioinformatics and broader biological research communities by providing training and resources that help researchers to find and share data, exchange expertise, and agree on best practices at national, European and international levels. The review panel describes PomBase as a “mature, leading model organism database which is popular, unique, well used, and has a strong user community.”</p>
</div>
<div id="2020-01-07-new-stable-integration-vectors" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/stable_pAde6_vector.png"/></p>
<h3 id="new-vectors-simple-reliable-s.-pombe-molecular-biology">New vectors for simple, reliable <em>S. pombe</em> molecular biology</h3>
<p><em>2020-01-07</em></p>
<p>To enable fission yeast researchers to manipulate <em>S. pombe</em> molecular biology reproducibly and easily, Aleks Vještica and Magdalena Marek in Sophie Martin’s lab have designed and constructed a series of simple, fully characterized plasmids.</p>
<p>The Stable Integration Vector (SIV) series provides a highly modular toolbox to introduce heterologous sequences more stably was possible with than previously available vectors. The toolkit includes antibiotic resistance markers, promoters, fluorescent tags, and terminators, as well as large set of ready-to-use fluorescent probes to mark organelles and visualize cellular processes.</p>
<p>The work is published in the <a href="https://jcs.biologists.org/content/early/2019/11/29/jcs.240754.long">Journal of Cell Science</a>, and a PomBase <a href="https://www.pombase.org/reference/PMID:31801797">publication page</a> is available.</p>
</div>
<div id="2019-11-28-shareable-result-links" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/advanced_search.png"/></p>
<h3 id="unique-permanent-urls-search-results">Unique permanent URLs for search results</h3>
<p><em>2019-11-28</em></p>
<p>All result pages from the <a routerLink="/query">Advanced search</a> now have a unique permanent URL that can be bookmarked and shared with your colleagues.</p>
<p>The <a routerLink="/documentation/quick-little-tool">QuiLT</a> and <a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">GO slim</a> pages also now have permanent URLs.</p>
</div>
<div id="2019-10-16-JBrowse-tracks-loadable-from-publication-pages" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/new_tracks.png"/></p>
<h3 id="browser-tracks-now-loadable-publication-pages">Browser tracks now loadable from publication pages</h3>
<p><em>2019-10-16</em></p>
<p>Data tracks from datasets hosted in the PomBase genome browser can now be browsed and loaded from their respective publication pages. For an example, see <a href="https://www.pombase.org/reference/PMID:29914874">Atkinson et al. (2018)</a>. Data tracks are now also downloadable from the publication pages.</p>
</div>
<div id="2019-10-07-new-wt-pombase-funding" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="new-pombase-funding-wellcome-trust">New PomBase funding from the Wellcome Trust</h3>
<p><em>2019-10-07</em></p>
<p>We are pleased to announce that the recent PomBase application for continued Wellcome Trust funding was successful. Although the grant was not fully funded, we are confident that we can cover the shortfall by small grants for stand-alone projects and collaborations. We would like to thank the pombe community for their support with the application, and the Wellcome Trust for their continued funding. We look forward to supporting your research until 2025 (and beyond).</p>
</div>
<div id="2019-10-06-interpro-v76-loaded" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-now-uses-interpro-version-76.0">PomBase now uses InterPro Version 76.0</h3>
<p><em>2019-10-06</em></p>
<p>PomBase now uses InterPro Version 76.0, which integrates 277 new methods from the CATH-Gene3D (1), PANTHER (178) and CDD (98) databases. InterPro cites 59846 publications in PubMed. See the InterPro <a href="https://www.ebi.ac.uk/interpro/release_notes/">release notes</a> for further information.</p>
</div>
<div id="2019-08-30-replication-origin-tracks-loaded" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/></p>
<h3 id="replication-origin-data-loaded-into-jbrowse">Replication origin data loaded into JBrowse</h3>
<p><em>2019-08-30</em></p>
<p>We have loaded data from: <a href="https://www.pombase.org/reference/PMID:14566325">Segurado et al. (2003) “A+T-rich islands”</a>, <a href="https://www.pombase.org/reference/PMID:17304213">Hayashi et al. (2007) “Pre-replicative complex localization; early and late firing origins”</a>, and <a href="https://www.pombase.org/reference/PMID:18093330">Mickle et al. (2007) “Replication origins with functional classification”</a>.</p>
<p>To view the tracks, either follow the hyperlinks above to the respective PomBase publication pages, and click on the “view” link after “Datasets from this publication are available in the PomBase JBrowse genome browser”, or go directly to <a href="http://www.pombase.org/jbrowse/">the browser</a> and click on the “select tracks” button to find the tracks manually.</p>
<p>For anyone new to JBrowse we have a <a href="https://www.pombase.org/documentation/JBrowse_quick_start">quick start guide</a>.</p>
</div>
<div id="2019-07-19-join-the-conversations-on-slack" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/slack_icon-32px.png"/></p>
<h3 id="join-conversations-on-slack">Join the conversations on Slack</h3>
<p><em>2019-07-19</em></p>
<p>The vibrant fission yeast community now has a Slack channel. Slack provides a forum for the research community. Follow conversations you care about, message colleagues privately, or in groups, ask questions, post responses. All archived and searchable.</p>
<ul>
<li>See the <a href="/assets/external_media/PombeSlack-flyer.pdf">PombeSlack flyer</a> for more information, including how to join.</li>
</ul>
<!-- included with original news item; commented out after expiry:
 - <a href="https://bit.ly/2XYkcEi">Join PombeSlack</a> -->
</div>
<div id="2019-07-11-Fitness_Landscape_of_the_Fission_Yeast_Genome-data-loaded" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/></p>
<h3 id="fitness-landscape-fission-yeast-genome-dataset-loaded-into-jbrowse">“Fitness Landscape of the Fission Yeast Genome” dataset loaded into JBrowse</h3>
<p><em>2019-07-11</em></p>
<p>We have loaded the <a href="https://www.pombase.org/reference/PMID:31077324">Grech et al. (2019) “Fitness Landscape of the Fission Yeast Genome”</a> dataset into JBrowse. In this study, transposon mutagenesis libraries were created to map transposon insertion sites in the S. pombe genome. From this data, functional elements of the genome were inferred. The tracks from this study can be loaded by a single click from the linked publication page above</p>
<p>Thanks Dan Jeffares for sending us the data.</p>
<p>For anyone new to JBrowse we have a <a href="https://www.pombase.org/documentation/JBrowse_quick_start">quick start guide</a>.</p>
</div>
<div id="2019-05-19-tRNA_metabolism_update" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/rna_metabolism_32px.png"/></p>
<h3 id="trna-metabolism-go-annotation-update">tRNA metabolism GO annotation update</h3>
<p><em>2019-05-19</em></p>
<p>The process of tRNA metabolism, and the associated molecular functions have recently been reviewed.</p>
<ul>
<li><a href="https://www.pombase.org/term/GO:0006399">“tRNA metabolic process” annotation</a></li>
<li><a href="https://www.pombase.org/term_genes/GO:0006399">genes for “tRNA metabolic process”</a></li>
</ul>
<p>Please let us know if the annotation can be further improved.</p>
</div>
<div id="2019-04-18-customisable-fasta" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombelist.png"/></p>
<h3 id="customisable-fasta-download">Customisable FASTA download</h3>
<p><em>2019-04-18</em></p>
<p>You can now download nucleotide or peptide sequences for genes in Advanced search results in FASTA format, and customise what is included in the FASTA headers (e.g. gene names, product descriptions, sequence coordinates, or various IDs can be included).</p>
</div>
<div id="2019-04-18-prediction-links" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/tree_for_news.png"/></p>
<h3 id="new-homology-modelling-and-ortholog-links-on-gene-pages">New homology modelling and ortholog links on gene pages</h3>
<p><em>2019-04-18</em></p>
<p>We have added new external links to PomBase gene pages for structure and ortholog predictions:</p>
<ul>
<li><p>Protein-specific links to <a href="https://swissmodel.expasy.org/">SWISS-MODEL</a>, a fully automated protein structure homology-modelling server, accessible via the ExPASy web server, lead to a SWISS-MODEL Repository page for each sequence and present results. If no structure or model is available, you can either trigger adding an entry to the repository with a single click or easily interactively search for templates and build models in your own SWISS-MODEL workspace.</p></li>
<li><p>Ensembl Fungi Compara and Ensembl Pan-taxonomic Compara links lead to orthology predictions from the Ensembl Compara pipeline for fungi and all species, respectively.</p></li>
<li><p><a href="http://www.pantherdb.org/">PANTHER</a> links retrieve gene information, classification, and predicted orthologs.</p></li>
</ul>
</div>
<div id="2019-04-17-pombase-interpro-update" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-interpro-update">PomBase InterPro Update</h3>
<p><em>2019-04-17</em></p>
<p>PomBase now uses InterPro Version 73.0, which integrates 1,531 new methods from the CATH-Gene3D (122), CDD (330), PANTHER (1075), Pfam (2), PROSITE profiles (1) and TIGRFAMs (1) databases, and covers 81.2% of UniProt Knowledgebase release 2019_02.</p>
<p>See <a href="https://www.ebi.ac.uk/about/news/service-news/InterPro-73.0">the news item at InterPro</a> for additional information, including release notes.</p>
</div>
<div id="2019-04-16-pombe-gene-info-extension" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/gix_icon.png"/></p>
<h3 id="s.-pombe-included-gene-info-browser-extension"><em>S. pombe</em> included in Gene Info browser extension</h3>
<p><em>2019-04-16</em></p>
<p><em>S. pombe</em> gene information is now included in the <a href="https://gene-info.org/">Gene Info</a> extension (GIX) for the Chrome and Firefox web browsers. GIX allows you to retrieve information about a gene product directly on any webpage simply by double clicking an official gene name, synonym or supported accession. Searching or double-clicking on text terms retrieves gene function annotation, GO terms, external database links, and interaction data drawn from BioGRID and IntAct. Retrieved gene names are automatically hyperlinked for rapid recursive searches.</p>
<p>GeneInfo is fully open source, available online at <a href="https://github.com/knightjdr/gene-info/">GitHub</a>. Tutorial videos, a step-by-step guide, and download links for Firefox Add-ons and the Chrome web storeare available <a href="https://gene-info.org/">online</a>. GeneInfo was developed by James Knight in the <a href="https://gingraslab.lunenfeld.ca/">Gingras Lab</a> at the <a href="http://www.lunenfeld.ca/">Lunenfeld-Tanenbaum Research Institute</a> in Toronto, Canada.</p>
</div>
<div id="2019-03-19-motif-search-integrated" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/motif_search_cropped.png"/></p>
<h3 id="integrated-motif-search">Integrated motif search</h3>
<p><em>2019-03-19</em></p>
<p>The PomBase <a routerLink="/motif_search">motif search</a> has been fully integrated into the website, and allows users to find protein motifs and send them directly to the PomBase <a routerLink="/query">advanced search</a>.</p>
</div>
<div id="2019-03-13-icygmb2019" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/icygmb_tiny.png"/></p>
<h3 id="icygmb-2019-registration-open">ICYGMB 2019 - registration open</h3>
<p><em>2019-03-13</em></p>
<p>Registration is now open for the <a href="https://yeast2019.org">29th International Conference on Yeast Genetics and Molecular Biology (ICYGMB)</a>, which returns to Gothenburg, Sweden, August 18-22, 2019.</p>
<p>Yeast2019 is the meeting of the international yeast research community where the latest, and even unpublished results are exchanged, and new projects, alliances, and collaborations are founded. Featuring 55 confirmed speakers including keynote lectures by Susan Gasser, Roger Kornberg and Frederick Roth, this conference will contain important news and information for all yeast researchers. A do-not-miss-event.</p>
</div>
<div id="2019-03-05-list-go-slim-tool" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/slim_tiny.png"/></p>
<h3 id="go-slim-any-s.-pombe-gene-list">GO slim for any <em>S. pombe</em> gene list</h3>
<p><em>2019-03-05</em></p>
<p>PomBase’s advanced search now allows you to retrieve <a href="documentation/pombase-go-slim-documentation">GO slim</a> annotations for any set of search results. To find GO slim annotations for your own list of <em>S. pombe</em> genes, use the advanced search “Gene names and IDs” option, and then use the “Slim” button on the search results page.</p>
<p>See the <a href="browse-curation/fission-yeast-bp-go-slim-terms">fission yeast GO slim page</a> and the <a href="documentation/advanced-search">advanced search documentation</a> for more information.</p>
</div>
<div id="2019-03-04-serym2019" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/serym2019tiny.png"/></p>
<h3 id="south-eastern-regional-yeast-meeting-serym-registration-open">South Eastern Regional Yeast Meeting (SERYM) - registration open</h3>
<p><em>2019-03-04</em></p>
<p>Registration is now open for the <a href="https://petitinstitute.gatech.edu/serym-2019">26th annual South Eastern Regional Yeast Meeting (SERYM)</a>, which will be held April 12-14, 2019, in Atlanta, GA, USA.</p>
<p>Fission yeast’s own Susan Forsburg is the keynote speaker. The meeting brings together researchers who use any type of yeast as a model system, covering diverse, interdisciplinary topics from strategies for treatment of fungal disease to modeling human disease in yeast.</p>
<p>Icon: SERYM 2019</p>
</div>
<div id="2019-02-27-trieste-cell-cycle-meeting" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/chantal_roubinet_tiny.png"/></p>
<h3 id="international-cell-cycle-meeting-registration-open">International Cell Cycle meeting - registration open</h3>
<p><em>2019-02-27</em></p>
<p>Registration is now open for the <a href="https://www.ucl.ac.uk/lmcb/cell-cycle-meeting-2019">Inaugural Trieste Cell Cycle Meeting</a>, which will be held June 3-6, 2019, in Trieste, Italy.</p>
<p>This is the first of a planned series of biennial cell cycle meetings that will take place in Europe, and will alternate with the Salk Cell Cycle meetings held on the US west coast.</p>
<p>Organisers Rob de Bruin, Snezhana Oliferenko, Rosella Visintin and Peter Thorpe hope to see you there!</p>
<p>Icon derived from meeting image; credit: Chantal Roubinet, Baum lab</p>
</div>
<div id="2019-02-20-conserved-unknowns-open-biol" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/unknowns.png"/></p>
<h3 id="published-hidden-plain-sight-what-remains-be-discovered-eukaryotic-proteome">Published: Hidden in plain sight: What remains to be discovered in the eukaryotic proteome?</h3>
<p><em>2019-02-20</em></p>
<p>Our analysis of conserved unknown proteins has now been published in <a href="https://royalsocietypublishing.org/doi/10.1098/rsob.180241">Open Biology</a>. In it, PomBase curators consider the challenges and opportunities that conserved, but persistently unstudied, proteins pose for diverse areas of basic and applied biology. We develop metrics to define unknown lists, provide unknown inventories for human and yeast, and classify <em>S. pombe</em> unknowns by numerous orthogonal attributes, all with a view to drawing attention to the unknowns to alleviate their neglect.</p>
</div>
<div id="2019-02-15-pombe-2019-registration-open" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombe2019tiny.png"/></p>
<h3 id="pombe-2019-registration-open">Pombe 2019 - registration open</h3>
<p><em>2019-02-15</em></p>
<p>Registration for the <a href="http://www.pombe2019.com">10th International Fission Yeast Meeting</a> is now open!</p>
<p>The conference will take place July 14-19, 2019, in Barcelona, Spain. Early registration closes on April 15th — or when capacity is reached. Please see the <a href="http://www.pombe2019.com">conference website</a> for more information, including registration final deadline and costs (some travel grants are available), abstract submission, programme, accommodation, and logistics.</p>
</div>
<div id="2019-02-12-wood-isb-award" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/isb_logo_32px.png"/></p>
<h3 id="val-wood-wins-biocuration-society-award">Val Wood wins Biocuration society award</h3>
<p><em>2019-02-12</em></p>
<p>Congratulations to PomBase project leader Val Wood, who has received the 2019 Exceptional Contributions to Biocuration Award from the International Society for Biocuration. Read more at the <a href="https://www.biocuration.org/anne-morgat-and-val-wood-recipients-of-the-2019-biocuration-awards/">ISB site</a></p>
</div>
<div id="2019-01-09-improved-disease-association-dataset-released" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/human_disease.png"/></p>
<h3 id="improved-disease-association-dataset-released">Improved disease association dataset released</h3>
<p><em>2019-01-09</em></p>
<p>We are pleased to announce the release of our improved human disease mappings dataset. This dataset connects human disease causing genes to their <em>S. pombe</em> orthologs.</p>
<p>Diseases are now mapped to the Disease Ontology (DO) and the dataset has been extended by data from Malacards. All disease associations can be accessed from the top level disease page. A disease slim has been created to facilitate browsing of disease categories. Currently, 907 S. pombe genes are associated with disease (up from 610 in the original dataset). This number is due to increase as mappings are still in progress.</p>
<p>Many thanks to DO and Malacards for help in improving this annotation set. Icon courtesy of <a href="https://pixabay.com/en/users/mcmurryjulie-2375405/">Julie McMurry</a>.</p>
</div>
<div id="2018-12-17-mitochondrial-protein-review" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/mito_news.png"/></p>
<h3 id="mitochondrial-go-annotation-update">Mitochondrial GO annotation update</h3>
<p><em>2018-12-17</em></p>
<p>Responding to increasing interest in mitochondrial biology, especially relating to ageing, neurogenerative diseases, and processes at the ER-mitochondrion interface, we have reviewed <em>S. pombe</em> mitochondrial GO annotations. Although there is still relatively little fission yeast-derived experimental data in this area, we have refined many inferred annotations for mitochondrial complexes and sub-components as well as some for processes.</p>
<p>You can see all 753 <em>S. pombe</em> mitochondrial annotations on the ontology term page for <a href="https://www.pombase.org/term/GO:0005739">mitochondrion (GO:0005739)</a>.</p>
<p>Icon courtesy of Reactome.</p>
</div>
<div id="2018-12-03-new-nucleosome-occupancy-browser-maps-loaded" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/></p>
<h3 id="new-nucleosome-occupancy-maps-loaded">New nucleosome occupancy maps loaded</h3>
<p><em>2018-12-03</em></p>
<p>We have loaded the nucleosome occupancy maps as described in González et al. (2016) <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27662899">PMID: 27662899</a>. This dataset was generated using the paired-end sequencing protocol of Illumina and thus those maps are of higher resolution than those made with single-end (SE) sequencing hosted in the browser since before.</p>
<p>Here is a link that <a href="https://www.pombase.org/jbrowse/?loc=II%3A2751621..2767510&tracks=Forward%20strand%20features%2CReverse%20strand%20features%2Cnucleosome%20positioning%20during%20vegetative%20growth%3B%20repeat%201%20-%20Gonz%C3%A1lez%20et%20al.%20%282016%29%2Cnucleosome%20positioning%20during%20vegetative%20growth%3B%20repeat%202%20-%20Gonz%C3%A1lez%20et%20al.%20%282016%29%2CDNA%20sequence&highlight=">loads the tracks in PomBase JBrowse</a>. And here is a link to our <a href="https://www.pombase.org/documentation/JBrowse_quick_start">JBrowse quickstart guide</a>.</p>
<p>Many thanks to Paco Antequera for sending us the bigwig files! If you would like us to load any datasets then please get in touch.</p>
</div>
<div id="2018-11-21-quick-little-tool" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/QLT.png"/></p>
<h3 id="see-your-genes-quilt">See your genes in a QuiLT</h3>
<p><em>2018-11-21</em></p>
<p>PomBase now offers a new way to display gene lists graphically based on multiple orthogonal annotation types — the Quick Little Tool (QuiLT) for visualisation.</p>
<p>Inspired by our recent analysis of conserved unstudied proteins (see figures 4 and S1 in the <a href="https://www.biorxiv.org/content/early/2018/11/16/469569">manuscript at bioRxiv</a>), QuiLT allows you to create a similar figure for any gene list you create or import using the advanced search. To use QuiLT, follow the link to your search results, then click the “Visualise” button. QuiLT visualisation is also available from the PomBase pages that list genes annotated to an ontology term, and on the Priority unstudied genes page.</p>
<p>To see the Unknowns dataset in QuiLT, visit the <a routerLink="/results/from/predefined/priority-unstudied-genes:conserved_unknown">unknowns results page</a> and click “Visualise”.</p>
<p>The QuiLT display is interactive, and you can:</p>
<ul>
<li>Highlight subsets of the list, and filter the display</li>
<li>Toggle annotation types on and off</li>
<li>Reorder the list to focus on features of most interest</li>
<li>Download the image</li>
</ul>
<p>See the <a routerLink="/documentation/quick-little-tool">QuiLT documentation</a> for more information, and <a href="mailto:helpdesk@pombase.org">contact the curators</a> if you have comments, questions or suggestions.</p>
<p>Many thanks to our star (and only) programmer, Kim Rutherford, for developing QuiLT.</p>
</div>
<div id="2018-11-20-tm-transport-overhaul" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/antiporter.png"/></p>
<h3 id="fission-yeast-transmembrane-transport-overhaul">Fission yeast transmembrane transport overhaul</h3>
<p><em>2018-11-20</em></p>
<p>The Gene Ontology “<a href="https://www.pombase.org/term_genes/GO:0055085">transmembrane transport</a>” branch has recently been substantially revised. In line with these revisions, PomBase has standardised gene product descriptions for transporters, and overhauled GO annotations to be as complete and comprehensive as possible based on current knowledge.</p>
<p>Icon courtesy of <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29077811">Reactome</a>.</p>
</div>
<div id="2018-11-17-conserved-unknowns-pub" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/unknowns.png"/></p>
<h3 id="hidden-plain-sight-what-remains-be-discovered-eukaryotic-proteome">Hidden in plain sight: What remains to be discovered in the eukaryotic proteome?</h3>
<p><em>2018-11-17</em></p>
<p>In a new publication, PomBase curators consider the challenges and opportunities that conserved, but persistently unstudied, proteins pose for diverse areas of basic and applied biology. To draw attention to these proteins, we develop metrics to define unknown lists, provide unknown inventories for human and yeast, classify <em>S. pombe</em> unknowns by numerous orthogonal attributes, and speculate about reasons for their neglect.</p>
<p>A <a href="https://www.biorxiv.org/content/early/2018/11/16/469569">pre-publication manuscript</a> is available at bioRxiv.</p>
</div>
<div id="2018-11-14-S.-pombe-in-your-pocket" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/mobile_thumb.png"/></p>
<h3 id="pombase-your-pocket">PomBase in your pocket</h3>
<p><em>2018-11-14</em></p>
<p>Our usage statistics informed us that over 20% of devices accessing PomBase are smartphones or tablets. We therefore spent some time optimizing the display for small screens. We hope that you will continue to enjoy PomBase on the go!</p>
</div>
<div id="2018-11-08-20-years-GO" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/go-logo-icon.mini.png"/></p>
<h3 id="celebrating-20-years-go">Celebrating 20 years of GO</h3>
<p><em>2018-11-08</em></p>
<p>PomBase curators are major contributors to the <a href="http://geneontology.org/">Gene Ontology</a> (GO) project — ontology content, annotations, and QC procedures — and co-authors on the new <a href="https://doi.org/10.1093/nar/gky1055">GO NAR Database Issue paper</a>.</p>
<p>We recommend citing the GO and PomBase NAR papers when you use GO data in your analyses.</p>
</div>
<div id="2018-11-06-rna-central" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/rna_central_icon.png"/></p>
<h3 id="rna-central-and-pombase">RNA central and PomBase</h3>
<p><em>2018-11-06</em></p>
<p>RNAcentral is a comprehensive database of non-coding RNA sequences. PomBase is an RNAcentral Consortium member, and all of the curated non-coding RNAs from PomBase will be available in RNAcentral soon. For more information, see their recent <a href="https://doi.org/10.1093/nar/gky1034">NAR Database Issue paper</a>, as well as current <a href="https://rnacentral.org/search?q=pombe">search results for <em>S. pombe</em> RNAs</a>.</p>
</div>
<div id="2018-10-23-jbrowse-graphics-gene-page" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/></p>
<h3 id="new-pombase-genomic-region-graphics">New PomBase genomic region graphics</h3>
<p><em>2018-10-23</em></p>
<p>PomBase gene pages now use interactive graphics from PomBase JBrowse to depict the genomic region around the gene. Drag to scroll left and right, double-click to zoom in, shift-double-click to zoom out, and click a feature to see details in a popup. The “Full-screen view” link in the corner opens the fully functional JBrowse in a new tab or window. Reloading a gene page restores the display to the default location and zoom level.</p>
</div>
<div id="2018-10-15-nar-database-issue" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/NAR_paper.png"/></p>
<h3 id="pombase-nar-database-issue">PomBase NAR Database Issue</h3>
<p><em>2018-10-15</em></p>
<p>Our NAR database update “PomBase 2018: user-driven reimplementation of the fission yeast database provides rapid and intuitive access to diverse, interconnected information” is <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky961/5128936">now available</a>. We have updated the <a routerLink="/about/citing-pombase">Citing PomBase</a> to recommend citing this new paper. Thank you all for guiding the development of the new, improved PomBase, and for your continued usage, curation contributions, and suggestions!</p>
</div>
<div id="2018-10-10-2019-fungal-pathogen-genomics-course-registration-open" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/WT_course.png"/></p>
<h3 id="fungal-pathogen-genomics-course-2019-registration-open">Fungal Pathogen Genomics Course 2019 - registration open</h3>
<p><em>2018-10-10</em></p>
<p>Registration for the <a href="https://coursesandconferences.wellcomegenomecampus.org/our-events/fungal-pathogen-genomics-2019/">2019 Fungal Pathogen Genomics Course</a> is now open. The course is hosted by Wellcome Genome Advanced Courses and Scientific Conferences, and will take place May 7-12, 2019, at the Wellcome Genome Campus, Hinxton, UK. Course content provides hands-on training on how to: - Take advantage of unique tools offered by FungiDB, EnsemblFungi, PomBase, SGD/CGD, and MycoCosm/JGI; - Develop testable hypotheses; - Investigate transcriptomics, proteomics and genomics datasets across multiple databases and different user interfaces. Please see the <a href="https://coursesandconferences.wellcomegenomecampus.org/our-events/fungal-pathogen-genomics-2019/">course website</a> for more information, including how to apply, costs (limited bursaries are available), programme, and logistics.</p>
</div>
<div id="2018-10-06-transcript-tracks-from-Atkinson-et-al-(2018)-loaded" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/Atkinson.png"/></p>
<h3 id="transcript-tracks-atkinson-et-al.-2018-loaded">Transcript tracks from Atkinson et al. (2018) loaded</h3>
<p><em>2018-10-06</em></p>
<p>We are very pleased to announce that we have loaded the transcript tracks from <a href="https://www.ncbi.nlm.nih.gov/pubmed/29914874">Atkinson et al. (2018)</a> into the PomBase JBrowse genome browser. For a brief introduction to getting started with PomBase JBrowse, please see our <a href="https://www.pombase.org/documentation/JBrowse_quick_start">documentation page</a>. If you have published data that you would like to see hosted, please get in touch.</p>
</div>
<div id="2018-08-31-mailing-list-move-to-cam" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/pombelist.png"/></p>
<h3 id="pombelist-changes">pombelist changes</h3>
<p><em>2018-08-31</em></p>
<p>The pombe community mailing list, “pombelist”, is now hosted by the University of Cambridge. The new address for posting messages is &lt;pombelist{{"@"}}pombase.org&gt;. The link to <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">subscribe</a> has also changed.</p>
</div>
<div id="2018-05-28-new-browse-tracks" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/new_tracks.png"/></p>
<h3 id="new-genome-browser-tracks">New genome browser tracks</h3>
<p><em>2018-05-28</em></p>
<p>We are very pleased to announce that we have loaded a number of new datasets into the PomBase [JBrowse genome browser (https://www.pombase.org/jbrowse/). These include:</p>
<ul>
<li><a href="https://www.biorxiv.org/content/early/2018/03/13/281642">Thodberg <em>et al.</em> (2018)</a> - CAGE-defined <a
href="https://www.pombase.org/jbrowse/?loc=II%3A27586..65254&amp;tracks=DNA%20sequence%2CForward%20strand%20features%2CReverse%20strand%20features%2CConsensus%20transcription%20start%20sites%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20heat%20shock%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20heat%20shock%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20nitrogen%20starvation%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20nitrogen%20starvation%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20oxidative%20stress%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20oxidative%20stress%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20YES%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20YES%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20glucose%20minimal%20medium%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20glucose%20minimal%20medium%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)&amp;highlight=">transcription
start sites</a> across 5 different conditions</li>
<li><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23163955">Yadav <em>et al.</em> (2012)</a> - <a href="https://www.pombase.org/jbrowse/?loc=I%3A2364901..2436900&amp;tracks=Forward%20strand%20features%2CReverse%20strand%20features%2CPosition%20specific%20G(x)%20scores%20(free%20energy%20needed%20to%20separate%20a%20base%20pair)%20-%20Yadav%20et%20al.%20(2012)&amp;highlight=">G(x) scores</a> specifying the amount of free energy needed to melt base pairs in the DNA duplex at different genomic locations</li>
<li>A PomBase-generated <a href="https://www.pombase.org/jbrowse/?loc=III%3A725762..736349&amp;tracks=Forward%20strand%20features%2CReverse%20strand%20features%2CConsensus%20transcription%20factor%20DNA%20binding%20motifs&amp;highlight=">dataset of promoter elements</a> across the genome (computational matching of the <a routerLink="/browse-curation/dna-binding-sites">consensus promoter sequences</a> to the reference DNA sequence)</li>
<li><a href="https://www.pombase.org/jbrowse/?loc=I%3A2178501..2207300&amp;tracks=Forward%20strand%20features%2CReverse%20strand%20features%2CIntron%20Branch%20Points%20-%20Bitton%20et%20al.%20(2014)&amp;highlight=">Intron branch points</a> from <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24709818">Bitton <em>et al.</em> (2014)</a> which was also available in the Ensembl browser.</li>
</ul>
<p>For anyone wanting a quick introduction to our genome browser, Antonia Lock has written <a routerLink="/documentation/JBrowse_quick_start">“Getting started with PomBase JBrowse”</a>, a basic guide that covers loading tracks, navigating the browser, what metadata we provide, and more.</p>
</div>
<div id="2018-05-22-book-chapter" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/book_chapter.png"/></p>
<h3 id="new-book-chapter-about-pombase">New book chapter about PomBase</h3>
<p><em>2018-05-22</em></p>
<p>PomBase has a <a href="https://link.springer.com/protocol/10.1007/978-1-4939-7737-6_4">new book chapter</a> in <a href="https://link.springer.com/book/10.1007/978-1-4939-7737-6"><em>Eukaryotic Genomic Databases</em> (Methods and Protocols)</a>. This chapter provides insight into the curation philosophy and data organization at PomBase, and provides a guide to using PomBase tailored for infrequent visitors and anyone considering extending their research to include <em>S. pombe</em>. The chapter is <a href="https://link.springer.com/protocol/10.1007/978-1-4939-7737-6_4">free to download</a> courtesy of the <a href="https://wellcome.ac.uk/">Wellcome Trust</a>.</p>
</div>
<div id="2018-04-16-pombase-deploys-jbrowse" class="news-item">
<p><img loading="lazy" class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/></p>
<h3 id="pombase-releases-jbrowse">PomBase releases JBrowse</h3>
<p><em>2018-04-16</em></p>
<p>PomBase has now implemented <a href="https://www.pombase.org/jbrowse/?loc=II%3A564641..602220&amp;tracks=DNA%2CForward%20strand%20features%2CPomBase%20%20reverse%20strand%20features&amp;highlight=">JBrowse</a>, from the <a href="http://gmod.org/wiki/Main_Page">GMOD</a> project, as its genome browser. The new browser offers a number of improvements over the old:</p>
<ul>
<li>Quick, responsive scrolling and zooming</li>
<li>Simple track selection interface</li>
<li>Intuitive controls</li>
<li>Simplified data submission pipeline behind the scenes</li>
<li>More informative track metadata</li>
</ul>
</div>
<div id="2018-04-05-andre-goffeau" class="news-item">
<h3 id="in-memory-andr-eacute-goffeau">In memory of André Goffeau</h3>
<p><em>2018-04-05</em></p>
<p>Sadly, PomBase staff and the fission yeast community note the death of André Goffeau on April 2, 2018. In addition to initiating and coordinating the sequencing of the budding yeast genome, Prof. Goffeau will be remembered for his contributions to the fission yeast genome project and for his knowledge, leadership, and friendship.</p>
</div>
<div id="2017-11-24-congratulations-to-gsa-award-winners" class="news-item">
<h3 id="congratulations-gsa-award-winners">Congratulations to GSA award winners</h3>
<p><em>2017-11-24</em></p>
<p>The <a href="http://www.genetics-gsa.org">Genetics Society of America</a> (GSA) has announced two award winners familiar to the model organism database world:</p>
<ul>
<li>Ira Herskowitz Award: Mike Cherry, Stanford University</li>
<li>Lifetime Achievement Award: Steve Oliver, University of Cambridge</li>
</ul>
<p>The awards will be presented at the next <a href="http://conferences.genetics-gsa.org/yeast/2018/index">Yeast Genetics Meeting</a>, at Stanford University in August 2018. Congratulations and thanks to Mike and Steve!</p>
</div>
<div id="2017-10-24-new-improved-pombase-goes-live" class="news-item">
<h3 id="new-improved-pombase-goes-live">New, improved PomBase goes live</h3>
<p><em>2017-10-24</em></p>
<p>The new PomBase web site, which has been under development during 2017, has been released. The new site features:</p>
<ul>
<li>Nightly data updates</li>
<li>New publication pages</li>
<li>New genotype pages</li>
<li>Improved ontology term pages</li>
<li>Improved summary views for annotation displays</li>
<li>Phenotype annotation display filtering</li>
<li>Faster querying in the advanced search</li>
<li>Front page research and community curation highlights</li>
<li>Streamlined back-end data storage and retrieval</li>
</ul>
<p>We thank the members of the fission yeast research community who have followed its progress via the preview site, and welcome feedback from all users.</p>
</div>
<div id="2016-12-11-9th-international-fission-yeast-meeting-early-registration-closes-soon" class="news-item">
<h3 id="id-9th-international-fission-yeast-meeting-early-registration-closes-soon">9th International Fission Yeast Meeting - early registration closes soon</h3>
<p><em>2016-12-11</em></p>
<p>Reminder: early registration for the 9th International Fission Yeast Meeting in Banff closes <strong>Dec. 31, 2016</strong>. Please see the conference website at <a href="http://www.pombe2017.com">www.pombe2017.com</a> for details.</p>
</div>
<div id="2016-10-31-9th-international-fission-yeast-meeting-registration-open" class="news-item">
<h3 id="id-9th-international-fission-yeast-meeting-registration-open">9th International Fission Yeast Meeting registration open</h3>
<p><em>2016-10-31</em></p>
<p>Registration for the 9th International Fission Yeast Meeting is now open. The meeting will be held in Banff, Canada from May 14-19, 2017. Early registration closes Dec 1, 2016! Please see our website at <a href="http://www.pombe2017.com">www.pombe2017.com</a> for details. We look forward to seeing you in Banff!<br />
<br />
- Conference Organizers: Dallan Young, Gordon Chua, Paul Young</p>
</div>
<div id="2016-10-19-pombase-data-update-23" class="news-item">
<h3 id="pombase-data-update-2016-10-19">PomBase data update 2016-10-19</h3>
<p><em>2016-10-19</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through September 11, 2016.</p>
</div>
<div id="2016-06-27-show-your-support-database-funding" class="news-item">
<h3 id="show-your-support-database-funding">Show your support for database funding</h3>
<p><em>2016-06-27</em></p>
<p>In response to planned cuts to database funding, leading model organism researchers have prepared an open letter to NIH Director Dr. Francis Collins to demonstrate support for the independent community-focused databases that are essential to their work. Although PomBase is not directly funded by NIH, we collaborate extensively with those that are, including the GO Consortium and several model organism databases.<br />
<br />
The Genetics Society of America website where the letter can be viewed and signed is at <a href="http://www.genetics-gsa.org/MODsupport" class="uri">http://www.genetics-gsa.org/MODsupport</a><br />
<br />
Please sign the letter to add your voice in support of the databases that help make your research possible. For more information, we recommend an email that Mike Cherry sent to the GO-Friends mailing list, archived at <a href="https://mailman.stanford.edu/pipermail/go-friends/2016-June/002355.html" class="uri">https://mailman.stanford.edu/pipermail/go-friends/2016-June/002355.html</a></p>
</div>
<div id="2016-06-15-our-model-organism-database-commentary" class="news-item">
<h3 id="our-model-organism-database-commentary">Our model organism database commentary</h3>
<p><em>2016-06-15</em></p>
<p>Several of the PomBase staff, joined by our advisor Sir Paul Nurse, have published a Comment in <em>BMC Biology</em> briefly describing the importance of model organism databases to the success of modern biomedical research:<br />
<br />
Oliver SG, Lock A, Harris MA, Nurse P, Wood V. 2016. Model organism databases: essential resources that need the support of both funders and users.<br />
<em>BMC Biol.</em> 2016 <strong>14(1)</strong>: 49. doi: <a href="http://dx.doi.org/10.1186/s12915-016-0276-z">10.1186/s12915-016-0276-z</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27334346">PMID:27334346</a></p>
</div>
<div id="2016-05-31-pombase-data-update-22" class="news-item">
<h3 id="pombase-data-update-2016-05-31">PomBase data update 2016-05-31</h3>
<p><em>2016-05-31</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through May 12, 2016.</p>
</div>
<div id="2016-05-09-pombase-data-update-21" class="news-item">
<h3 id="pombase-data-update-2016-05-09">PomBase data update 2016-05-09</h3>
<p><em>2016-05-09</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through April 8, 2016.</p>
</div>
<div id="2016-04-11-pombase-data-update-20" class="news-item">
<h3 id="pombase-data-update-2016-04-11">PomBase data update 2016-04-11</h3>
<p><em>2016-04-11</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through March 9, 2016.</p>
<p><strong>Important:</strong> We have corrected a problem that made erroneous interaction data and literature appear on some gene pages.</p>
<p>The gene pages now include interaction data from the Vo et al. proteome-wide study (curated by BioGRID and imported into PomBase):<br />
Vo TV <em>et al.</em> 2016. A Proteome-wide Fission Yeast Interactome Reveals Network Evolution Principles from Yeasts to Human. <em>Cell</em> <strong>164(1-2)</strong>: 310-23. doi: <a href="http://dx.doi.org/10.1016/j.cell.2015.11.037">10.1016/j.cell.2015.11.037</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=26771498">PMID:26771498</a>.</p>
<p>The genome browser now includes transcriptome data published in:<br />
<br />
Eser P, Wachutka L, Maier KC, Demel C, Boroni M, Iyer S, Cramer P, Gagneur J. 2016. Determinants of RNA metabolism in the Schizosaccharomyces pombe genome. <em>Mol Syst Biol</em>. <strong>12(2):</strong> 857. doi: <a href="http://dx.doi.org/10.15252/msb.20156526">10.15252/msb.20156526</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=26883383">PMID:26883383</a>.</p>
</div>
<div id="2016-02-11-pombase-data-update-19" class="news-item">
<h3 id="pombase-data-update-2016-02-11">PomBase data update 2016-02-11</h3>
<p><em>2016-02-11</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through January 25, 2016.<br />
The genome browser includes variation data, in tracks under “Variation”, from natural <em>S. pombe</em> isolates, published in:<br />
<br />
Jeffares DC <em>et al.</em> 2015. The genomic and phenotypic diversity of Schizosaccharomyces pombe. <em>Nat Genet.</em> <strong>47(3):</strong> 235-241. doi:<a href="http://dx.doi.org/10.1038/ng.3215">10.1038/ng.3215</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25665008">PMID:25665008</a></p>
<p>New files are now available from the PomBase FTP site, and are linked from pages in the Download Datasets area:</p>
<ul>
<li>Non-coding RNA sequence feature coordinates (available via the <a href="/downloads/data-mappings">Data Mappings</a> page);</li>
<li>Protein features, such as domains and family assignments (available via the <a routerLink="/downloads/protein-datasets">Protein Datasets</a> page);</li>
<li>Protein modification annotations (also in Protein Datasets).</li>
</ul>
<p>The <a routerLink="/status/new-and-removed-genes">New and Removed Genes page</a> has been updated to reflect recent deletions and merges.<br />
<br />
Note: Ontology graph views are no longer available in the genome browser, so links have been removed from the GO, FYPO, and modification tables on the gene pages. For GO and FYPO, links to external ontology browsers that offer graphical views are available on the <a href="/documentation/ontology-term-page">Ontology Term pages</a>.<br />
<br />
</p>
</div>
<div id="2015-12-02-pombase-data-update-18" class="news-item">
<h3 id="pombase-data-update-2015-12-02">PomBase data update 2015-12-02</h3>
<p><em>2015-12-02</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through November 9, 2015, including 340 community-curated publications.</p>
</div>
<div id="2015-12-02-new-advanced-search-features" class="news-item">
<h3 id="new-advanced-search-features">New Advanced Search features</h3>
<p><em>2015-12-02</em></p>
<p>We have introduced new features to the <a href="/spombe/query/builder">Advanced Search</a>:</p>
<ul>
<li>There are now two query reuse options: store a query in your web browser cache, or download a JSON file that can be uploaded later to re-run.</li>
<li>You can now query for genes that interact genetically or physically with a specified gene.</li>
<li>The FYPO query now offers options to choose nulls (deletions or disruptions), wild-type overexpression, or all alleles. The search results will include any genes that have an allele that matches the allele criteria and the chosen phenotype.</li>
</ul>
</div>
<div id="2015-10-19-new-genetics-primer-fission-yeast" class="news-item">
<h3 id="new-genetics-primer-fission-yeast">New genetics primer for fission yeast</h3>
<p><em>2015-10-19</em></p>
<p>A new genetics primer, aimed at researchers interested in using fission yeast as a model system, has recently been published. The primer includes a brief history of fission yeast research, an introduction to available genetic tools, and the use of PomBase for data analysis</p>
<p>Hoffman CS, Wood V, Fantes PA. (2015) An Ancient Yeast for Young Geneticists: A Primer on the <em>Schizosaccharomyces pombe</em> Model System. <em>Genetics</em> <strong>201</strong>:403-423. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26447128">PMID:26447128</a> <a href="http://dx.doi.org/10.1534/genetics.115.181503">DOI:10.1534/genetics.115.181503</a></p>
</div>
<div id="2015-09-28-pombase-data-update-viability-summary-correction-alert" class="news-item">
<h3 id="pombase-data-update-viability-summary-correction-alert">PomBase data update; viability summary correction alert</h3>
<p><em>2015-09-28</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through September 6, 2015.</p>
<p>Errors in the previous FYPOviability.tsv file have been corrected, and we recommend that all users update this file, especially those who downloaded it earlier in September 2015.</p>
</div>
<div id="2015-09-03-pombase-data-update-multi-allele-phenotypes" class="news-item">
<h3 id="pombase-data-update-multi-allele-phenotypes">PomBase data update with multi-allele phenotypes</h3>
<p><em>2015-09-03</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through August 13, 2015, including 300 community-curated publications.</p>
<p>PomBase gene pages now include multi-allele phenotype annotations (i.e. phenotypes of double mutants, triple mutants, etc.). New sub-sections of the gene pages display multi-allele phenotypes at the population and individual cell level, paralleling the organisation of the single allele phenotype display. Compact and full views are available; both show phenotypes with the relevant genotypes and the alleles that make them up, and the full view adds details for evidence, expression, conditions, and references.</p>
<p>The genome browser now includes data tracks for two more publications:<br />
<br />
DNA polymerase usage from:<br />
Daigaku Y, Keszthelyi A, Müller CA, Miyabe I, Brooks T, Retkute R, Hubank M, Nieduszynski CA, Carr AM. 2015. A global profile of replicative polymerase usage. <em>Nat Struct Mol Biol.</em> 2015 Mar;<strong>22(3):</strong>192-8. doi: <a href="http://dx.doi.org/10.1038/nsmb.2962">10.1038/nsmb.2962</a> PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25664722">25664722</a><br />
<br />
Promoters and transcription start sites from:<br />
Li H, Hou J, Bai L, Hu C, Tong P, Kang Y, Zhao X, Shao Z. 2015. Genome-wide analysis of core promoter structures in Schizosaccharomyces pombe with DeepCAGE. <em>RNA Biol.</em> 2015;<strong>12(5):</strong>525-37. doi: <a href="http://dx.doi.org/10.1080/15476286.2015.1022704">10.1080/15476286.2015</a>.1022704 PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25747261">25747261</a></p>
<p>Codon adaptation index (CAI) values are now included in the Protein Properties section of the gene pages and in the downloadable PeptideStats.tsv file. A file of amino acid composition data is also available from the FTP site and the <a href="/downloads/protein-datasets">Protein Datasets</a> page.</p>
<p>The gene page section that was formerly misnamed “species distribution” is now called “taxonomic conservation”.</p>
</div>
<div id="2015-06-16-pombase-data-update-17" class="news-item">
<h3 id="pombase-data-update-2015-06-16">PomBase data update 2015-06-16</h3>
<p><em>2015-06-16</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through May 26, 2015, including 270 community-curated publications. See you at Pombe 2015 in Kobe!</p>
</div>
<div id="2015-05-26-canto-downtime-new-version" class="news-item">
<h3 id="canto-downtime-new-version">Canto downtime &amp; new version</h3>
<p><em>2015-05-26</em></p>
<p>Canto, PomBase’s literature curation tool, will be unavailable for approximately 3 weeks starting at 12:00 midnight UK time (BST) tonight, 27 May 2015, while we deploy an upgraded version.<br />
<br />
The upgraded Canto will feature an entirely new interface for annotating multi-allele phenotypes and the corresponding genotypes, as well as improved workflows for single-allele phenotypes, GO, etc. All existing annotations will be retained, and users can resume curation using the new and improved features in any unfinished sessions when Canto is back online.<br />
<br />
We will announce when the new version of Canto is released to the public.</p>
</div>
<div id="2015-05-26-pombase-data-update-16" class="news-item">
<h3 id="pombase-data-update-2015-05-26">PomBase data update 2015-05-26</h3>
<p><em>2015-05-26</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through May 8, 2015, including 265 community-curated publications.</p>
</div>
<div id="2015-04-23-pombe-2015-travel-fellowships" class="news-item">
<h3 id="pombe-2015-travel-fellowships">Pombe 2015 travel fellowships</h3>
<p><em>2015-04-23</em></p>
<p>Applications are now being accepted for fellowships to provide financial support for students and postdocs attending the 8th International Fission Yeast Meeting in Kobe, Japan. To apply, follow the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2015/004246.html">instructions sent to the pombase mailing list</a>. The deadline is may 17, 2015 (same as the registration deadline).</p>
</div>
<div id="2015-04-19-pombase-data-update-15" class="news-item">
<h3 id="pombase-data-update-2015-04-19">PomBase data update 2015-04-19</h3>
<p><em>2015-04-19</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through April 7, 2015, including 260 community-curated publications.The Advanced Search now supports queries for proteins with a specified number of transmembrane domains.</p>
</div>
<div id="2015-04-19-pombe-2015-poster-abstract-deadline-extended" class="news-item">
<h3 id="pombe-2015-poster-abstract-deadline-extended">Pombe 2015 poster abstract deadline extended</h3>
<p><em>2015-04-19</em></p>
<p>The abstract submission deadline for the <a href="https://amarys-jtb.jp/web/Pombe2015/index.html">8th International Fission Yeast Meeting</a> in Kobe, Japan has been extended until midnight Friday, April 24 for <strong>posters only</strong>. Registration is open until May 17.</p>
</div>
<div id="2015-04-09-pombe-2015-abstract-deadline-approaching" class="news-item">
<h3 id="pombe-2015-abstract-deadline-approaching">Pombe 2015 abstract deadline approaching</h3>
<p><em>2015-04-09</em></p>
<p>Abstracts are due on Sunday, April 19, 2015 for the <a href="https://amarys-jtb.jp/web/Pombe2015/index.html">8th International Fission Yeast Meeting</a> in Kobe, Japan. Registration will remain open until May 17, but the abstract submission deadline cannot be extended.</p>
</div>
<div id="2015-03-23-pombase-data-update-14" class="news-item">
<h3 id="pombase-data-update-2015-03-23">PomBase data update 2015-03-23</h3>
<p><em>2015-03-23</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through March7, 2015, including 250 community-curated publications.The autocomplete feature of the Advanced Search ontology term filter has been improved with respect to response time and relevance of suggested terms.</p>
</div>
<div id="2015-02-26-pombe-2015-registration-now-open" class="news-item">
<h3 id="pombe-2015-registration-now-open">Pombe 2015 registration now open</h3>
<p><em>2015-02-26</em></p>
<p>Registration for Pombe 2015: 8th International Fission Yeast Meeting is now open at the conference web site, <a href="https://amarys-jtb.jp/web/Pombe2015/index.html" class="uri">https://amarys-jtb.jp/web/Pombe2015/index.html</a></p>
<p>The registration deadline is 17 May 2015.</p>
<p>Thanks to Yasushi Hiraoka for this item.</p>
</div>
<div id="2015-02-16-pombase-data-update-13" class="news-item">
<h3 id="pombase-data-update-2015-02-16">PomBase data update 2015-02-16</h3>
<p><em>2015-02-16</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through February 2, 2015, including 245 community-curated publications. On the gene pages, the interaction tables now provides a bit of descriptive text for each annotation, indicating the nature and direction of the interaction.</p>
</div>
<div id="2015-01-26-pombase-data-update-12" class="news-item">
<h3 id="pombase-data-update-2015-01-26">PomBase data update 2015-01-26</h3>
<p><em>2015-01-26</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through January 12, 2015, including 240 community-curated publications. The gene page Phenotype section now features a compact default display. A downloadable “viability summary” data file is now available. The PomBase BLAST server has incorporated interface changes made Ensembl-wide.</p>
</div>
<div id="2014-12-10-new-compact-go-annotation-display" class="news-item">
<h3 id="new-compact-go-annotation-display">New compact GO annotation display</h3>
<p><em>2014-12-10</em></p>
<p>To make the Gene Ontology (GO) annotations easier to read on PomBase gene pages, we have introduced a new, streamlined display that presents just the essentials. The summary shows the term name (hyperlinked to the ontology term page), the count of genes annotated to the term, and any annotation extensions. All of the previously visible annotation details are still available – simply click the “Summary” button to switch to the “Full” view. Or click the “+” and “-” icons to expand or collapse the annotation to a single term.<br />
 <br />
In addition, the top of the Biological Process table now lists any GO slim terms applicable to the gene.</p>
</div>
<div id="2014-12-10-esyn-network-visualizations-pombase" class="news-item">
<h3 id="esyn-network-visualizations-pombase">esyN network visualizations in PomBase</h3>
<p><em>2014-12-10</em></p>
<p>PomBase has implemented network visualisations for fission yeast in “esyN, using data curated by BioGRID and PomBase. esyN is a web-based tool for building, sharing, and viewing network data developed by Dan Bean and Giorgio Favrin in the <a href="http://www.sysbiol.cam.ac.uk">Cambridge Systems Biology Centre</a>, University of Cambridge, UK.<br />
<br />
On gene pages, we have links to gene-specific interaction networks in esyN in the table headers of the Interactions sections:</p>
<ul>
<li>The Genetic Interactions section links to all interactions centred on the gene and curated in BioGRID</li>
<li>The Physical interactions section has links to two datasets:
<ul>
<li>All physical interactions curated in BioGRID for the gene</li>
<li>Interactions for the gene in the PomBase <a href="/documentation/high-confidence-physical-interaction-network">High Confidence Physical Interaction Network (HCPIN)</a></li>
</ul></li>
</ul>
</div>
<div id="2014-11-12-pombase-data-update-11" class="news-item">
<h3 id="pombase-data-update-2014-11-12">PomBase data update 2014-11-12</h3>
<p><em>2014-11-12</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through October 27, 2014, including 225 community-curated publications. The gene page Phenotype section now includes data from the high-throughput microscopy analysis of viable deletion mutants reported in:<br />
<br />
Graml V, Studera X, Lawson JL, Chessel A, Geymonat M, Bortfeld-Miller M, Walter T, Wagstaff L, Piddini E, Carazo-Salas RE. A Genomic Multiprocess Survey of Machineries that Control and Link Cell Shape, Microtubule Organization, and Cell-Cycle Progression. Dev Cell. 2014 Oct 27;31(2):227-39. doi: <a href="http://dx.doi.org/10.1016/j.devcel.2014.09.005">10.1016/j.devcel.2014.09.005</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25373780">PMID:25373780</a>. Links to the accompanying SYSGRO resource have been added to the External References section of the gene pages.</p>
<p>The genome browser now includes tracks for intron branch point data from:</p>
<p>Bitton DA, Rallis C, Jeffares DC, Smith GC, Chen YY, Codlin S, Marguerat  S, Bähler J. LaSSO, a strategy for genome-wide mapping of intronic  lariats and branch points using RNA-seq. Genome Res. 2014 Jul;24(7):1169-79. doi: <a href="http://dx.doi.org/10.1101/gr.166819.113">10.1101/gr.166819.113</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24709818">PMID:24709818</a>.</p>
<p>We have greatly improved search results for GO and FYPO annotations: both now follow more relationship types within the ontology to retrieve genes annotated to a term. The PomBase GO search now includes the <em>regulates</em> relationships, so its search results are consistent with those in the GO Consortium’s AmiGO browser. The FYPO search now uses <em>has_part,</em> <em>has_output</em>, and <em>output_of</em> as well as <em>is_a</em> and <em>part_of</em>. The Phenotype section now includes a highlighted sub-header that indicates whether a deletion mutant is viable or inviable. A file of protein complex subunits is available for download, and numerous smaller improvements have been made in the gene pages and static pages.</p>
</div>
<div id="2014-09-16-pombase-data-update-10" class="news-item">
<h3 id="pombase-data-update-2014-09-16">PomBase data update 2014-09-16</h3>
<p><em>2014-09-16</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through August 30, 2014. Community curation now covers over 200 papers.</p>
</div>
<div id="2014-08-18-pombase-data-update-9" class="news-item">
<h3 id="pombase-data-update-2014-08-18">PomBase data update 2014-08-18</h3>
<p><em>2014-08-18</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through August 8, 2014. Community curation now covers over 190 papers. Gene pages now include links to the <a href="https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/buildDetails?atlas_build_id=355"><em>S. pombe</em> PeptideAtlas</a>, a database of peptides identified in tandem mass spectrometry proteomics experiments.</p>
</div>
<div id="2014-07-17-pombase-data-update-8" class="news-item">
<h3 id="pombase-data-update-2014-07-17">PomBase data update 2014-07-17</h3>
<p><em>2014-07-17</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through July 8, 2014. The gene pages also now display protein modification data from an additional large-scale dataset:</p>
<p>Koch A, Krug K, Pengelley S, Macek B, Hauf S. 2011. Mitotic substrates of the kinase aurora with roles in chromatin regulation identified through quantitative phosphoproteomics of fission yeast. <em>Sci Signal</em>. <strong>4</strong>(179): rs6 doi: 10.1126/scisignal.2001588 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21712547">PMID:21712547</a><br />
<br />
We have also made corrections to some residue positions affected by sequence updates in one of the modification datasets we added last month:</p>
<p>Carpy A, Krug K, Graf S, Koch A, Popic S, Hauf S, Macek B. 2014. Absolute proteome and phosphoproteome dynamics during the cell cycle of fission yeast. <em>Mol Cell Proteomics</em>. 2014 Apr 23. [Epub ahead of print] <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24763107">PMID:24763107</a></p>
</div>
<div id="2014-07-08-pombase-data-update-7" class="news-item">
<h3 id="pombase-data-update-2014-07-08">PomBase data update 2014-07-08</h3>
<p><em>2014-07-08</em></p>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through June 6, 2014. In other improvements, a downloadable file of <a href="https://www.pombase.org/data/genome_sequence_and_features/feature_sequences/cds+introns+utrs.fa.gz">intron sequence data</a> (FASTA format) is now available, and phenotypes are now included in the Target Of section on gene pages.</p>
<p>The gene pages also now display protein modification data from two large-scale datasets:</p>
<ul>
<li>Wilson-Grady JT, Villén J, Gygi SP. 2008 .Phosphoproteome analysis of fission yeast. <em>J Proteome Res</em>. 2008 Mar;<strong>7</strong>(3):1088-97. doi:10.1021/pr7006335. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18257517">PMID:18257517</a></li>
<li>Carpy A, Krug K, Graf S, Koch A, Popic S, Hauf S, Macek B. 2014. Absolute proteome and phosphoproteome dynamics during the cell cycle of fission yeast. <em>Mol Cell Proteomics</em>. 2014 Apr 23. [Epub ahead of print] <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24763107">PMID:24763107</a></li>
</ul>
<p>Link updated 2021-02-04</p>
</div>
<div id="2014-06-29-gene-ontology-publication-annotation-extensions" class="news-item">
<h3 id="gene-ontology-publication-on-annotation-extensions">Gene Ontology publication on annotation extensions</h3>
<p><em>2014-06-29</em></p>
<p>PomBase was an early adopter of <a routerLink="/documentation/gene-page-gene-ontology#ann_ext">annotation extensions</a>, which add spatial, temporal, or substrate/target details to GO annotations. The GO Consortium has now published a paper describing its implementation of annotation extensions, in which PomBase examples and its gene page display figure prominently:</p>
<p>Huntley, R.P. <em>et al.</em> (2014) A method for increasing expressivity of Gene Ontology annotations using a compositional approach. <em>BMC Bioinformatics</em> 2014, <strong>15</strong>:155. <a href="http://www.biomedcentral.com/1471-2105/15/155/abstract">doi:10.1186/1471-2105-15-155</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24885854">PMID:24885854</a></p>
</div>
<div id="2014-05-15-pombase-data-update-6" class="news-item">
<h3 id="pombase-data-update-2014-05-15">PomBase data update 2014-05-15</h3>
<p><em>2014-05-15</em></p>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through April 28, 2014. Transcriptome data from <a href="http://europepmc.org/abstract/MED/23101633">Margeurat <em>et al</em> (2012)</a> is now available as Ensembl Browser tracks.</p>
<p>Thank you to all who have done, or are doing, paper curation in Canto. Over 159 community-curated papers are now included in PomBase.</p>
<p>There are a number of routes to accelerate your data into PomBase, (either through community curation, or by supplying HTP sequence, modification or phenotype data in one of our specified formats), see <a href="http://www.pombase.org/submit-data" class="uri">http://www.pombase.org/submit-data</a> for more details.</p>
<p>As usual, please don’t hesitate to alert us of any other problems with data or site performance, or if you have any questions.</p>
<p>Sincerely yours,<br />
The PomBase Staff</p>
</div>
<div id="2014-03-20-pombase-data-update-5" class="news-item">
<h3 id="pombase-data-update-2014-03-20">PomBase data update 2014-03-20</h3>
<p><em>2014-03-20</em></p>
<p>Data on the PomBase web site now includes manual curation through February 24, 2014. Human orthologs that went missing from gene pages have been restored, and other small improvements have been made to gene pages. Community curation now covers over 130 publications.</p>
</div>
<div id="2014-02-20-pombase-data-update-4" class="news-item">
<h3 id="pombase-data-update-2014-02-20">PomBase data update 2014-02-20</h3>
<p><em>2014-02-20</em></p>
<p>We have once again updated the data available on the PomBase web site. The data now includes manual curation through January 10, 2014, and covers over 100 papers that have been curated in Canto by community members. We again thank all who have contributed curation via Canto.</p>
<p>We have made some improvements to the gene pages. Highlights:</p>
<ul>
<li>The Sequence section now has links to NCBI BLAST as well as Ensembl BLAST.</li>
<li>The External References section now links to the Pomb(A) polyadenylation viewer.</li>
</ul>
<p>In the genome browser, new data tracks are now available for data from these publications:</p>
<ul>
<li>Rhind N, [and many more]. 2011. Comparative functional genomics of the fission yeasts. Science 332(6032):930-6. doi: 10.1126/science.1203357. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:21511999</a></li>
<li>Schlackow M, Marguerat S, Proudfoot NJ, Bähler J, Erban R, Gullerova M. 2013. Genome-wide analysis of poly(A) site selection in <em>Schizosaccharomyces pombe</em>. RNA. 19(12):1617-31. doi:10.1261/rna.040675.113. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24152550">PMID:24152550</a></li>
<li>Soriano I, Quintales L, Antequera F. 2013. Clustered regulatory elements at nucleosome-depleted regions punctuate a constant nucleosomal landscape in Schizosaccharomyces pombe. BMC Genomics. 14:813. doi:10.1186/1471-2164-14-813. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24256300">PMID:24256300</a> (partial data;  remainder coming in the next update)</li>
<li>Xu J, Yanagisawa Y, Tsankov AM, Hart C, Aoki K, Kommajosyula N, Steinmann KE, Bochicchio J, Russ C, Regev A, Rando OJ, Nusbaum C, Niki H, Milos P, Weng Z, Rhind N. 2012. Genome-wide identification and characterization of replication origins by deep sequencing. Genome Biol. 13(4):R27. doi:10.1186/gb-2012-13-4-r27. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22531001">PMID:22531001</a></li>
</ul>
<p>Now that more data tracks are available, we have added some categories to the track configuration section to improve organization. Additional documentation is in preparation, and will be announced here when available.<br />
<br />
Genome sequences for additional <em>Schizosaccharomyces</em> species (<em>S. japonicus</em>, <em>S. octosporus</em>, and <em>S. cryophilus</em>) have recently become available in Ensembl Fungi, and the PomBase genome browser now includes comparative genomics data, with a view of region comparisons between each new genome and <em>S. pombe</em>.</p>
</div>
<div id="2014-02-19-human-ortholog-data-correction-coming-next-month" class="news-item">
<h3 id="human-ortholog-data-correction-coming-next-month">Human ortholog data correction coming next month</h3>
<p><em>2014-02-19</em></p>
<p>We are about to release a data update for PomBase. Please note that there is still a problem with the human orthologs, as originally described on this list in mid-December (see archived message at <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2013/003926.html" class="uri">http://listserver.ebi.ac.uk/pipermail/pombelist/2013/003926.html</a>). We will correct this problem in the next PomBase release, and apologise for any inconvenience in the meantime.</p>
</div>
<div id="2013-12-08-pombase-data-update-3" class="news-item">
<h3 id="pombase-data-update-2013-12-08">PomBase data update 2013-12-08</h3>
<p><em>2013-12-08</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through November 11, 2013. We now have future meetings available as a calendar or a list. The <a routerLink="/faq">FAQ</a> and some documentation pages have also been updated.<br />
</p>
<p>2021-08-18: Updated to remove out-of-date links (events are now listed only as news items).</p>
</div>
<div id="2013-11-24-2013-meeting-mini-reviews-published" class="news-item">
<h3 id="id-2013-meeting-mini-reviews-published">2013 meeting mini-reviews published</h3>
<p><em>2013-11-24</em></p>
<p>A series of mini-reviews, which were invited in association with the International Fission Yeast Meeting in London, have now been published in Biochemical Society Transactions: <a href="http://www.biochemsoctrans.org/bst/041/6/default.htm#c" class="uri">http://www.biochemsoctrans.org/bst/041/6/default.htm#c</a></p>
<p>(Thanks to Jürg Bahler for this item)</p>
</div>
<div id="2013-11-20-pombase-survey-results-available" class="news-item">
<h3 id="pombase-survey-results-available">PomBase survey results available</h3>
<p><em>2013-11-20</em></p>
<p>The 2013 PomBase user survey closed at the end of October, and the results are available <a href="https://www.pombase.org/data/documents/2013_pombase_survey_summary.pdf">here</a> (PDF at FTP site). Some highlights have been sent to the pombe mailing list. Many thanks to all who completed the survey.</p>
<p>Link updated 2021-02-04</p>
</div>
<div id="2013-10-27-new-target-gene-page-section" class="news-item">
<h3 id="new-target-gene-page-section">New “Target Of” gene page section</h3>
<p><em>2013-10-27</em></p>
<p>With the October 2013 update, gene pages now include “<a href="/documentation/gene-page-target">Target Of</a>” annotations, which describe genes that affect the gene of interest. These annotations are essentially the reciprocal of <a href="/faqs/what-annotation-extension">ontology annotation extensions</a>. Each “Target Of” annotation includes a relationship that indicates how the genes are connected, the name and product of the second gene, and a reference. Genes listed under “Target Of” may include upstream regulators or enzymes that modify the product of the gene of interest. For example, the “Target Of” annotations for <a routerLink="/spombe/result/SPBC11B10.09#recipext">cdc2</a> indicate that it is a substrate of, and regulated by, the kinase Wee1 and the phosphatase Cdc25 (among others). At present, “Target Of” data includes annotations derived from <a href="/documentation/gene-page-gene-ontology#ann_ext">GO annotation extensions</a>. We will soon extend it to include data
from <a routerLink="/documentation/gene-page-phenotypes">phenotype</a> annotation extensions.</p>
</div>
<div id="2013-10-21-pombase-data-update-2" class="news-item">
<h3 id="pombase-data-update-2013-10-21">PomBase data update 2013-10-21</h3>
<p><em>2013-10-21</em></p>
<p>The PomBase web site has been updated and now includes manually curated data through October 6, 2013. The number of community-curated papers continues to increase, ensuring that PomBase gene pages contain complete and up-to-date information. We are also pleased to announce that data tracks are now available in the genome browser for data from these two publications:</p>
<ul>
<li>Woolcock KJ, Gaidatzis D, Punga T, Bühler M. 2010. Dicer associates with chromatin to repress genome activity in Schizosaccharomyces pombe. Nat Struct Mol Biol. 2011 Jan;18(1):94-9. doi: 10.1038/nsmb.1935 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21151114">PMID:21151114</a></li>
<li>Mata J. 2013. Genome-wide mapping of polyadenylation sites in fission yeast reveals widespread alternative polyadenylation. RNA Biol. 2013 Aug 1;10(8):1407-14. doi: 10.4161/rna.25758 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23900342">PMID:23900342</a></li>
</ul>
</div>
<div id="2013-09-18-pombase-user-survey-open" class="news-item">
<h3 id="pombase-user-survey-open">PomBase User Survey open</h3>
<p><em>2013-09-18</em></p>
<p>To guide current and future development, PomBase is now conducting a <a href="https://www.surveymonkey.com/s/NDM2BQX">user survey</a>, where we invite the fission yeast research community and any other PomBase users to evaluate the resources provided so far and comment on future priorities. The survey should take about 10 minutes to complete. Thank you for your participation!</p>
<p><a href="https://www.surveymonkey.com/s/NDM2BQX" class="uri">https://www.surveymonkey.com/s/NDM2BQX</a></p>
</div>
<div id="2013-09-15-pombase-data-update-1" class="news-item">
<h3 id="pombase-data-update-2013-09-15">PomBase data update 2013-09-15</h3>
<p><em>2013-09-15</em></p>
<p>We have once again updated the data available on the PomBase web site. The data now includes manual curation through August 11, 2013. We are particularly pleased to note that this update includes annotations from several dozen papers curated by the <em>S. pombe</em> community. Many thanks to all who have done, or are doing, paper curation in <a href="http://curation.pombase.org/pombe/">Canto</a>.<br />
<br />
We also have an updated version of the <em>S. pombe</em>/human ortholog table available upon <a href="mailto:helpdesk@pombase.org">request</a>.</p>
</div>
<div id="2013-08-18-send-htp-data-pombase" class="news-item">
<h3 id="send-htp-data-pombase">Send HTP data to PomBase</h3>
<p><em>2013-08-18</em></p>
<p>At the pombe 2013 meeting in London, PomBase staff received numerous requests display various published data, such as gene expression, histone modifications, etc. in the genome browser. To provide this, we now invite pombe researchers to send data: If you have published any high-throughput experiments that produced data  that can be associated with genome sequence coordinates, and thereby displayed as tracks on the PomBase genome browser, please fill out the <a href="/submit-data/data-submission-form">HTP Data Submission Form</a>. We can also accept large sets of phenotype data via the Phenotype Data Submission Form. If you have any problems or questions, contact us via the <a href="mailto:helpdesk%7B%7B%22@%22%7D%7Dpombase.org">PomBase Helpdesk</a> at any time.</p>
</div>
<div id="2013-07-29-connecting-pombase" class="news-item">
<h3 id="connecting-pombase">Connecting With PomBase</h3>
<p><em>2013-07-29</em></p>
<p>To complement the <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">mailing list</a> and twitter (<a href="http://twitter.com/PomBase">{{"@"}}PomBase</a>) it is now possible to follow the activities of PomBase and interact with other members of the pombe community via the new <a href="http://www.linkedin.com/groups/PomBase-5122686">LinkedIn Group</a> and <a href="https://plus.google.com/114842152049074799331">Google+</a>.</p>
<p>Links to these are also available from the front page of the PomBase.org site.</p>
</div>
<div id="2013-07-21-pombelist-has-moved" class="news-item">
<h3 id="pombelist-has-moved">pombelist has moved</h3>
<p><em>2013-07-21</em></p>
<p><strong>Update: This item dates from July 2013, and the links in it no longer work.</strong> \ Please see the <a routerLink="/community">Fission Yeast Community</a> page for the current mailing list link. \ (2020-02-18)</p>
<p>The pombe community mailing list, pombelist, has migrated from the Wellcome Trust Sanger Institute and is now hosted by EBI. The new address is &lt;pombelist{{"@"}}ebi.ac.uk&gt; (please note that the old address no longer works, and will generate an automatic notification including the new address). The link to <a href="http://listserver.ebi.ac.uk/mailman/listinfo/pombelist">subscribe</a> has also been updated, and the entire <a href="http://listserver.ebi.ac.uk/mailman/private/pombelist/">archive</a> is available at the new location.</p>
</div>
<div id="2013-07-18-pombase-website-update" class="news-item">
<h3 id="pombase-website-update">PomBase website update</h3>
<p><em>2013-07-18</em></p>
<p>We’d like to highlight a few improvements we’ve just made to the PomBase website. Most of the changes affect the gene pages:</p>
<ul>
<li>The basic information display at the top of each gene page is more compact.</li>
<li>For ontology annotations, the number of genes annotated is now shown, in a column labeled “Count” (also, changes behind the scenes involving this data mean that pages should load faster).</li>
<li>Annotation extensions for GO are displayed using human-friendly text instead of internal “relation” labels.</li>
<li>The Quick Links box can now be collapsed and expanded by clicking its header.</li>
<li>Display of modification annotations using PSI-MOD is improved.</li>
</ul>
<p>In addition, the Motif Search output now includes standard gene names and product descriptions. As we noted in a separate message, CDS coordinate files are once again available from the Downloads, with accurate and up-to-date data.<br />
<br />
</p>
</div>
<div id="2013-06-23-pombase-launches-community-curation" class="news-item">
<h3 id="pombase-launches-community-curation">PomBase launches community curation</h3>
<p><em>2013-06-23</em></p>
<p>At the pombe 2013 conference in London, PomBase officially launched its community curation initiative, which allows researchers to contribute publication-based annotations directly to the database. PomBase curators invite lab heads by individual email to curate newly published papers, providing links to the online curation system and its documentation. Researchers can also initiate curation of any older fission yeast publication in PubMed. Community curation uses the open-source online tool <a href="http://curation.pombase.org/pombe/">Canto</a>.</p>
</div>
<div id="2013-06-20-pombase-data-update-0" class="news-item">
<h3 id="pombase-data-update-2013-06-20">PomBase data update 2013-06-20</h3>
<p><em>2013-06-20</em></p>
<p>PomBase data now includes manual curation through June 9, 2013, and represents complete annotation for 664 publications (as well as partial curation of many more). A highlight of this month’s literature curation update is the addition of over 9400 phenotype annotations, representing about 95% of the phenotype data from the recently published genome-wide study of cell cycle and cell morphology (<a href="http://rsob.royalsocietypublishing.org/content/3/5/130053.abstract">Hayles <em>et al.</em> Open Biology May 2013</a>; <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23697806">PMID:23697806</a>). We have also improved the display of allele details for phenotype annotations. Other changes include better support for gene synonyms in the simple search, regular updates to the UTR data files, and a number of minor adjustments to external links in the annotation data tables and the external references section.</p>
</div>
<div id="2013-05-20-pombase-data-update" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2013-05-20</em></p>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through 13 May, 2013.</p>
</div>
<div id="2013-05-13-genedb-s-pombe-decommissioned" class="news-item">
<h3 id="genedb-s.-pombe-decommissioned">GeneDB S. pombe decommissioned</h3>
<p><em>2013-05-13</em></p>
<p>As of 14 May 2013, the old GeneDB database for <em>S. pombe</em> is no longer available. This resource consisted of static web pages, was not updated after March 2012, and not supported by an underlying relational database. The PomBase site fully supersedes GeneDB <em>S. pombe</em>, and provides improved infrastructure that will meet the current and future needs of the fission yeast community. Please e-mail the <a href="mailto:helpdesk@pombase.org">helpdesk</a> if you cannot find a replacement for any GeneDB functionality in PomBase.</p>
</div>
<div id="2013-05-07-quantitative-gene-expression-data-available-pombase" class="news-item">
<h3 id="quantitative-gene-expression-data-available-pombase">Quantitative gene expression data available in PomBase</h3>
<p><em>2013-05-07</em></p>
<p>We have extended the Gene Expression section of each gene page to support the display of quantitative expression data, and are now showing data from two publications:</p>
<ul>
<li>Marguerat S, Schmidt A, Codlin S, Chen W, Aebersold R, BählerJ. 2012. “Quantitative analysis of fission yeast transcriptomes and proteomes in proliferating and quiescent cells.” <em>Cell</em> <strong>151:</strong>671-683.</li>
<li>Wu JQ, Pollard TD. 2005. “Counting cytokinesis proteins globally and locally in fission yeast.” <em>Science</em> <strong>310:</strong>310-314.</li>
</ul>
<p>We will also soon refine the display of the new expression data, and can add more datasets upon request. We thank Sam Marguerat for preparing the data from both papers for inclusion in PomBase.</p>
<p>We have also updated the PomBase site to include manual curation through April 4, 2013, and we have updated the “all gene names” file on the PomBase ftp site. The new file is available at<br />
<a href="https://www.pombase.org/data/names_and_identifiers/gene_IDs_names.tsv" class="uri">https://www.pombase.org/data/names_and_identifiers/gene_IDs_names.tsv</a></p>
<p>Link updated 2021-02-04</p>
</div>
<div id="2013-04-11-carl-singer-foundation-established" class="news-item">
<h3 id="carl-singer-foundation-established">Carl Singer Foundation Established</h3>
<p><em>2013-04-11</em></p>
<p>Carl Singer, who was an integral part of the yeast research community for many years, passed away on February 8, 2013. Throughout his career, Carl supported yeast research both with his engineering expertise and with his good cheer. In tribute to Carl, the Singer family has now set up <a href="http://www.thecarlsingerfoundation.org/">The Carl Singer Foundation</a>, a charitable foundation dedicated to supporting scientific education in the field of yeast genetics. Questions about the foundation may be directed to Harry Singer at harry [at] thecarlsingerfoundation.org.<br />
<br />
Carl’s family would be happy to receive memories of Carl’s life at regards [at] singerinstruments.com.</p>
<p>H/T <a href="http://www.yeastgenome.org/carl-singer-foundation-established">SGD</a></p>
</div>
<div id="2013-04-02-pombe-2013-registration-abstracts-mon-8th-april" class="news-item">
<h3 id="pombe-2013-registration-abstracts-by-mon-8th-april">Pombe 2013: registration &amp; abstracts by Mon 8th April</h3>
<p><em>2013-04-02</em></p>
<pre><code>Dear Pombe Fans,
      Please remember the imminent deadline (Monday 8th April) to register and submit abstracts for Pombe 2013: http://events.embo.org/13-pombe
      Abstracts are also required from all who have already been invited to talk.
      And do book your accommodation if you haven&#39;t yet done so.
      More details are in previous email forwarded below.
      Cheers,
      -Jürg &amp; Jacky
      From: On Behalf Of Bahler, Jurg
      Sent: 18 March 2013 17:49
      To: pombelist at sanger.ac.uk
      Subject: [Pombelist] Pombe 2013: Accommodation, registration &amp; abstracts
      Dear Pombe Afficionados,
      Only three weeks left to register and submit abstracts for Pombe 2013, by Monday 8th April: http://events.embo.org/13-pombe
      Speakers for 10 plenary talks and all workshop talks will be selected from abstracts, and there will be attractive poster prizes.
      Payment is only requested after registration, by 10th May.
      Important: if you require accommodation, please do book this real soon now. Especially the most cost-effective student accommodation (comfortable, with private bathrooms) may not be available much longer, as it will be put on general sale shortly. Both hotels and student accommodation will sell out in June, so you have to arrange it now. Information on accommodation is available here: http://events.embo.org/13-pombe/application.html
      We will provide a number of free registrations for which you can apply during online registration (a few of which are reserved for student members of The Genetics Society: you become eligible if you join them now). The meeting is also supported by the Biochemical Society, so if you are, or become, a member you can apply to them for student bursaries or, if you have been a member for at least 1 year, also for travel grants.
      We highly appreciate all the generous contributions from our sponsors so far:
      Platinum: EMBO
      Gold: Biochemical Society, Genetics Society, Formedium, Sunrise Science Products, Singer Instruments, F1000Research, PomBase/Wellcome Trust
      Silver: MDPI - Open Access Publishing, Hybrigenics, Infors, Life Technologies, Bioneer
      Bronze: Nature Communications, m2p labs, Imsol, Open Biology
      We look very much forward to welcoming you in London this June!
      All the best,
      -Jürg &amp; Jacky</code></pre>
</div>
<div id="2013-04-01-data-update-pombase-web-site" class="news-item">
<h3 id="data-update-on-pombase-web-site">Data update on PomBase web site</h3>
<p><em>2013-04-01</em></p>
<p>We have once again updated the data available on the PomBase web site. The data now includes manual curation through March 6, 2013.<br />
<br />
We now expect to be able to update PomBase data every month, and will soon have an automated pipeline in place. We thank all of you for your patience during the long months when updates were infrequent.<br />
<br />
You should also see a few small improvements in the site:</p>
<ul>
<li>Ontology term pages now display the text definition for each term.</li>
<li>FASTA sequence retrieval should be quicker, and less likely to time out, for large gene lists.</li>
<li>There has been some tidying of the display of “extension” data for GO and phenotype annotations.</li>
</ul>
<p>Last month we noted an intermittent problem with the “Reference” column display in the data tables. The occurrence of this problem should now be greatly reduced, so please let us know if you see it recurring.<br />
<br />
As usual, please don’t hesitate to alert us of any other problems with data or site performance, or if you have any questions.<br />
<br />
</p>
</div>
<div id="2013-03-01-speed-improvements-and-new-data-pombase-web-site" class="news-item">
<h3 id="speed-improvements-and-new-data-on-pombase-web-site">Speed improvements and new data on PomBase web site</h3>
<p><em>2013-03-01</em></p>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through December 17, 2012, and reflects complete curation of an additional 70 papers.<br />
<br />
We have also made some improvements “under the hood” that should make gene page loading much faster. Please let us know if you have any problems with gene pages loading slowly or incompletely, whether or not you have reported issues in the past.<br />
<br />
We are aware that there is an intermittent problem with the “Reference” column display in the data tables – sometimes a PubMed ID appears instead of an author name and year. This problem will be fixed as soon as possible. Please alert us if you notice anything else odd or wrong.</p>
</div>
<div id="2012-11-06-new-data-and-new-features-pombase-web-site" class="news-item">
<h3 id="new-data-and-new-features-on-pombase-web-site">New data and new features on PomBase web site</h3>
<p><em>2012-11-06</em></p>
<p>We are pleased to announce that we have updated both data and web site features for PomBase.</p>
<p>Most importantly, we have added new data types, and upgraded the gene pages to display them.</p>
<p>We have also added more annotations of existing data types, bringing the web site content up to September 11, 2012. The new annotations include the first contributions to come in via the new community curation system, and we thank the researchers who are participating in the initial phase of community curation.</p>
<p><strong>New annotation types:</strong></p>
<ul>
<li>Phenotype annotations now use the Fission Yeast Phenotype Ontology (FYPO), and include allele details, expression levels, and experimental conditions. With FYPO, more detailed phenotypes can be described, and links between terms for related phenotypes support improved phenotype searches.</li>
<li>Many GO annotations now include “annotation extensions” that provide additional specificity. For example, extensions may capture the substrate of a catalytic activity, the cell cycle phase during which a function or process occurs, or any of several other types of supporting information for the annotation. Annotation extensions are described in more detail below.</li>
</ul>
<p>You can see these new data types on many gene pages, such as <a routerLink="/spombe/result/SPBC11B10.09">cdc2</a> or <a routerLink="/spombe/result/SPBC106.10">pka1</a>.</p>
<p><strong>New web site features:</strong></p>
<ul>
<li>Annotation display - Gene page GO and phenotype displays have been revamped to show new annotation types described above.</li>
<li>Ontology term pages - Each ontology term ID now links to pages with information about the term and lists of genes annotated to it.</li>
<li>Ontology graph links - GO and phenotype annotation sections now include links to graphical ontology displays in the genome browser.</li>
<li>Sequence highlighting - Sequence download now offers an option to show colour highlighting of regions such as UTRs, introns and exons.</li>
<li>Versions - Each gene page now shows the current data version in the format PomBase:x.y, where <em>x</em> is the Ensembl Genomes (EG) version, and <em>y</em> is the Chado version. The sequence, and sequence feature locations, remain stable within any EG version, whereas annotations change with each Chado update.</li>
<li>Protein family information is now included in the Protein Features gene page section.</li>
<li>The Protein Feature section includes a link to the Pfam entry for a protein.</li>
<li>Transcript source data (e.g. for UTR coordinates) is now displayed in the Transcript Features section.</li>
<li>A Documentation page contains links to relevant portions of the Ensembl Genomes documentation. (More documentation will be added over the coming months.)</li>
</ul>
<p><strong>What are annotation extensions?</strong></p>
<p>Annotation extensions are a form of supporting data that can be added GO annotations (or other ontology annotations) to capture additional details not provided by the ontology term itself.</p>
<p>The information in GO annotation extensions encompasses several effector-target relationships, such as</p>
<ul>
<li>localisation dependencies</li>
<li>substrates of functions, e.g. targets of a protein kinase – see the has_substrate extensions on Cdc2’s “protein serine/threonine kinase” (GO:0004674) annotations</li>
<li>activators and inhibitors</li>
<li>regulation targets of signalling pathways or transcription factors</li>
</ul>
<p>Additional extensions describe spatial and temporal aspects of processes. For example, several S. pombe annotations now include extensions that indicate in which phase of the cell cycle a gene product is found in a cellular component or involved in a process – see the pka1 annotations to “nucleus” (GO:0005634) and “cytoplasm” (GO:0005737).</p>
<p>You may also find the <a href="http://wiki.geneontology.org/index.php/Annotation_Extension">GO wiki page</a> on annotation extensions informative, although it is primarily aimed at curators.</p>
<p>Annotation extensions can also be used with phenotype annotations. The most common usage of phenotype annotation extensions is to capture which gene, protein, etc. was used in an assay. For example, the sam5 (G441E) mutation of <em>pka1</em> causes nuclear accumulation of Ste11. This is represented by annotation to the ontology term “nuclear protein accumulation” (FYPO:0000255), with the extension “assayed_using(PomBase:SPBC32C12.02)”. Extensions can also indicate expressivity or penetrance for a phenotype.</p>
</div>
<div id="2012-07-01-pombase-web-site-fully-live" class="news-item">
<h3 id="pombase-web-site-fully-live">PomBase web site fully live</h3>
<p><em>2012-07-01</em></p>
<p><strong>Note (2023-06-09): This is an archived news item about PomBase V1. See <a routerLink="/documentation/advanced-search">the documentation page</a> to learn new <a routerLink="/query">Advanced search</a> in PomBase V2. </strong></p>
<p>We are pleased to announce that the PomBase web site, <a href="http://www.pombase.org">www.pombase.org</a>, is now fully live; the preview phase has ended. The site has been updated with an assortment of new features, datatypes, and bug fixes.</p>
<p>More recent data, reflecting additions and changes through March 20, 2012, are now available on gene pages and in search results.</p>
<p>The updated site features a Gene List Search that provides behavior equivalent to GeneDB’s List Download. You can now type or paste lists into the Gene Systematic IDs and Gene Names filters, and use the Query History to combine a gene list search with other search options. For convenience, there is a direct link to a search page pre-configured to accept a list of systematic IDs available in the Find menu, on the Find page, and here: <a routerLink="/spombe/query/builder?filter=12">http://www.pombase.org/spombe/query/builder?filter=12</a></p>
<p>The Advanced Search also now offers:</p>
<ul>
<li>options to search GO, FYPO, and other ontologies by term name or ID;</li>
<li>autocomplete for ontology term name search;</li>
<li>ability to search for genes in a region, such as centromeres or telomeres;</li>
<li>improved organization of filter selections.</li>
</ul>
<p>We have also fixed a Sequence Download error reported by some users, so that the “CDS”, CDS + UTRs”, and “CDS + UTRs + Introns” options now retrieve the correct sequences.<br />
<br />
In addition, numerous minor improvements have been made. Please send any questions or comments on the PomBase web site to us at &lt;&lt;helpdesk{{"@"}}pombase.org&gt;&gt;.</p>
</div>
<div id="2011-11-27-pombase-preview-launch" class="news-item">
<h3 id="pombase-preview-launch">PomBase preview launch</h3>
<p><em>2011-11-27</em></p>
<p>A preview of <a href="http://www.pombase.org">PomBase</a>, the new model organism database for the fission yeast <em>Schizosaccharomyces pombe</em>, has been <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000970.html">announced to the <em>S. pombe</em> community</a> for testing and feedback. For more on PomBase, see the NAR Database Issue paper (<a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153">PubMed abstract</a>) or <a href="/about/contacts">contact the PomBase staff</a>.</p>
</div>
<div id="2011-10-27-pombase-nar-paper-published-online" class="news-item">
<h3 id="pombase-nar-paper-published-online">PomBase NAR paper published online</h3>
<p><em>2011-10-27</em></p>
<p>A paper describing <a href="http://www.pombase.org">PomBase</a> has been published online will be included in the 2012 Database Issue of <em>Nucleic Acids Research</em>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153">Abstract</a> and <a href="http://nar.oxfordjournals.org/content/early/2011/10/28/nar.gkr853">open access full text</a> are available.</p>
</div>
<div id="2011-04-28-genedb-s-pombe-now-uses-latest-release-pfam-protein-family-database250" class="news-item">
<h3 id="genedb-s.-pombe-now-uses-latest-release-pfam-protein-family-database-25.0-.">GeneDB (S. pombe) now uses the latest release of the Pfam protein family database(25.0).</h3>
<p><em>2011-04-28</em></p>
</div>
<div id="2011-04-21-schizosaccharomyces-comparative-genome-paper-published" class="news-item">
<h3 id="schizosaccharomyces-comparative-genome-paper-published">Schizosaccharomyces Comparative Genome Paper Published</h3>
<p><em>2011-04-21</em></p>
<p>A paper describing the major findings of the <em>Schizosaccharomyces</em> Comparative Genome Project was <a href="http://www.sciencemag.org/content/early/2011/04/20/science.1203357">published today</a> in Science Express and reported changes are included in GeneDB.</p>
<p>Further details are described in the pombe mailing list posts:</p>
<ul>
<li><a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000855.html">Schizosaccharomyces Comparative Genome Paper Published</a></li>
<li><a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000856.html">Import of the fission yeast revisions from the Broad Institute comparative genome paper into GeneDB</a>.</li>
</ul>
</div>
<div id="2011-02-01-genome-reappraisal-reveals-new-genes-and-revised-gene-structures" class="news-item">
<h3 id="genome-reappraisal-reveals-new-genes-and-revised-gene-structures">Genome reappraisal reveals new genes and revised gene structures</h3>
<p><em>2011-02-01</em></p>
<p>Further information on the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000815.html">pombe mailing list</a>.</p>
</div>
<div id="2011-01-31-annotated-transcription-start-and-termination-sites-fission-yeast" class="news-item">
<h3 id="annotated-transcription-start-and-termination-sites-fission-yeast">Annotated transcription start and termination sites for fission yeast</h3>
<p><em>2011-01-31</em></p>
<p>Further details are available on the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000814.html">pombe mailing list</a>.</p>
</div>
<div id="2010-05-15-analysis-fission-yeast-deletion-publication" class="news-item">
<h3 id="analysis-fission-yeast-deletion-publication">Analysis of Fission Yeast Deletion Publication</h3>
<p><em>2010-05-15</em></p>
<p>The analysis of the fission yeast deletion collection is now published online in <a href="http://www.nature.com/nbt/journal/v28/n6/abs/nbt.1628.html">Nature Biotechnology</a>.</p>
</div>
<div id="2010-02-28-funding-pombase" class="news-item">
<h3 id="funding-pombase">Funding for PomBase</h3>
<p><em>2010-02-28</em></p>
<p>Funding was awarded by the Wellcome Trust for a fission yeast Model Organism Database, <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2010-March/002731.html">PomBase</a>.</p>
</div>
<div id="2009-11-30-fission-yeast-one-12-key-organisms-reference-genomes-project" class="news-item">
<h3 id="fission-yeast-one-12-key-organisms-reference-genomes-project">Fission yeast is one of the 12 key organisms of the reference genomes project</h3>
<p><em>2009-11-30</em></p>
<p>Fission yeast is one of the 12 key organisms of the <a href="http://geneontology.org/GO.refgenome.shtml?all">reference genomes project</a>. The goal of this project is to completely annotate twelve reference genomes so that those annotations may be used to effectively seed the automatic annotation efforts of other genome.</p>
</div>
<div id="2009-10-31-genedb-s-pombe-now-uses-latest-update-pfam-release-240" class="news-item">
<h3 id="genedb-s.-pombe-now-uses-latest-update-pfam-release-24.0">GeneDB (S. pombe) now uses the latest update to Pfam, release 24.0</h3>
<p><em>2009-10-31</em></p>
<p>GeneDB (S. pombe) now uses the latest update to Pfam, release 24.0 and 88.5% of fission yeast proteins now contain a match to at least one Pfam domain (increased from 83% in version 23).</p>
</div>
<div id="2009-09-30-fission-yeast-ensembl-fungi" class="news-item">
<h3 id="fission-yeast-ensembl-fungi">Fission yeast in Ensembl Fungi</h3>
<p><em>2009-09-30</em></p>
<p>The fission yeast genome and annotation dataset is now available as part of <a href="http://fungi.ensembl.org/index.html">Ensembl Fungi</a>.</p>
</div>
<div id="2009-08-31-genedb-now-using-version-23-pfam-protein-family-database" class="news-item">
<h3 id="genedb-now-using-version-23-pfam-protein-family-database.">GeneDB is now using Version 23 of the Pfam protein family database.</h3>
<p><em>2009-08-31</em></p>
<p>GeneDB is now using Version 23 of the Pfam protein family database. A total of 4154 (83%) S. pombe proteins now have at least one Pfam domain or family assignment (compared to 76% for S. cerevisiae), the highest percentage coverage for any eukaryote.</p>
</div>
<div id="2008-11-30-s-pombe-genedb-now-includes-deep-links-biological-general-repository-interaction-datasets-biogr" class="news-item">
<h3 id="s.-pombe-genedb-now-includes-deep-links-biological-general-repository-interaction-datasets-biogrid">S. pombe GeneDB now includes “deep links” to the Biological General Repository for Interaction Datasets (BioGRID)</h3>
<p><em>2008-11-30</em></p>
<p><em>S. pombe</em> GeneDB now includes “deep links” to the Biological General Repository for Interaction Datasets (BioGRID) interaction datasets from the ‘Database Cross References’ section of the individual Gene Pages.</p>
</div>
<div id="2008-04-30-global-sequence-and-chip-study-examines-eukaryotic-transcription" class="news-item">
<h3 id="global-sequence-and-chip-study-examines-eukaryotic-transcription">Global sequence and chip study examines eukaryotic transcription</h3>
<p><em>2008-04-30</em></p>
<p>Dynamic repertoire of the fission yeast transcriptome reveals: 94% of the genome is transcribed; extensive variation in different stages and conditions; global and condition-specific coupling between splicing efficiency and transcription; confirms the majority of introns; refines ~75 gene structures; identifies 453 new transcripts 26 of which were predicted to code for proteins.</p>
</div>
<div id="2008-01-31-h-mating-type-region-has-been-provided" class="news-item">
<h3 id="the-h-mating-type-region-has-been-provided">The h- mating type region has been provided</h3>
<p><em>2008-01-31</em></p>
<p>The <a routerLink="/status/mating-type-region">h- mating type region</a> has been provided by Xavier Marsellach and Lorena Aguilar.</p>
</div>
<div id="2007-12-31-baumann-and-zakian-labs-identify-telomerase-rna" class="news-item">
<h3 id="baumann-and-zakian-labs-identify-telomerase-rna">Baumann and Zakian labs identify telomerase RNA</h3>
<p><em>2007-12-31</em></p>
<p>Baumann and Zakian labs identify elusive <a href="/spombe/result/SPNCRNA.214">telomerase RNA</a> (PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18157152">18157152</a> and PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18157149">18157149</a>)</p>
</div>
<div id="2007-09-30-wellcome-trust-advanced-course-genome-wide-approaches-fission-yeast" class="news-item">
<h3 id="wellcome-trust-advanced-course-genome-wide-approaches-fission-yeast">Wellcome Trust Advanced Course ’Genome-wide approaches with fission yeast</h3>
<p><em>2007-09-30</em></p>
<p>Wellcome Trust Advanced Course ‘Genome-wide approaches with fission yeast’ held in Hinxton.</p>
</div>
<div id="2007-05-31-4th-international-fission-yeast-meeting" class="news-item">
<h3 id="id-4th-international-fission-yeast-meeting">4th International Fission Yeast Meeting</h3>
<p><em>2007-05-31</em></p>
<p>4th International Fission Yeast Meeting held in Copenhagen.</p>
</div>
<div id="2006-12-31-genedb-representation-fission-yeast-data-moved-contigs-chromosomes" class="news-item">
<h3 id="genedb-representation-fission-yeast-data-moved-contigs-chromosomes">GeneDB representation of the fission yeast data moved from contigs to chromosomes</h3>
<p><em>2006-12-31</em></p>
<p>GeneDB representation of the fission yeast data moved from contigs to chromosomes. See the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2007/000128.html">pombelist archive</a> for details.</p>
</div>
<div id="2006-09-30-yeast-special-issue-2006-european-fission-yeast-meeting" class="news-item">
<h3 id="yeast-special-issue-2006-european-fission-yeast-meeting">Yeast Special Issue from the 2006 European Fission Yeast Meeting</h3>
<p><em>2006-09-30</em></p>
<p>The October issue of the journal Yeast is a <a href="http://onlinelibrary.wiley.com/doi/10.1002/yea.v23:13/issuetoc">fission yeast special issue</a> containing 13 articles and reviews commissioned as a result of the European Fission Yeast Meeting, which are FREE to download.</p>
</div>
<div id="2006-06-30-first-fission-yeast-whole-proteome-localization-study-now-published" class="news-item">
<h3 id="the-first-fission-yeast-whole-proteome-localization-study-now-published">The first fission yeast whole proteome localization study is now published</h3>
<p><em>2006-06-30</em></p>
<p>The first fission yeast whole proteome localization study is now published: <a href="http://www.nature.com/nbt/journal/v24/n7/abs/nbt1222.html">Matsuyama A. et al (2006): ORFeome cloning and global analysis of protein localization in the fission yeast Schizosaccharomyces pombe. Nat Biotech 24, 841-7.</a><br />
 </p>
</div>
<div id="2006-04-30-fission-yeast-database-survey" class="news-item">
<h3 id="fission-yeast-database-survey">Fission yeast database survey</h3>
<p><em>2006-04-30</em></p>
<p>The fission yeast database survey is now closed. You can view the survey results <a routerLink="/community/pombase_2006_survey_summary">here</a>.</p>
</div>
<div id="2006-03-17-european-fission-yeast-meeting" class="news-item">
<h3 id="european-fission-yeast-meeting">European Fission Yeast Meeting</h3>
<p><em>2006-03-17</em></p>
<p>The European Fission Yeast Meeting (16th-18th March 2006) and The Fission Yeast Bioinformatics workshop (15th - 16th Mar 2006) both took place at the Wellcome Trust Genome Campus in Hinxton (Cambridge, UK).</p>
</div>
<div id="2005-11-16-comparative-genomics-eukaryotic-microorganisms" class="news-item">
<h3 id="comparative-genomics-eukaryotic-microorganisms">Comparative Genomics of Eukaryotic Microorganisms</h3>
<p><em>2005-11-16</em></p>
<p>Comparative Genomics of Eukaryotic Microorganisms:<br />
Eukaryotic Genome Evolution, Approaches with Yeasts and Fungi</p>
<p><br />
This conference took place from 12th-17th November 2005 in Sant Feliu de Guixols, Spain. Full details can be found <a href="http://www.esf.org/index.php?id=940">here</a>.</p>
</div>
<div id="2005-11-12-second-east-coast-regional-pombe-meeting" class="news-item">
<h3 id="second-east-coast-regional-pombe-meeting">Second East Coast Regional pombe Meeting</h3>
<p><em>2005-11-12</em></p>
<p>Second East Coast Regional pombe Meeting<br />
<br />
This meeting took place from November 11-13, 2005 in Miami Beach, Florida.</p>
</div>
<div id="2004-08-31-general-repository-interaction-datasets" class="news-item">
<h3 id="general-repository-interaction-datasets">General Repository for Interaction Datasets</h3>
<p><em>2004-08-31</em></p>
<p>A project to record published genetic and physical interactions is underway with Mike Tyers and the GRID group at Toronto.</p>
</div>
<div id="2004-08-29-third-international-fission-yeast-meeting" class="news-item">
<h3 id="the-third-international-fission-yeast-meeting">The Third International Fission Yeast Meeting</h3>
<p><em>2004-08-29</em></p>
<p>The meeting was held at UC San Diego on August 24-29, 2004.</p>
</div>
<div id="2004-04-30-methods-volume-33-issue-3" class="news-item">
<h3 id="methods-volume-33-issue-3">Methods Volume 33 Issue 3</h3>
<p><em>2004-04-30</em></p>
<p>This issue of Methods includes 11 papers for fission yeast protocols including DNA damage checkpoint assays, cell wall analysis, TAP, nuclear envelope integrity assays, GFP imaging, TS mutant creation and plasmid use and construction. See the <a href="http://www.sciencedirect.com/science/journal/10462023/33/3">Methods site</a> for details of the papers including PMIDs.</p>
<p>2021-08-18: Updated to remove out-of-date link.</p>
</div>
<div id="2003-10-31-recent-genome-wide-surveys" class="news-item">
<h3 id="recent-genome-wide-surveys">Recent Genome wide surveys</h3>
<p><em>2003-10-31</em></p>
<p>Correlations Between Gene Expression and Gene Conservation in Fission Yeast. Mata J, Bahler J. Genome Res. 2003 Nov 12 <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14613978">PMID:14613978</a><br />
<br />
FELINES: a utility for extracting and examining EST-defined introns and exons. Drabenstot SD et al Nucleic Acids Res. 2003 Nov 15;31(22):e141. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14602934">PMID:14602934</a><br />
<br />
Genome-wide distribution of DNA replication origins at A+T-rich islands in Schizosaccharomyces pombe. Segurado M, De Luis A, Antequera F. EMBO Rep. 2003 Nov;4(11):1048-53. Epub 2003 Oct 17. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14566325">PMID:14566325</a><br />
<br />
Retrotransposons and their recognition of pol II promoters: a comprehensive survey… Bowen NJ et al Genome Res. 2003 Sep;13(9):1984-97. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12952871">PMID:12952871</a></p>
</div>
<div id="2003-08-31-new-fission-yeast-book-now-published" class="news-item">
<h3 id="the-new-fission-yeast-book-now-published">The ‘new’ fission yeast book is now published</h3>
<p><em>2003-08-31</em></p>
<p>Egel, R., Copenhagen, Denmark (Ed.) The Molecular Biology of Schizosaccharomyces pombe Genetics, Genomics and Beyond ISBN:3-540-00693-1</p>
</div>
<div id="2003-02-28-schizosaccharomyces-pombe-essential-genes-pilot-study" class="news-item">
<h3 id="schizosaccharomyces-pombe-essential-genes-pilot-study">Schizosaccharomyces pombe Essential Genes: A pilot Study</h3>
<p><em>2003-02-28</em></p>
<p>Decottignies A, Sanchez-Perez I, Nurse P Genome Res. 2003 Mar;13(3):399-406. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12618370">PMID:12618370</a></p>
</div>
<div id="2002-12-31-global-transcriptional-responses-fission-yeast-environmental-stress" class="news-item">
<h3 id="global-transcriptional-responses-fission-yeast-environmental-stress">Global transcriptional responses of fission yeast to environmental stress</h3>
<p><em>2002-12-31</em></p>
<p>Chen D, Toone WM, Mata J, Lyne R, Burns G, Kivinen K, Brazma A, Jones N, Bähler J. Mol Biol Cell. 2003 Jan;14(1):214-29. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12529438">PMID:12529438</a></p>
</div>
</div>

  </div>
  </div>
</div>
<div *ngIf="section == 'status'">
<app-page-contents-menu title="Status" titleRoute="/status">

<div class="left-menu-part left-menu-item"><a routerLink="/status/genome-overview">Statistics</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/status/sequencing-status">Sequencing status</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/status/centromeres">Centromeres</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/status/mating-type-region">Mating type region</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/status/telomeres">Telomeres</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/status/sequencing-updates">Sequence updates</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/status/sequence-updates-pending">Sequence updates pending</a></div>
<div class="left-menu-part left-menu-item"><span>Gene coordinate changes</span></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/status/gene-coordinate-changes-protein-coding">Protein coding genes</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/status/gene-coordinate-changes-RNA">RNA genes</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/status/new-and-removed-genes">New and removed genes</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/status/gene-characterisation">Gene characterisation</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/status/priority-unstudied-genes">Priority unstudied genes</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/status/unmapped_genes">Unmapped genes</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'gene-characterisation'">
<h3 id="protein-coding-gene-characterisation-status">Protein-coding gene characterisation status</h3>
<p>Unknown proteins have become an increasing concern, as rates of gene characterisation have plateaued and a substantial proportion of proteins remain unstudied or understudied, even in the most intensively investigated model organisms. To monitor this issue, PomBase maintains an inventory of unknown proteins, originally described in <a href="https://doi.org/10.1098/rsob.180241">Wood <em>et al.</em> 2019</a>.</p>
There are several ways to determine whether a gene is functionally
characterised, and characterisation is best considered as a
continuum. As a pragmatic first step, however, we applied a simple
binary classification of "known" versus "unknown", based on whether
the gene has a biological process annotation from the fission yeast GO
slim set (i.e., whether its broad cellular role has been defined). A
detailed justification for this process-focused approach is provided
in <a href="https://doi.org/10.1093/genetics/iyae007">Rutherford <i>et al.</i> 2024</a>.
It should be noted that
<a (click)="gotoPredefinedResults('canned_query:coding_genes_with_mf_annotation')">
{{getPredefinedQueryCount('canned_query:coding_genes_with_mf_annotation') | async}}
</a>
of the
{{getPredefinedQueryCount('canned_query:coding_genes_with_bp_annotation') | async}}
proteins with a GO-slim biological process also have an assigned GO
molecular function.
<p>
<p>We regularly review genes in the “unknown” category to determine whether new functional data are available from other species. A gene is classified as unknown when no broad biological role can be inferred from experimental data in S. pombe or from an experimentally characterised ortholog in any other organism. Importantly, we require robust supporting evidence to assign a GO biological process annotation. Finally, because every S. pombe gene has been reviewed for GO biological process, these genes are “known unknowns” rather than “unannotated”.</p>
<h4 id="related-gene-lists">Related gene lists</h4>
<table>
<thead>
  <tr>
    <th>List</th>
    <th>Description</th>
  </tr>
</thead>
<tr>
  <td style="white-space: nowrap;"><a (click)="gotoPredefinedResults('canned_query:protein_coding_genes_unknown_process')">
Unknown genes ({{getPredefinedQueryCount('canned_query:protein_coding_genes_unknown_process') | async}})
  </a>
  </td>
  <td>Protein coding genes, unknown process</td>
</tr>
<tr>
  <td style="white-space: nowrap;"><a (click)="gotoPredefinedResults('priority-unstudied-genes:conserved_unknown')">
Conserved unknown ({{getPredefinedQueryCount('priority-unstudied-genes:conserved_unknown') | async}})
  </a>
  </td>
  <td>
    Genes of unknown process, conserved outside the Schizosaccharomyces clade
  </td>
</tr>
<tr>
  <td style="white-space: nowrap;"><a (click)="gotoPredefinedResults('priority-unstudied-genes:conserved_unknown_AND_conserved_in_vertebrates')">
Priority unstudied genes ({{getPredefinedQueryCount('priority-unstudied-genes:conserved_unknown_AND_conserved_in_vertebrates') | async}})
  </a>
  </td>
  <td>
    Priority unknowns are the subset conserved to vertebrates (all
    present in human), considered over at least ~1 billion years of
    evolution. Some of these are universally conserved in bacteria and
    archaea
  </td>
</tr>
<tr>
  <td style="white-space: nowrap;"><a routerLink="/gocam/missing-activities">Missing activities ({{getMissingActivityCount() | async}})</a></td>
  <td>
    A list of activities known or suspected to occur in fission
    yeast but not yet associated with a gene product. These "pathway
    holes" have been identified through GO-CAM causal model curation.
  </td>
</tr>
</table>
<h4 id="changes-over-time">Changes over time</h4>
<p><a href="assets/gene_characterisation_status_figure.svg"><img src="assets/gene_characterisation_status_figure.svg" class="screenshot" alt="Gene characterisation status" /></a></p>
<h4 id="current-status">Current status</h4>
<p><app-characterisation-status-table></app-characterisation-status-table></p>
<h4 id="protein-coding-gene-characterisation-status-descriptions">Protein-coding gene characterisation status descriptions</h4>
<p><strong>Biological role published:</strong> Completely or partially characterised in a small scale experiment, with some published information about the biological role (corresponding to any of the <a href="browse-curation/fission-yeast-bp-go-slim-terms">fission yeast GO biological process slim</a> terms)</p>
<p><strong>Biological role inferred:</strong> A biological role (as above, a <a href="browse-curation/fission-yeast-bp-go-slim-terms">fission yeast GO Process slim</a> term) is inferred from homology to an experimentally characterised gene product</p>
<p><strong>Conserved protein (unknown biological role):</strong> Conserved outside the <i>Schizosaccharomyces</i>, but nothing known about the biological role in any organism</p>
<p><strong><i>Schizosaccharomyces</i> specific protein, uncharacterized:</strong> Unpublished and found only in fission yeast (<i>S. pombe</i>, <i>S. octosporus</i>, <i>S. japonicus</i>, <i>S. cryophilus</i>); nothing known about biological role. May be single copy or a member of a multi-member family.</p>
<p><strong><i>S. pombe</i> specific protein, uncharacterized:</strong> Unpublished and found only in <i>S. pombe</i> (not detected in other <i>Schizosaccharomyces</i> species); nothing known about biological role</p>
<p><strong>Dubious:</strong> Unlikely to be protein coding</p>
<p><strong>Transposon:</strong> A predicted or experimentally verified transposable element.</p>
<p>Note: You can retrieve current lists of genes with each characterisation status using the <a routerLink="/query">advanced search</a>. Select the Characterisation status query, then choose a status from the pulldown menu, and submit.</p>

  </div>
  <div *ngIf="pageName == 'gene-characterisation-statistics-history'">
<h3 id="protein-coding-gene-characterisation-statistics-history">Protein-coding gene characterisation statistics history</h3>
<h4 id="history-of-characterisation-status-using-current-status-descriptions">History of characterisation status using current status descriptions</h4>
<h5 id="introduced-with-pombase-chado-release-46-august-2014">(introduced with PomBase Chado release 46, August 2014):</h5>
<table>
 <thead>
  <tr>
   <th style="width: 100px;">Date</th>
   <th style="width: 100px;">Published</th>
   <th style="width: 100px;">Role inferred</th>
   <th style="width: 100px;">Conserved unknown</th>
   <th style="width: 100px;"><em><em><strong>Schizo.</strong></em></em></th>
   <th style="width: 100px;"><em>S. pombe</em></th>
   <th style="width: 100px;">Dubious</th>
   <th style="width: 100px;">Transposon</th>
   <th style="width: 100px;">Total</th>
  </tr>
 </thead>
 <tbody>
  <tr>
   <td>2022-12-31</td>
   <td>2477</td>
   <td>1931</td>
   <td>385</td>
   <td>155</td>
   <td>103</td>
   <td>69</td>
   <td>13</td>
   <td>5133</td>
  </tr>
  <tr>
   <td>2021-08-31</td>
   <td>2441</td>
   <td>1953</td>
   <td>390</td>
   <td>164</td>
   <td>117</td>
   <td>55</td>
   <td>13</td>
   <td>5133</td>
  </tr>
 <tr>
  <td>2020-09-11</td>
  <td>2389</td>
  <td>1985</td>
  <td>408</td>
  <td>162</td>
  <td>121</td>
  <td>55</td>
  <td>13</td>
  <td>5133</td>
 </tr>
 <tr>
  <td>2019-12-31</td>
  <td>2368</td>
  <td>2010</td>
  <td>406</td>
  <td>162</td>
  <td>122</td>
  <td>55</td>
  <td>13</td>
  <td>5136</td>
 </tr>
 <tr>
  <td>2018-10-05</td>
  <td>2339</td>
  <td>2035</td>
  <td>410</td>
  <td>163</td>
  <td>122</td>
  <td>55</td>
  <td>13</td>
  <td>5137</td>
 </tr>
 <tr>
  <td>2017-01-30</td>
  <td>2235</td>
  <td>1996</td>
  <td>511</td>
  <td>180</td>
  <td>145</td>
  <td>54</td>
  <td>13</td>
  <td>5171</td>
 </tr>
 <tr>
  <td>2016-09-12</td>
  <td>2227</td>
  <td>2034</td>
  <td>503</td>
  <td>190</td>
  <td>149</td>
  <td>55</td>
  <td>13</td>
  <td>5171</td>
 </tr>
 <tr>
  <td>2015-01-19</td>
  <td>2154</td>
  <td>2050</td>
  <td>529</td>
  <td>178</td>
  <td>143</td>
  <td>76</td>
  <td>13</td>
  <td>5143</td>
 </tr>
 <tr>
  <td>2014-09-17</td>
  <td>2124</td>
  <td>2066</td>
  <td>537</td>
  <td>183</td>
  <td>143</td>
  <td>77</td>
  <td>&nbsp;13</td>
  <td>5143</td>
 </tr>
 </tbody>
</table>
<h4 id="history-of-characterisation-status-using-previous-status-descriptions">History of characterisation status using previous status descriptions:</h4>
<table>
 <thead>
  <tr>
   <th style="width: 100px;">Date</th>
   <th style="width: 100px;">Published</th>
   <th style="width: 100px;">Role inferred</th>
   <th style="width: 100px;">Conserved unknown</th>
   <th style="width: 100px;"><em>S. pombe</em></th>
   <th style="width: 100px;">Orphan</th>
   <th style="width: 100px;">Dubious</th>
   <th style="width: 100px;">Transposon</th>
   <th style="width: 100px;">Total</th>
  </tr>
 </thead>
 <tbody>
 <tr>
  <td>2006-08-25</td>
  <td>1560</td>
  <td>2433</td>
  <td>458</td>
  <td>68</td>
  <td>403</td>
  <td>57</td>
  <td>&nbsp;</td>
  <td>4979</td>
 </tr>
 <tr>
  <td>2007-02-27</td>
  <td>1607</td>
  <td>2329</td>
  <td>572</td>
  <td>47</td>
  <td>364</td>
  <td>60</td>
  <td>&nbsp;</td>
  <td>4979</td>
 </tr>
 <tr>
  <td>2008-06-05</td>
  <td>1715</td>
  <td>2210</td>
  <td>634</td>
  <td>42</td>
  <td>344</td>
  <td>57</td>
  <td>&nbsp;</td>
  <td>5002</td>
 </tr>
 <tr>
  <td>2009-01-14</td>
  <td>1812</td>
  <td>2132</td>
  <td>636</td>
  <td>56</td>
  <td>318</td>
  <td>57</td>
  <td>&nbsp;</td>
  <td>5011</td>
 </tr>
 <tr>
  <td>2010-02-22</td>
  <td>1848</td>
  <td>2160</td>
  <td>614</td>
  <td>55</td>
  <td>286</td>
  <td>57</td>
  <td>&nbsp;</td>
  <td>5020</td>
 </tr>
 <tr>
  <td>2010-06-30</td>
  <td>1916</td>
  <td>2163</td>
  <td>592</td>
  <td>54</td>
  <td>285</td>
  <td>58</td>
  <td>11</td>
  <td>5025</td>
 </tr>
 <tr>
  <td>2011-04-27</td>
  <td>1916</td>
  <td>2169</td>
  <td>588</td>
  <td>64</td>
  <td>315</td>
  <td>79</td>
  <td>11</td>
  <td>5142</td>
 </tr>
 <tr>
  <td>2011-12-16</td>
  <td>1936</td>
  <td>2165</td>
  <td>582</td>
  <td>66</td>
  <td>309</td>
  <td>73</td>
  <td>11</td>
  <td>5142</td>
 </tr>
 <tr>
  <td>2013-05-23</td>
  <td>2012</td>
  <td>2137</td>
  <td>541</td>
  <td>66</td>
  <td>303</td>
  <td>71</td>
  <td>13</td>
  <td>5143</td>
 </tr>
 <tr>
  <td>2013-12-06</td>
  <td>2083</td>
  <td>2096</td>
  <td>522</td>
  <td>64</td>
  <td>294</td>
  <td>71</td>
  <td>13</td>
  <td>5143</td>
 </tr>
 <tr>
  <td>2014-07-14</td>
  <td>2110</td>
  <td>2076</td>
  <td>517</td>
  <td>64</td>
  <td>292</td>
  <td>71</td>
  <td>13</td>
  <td>5143</td>
 </tr>
 </tbody>
</table>
<h3 id="status-descriptions">Status Descriptions</h3>
<table>
<thead>
 <tr>
  <th>Shorthand</th>
  <th>Full name</th>
  <th>Description</th>
  <th>Comment</th>
 </tr>
</thead>
<tbody>
 <tr>
  <td>Published</td>
  <td>Experimentally characterised (or published)</td>
  <td>Completely or partially characterised in a small scale experiment, with some published information about the biological role (corresponding to any of the fission yeast GO biological process slim terms)</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>Role inferred</td>
  <td>Role inferred from homology</td>
  <td>A biological role (as above, a fission yeast GO process slim term) is inferred from homology to an experimentally characterised gene product</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>Conserved unknown</td>
  <td>Conserved protein (unknown biological role)</td>
  <td>Conserved outside the <em>Schizosaccharomyces</em>, but nothing known about the biological role in any organism</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><em>Schizo.</em></td>
  <td><em><em>Schizosaccharomyces</em>&nbsp;</em>specific protein, uncharacterised</td>
  <td>Unpublished and found only in fission yeast (<em>S. pombe</em>,&nbsp;<em>S. octosporus,&nbsp;<em>S. japonicus, S. cryophilus</em></em><em></em>); nothing known about biological role. May be single copy or a member of a multi-member family.</td>
  <td>Introduced Sept. 2014</td>
 </tr>
 <tr>
  <td><em>S. pombe</em></td>
  <td><em><em>S. pombe</em>&nbsp;</em>specific protein, uncharacterised</td>
  <td>Unpublished and found only in&nbsp;<em>S. pombe&nbsp;</em>(not detected in other&nbsp;<em>Schizosaccharomyces</em>&nbsp;species); nothing known about biological role</td>
  <td>Introduced Sept. 2014&nbsp;</td>
 </tr>
 <tr>
  <td>Transposon</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>Dubious</td>
  <td>&nbsp;</td>
  <td>Unlikely to be protein coding</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><em>S. pombe</em></td>
  <td>&nbsp;<em>S. pombe</em>&nbsp;specific families</td>
  <td>Unpublished and found only in fission yeast (<em>S. pombe</em>,&nbsp;<em>S. octosporus,&nbsp;</em><em>S. japonicus, S. cryophilus</em>); nothing known about biological role, but are not single copy (duplications in fission yeast)</td>
  <td>&nbsp;Used Aug. 2006-Aug. 2014</td>
 </tr>
 <tr>
  <td>&nbsp;Orphan</td>
  <td>Sequence orphan, uncharacterised</td>
  <td>Unpublished and found only in fission yeast (<em>S. pombe</em>,&nbsp;<em>S. octosporus</em>,&nbsp;<em>S. japonicus, S. cryophilus</em>); nothing known about biological role</td>
  <td>Used Aug. 2006-Aug. 2014</td>
 </tr>
 </tbody>
 </table>

  </div>
  <div *ngIf="pageName == 'index'">
<h2 id="genome-status">Genome Status</h2>
<table>
<tbody>
<tr class="odd">
<td><a routerLink="/status/genome-overview">Genome overview</a></td>
<td>Graphic and basic information about each chromosome</td>
</tr>
<tr class="even">
<td><a routerLink="/status/sequencing-status">Sequencing status</a></td>
<td>Contig size, gap status and progress, including <a routerLink="/status/centromeres">centromeres</a> and <a routerLink="/status/telomeres">telomeres</a></td>
</tr>
<tr class="odd">
<td><a routerLink="/status/sequencing-updates">Sequence updates</a></td>
<td>Changes to the genome sequence since July 2003</td>
</tr>
<tr class="even">
<td><a routerLink="/status/sequence-updates-pending">Sequence updates pending</a></td>
<td>Pending changes, mainly from the Broad Institute, some supported by data from other sources</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/gene-coordinate-changes-protein-coding">Protein coding gene coordinate changes</a></td>
<td>Changes to coordinates of individual protein coding genes since publication</td>
</tr>
<tr class="even">
<td><a routerLink="/status/gene-coordinate-changes-RNA">RNA gene coordinate changes</a></td>
<td>Changes to coordinates of RNA genes since publication</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/new-and-removed-genes">New and removed genes</a></td>
<td>Genes identified or removed since publication</td>
</tr>
<tr class="even">
<td><a routerLink="/status/gene-characterisation">Gene characterisation</a></td>
<td>Current counts of protein coding gene status, as published in small scale experiments</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/statistics">Genome statistics</a></td>
<td>Information on the status of the genome (Note: Last updated January 2017)</td>
</tr>
<tr class="even">
<td><a routerLink="/status/priority-unstudied-genes">Priority unstudied genes</a></td>
<td>Unstudied nuclear-encoded protein-coding genes conserved 1:1 in human</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/unmapped_genes">Unmapped genes</a></td>
<td>List of genes identified genetically but not cloned or physically mapped</td>
</tr>
</tbody>
</table>
<p>Note: Many older <em>S. pombe</em> sequence submissions to the DNA databases (International Nucleotide Sequence Database Collaboration databases, i.e. ENA, GenBank, DDBJ) contain one or more errors, and we do not have the resources to maintain past sequences or flag every error in PomBase.</p>

  </div>
  <div *ngIf="pageName == 'priority-unstudied-genes'">
<h3 id="priority-unstudied-genes">Priority unstudied genes</h3>
<p>Fission yeast genes are classed as “unknown” if there is no information about the broad cellular role (biological process) in which it participates (corresponding to any of the <a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">high level GO biological process slim classes</a>). For these, we have been unable to identify a broad biological role based on experimental data in <em>S. pombe</em> or any other organism. Note that, as all genes in <em>S. pombe</em> have been curated, these genes are documented as “unknown” (they are not “unannotated”).</p>
<p>In <em>S. pombe</em> the “unknown” inventory is <app-query-link [goToResults]="true" [predefinedQueryId]="'priority-unstudied-genes:unknown_genes'"></app-query-link> entries, many of which are apparently found only in the fission yeast clade. However, a large number (<app-query-link [goToResults]="true" [predefinedQueryId]="'priority-unstudied-genes:conserved_unknown'"></app-query-link>) are conserved, and a significant number of these (<app-query-link [goToResults]="true" [predefinedQueryId]="'priority-unstudied-genes:conserved_unknown_AND_conserved_in_vertebrates'"></app-query-link>) have orthologs in vertebrates. These genes are listed below.</p>
<p>You can recreate this query, or variations of it, using the Advanced Search. See the <a routerLink="/faq/how-can-i-find-all-s.-pombe-genes-are-conserved-human">relevant FAQ</a> and the <a routerLink="/query">Advanced Search</a> documentation for details. You can also send this list directly to the advanced search (For example to mine for phenotypes or locations of interest).</p>
<p><a routerLink="/status/gene-characterisation">Our protein-coding gene characterisation status page</a> shows changes in the number of unknowns over time.</p>
<p><app-predefined-query-results [description]="'Unstudied genes'" [predefinedQueryId]="'priority-unstudied-genes:conserved_unknown_AND_conserved_in_vertebrates'"></app-predefined-query-results></p>

  </div>
  <div *ngIf="pageName == 'sequencing-status'">
<h3 id="sequencing-status">Sequencing status</h3>
<p>Note: Many older <em>S. pombe</em> sequence submissions to the DNA databases (International Nucleotide Sequence Database Collaboration databases, i.e. ENA, GenBank, DDBJ) contain one or more errors, and we do not have the resources to maintain past sequences or flag every error in PomBase.</p>
<h4 id="chromosome-1">Chromosome 1</h4>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">Contig Name</th>
<th style="text-align: left;">Region</th>
<th style="text-align: left;">Size</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;">unsequenced to chr1 left telomere</td>
<td style="text-align: left;">10 ± 2 kb*</td>
</tr>
<tr class="even">
<td style="text-align: left;">c212</td>
<td style="text-align: left;">sub-telomeric left arm</td>
<td style="text-align: left;">29,663 bp</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;">Gap</td>
<td style="text-align: left;">&lt;5 kb*</td>
</tr>
<tr class="even">
<td style="text-align: left;">c977</td>
<td style="text-align: left;">left arm and right arm</td>
<td style="text-align: left;">5,549,370 bp</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;">unsequenced to chr1 left telomere</td>
<td style="text-align: left;">18 ± 3 kb*</td>
</tr>
</tbody>
</table>
<h4 id="chromosome-2">Chromosome 2</h4>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">Contig Name</th>
<th style="text-align: left;">Region</th>
<th style="text-align: left;">Size</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">AB325691</td>
<td style="text-align: left;">chr2 left arm gap-filling contig</td>
<td style="text-align: left;">20,000 bp**</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;">Gap</td>
<td style="text-align: left;">5 ± 5 kb</td>
</tr>
<tr class="odd">
<td style="text-align: left;">c1348</td>
<td style="text-align: left;">sub-telomeric left arm</td>
<td style="text-align: left;">80,201 bp</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;">Gap</td>
<td style="text-align: left;">22 ± 5 kb*</td>
</tr>
<tr class="odd">
<td style="text-align: left;">pB10D8</td>
<td style="text-align: left;">left arm to centromeric gap</td>
<td style="text-align: left;">1,536,269 bp</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;">Gap</td>
<td style="text-align: left;">~6 kb</td>
</tr>
<tr class="odd">
<td style="text-align: left;">pJ5566</td>
<td style="text-align: left;">right arm from centromeric gap to telomeric repeats</td>
<td style="text-align: left;">2,923,134 bp</td>
</tr>
</tbody>
</table>
<h4 id="chromosome-3">Chromosome 3</h4>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">Contig Name</th>
<th style="text-align: left;">Region</th>
<th style="text-align: left;">Size</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">p20C8</td>
<td style="text-align: left;">left arm from centromeric gap</td>
<td style="text-align: left;">1,083,348 bp</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;">Gap</td>
<td style="text-align: left;">25.3 ± 6 kb***</td>
</tr>
<tr class="odd">
<td style="text-align: left;">c1676</td>
<td style="text-align: left;">right arm from centromeric gap</td>
<td style="text-align: left;">1,369,435 bp</td>
</tr>
</tbody>
</table>
<p>* pers comm. Randy Hyppa and Gerry Smith.</p>
<p>** Sasaki <em>et al.</em> (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18727152">PMID:18727152</a>)</p>
<p>*** pers comm. Chad Ellermeier and Gerry Smith.</p>
<h4 id="centromere-clone-information"><a routerLink="/status/centromeres">Centromere clone information</a></h4>
<h4 id="telomere-clone-information"><a routerLink="/status/telomeres">Telomere clone information</a></h4>
<h4 id="mating-type-region-information"><a routerLink="/status/mating-type-region">Mating type region information</a></h4>
<h4 id="ribosomal-dna-information-faq"><a routerLink="/faq/there-any-rdna-repeat-sequences-pombase">Ribosomal DNA information (FAQ)</a></h4>

  </div>
  </div>
</div>
<div *ngIf="section == 'submit-data'">
<app-page-contents-menu title="Submit data" titleRoute="/submit-data">

<div class="left-menu-part left-menu-item"><a routerLink="/submit-data/gene-names">Gene name registration</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/submit-data/gene-naming-guidelines"><i>S. pombe</i> gene naming guidelines</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/submit-data/gene-naming-committee-members">Gene Naming Committee members</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/submit-data/bulk-annotation">Annotation datasets</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/data-submission-form-for-HTP-sequence-linked-data">Sequence-linked HTP data</a></div>


</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'index'">
<h2 id="pombase-data-submission">PomBase Data Submission</h2>
<h3 id="gene-names"><a routerLink="/submit-data/gene-names">Gene names</a></h3>
<p>Gene naming guidelines and how to submit a new gene name</p>
<h3 id="annotation-datasets"><a routerLink="/submit-data/bulk-annotation">Annotation datasets</a></h3>
<p>Data formats and instructions for submitting large sets of GO, phenotype, modification, or gene expression data.</p>
<h3 id="sequence-linked-htp-data"><a routerLink="/documentation/data-submission-form-for-HTP-sequence-linked-data">Sequence-linked HTP data</a></h3>
<p>Metadata format and instructions for submitting sequence-linked data to be displayed as a track in the genome browser.</p>
<h3 id="gene-specific-data">Gene-specific data</h3>
<p>To submit information on individual genes, use the Contact Curators link provided on the relevant <a href="https://www.pombase.org">PomBase</a> gene page</p>

  </div>
  </div>
</div>
</div>
