<div class="docs">
<app-social-contact></app-social-contact>
<div *ngIf="section == 'about'">
<app-page-contents-menu title="About" titleRoute="/about">

<div class="left-menu-part left-menu-item"><a routerLink="/about/citing-pombase">Citing PomBase</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/about/published-by-pombase">PomBase Publications</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/about/terms-of-use">Terms of Use</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/about/pombase-sab">SAB</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/about/privacy-policy">Privacy Policy</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/help">Help</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/about/resource-metrics">Resource metrics</a></div>


</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'citing-pombase'">
<h2 id="citing-pombase">Citing PomBase</h2>
<p>Also see the <a routerLink="/about/terms-of-use">Terms of Use</a> and the complete list of <a routerLink="/about/published-by-pombase">papers published by PomBase</a>.</p>
<p>To cite anything from PomBase, including</p>
<ul>
<li><em>S. pombe</em> genome sequence and feature annotation</li>
<li>Curated data (GO, phenotypes, modifications, gene expression, etc.)</li>
<li>GO or disease slim sets</li>
<li>Tools (advanced search, motif search, QuiLT, etc.)</li>
</ul>
<p>please use:</p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gky961">

</div>
<div class="pub-citation">
<p>Lock A, Rutherford K, Harris MA, Hayles J, Oliver SG, Bähler J; Wood V.<br />
PomBase 2018: user-driven reimplementation of the fission yeast database provides rapid and intuitive access to diverse, interconnected information.<br />
<em>Nucleic Acids Res.</em> 2018 (Database issue): gky961 Epub 2018 Oct 13.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/30321395">PMID:30321395</a> - <a href="https://doi.org/10.1093/nar/gky961">DOI:10.1093/nar/gky961</a></p>
<h3 id="genome-sequence">Genome Sequence</h3>
<p>To cite the <em>S. pombe</em> genome sequence, use:</p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1038/nature724">

</div>
<div class="pub-citation">
<p>Wood V, Gwilliam R, Rajandream MA, Lyne M, Lyne R, Stewart A, Sgouros J, Peat N, Hayles J, Baker S, <em>et al.</em><br />
The genome sequence of <em>Schizosaccharomyces pombe</em>.<br />
<em>Nature</em>. 2002 Feb 21;415(6874):871-80. Erratum in: Nature 2003 Jan 2;421(6918):94. Cerrutti L [corrected to Cerutti L].</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/11859360">PMID:11859360</a> - <a href="http://www.nature.com/nature/journal/v415/n6874/full/nature724.html">Full text at Nature</a></p>
<h3 id="go-annotations">GO Annotations</h3>
<p>To cite the <em>S. pombe</em> Gene Ontology annotation data, use two papers from the 2018 NAR Database Issue:</p>
<ol type="1">
<li>The PomBase paper noted above</li>
<li>The GO Consortium paper:</li>
</ol>
<p>The Gene Ontology Consortium.<br />
The Gene Ontology Resource: 20 years and still GOing strong.<br />
<em>Nucleic Acids Res.</em> 2018 (Database issue): gky1055 Epub 2018 Nov 5.<br />
</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/30395331">PMID:30395331</a> - <a href="https://doi.org/10.1093/nar/gky1055">DOI:10.1093/nar/gky1055</a></p>
<p>(Note that GO annotation for <em>S. pombe</em> has also been previously published, but we now recommend the 2018 NAR paper as it is more up-to-date. The older publication is Aslett M, Wood V. Gene Ontology annotation status of the fission yeast genome: preliminary coverage approaches 100%. <em>Yeast</em> 2006 Oct 15;23(13):913-9. Erratum in <em>Yeast</em> 2007 May;24(5):457. PMID:17072883)</p>
<h3 id="phenotype-ontology-and-annotations">Phenotype Ontology and Annotations</h3>
<p>To cite the Fission Yeast Phenotype Ontology (FYPO) and/or <em>S. pombe</em> FYPO annotation data, use:</p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/bioinformatics/btt266">

</div>
<div class="pub-citation">
<p>Harris MA, Lock A, Bähler J, Oliver SG, Wood V.<br />
FYPO: The Fission Yeast Phenotype Ontology.<br />
<em>Bioinformatics</em>. 2013 Jul 1;29(13):1671-8. Epub 2013 May 8.</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/23658422">PMID:23658422</a> - <a href="http://dx.doi.org/10.1093/bioinformatics/btt266">DOI:10.1093/bioinformatics/btt266</a>.</p>
<p>Full text at Bioinformatics: <a href="http://bioinformatics.oxfordjournals.org/content/29/13/1671.long">HTML</a> or <a href="http://bioinformatics.oxfordjournals.org/content/29/13/1671.full.pdf+html">PDF</a></p>
<h3 id="canto">Canto</h3>
<p>To cite the online curaton tool Canto, use:</p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/bioinformatics/btu103">

</div>
<div class="pub-citation">
<p>Rutherford KM, Harris MA, Lock A, Oliver SG, Wood V.<br />
Canto: An online tool for community literature curation.<br />
<em>Bioinformatics</em>. 2014 Jun 15;30(12):1791-2. Epub 2014 Feb 25.</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/24574118">PMID:24574118</a> - <a href="http://dx.doi.org/10.1093/bioinformatics/btu103">DOI: 10.1093/bioinformatics/btu103</a></p>
<h3 id="manually-curated-orthologs">Manually Curated Orthologs</h3>
<p>To cite the manually curated <em>S. cerevisiae</em> or human orthologs (displayed on gene pages or downloaded), use the NAR database paper as noted above. Note that the manually curated orthologs use several data sources including Compara, but are distinct from the Compara results themselves.</p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gkr853">

</div>
<div class="pub-citation">
<p>Wood V, Harris MA, McDowall MD, Rutherford K, Vaughan BW, Staines DM, Aslett M, Lock A, Bähler J, Kersey PJ, Oliver SG.<br />
PomBase: a comprehensive online resource for fission yeast.<br />
<em>Nucleic Acids Res.</em> 2012 Jan;40(Database issue):D695-9. Epub 2011 Oct 28.</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153">PMID:22039153</a> - <a href="http://dx.doi.org/10.1093/nar/gkr853">DOI: 10.1093/nar/gkr853</a> - <a href="http://nar.oxfordjournals.org/content/40/D1/D695.long">Full text at NAR</a></p>
<p><em>Altmetric badges link to data provided by <a href="https://www.altmetric.com/">Altmetric.com</a>, a research metrics company who track and collect the online conversations around millions of scholarly outputs. Read <a href="https://www.altmetric.com/about-altmetrics/">more about Altmetrics</a>.</em></p>

  </div>
  <div *ngIf="pageName == 'cookie-policy'">
<h2 id="pombase-cookie-policy">PomBase Cookie Policy</h2>
<h3 id="what-are-cookies">What are cookies?</h3>
<p>We will not use cookies to collect personally identifiable information about you.</p>
<p>A cookie is a file that is stored on your computer when you visit a website. Most websites use them and they are generally harmless. When you revisit the website later or visit a different webpage a copy of the cookie file is sent to the website.</p>
<p>Cookies can be used to store information and have many uses. With the most common being tracking, remembering your details or settings, and to keep you logged in to an account.</p>
<p>You may refuse to accept cookies by activating the setting on your browser which allows you to refuse the setting of cookies. However, if you select this setting you may be unable to access certain parts of our site. Unless you have adjusted your browser setting so that it will refuse cookies, our system will issue cookies when you visit our site.</p>
<p>You can find out more information about cookies from the following websites:</p>
<ul>
<li><a href="https://en.wikipedia.org/wiki/HTTP_cookie">Wikipedia</a></li>
<li><a href="https://support.mozilla.org/en-US/kb/delete-cookies-remove-info-websites-stored">Mozilla</a></li>
<li><a href="https://policies.google.com/technologies/cookies?hl=en&gl=uk">Google</a></li>
</ul>
<h3 id="how-we-use-cookies">How we use cookies</h3>
<p>Like most interactive websites our site uses cookies to enable us to retrieve user details for each visit. These pieces of information are used to improve services for you, for example, by:</p>
<ul>
<li><p>measuring how many people are using our website, so they can be made easier to use and to ensure there is enough capacity</p></li>
<li><p>analysing anonymised data to help us understand how people interact with our website so we can make it better.</p></li>
</ul>
<h3 id="the-cookies-we-use">The cookies we use</h3>
<p>This website contains links to and from third-party sites. This policy only covers use of cookies on this site. You should consult privacy and cookie policies on third-party sites before you submit any personal data. PomBase is not responsible for any cookies that may be set by third-party websites that are linked to from pages on www.pombase.org.</p>
<p>Table of cookies we use on our site:</p>
<table>
<thead>
<tr class="header">
<th>Name</th>
<th>Description</th>
<th>More information</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Canto_pombe__session</td>
<td>Cookie set by Canto</td>
<td><a routerLink="/about/privacy-policy">PomBase privacy policy</a></td>
</tr>
<tr class="even">
<td>_ga<br>_gali<br>_gid<br>_gat<br>_gat_jbrowseTracker<br>__utma<br>__utmb<br>__utmc<br>__utmt<br>__utmz</td>
<td>Cookies used by Google Analytics, which we use to measure website usage</td>
<td><a href="http://www.google.com/policies/privacy/">Google’s privacy policy</a> and <a href="https://developers.google.com/analytics/devguides/collection/analyticsjs/cookie-usage">Google Analytics Cookie descriptions</a></td>
</tr>
</tbody>
</table>
<h4 id="strictly-necessary-cookies">Strictly Necessary Cookies</h4>
<p>These cookies are essential to enable you to move around the website and use its features, such as accessing secure areas of the website. Without these cookies, our website won’t work properly.</p>
<h4 id="performance-cookies">Performance Cookies</h4>
<p>These cookies collect information about how you use our website, for instance, which pages you visit most often and if you get any error messages from those pages. Google Analytics sets cookies to help us accurately estimate the number of visitors to the website and volumes of usage. This is to ensure that we provide a fast service that is available when you want it.</p>
<h4 id="social-media-cookies">Social media cookies</h4>
<p>PomBase has accounts or pages with external resources including Twitter (<span class="citation" data-cites="PomBase">@PomBase</span>), Facebook (www.facebook.com/pombase), LinkedIn (www.linkedin.com/groups/5122686), and GithHub (github.com/pombase). All use cookie technology, and maintain their own privacy policies.</p>
<h4 id="what-if-i-see-an-unlisted-cookie">What if I see an unlisted cookie?</h4>
<p>If you notice a cookie that slips past us or have a question about cookies then please contact us.</p>
<h3 id="how-can-i-disable-or-block-cookies">How can I disable or block cookies?</h3>
<p>You may refuse to accept cookies by activating the setting on your browser which allows you to refuse the setting of cookies. However, if you select this setting you may be unable to access certain parts of our site. Unless you have adjusted your browser setting so that it will refuse cookies, our system will issue cookies when you visit our site. We recommend visiting your browser help section to find out more about managing cookies.</p>
<ul>
<li><a href="http://support.mozilla.org/en-US/kb/Enabling%20and%20disabling%20cookies">Firefox</a></li>
<li><a href="http://windows.microsoft.com/en-US/internet-explorer/help">Internet Explorer</a></li>
<li><a href="https://support.google.com/chrome/bin/answer.py?hl=en&answer=95647&p=cpn_cookies">Google Chrome</a></li>
<li><a href="https://support.apple.com/en-gb/guide/safari/manage-cookies-and-website-data-sfri11471/mac">Safari</a></li>
<li><a href="http://www.aboutcookies.org/Default.aspx?page=2">Other web browsers</a></li>
</ul>
<h3 id="do-not-track-me">Do not track me!</h3>
<p>Do not track is a relatively new feature in web browsers which allows you to tell websites that you do not want to be tracked. While not all websites currently use this, you can find out more how to set it up on your browser from the following websites:</p>
<ul>
<li><a href="https://support.mozilla.org/en-US/kb/how-do-i-turn-do-not-track-feature">Firefox</a></li>
<li><a href="http://windows.microsoft.com/en-US/internet-explorer/help">Internet Explorer</a></li>
<li><a href="https://support.google.com/chrome/answer/114836?hl=en&ref_topic=7437824">Google Chrome</a></li>
<li><a href="https://support.apple.com/en-gb/guide/safari/prevent-tracking-sfri40732/mac">Safari</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'data-sources'">
<h2 id="pombase-data-sources">PomBase Data Sources</h2>
<p>Also see the <a routerLink="/about/terms-of-use">Terms of Use</a>.</p>
<h3 id="research-spotlight-images">Research spotlight images</h3>
<p>For images used on the home page, see the <a routerLink="/archive/spotlight">Research Spotlight archive</a> and links therein.</p>
<h3 id="genome-browser-data-tracks">Genome browser data tracks</h3>
<p>These publications describe datasets displayed as genome browser tracks. Each PubMed ID links to the PomBase <a routerLink="/documentation/publication-page">publication page</a> for the paper, which in turn links to PubMed and EuropePMC. Full text and repository database links will be available via PubMed and EuropePMC where relevant.</p>
<ul>
<li>Bitton DA, Rallis C, Jeffares DC, Smith GC, Chen YY, Codlin S, Marguerat S, Bähler J.<br />
<em>Genome Res.</em> 2014 Jul;24(7):1169-79.<br />
<a routerLink="/reference/PMID:24709818">PMID:24709818</a></li>
<li>Jeffares DC, Rallis C, Rieux A, Speed D, Převorovský M, Mourier T, Marsellach FX, Iqbal Z, Lau W, Cheng TM, Pracana R, Mülleder M, Lawson JL, Chessel A, Bala S, Hellenthal G, O’Fallon B, Keane T, Simpson JT, Bischof L, Tomiczek B, Bitton DA, Sideri T, Codlin S, Hellberg JE, van Trigt L, Jeffery L, Li JJ, Atkinson S, Thodberg M, Febrer M, McLay K, Drou N, Brown W, Hayles J, Carazo Salas RE, Ralser M, Maniatis N, Balding DJ, Balloux F, Durbin R, Bähler J.<br />
<em>Nat. Genet.</em> 2015 Mar;47(3):235-41<br />
<a routerLink="/reference/PMID:25665008">PMID:25665008</a></li>
<li>Marguerat S, Schmidt A, Codlin S, Chen W, Aebersold R, Bähler J<br />
<em>Cell</em> 2012 Oct 26;151(3):671-83<br />
<a routerLink="/reference/PMID:23101633">PMID:23101633</a></li>
<li>Mata J.<br />
<em>RNA Biol 2013</em> Aug;10(8):1407-14<br />
<a routerLink="/reference/PMID:23900342">PMID:23900342</a></li>
<li>Rhind N, Chen Z, Yassour M, Thompson DA, Haas BJ, Habib N, Wapinski I, Roy S, Lin MF, Heiman DI, Young SK, Furuya K, Guo Y, Pidoux A, Chen HM, Robbertse B, Goldberg JM, Aoki K, Bayne EH, Berlin AM, Desjardins CA, Dobbs E, Dukaj L, Fan L, FitzGerald MG, French C, Gujja S, Hansen K, Keifenheim D, Levin JZ, Mosher RA, Müller CA, Pfiffner J, Priest M, Russ C, Smialowska A, Swoboda P, Sykes SM, Vaughn M, Vengrova S, Yoder R, Zeng Q, Allshire R, Baulcombe D, Birren BW, Brown W, Ekwall K, Kellis M, Leatherwood J, Levin H, Margalit H, Martienssen R, Nieduszynski CA, Spatafora JW, Friedman N, Dalgaard JZ, Baumann P, Niki H, Regev A, Nusbaum C.<br />
<em>Science</em> 2011 May 20;332(6032):930-6<br />
<a routerLink="/reference/PMID:21511999">PMID:21511999</a></li>
<li>Schlackow M, Marguerat S, Proudfoot NJ, Bähler J, Erban R, Gullerova M.<br />
<em>RNA</em> 2013 Dec;19(12):1617-31<br />
<a routerLink="/reference/PMID:24152550">PMID:24152550</a></li>
<li>Soriano I, Quintales L, Antequera F.<br />
<em>BMC Genomics</em> 2013 Nov 21;14:813<br />
<a routerLink="/reference/PMID:24256300">PMID:24256300</a></li>
<li>Woolcock KJ, Gaidatzis D, Punga T, Bühler M.<br />
<em>Nat. Struct. Mol. Biol.</em> 2011 Jan;18(1):94-9<br />
<a routerLink="/reference/PMID:21151114">PMID:21151114</a></li>
<li>Xu J, Yanagisawa Y, Tsankov AM, Hart C, Aoki K, Kommajosyula N, Steinmann KE, Bochicchio J, Russ C, Regev A, Rando OJ, Nusbaum C, Niki H, Milos P, Weng Z, Rhind N.<br />
<em>Genome Biol.</em> 2012 Apr 24;13(4):R27<br />
<a routerLink="/reference/PMID:22531001">PMID:22531001</a></li>
</ul>
<h3 id="ontology-versions">Ontology versions</h3>
<ul>
<li><p>GO: <app-cv-version cvName="molecular_function"></app-cv-version></p></li>
<li><p>FYPO: <app-cv-version cvName="fission_yeast_phenotype"></app-cv-version></p></li>
<li><p>PSI-MOD: <app-cv-version cvName="PSI-MOD"></app-cv-version></p>
<p><a routerLink="/internal-details">All ontologies and controlled vocabularies</a></p></li>
</ul>

  </div>
  <div *ngIf="pageName == 'index'">
<h2 id="about-pombase">About PomBase</h2>
<p>PomBase is a model organism database that provides organization of and access to scientific data for the fission yeast <em>Schizosaccharomyces pombe</em>. PomBase supports genomic sequence and features, genome-wide datasets and manual literature curation.</p>
<p>PomBase also provides a community hub for researchers, providing genome statistics, a community curation interface, news, events, documentation and mailing lists.</p>
<hr />
<h3 id="contact-pombase-staff">Contact PomBase staff</h3>
<p>Email: <a href="mailto:helpdesk@pombase.org">helpdesk@pombase.org</a></p>
<p>Post:</p>
<p>PomBase<br />
Department of Biochemistry<br />
University of Cambridge<br />
80 Tennis Court Road<br />
Cambridge<br />
CB2 1GA<br />
United Kingdom</p>
<h3 id="submit-data">Submit Data</h3>
<p>See the <a routerLink="/submit-data">Submit data</a> page</p>
<h3 id="data-sources">Data sources</h3>
<p>Source publications for external data used in Pombase pages and genome browser.</p>
<p><a routerLink="/about/data-sources">Data sources</a></p>
<h3 id="citing-pombase">Citing PomBase</h3>
<p>The <a routerLink="/about/citing-pombase">Citing PomBase</a> page lists papers to cite for PomBase, the <em>S. pombe</em> genome sequence, annotation data, and other resources.</p>
<h3 id="terms-of-use">Terms of Use</h3>
<p><a routerLink="/about/terms-of-use">Terms of Use</a> for PomBase data and Canto.</p>
<h3 id="pombase-publications">PomBase Publications</h3>
<p>Complete list of <a routerLink="/about/published-by-pombase">papers published by PomBase</a>.</p>
<hr />
<h3 id="pombase-sab">PomBase SAB</h3>
<p><a routerLink="/about/pombase-sab">Scientific Advisory Board members</a></p>
<hr />
<h3 id="pombase-staff">PomBase staff</h3>
<h4 id="university-of-cambridge"><a href="http://www.cam.ac.uk/">University of Cambridge</a></h4>
<ul>
<li>Val Wood (Curator &amp; co-PI)</li>
<li>Midori Harris (Curator)</li>
<li>Kim Rutherford (Programmer)</li>
<li>Steve Oliver (PI)</li>
</ul>
<h4 id="university-college-london"><a href="http://www.ucl.ac.uk/">University College London</a></h4>
<ul>
<li>Antonia Lock (Curator)</li>
<li>Jürg Bähler (co-PI)</li>
</ul>
<hr />
<h3 id="data-versions">Data versions</h3>
<p>Versions of internal and external data loaded into current and past Pombase releases.</p>
<p><a routerLink="/about/version-history">Version history</a></p>
<h3 id="receive-project-updates">Receive project updates</h3>
<p>Subscribe to <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">Pombelist</a> to receive annotation and data updates</p>
<hr />
<h3 id="privacy">Privacy</h3>
<p>See the PomBase <a routerLink="/about/privacy-policy">privacy policy</a> and <a routerLink="/about/cookie-policy">cookie policy</a>.</p>
<h3 id="project-funding">Project funding</h3>
<p>PomBase is funded by the <a href="https://wellcome.ac.uk/">Wellcome Trust</a> [104967/Z/14/Z].</p>
<p><a href="https://wellcome.ac.uk/"> <img style="background-color: black" height="127px" src="https://wellcome.ac.uk/sites/all/themes/corp_base/assets/images/new-logos/wellcome-logo.svg" alt="Wellcome"/> </a></p>

  </div>
  <div *ngIf="pageName == 'new-pombase'">
<h2 id="new-improved-pombase">New, improved PomBase</h2>
<p>The new PomBase web site was released on October 24, 2017.</p>
<p>The new site uses streamlined back-end data storage and retrieval procedures that support nightly data updates and a host of other improvements:</p>
<ul>
<li>New publication pages: For each publication loaded into PomBase, a page summarizes essential details about paper, and shows any annotations that cite it.<br />
<a routerLink="/reference/PMID:27984725">Example page (PMID:27984725)</a> and <a routerLink="/documentation/publication-page">Documentation</a></li>
<li>New genotype pages: Genotype pages are analogous to gene pages, but describe the alleles that make up a genotype, and show phenotypes associated with the genotype.<br />
<a routerLink="/genotype/e5a15cdcd4aa21c4-genotype-6">Example page (cdc25-22)</a> and <a routerLink="/documentation/genotype-page">Documentation</a></li>
<li>Improved ontology term pages: Each ontology term page now includes annotation details for annotated genes (or genotypes, for FYPO), with summary and detailed views. The detailed views show descendant terms with ontology relations, and distinguish annotations to more or less specific terms.<br />
<a routerLink="/term/GO:0000070">Example page (GO:0000070)</a> and <a routerLink="/documentation/ontology-term-page">Documentation</a></li>
<li>Improved summary views for annotation displays: On gene, genotype, term, and publication pages, the summary view is now non-redundant. See the gene page <a routerLink="/documentation">documentation</a>, example pages above, and an <a routerLink="/gene/SPCC18B5.03">example gene page (wee1)</a> for more information.</li>
<li>Phenotype annotation display filtering: The FYPO display on gene, genotype, and publication can be filtered to show annotation subsets based on broad phenotypic categories. See the gene page <a routerLink="/documentation/gene-page-phenotypes">pheontype documentation</a> and the example pages above for more information.</li>
<li>Faster querying in the <a routerLink="/query">advanced search</a></li>
<li>Front page research and community curation highlights</li>
</ul>
<p>We thank the members of the fission yeast research community who have followed its progress via the preview site, and welcome <a href="mailto:helpdesk@pombase.org?subject=Comment%20on%20the%20new%20PomBase">feedback</a> from all users.</p>

  </div>
  <div *ngIf="pageName == 'pombase-sab'">
<h2 id="pombase-scientific-advisory-board">PomBase Scientific Advisory Board</h2>
<h4 id="current-members">Current members</h4>
<p><em>Li-Lin Du</em><br> National Institute of Biological Sciences, Beijing, China</p>
<p><em>Kathleen Gould</em><br> Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN, USA</p>
<p><em>Doug Howe</em> The Zebrafish Information Network, University of Oregon, Eugene, OR, USA</p>
<p><em>Sue Jaspersen</em><br> Stowers Institute for Medical Research, Kansas City, MO, USA &amp; Department of Molecular and Integrative Physiology, University of Kansas Medical Center, Kansas City, KS, USA</p>
<p><em>Sabina Leonelli</em><br> Exeter Centre for the Study of the Life Sciences &amp; Department of Sociology, Philosophy and Anthropology, University of Exeter, Exeter, UK</p>
<p><em>Samuel Marguerat</em><br> MRC London Institute of Medical Sciences, London, UK &amp; Institute of Clinical Sciences, Faculty of Medicine, Imperial College London, London, UK</p>
<p><em>Sophie Martin</em><br> Department of Fundamental Microbiology, University of Lausanne, Lausanne, Switzerland</p>
<!--
#### Past members
(uncomment when anyone rotates off)
-->

  </div>
  <div *ngIf="pageName == 'pombase-staff'">
<h2 id="contact-us">Contact us</h2>
<p>Email: <a href="mailto:helpdesk@pombase.org">helpdesk@pombase.org</a></p>
<p>Post:</p>
<p>PomBase<br />
Department of Biochemistry<br />
University of Cambridge<br />
80 Tennis Court Road<br />
Cambridge<br />
CB2 1GA<br />
United Kingdom</p>
<h4 id="university-of-cambridge"><a href="http://www.cam.ac.uk/">University of Cambridge</a></h4>
<ul>
<li>Val Wood (Curator &amp; co-PI)</li>
<li>Midori Harris (Curator)</li>
<li>Kim Rutherford (Programmer)</li>
<li>Steve Oliver (PI)</li>
</ul>
<h4 id="university-college-london"><a href="http://www.ucl.ac.uk/">University College London</a></h4>
<p>&lt;! — Antonia Lock (Curator) –&gt; - Jürg Bähler (co-PI)</p>

  </div>
  <div *ngIf="pageName == 'privacy-policy'">
<h2 id="pombase-privacy-policy">PomBase Privacy Policy</h2>
<p>PomBase (<strong>we</strong> or <strong>us</strong>) is committed to protecting and respecting your privacy.</p>
<p>This policy (together with our Cookie Policy, Terms of Use and any other documents referred to) sets out the basis on which any personal data we collect from you, or that you provide to us, will be processed by us. Please read the following carefully to understand our views and practices regarding your personal data and how we will treat it.</p>
<p>For the purpose of the General Data Protection Regulation (GDPR), PomBase is the data controller. To contact us, email <a href="mailto:helpdesk@pombase.org">helpdesk@pombase.org</a> or see the <a href="/about">About PomBase</a> page for address details.</p>
<h3 id="information-we-may-collect-from-you">Information we may collect from you</h3>
<p>Through our website (<a href="https://www.pombase.org/">www.pombase.org</a>), we may collect and process the following data about you:</p>
<ul>
<li>Information that you provide by filling in forms on our website. This includes information provided when submitting data, including Canto curation, or when reporting a problem or contacting PomBase staff for any other purpose.</li>
<li>If you contact us, we may keep a record of that correspondence.</li>
<li>We may also ask you to complete surveys to help us improve our services, although you do not have to respond to them.</li>
<li>When you browse the website, we collect details of your visits to our site including, but not limited to, traffic data, location data, weblogs and other communication data, as well as the resources that you access.</li>
<li>Cookies and analytics information that make our site easier to use and allow us to continue to improve our website. (Read our <a href="about/cookie-policy">Cookie policy</a>.</li>
</ul>
<p>We may also access information from other sources when you interact with PomBase online:</p>
<ul>
<li>We can collect information about your interests and browsing behaviour from Google Analytics.</li>
</ul>
<h3 id="how-we-use-personal-information">How we use personal information</h3>
<p>We use information held about you in the following ways:</p>
<ul>
<li>To respond to your enquiries and keep a record of our communications with you</li>
<li>To ensure that content from our site is presented in the most effective manner for you and for your computer</li>
<li>To provide you with information, events or services that you request from us or which we feel may interest you, where you have consented to be contacted for such purposes</li>
<li>To allow you to participate in interactive features of our service, when you choose to do so</li>
<li>To notify you about changes to our site</li>
<li>To send you updates, information and newsletters if you request them</li>
</ul>
<h3 id="collecting-and-using-personal-data">Collecting and using personal data</h3>
<p>you are consenting to PomBase storing and using it for the purposes for which it was provided. We will not use your personal information to contact you unless you have specifically requested to receive information from us.</p>
<p>We will hold your personal data for no longer than necessary for the purposes explained in this document.</p>
<h3 id="reasons-we-can-collect-and-use-your-personal-information">Reasons we can collect and use your personal information</h3>
<p>We rely on Article 6(1)(f) of the GDPR as the lawful basis on which we collect and use your personal data. That means that we can collect and use your personal data on the basis that it is necessary for the purposes of our legitimate interests, except where our interests are overridden by your privacy interests.</p>
<p>Our legitimate interests are maintaining our data resources, providing data and services to our users, and raising awareness of PomBase and fission yeast research online, in print, and in person.</p>
<h3 id="disclosure-of-your-information">Disclosure of your information</h3>
<p>We may disclose your personal information to the PomBase collaboration’s member institutions (<a href="http://www.cam.ac.uk/">University of Cambridge</a>, <a href="http://www.ucl.ac.uk/">University College London</a>, the <a href="https://babraham.ac.uk/">Babraham Institute</a>) as necessary for the purposes of our legitimate interests as given above.</p>
<p>We may also disclose your personal information to third parties if we are under a duty to disclose or share your personal data in order to comply with any legal obligation, or in order to enforce or apply our Terms of use and other agreements; or to protect the rights, property, or safety of our member institutions or others.</p>
<p>Our site may, from time to time, contain links to and from the websites of other online biological databases and similar resources. If you follow a link to any of these websites, please note that these websites have their own privacy policies and that we do not accept any responsibility or liability for these policies. Please check these policies before you submit any personal data to these websites.</p>
<p>We will not share your personal information with any other third party.</p>
<h3 id="your-rights">Your rights</h3>
<p>Under the General Data Protection Regulation you have a number of important rights free of charge. In summary, those include rights to:</p>
<ul>
<li>fair processing of information and transparency over how we use your use personal information</li>
<li>access to your personal information and to certain other supplementary information that this Privacy Notice is already designed to address</li>
<li>require us to correct any mistakes in your information which we hold</li>
<li>require the erasure of personal information concerning you in certain situations</li>
<li>receive the personal information concerning you which you have provided to us, in a structured, commonly used and machine-readable format and have the right to transmit those data to a third party in certain situations</li>
<li>object at any time to processing of personal information concerning you for direct marketing</li>
<li>object to decisions being taken by automated means which produce legal effects concerning you or similarly significantly affect you</li>
<li>object in certain other situations to our continued processing of your personal information</li>
<li>otherwise restrict our processing of your personal information in certain circumstances</li>
</ul>
<p>For further information on each of those rights, including the circumstances in which they apply, see the Guidance from the UK [Information Commissioner’s Office[(https://ico.org.uk/) (ICO) on individuals’ rights under the General Data Protection Regulation.</p>
<p>If you would like to exercise any of those rights, please contact us (see below).</p>
<h3 id="challenging-our-use-of-personal-data">Challenging our use of personal data</h3>
<p>We hope that we can resolve any query or concern you raise about our use of your information.</p>
<p>The General Data Protection Regulation gives you right to lodge a complaint with a supervisory authority, in particular in the European Union (or European Economic Area) state where you work, normally live or where any alleged infringement of data protection laws occurred. The supervisory authority in the UK is the Information Commissioner, who may be contacted at <a href="https://ico.org.uk/make-a-complaint/">ico.org.uk/concerns</a>.</p>
<h3 id="changes-to-our-privacy-policy">Changes to our privacy policy</h3>
<p>This privacy notice was published on 20 September 2018.</p>
<p>We may change this privacy notice from time to time. You should check this policy occasionally to ensure you are aware of the most recent version.</p>
<h3 id="contact-us">Contact us</h3>
<p>For any further enquiries about the PomBase’s Privacy Policy, or if you have concerns about how your data is being processed or would like to report a possible data breach, please contact <a href="mailto:helpdesk@pombase.org">helpdesk@pombase.org</a>.</p>

  </div>
  <div *ngIf="pageName == 'published-by-pombase'">
<h2 id="publications-authored-by-pombase-staff">Publications authored by PomBase staff</h2>
<p>Please see the <a routerLink="/about/citing-pombase">Citing PomBase</a> page, which lists papers to cite for PomBase, the <em>S. pombe</em> genome sequence, Canto, FYPO, annotations and Compara.</p>
<h3 id="journal-articles">Journal articles</h3>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1098/rsob.200149">

</div>
<div class="pub-citation">
<p>Valerie Wood, Seth Carbon, Midori A. Harris, Antonia Lock†, Stacia R. Engel, David P. Hill, Kimberly Van Auken, Helen Attrill, Marc Feuermann, Pascale Gaudet, Ruth C. Lovering, Sylvain Poux, Kim M. Rutherford and Christopher J. Mungall<br />
Term Matrix: a novel Gene Ontology annotation quality control system based on ontology term co-annotation patterns<br />
<em>Open Biol.</em> 10:200149</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/32875947">PMID:32875947</a> <a href="https://doi.org/10.1098/rsob.200149">10.1098/rsob.200149</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/database/baaa028.">

</div>
<div class="pub-citation">
<p>Lock A, Harris MA, Rutherford K, Hayles J, Wood V.<br />
Community curation in PomBase: enabling fission yeast experts to provide detailed, standardized, sharable annotation from research publications. <em>Database (Oxford)</em> 2020 Jan 1;2020. pii: baaa028.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/32353878">PMID:32353878</a> <a href="https://doi.org/10.1093/database/baaa028">DOI:10.1093/database/baaa028</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1098/rsob.180241">

</div>
<div class="pub-citation">
<p>Wood V, Lock A, Harris MA, Rutherford K, Bähler J, Oliver SG.<br />
Hidden in plain sight: What remains to be discovered in the eukaryotic proteome?<br />
<em>Open Biol</em> 2019 Feb 20;9(2):180241.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/30938578">PMID:30938578</a> <a href="https://doi.org/10.1098/rsob.180241">DOI:10.1098/rsob.180241</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gky961">

</div>
<div class="pub-citation">
<p>Lock A, Rutherford K, Harris MA, Hayles J, Oliver SG, Bähler J; Wood V.<br />
PomBase 2018: user-driven reimplementation of the fission yeast database provides rapid and intuitive access to diverse, interconnected information.<br />
<em>Nucleic Acids Res.</em> 2019 Jan 8;47(Database issue): D821-D827. Epub 2018 Oct 13.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/30321395">PMID:30321395</a> <a href="https://doi.org/10.1093/nar/gky961">DOI:10.1093/nar/gky961</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1186/s12915-016-0276-z">

</div>
<div class="pub-citation">
<p>Oliver SG, Lock A, Harris MA, Nurse P, Wood V.<br />
Model organism databases: essential resources that need the support of both funders and users.<br />
<em>BMC Biol.</em> 2016 14(1): 49.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/27334346">PMID:27334346</a> <a href="http://dx.doi.org/10.1186/s12915-016-0276-z">DOI:10.1186/s12915-016-0276-z</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gku1040">

</div>
<div class="pub-citation">
<p>McDowall MD, Harris MA, Lock A, Rutherford K, Staines DM, Bähler J, Kersey PJ, Oliver SG, Wood V.<br />
PomBase 2015: updates to the fission yeast database.<br />
<em>Nucleic Acids Res.</em> 2015 Jan;43(Database issue):D665-61. Epub 2014 Oct 31.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25361970">PMID:25361970</a> <a href="http://dx.doi.org/10.1093/nar/gku1040">DOI:10.1093/nar/gku1040</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/bioinformatics/btu103">

</div>
<div class="pub-citation">
<p>Rutherford KM, Harris MA, Lock A, Oliver SG, Wood V.<br />
Canto: An online tool for community literature curation.<br />
<em>Bioinformatics</em>. 2014 Jun 15;30(12):1791-2. Epub 2014 Feb 25.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/24574118">PMID:24574118</a> <a href="http://dx.doi.org/10.1093/bioinformatics/btu103">DOI:10.1093/bioinformatics/btu103</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/bioinformatics/btt266">

</div>
<div class="pub-citation">
<p>Harris MA, Lock A, Bähler J, Oliver SG, Wood V.<br />
FYPO: The Fission Yeast Phenotype Ontology.<br />
<em>Bioinformatics</em>. 2013 Jul 1;29(13):1671-8. Epub 2013 May 8.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/23658422">PMID:23658422</a> <a href="http://dx.doi.org/10.1093/bioinformatics/btt266">DOI:10.1093/bioinformatics/btt266</a>.</p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gkr853">

</div>
<div class="pub-citation">
<p>Wood V, Harris MA, McDowall MD, Rutherford K, Vaughan BW, Staines DM, Aslett M, Lock A, Bähler J, Kersey PJ, Oliver SG.<br />
PomBase: a comprehensive online resource for fission yeast.<br />
<em>Nucleic Acids Res.</em> 2012 Jan;40(Database issue):D695-9. Epub 2011 Oct 28.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153">PMID:22039153</a> <a href="http://dx.doi.org/DOI:10.1093/nar/gkr853">DOI:10.1093/nar/gkr853</a></p>
<!-- comment out preprint section when there aren't any?
### Preprints 

Wood V, Carbon S, Harris MA, Lock A, Engel SR, Hill DP, Van Auken K, Attrill H, Feuermann M, Gaudet P, Lovering RC, Poux S, Rutherford KM, Mungall CJ.\
Term Matrix: A novel Gene Ontology annotation quality control system based on ontology term co-annotation patterns\.
<a href="https://www.biorxiv.org/content/10.1101/2020.04.21.045195v1.full">Pre-publication manuscript</a> available at bioRxiv.

Wood V, Lock A, Harris MA, Rutherford K, Bähler J, Oliver SG.\
Hidden in plain sight: What remains to be discovered in the eukaryotic proteome?\
<a href="https://www.biorxiv.org/content/early/2018/11/16/469569">Pre-publication manuscript</a> available at bioRxiv.
-->
<h3 id="book-chapter">Book chapter</h3>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1007/978-1-4939-7737-6_4">

</div>
<div class="pub-citation">
<p>Lock A, Rutherford K, Harris MA, Wood V. 2018.<br />
PomBase: The Scientific Resource for Fission Yeast.<br />
In Kollar M (ed) <em>Eukaryotic Genomic Databases. Methods in Molecular Biology</em>, vol 1757. Humana Press, New York, NY<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://dx.doi.org/DOI:10.1007/978-1-4939-7737-6_4">DOI:10.1007/978-1-4939-7737-6_4</a></p>
<h3 id="infographics">Infographics</h3>
<p><a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2017/pombe2017_infographic_p132.pdf">2017 PomBase infographic</a> (PDF; file at FTP site)</p>
<h3 id="more">More</h3>
<p>Posters and slides from conferences and workshops are available on the <a routerLink="/documents">Documents</a> page.</p>
<h2 id="collaborative-publications-including-pombase-authors">Collaborative publications including PomBase authors</h2>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gkz997">

</div>
<div class="pub-citation">
<p>Shefchek KA et al. (including Harris MA).<br />
The Monarch Initiative in 2019: an integrative data and analytic platform connecting phenotypes to genotypes across species.<br />
<em>Nucleic Acids Res.</em> 2020 Jan 8;48(Database issue): D704-D715.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/31701156">PMID:31701156</a> <a href="https://doi.org/10.1093/nar/gkz997">DOI:10.1093/nar/gkz997</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gkz904">

</div>
<div class="pub-citation">
<p>Urban M, Cuzick A, Seager J, Wood V, Rutherford K, Venkatesh SY, De Silva N, Martinez MC, Pedro H, Yates AD, Hassani-Pak K, Hammond-Kosack KE.<br />
PHI-base: the pathogen-host interactions database.<br />
<em>Nucleic Acids Res.</em> 2020 Jan 8;48(Database issue): D613-D620.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/31733065">PMID:31733065</a> <a href="https://doi.org/10.1093/nar/gkz904">DOI:10.1093/nar/gkz904</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gky1055">

</div>
<div class="pub-citation">
<p>The Gene Ontology Consortium.<br />
The Gene Ontology Resource: 20 years and still GOing strong.<br />
<em>Nucleic Acids Res.</em> 2019 Jan 8;47(Database issue):D330-D338.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/30395331">PMID:30395331</a> <a href="http://dx.doi.org/10.1093/nar/gky1055">DOI:10.1093/nar/gky1055</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gky1034">

</div>
<div class="pub-citation">
<p>The RNAcentral Consortium.<br />
RNAcentral: a hub of information for non-coding RNA sequences.<br />
<em>Nucleic Acids Res.</em> 2019 Jan 8;47(Database issue):D221-D229. Epub 2018 Nov 5.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/30395267">PMID:30395267</a> <a href="http://dx.doi.org/10.1093/nar/gky1034">DOI:10.1093/nar/gky1034</a> (Correction in <a href="http://dx.doi.org/10.1093/nar/gky1206">DOI:10.1093/nar/gky1206</a>)</p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gkw1089">

</div>
<div class="pub-citation">
<p>Urban M, Cuzick A, Rutherford K, Irvine A, Pedro H, Pant R, Sadanadan V, Khamari L, Billal S, Mohanty S, Hammond-Kosack KE.<br />
PHI-base: a new interface and further additions for the multi-species pathogen-host interactions database.<br />
<em>Nucleic Acids Res.</em> 2017 Jan 4;45(D1):D604-D610. Epub 2016 Dec 3.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/27915230">PMID:27915230</a> <a href="http://dx.doi.org/10.1093/nar/gkw1089">DOI:10.1093/nar/gkw1089</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gkw1108">

</div>
<div class="pub-citation">
<p>The Gene Ontology Consortium<br />
Expansion of the Gene Ontology knowledgebase and resources.<br />
<em>Nucleic Acids Res.</em> 2017 Jan 4;45(D1):D331-D338. Epub 2016 Nov 29.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/27899567">PMID:27899567</a> <a href="http://dx.doi.org/10.1093/nar/gkw1108">DOI:10.1093/nar/gkw1108</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gkw1008">

</div>
<div class="pub-citation">
<p>The RNAcentral Consortium<br />
RNAcentral: a comprehensive database of non-coding RNA sequences.<br />
<em>Nucleic Acids Res.</em> 2017 Jan 4;45(D1):D128-D134. Epub 2016 Oct 28.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/27794554">PMID:27794554</a> <a href="http://dx.doi.org/10.1093/nar/gkw1008">DOI:10.1093/nar/gkw1008</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1261/rna.055301.115">

</div>
<div class="pub-citation">
<p>Huntley RP, Sitnikov D, Orlic-Milacic M, Balakrishnan R, D’Eustachio P, Gillespie ME, Howe D, Kalea AZ, Maegdefessel L, Osumi-Sutherland D, Petri V, Smith JR, Van Auken K, Wood V, Zampetaki A, Mayr M, Lovering RC.<br />
Guidelines for the functional annotation of microRNAs using the Gene Ontology.<br />
<em>RNA.</em> 2016 May;22(5):667-76. Epub 2016 Feb 25.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/26917558">PMID:26917558</a> <a href="http://dx.doi.org/10.1261/rna.055301.115">DOI:10.1261/rna.055301.115</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.3389/fgene.2015.00330">

</div>
<div class="pub-citation">
<p>Bitton DA, Schubert F, Dey S, Okoniewski M, Smith GC, Khadayate S, Pancaldi V, Wood V, Bähler J.<br />
AnGeLi: A Tool for the Analysis of Gene Lists from Fission Yeast.<br />
<em>Front Genet.</em> 2015 Nov 16;6:330. eCollection 2015.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/26635866">PMID:26635866</a> <a href="http://dx.doi.org/10.3389/fgene.2015.00330">DOI:10.3389/fgene.2015.00330</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1534/genetics.115.181503">

</div>
<div class="pub-citation">
<p>Hoffman CS, Wood V, Fantes PA.<br />
An Ancient Yeast for Young Geneticists: A Primer on the <em>Schizosaccharomyces pombe</em> Model System. <em>Genetics</em>. 2015 Oct;201(2):403-23.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/26447128">PMID:26447128</a> <a href="http://dx.doi.org/10.1534/genetics.115.181503">DOI:10.1534/genetics.115.181503</a><br />
(Erratum in <a href="http://www.ncbi.nlm.nih.gov/pubmed/26953269">PMID:26953269</a> <a href="http://dx.doi.org/10.1534/genetics.116.187088">DOI:10.1534/genetics.116.187088</a>)</p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gku1179">

</div>
<div class="pub-citation">
<p>Gene Ontology Consortium<br />
Gene Ontology Consortium: going forward.<br />
<em>Nucleic Acids Res.</em> 2015 Jan;43(Database issue):D1049-56. Epub 2014 Nov 26.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/25428369">PMID:25428369</a> <a href="http://dx.doi.org/10.1093/nar/gku1179">DOI:10.1093/nar/gku1179</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1016/j.tibtech.2014.05.003">

</div>
<div class="pub-citation">
<p>Dikicioglu D, Wood V, Rutherford KM, McDowall MD, Oliver SG.<br />
Improving functional annotation for industrial microbes: a case study with Pichia pastoris.<br />
<em>Trends Biotechnol.</em> 2014 Aug;32(8):396-9. Epub 2014 Jun 11.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929579">PMID:24929579</a> <a href="http://dx.doi.org/10.1016/j.tibtech.2014.05.003">DOI:10.1016/j.tibtech.2014.05.003</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1371/journal.pone.0099864">

</div>
<div class="pub-citation">
<p>Alam-Faruque Y, Hill DP, Dimmer EC, Harris MA, Foulger RE, Tweedie S, Attrill H, Howe DG, Thomas SR, Davidson D, Woolf AS, Blake JA, Mungall CJ, O’Donovan C, Apweiler R, Huntley RP.<br />
Representing kidney development using the gene ontology.<br />
<em>PLoS One.</em> 2014 Jun 18;9(6):e99864. eCollection 2014.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/24941002">PMID:24941002</a> <a href="http://dx.doi.org/10.1371/journal.pone.0099864">DOI:10.1371/journal.pone.0099864</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1186/1471-2105-15-155">

</div>
<div class="pub-citation">
<p>Huntley RP, Harris MA, Alam-Faruque Y, Blake JA, Carbon S, Dietze H, Dimmer EC, Foulger RE, Hill DP, Khodiyar VK, Lock A, Lomax J, Lovering RC, Mutowo-Meullenet P, Sawford T, Van Auken K, Wood V, Mungall CJ.<br />
A method for increasing expressivity of Gene Ontology annotations using a compositional approach.<br />
<em>BMC Bioinformatics.</em> 2014 May 21;15:155.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/24885854">PMID:24885854</a> <a href="http://dx.doi.org/10.1186/1471-2105-15-155">DOI:10.1186/1471-2105-15-155</a>.</p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1186/1471-2164-14-513">

</div>
<div class="pub-citation">
<p>Hill DP, Adams N, Bada M, Batchelor C, Berardini TZ, Dietze H, Drabkin HJ, Ennis M, Foulger RE, Harris MA, Hastings J, Kale NS, de Matos P, Mungall CJ, Owen G, Roncaglia P, Steinbeck C, Turner S, Lomax J.<br />
Dovetailing biology and chemistry: integrating the Gene Ontology with the ChEBI chemical ontology.<br />
<em>BMC Genomics.</em> 2013 Jul 29;14:513.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/23895341">PMID:23895341</a> <a href="http://dx.doi.org/10.1186/1471-2164-14-513">DOI:10.1186/1471-2164-14-513</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/database/bat054">

</div>
<div class="pub-citation">
<p>Balakrishnan R1, Harris MA, Huntley R, Van Auken K, Cherry JM.<br />
A guide to best practices for Gene Ontology (GO) manual annotation.<br />
<em>Database (Oxford).</em> 2013 Jul 9;2013:bat054. Print 2013.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/23842463">PMID:23842463</a> <a href="http://dx.doi.org/10.1093/database/bat054">DOI:10.1093/database/bat054</a>.</p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1098/rsob.130053">

</div>
<div class="pub-citation">
<p>Hayles J, Wood V, Jeffery L, Hoe KL, Kim DU, Park HO, Salas-Pino S, Heichinger C, Nurse P.<br />
A genome-wide resource of cell cycle and cell shape genes of fission yeast.<br />
<em>Open Biol.</em> 2013 May 22;3(5):130053.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/23697806">PMID:23697806</a> <a href="http://dx.doi.org/10.1098/rsob.130053">DOI:10.1098/rsob.130053</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gks1050">

</div>
<div class="pub-citation">
<p>Gene Ontology Consortium<br />
Gene Ontology annotations and resources.<br />
<em>Nucleic Acids Res.</em> 2013 Jan;41(Database issue):D530-5. Epub 2012 Nov 17.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/23161678">PMID:23161678</a> <a href="http://dx.doi.org/10.1093/nar/gks1050">DOI:10.1093/nar/gks1050</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/bioinformatics/bts577">

</div>
<div class="pub-citation">
<p>Smith RN1, Aleksic J, Butano D, Carr A, Contrino S, Hu F, Lyne M, Lyne R, Kalderimis A, Rutherford K, Stepan R, Sullivan J, Wakeling M, Watkins X, Micklem G.<br />
InterMine: a flexible data warehouse system for the integration and analysis of heterogeneous biological data.<br />
<em>Bioinformatics.</em> 2012 Dec 1;28(23):3163-5. Epub 2012 Sep 27.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/23023984">PMID:23023984</a> <a href="http://dx.doi.org/10.1093/bioinformatics/bts577">DOI:10.1093/bioinformatics/bts577</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/bioinformatics/bts250">

</div>
<div class="pub-citation">
<p>Hoehndorf R, Harris MA, Herre H, Rustici G, Gkoutos GV.<br />
Semantic integration of physiology phenotypes with an application to the Cellular Phenotype Ontology.<br />
<em>Bioinformatics.</em> 2012 Jul 1;28(13):1783-9. Epub 2012 Apr 26.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/22539675">PMID:22539675</a> <a href="http://dx.doi.org/10.1093/bioinformatics/bts250">DOI:10.1093/bioinformatics/bts250</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1371/journal.pcbi.1002386">

</div>
<div class="pub-citation">
<p>Thomas PD, Wood V, Mungall CJ, Lewis SE, Blake JA; Gene Ontology Consortium.- On the Use of Gene Ontology Annotations to Assess Functional Similarity among Orthologs and Paralogs: A Short Report.<br />
<em>PLoS Comput Biol.</em> 2012;8(2):e1002386. Epub 2012 Feb 16.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/22359495">PMID:22359495</a> <a href="http://dx.doi.org/10.1371/journal.pcbi.1002386">DOI:10.1371/journal.pcbi.1002386</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gkr921">

</div>
<div class="pub-citation">
<p>Contrino S, Smith RN, Butano D, Carr A, Hu F, Lyne R, Rutherford K, Kalderimis A, Sullivan J, Carbon S, Kephart ET, Lloyd P, Stinson EO, Washington NL, Perry MD, Ruzanov P, Zha Z, Lewis SE, Stein LD, Micklem G.<br />
modMine: flexible access to modENCODE data.<br />
<em>Nucleic Acids Res.</em> 2012 Jan;40(Database issue):D1082-8. Epub 2011 Nov 12.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/22080565">PMID:22080565</a> <a href="http://dx.doi.org/10.1093/nar/gkr921">DOI:10.1093/nar/gkr921</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1186/1471-2105-12-325">

</div>
<div class="pub-citation">
<p>Leonelli S1, Diehl AD, Christie KR, Harris MA, Lomax J.<br />
How the gene ontology evolves.<br />
<em>BMC Bioinformatics.</em> 2011 Aug 5;12:325.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/">PMID:21819553</a> <a href="http://dx.doi.org/10.1186/1471-2105-12-325">DOI:10.1186/1471-2105-12-325</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1534/genetics.110.123497">

</div>
<div class="pub-citation">
<p>Bitton DA, Wood V, Scutt PJ, Grallert A, Yates T, Smith DL, Hagan IM, Miller CJ.<br />
Augmented annotation of the Schizosaccharomyces pombe genome reveals additional genes required for growth and viability.<br />
<em>Genetics.</em> 2011 Apr;187(4):1207-17. Epub 2011 Jan 26.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/21270388">PMID:21270388</a> <a href="http://dx.doi.org/10.1534/genetics.110.123497">DOI:10.1534/genetics.110.123497</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gkr1028">

</div>
<div class="pub-citation">
<p>Gene Ontology Consortium<br />
The Gene Ontology:enhancements for 2011.<br />
<em>Nucleic Acids Res.</em> 2011 Jan;40(Database issue):D559-64. Epub 2012 Nov 17.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/22102568">PMID:22102568</a> <a href="http://dx.doi.org/10.1093/nar/gkr1028">DOI:10.1093/nar/gkr1028</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1038/nbt.1628">

</div>
<div class="pub-citation">
<p>Kim DU, Hayles J, Kim D, Wood V, Park HO, Won M, Yoo HS, Duhig T, Nam M, Palmer G, Han S, Jeffery L, Baek ST, Lee H, Shim YS, Lee M, Kim L, Heo KS, Noh EJ, Lee AR, Jang YJ, Chung KS, Choi SJ, Park JY, Park Y, Kim HM, Park SK, Park HJ, Kang EJ, Kim HB, Kang HS, Park HM, Kim K, Song K, Song KB, Nurse P, Hoe KL.<br />
Analysis of a genome-wide set of gene deletions in the fission yeast Schizosaccharomyces pombe.<br />
<em>Nat Biotechnol.</em> 2010 Jun;28(6):617-623. Epub 2010 May 16.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/20473289">PMID:20473289</a> <a href="http://dx.doi.org/10.1038/nbt.1628">DOI:10.1038/nbt.1628</a></p>
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1093/nar/gkp1018">

</div>
<div class="pub-citation">
<p>Gene Ontology Consortium<br />
The Gene Ontology in 2010: extensions and refinements.<br />
<em>Nucleic Acids Res.</em> 2010 Jan;38(Database issue):D331-5. Epub 2009 Nov 17.<br />
</p>
</div>
<div class="clear-float">

</div>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/19920128">PMID:19920128</a> <a href="http://dx.doi.org/10.1093/nar/gkp1018">DOI:10.1093/nar/gkp1018</a></p>
<p><em>Altmetric badges link to data provided by <a href="https://www.altmetric.com/">Altmetric.com</a>, a research metrics company who track and collect the online conversations around millions of scholarly outputs. Read <a href="https://www.altmetric.com/about-altmetrics/">more about Altmetrics</a>.</em></p>

  </div>
  <div *ngIf="pageName == 'resource-metrics'">
<h1 id="resource-metrics">Resource Metrics</h1>
<h2 id="curation">Curation</h2>
<p>The <a href="https://curation.pombase.org/pombe/stats/annotation">Curation Statistics</a> page (currently part of PomBase Canto) provides a set of metrics reflecting the current state of literature curation in PomBase.</p>
<h2 id="web-usage">Web usage</h2>
<p>Statistics forthcoming</p>

  </div>
  <div *ngIf="pageName == 'terms-of-use'">
<h2 id="pombase-terms-of-use">PomBase Terms of Use</h2>
<p>All data curated by PomBase, including data from Canto community curation, are licensed under the Creative Commons Attribution 4.0 International license (CC-BY; see the Creative Commons site for a <a href="https://creativecommons.org/licenses/by/4.0/">summary</a> and the <a href="https://creativecommons.org/licenses/by/4.0/legalcode">full license</a>).</p>
<p>Datasets hosted in the genome browser and front-page Research Spotlight images are used with permission of the generating researchers. Specific copyright and licensing terms may apply; consult the relevant publications (listed on the <a routerLink="/about/data-sources">Data Sources</a> page) for details.</p>
<p>Canto is is licensed under the GNU General Public License v3.0; see the <a href="https://github.com/pombase/canto/blob/master/LICENSE">Canto License page</a> for more information.</p>
<p>Also see the <a routerLink="/about/citing-pombase">Citing PomBase</a> page.</p>

  </div>
  <div *ngIf="pageName == 'version-history'">
<h1 id="pombase-data-version-history">PomBase data version history</h1>
<p>This table records versions of software, annotations, etc. used in the numbered releases of Pombase made between September 2012 and January 2017. See the <a routerLink="/downloads/chado-database-dumps">Chado database dumps</a> page for a link to the data represented by the “Annotation Version” column.</p>
<h4 id="past-pombase-releases">Past PomBase Releases</h4>
<table border="1">
 <tbody>
 <tr>
  <td>Release Date</td>
  <td>INSDC</td>
  <td>Gene Build</td>
  <td>Ensembl Software Release</td>
  <td>Annotation Version</td>
  <td>GO GAF</td>
  <td>InterPro</td>
  <td>Manually Curated Ortholog table</td>
 </tr>
 <tr>
  <td><span class="no-break">2017-01-30</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!84, EG30</span></td>
  <td>62</td>
  <td><span class="no-break">2017-03-17</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2016-10-21</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!84, EG30</span></td>
  <td>61</td>
  <td><span class="no-break">2016-10-21</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2016-06-01</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!84, EG30</span></td>
  <td>60</td>
  <td><span class="no-break">2016-06-01</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2016-05-10</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!84, EG30</span></td>
  <td>59</td>
  <td><span class="no-break">2016-05-11</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2016-04-12</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!84, EG30</span></td>
  <td>58</td>
  <td><span class="no-break">2016-04-12</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2016-02-11</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!83, EG30</span></td>
  <td>57</td>
  <td><span class="no-break">2016-02-11</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-12-02</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!82, EG29</span></td>
  <td>56</td>
  <td><span class="no-break">2015-12-02</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-09-29</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!81, EG28</span></td>
  <td>55</td>
  <td><span class="no-break">2015-09-29</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-09-07</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!81, EG28</span></td>
  <td>54</td>
  <td><span class="no-break">2015-09-01</span></td>
  <td><span>InterProScan5 (5.14-53.0 23 July 2015)</span></td>
  <td>pombe/cerevisiae: 2.21 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-08-13</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-06-17</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!79, EG26</span></td>
  <td>53</td>
  <td><span class="no-break">2015-06-17</span></td>
  <td>InterProScan5 (5.8-49.0 20-November-2014)</td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-05-27</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-05-28</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!79, EG26</span></td>
  <td>52</td>
  <td><span class="no-break">2015-05-29</span></td>
  <td><span>InterProScan5 (5.8-49.0 20-November-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-05-27</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-04-21</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!79, EG26</span></td>
  <td>51</td>
  <td><span class="no-break">2015-04-21</span></td>
  <td><span>InterProScan5 (5.8-49.0 20-November-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-03-10</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-03-25</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!78, EG25</span></td>
  <td>50</td>
  <td><span class="no-break">2015-03-25</span></td>
  <td><span>InterProScan5 (5.8-49.0 20-November-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2015-03-10</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-02-19</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!78, EG25</span></td>
  <td>49</td>
  <td><span class="no-break">2015-02-17</span></td>
  <td><span>InterProScan5 (5.8-49.0 20-November-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2014-09-01</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2015-01-27</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!78, EG25</span></td>
  <td>48</td>
  <td><span class="no-break">2015-01-20</span></td>
  <td><span>InterProScan5 (5.8-49.0 20-November-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2014-09-01</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2014-11-13</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!76, EG23</span></td>
  <td>47</td>
  <td><span class="no-break">2014-11-13</span></td>
  <td><span>InterProScan5 (5.7-48.0 August-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2014-09-01</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2014-09-16</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!76, EG23</span></td>
  <td>46</td>
  <td><span class="no-break">2014-09-15</span></td>
  <td><span>InterProScan5 (5.7-48.0 August-2014)</span></td>
  <td>pombe/cerevisiae: 2.20 (<span class="no-break">2014-08-31</span>); pombe/human <span class="no-break">2014-09-01</span></td>
 </tr>
 <tr>
  <td><span class="no-break">2014-08-18</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!76, EG23</span></td>
  <td>45</td>
  <td><span class="no-break">2014-08-18</span></td>
  <td><span>InterProScan5 (5.7-48.0 August-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-07-18</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!75, EG22</span></td>
  <td>44</td>
  <td><span class="no-break">2014-07-18</span></td>
  <td><span>InterProScan5 (5.4-47.0 June-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-07-08</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!75, EG22</span></td>
  <td>43</td>
  <td><span class="no-break">2014-07-08</span></td>
  <td><span>InterProScan5 (5.4-47.0 June-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-05-15</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!75, EG22</span></td>
  <td>42</td>
  <td><span class="no-break">2014-05-13</span></td>
  <td><span>InterProScan5 (5.3-46.0 January-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-03-20</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!74, EG21</span></td>
  <td>41</td>
  <td><span class="no-break">2014-03-20</span></td>
  <td><span>InterProScan5 (5.3-46.0 January-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-02-20</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!74, EG21</span></td>
  <td>40</td>
  <td><span class="no-break">2014-02-20</span></td>
  <td><span>InterProScan5 (5.3-46.0 January-2014)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-12-10</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!73, EG20</span></td>
  <td>39</td>
  <td><span class="no-break">2013-12-04</span></td>
  <td><span>InterProScan5 (<span class="no-break">2013-04-24</span>)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-10-22</span></td>
  <td>ASM294v2</td>
  <td>2.1&nbsp;</td>
  <td><span>e!73, EG20</span></td>
  <td>38</td>
  <td><span class="no-break">2013-11-11</span></td>
  <td><span>InterProScan5 (<span class="no-break">2013-04-24</span>)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-09-16</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!71, EG18</span></td>
  <td>37</td>
  <td><span class="no-break">2013-09-18</span></td>
  <td><span>InterProScan5 (<span class="no-break">2013-04-24</span>)</span></td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-06-21</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td>e!71, EG18</td>
  <td>36</td>
  <td><span class="no-break">2013-07-02</span></td>
  <td>InterProScan5 (<span class="no-break">2013-04-24</span>)</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-05-21</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!71, EG18</span>&nbsp;</td>
  <td>35</td>
  <td><span class="no-break">2013-06-06</span></td>
  <td>&nbsp;</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-05-07</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!71, EG18</span>&nbsp;</td>
  <td>34</td>
  <td><span class="no-break">2013-04-18</span></td>
  <td>&nbsp;</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-04-02</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td>e!70, EG17</td>
  <td>33</td>
  <td><span class="no-break">2013-03-07</span></td>
  <td>&nbsp;</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-02-26</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!70, EG17</span>&nbsp;</td>
  <td>31</td>
  <td><span class="no-break">2013-01-16</span></td>
  <td>&nbsp;</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span class="no-break">2012-11-19</span></td>
  <td>ASM294v2</td>
  <td>2.1</td>
  <td><span>e!69, EG16</span>&nbsp;</td>
  <td>30</td>
  <td><span class="no-break">2012-11-27</span></td>
  <td>&nbsp;</td>
  <td>pombe/cerevisiae: 2.18 (<span class="no-break">2012-11-07</span>)</td>
 </tr>
 <tr>
  <td><span>2012-10</span></td>
  <td><span>ASM294v2</span></td>
  <td>2.1</td>
  <td><span>e!69, EG16</span>&nbsp;</td>
  <td>29&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><span>2012-09</span></td>
  <td><span>ASM294v2</span></td>
  <td>2.1&nbsp;</td>
  <td><span>e!68, EG15</span>&nbsp;</td>
  <td>28&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 </tbody>
</table>
<h4 id="brief-explanations">Brief explanations</h4>
<table>
<thead>
<tr class="header">
<th>Item</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>INSDC</td>
<td>Accession of the current <em>S. pombe</em> sequence in the International Nucleotide Sequence Data Collaboration (ENA, GenBank, DDBJ) databases</td>
</tr>
<tr class="even">
<td>Gene Build</td>
<td>Version of gene feature annotation</td>
</tr>
<tr class="odd">
<td>Ensembl Software Release</td>
<td>Versions of ensembl (e!) and Ensembl Genomes (EG) data and software used to build PomBase</td>
</tr>
<tr class="even">
<td>Annotation Version</td>
<td>Version of curated PomBase annotations (sequence features, ontology annotations, etc.)</td>
</tr>
<tr class="odd">
<td>GO GAF</td>
<td>Date on which GO annotation data were retrieved from the GO repository</td>
</tr>
<tr class="even">
<td>InterPro</td>
<td>Date and version of InterProScan used to find matches to InterPro entries</td>
</tr>
<tr class="odd">
<td>Manually Curated Ortholog table</td>
<td>Date and version of the manually curated sets of pombe/cerevisiae (and, where included, pombe/human) orthologs</td>
</tr>
</tbody>
</table>

  </div>
  </div>
</div>
<div *ngIf="section == 'browse-curation'">
<app-page-contents-menu title="Browse curation" titleRoute="/browse-curation">

<div class="left-menu-part left-menu-item"><a routerLink="/browse-curation/fission-yeast-phenotype-ontology">Fission Yeast Phenotype Ontology</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/browse-curation/fission-yeast-mf-go-slim-terms">GO function slim terms</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">GO process slim terms</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/browse-curation/fission-yeast-cc-go-slim-terms">GO component slim terms</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/browse-curation/dna-binding-sites">DNA binding sites</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/browse-curation/drugs-known-pombe-targets">Drugs with known targets</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/browse-curation/disease-slim">Mondo Disease Ontology slim</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'disease-slim'">
<h2 id="monarch-disease-ontology-slim">Monarch Disease Ontology slim</h2>
<p>A “slim” is a high-level subset of terms from an ontology, used to analyze sets of annotated genes. The table below shows a slim set of grouping terms that we have selected from the <a href="https://mondo.monarchinitiative.org/">Mondo Disease Ontology</a> (Mondo) and the number of annotations to each term. Mondo IDs link to PomBase ontology term pages. The annotation totals link to pages with information about the term and a list of annotated genes. A gene may be included in the annotation sets for more than one slim term.</p>
<p>You can download a list of current <a href="ftp://ftp.pombase.org/nightly_update/misc/disease_ontology_slim_ids_and_names.tsv">PomBase Mondo slim IDs and term names</a> from the PomBase FTP site.</p>
<p>If you would like to have any terms added to the PomBase Mondo slim set, please contact the <a href="mailto:helpdesk@pombase.org">helpdesk</a>.</p>
<p><app-slim-table [slimName]="'pombe_mondo_slim'"></app-slim-table></p>

  </div>
  <div *ngIf="pageName == 'dna-binding-sites'">
<h3 id="dna-binding-site-consensus-sequences-in-s.-pombe">DNA binding site consensus sequences in <em>S. pombe</em></h3>
<p>Also see <a href="https://www.pombase.org/jbrowse/?loc=III%3A725762..736349&amp;tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CConsensus%20transcription%20factor%20DNA%20binding%20motifs&amp;highlight=">PomBase JBrowse with binding site track enabled</a>.</p>
<h4 id="transcription-factor-binding-sites">Transcription factor binding sites</h4>
<table>
<thead>
<tr class="header">
<th>Site Name</th>
<th>Consensus Sequence</th>
<th>bound by</th>
<th>Sequence Ontology ID</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>AACCCT box</td>
<td>ATCA(C/A)AACCCTAACCCT</td>
<td><a href="https://www.pombase.org/gene/SPCC290.04">Ams2</a>, <a href="https://www.pombase.org/gene/SPAC13G7.10">Teb1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001901">SO:0001901</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17452352">PMID:17452352</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/4092687">PMID:4092687</a></td>
</tr>
<tr class="even">
<td>Ace2 UAS</td>
<td>CCAGCC</td>
<td><a href="https://www.pombase.org/gene/SPAC6G10.12c">Ace2</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001857">SO:0001857</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16678171">PMID:16678171</a></td>
</tr>
<tr class="odd">
<td>AP 1 binding site</td>
<td>TGACTCA</td>
<td><a href="https://www.pombase.org/gene/SPAC1783.07c">Pap1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001842">SO:0001842</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/1899230">PMID:1899230</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/3034432">PMID:3034432</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/3125983">PMID:3125983</a></td>
</tr>
<tr class="even">
<td>calcineurin-dependent response element (CDRE motif)</td>
<td>GNGGCKCA</td>
<td><a href="https://www.pombase.org/gene/SPAC4G8.13c">Prz1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001865">SO:0001865</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16928959">PMID:16928959</a></td>
</tr>
<tr class="odd">
<td>CCAAT motif</td>
<td>CCAAT</td>
<td><a href="https://www.pombase.org/gene/SPBC725.11c">Php2</a>, <a href="https://www.pombase.org/gene/SPAC23C11.08">Php3</a>, <a href="https://www.pombase.org/gene/SPBC16E9.01c">Php4</a>, <a href="https://www.pombase.org/gene/SPBC3B8.02">Php5</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001856">SO:0001856</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16963626">PMID:16963626</a></td>
</tr>
<tr class="even">
<td>copper-response element (CuRE)</td>
<td>HTHNNGCTGD</td>
<td><a href="https://www.pombase.org/gene/SPAC31A2.11c">Cuf1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001844">SO:0001844</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/10593913">PMID:10593913</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/9188496">PMID:9188496</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/9211922">PMID:9211922</a></td>
</tr>
<tr class="odd">
<td>CSL response element</td>
<td>GTGRGAA</td>
<td><a href="https://www.pombase.org/gene/SPCC736.08">Cbf11</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001839">SO:0001839</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19101542">PMID:19101542</a></td>
</tr>
<tr class="even">
<td>cyclic AMP response element (CRE)</td>
<td>TGACGTCA</td>
<td><a href="https://www.pombase.org/gene/SPBC29B5.01">Atf1</a>, <a href="https://www.pombase.org/gene/SPAC1783.07c">Pap1</a>, <a href="https://www.pombase.org/gene/SPAC21E11.03c">Pcr1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001843">SO:0001843</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/11483355">PMID:11483355</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/11483993">PMID:11483993</a></td>
</tr>
<tr class="odd">
<td>DNA damage response element (DRE)</td>
<td>CGWGGWNGMM</td>
<td><a href="https://www.pombase.org/gene/SPAC1B1.01">Deb1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001845">SO:0001845</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/11073995">PMID:11073995</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/8668127">PMID:8668127</a></td>
</tr>
<tr class="even">
<td>FLEX element</td>
<td>GTAAACAAACAAAM</td>
<td><a href="https://www.pombase.org/gene/SPBC16G5.15c">Fkh2</a>, <a href="https://www.pombase.org/gene/SPBC32H8.11">Mei4</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001846">SO:0001846</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/10747048">PMID:10747048</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/14871934">PMID:14871934</a></td>
</tr>
<tr class="odd">
<td>forkhead motif</td>
<td>TTTRTTTACA</td>
<td><a href="https://www.pombase.org/gene/SPBC4C3.12">Sep1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001847">SO:0001847</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15195092">PMID:15195092</a></td>
</tr>
<tr class="even">
<td>GATA box</td>
<td>WGATAR</td>
<td><a href="https://www.pombase.org/gene/SPCC1902.01">Gaf1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001840">SO:0001840</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/8321208">PMID:8321208</a></td>
</tr>
<tr class="odd">
<td>heat shock element (HSE)</td>
<td>NGAAN (at least 3 copies)</td>
<td><a href="https://www.pombase.org/gene/SPAC2E12.02">Hsf1</a>, <a href="https://www.pombase.org/gene/SPAC8C9.14">Prr1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001850">SO:0001850</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17347150">PMID:17347150</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/8689565">PMID:8689565</a></td>
</tr>
<tr class="even">
<td>homol D box</td>
<td>CAGTCACA (or inverted form TGTGACTG)</td>
<td><a href="https://www.pombase.org/gene/SPAC1142.08">Fhl1</a>, <a href="https://www.pombase.org/gene/SPBC336.09c">Rrn7</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001848">SO:0001848</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21673110">PMID:21673110</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/7501449">PMID:7501449</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/8458332">PMID:8458332</a></td>
</tr>
<tr class="odd">
<td>homol E box</td>
<td>ACCCTACCCT (or inverted form AGGGTAGGGT)</td>
<td><a href="https://www.pombase.org/gene/SPAC1142.08">Fhl1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001849">SO:0001849</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/7501449">PMID:7501449</a></td>
</tr>
<tr class="even">
<td>iron repressed GATA element</td>
<td>WGATAA</td>
<td><a href="https://www.pombase.org/gene/SPBC29B5.01">Atf1</a>, <a href="https://www.pombase.org/gene/SPAC23E2.01">Fep1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001851">SO:0001851</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/11956219">PMID:11956219</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/17211681">PMID:17211681</a></td>
</tr>
<tr class="odd">
<td>LOZ1 response element</td>
<td>CGNMGATCNTY</td>
<td><a href="https://www.pombase.org/gene/SPAC25B8.19c">Loz1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0002225">SO:0002225</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31515876">PMID:31515876</a></td>
</tr>
<tr class="even">
<td>mating type M-box</td>
<td>ACAAT</td>
<td><a href="https://www.pombase.org/gene/SPBC23G7.09">Mat1-Mc</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001852">SO:0001852</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/9233811">PMID:9233811</a></td>
</tr>
<tr class="odd">
<td>MluI cell cycle box (MCB)</td>
<td>ACGCGT</td>
<td><a href="https://www.pombase.org/gene/SPBC336.12c">Cdc10</a>, <a href="https://www.pombase.org/gene/SPBC725.16">Res1</a>, <a href="https://www.pombase.org/gene/SPAC22F3.09c">Res2</a>, <a href="https://www.pombase.org/gene/SPBC21B10.13c">Yox1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001855">SO:0001855</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16285853">PMID:16285853</a></td>
</tr>
<tr class="even">
<td>Pho7 binding site</td>
<td>[5’-TCG(G/C)(A/T)xxTTxAA]</td>
<td><a href="https://www.pombase.org/gene/SPBC27B12.11c">Pho7</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0002216">SO:0002216</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28811350">PMID:28811350</a></td>
</tr>
<tr class="odd">
<td>pombe cell cycle box (PCB)</td>
<td>GNAACR</td>
<td><a href="https://www.pombase.org/gene/SPBC4C3.12">Sep1</a>, <a href="https://www.pombase.org/gene/SPBC16G5.15c">Fkh2</a>, <a href="https://www.pombase.org/gene/SO:0001871">Mbx1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001871">SO:0001871</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/12411492">PMID:12411492</a></td>
</tr>
<tr class="even">
<td>Sap1 recognition motif</td>
<td>TARGCAGNTNYAACGMG</td>
<td><a href="https://www.pombase.org/gene/SPCC1672.02c">Sap1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001864">SO:0001864</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16166653">PMID:16166653</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/7651412">PMID:7651412</a></td>
</tr>
<tr class="odd">
<td>sterol regulatory element</td>
<td>ATCACCCCAC (and variants)</td>
<td><a href="https://www.pombase.org/gene/SPBC19C2.09">Sre1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001861">SO:0001861</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/11111080">PMID:11111080</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/16537923">PMID:16537923</a></td>
</tr>
<tr class="even">
<td>STREP motif</td>
<td>CCCCTC</td>
<td><a href="https://www.pombase.org/gene/SPAC6F12.02">Rst2</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001859">SO:0001859</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/11739717">PMID:11739717</a></td>
</tr>
<tr class="odd">
<td>TATA box</td>
<td>TATA(A|T)A(A|T)</td>
<td><a href="https://www.pombase.org/gene/SPBC13E7.10c">Brf1</a>, <a href="https://www.pombase.org/gene/SPAC29E6.08">Tbp1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0000174">SO:0000174</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16858867">PMID:16858867</a></td>
</tr>
<tr class="even">
<td>TR box</td>
<td>TTCTTTGTTY</td>
<td><a href="https://www.pombase.org/gene/SPBC32C12.02">Ste11</a>, <a href="https://www.pombase.org/gene/SPBC23G7.09">Mat1-Mc</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001858">SO:0001858</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/1657709">PMID:1657709</a></td>
</tr>
<tr class="odd">
<td>Zas1 recognition motif</td>
<td>(Y)CCCCAY</td>
<td><a href="https://www.pombase.org/gene/SPBC1198.04c">Zas1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0002215">SO:0002215</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29735745">PMID:29735745</a></td>
</tr>
<tr class="even">
<td>zinc repressed element</td>
<td>GNMGATC</td>
<td><a href="https://www.pombase.org/gene/SPAC25B8.19c">Loz1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0002006">SO:0002006</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24003116">PMID:24003116</a></td>
</tr>
</tbody>
</table>
<h4 id="other-dna-binding-sites">Other DNA binding sites</h4>
<table>
<thead>
<tr class="header">
<th>Site Name</th>
<th>Consensus Sequence</th>
<th>bound by</th>
<th>Sequence Ontology ID</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>ARS consensus sequence</td>
<td>WTTTAYRTTTW</td>
<td><a href="https://www.pombase.org/gene/SPBC1105.04c">Cbp1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0002004">SO:0002004</a></td>
<td></td>
</tr>
<tr class="even">
<td>GT dinucleotide repeat</td>
<td>(GT)n</td>
<td><a href="https://www.pombase.org/gene/SPAC30.03c">Tsn1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001862">SO:0001862</a></td>
<td></td>
</tr>
<tr class="odd">
<td>GTT trinucleotide repeat</td>
<td>(GTT)n</td>
<td><a href="https://www.pombase.org/gene/SPAC30.03c">Tsn1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001863">SO:0001863</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/">PMID:</a></td>
</tr>
<tr class="even">
<td>rDNA intergenic spacer element</td>
<td>AGGTAAGGGTAATGCAC</td>
<td><a href="https://www.pombase.org/gene/SPBC1198.11c">Reb1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001860">SO:0001860</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/9016645">PMID:9016645</a></td>
</tr>
<tr class="odd">
<td>telomeric repeat</td>
<td>TTAC(0-1)A(0-1)G(1-8)</td>
<td><a href="https://www.pombase.org/gene/SPAC16A10.07c">Taz1</a></td>
<td><a href="http://sequenceontology.org/browser/current_svn/term/SO:0001496">SO:0001496</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/8720065">PMID:8720065</a></td>
</tr>
</tbody>
</table>
<h4 id="notes">Notes</h4>
<p>SO IDs link to <a href="http://sequenceontology.org/browser/obob.cgi">MISO</a>, the <a href="http://sequenceontology.org/">Sequence Ontology</a> browser. Consensus sequences use the <a href="https://www.qmul.ac.uk/sbcs/iubmb/misc/naseq.html">IUBMB Nomenclature for Incompletely Specified Bases in Nucleic Acid Sequences</a>; briefly:</p>
<ul>
<li>R = purine (A or G)</li>
<li>Y = pyrimidine (C or T)</li>
<li>W = A or T</li>
<li>M = A or C</li>
<li>K = G or T</li>
<li>H = A, C or T</li>
</ul>

  </div>
  <div *ngIf="pageName == 'drugs-known-pombe-targets'">
<h3 id="drugs-with-known-s.-pombe-targets">Drugs with known <em>S. pombe</em> targets</h3>
<p>This table lists drugs that affect <em>S. pombe</em>, and summarizes what is known about their targets or mechanisms of action.</p>
<table>
<thead>
<tr class="header">
<th>Drug</th>
<th>Target gene(s)</th>
<th>Cellular target</th>
<th>Process target</th>
<th>Other</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:44975">1,10-phenanthroline</a></td>
<td></td>
<td>RNA polymerases II and III</td>
<td></td>
<td></td>
<td><a href="https://www.pombase.org/reference/PMID:27518095">PMID:27518095</a></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:65678">CT2108A</a></td>
<td></td>
<td>fatty acid synthase</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:65679">CT2108B</a></td>
<td></td>
<td>fatty acid synthase</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:37415">α-amanitin</a></td>
<td><a href="https://www.pombase.org/gene/SPBC28F2.12">rpb1</a></td>
<td>RNA polymerase II</td>
<td>transcription <a href="https://www.pombase.org/term/GO:0006367">initiation</a> and <a href="https://www.pombase.org/term/GO:0006368">elongation</a></td>
<td>interferes with a protein conformational change underlying the transcription mechanism</td>
<td><a href="https://www.pombase.org/reference/PMID:11805306">PMID:11805306</a></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:53785">alverine citrate</a></td>
<td></td>
<td></td>
<td>proteasome/lipid synthesis?</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:2639">amiloride</a></td>
<td><a href="https://www.pombase.org/gene/SPAC17A2.01">bsu1</a></td>
<td>plasma membrane importer</td>
<td><a href="https://www.pombase.org/term/GO:0006006">glucose metabolism</a></td>
<td></td>
<td><a href="https://www.pombase.org/reference/PMID:15701794">PMID:15701794</a>, <a href="https://www.pombase.org/reference/PMID:8431459">PMID:8431459</a></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:2682">amphotericin B</a></td>
<td></td>
<td>ergosterol binding</td>
<td><a href="https://www.pombase.org/term/GO:0016126">sterol biosynthesis</a></td>
<td>forms membrane pores</td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:338412">anisomycin</a></td>
<td></td>
<td></td>
<td><a href="https://www.pombase.org/term/GO:0002181">translation</a></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>antifungal azoles</td>
<td></td>
<td></td>
<td><a href="https://www.pombase.org/term/GO:0016126">sterol biosynthesis</a></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:2762">antimycin A</a></td>
<td></td>
<td>electron transport chain? cytochrome bc1 complex?</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>arborcandin C</td>
<td></td>
<td>1,3-beta-glucan synthase</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:3015">benomyl</a></td>
<td><a href="https://www.pombase.org/gene/SPBC26H8.07c">nda3</a></td>
<td></td>
<td>microtubule destabilizer</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:22907">bleomycins</a></td>
<td></td>
<td>fungal cell wall</td>
<td><a href="https://www.pombase.org/term/GO:0000917">septation</a>, <a href="https://www.pombase.org/term/GO:0000281">cytokinesis</a></td>
<td>gamma irradiation mimetic</td>
<td><a href="https://dornsife.usc.edu/pombenet/drugs/">Forsburg lab</a></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:52717">bortezomib</a></td>
<td></td>
<td>proteasome</td>
<td></td>
<td></td>
<td><a href="https://www.pombase.org/reference/PMID:25908789">PMID:25908789</a></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:48080">brefeldin A</a></td>
<td><a href="https://www.pombase.org/gene/SPBC211.03c">gea1</a></td>
<td></td>
<td>intracellular <a href="https://www.pombase.org/term/GO:0016192">vesicle-mediated transport</a></td>
<td>inhibits the GEFs for class II ARFs; blocks coat assembly and vesicle budding</td>
<td><a href="https://www.pombase.org/reference/PMID:27191590">PMID:27191590</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/1448152/">PMID:1448152</a> (PubMed link)</td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:50011">Calcofluor White</a></td>
<td></td>
<td></td>
<td><a href="https://www.pombase.org/term/GO:0009272">cell wall synthesis</a></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:27656">camptothecin</a></td>
<td><a href="https://www.pombase.org/gene/SPBC1703.14c">Top1</a></td>
<td>topoisomerase 1</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:609827">canavanine</a></td>
<td></td>
<td></td>
<td></td>
<td>toxic arginine analog</td>
<td></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:171741">cerulenin</a></td>
<td></td>
<td>β-keto-acyl ACP synthase, HMG-CoA synthase</td>
<td></td>
<td></td>
<td><a href="https://www.pombase.org/reference/PMID:30003614">PMID:30003614</a></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:17698">chloramphenicol</a></td>
<td></td>
<td></td>
<td><a href="https://www.pombase.org/term/GO:0002181">translation</a></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:34630">chlorpropham</a> (isopropyl N-3-chlorophenyl carbamate)</td>
<td>spindle</td>
<td></td>
<td></td>
<td>spindle poison</td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:133750">Cutin-1</a></td>
<td><a href="https://www.pombase.org/gene/SPAC926.09c">fas1</a></td>
<td>fatty acid synthase</td>
<td>nuclear envelope expansion during mitosis</td>
<td></td>
<td><a href="https://www.pombase.org/reference/PMID:26869222">PMID:26869222</a></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:27641">cycloheximideg</a></td>
<td></td>
<td>60S ribosomal subunit</td>
<td><a href="https://www.pombase.org/term/GO:0002181">translation</a></td>
<td>prevents release of deacetylated tRNA from the E site</td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:4031">cyclosporin A</a></td>
<td></td>
<td>calcineurin and cyclophilins</td>
<td></td>
<td>several cyclophilins described in <em>S. pombe</em></td>
<td><a href="https://www.pombase.org/reference/PMID:16134115">PMID:16134115</a></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:82642">dihydromotuporamine C</a></td>
<td></td>
<td>proteasome?</td>
<td>lipid synthesis?</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:57248">echinocandin</a></td>
<td></td>
<td></td>
<td><a href="https://www.pombase.org/term/GO:0009272">cell wall synthesis</a></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:50145">fenpropimorph</a></td>
<td></td>
<td>proteasome?</td>
<td>lipid synthesis?</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:44423">hydroxyurea</a></td>
<td><a href="https://www.pombase.org/gene/SPBC25D12.04">suc22</a></td>
<td>ribonucleotide reductase small subunit</td>
<td>slows DNA synthesis</td>
<td>causes replication fork stalling</td>
<td><a href="https://www.pombase.org/reference/PMID:27869662">PMID:27869662</a></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:16976">hygromycin B</a></td>
<td></td>
<td>40S ribosomal subunit</td>
<td><a href="https://www.pombase.org/term/GO:0002181">translation</a></td>
<td>interferes with translocation of tRNA from the A site to the P site of the ribosome</td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:6076">itraconazole</a></td>
<td>spindle</td>
<td></td>
<td></td>
<td>spindle poison</td>
<td></td>
</tr>
<tr class="odd">
<td>latrunculin <a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:49703">A</a> or <a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:49703">B</a></td>
<td></td>
<td></td>
<td><a href="https://www.pombase.org/term/GO:0030041">actin polymerization</a></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:52646">leptomycin B</a></td>
<td><a href="https://www.pombase.org/gene/SPAC1805.17">crm1</a></td>
<td></td>
<td><a href="https://www.pombase.org/term/GO:0051168">nuclear export</a></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:40303">lovastatin</a></td>
<td><a href="https://www.pombase.org/gene/SPCC162.09c">hmg1</a></td>
<td>HMG CoA-reductase (competitive inhibitor)</td>
<td><a href="https://www.pombase.org/term/GO:0006696">ergosterol biosynthesis</a></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:6852">methyl 5-hydroxy-2-benzimidazole carbamate</a></td>
<td></td>
<td>microtubules</td>
<td></td>
<td>reversible MT inhibitor</td>
<td></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:34848">mevastatin</a> (compactin)</td>
<td><a href="https://www.pombase.org/gene/SPCC162.09c">hmg1</a></td>
<td>HMG CoA-reductase (competitive inhibitor)</td>
<td><a href="https://www.pombase.org/term/GO:0006696">ergosterol biosynthesis</a></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:48041">microcystin</a></td>
<td></td>
<td>PP2A</td>
<td></td>
<td></td>
<td><a href="https://www.pombase.org/reference/PMID:29079657">PMID:29079657</a></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:27617">monensin</a></td>
<td></td>
<td>Golgi transmembrane Na+/H+ antiporter</td>
<td><a href="https://www.pombase.org/term/GO:0034220">ion transport</a></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:168396">mycophenolic acid</a></td>
<td><a href="https://www.pombase.org/gene/SPBC2F12.14c">gua1</a></td>
<td>IMP dehydrogenase</td>
<td></td>
<td>limits cellular GTP pools</td>
<td><a href="https://www.pombase.org/reference/PMID:11535588">PMID:11535588</a></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:7488">natamycin</a></td>
<td></td>
<td>ergosterol binding</td>
<td></td>
<td>blocks fungal growth without permeabilizing the membrane</td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:7660">nystatin</a></td>
<td></td>
<td>ergosta-5,7,22,24,(28)-tetraenol binding</td>
<td><a href="https://www.pombase.org/term/GO:0006696">ergosterol biosynthesis</a></td>
<td>antifungal</td>
<td></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:46442">orthovanadate</a></td>
<td></td>
<td>H(+)-ATPase</td>
<td></td>
<td></td>
<td><a href="https://www.pombase.org/reference/PMID:8431459">PMID:8431459</a></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:55374">oxazolidinones</a></td>
<td></td>
<td></td>
<td><a href="https://www.pombase.org/term/GO:0002181">translation</a></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:75044">phleomycin</a></td>
<td></td>
<td>nucleic acids</td>
<td></td>
<td>antibiotic; damages nucleic acids in the presence of iron when cells are grown aerobically</td>
<td><a href="https://www.pombase.org/reference/PMID:17724773">PMID:17724773</a></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:63618">pravastatin</a></td>
<td><a href="https://www.pombase.org/gene/SPCC162.09c">hmg1</a></td>
<td>HMG-CoA reductase</td>
<td><a href="https://www.pombase.org/term/GO:0006696">ergosterol biosynthesis</a></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:9168">rapamycin</a> (sirolimus)</td>
<td><a href="https://www.pombase.org/gene/SPBC839.17c">fkh1</a></td>
<td>TOR kinase (inhibitor)</td>
<td><a href="https://www.pombase.org/term/GO:0000278">cell cycle</a> progression</td>
<td><a href="https://www.pombase.org/reference/PMID:11335722">PMID:11335722</a></td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:8852">ricin</a></td>
<td></td>
<td>ribosome</td>
<td><a href="https://www.pombase.org/term/GO:0002181">translation</a></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>sanglifehrin A</td>
<td></td>
<td>cyclophilin binding</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>spliceostatin A</td>
<td></td>
<td>binds SF3b components</td>
<td><a href="https://www.pombase.org/term/GO:0008380">splicing</a></td>
<td></td>
<td><a href="https://www.pombase.org/reference/PMID:17961508">PMID:17961508</a></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:15738">staurosporine</a></td>
<td></td>
<td>protein kinase</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:61049">tacrolimus</a> (FK506)</td>
<td><a href="https://www.pombase.org/gene/SPBC839.17c">fkh1</a></td>
<td>TOR kinase (inhibitor)</td>
<td><a href="https://www.pombase.org/term/GO:0000278">cell cycle</a> progression</td>
<td></td>
<td><a href="https://www.pombase.org/reference/PMID:11335722">PMID:11335722</a></td>
</tr>
<tr class="odd">
<td>tagetitoxin (tagetin)</td>
<td><a href="https://www.pombase.org/gene/SPCC18.07">rpc53</a></td>
<td>RNA polymerase III</td>
<td></td>
<td><a href="https://books.google.co.uk/books?id=zYlqeJAi0igC&pg=PA173&lpg=PA173&dq=tagetitoxin+pombe&source=bl&ots=zENTjUISTy&sig=ACfU3U2_CsWzJPojjmBVgRvbvm3bi0x5lQ&hl=en&sa=X&ved=2ahUKEwjh0IDe75nnAhUHZMAKHcM7CV8Q6AEwBXoECFgQAQ#v=onepage&q=tagetitoxin%20pombe&f=false">RNA polymerases and associated factors (book)</a></td>
<td></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:41774">tamoxifen</a></td>
<td><a href="https://www.pombase.org/gene/SPBC29A10.01">ccr1</a></td>
<td>NADPH-cytochrome p450 reductase</td>
<td></td>
<td>affects cell wall integrity</td>
<td><a href="https://www.pombase.org/reference/PMID:32571823">PMID:32571823</a></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:9448">terbinafine</a></td>
<td><a href="https://www.pombase.org/gene/SPBC713.12">erg1</a></td>
<td>squalene monooxygenase</td>
<td><a href="https://www.pombase.org/term/GO:0016126">sterol biosynthesis</a></td>
<td></td>
<td><a href="https://www.pombase.org/reference/PMID:32320462">PMID:32320462</a></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:497734">thialysine</a></td>
<td></td>
<td></td>
<td></td>
<td>naturally occurring, toxic lysine analog</td>
<td></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:">torin1</a></td>
<td><a href="https://www.pombase.org/gene/SPBC30D10.10c">tor1</a>, <a href="https://www.pombase.org/gene/SPBC216.07c">tor2</a></td>
<td></td>
<td><a href="https://www.pombase.org/term/GO:0031929">TOR signaling</a></td>
<td></td>
<td><a href="https://www.pombase.org/reference/PMID:24424027">PMID:24424027</a></td>
</tr>
<tr class="even">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:46024">trichostatin A</a> (TSA)</td>
<td><a href="https://www.pombase.org/gene/SPBC800.03">clr3</a>, <a href="https://www.pombase.org/gene/SPBC36.05c">clr6</a>, <a href="https://www.pombase.org/gene/SPAC3G9.07c">hos2</a></td>
<td>histone deacetylase</td>
<td></td>
<td></td>
<td><a href="https://www.pombase.org/reference/PMID:19723888">PMID:19723888</a></td>
</tr>
<tr class="odd">
<td><a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:29699">tunicamycin</a></td>
<td></td>
<td></td>
<td><a href="https://www.pombase.org/term/GO:0006487">N-glycosylation</a></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>zaragozic acid</td>
<td></td>
<td></td>
<td><a href="https://www.pombase.org/term/GO:0006487">N-glycosylation</a></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>

  </div>
  <div *ngIf="pageName == 'fission-yeast-bp-go-slim-terms'">
<h2 id="fission-yeast-go-biological-process-slim">Fission yeast GO Biological Process slim</h2>
<p>A “GO slim” is a subset of the Gene Ontology terms selected for a specific purpose in interpreting large-scale data, such as functional annotation of a genome or high-throughput experimental results. PomBase uses a GO slim to provide a simple summary of <em>S. pombe’s</em> biological capabilities by grouping gene products using broad classifiers.</p>
<p>The table below shows terms in the current fission yeast biological process GO slim, and the number of annotations to each term. GO IDs link to PomBase <a routerLink="/documentation/ontology-term-page">ontology term pages</a>. Icons beside each GO term link to <a href="http://www.esyn.org/">esyN</a>, which provides a graphical view of interactions for the genes from the PomBase <a routerLink="/documentation/high-confidence-physical-interaction-network">High Confidence Physical Interaction Network (HCPIN)</a> dataset. Only the subset of genes linked into the interaction network will be displayed in the esyN network view. The annotation totals link to pages with information about the term and a list of annotated genes.</p>
<p>Further information is available from the <a href="documentation/pombase-go-slim-documentation">PomBase GO slim documentation</a> and additional pages linked there. You can also download a list of current <a href="ftp://ftp.pombase.org/nightly_update/misc/bp_goslim_pombe_ids_and_names.tsv">GO process slim IDs and term names</a> from the PomBase FTP site.</p>
<!-- <a href="http://www.esyn.org/tutorial.html">esyN documentation</a> -->
<!--
Note that both proteins and RNAs can be annotated to GO terms, and the
tables on this page include annotated RNAs. For some GO terms, notably
'cytoplasmic translation', RNAs make up a significant proportion of
the total annotated genes.
-->
<p><app-slim-table [slimName]="'bp_goslim_pombe'"></app-slim-table></p>

  </div>
  <div *ngIf="pageName == 'fission-yeast-cc-go-slim-terms'">
<h2 id="fission-yeast-go-cellular-component-slim">Fission yeast GO Cellular Component slim</h2>
<p>A “GO slim” is a subset of the Gene Ontology terms selected for a specific purpose in interpreting large-scale data, such as functional annotation of a genome or high-throughput experimental results. PomBase uses GO slims to provide a simple summary of <em>S. pombe’s</em> biological capabilities by grouping gene products using broad classifiers.</p>
<p>The table below shows terms in the current fission yeast cellular component GO slim, and the number of annotations to each term. GO IDs link to PomBase <a href="/documentation/ontology-term-page">ontology term pages</a>. The annotation totals link to pages with information about the term and a list of annotated genes.</p>
<p>Further information is available from the <a href="documentation/pombase-go-slim-documentation">PomBase GO slim documentation</a> and additional pages linked there. You can also download a list of current <a href="ftp://ftp.pombase.org/nightly_update/misc/cc_goslim_pombe_ids_and_names.tsv">GO component slim IDs and term names</a> from the PomBase FTP site.</p>
<p><app-slim-table [slimName]="'cc_goslim_pombe'"></app-slim-table></p>

  </div>
  <div *ngIf="pageName == 'fission-yeast-go-slimming-tips'">
<h3 id="pombase-go-slim-usage-tips">PomBase GO Slim Usage Tips</h3>
<h4 id="general-go-slim-hints">General GO Slim hints</h4>
<p>These points will help you understand GO slims, and highlight some features of the fission yeast slim terms and annotations.</p>
<ol type="1">
<li>Note that the counts in the “Genes” column of the GO slim tables are not additive, because many gene products are annotated to multiple terms.</li>
<li>It not possible to create a slim with no overlaps between terms. Although the <em>S. pombe</em> slim has been defined to include biologically informative terms, and minimise overlaps between terms, large overlaps cannot be entirely avoided. For example, most of the gene products annotated to <em>signal transduction</em> are also annotated other terms in the slim.</li>
<li>As a consequence of points 1 and 2, GO slim annotation summaries should not be presented using pie charts. Although a pie chart could show the fraction of total annotations for any slim term, it can too easily be mistaken for the fraction of total annotated gene products, which is not the same.</li>
<li>It is difficult to define a slim which includes all annotated gene products without including terms with very small numbers of annotations (for example, the peroxisome organization branch has very few annotations), or very high level terms which are not particularly biologically informative (e.g. cellular process). Because we have opted not to include such terms, some gene products are annotated to process terms but do not appear in the slim annotation set.</li>
<li>Bear in mind that both proteins and RNAs can be annotated to GO terms. If you are working only with proteins you will need to make adjustments for this. For example, many tRNAs and rRNAs are annotated to cytoplasmic translation, and are therefore included in the slim set.</li>
<li>There is a difference between “unknown” and “unannotated”. All fission yeast and budding yeast gene products have been assessed and are classed as “unknown” for biological process if no biological process information is found (experimental or inferred). If you are making comparisons with other organisms, remember that it is possible that not all gene products have been assessed and that the “unknown” set is underestimated.</li>
<li>The default S. pombe slim includes all evidence codes for fission yeast. The evidence code <strong>IEA</strong> (inferred from electronic annotation) is often considered to be less accurate than other evidence codes, but it is very useful for increasing the coverage of some of the high level GO terms. Accurate annotation counts for some terms currently depends on including this evidence code (for example, there are 16 gene products annotated to transmembrane transport with IEA evidence, which are not yet covered by a manual annotation). For fission yeast, the IEA annotations improve slim coverage, but only represent a small number of annotations (519 biological process annotations as of May 2020), and have a low rate of false positives. We therefore recommend that you include them.</li>
</ol>
<h4 id="creating-a-user-defined-slim">Creating a user-defined slim</h4>
<ol type="1">
<li>You can create your own slim, or retrieve slim annotations for a gene set, using online slimming tools such as the <a href="http://go.princeton.edu/cgi-bin/GOTermMapper">GOTermMapper</a> at Princeton or <a href="https://www.ebi.ac.uk/QuickGO/slimming">QuickGO Explore Biology</a> at EBI (note that QuickGO requires UniProtKB accessions; PomBase IDs will not work).</li>
<li>When creating a slim for the entire genome, you should try to ensure that it covers as many annotated genes in your set as possible (see #3 in the list above). You should be aware of how many genes are annotated but not in your slim, and how many are “unknown” (i.e., annotated only to the root node; see #5 above).</li>
<li>For display purposes you usually want to keep the number of terms as small as possible to convey your results. However, you should ensure that the terms you include are specific enough to capture biologically relevant information. Many terms (e.g. metabolic process (3229 annotations), cellular process (4581 annotations)) are too general for the purpose of most slim-based analyses.</li>
<li>On a related note, if you are using your slim for data analysis (e.g. to summarize an enrichment), you should ensure that the terms are specific enough to demonstrate their relevance to the biological topic of interest. For example, lumping all genes involved in transport my mask overrepresentation of transmembrane transport vs. underrepresentation of vesicle-mediated transport in your results set, so you need to ensure that the slim has categories to represent your results effectively.</li>
<li>Most current implementations of software to create “GO slims” include the <em>regulates</em> relationship by default, so that (for example) genes involved in <em>regulation of cytokinesis</em> will be included with the set of genes annotated to <em>cytokinesis</em>. See the <a href="http://geneontology.org/docs/ontology-relations/">GO Ontology Relations documentation</a> for further information about relationships in GO. The annotation totals presented for the <a href="browse-curation/fission-yeast-bp-go-slim-terms">PomBase default <em>S. pombe</em> process slim</a>, by contrast, are calculated explicitly including the genes which are involved in a process via regulation only. We expect the distinction to be available in future versions of slimming software.</li>
</ol>
<!-- restore commented-out version when ticket 1521 done
5.  Most current implementations of software to create "GO slims"
    include the *regulates* relationship by default, so that (for
    example) genes involved in *regulation of cytokinesis* will be
    included with the set of genes annotated to *cytokinesis*. See the
    [GO Ontology Relations
    documentation](http://geneontology.org/docs/ontology-relations/)
    for further information about relationships in GO. The annotation
    totals presented for the <a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">PomBase default *S. pombe* process slim</a>, by contrast,
    are calculated both explicitly including or excluding the genes
    which are involved in a process via regulation only. We expect this
    distinction to be available in future versions of slimming software.
-->

  </div>
  <div *ngIf="pageName == 'fission-yeast-mf-go-slim-terms'">
<h2 id="fission-yeast-go-molecular-function-slim">Fission yeast GO Molecular Function slim</h2>
<p>A “GO slim” is a subset of the Gene Ontology terms selected for a specific purpose in interpreting large-scale data, such as functional annotation of a genome or high-throughput experimental results. PomBase uses GO slims to provide a simple summary of <em>S. pombe’s</em> biological capabilities by grouping gene products using broad classifiers.</p>
<p>The table below shows terms in the current fission yeast molecular function GO slim, and the number of annotations to each term. GO IDs link to PomBase <a href="/documentation/ontology-term-page">ontology term pages</a>. The annotation totals link to pages with information about the term and a list of annotated genes.</p>
<p>Further information is available from the <a href="documentation/pombase-go-slim-documentation">PomBase GO slim documentation</a> and additional pages linked there. You can also download a list of current <a href="ftp://ftp.pombase.org/nightly_update/misc/mf_goslim_pombe_ids_and_names.tsv">GO function slim IDs and term names</a> from the PomBase FTP site.</p>
<p><app-slim-table [slimName]="'mf_goslim_pombe'"></app-slim-table></p>

  </div>
  <div *ngIf="pageName == 'fission-yeast-phenotype-ontology'">
<h2 id="fission-yeast-phenotype-ontology">Fission Yeast Phenotype Ontology</h2>
<p>PomBase curators develop and use the Fission Yeast Phenotype Ontology (FYPO) to annotate phenotypes of mutant alleles and of over- or under-expressing wild type genes.</p>
<p>The contents and structure of FYPO are described on the <a href="http://curation.pombase.org/pombase-trac/wiki/FissionYeastPhenotypeOntology">FYPO page of the PomBase wiki</a> and in the publication:</p>
<p>Harris MA, Lock A, Bähler J, Oliver SG, Wood V. FYPO: The Fission Yeast Phenotype Ontology. Bioinformatics. 2013 July 1;29(13): 1671–1678. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23658422">PubMed abstract (PMID:23658422)</a> or <a href="http://bioinformatics.oxfordjournals.org/content/29/13/1671.long">Full text at Bioinformatics</a></p>
<h3 id="browsing-fypo-terms">Browsing FYPO terms</h3>
<p>At present, FYPO is included collections such as the National Center for Biomedical Ontology’s <a href="http://bioportal.bioontology.org/">BioPortal</a> and EBI’s <a href="http://www.ebi.ac.uk/ontology-lookup/">Ontology Lookup Service (OLS)</a>, both of which allow searching and browsing.</p>
<ul>
<li>BioPortal <a href="http://bioportal.bioontology.org/ontologies/FYPO">FYPO summary page</a></li>
<li>BioPortal <a href="http://bioportal.bioontology.org/ontologies/FYPO/?p=classes">FYPO terms</a> (the term display supports tree-viewer browsing, text searches, and a graphical display)</li>
<li><a href="http://www.ebi.ac.uk/ontology-lookup/browse.do?ontName=FYPO">OLS FYPO browser</a></li>
</ul>
<p>We hope to deploy a browser at PomBase in the not-too-distant future that will include annotations as well as terms.</p>
<h3 id="finding-fypo-terms-and-annotations-in-pombase">Finding FYPO terms and annotations in PomBase</h3>
<p>FYPO terms are displayed on PomBase gene pages, along with supporting evidence and allele and expression details, as described in the PomBase gene page documentation.</p>
<p>The PomBase advanced search can use either term names or IDs to search FYPO, and returns a list of genes that have alleles annotated to the specified term or any of its descendants. For example, see the <a routerLink="/faq/can-i-get-list-essential-pombe-genes">FAQ on finding essential genes</a>.</p>
<h3 id="curating-phenotypes-with-fypo">Curating phenotypes with FYPO</h3>
<p>The PomBase online curation tool, <a href="http://curation.pombase.org/pombe">Canto</a>, uses FYPO terms for annotation of S. pombe phenotypes. <a href="http://curation.pombase.org/pombe/docs/fypo_annotation">Phenotype curation documentation</a> is available via Canto.</p>
<h3 id="submitting-bulk-data">Submitting Bulk Data</h3>
<p>If you have a large set of phenotype data to submit, you may want to do a bulk submission. See the documentation on the <a href="http://www.pombase.org/submit-data/phenotype-data-bulk-upload-format">recommended file format</a>, and use the Phenotype Data Submission Form.</p>

  </div>
  <div *ngIf="pageName == 'index'">
<h2 id="browse-curation">Browse curation</h2>
<ul>
<li><a routerLink="/browse-curation/fission-yeast-phenotype-ontology">Fission Yeast Phenotype Ontology</a>: Summary page and links to additional information</li>
<li>GO slim annotation summaries
<ul>
<li><a routerLink="/browse-curation/fission-yeast-mf-go-slim-terms">Molecular function GO slim</a>: terms and annotation totals</li>
<li><a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">Biological process GO slim</a>: terms and annotation totals</li>
<li><a routerLink="/browse-curation/fission-yeast-cc-go-slim-terms">Cellular component GO slim</a>: terms and annotation totals</li>
</ul></li>
<li><a routerLink="/browse-curation/disease-slim">Disease association overview</a>: Fission yeast Mondo slim terms and annotation totals</li>
<li><a routerLink="/browse-curation/dna-binding-sites">DNA binding sites</a>: Consensus binding sequences found in <em>S. pombe</em></li>
<li><a routerLink="/browse-curation/drugs-known-pombe-targets">Drug targets</a>: List of drugs known to affect <em>S. pombe</em>, and their targets or mechanism of action</li>
</ul>

  </div>
  </div>
</div>
<div *ngIf="section == 'community'">
<app-page-contents-menu title="Community" titleRoute="/community">

<div class="left-menu-part left-menu-item"><a routerLink="/community/fission-yeast-community-curation-project">Community curation project</a></div>
<div class="left-menu-part left-sub-menu-item"><a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">Mailing list</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/community/internet-resources">External resources</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == '2006-survey-q16'">
<h3 id="genedb-s.-pombe-survey-question-16-responses">2006 GeneDB <em>S. pombe</em> survey, question 16 responses</h3>
<p><strong>If you could add one feature to the database immediately, or improve one feature immediately, what would it be?</strong></p>
<ol>
<li>Location of the genes based on their chromosomal positions.</li>
<li>sequence variation in S. pombe related strains</li>
<li>Links to S. cerevisiae and other eukaryotic databases for orthologues etc.</li>
<li>Adding protein/gene interaction data. Either biochemical or from synthetic interaction studies.</li>
<li>-</li>
<li>Improve the link to microarrays results to obtain an easier view of the microarrays results, including the mutant strains</li>
<li>Sequences of upstream 1000bp + gene sequences + downstream 1000bp</li>
<li>Epigenome information!!</li>
<li>Improve the gene search tool by including full-text genepage search. Add transcriptional regulatory motifs of genes.</li>
<li>Promoter Motif Finder</li>
<li>Add phenotypes.</li>
<li>direct link to orthologs</li>
<li>mainly I want to know the best homolgues in S. cerevisiae, C.elegans and mammals.</li>
<li>mutant and deletion phenotypes</li>
<li>as indicaeted above</li>
<li>The web page needs to be more attractive.</li>
<li>More information on the gene functions curated from the literature.</li>
<li>the sited literature for a specific gene.</li>
<li>transcription start and stop</li>
<li>Link to find orthologs of my pombe gene of interest in other species directly from the &quot;Gene Page.&quot;</li>
<li>Add phylogenetic trees</li>
<li>I want to download the datasets for ORFs and chromosomal elements in a format that I can directly import into a relational database such as MS Access. Any tab-delimited format is fine, but the ORF start and stop on the chromosome coordinates AND the gene aliases (or blank spaces) need to be linkable and to import consistantly. The current format is unusable because, for example, the names in some lines are not tab-delimited spaces in all of the other lines where no name exists. The unique identifier for all elements needs to be on all tables. We need such information to do our own whole-genome searches and mutageneses.</li>
<li>looks easier</li>
<li>3&#39; UTR data</li>
<li>Quick access to a specific gene in the choromosome map. Quick access to the cosmid sequence that contains a specific gene.</li>
<li>The Gene Description would be more informative. Especially when you use the LIST Download feature.</li>
<li>genome browser - needs to be more user friendly</li>
<li>Making it easy to access to available resources such as mutant strains, plasmids for gene expression; probably by adding links to cooperative researchers or institutions (for example, NBRP in Japan http://yeast.lab.nig.ac.jp/nig/english/index.html).</li>
<li>useful</li>
<li>Finding orthologues. Integrating orthiologue&not;&yen;s information Comparing two or more genes for expression profiles or other features</li>
<li>I like SGD very much</li>
<li>Make it more easy to identify a query sequence by Blast. You often get the name of a cosmid, instead of teh name of the gene</li>
<li>Microarray datasets</li>
<li>improved more user-friendly motif searching</li>
<li>A tool tp search protein-protein interaction database</li>
<li>addition of a function: mapping of two genes, quick calculation of intergenic distance between linked genes.</li>
<li>the information from the sequencing for plasmid vs cosmid is confusing. e.g. you find your gene and blast search a region of it within the pombe sequence, for example to check that a primer only binds to your gene and not elsewhere. the ID for the hits (which should be 100% matching with your gene) are the plasmid numbers which don&#39;t correspnd to the cosmid number which makes things a bit confusing and requires you to browse through all the balst hits (e.g. the systematic gene name - SPCC###.##). Maybe when you see results of the blast the systematic gene names/cosmid could be immediately visible rather than the plasmid name?</li>
<li>better links to other orthologs ie. human</li>
<li>Don&#39;t know</li>
<li>interacting factors if known (genetic and physical)</li>
<li>FASTA format of sequence data not punctuated by space</li>
<li>email addresses of experts for the gene</li>
<li>Phenotypes of the mutant</li>
<li>Improve Gene Ontology Annotaions. It is often not trivial to track down why a specific term is associated with a gene. Thi smay just be my lack of knowledge about how to correctly use the resource.</li>
<li>gene boundary: where on the chromosome a gene start and where it ends (promoter and transcription termination signal).</li>
<li>phenotype information</li>
<li>Two things actually: 1. To have chromosomal position along with contig position. 2. Consolidated information about interacting partners and/or functional partners.</li>
<li>Ability to search for partial systematic ORF names</li>
<li>protein- protein interaction</li>
<li>more clear identification of physical distances between pairs of genes</li>
<li>Add transcriptional regulatory regions of genes</li>
<li>qq</li>
<li>Overview of UTRs</li>
<li>Direct links to major publications for genes.</li>
<li>microarray data</li>
<li>show orthologs with % similarity and sequence alignments</li>
<li>transcript start/stop</li>
<li>add a clustal multiple alignment tool, to compare all orthologues of a given fission yeast gene.</li>
<li>Full text literature annotation with references to all mutant alleles of a gene and phenotypes of the mutants (often a paper contains information on mutant alleles that is not reflected in the abstract) note: There are many features I consider a very close second in priority.</li>
<li>Easy access to genome/proteome interaction studies</li>
<li>otholog sequence alignment, such as clustal and shown as box-shadowing type.</li>
<li>description of mutant phenotype and available alleles, description of localisation</li>
<li>It would be very helpful to have a section on published mutants for each gene along with a reference. Often I would like to know what kind of mutations of a particular gene have been constructed and what their phenotypes are. If there was a brief compilation of that data in one place it would make GeneDB much better.</li>
<li>Link the gene names that appear in Blast searches to the genedb page for that gene so that the info on the gene is jus a click away.</li>
<li>List Download</li>
<li>I find the graphic interface of amiGO to be very difficult to use. I really have never found it to be useful. It also has mistakes in process names. For example it refers to the transposons as Ty1 not the true name Tf1.</li>
<li>ability to download gene sequence data from the gene page, plus the ability to add a specified distance of the sequence either side of the gene. For example, retrieving sequence data for gene Y plus 0.5 Kb of sequence either side.</li>
<li>something that alows me to look at at gene in the genome, then click on said gene and expending it into its DNA sequence along with adjacent sequence</li>
<li>It would be helpful to be able to get sequence of gene plus bases up and downstream, as with SGD, instead of having to look through the cosmid library</li>
<li>protein interactions</li>
<li>x</li>
<li>Direct access link to PombePD under some form of licensing or one off fee pricing model (although nocost is obviously the ideal).</li>
<li>blast features, display of results in the form of map like in cerevisiae geome project</li>
<li>Mutant phenotypes and protein interaction data</li>
<li>to become like pombePD</li>
<li>Direct link to NCBI/Entrez</li>
<li>Protein regulation</li>
<li>(1)possibility of easily carrying out certain predictions manually, like prediction of subcellular localisation, introns, homologues. This would enable the user to test how good the predictions are and if there are alternative possibilities. (2)facility for getting a list of genes with similar expression profiles ; also more DNA microarray data (e.g. DNA damage) (3) better (i.e. more complex) facility for searching the whole set of genes: e.g. enable the user to specify a number of &#39;ideal&#39; criteria of a gene/protein (combination of pI, size, expression profile, protein domains, kinase motifs, other sequence motifs, predicted subcellular localisation), which would then return a set of genes/proteins that best fulfill these criteria, without necessarilly fulfilling all of them. Such a feature would be very useful in the identification of very distant homologues that are not easily identified in Blast searches.</li>
<li>More informations on phenotypes</li>
<li>Community annotation/discussion</li>
<li>easier access to batch downloads of gene/protein information and more information on what it exactly is (how it was constructed)</li>
<li>I find it very good. For my purposes all the available features are already a lot. I understand however the need for supporting curation of annotation and constant update. This of cours requires more than one curator and appropriate funding.</li>
<li>Phenotype data</li>
<li>Phenotypes of deletion mutants</li>
<li>Search information by gene list</li>
<li>SDS PAGE mobility</li>
<li>No comments</li>
<li>Complete list of phenotypes</li>
<li>no</li>
<li>metabolic pathway maps tuned to yeast</li>
<li>Position of the introns into the sequence add as colored letters</li>
<li>-</li>
<li>I would like to wish to get also the information of the S.cerevisiae gene name of the respective gene</li>
<li>To easy up data down-loads: where to find what</li>
<li>interactions (Y 2 H) IP info for each gene genetic interactions (null, slow growth)</li>
<li>An end-to-end searchable gene map of each chromosome (or contig), so that I can easily see what genes are closely linked to my gene-of-interest.</li>
<li>Make it easier</li>
<li>--</li>
<li>integrated resource for accessing all microarray data</li>
<li>Completion of the genome sequence, with linking of all contigs and addition of telomeres.</li>
<li>somewhat more details on the phenotype</li>
<li>Add mutant phenotype description on each Gene Page</li>
<li>Annotate phenotypes associated with a given gene (deletion, mutants, overexpression, doublemutants,..)</li>
<li>clear literature summary</li>
<li>Phenotypes and lit</li>
<li>IDs of and links to physical and genetic interaction partners.</li>
<li>Results of prerun BLAST against other organisms with links built in to other publicly available databases.</li>
<li>don&#39;t know</li>
<li>Provide the ORF+ 1KB upstream and 1 Kb downstream of genes</li>
<li>I would be useful to have the identity/similarity % with homologs of each gene compared to other organisms</li>
<li>phenotypes, and genetic and physical interactions as is found in SGD and YPD would make the database hugely more powerful.</li>
<li>when we could access PombePD the cross referencing between orthologues in Worms, humans and cerevisiae was so useful. YOGY has been great but it would be good if it was linked into Wormbase, SGD more so you could move between Gene pages in different organisms more quickly.</li>
<li>Web services for better integration with BioPerl and automated scripts in general.</li>
<li>Description of homologues in higher organisms.</li>
<li>Information on transciption start and stop sites and promoter elements.</li>
<li>No Thanks</li>
<li>more microarray datasets</li>
<li>mutant gene phenotype and genetic interactions</li>
<li>Better hyperlinks to look at orthologues in other organisms</li>
<li>Having easy access to information about regulatory motifs and promoters for each gene.</li>
<li>phosphorylation site predictor</li>
<li>Integrating GBrowse with expression data.</li>
<li>Probably better links to as much as possible of published literature relating to the gene, so one can read the details of a gene of interest when the gene name is not mentioned explicitly in the abstract of the paper. however, this could be huge for some genes such as cdc2.</li>
<li>I would change the genome browser to be based on chromosomal coordinates rather than contig coordinates. Right now I&#39;m using the &quot;chromosome&quot; assembly of the genome. Since your implementation of gbrowse does not allow genome browsing based on chromosomal coordinates, I&#39;ve been using this one that does: http://fungal.genome.duke.edu/cgi-bin/gbrowse/spom/ . I understand that the chromosomal coordinates may not be as precise, but if they&#39;re good enough to create chromosome assembly files, why not just allow people to access the genome that way as well? I think that&#39;s what most people have come to expect from genome browsers elsewhere.</li>
<li>Add a Candida albicans GeneDB and a Cryptococcus neoformans GeneDB.</li>
<li>curated (or non-curated) interaction grids genetic, physical, text mining based. with links to data</li>
<li>identification of genes with human orthologues especially causing disease</li>
<li>Non-coding sequences, particularly replication origins.</li>
<li>Identification of transcription units.</li>
<li>Be able to get alignments of my favorite gene with the closest ortologs directly</li>
<li>To show the homolog names in other model organisms</li>
<li>Direct X-ref to SGD</li>
<li>Graphic Interface</li>
<li>More GO curation</li>
<li>UTR data for genes</li>
</ol>

  </div>
  <div *ngIf="pageName == '2006-survey-q17'">
<h3 id="genedb-s.-pombe-survey-question-17-responses">2006 GeneDB <em>S. pombe</em> survey, question 17 responses</h3>
<p><strong>How have the data and curation helped your research? Please provide specific examples and publications if possible. These examples will be helpful to secure future funding.</strong></p>
<ol>
<li>&quot;Published expression profiles&quot; has been very useful and informative.</li>
<li>It helped me to study some genome wide approche.</li>
<li>Vey useful and crucial for our reserach, especially downloads for microarray work, browsers, GO terms etc and also gene name issues</li>
<li>Invaluable to most of my research. Example publications which have relied on this: Aves SJ et al. (2002) The mei3 region of the Schizosaccharomyces pombe genome. Yeast 19, 521-527 Hunt C et al. (2001) Subtelomeric sequence from the right arm of Schizosaccharomyces pombe chromosome 1 contains seven permease genes. Yeast 18, 355-361 Xiang Z et al. (2000) The mating-type region of Schizosaccharomyces pombe h-S 972: sequencing and analysis of 69 kb including the expressed mat1 locus. Yeast 16, 1061-1067 Xiang Z et al. (2000) Analysis of 114 kb of DNA sequence from fission yeast chromosome 2 immediately centromere-distal to his5. Yeast 16, 1405-1411</li>
<li>Helpful on a daily basis-I can&#39;t imagine trying to work without it!</li>
<li>Vital for gene searches.</li>
<li>-</li>
<li>I used this data to choose gene as probe for the northern blot, to choose oligos for the PCRs, to have a general view of the phenotypes in mutants strains, to confirm, or not, previous results.</li>
<li>1. Search for S. cerevisiae orthologues 2. pubilished literatures of interesting genes 3. News of S. pombe community (e.g. conferences)</li>
<li>We searched the genedb database for genes with no homologs in cerevisiae as candidates for RNAi mediated chromatin modification machinery and are currently knocking them out and assaying their effect.</li>
<li>web sites</li>
<li>Helped to understand my data and identify links to homologs/orthologs in other species, especially budding yeast.</li>
<li>The genome wide deletion project has used the annotated sequence ( Wood et al Nature 2002) as the basis for constructing the deletions. The deletion database is linked to gene DB and the bioinformatic tools will be used for analysis of the deletion dataset</li>
<li>For our research we use the pombe gene DB everyday, we use the SMART program, Fasta, consult the expression profiles from the microarrays etc. Tajadura, V., Garc&radic;&ne;a, B., Garc&radic;&ne;a, I., Garc&radic;&ne;a, P., and S&radic;&deg;nchez, Y. (2004) &quot; Schizosaccharomyces pombe Rgf3p is a specific Rho1-GEF that regulates cell wall b-glucan biosynthesis through the GTPase Rho1p? Journal of Cell Science 117: 6163-6174 Garc&radic;&ne;a-Cort&radic;&copy;s, J. C., Carnero, E., Ishiguro, J., S&radic;&deg;nchez, Y., Duran, A. and Ribas, J. C.(2005) &quot;The novel fission yeast (1,3)-a-D-glucan synthase catalityc subunit Bgs4p is essential during both cytokinesis and polarized growth ? Journal of Cell Science 118: 157-174 Garc&radic;&ne;a, I., Jim&radic;&copy;nez, D., Mart&radic;&ne;n, V., Dur&radic;&deg;n, A. S&radic;&deg;nchez, Y. (2005) &quot; Agn1p, a putative a-1,3 glucanase, is required for cell separation in Schizosaccharomyces pombe? Biology of the Cell 97: 569-576 Garc&radic;&ne;a, I., Mart&radic;&ne;n, V., Tajadura, V., Toda, T. and S&radic;&deg;nchez, Y. (2006) &quot;Synthesis of a?glucans in fission yeast spores is carried out by three a-glucan synthase paralogs, Mok12p, Mok13p and Mok14p? Mol Microbiol. 59:836-853. Garc&radic;&ne;a, P., Tajadura, V., Garc&radic;&ne;a, I., and S&radic;&deg;nchez, Y. (2006) ?Rgf1p is a specific Rho1-GEF that coordinates cell polarization with cell wall biogenesis in fission yeast? Molecular Biology of the Cell. 17:1620-1631</li>
<li>identification and structural analyses of putative genes ...</li>
<li>I use genedb to access sequence information easily which I can then use to design primers/constructs for my research. I also use it to obtain information quickly about genes that I am interested in such as the motif information.</li>
<li>cloned a gene. testing linkage.</li>
<li>The database is an essential part of our &quot;gene discovery&quot; studies. Examples in which the database has facilliated our studies include: 1. Martin, V., Chahwan, C., Gao, H., Blais, V., Wohlschlegel, J., Yates 3rd, J.R., McGowan, C.H., Russell, P. (2006) Sws1 is a conserved regulator of homologous recombination in eukaryotic cells. EMBO J. in press. 2. Mart&radic;&ne;n, V., Rodr&radic;&ne;guez-Gabriel, M.A., McDonald, W. H., Watt, S., Yates 3rd , J.R., B&radic;&sect;hler, J., Russell, P. (2006) Cip1 and Cip2 are novel RNA-Recognition-Motif proteins that counteract Csx1 function during oxidative stress. Mol. Biol. Cell. 17, 1176-1183. 3. Noguchi, E., Noguchi, C., McDonald, W. H., Yates, J. R. 3rd,, Russell, P. (2004) Swi1 and Swi3 are components of a replication fork protection complex in fission yeast. Mol. Cell. Biol. 24, 8342-8355. 4. Saitoh, S., Chabes, A., McDonald, W. H., Thelander, L., Yates 3rd, J. R., Russell, P. (2002). Cid13 is a cytoplasmic poly(A) polymerase that regulates ribonucleotide reductase mRNA. Cell 109, 563-573.</li>
<li>Current work on newly identified cytoskeletal proteins. Cell cycle analysis. Phenotype &amp; Protein Complex analysis for understanding mechanisms across species.</li>
<li>The data allowed us to make specific gene knockouts and fusions. Publications are in preparation, but not yet available. Please put an announcement link of how to submit such publications on the S.pombe GD entry page and we can submit them as they come in. We only started working in S. pombe because the genome was finished and partially annotated.</li>
<li>Genome sequence. J Cell Sci. 2004 May 1;117(Pt 11):2283-93.</li>
<li>The curation correctly suggested that what we had previously thought one large gene was in fact two closely-spaced genes (cid14 and pfs2). This information was vitally important for two recent publications: 1. Win, T. Z., Draper, S., Read, R. L., Pearce, J., Norbury, C. J. &amp; Wang, S. W. Requirement of fission yeast Cid14 in polyadenylation of rRNAs. Mol Cell Biol 26, 1710-21 (2006). 2. Wang, S. W., Asakawa, K., Win, T. Z., Toda, T. &amp; Norbury, C. J. Inactivation of the pre-mRNA cleavage and polyadenylation factor Pfs2 in fission yeast causes lethal cell cycle defects. Mol Cell Biol 25, 2288-96 (2005).</li>
<li>It helped me to find bgs4+ and clone it from its cosmid. It has been published in: The novel fission yeast (1,3)beta-D-glucan synthase catalytic subunit Bgs4p is essential during both cytokinesis and polarized growth. (2005) J. Cell Sci. 118:157-174. It also helped me to delete bg3+ and to find many things of myosin II proteins, heavy, light and regulatory proteins.</li>
<li>I&#39;ve used the LIST Download feature to help me curate a database for microarray analysis.</li>
<li>Facilitated identification by microarray hybridization of gene clusters regulated by TFIIH-associated kinase in fission yeast (Lee et al., Mol. Biol. Cell 16:2734-2745, 2005). Enabled mass spectrometric identification of subunits of fission yeast Cdk9 complex, ortholog of mammalian P-TEFb (Pei et al., Mol. Cell. Biol. 26: 777-788, 2006).</li>
<li>We have done many genetic screens, cloned genes from libraries, done a small amount of sequencing and then used GeneDB to identify the gene. Many publications have resulted from this type of strategy.</li>
<li>It is fast and easy to find information on genes that I find a new interest in, before I devote time and energy to learn more. There is good information about potential protein interactions, and it is wonderful to have information all in one place and format.</li>
<li>In our reaserch using MS-specs to identify the interactants with a protein of our interest of which expression is restricted only during sporulation, links to microarray data provided by Dr. Bahler&#39;s team were very helpful to select candidates to focus on.</li>
<li>with information for my work</li>
<li>It has been fundamental to many research projects in my lab, in particular to the following ones: Blanco, M., Pelloquin, L. and Moreno, S. (2001). Fission yeast mfr1 activates APC and coordinates meiotic nuclear division with sporulation. J. Cell Sci. 114, 2135-2141. Cueille, N., Salimova, E., Esteban, V., Blanco, M.A., Moreno, S., Bueno, A. and Simanis, V. (2001). Flp1, the fission yeast orthologue of the S. cerevisiae CDC14 gene, is not required for cyclin degradation or rum1p stabilisation at the end of mitosis. J. Cell Sci. 114, 2649-2664. P&radic;&copy;rez-Hidalgo, L., Moreno, S. and San Segundo, P. (2003). Regulation of meiotic progression by the meiosis-specific chechpoint kinase Mek1 in fission yeast. J. Cell Sci. 116, 259-271. Mart&radic;&ne;n-Castellanos, C., Blanco, M., Rozal&radic;&copy;n, A.E., P&radic;&copy;rez-Hidalgo, L., Garc&radic;&ne;a, A.I., Conde, F., Mata, J., Ellermeier, C., Davis, L., San-Segundo, P., Smith, G.R. and Moreno, S. (2005). A large-scale screen in S. pombe identifies novel genes required for meiotic recombination and chromosome segregation. Curr. Biol. 15, 2056-2062.</li>
<li>We have done an screening in which expression profile during sexual diferentation and gene information have been constantly checked out. Data from our screening have been added to the data base. During the course of the project the data base gave us up-dating information about genes that were in study by other groups.</li>
<li>It has proved invaluable for identifying new genes, predicting possible gene function, strain engineering, and plasmid construction. This is an essential tool for our research, and has played a part in every one of our publications in the last few years (approximately 10 papers in prominent journals).</li>
<li>this is a crucial tool for all experiments. at the most basic level, finding regions of homology to genes of interest in other organisms and to genes identified in screens in pombe is indispensable. this should be self-evident to funding agencies and the value of pombe research is obvious from the numerous concepts and insights gained from it. can&#39;t have pombe research without this data and curation.</li>
<li>It is very easier to find a gene and get nucleotide and amino acid sequences than other databases. This database is very useful to speculate its function, lacalization etc, because it contains biochemical information, such as domain motif.</li>
<li>Current research on snoRNAs</li>
<li>design of all molecular biology/ cloning and protein analysis</li>
<li>very helpful. use often for designing tagging constructs</li>
<li>It has been very useful to focus my research in proteins wich have cell-wall caracteristics because in my lab we are interested in this kind of proteins. (Alonso-n&radic;&int;&radic;&plusmn;ez et al. Mol Biol Cell. 2005 Apr;16(4):2003-17)</li>
<li>Huang, Y., Garrison, P.N., and Barnes, L.D. Molecular cloning of the Schizosaccharomyces pombe gene encoding diadenosine 5&#39;, 5&quot;&#39;-P1, P4-tetraphosphate (Ap4A) asymmetrical hydrolase: sequence similarity with the histidine triad (HIT) protein family. Biochem. J. 312, 925-932 (1995). Ingram, S.W., Stratemann, S.A., and Barnes, L.D. Schizosaccharomyces pombe Aps1, a diadenosine 5&#39;, 5&quot;&#39;-P1, P6-hexaphosphate hydrolase that is a member of the nudix (mutT) family of hydrolases: cloning of the gene and characterization of the purified enzyme. Biochemistry 38, 3649-3655 (1999). Safrany, S. T., Ingram, S. W., Cartwright, J. L., Falck, J. R., McLennan, A. G., Barnes, L. D., and Shears, S. B. The diadenosine hexaphosphate hydrolases from Schizosaccharomyces pombe and Saccharomyces cerevisiae are homologues of the human diphosphoinositol polyphosphate phosphohydrolase: the significance of the overlapping substrate specificities in a mutT motif. J. Biol. Chem. 274, 21735-21740 (1999) Ingram, S. W. and Barnes, L. D. Disruption and overexpression of the Schizosaccharomyces pombe aph1 gene and the effects on intracellular diadenosine 5&#39;, 5&#39;&#39;&#39;-P1, P4-tetraphosphate (Ap4A), ATP, and ADP concentrations. Biochem. J. 350, 663-669 (2000)</li>
<li>extremely useful as an everyday tool e.g. for designing primers and retrieving an overview of genes of interest</li>
<li>Don&#39;t know</li>
<li>I have used the data to get the sequences to clone S. pombe orthologs; also used the expression profile data ;and also identify the functions of many genes thht I have identified in my microarray experiments. We have just submitted a paper including all this information.</li>
<li>To know the position of a gene on the genome, Entrez Genome Map Viewer is insufficient and not useful, so GeneDB Genome Browser is necessary for my work.</li>
<li>finding homologs designing deletions designing epitope tagging</li>
<li>Mostly in gathering information on proteins identified in genetic screens and by MS analysis of purified protein complexes.</li>
<li>We have performed numberous genetic screens. We only needed very short sequences to find out what genes we had actually isolated. A few examples of these are as follows: Chen et al 1999; Chen et al 2000; Yen and Chang 2000; Papadaki et al 2002; Yen et al 2003.</li>
<li>The database allows me to do all of my recombinant DNA work very quickly and rapidly screen the role of various proteins in cell division. I&#39;ve only started using it recently (I just started my postdoc postion), but have used it so much I couldn&#39;t imagine where I would find the information if this was unavailable.</li>
<li>1. I have been doing a lot of search with LTRs, I love the BLUE colour easy to see LTRs in Artemis. They have made my life so much easier. 2. Ease of getting different amount DNA sequence at each end of the &#39;gene of interest&#39;, is extremely useful for genomic analysis and Southern blots. 3. Many references cited are extremely useful, e.g. about non-coding RNA (prl. I may have never seen that otherwise!</li>
<li>GeneDB is instrumental in quickly locating general information about all pombe genes, in addition to valuable bioinformatic tools.</li>
<li>The database has helped me to look up lists of genes I identify in the screens. In general, it&#39;s very helpful whenever I encouter a fission yeast gene which I don&#39;t know well about. Although these things are possible by other means (such as NCBI), GENE DB for S.pombe is so far most convenient as I need not to eliminate any entry of genes of any other organisms.</li>
<li>I am doing a lot of work on bioinformatics analysis of the mitotic cell cycle in various organisms. GeneDB is my reference database for S. pombe. I have published one paper specifically on the analysis of S. pombe data (PMID:16544289) for which GeneDB was extensively used.</li>
<li>excellent teaching tool</li>
<li>Identification of specific domains. Identification of homolgoues (Lorenz et al., 2004).</li>
<li>They are central to everything we do!</li>
<li>qq</li>
<li>I have screened an insertion library and cloned sequences by inverse pcr and used the database to identify loci and find about whether they are relevant candidates</li>
<li>finding orthologs, creating primers for tagging and knockouts</li>
<li>We have done a large proteomic screen of fission yeast protein. the annotation data was invaluable. The study is submitted for publication: Comparative proteomic and transcriptiomic profiling of the fission yeast Schizosaccharomyces pombe Michael W. Schmidt1,2, Andres Houseman3, Alexander R. Ivanov1,2 Dieter A. Wolf1,2 Address: 1 NIEHS Center for Environmental Health Proteomics Facility, 2 Department of Genetics and Complex Diseases, 3 Department of Biostatistics, Harvard School of Public Health, 665 Huntington Avenue, Boston, Massachusetts, 02115, USA;</li>
<li>Used gene annotations to map DNA replication origins. Sequence data used for mapping origins, for identifying DNA sequence features (AT-hook domains), for mapping AT-rich islands. GeneDB regularly used as a first resource for characterisation of genes of interest as well as a resource for links to related databases.</li>
<li>We used the database to annotate our custom genome tiling arrays. No publications yet.</li>
<li>in general I use this database most frequently to do most of my work</li>
<li>the blast tool on S. pombe GeneDB was crucial for me to identify in fission yeast the orthologous subunits of a transcriptional complex conserved from budding yeast to humans.</li>
<li>The data and curation have been essential for my research. The data and curation have contributed significantly in grant writing and experimental design and for routine S. pombe molecular biology (i. e. for genetic crosses, we can see if two genes are likely to be linked, the database is our primary source of information for making gene mutations and disruptions and allows us to check if we might accidentally affect a neighboring gene with predicted functions). My three most recent papers simply would not exist without this resource. They are 1) Oliva A, Rosebrock A, Ferrezuelo F, Pyne S, Chen H, Skiena S, Futcher B, Leatherwood J., The cell cycle-regulated genes of Schizosaccharomyces pombe. PLoS Biol. 2005 Jul;3(7):e225. 2) Zhou C, Arslan F, Wee S, Krishnan S, Ivanov AR, Oliva A, Leatherwood J, Wolf DA., PCI proteins eIF3e and eIF3m define distinct translation initiation factor 3 complexes. BMC Biol. 2005 May 17;3(1):14. 3) Averbeck N, Sunder S, Sample N, Wise JA, Leatherwood J., Negative control contributes to an extensive program of meiotic splicing in fission yeast. Mol Cell. 2005 May 13;18(4):491-8. Two of these papers involve microarray analyses and the resource was used extensively in design of the microarrays themselves and even more extensively in interpretation of the microarray data. We typically use JavaTree free-ware to view cluster analyses of microarray data and we have this configured to link directly to the S. Pombe GeneDB. Our future use will rely much more heavily on the GO annotations as they become more complete and refined. Our recent publications often use the Sanger database as a reference for groups of related genes or even gene functions rather than referencing a long list of primary papers. For genomics studies, such a common accesable and stable resource is essential. The third of the recent papers, published in Molecular Cell, does not contain microarray data, but it too would have been completely impossible without the Sancer Center Pombe Information resources. We used information in many different ways to guide our discovery of a major program of splicing regulation during S. pombe meiosis that is reported in this paper. The resource is also proving absolutely essential to our progress in studying replication timing in S. pombe. In particular, Artemis has been invaluable. I often prefer to do blast searches via the Sanger center rather than NCBI, though the convenient links to NCBI and other resources such as SGD are quite appreciated Finally, I depend on the resource when I receive S. pombe related papers for review. Even for non - S. pombe related questions, I often think of a relevant gene-name for S. pombe and not for other organisms and so I can quickly find out what the ortholog may be and this sometimes gives me useful insights or background knowledge.</li>
<li>Use quite frequently to screen for new enzymes/binding domains. See Sanders et al Cell 2004</li>
<li>Very convenient database. I use it more as a reference information source than as bioinformatic tools.</li>
<li>It helped me to find and compare the differents sequences of genes or proteins of the family Bgs. Its is useful to find their expresion patrons.</li>
<li>To build KO strains and identify putative homologous proteins.</li>
<li>It has allowed me to easily access protein and DNA sequence for molecular work such as constructing knockouts or tagged proteins. Curation helps give insight as to what is known about gene of interest and its homologs and paralogs which help decide which direction to take with experiments.</li>
<li>Get info quickly</li>
<li>Analysing centromeres and telomeres Analysing specific gene regions. Retrieval of information and sequnce of Rad21 and other genes/orf analysed. Bernard P, Maure J.F., Partridge J.F., Genier S, Javerzat J.P., Allshire R.C. (2001) Requirement of heterochromatin for cohesion at centromeres. Science. 294:2539-42. Retrieval of information and sequnce of Chp1 and other genes/orf analysed. Partridge J.F., Scott K.S.C., Bannister, A.J., Kouzarides T., and Allshire R.C. (2002) A cis-acting element from the fission yeast mediates histone H3 methylation and recruitment of silencing factors and cohesin to an ectopic site. Curr Biol. 12: 1652. Retrieval on information and sequences features of H3/H4 .1/.2/.3 genes/ORFs Mellone, BG, Ball L, Suka N, Grunstein MR, Partridge JF, Allshire RC. (2003) Centromere silencing and function in fission yeast is governed by the amino terminus of histone H3. Curr Biol. 13: 1748-57. Retrieval on information and sequences features of Sim4 gene/ORF Pidoux, AL, Richardson, W, Allshire R.C. (2003) Sim 4: a novel fission yeast kinetochore protein required for centromeric silencing and chromosome segregation. J. Cell Biol. 161: 295. Retrival of information and features of various SET proteins in the S. pombe genome: Sanders, S. L., Portoso, M., Mata, J., Bahler, J., Allshire, R. C., and Kouzarides, T. (2004). Methylation of histone H4 lysine 20 controls recruitment of Crb2 to sites of DNA damage. Cell 119, 603-614. Retrieval of information and sequences feature of rpb7: Djupedal, I., Portoso, M., Sp&radic;&bull;hr, H., Bonilla, C., Gustafsson, C. M., Allshire, R. C. and Ekwall, K. (2005). RNA Pol II subunit Rpb7 promotes centromeric transcription and RNAi-directed chromatin silencing. Genes Dev. 19, 2301-2306.</li>
<li>The data has been very helpful in order to allow for the research of gene sequences, homologous genes, for searching for genes that have several nomenclatures like mto1, mod20 and mbo1 which are the same gene.</li>
<li>In 2002 we published a paper on the insertion sites of the retrotransposon Tf1. We sequenced the DNA flanking the insertions and were able to determine their position in the genome using your blast tools. In addition, the annotations of each gene were very useful in analyzing the insertions.</li>
<li>I use it routinely as a start-point for the analysis of any S. pombe gene.</li>
<li>our pombe work is as yet unfunded</li>
<li>Gene structure (introns, exons), amplification of cDNA, targeting, interacting partners</li>
<li>cloning-retrieveing sequences protein expected sizes pI basic informations about function</li>
<li>yes! We do fission yeast proteomics and the data base helped us in every aspect of our work</li>
<li>The data and curation have been fundamental to all my recent publications.Without this these publications would not have been possible. Greenall A, Williams ES, Martin KA, Palmer JM, Gray J, Liu C, Whitehall SK. (2006) Hip3 interacts with the HIRA proteins Hip1 and Slm9 and is required for transcriptional silencing and accurate chromosome segregation. J Biol Chem. 281(13):8732-9. Blackwell, C., Martin, K.A., Greenall, A., Pidoux, A. Allshire, R.C. &amp; Whitehall, S.K. (2004) The S. pombe HIRA-like protein Hip1 is required for the periodic expression of histone genes and contributes to the function of complex centromeres. Mol. Cell. Biol. 24: 4309-4320. Stiefel J., Wang, L., Kelly, D.A., Janoo, R.T.K., Seitz, J., Whitehall, S.K. &amp; Hoffman, C.S. (2004) Isolation and characterization of suppressors of an adenylate cyclase in the fission yeast Schizosaccharomyces pombe. Euk. Cell 3: 610-619. Borrelly, G.P.M., Harrison, M.D., Robinson, A.K., Cox, S.G., Robinson, N.J. &amp; Whitehall, S.K. (2002) Surplus zinc is handled by Zym1 metallothionein and Zhf endoplasmic reticulum transporter in S. pombe. J. Biol Chem. 277: 30394-30400. Greenall, A., Hadcroft, A.P., Malakasi P., Jones N., Morgan B.A., Hoffman C.S., &amp; Whitehall S.K. (2002) Role of the fission yeast Tup1-like repressors and the Prr1 transcription factor in the response to salt stress Mol. Biol. Cell. 13: 2977-2989.</li>
<li>Identification of homologues for membrane-trafficking genes, including some in S.cervisiae which we suspected may be represented in S.pombe. This has allowed construction of GFP fusions, and determination of phenotypes; we&#39;ve managed to dissect part of the previously-unknown cvt pathway for vacuolar targetting in recent months as a result.</li>
<li>Quick identification of S. pombe homologues of genes from other species. This has formed the basis of much of the group&#39;s current work. This would be possible without GeneDB but much more laborious.</li>
<li>The meiotic expression profile of genes is very useful to know the character of the isolated genes, which might be involved in meiotic progression. Kitajima, T. S., Kawashima, S. A. and Watanabe, Y. The conserved kinetochore protein shugoshin protects centromeric cohesion during meiosis. Nature 427, 510-517 (2004). Yokobayashi, S. and Watanabe, Y. The kinetochore protein Moa1 enables cohesion-mediated monopolar attachment at meiosis I. Cell 123, 803-817 (2005)</li>
<li>Necessary background for the day by day work in the lab: design of new experiments, construction of strains etc.</li>
<li>Identification of homologues and of interesting characteristics, as yet unpublished.</li>
<li>Data was vital for Identification and verification of microarray targets</li>
<li>Data and curation are essential for our research on a day to day basis.</li>
<li>Very important to identify homologues</li>
<li>I used the information for cross species comparisons of expression data (manuscript in prep)</li>
<li>I have been helped and supported in the context of a collaborative study with Jurg Bahler group on cisplatin injury response that has been published in Cell Mol Life Sciences 61:2253-2263, 2004 - Gatti L et al.I strongly feel that model organisms like fission yeast could improve the understanding of processes relevant to the cure of cancer i. e pathways activated by antitumor drugs in mammalian cells (see Perego P et al., Pharmacol Reviews 52: 477-491, 2000.</li>
<li>Quick and easy access to commonly required gene and sequence information.</li>
<li>Peng et al (2005) Mol. Biol. Cell 16:1026 Bimbo et al (2005) Euk. Cell 4:799-813</li>
<li>Gene expression microarrys data: this allowed us to know that chs2 (a gene with similarity to Chitin Synthases genes that codifies a protein lacking such catalythic activity) was being expressed in vegetative cell cycle in fission yeast-it was not a pseudogene. To gain basic (and quickly)information about genes with which chs2 presents genetic interaction. Publications: ?In Schizosaccharomyces pombe chs2p has no chitin synthase activity but is related to septum formation?. Mart&radic;&ne;n Garc&radic;&ne;a, R.M., Dur&radic;&deg;n, A., Valdivieso, M.H. 2003. FEBS Letters 549: 176-180. -?The fission yeast Chs2p interacts with the type-II myosin Myo3p and is required for the integrity of the actomyosin ring?. Mart&radic;&ne;n Garc&radic;&ne;a, R.M., Valdivieso, M.H. 2006. J.Cell Science. (In Press).</li>
<li>Annotations of some bgs genes helped in their cloning and deletion.</li>
<li>no</li>
<li>essential for validation of gene regulatory network inference from microarray data</li>
<li>to identify microarrays candidates (to annotate), functions by homology Look for sequences of gene, to design primer, design fusions ...</li>
<li>identification of the spombe homolog of Ace2 (Mart&radic;&ne;n-Cuadrado et al (2003) J. Cell Sci. 116, 1689-1698)</li>
<li>I found out that a second homologue of my gene of interest exist and have therefore started to analyse this in addition.</li>
<li>Have been immesely useful in identifying orthologues for proteins studied in other organisms. The DB is also very important for obtaining first glance information on expression profiles in various conditions.</li>
<li>mRNA expression profile (meiosis) helped a lot to decide on certain genes to include in a complex finding strategy.</li>
<li>sequence info for genertaing KOs and tagged genes comparisons to yeast microarray data for groups of genes with similar expression profiles gene function descriptions link to forsburg page for plasmid info etc</li>
<li>Yes, enormously. Too many examples to list.</li>
<li>The Fission Yeast Database is an essential tool for all the research done in my lab. Most if not all of the publications from my lab. have in some way or another relied on information from it. I do not believe I could continue research using this model organism in the post-genomic era without this facility.</li>
<li>The domain structure prediction gave us an idea of the function. Tajadura et al. 2004 Journal of cell Science</li>
<li>Curation of GO terms is essential for all work with pombe. GeneDB is an essential resource</li>
<li>The data and curation are continually helpful, because they permit me to quickly evaluate what is know about each new fission yeast gene that I come across.</li>
<li>very helpful for my research since i started working with homologs of budding yeast knowing nothing about them in fission yeast</li>
<li>Gene and genome evolution</li>
<li>My students are using the DB much more extensively than I, almost on a daily basis, for primer design, search for chromosomal locations of cosmids for the use as FISH probes, etc. For them, and hence for the group it is indispensable.</li>
<li>All my research and publications for the last several years were not possible without the S. pombe database. I use the database everyday to determine which gene I should work on and retrieve its sequence information to design strategies for gene disruption and gene tagging. Also easy to retrieve DNA and protein sequences for homolog search and analyze domain structures of proteins.</li>
<li>1. Sequence retrieval to design PCRs and blast genome for primer sequences. 2. GO annotations to identify groups of genes (e.g. localizing to the nuclear envelope). This as a starting point for litterature searches and definition of hypotheses.</li>
<li>it helps enormously to have all information at the same place, to find the right sequence, micro-arry data and so on</li>
<li>This resource has saved time in cloning genes, analyzing plasmids that have rescued mutant strains, etc.</li>
<li>writing my mini master&#39;s thesis...it helped me find relevant papers</li>
<li>Orthologue relationships have been used in informal validation of automated orthologue prediction methods. GO annotation is being used in validation of bioinformatic predictions of functional linkage between gene products. The latter has previously only been systematically tested for S. cerevisiae and only with protein complexes (Barker D &amp; Pagel M, 2005, PLoS Comput Biol, 1, 24-31).</li>
<li>Find Artemis extremely useful in design of primers for homologous recombination and for determining relevant gene regulatory regions.</li>
<li>- identification of structural and functional features of pombe genes and proteins - identification of pombe homologs to genes of ther organisms - correlation of gene orders and genetic distances with physical map - planning of recombination analysis along chromosomes and at specific genes: Gregan et al (2005) Novel genes required for meiotic chromosome segregation are identified by a high-throughput knockout screen in fission yeast. Curr Biol, 15 1663- 1669, several publications in preparation</li>
<li>The expression pattern profiles have helped to identify my genes of interest.</li>
<li>It is the first place to search when moving onto a new field and gives information on where to continue looking for data. After carefully looking through all the infomration and all teh links on the gene of interest I get a very good picture of the function of teh gene.</li>
<li>Virtually every one of our publications has benefited enormously from having the easily accessible annotated genome information easily available. Examples are listed below. B&radic;&sect;hler, J., Steever, A.B., Wheatley, S., Wang, Y-L., Pringle, J.R., Gould K.L., and McCollum, D. (1998). Role of polo kinase and Mid1p in determining the site of cell division in fission yeast. J. Cell Biol. 143:1603-1616. Feoktistova, A., McCollum, D., Ohi, R., and Gould, K. L. (1999). Identification and characterization of Schizosaccharomyces pombe asp1+, a gene which interacts with mutations in the Arp2/3 complex and actin. Genetics. 152:895-908. Sparks, C.A., Morphew, M., and McCollum, D. (1999). Sid2p, a spindle pole body kinase that regulates the onset of cytokinesis. J. Cell Biol. 146:777-790. McCollum, D., Balasubramanian, M., and Gould, K.L. (1999). Isolation of cold-sensitive actin mutants defective for cytokinesis and polarized cell growth in fission yeast. FEBS Lett. 451:321-326. Guertin, D., Chang, L., Irshad, F., Gould, K.L., and McCollum, D. (2000). The role of the Sid1p kinase and Cdc14p in a novel signaling pathway that regulates the onset of cytokinesis in fission yeast. EMBO J. 19:1803-1815. Hou, M.C., and McCollum, D. (2000). Mob1p interacts with the Sid2p kinase and is required for cytokinesis in fission yeast. Curr. Biol. 10:619-622. Trautmann, S., Wolfe, B.A., Jorgensen, P., Tyers, M., Gould K.L., and McCollum. D. (2001). Fission yeast Clp1p phosphatase regulates G2/M transition and coordination of cytokinesis with cell cycle progression. Curr. Biol. 11:931-940. Wang, H., Tang, X., Liu, J., Trautmann, S., Balasundaram, D., McCollum, D., and Balasubramanian, M. (2002). The multiprotein exocyst complex is essential for cell separation in Schizosaccharomyces pombe. Mol. Biol. Cell. 13: 515-529. Guertin, D.A., Venkatram, S., Gould, K.L., and McCollum D. (2002). Dma1p prevents mitotic exit and cytokinesis by inhibiting the Septation Initiation Network (SIN). Dev. Cell. 3:779-790. Hou, M.C., Wiley, D.J., Verde, F., and McCollum, D. (2003). Mob2p interacts with Orb6p kinase to promote coordination of cell polarity with the cell cycle progression. J. Cell Sci. 116:125-35 Jin, Q.-W., and McCollum, D. (2003) Scw1p antagonizes the septation initiation network to regulate septum formation and cell separation in fission yeast. Euk. Cell. 2:510-520. Hou, M.-C., Guertin, D.A., and McCollum, D. (2004) Initiation of Cytokinesis is Controlled through Multiple Modes of Regulation of the Sid2p-Mob1p Kinase Complex. Mol. Cell. Biol. 24:3262-3276. Mitra, P., Zhang, Y., Rameh, L., Ivshina, M., McCollum, D., Nunnari, J., Hendricks, G., Kerr, M., Field, S., Cantley, L., and Ross, A. (2004). A Novel Phosphatidylinositol(3,4,5)P3 Pathway in Fission Yeast. J. Cell Biol. 166:205-11. Mishra M, Karagiannis, J., Trautmann, S., Wang, H., McCollum, D., Balasubramanian , M.K. (2004). The Clp1p/Flp1p phosphatase ensures completion of cytokinesis in response to minor perturbation of the cell division machinery in Schizosaccharomyces pombe. J Cell Sci. 117:3897-910. Trautmann, S., Rajagopalan, S., and McCollum, D. (2004) The S. pombe Cdc14-like phosphatase Clp1p regulates chromosome bi-orientation and interacts with aurora kinase. Dev. Cell. 7:755-62 Trautmann, S., and McCollum, D. (2005) Distinct nuclear and cytoplasmic functions of the S. pombe Cdc14-like phosphatase Clp1p/Flp1p and a role for nuclear shuttling in its regulation. Curr. Biol. 15:1384-1389.</li>
<li>In the last 4 years we have published 2 papers in J Biol Chem and 1 in Molecular Cell for which pombe gene database has been invaluable.</li>
<li>Link to microarray data base is extremely useful. We have found that pof1 is highly induced by cadnium, which led to publicatoin in EMBO J. (2005, Harrison et al.).</li>
<li>General improvement in speed and efficency by combining data into a single resource so that it reduces time spent searching primary literature or general gene databases to locate information. This is especially important working in a smaller institute where because of limited local library facilities much of the information is not available without delays ordering reprints or contacting authors.</li>
<li>No Thanks</li>
<li>I was able to conduct an in silico promoter analysis to predict my protein&#39;s binding sites in the genome.</li>
<li>when quoting a gene of interest in a publication it is easy to check the basic properties of the gene (function, location) and to find rapidly the relevant publications.</li>
<li>Helped a lot. Since the sequencing project was completed a lot of my research has been domain driven ie finding the function of a particular domain then looking for other pombe orf&#39;s which also contain this domain.eg UBA domain, Ubx domain, UBL domain Wikinson et al 2001, Nat Cell Biol, Vol3, 939-943. Hartmann-Petersen et al, 2004, Current Biology, Vol14, 824-828.</li>
<li>It is definitely an important tool, I do not want to miss. It is indispensable for planning crosses and design diverse constructs (deletions, tags, artificial markers for recombination assays, and so on), do bioinformatic searches and check orthologs in other organisms. I actually cannot, and fortunately do not have to, imagine how cumbersome it would be to get all this information without a very well maintained and curated database such as the pombe geneDB.</li>
<li>The unification of gene names among data is essential for combining data from various sources. I think GeneDB is the only manager for S.pombe gene names.</li>
<li>In general terms GeneDB has vastly improved the efficiency of my work. It allows me to get a handle on a gene of interest as soon as that gene has been identified as interesting.</li>
<li>Producing primers for construction of microarray data. Annotation of microarray datasets. Finding and annotating pombe orthologs.</li>
<li>Mostly we use it for designing primers for tagging and deletion, especially to know about upstream and downstream regions, and also to ensure that we do not disrupt or affect neighboring genes. Essentially all publications from my lab use this. Here are some publications: Snaith, H.A. and K.E. Sawin (2003) Fission yeast mod5p regulates polarized growth through anchoring of tea1p at cell tips. Nature 423:647-651. Sawin, K.E., P.C.C. Lourenco, H.A. Snaith (2004) Microtubule nucleation at non-spindle pole body microtubule-organizing centers requires fission yeast centrosomin-related protein mod20p. Curr. Biol. 14:763-775. Samejima, I., P.C.C. Lourenco, H. A. Snaith, K.E. Sawin (2005) Fission yeast mto2p regulates microtubule nucleation by the centrosomin-related protein mto1p. Mol. Biol. Cell 16:3040-3051. Snaith, H.A., I. Samejima, K.E. Sawin (2005) Multistep and multimode cortical anchoring of tea1p at cell tips in fission yeast. EMBO J. 24:3690-3699.</li>
<li>The sequence and annotation data for the genome was the starting point for our new oligonucleotide microarray using 70-mers for each annotated exon, intron, and 3&#39; UTR. It would not have been possible without it.</li>
<li>The gene name for S. pombe alpha-glucanases was changed from my personal preference &#39;agl&#39; to the reserved gene name &#39;agn&#39; during the review process of our manuscript Mol. Biol. Cell. 2004; 15: 3903-14 (see title), based on comments of an anonymous reviewer who recommended to keep gene names consistent within the S. pombe community. I now double-check your list carefully before submission of a new manuscript.</li>
<li>Quick at a glance, searches of Genes-of-interest e.g. candidate comes up in a screen. Additional validation for the hit is quickly needed. Information from geneDB is crucial. Also Comparisions of conserved pathways in different organims is important and GeneDG is a place where you comfortably find data over several model organisms,</li>
<li>identification of human disease homologue (Gachet et al 2005)</li>
<li>It is absolutely mandatory to have a very reliable database to carry on research in any lab.</li>
<li>Finding information on genes of interest</li>
<li>Allowed us to identify Nbs1 (Chahwan 2003 MCB 23:6564), the RNAi machinery (Sigova 2004 G&amp;D 18:2359) and replication origins (Patel 2006 MBC 17:308). None of these could have been done by simple BLAST searches.</li>
<li>We have used it extensively for information regarding genes we identify and for predicting possible candidate genes together with our mapping experiment. I use it to quickly access ORF sequences to use in PCR primer designs. In the past few years, the research or writing of these papers heavily depended on access to GeneDB. Curr Genet. 2006 Feb 21; [Epub ahead of print] Genetics. 2006 Feb 19; [Epub ahead of print] Genetics. 2005 Dec;171(4):1523-33. Epub 2005 Sep 2. Proc Natl Acad Sci U S A. 2005 Apr 26;102(17):6108-13. Epub 2005 Apr 14. Eukaryot Cell. 2004 Jun;3(3):610-9. Eukaryot Cell. 2002 Aug;1(4):558-67. Genetics. 2001 Mar;157(3):1205-15. Genetics. 2001 Mar;157(3):1159-68. Genetics. 2000 Oct;156(2):513-21.</li>
<li>since inception it has influenced in some way, all of my publications http://www.paulgyoung.com/gradbrochure.htm</li>
<li>Example: we conducted a screen for novel replication mutants (Gomez. angeles and Forsburg, Genetics 2005). This required that we identify the gene corresponding to the mutant. We did this by cloning from a genomic library; naturally tehre were multiple genes per clone. The curation was invaluable in sorting out likely candidates for novel or known replication genes.</li>
<li>Find orthologues of my genes of interest</li>
<li>Yes, all the information I need about sequences, consensus sites, and more I have it from Sp database. I add publications were to make the work I had the information from Sp database. .- Maribel S&radic;&deg;nchez-Piris,Francesc Posas, Vicen&radic;ß Alemany, Ingerborg Winge, Elena Hidalgo, Oriol Bachs and Rosa Aligue. The serine/threonine kinase Cmk2 is a target of the Sty1 MAPK and is required for oxidative stress response in fission yeast. Journal of Biological Chemistry 227: 17722-17727. 2002. .- Vicen&radic;ß Alemany, Maribel S&radic;&deg;nchez-Piris, Oriol Bachs and Rosa Aligue. Characterization of Cmk2, a new serine/threonine kinase in fission yeast. FEBS Letters 524:79-86. 2002 .- Sandra L&radic;&ge;pez-Avil&radic;&copy;s, Maribel Grande, marta Gonz&radic;&deg;lez, Oriol Bachs and Rosa Aligue. .- Sandra L&radic;&ge;pez-Avil&radic;&copy;s, Marta Gonz&radic;&deg;lez, Asse-Lill Helgesen, Maribel Grande, Vicen&radic;ß Alemany, Maribel Sanchez-Piris, Oriol Bachs, Jonathan Millar and Rosa Aligue. Regulation of the Cdc25 phosphatase by the stress-activated kinase Srk1 in fission yeast Molecular Cell 17: 49-59. 2005.</li>
<li>It has made it easier to quickly check gene information, resulting in less wasted time in searching multiple databases for bits of information</li>
</ol>

  </div>
  <div *ngIf="pageName == '2006-survey-q18'">
<h3 id="genedb-s.-pombe-survey-question-18-responses">2006 GeneDB <em>S. pombe</em> survey, question 18 responses</h3>
<p><strong>Do you work experimentally on any other organisms in addition to fission yeast for your area of research, and if so which? (list all, including fission yeast, in order of relative importance to your work/degree of use)</strong></p>
<ol>
<li>S. cerevisiae</li>
<li>No</li>
<li>S. japonicus (starting soon) fission yeasts of different geographical origins (starting soon)</li>
<li>S. pombe S. cerevisiae</li>
<li>Fission yeast mouse and human cells</li>
<li>fission yeast, budding yeast</li>
<li>fission yeast: characterization of protein probably implicated in stress response E. coli: clone and protein purification</li>
<li>fission yeast budding yeast</li>
<li>Schizosaccharomyces pombe E. coli Saccharomyces cerevisiae</li>
<li>Fission yeast Budding yeast E.coli</li>
<li>S. cerevisiae C.elegans Mice</li>
<li>Fission yeast</li>
<li>fission yeast budding yeast mammalian cells</li>
<li>No</li>
<li>Fission yeast Human cell tissue culture Xenopus oocytes</li>
<li>No.</li>
<li>fission yeast stem cells</li>
<li>Primary organism: S. cerevisiae - 80% of lab Secondary organism: S. pombe - 20% of lab</li>
<li>no</li>
<li>Xenopus egg extracts</li>
<li>fission yeast human cell lines zebrafish</li>
<li>I only work on fission yeast.</li>
<li>fission yeast, mammalian tissue culture cells</li>
<li>human fission yeast mouse (cell culture) Drosophila</li>
<li>Fission yeast Saccharomyces cerevisiae</li>
<li>only fission yeast</li>
<li>No, only fission yeast right now</li>
<li>fission yeast budding yeast E. coli insect cell</li>
<li>Fission Yeast Caenorhabditis elegans Saccharomyces cerevisiae</li>
<li>Only fission yeast</li>
<li>S. cerevisiae E. coli C. neoformans</li>
<li>fission yeast budding yeast</li>
<li>fission yeast budding yeast</li>
<li>no</li>
<li>Tomato</li>
<li>none</li>
<li>No, i don&#39;t</li>
<li>1.fision yeast 2.e.coli</li>
<li>S. pombe E.coli S. cerevisiae</li>
<li>budding yeast</li>
<li>Caenorhabditis elegans</li>
<li>no - there&#39;s only one true model organism ;-)</li>
<li>S. pombe E. coli</li>
<li>S.pombe Candida spp. s.cerevisiae Phaffia rhodosima Mucor circinelloides</li>
<li>I also work on S. cerevisiae. But presently, I am focussing on the use of S. pombe as a model organism.</li>
<li>fission yeast</li>
<li>fission yeast human cell lines mice</li>
<li>fission yeast human cells Aspidoscelis spp</li>
<li>We also use mammalian tissue culture cells, such as HeLa, NIH3T3, and MCF7 cells. S. pombe remains our major model organism.</li>
<li>fission yeast mammalian cell culture mice budding yeast</li>
<li>fission yeast only, though I often compare protein sequences from different organisms to identify conserved regions- usually human and budding yeast</li>
<li>Fission yeast Mammalian cell lines</li>
<li>No.</li>
<li>no</li>
<li>I don&#39;t do experimental work</li>
<li>fission yeast budding yeast</li>
<li>Fission yeast</li>
<li>fission yeast</li>
<li>qq</li>
<li>Fission yeast mammalian cells budding yeast</li>
<li>no</li>
<li>Pombe</li>
<li>no</li>
<li>fission yeast E.coli</li>
<li>1. fission yeast 2. budding yeast</li>
<li>1) fission yeast 2) budding yeast</li>
<li>Sometimes in S. cerevisiae</li>
<li>Fission yeast only</li>
<li>No</li>
<li>Drosophila S2 cells 1%</li>
<li>No.</li>
<li>no</li>
<li>no</li>
<li>fission yeast</li>
<li>The lab I am in does (budding yeast) but I only work with S. pombe.</li>
<li>S. pombe E. coli</li>
<li>S. pombe E. coli</li>
<li>mammalian cells pombe zebrafish</li>
<li>budding</li>
<li>Budding yeast Fission yeast</li>
<li>saccharomyces cerevisiae</li>
<li>buddin yeast</li>
<li>Very occasionally we are forced to look at S. cerevisiae...</li>
<li>Schizosaccharomyces pombe Saccharomyces cerevisiae</li>
<li>Fission yeast S. pombe Fission yeast S. japonicus</li>
<li>Fission yeast only</li>
<li>Budding yeast HeLa cell</li>
<li>Taphrina wiesneri</li>
<li>no</li>
<li>Fission yeast Budding yeast E. coli B. cereus</li>
<li>no</li>
<li>S. cerevisiae S. Pombe</li>
<li>none</li>
<li>Arabidopsis fission yeast budding yeast H. sapiens bacteria</li>
<li>Human tumor cell lines (60%) Bacterial strains for cloning purposes (30%) Yeast strains (10%) Such percentages are dependent on specific projects. At the time of a specific collaborative effort focus on yeast was 70%</li>
<li>No</li>
<li>fission yeast</li>
<li>Fission yeast Saccharomyces cerevisiae</li>
<li>budding yeast</li>
<li>No, ONLY S.pombe</li>
<li>No at this point.</li>
<li>No</li>
<li>no</li>
<li>fusion yeast lactobacillus</li>
<li>Schizosaccharomyces pombe Saccharomyces cerevisiae</li>
<li>no</li>
<li>S. pombe S. cerevisiae C. albicans</li>
<li>S. pombe S. cerevisiae (mostly in past) human zebrafish (in future) mouse (in future)</li>
<li>Budding yeast Fission yeast The relative importance is likely to shift to Fission yeast soon.</li>
<li>Fission yeast</li>
<li>fission yeast</li>
<li>Fission yeast Haloferax volcanii (if only there was an equivalent database for that organism...)</li>
<li>no</li>
<li>fission yeast mammalian cell lines budding yeast</li>
<li>fission yeast budding yeast</li>
<li>Fission yeast, Budding yeast, Rice</li>
<li>Candida albicans Yarrowia lipolytica Saccharomyces cerevisiae Schizosaccharomyces pombe</li>
<li>C. elegans S. cerevisiae</li>
<li>No.</li>
<li>fisson yeast budding yeast</li>
<li>fission yeast C.elegans (nematode) Caulobacter crescentus (bacteria)</li>
<li>fission yeast Drosophila</li>
<li>fission yeast, budding yeast</li>
<li>bioinformatically (on fully sequenced genomes) - fission yeast, bakers yeast, other fungi, eukaryotic parasites, eukaryotes in general, prokaroytes.</li>
<li>no</li>
<li>Fission yeast mammalian tissue culture cells cryptococcus neoformans Pichia pastoris</li>
<li>no</li>
<li>S. pombe S. cerevisiae</li>
<li>no</li>
<li>No</li>
<li>FISSION YEAST E. COLI B. SUBTILIS</li>
<li>S.pombe, C. elegans and S.cerevisiae</li>
<li>So far only fission yeast</li>
<li>No</li>
<li>S. cerevisiae S. pombe B. subtilis</li>
<li>No Thanks</li>
<li>fission yeast</li>
<li>no</li>
<li>fission yeast budding yeast</li>
<li>no</li>
<li>Collaborate with cereviae workers.</li>
<li>Fission yeast Budding yeast (two-hybrid only so far)</li>
<li>I&#39;m a programmer for the microarray group here, so my work can vary. So far, it&#39;s only been on fission yeast and budding yeast, but that will likely change in the future. Right now, I&#39;ve worked on fission yeast the most, I&#39;d say.</li>
<li>(1) S. pombe, (2) S. cerevisiae, (3) Candida albicans, (4) Cryptococcus neoformans.</li>
<li>Mouse Fission yeast Mammalian Cell culture models Drosophila Schneider S2 cells S. cerevisiae</li>
<li>fission yeast mammalian cells C. elgeans</li>
<li>Fission yeast, budding yeast and higher eukariotic cells</li>
<li>Mammalian cells, rodent and human</li>
<li>Only fission yeast</li>
<li>No</li>
<li>Only fission yeast</li>
<li>Fission Yeast Budding Yeast Xenopus</li>
<li>S. cerevisiae S. pombe</li>
<li>RARELY S cerevisiae, mammalian cells</li>
<li>Xenopus laevis</li>
<li>Fission yeast 80% mammalian culture cells 20%</li>
<li>no</li>
</ol>

  </div>
  <div *ngIf="pageName == '2006-survey-q19'">
<h3 id="genedb-s.-pombe-survey-question-19-responses">2006 GeneDB <em>S. pombe</em> survey, question 19 responses</h3>
<p><strong>Please provide any additional comments and information about the current resource which you feel were not covered by this survey</strong></p>
<ol>
<li>-</li>
<li>I often use published literatures of my interesting genes. I sometimes found that some literatures was not related to the gene I was interested in. One thing I noticed was that the same name was shown in those papers. Gene expression profiles are very usful. If possible, I would like to see the links of other eukaryotic orthologues not just only S. cerevisiae orthologues.</li>
<li>Genedb is a great resource. It is amazing that such a fantastic resource has been built so quickly with such limited funding. I hope that it can be extended as it provides a very valuable resource for the community.</li>
<li>the access to the site and downloading information is very slow at times.</li>
<li>It is an excellent resource.</li>
<li>I cannot understand what some of the items or screens are that you refer to in question 4? What is a context map? Some of the others required me to guess, even after looking at web page to answer this survey. I suggest approaching the answers to this question with caution. As a budding yeast researcher, SGD has been a godsend for our speed of work. The closer you get to this level of functionality, the better S. pombe will be as an experimental system.</li>
<li>Val and the people who work on the database are very helpful in terms of answering questions and helping with specific problems.</li>
<li>form stonger ties/links with SGD and other yeast databases</li>
<li>Most up-to-date information in pombe community</li>
<li>Set the default sequence download range to span only the gene ORF, with options to extend up or downstream.</li>
<li>great resource - thanks v much!</li>
<li>NA</li>
<li>Not all the data published is curated.</li>
<li>It would be useful to suggest promoter region, if possible. This is necessary when we plan to clone the genomic version of the gene for expression study.</li>
<li>Thank you</li>
<li>I think it is a wonderful source...Long Live GeneDB!!!</li>
<li>This is an essential service which must be maintained to the highest standard.</li>
<li>qq</li>
<li>This is an absolutely essential resource for our community. Valerie Wood had done a truly remarkable job so far. She is proactive in seeking input from the community and she is also very responsive to our needs and suggestions. It is clear that she cares that this database and resource should be of the highest quality, though I cannot imagine how she accomplishes all that she does. It seems to me that this project must require many many qualified people. I don&#39;t know how many additional people are involved. Valerie Wood has always been the person to respond to us when we have requested information or guidance.</li>
<li>This is a fantastic resource for teh fission yeast community and should be maintained and continually improved.</li>
<li>It seems to me the most valuable tool to include in this database is a comprehensive collection of annotations with references to the literature. This will require continous updating.</li>
<li>S. pombe GeneDB does not appear to have the gene sequence for the mating type plus information (mat2 sequence for h+), is this because the genome sequence was from a h- strain? I think that the plus genes should be added.</li>
<li>none</li>
<li>important resource !</li>
<li>The data base is wonderful. We all appreciate the administration very much.</li>
<li>none</li>
<li>I found it very well organized</li>
<li>No comments</li>
<li>no</li>
<li>- very important resource, like the new YOGY a lot. - need easier community annotation, literature linking (e.g. for specific genes mentioned in own papers, authors could link them to YOGY resource directly, rather than waiting for the indexing services to put it there as a general reference only).</li>
<li>In general DNA regulatory elements are very badely annoted in the pombe data base. For example, whereas the mating locus organisation is very well known, it still very difficult to find in the pombe genome data base (using artemis)where mat1, 2 and 3 are exactely located.</li>
<li>genetic interactions physical interactions</li>
<li>The Fission Yeast database is an ESSENTIAL part of the bioinformatic framework of S.pombe genome analysis.</li>
<li>--</li>
<li>GeneDB is a minimal resource which is essential for modern fission yeast research. Even so many things are missing and need to be built out.</li>
<li>its fine</li>
<li>I&#39;m just starting this postdoc in pombe. I have recently joined pombelist (1 month), and so I cannot comment about many of the services. In general, I have found it to be useful although my use of it has been limited. Perhaps its greatest utility is by linking the pombe community? I anticipate using the database a great deal in the coming months, and I will be happy to report back in the future on my experiences.</li>
<li>Another point for the wish list: genetic and physical intaractions would be very helpfull too...</li>
<li>none</li>
<li>please provide web services</li>
<li>The current resource does not have logical species coverage. It just covers what the Pathogens Sequencing Unit have been working on. An S. pombe-based resource would be more relevant and even more useful than the current resource, and is essential for smooth rational progress of research with this important model organism.</li>
<li>Val Wood does an outstanding job keeping the database updated and informing the Pombe community about changes to GeneDB. Her hard work has made this database an invaluable resource for the fission yeast research community.</li>
<li>Protein localisation</li>
<li>So far GeneDB is supurb. Sometimes I find pombe GeneDB is better than budding yeast equivalent. I hope that the Sanger Centre continues to make effort to impove this important resource. It is essential for our research and community.</li>
<li>This has really become an essential resource for S. pombe research even for an occaisional user like myself.</li>
<li>Since we use S. cerevisiae as our primary research organism, I am a frequent visitor to the SGD homepage. I would benefit a lot if the primary pombe and cerevisiae pages would have a similar structure/navigation and have similar features.</li>
<li>No Thanks</li>
<li>Maybe I missed it but I would appreciate a possibility to download GO terms (including parents) using the ListDownload feature.</li>
<li>Keep up the good work!</li>
<li>I think joining up with Saccharomyces would be a good idea.</li>
<li>Thanks for all the work you do in bringing fission yeast up to the level of &quot;model organism&quot;. Keep up the good work!</li>
<li>Please note that the genomes of all four yeasts mentioned have been sequences and that inclusion of these yeasts would be extremely helpful for cross-species analyses.</li>
<li>aim for database to facilitate/increase the usefulness of pombe as a model organism -for human disease, for conserved genes and gene families</li>
<li>The people associated with this are exceptional. In particular Val Wood is quick to respond to questions and comments and has provided us with considerable support and information. She is acknowledged in both of our 2006 papers for significant intellectual contributions.</li>
<li>The GeneDB is an absolutely inevitable knowledge resource for the pombe researchers.</li>
<li>very valuable resource and essential for the pombe community</li>
<li>The GeneDB front end works very well.</li>
<li>It is absolutely essential to secure funding for this !!!!</li>
</ol>

  </div>
  <div *ngIf="pageName == 'fission-yeast-community-curation-pilot-project'">
<h2 id="participating-groups-annotated-papers">Participating groups &amp; Annotated papers</h2>
<p>This table summarizes the results of the pilot project that preceded the <a routerLink="/community/fission-yeast-community-curation-project">Fission Yeast Community Curation Project</a>. The first phase of the project began in January 2009, and the second in November 2009. We thank all of the researchers who participated for making the pilot project a success.</p>
<table border="0" cellpadding="2" width="100%">
 <thead>
  <tr>
  <th><span>PMID</span></th>
  <th>Laboratory</th>
  <th>Submitter if different</th>
  <th>GO terms</th>
  <th>Interactions</th>
  <th>Phenotypes</th>
  <th>Other <span>curation</span></th>
  <th>Total</th>
  <th>Date</th>
  </tr>
 </thead>
 <tbody>
 <tr>
  <td>PHASE 1</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19101542">19101542</a></td>
  <td>Peter Folk</td>
  <td>Martin <span>Převorovský</span></td>
  <td>19</td>
  <td>0</td>
  <td>15</td>
  <td>3</td>
  <td>37</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19098712">19098712</a></td>
  <td>Juan Mata</td>
  <td>&nbsp;</td>
  <td>3</td>
  <td>7</td>
  <td>0</td>
  <td>1</td>
  <td>11</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19076239">19076239</a></td>
  <td>Paul Russell</td>
  <td>&nbsp;</td>
  <td>5</td>
  <td>0</td>
  <td>2</td>
  <td>2</td>
  <td>9</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18675827">18675827</a></td>
  <td>Per <span>Sunnerhagen</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>1</td>
  <td>3</td>
  <td>0</td>
  <td>6</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17353264">17353264</a></td>
  <td>Chris <span>Norbury</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>0</td>
  <td>0</td>
  <td>0</td>
  <td>2</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19057642">19057642</a></td>
  <td>Luis <span>Rokeach</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19056896">19056896</a></td>
  <td>Fred Winston</td>
  <td>Dom <span>Helmlinger</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19041767">19041767</a></td>
  <td>Peter <span>Espenshade</span></td>
  <td>John Burg</td>
  <td>4</td>
  <td>2</td>
  <td>2</td>
  <td>1</td>
  <td>9</td>
  <td><span class="no-break">2009-01-04</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18556659">18556659</a></td>
  <td>Kevin <span>Hardwick</span></td>
  <td>&nbsp;</td>
  <td>16</td>
  <td>16</td>
  <td>2</td>
  <td>0</td>
  <td>34</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18430926">18430926</a></td>
  <td>Charlie Hoffman</td>
  <td>&nbsp;</td>
  <td>4</td>
  <td>0</td>
  <td>2</td>
  <td>0</td>
  <td>6</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17304215">17304215</a></td>
  <td><span>Hiroshi</span> <span>Iwasaki</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>5</td>
  <td>9</td>
  <td>19</td>
  <td>35</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18426916">18426916</a></td>
  <td><span>Hiro</span> <span>Yamano</span></td>
  <td>Michelle Tricky</td>
  <td>2</td>
  <td>1</td>
  <td>2</td>
  <td>1</td>
  <td>6</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19037101">19037101</a></td>
  <td>Nick <span>Rhind</span></td>
  <td>Nicholas Willis</td>
  <td>7</td>
  <td>6</td>
  <td>7</td>
  <td>0</td>
  <td>20</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19001497">19001497</a></td>
  <td>Ken <span>Sawin</span></td>
  <td>Lynda <span>Groocock</span></td>
  <td>9</td>
  <td>6</td>
  <td>2</td>
  <td>1</td>
  <td>18</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18368917">18368917</a></td>
  <td>Steve Aves</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18799626">18799626</a></td>
  <td>Takashi <span>Toda</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>3</td>
  <td>3</td>
  <td>0</td>
  <td>8</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19109429">19109429</a></td>
  <td>Stephen <span>Kearsey</span></td>
  <td>&nbsp;</td>
  <td>9</td>
  <td>2</td>
  <td>2</td>
  <td>0</td>
  <td>13</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17688408">17688408</a></td>
  <td>Susan <span>Forsburg</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17409062">17409062</a></td>
  <td>Iain Hagan</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-06</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19139281">19139281</a></td>
  <td>Tony Carr</td>
  <td>Edgar <span>Hartsuiker</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-01-15</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18065650">18065650</a></td>
  <td>Per <span>Sunnerhagen</span></td>
  <td>&nbsp;</td>
  <td>5</td>
  <td>2</td>
  <td>4</td>
  <td>0</td>
  <td>11</td>
  <td>-</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18406331">18406331</a></td>
  <td><span>Kaz</span> <span>Shiozaki</span></td>
  <td>&nbsp;</td>
  <td>4</td>
  <td>4</td>
  <td>1</td>
  <td>1</td>
  <td>10</td>
  <td><span class="no-break">2009-01-15</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19139265">19139265</a></td>
  <td>Kathy Gould</td>
  <td>Rachel Roberts</td>
  <td>8</td>
  <td>11</td>
  <td>1</td>
  <td>1</td>
  <td>21</td>
  <td><span class="no-break">2009-01-17</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19197239">19197239</a></td>
  <td>Benoit <span>Arcangioli</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>0</td>
  <td>4</td>
  <td>0</td>
  <td>6</td>
  <td><span class="no-break">2009-02-10</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19328067">19328067</a></td>
  <td>Rob Fisher</td>
  <td>&nbsp;</td>
  <td>5</td>
  <td>4</td>
  <td>2</td>
  <td>25</td>
  <td>36</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19330768">19330768</a></td>
  <td><span>Katya</span> <span>Grishchuk</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19217404">19217404</a></td>
  <td>Robin <span>Allshire</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19217403">19217403</a></td>
  <td>Paul Russell</td>
  <td>Jessica Williams</td>
  <td>5</td>
  <td>7</td>
  <td>3</td>
  <td>0</td>
  <td>15</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19362535">19362535</a></td>
  <td>Janet Partridge</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19250904">19250904</a></td>
  <td><span>Songtao</span> <span>Jia</span></td>
  <td><span>Bahrat</span> Reddy</td>
  <td>4</td>
  <td>2</td>
  <td>2</td>
  <td>2</td>
  <td>10</td>
  <td><span class="no-break">2009-03-01</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19373772">19373772</a></td>
  <td>Jeremy <span>Hyams</span></td>
  <td>Isabelle <span>Jourdain</span></td>
  <td>5</td>
  <td>1</td>
  <td>6</td>
  <td>0</td>
  <td>12</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19299465">19299465</a></td>
  <td>Sara Mole</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19299465">19299465</a></td>
  <td>Julie Cooper</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19286980">19286980</a></td>
  <td><span>Stevan</span> Marcus</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19033386">19033386</a></td>
  <td>Iain Hagan</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19351719">19351719</a></td>
  <td>Luis <span>Rokeach</span></td>
  <td><span>Pascale</span> Beauregard</td>
  <td>2</td>
  <td>1</td>
  <td>1</td>
  <td>0</td>
  <td>4</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19160458">19160458</a></td>
  <td>Ken <span>Sawin</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19211838">19211838</a></td>
  <td>Nick <span>Rhind</span></td>
  <td>Mary Porter-Goff</td>
  <td>2</td>
  <td>2</td>
  <td>8</td>
  <td>0</td>
  <td>12</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19273851">19273851</a></td>
  <td>Ramsay <span>Macfarlane</span></td>
  <td>&nbsp;</td>
  <td>1</td>
  <td>0</td>
  <td>1</td>
  <td>0</td>
  <td>2</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19243310">19243310</a></td>
  <td>Takashi <span>Toda</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>4</td>
  <td>1</td>
  <td>0</td>
  <td>7</td>
  <td><span class="no-break">2009-04-20</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19158663">19158663</a></td>
  <td>Peter <span>Espenshade</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>1</td>
  <td>1</td>
  <td>0</td>
  <td>4</td>
  <td><span class="no-break">2009-04-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19237545">19237545</a></td>
  <td><span>Henar</span> <span>Valdivieso</span></td>
  <td>&nbsp;</td>
  <td>1</td>
  <td>1</td>
  <td>3</td>
  <td>0</td>
  <td>5</td>
  <td><span class="no-break">2009-04-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18562692">18562692</a></td>
  <td>Jonathan Miller</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-04-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16143612">16143612</a></td>
  <td>Charlie Hoffman</td>
  <td>&nbsp;</td>
  <td>0</td>
  <td>8</td>
  <td>0</td>
  <td>0</td>
  <td>8</td>
  <td><span class="no-break">2009-04-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15448137">15448137</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>15</td>
  <td>0</td>
  <td>0</td>
  <td>5</td>
  <td>20</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18375981">18375981</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>1</td>
  <td>0</td>
  <td>0</td>
  <td>0</td>
  <td>1</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9391101">9391101</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>0</td>
  <td>0</td>
  <td>0</td>
  <td>2</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7518718">7518718</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>0</td>
  <td>0</td>
  <td>0</td>
  <td>2</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7958849">7958849</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>7</td>
  <td>6</td>
  <td>0</td>
  <td>0</td>
  <td>13</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19542312">19542312</a></td>
  <td>John Armstrong</td>
  <td>James <span>Dodgeson</span></td>
  <td>18</td>
  <td>0</td>
  <td>16</td>
  <td>0</td>
  <td>34</td>
  <td>NONE</td>
 </tr>
 <tr>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>PHASE 2</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19620282">19620282</a></td>
  <td>Simon Whitehall</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19643199">19643199</a></td>
  <td><span>Jeramy</span> <span>Hyams</span></td>
  <td>Isabelle <span>Jourdain</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19436749">19436749</a></td>
  <td>Wayne <span>Wahls</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19627505">19627505</a></td>
  <td><span>Henar</span> <span>Valdivieso</span></td>
  <td>&nbsp;</td>
  <td>1</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>4</td>
  <td>5</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19664060">19664060</a></td>
  <td>Stuart <span>Macneill</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19686603">19686603</a></td>
  <td>Stuart <span>Macneill</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19666000">19666000</a></td>
  <td><span>Murakami</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19646873">19646873</a></td>
  <td><span>Fulvia</span> Verde</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19714215">19714215</a></td>
  <td><span>Jurg</span> <span>Bahler</span></td>
  <td>&nbsp;</td>
  <td>5</td>
  <td>4</td>
  <td>5</td>
  <td>2</td>
  <td>16</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18923417">18923417</a></td>
  <td>Nick <span>Boddy</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18667531">18667531</a></td>
  <td>Nick <span>Boddy</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19686339">19686339</a></td>
  <td>Luis <span>Rokeach</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17114925">17114925</a></td>
  <td><span>Danesh</span> <span>Moazed</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19707600">19707600</a></td>
  <td>Felicity Watts</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19394293">19394293</a></td>
  <td><span>Danesh</span> <span>Moazed</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18574244">18574244</a></td>
  <td>Peter <span>Baumann</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19520858">19520858</a></td>
  <td>Peter <span>Espenshade</span></td>
  <td>&nbsp;</td>
  <td>9</td>
  <td>1</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>10</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18503029">18503029</a></td>
  <td>Peter <span>Espenshade</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>1</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>1</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19427212">19427212</a></td>
  <td>Anna <span>Paoletti</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18223116">18223116</a></td>
  <td>Simon <span>Labbe</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19915076">19915076</a></td>
  <td>Simon <span>Labbe</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19571115">19571115</a></td>
  <td>Shelly <span>Sazer</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18272791">18272791</a></td>
  <td>Rosa <span>Aligue</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19838064">19838064</a></td>
  <td>Jan <span>Paluh</span></td>
  <td>&nbsp;</td>
  <td>3</td>
  <td>29</td>
  <td>9</td>
  <td>4</td>
  <td>55</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19696784">19696784</a></td>
  <td>Takashi <span>Toda</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>3</td>
  <td>2</td>
  <td>&nbsp;</td>
  <td>7</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17409354">17409354</a></td>
  <td>Elena Hidalgo</td>
  <td>&nbsp;</td>
  <td>14</td>
  <td>7</td>
  <td>3</td>
  <td>2</td>
  <td>26</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19416828">19416828</a></td>
  <td>Jacob <span>Dalgaard</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19473263">19473263</a></td>
  <td>Mathias <span>Sipiczki</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19836238">19836238</a></td>
  <td>Caroline Wilkinson</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18252721">18252721</a></td>
  <td>Caroline Wilkinson</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17986863">17986863</a></td>
  <td>Erik <span>Boye</span></td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>0</td>
  <td>3</td>
  <td>0</td>
  <td>5</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=20531409">20531409</a></td>
  <td>Juan Mata</td>
  <td>&nbsp;</td>
  <td>3</td>
  <td>90</td>
  <td>0</td>
  <td>0</td>
  <td>93</td>
  <td>N/A</td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19720063">19720063</a></td>
  <td><span>Nimisha</span> Sharma</td>
  <td>&nbsp;</td>
  <td>2</td>
  <td>3</td>
  <td>3</td>
  <td>2</td>
  <td>10</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19625445">19625445</a></td>
  <td>Jose <span>Cansado</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19486165">19486165</a></td>
  <td><span>Takayoshi</span> <span>Kuno</span></td>
  <td><span>Yue</span> Fang</td>
  <td>2</td>
  <td>10</td>
  <td>4</td>
  <td>0</td>
  <td>16</td>
  <td><span class="no-break">2009-11-23</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19713940">19713940</a></td>
  <td><span>Kentaro</span> <span>Nakano</span></td>
  <td><span>Masakatsu</span> <span>Takaine</span></td>
  <td>4</td>
  <td>2</td>
  <td>2</td>
  <td>0</td>
  <td>8</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19409973">19409973</a></td>
  <td>Kurt <span>Runge</span></td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td><span class="no-break">2009-11-24</span></td>
 </tr>
 <tr>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 </tbody>
</table>

  </div>
  <div *ngIf="pageName == 'fission-yeast-community-curation-project'">
<h1 id="the-fission-yeast-community-curation-project">The Fission Yeast Community Curation Project</h1>
<p>Comprehensive manual curation of the <em>S. pombe</em> literature is a primary goal of the PomBase project, and among the most important activities of the curation staff. Increasing numbers of new publications, and a large backlog of older papers, however, mean that complete literature curation will require input from the research community as well as the efforts of dedicated professional curators.</p>
<p>The PomBase Community Curation system enables researchers to contribute annotations directly to PomBase based on their publications using Canto, a web-based tool that allows both curators and researchers to create annotations. Canto supports GO, phenotype, interaction, and modification annotations, and can be configured for use with other ontologies as the need arises. Annotations made in the community curation system will be prioritized for inclusion in PomBase and will therefore also be more rapidly disseminated to other databases (e.g. GenBank/ENA/DDBJ, UniProtKB, BioGRID and GO), making data from annotated papers widely visible.</p>
<p><strong>How to contribute</strong>: For newly published papers, PomBase curators email authors inviting them to participate, with specific instructions and links. Lab members are also welcome to evaluate existing annotations and create new annotations from past publications — use the PubMed ID search on the <a href="https://curation.pombase.org/pombe">main PomBase Canto page</a>. Anyone can also try the <a href="https://curation.pombase.org/demo">demo version of Canto</a>.</p>
<p><strong>Progress and attribution</strong>: <a href="https://curation.pombase.org/pombe/stats/annotation">Curation statistics</a> track S. pombe papers curated in Canto by PomBase and community curators, and the annotations obtained. Canto will soon allow contributors to register <a href="http://orcid.org/">ORCID</a> identifiers in the system, which will support more formalised attribution of community contributions to PomBase, and thereby enable researchers to cite community curation in institutional reports, funding applications, etc. To supplement the existing permanent publication-centred links in Canto, we are currently investigating possible ways to link to external resources such as ORCID or OpenRIF via ORCID IDs.</p>
<p><strong>Further reading:</strong> For an in-depth analysis of our approach, and the status of community curation as of early 2019, see our publication:</p>
<p>Lock A, Harris MA, Rutherford K, Hayles J, Wood V.<br />
Community curation in PomBase: enabling fission yeast experts to provide detailed, standardized, sharable annotation from research publications. <em>Database (Oxford)</em> 2020 Jan 1;2020. pii: baaa028.<br />
<a href="http://www.ncbi.nlm.nih.gov/pubmed/32353878">PMID:32353878</a> <a href="https://doi.org/10.1093/database/baaa028">DOI:10.1093/database/baaa028</a></p>
<p>For a bit of historical perspective, you can browse the results of the <a routerLink="/community/fission-yeast-community-curation-pilot-project">Community Curation Pilot Project</a> that took place in 2009.</p>

  </div>
  <div *ngIf="pageName == 'index'">
<h1 id="fission-yeast-community">Fission Yeast Community</h1>
<ul>
<li><a routerLink="/community/fission-yeast-community-curation-project">Community curation project</a>, including the old <a routerLink="/community/fission-yeast-community-curation-pilot-project">pilot project</a></li>
<li><a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">Mailing list</a></li>
<li><a routerLink="/community/internet-resources">External resources</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'internet-resources'">
<h2 id="s.-pombe-community-websites"><em>S. pombe</em> Community Websites</h2>
<ul>
<li><a href="http://www-bcf.usc.edu/~forsburg/pombeweb.html">PombeWeb</a> maintained by Susan Forsburg
<ul>
<li><a href="http://www-bcf.usc.edu/~forsburg/pombeweb.html#people">Fission yeast researchers</a> at PombeWeb</li>
</ul></li>
<li><a href="http://www-bcf.usc.edu/~forsburg/main.html">PombeNet</a> maintained by Susan Forsburg
<ul>
<li><a href="http://www-bcf.usc.edu/~forsburg/main.html">About pombe</a> at PombeNet</li>
<li><a href="http://www-bcf.usc.edu/~forsburg/plasmids.html">Working with pombe</a> at PombeNet</li>
</ul></li>
<li><a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/erss.cgi?rss_guid=1ToPSyjDNLLW2R_UIOHFTxnRdlJouyMeQX4YfwMFXtO8d87M4r">PubMed RSS Feed</a> Feed for current <em>S. pombe</em> related publications.</li>
</ul>
<h3 id="ontologies">Ontologies</h3>
<ul>
<li><a href="http://www.geneontology.org/">The Gene Ontology Consortium</a> controlled vocabulary</li>
<li><a href="http://amigo.geneontology.org/">AmiGO</a> Search and browse GO annotation</li>
<li>The <a href="http://www.ebi.ac.uk/quickgo/">QuickGO</a> browser at EBI</li>
<li><a href="https://sites.google.com/view/yeastgenome-help/function-help/gene-ontology-go">SGD GO documentation</a> to help SGD users become familiar with GO</li>
<li><a href="http://vortex.cs.wayne.edu/projects.htm#Onto-Express">Onto-Express</a> find significant GO terms in lists of genes</li>
<li><a href="http://go.princeton.edu/cgi-bin/GOTermFinder">GOTermFinder</a> find significant GO terms in lists of genes</li>
<li><a href="http://bahlerweb.cs.ucl.ac.uk/cgi-bin/GLA/GLA_input">AnGeLi: Analysis of Gene Lists</a> (UCL) uses GO, phenotypes, interactions, etc.</li>
<li><a href="http://www.sequenceontology.org/">Sequence Ontology</a></li>
</ul>
<h3 id="interactions-pathways-and-networks">Interactions, Pathways and Networks</h3>
<ul>
<li><a href="http://www.thebiogrid.org/">BioGRID</a> Genetic and physical interactions</li>
<li><a href="http://www.ebi.ac.uk/intact/pages/interactions/interactions.xhtml?query=species:4896">Interactions (IntAct)</a> Known interactions for <em>S. pombe</em> from the IntAct database</li>
<li><a href="http://reactome.org/PathwayBrowser/#/SPECIES=68324">Reactome pathway data</a> <em>S. pombe</em> entry point.</li>
<li><a href="http://www.esyn.org/index.php">esyN</a> Web-based tool for building, sharing, and viewing network data</li>
<li><a href="http://string-db.org/">STRING</a> Protein–protein interaction network database</li>
</ul>
<h3 id="sequence-analysis-tools">Sequence Analysis Tools</h3>
<h4 id="general-tools">General tools</h4>
<!--
-   <a href="http://gbrowse.org/wiki/index.php/Schizosaccharomyces_pombe">GBrowse Wiki</a>
    for *S. pombe*
-->
<ul>
<li><a href="http://www.sanger.ac.uk/science/tools/artemis">Artemis</a> Sequence viewer and analysis tool (Note: for <em>S. pombe</em> we recommend using <a href="ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/v16/">Artemis Version 16</a>.</li>
<li><a href="http://www.ncbi.nlm.nih.gov/mapview/map_search.cgi?chr=spombe.inf">Map viewer</a> at the NCBI</li>
<li><a href="http://megasun.bch.umontreal.ca/People/lang/species/spo/spombe.html">Mitochondrial Genome Information</a></li>
</ul>
<h4 id="sequence-features-protein-families-orthologues">Sequence Features, Protein Families, Orthologues</h4>
<ul>
<li>Todd Lowe’s <em>S. pombe</em> genomic <a href="http://gtrnadb.ucsc.edu/GtRNAdb2/genomes/eukaryota/Schi_pomb_972h/">tRNA analysis</a></li>
<li><a href="http://wardlab.cbs.umn.edu/pombe/"><em>S. pombe</em> membrane protein library</a></li>
<li><a href="http://www.rsat.eu/">RSAT</a> Regulatory sequence analysis tools (includes <a href="http://rsat-tagc.univ-mrs.fr/rsat/">Fungi</a> server)</li>
<li><a href="http://pfam.xfam.org/">Pfam</a> protein families (search or browse)</li>
<li><a href="http://supfam.org/SUPERFAMILY/cgi-bin/gen_list.cgi?genome=iz">SCOP</a> Superfamily assignments</li>
<li><a href="https://rostlab.org/owiki/index.php/PredictNLS">PredictNLS</a> Nuclear localization signal prediction</li>
<li><a href="http://www.bahlerlab.info/YOGY/">YOGY</a> A resource for retrieving orthologous proteins</li>
<li><a href="http://epd.vital-it.ch/S_pombe/S_pombe_database.php">EPD</a> Eukaryotic Promoter Database, <em>S. pombe</em> section</li>
</ul>
<h3 id="gene-expression">Gene Expression</h3>
<ul>
<li><a href="http://www.bahlerlab.info/cgi-bin/SPGE/geexview">Gene Expression Viewer</a> at UCL</li>
<li><a href="https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/buildDetails?atlas_build_id=355"><em>S. pombe</em> PeptideAtlas</a> compendium of peptides identified in tandem mass spectrometry proteomics experiments</li>
</ul>
<h3 id="experimental-tools">Experimental tools</h3>
<ul>
<li><a href="http://www.massgeneral.org/cancer/research/motamediresources.aspx">PRIMED</a> Primers for deleting and tagging yeast genes (<em>S. pombe</em> and <em>S. cerevisiae</em>; scroll to bottom of page for downloadable spreadsheets)</li>
</ul>
<h3 id="strains-and-constructs">Strains and Constructs</h3>
<ul>
<li><a href="http://yeast.lab.nig.ac.jp/yeast/">Japanese National Bio Resource Project</a></li>
<li><a href="http://www.riken.jp/SPD/index.html">Postgenome database service</a> Chemical Genetics Laboratory, Riken</li>
<li><a href="http://dna.brc.riken.jp/en/yoshidayeast_en">Orfeome clones</a> Riken Bioresource Centre</li>
<li><a href="http://www.addgene.org">Addgene</a> Collects and distributes plasmids</li>
</ul>
<h3 id="experimental-protocols">Experimental protocols</h3>
<ul>
<li>The <a href="https://dornsife.usc.edu/pombenet/">Forsburg lab</a> technical information e.g. <a href="https://dornsife.usc.edu/pombenet/protocols/">Protocols</a></li>
</ul>
<h3 id="s.-cerevisiae-resources"><em>S. cerevisiae</em> Resources</h3>
<ul>
<li><a href="http://www.yeastgenome.org/">SGD</a> <em>Saccharomyces</em> Genome Database</li>
<li><a href="http://www.sanger.ac.uk/research/projects/genomeinformatics/sgrp.html"><em>Saccharomyces</em> genome resequencing</a> at Sanger</li>
</ul>

  </div>
  <div *ngIf="pageName == 'pombase_2006_survey_summary'">
<h2 id="survey-for-genedb-s.-pombe">2006 survey for GeneDB <em>S. pombe</em></h2>
<p>1. How often do you use S. pombe GeneDB?</p>
<table border="1" cellpadding="1" cellspacing="1" style="width: 500px; height: 153px;">
<tbody>
<tr>
  <td>&nbsp;</td>
  <td>Response Percent</td>
  <td>Response Total</td></tr>
<tr>
  <td>Every day</td>
  <td>18.2%</td>
  <td>66</td></tr>
<tr>
  <td>A few times a week</td>
  <td>52.9%</td>
  <td>192</td></tr>
<tr>
  <td>A few times a month</td>
  <td>25.1%</td>
  <td>91</td></tr>
<tr>
  <td>On rare occasions</td>
  <td>3.9%</td>
  <td>14</td></tr>
<tr><td colspan="2" rowspan="1">Total Respondents</td>
  <td>363</td></tr>
<tr><td colspan="2" rowspan="1">(skipped this question)</td>
  <td>0</td></tr>
</tbody>
</table>
<p>&nbsp;</p>
<p>2. For what type of work do you use GeneDB?</p>
<table border="1" cellpadding="1" cellspacing="1" style="width: 500px; height: 153px;">
<tbody>
<tr>
  <td>&nbsp;</td>
  <td>Response Percent</td>
  <td>Response Total</td></tr>
<tr>
  <td>To access annotation, curation and analysis for my gene(s) of interest</td>
  <td>91.2%</td>
  <td>331</td></tr>
<tr>
  <td>To identify candidate genes in my field of interest</td>
  <td>74.7%</td>
  <td>271</td></tr>
<tr>
  <td>Bioinformatics research</td>
  <td>44.9%</td>
  <td>163</td></tr>
<tr>
  <td>Teaching</td>
  <td>8.3%</td>
  <td>30</td></tr>
<tr>
  <td>Other</td>
  <td>4.4%</td>
  <td>16</td></tr>
<tr><td colspan="2" rowspan="1">Total Respondents</td>
  <td>363</td></tr>
<tr><td colspan="2" rowspan="1">(skipped this question)</td>
  <td>0</td></tr>
</tbody>
</table>
<p>&nbsp;</p>
<p>3. Which GeneDB features and associated tools do you use?</p>
<table border="1" cellpadding="1" cellspacing="1" style="width: 650px; height: 675px;">
<tbody>
<tr>
  <td>&nbsp;</td>
  <td>frequently</td>
  <td>occasionally</td>
  <td>rarely</td>
  <td>never</td>
  <td>I have not seen this feature</td>
  <td>Response Total</td></tr>
<tr>
  <td>The Gene Page</td>
  <td>83% (301)</td>
  <td>14% (52)</td>
  <td>2% (7)</td>
  <td>0% (1)</td>
  <td>1% (2)</td>
  <td>363</td></tr>
<tr>
  <td>AmiGO Gene Ontology (GO) browser</td>
  <td>12% (43)</td>
  <td>31% (112)</td>
  <td>30% (109)</td>
  <td>19% (68)</td>
  <td>9% (31)</td>
  <td>363</td></tr>
<tr>
  <td>Blast Tools</td>
  <td>45% (165)</td>
  <td>33% (121)</td>
  <td>13% (46)</td>
  <td>8% (28)</td>
  <td>1% (3)</td>
  <td>363</td></tr>
<tr>
  <td>Boolean Query Tool</td>
  <td>5% (19)</td>
  <td>14% (50)</td>
  <td>23% (85)</td>
  <td>31% (114)</td>
  <td>26% (95)</td>
  <td>363</td></tr>
<tr>
  <td>List Download</td>
  <td>9% (31)</td>
  <td>26% (95)</td>
  <td>25% (89)</td>
  <td>25% (89)</td>
  <td>16% (59)</td>
  <td>363</td></tr>
<tr>
  <td>Expression links</td>
  <td>26% (96)</td>
  <td>34% (123)</td>
  <td>17% (62)</td>
  <td>12% (45)</td>
  <td>10% (37)</td>
  <td>363</td></tr>
<tr>
  <td>YOGY (Retrieval of eukaryotic orthologs)</td>
  <td>13% (48)</td>
  <td>26% (93)</td>
  <td>23% (83)</td>
  <td>18% (67)</td>
  <td>20% (72)</td>
  <td>363</td></tr>
<tr>
  <td>User defined &#39;Motif Search&#39;</td>
  <td>14% (51)</td>
  <td>24% (87)</td>
  <td>27% (99)</td>
  <td>21% (75)</td>
  <td>14% (51)</td>
  <td>363</td></tr>
<tr>
  <td>Browsable Catalogues (products, curation, Pfam)</td>
  <td>18% (66)</td>
  <td>29% (104)</td>
  <td>26% (94)</td>
  <td>19% (69)</td>
  <td>8% (30)</td>
  <td>363</td></tr>
<tr>
  <td>Artemis (genome browser)</td>
  <td>23% (84)</td>
  <td>28% (102)</td>
  <td>25% (89)</td>
  <td>15% (53)</td>
  <td>10% (35)</td>
  <td>363</td></tr>
<tr>
  <td>GBrowse (genome browser)</td>
  <td>10% (36)</td>
  <td>26% (94)</td>
  <td>27% (98)</td>
  <td>23% (84)</td>
  <td>14% (51)</td>
  <td>363</td></tr>
<tr>
  <td>Gene Name Registry</td>
  <td>9% (33)</td>
  <td>22% (80)</td>
  <td>31% (114)</td>
  <td>28% (102)</td>
  <td>9% (34)</td>
  <td>363</td></tr>
<tr>
  <td>Cross-organism search page</td>
  <td>11% (39)</td>
  <td>35% (126)</td>
  <td>27% (99)</td>
  <td>15% (53)</td>
  <td>13% (46)</td>
  <td>363</td></tr>
<tr><td colspan="6" rowspan="1">Total Respondents</td>
  <td>363</td></tr>
<tr><td colspan="6" rowspan="1">(skipped this question)</td>
  <td>0</td></tr>
</tbody>
</table>
<p>4. Which GeneDB features and associated tools do you use?</p>
<table border="1" cellpadding="1" cellspacing="1" style="width: 650px; height: 567px;">
<tbody>
<tr>
  <td>&nbsp;</td>
  <td>very useful</td>
  <td>useful</td>
  <td>not very useful</td>
  <td>not at all useful</td>
  <td>I have not seen this feature</td>
  <td>Response Total</td></tr>
<tr>
  <td>General information</td>
  <td>77% (279)</td>
  <td>22% (81)</td>
  <td>0% (1)</td>
  <td>0% (0)</td>
  <td>1% (2)</td>
  <td>363</td></tr>
<tr>
  <td>Location</td>
  <td>53% (191)</td>
  <td>40% (144)</td>
  <td>6% (20)</td>
  <td>1% (3)</td>
  <td>1% (5)</td>
  <td>363</td></tr>
<tr>
  <td>Context map</td>
  <td>43% (157)</td>
  <td>44% (158)</td>
  <td>8% (28)</td>
  <td>2% (6)</td>
  <td>4% (14)</td>
  <td>363</td></tr>
<tr>
  <td>Curation, and curated ortholog data</td>
  <td>46% (166)</td>
  <td>42% (153)</td>
  <td>6% (23)</td>
  <td>0% (1)</td>
  <td>6% (20)</td>
  <td>363</td></tr>
<tr>
  <td>Predicted peptide properties</td>
  <td>37% (136)</td>
  <td>46% (167)</td>
  <td>10% (35)</td>
  <td>1% (2)</td>
  <td>6% (23)</td>
  <td>363</td></tr>
<tr>
  <td>Gene Ontology annotation</td>
  <td>33% (118)</td>
  <td>47% (171)</td>
  <td>13% (48)</td>
  <td>2% (6)</td>
  <td>6% (20)</td>
  <td>363</td></tr>
<tr>
  <td>Published expression profiles</td>
  <td>52% (188)</td>
  <td>37% (135)</td>
  <td>6% (23)</td>
  <td>1% (3)</td>
  <td>4% (14)</td>
  <td>363</td></tr>
<tr>
  <td>Literature link</td>
  <td>48% (174)</td>
  <td>42% (153)</td>
  <td>8% (28)</td>
  <td>0% (1)</td>
  <td>2% (7)</td>
  <td>363</td></tr>
<tr>
  <td>Domain information</td>
  <td>48% (176)</td>
  <td>44% (158)</td>
  <td>5% (19)</td>
  <td>1% (2)</td>
  <td>2% (8)</td>
  <td>363</td></tr>
<tr>
  <td>Database cross-references</td>
  <td>39% (142)</td>
  <td>45% (163)</td>
  <td>11% (40)</td>
  <td>0% (0)</td>
  <td>5% (18)</td>
  <td>363</td></tr>
<tr>
  <td>UniProt annotation for this protein</td>
  <td>27% (99)</td>
  <td>51% (186)</td>
  <td>12% (42)</td>
  <td>1% (4)</td>
  <td>9% (32)</td>
  <td>363</td></tr>
<tr><td colspan="6" rowspan="1">Total Respondents</td>
  <td>363</td></tr>
<tr><td colspan="6" rowspan="1">(skipped this question)</td>
  <td>0</td></tr>
</tbody>
</table>
<p>5. Which other internet resources do you use for your work? (select all which apply)</p>
<table border="1" cellpadding="1" cellspacing="1" style="width: 500px; height: 153px;">
<tbody>
<tr>
  <td>&nbsp;</td>
  <td>Response Percent</td>
  <td>Response Total</td></tr>
<tr>
  <td>PubMed</td>
  <td>99.2%</td>
  <td>360</td></tr>
<tr>
  <td>Gene Ontology (GO) Website</td>
  <td>25.3%</td>
  <td>92</td></tr>
<tr>
  <td>UniProt</td>
  <td>24.2%</td>
  <td>88</td></tr>
<tr>
  <td>Pfam</td>
  <td>49%</td>
  <td>178</td></tr>
<tr>
  <td>Interpro</td>
  <td>15.2%</td>
  <td>55</td></tr>
<tr>
  <td>Saccharomyces Database (SGD)</td>
  <td>67.5%</td>
  <td>245</td></tr>
<tr>
  <td>Comprehensive Yeast Genome Database (CYGD)</td>
  <td>17.1%</td>
  <td>62</td></tr>
<tr>
  <td>PombePD</td>
  <td>41.9%</td>
  <td>152</td></tr>
<tr>
  <td>NCBI (Entrez)</td>
  <td>90.4%</td>
  <td>328</td></tr>
<tr>
  <td>Google</td>
  <td>86%</td>
  <td>312</td></tr>
<tr>
  <td>GRID</td>
  <td>12.9%</td>
  <td>47</td></tr>
<tr>
  <td>Other (please specify)</td>
  <td>16.3%</td>
  <td>59</td></tr>
<tr><td colspan="2" rowspan="1">Total Respondents</td>
  <td>363</td></tr>
<tr><td colspan="2" rowspan="1">(skipped this question)</td>
  <td>0</td></tr>
</tbody>
</table>
<p>6. Which of these new types of information would you most like to see in the S. pombe database? Rank your results in order of preference from &#39;1&#39; MOST desirable to &#39;8&#39; LEAST desirable (each value can only be selected once)</p>
<table border="1" cellpadding="1" cellspacing="1" style="width: 800px; height: 403px;">
<tbody>
<tr>
  <td>&nbsp;</td>
  <td>1</td>
  <td>2</td>
  <td>3</td>
  <td>4</td>
  <td>5</td>
  <td>6</td>
  <td>7</td>
  <td>8</td>
  <td>Response Average</td></tr>
<tr>
  <td>Transcriptional start/stop</td>
  <td>17% (60)</td>
  <td>14% (50)</td>
  <td>9% (32)</td>
  <td>14% (49)</td>
  <td>13% (44)</td>
  <td>12% (42)</td>
  <td>11% (37)</td>
  <td>10% (34)</td>
  <td>4.18</td></tr>
<tr>
  <td>Regulatory motifs</td>
  <td>8% (26)</td>
  <td>22% (77)</td>
  <td>18% (61)</td>
  <td>16% (56)</td>
  <td>14% (49)</td>
  <td>12% (41)</td>
  <td>8% (27)</td>
  <td>2% (7)</td>
  <td>3.85</td></tr>
<tr>
  <td>Phenotypes</td>
  <td>52% (182)</td>
  <td>13% (47)</td>
  <td>16% (55)</td>
  <td>8% (27)</td>
  <td>4% (13)</td>
  <td>4% (15)</td>
  <td>2% (7)</td>
  <td>1% (3)</td>
  <td>2.23</td></tr>
<tr>
  <td>Signalling pathway data</td>
  <td>6% (20)</td>
  <td>8% (26)</td>
  <td>13% (46)</td>
  <td>17% (57)</td>
  <td>13% (46)</td>
  <td>18% (60)</td>
  <td>15% (53)</td>
  <td>10% (34)</td>
  <td>4.89</td></tr>
<tr>
  <td>Sequence/feature variation in related strains</td>
  <td>3% (9)</td>
  <td>5% (16)</td>
  <td>5% (18)</td>
  <td>11% (39)</td>
  <td>14% (49)</td>
  <td>13% (45)</td>
  <td>24% (83)</td>
  <td>25% (87)</td>
  <td>5.90</td></tr>
<tr>
  <td>Full text literature searching</td>
  <td>5% (16)</td>
  <td>8% (27)</td>
  <td>6% (22)</td>
  <td>7% (25)</td>
  <td>13% (44)</td>
  <td>16% (55)</td>
  <td>17% (58)</td>
  <td>29% (101)</td>
  <td>5.75</td></tr>
<tr>
  <td>Protein modifications</td>
  <td>7% (24)</td>
  <td>17% (60)</td>
  <td>18% (62)</td>
  <td>15% (51)</td>
  <td>15% (53)</td>
  <td>13% (46)</td>
  <td>11% (37)</td>
  <td>5% (17)</td>
  <td>4.20</td></tr>
<tr>
  <td>Microarray datasets</td>
  <td>6% (20)</td>
  <td>14% (50)</td>
  <td>16% (58)</td>
  <td>13% (46)</td>
  <td>14% (50)</td>
  <td>11% (40)</td>
  <td>11% (39)</td>
  <td>15% (52)</td>
  <td>4.67</td></tr>
<tr><td colspan="9" rowspan="1">Total Respondents</td>
  <td>361</td></tr>
<tr><td colspan="9" rowspan="1">(skipped this question)</td>
  <td>2</td></tr>
</tbody>
</table>
<p>7. How accurate, in general, do you consider the S. pombe curation and S. pombe GO curation?</p>
<table border="1" cellpadding="1" cellspacing="1" style="width: 500px; height: 153px;">
<tbody>
<tr>
  <td>&nbsp;</td>
  <td>Response Percent</td>
  <td>Response Total</td></tr>
<tr>
  <td>highly accurate</td>
  <td>26.1%</td>
  <td>95</td></tr>
<tr>
  <td>mostly accurate</td>
  <td>61.8%</td>
  <td>225</td></tr>
<tr>
  <td>badly inaccurate</td>
  <td>0%</td>
  <td>0</td></tr>
<tr>
  <td>cannot assess</td>
  <td>12.1%</td>
  <td>44</td></tr>
<tr><td colspan="2" rowspan="1">Total Respondents</td>
  <td>364</td></tr>
<tr><td colspan="2" rowspan="1">(skipped this question)</td>
  <td>0</td></tr>
</tbody>
</table>
<p>8. How accurate, in general, do you consider the S. pombe gene models?</p>
<table border="1" cellpadding="1" cellspacing="1" style="width: 500px; height: 153px;">
<tbody>
<tr>
  <td>&nbsp;</td>
  <td>Response Percent</td>
  <td>Response Total</td></tr>
<tr>
  <td>highly accurate</td>
  <td>19.2%</td>
  <td>70</td></tr>
<tr>
  <td>mostly accurate</td>
  <td>60.4%</td>
  <td>220</td></tr>
<tr>
  <td>badly inaccurate</td>
  <td>0.8%</td>
  <td>3</td></tr>
<tr>
  <td>cannot assess</td>
  <td>19.5%</td>
  <td>71</td></tr>
<tr><td colspan="2" rowspan="1">Total Respondents</td>
  <td>364</td></tr>
<tr><td colspan="2" rowspan="1">(skipped this question)</td>
  <td>0</td></tr>
</tbody>
</table>
<p>9. Have you ever contacted GeneDB, and if so, what was the reason? (select all which apply)</p>
<table border="1" cellpadding="1" cellspacing="1" style="width: 500px; height: 153px;">
<tbody>
<tr>
  <td>&nbsp;</td>
  <td>Response Percent</td>
  <td>Response Total</td></tr>
<tr>
  <td>Report an error</td>
  <td>35.6%</td>
  <td>47</td></tr>
<tr>
  <td>Reserve a gene name</td>
  <td>45.5%</td>
  <td>60</td></tr>
<tr>
  <td>Ask a question</td>
  <td>50.8%</td>
  <td>67</td></tr>
<tr>
  <td>Make a suggestion</td>
  <td>21.2%</td>
  <td>28</td></tr>
<tr>
  <td>Submit data/curation</td>
  <td>29.5%</td>
  <td>39</td></tr>
<tr><td colspan="2" rowspan="1">Total Respondents</td>
  <td>132</td></tr>
<tr><td colspan="2" rowspan="1">(skipped this question)</td>
  <td>231</td></tr>
</tbody>
</table>
<p>10. If you answered question 9, in general how satisfied were you with the response(s)?</p>
<table border="1" cellpadding="1" cellspacing="1" style="width: 500px; height: 153px;">
<tbody>
<tr>
  <td>&nbsp;</td>
  <td>Response Percent</td>
  <td>Response Total</td></tr>
<tr>
  <td>very satisfied</td>
  <td>90.4%</td>
  <td>122</td></tr>
<tr>
  <td>somewhat satisfied</td>
  <td>6.7%</td>
  <td>9</td></tr>
<tr>
  <td>somewhat unsatisfied</td>
  <td>3%</td>
  <td>4</td></tr>
<tr>
  <td>very unsatisfied</td>
  <td>0%</td>
  <td>0</td></tr>
<tr><td colspan="2" rowspan="1">Total Respondents</td>
  <td>135</td></tr>
<tr><td colspan="2" rowspan="1">(skipped this question)</td>
  <td>228</td></tr>
</tbody>
</table>
<p>11. How easy is it to find the information you require in S. pombe GeneDB?</p>
<table border="1" cellpadding="1" cellspacing="1" style="width: 500px; height: 153px;">
<tbody>
<tr>
  <td>&nbsp;</td>
  <td>Response Percent</td>
  <td>Response Total</td></tr>
<tr>
  <td>very easy</td>
  <td>32%</td>
  <td>116</td></tr>
<tr>
  <td>relatively easy</td>
  <td>55.1%</td>
  <td>200</td></tr>
<tr>
  <td>sometimes confusing</td>
  <td>11.6%</td>
  <td>42</td></tr>
<tr>
  <td>difficult</td>
  <td>0.3%</td>
  <td>1</td></tr>
<tr>
  <td>no opinion</td>
  <td>1.1%</td>
  <td>4</td></tr>
<tr><td colspan="2" rowspan="1">Total Respondents</td>
  <td>363</td></tr>
<tr><td colspan="2" rowspan="1">(skipped this question)</td>
  <td>0</td></tr>
</tbody>
</table>
<p>12. At what kind of institution do you work?</p>
<table border="1" cellpadding="1" cellspacing="1" style="width: 500px; height: 153px;">
<tbody>
<tr>
  <td>&nbsp;</td>
  <td>Response Percent</td>
  <td>Response Total</td></tr>
<tr>
  <td>University</td>
  <td>63.6%</td>
  <td>231</td></tr>
<tr>
  <td>Research Institute</td>
  <td>33.9%</td>
  <td>123</td></tr>
<tr>
  <td>Company</td>
  <td>0%</td>
  <td>0</td></tr>
<tr>
  <td>Government</td>
  <td>2.5%</td>
  <td>9</td></tr>
<tr>
  <td>Other</td>
  <td>0%</td>
  <td>0</td></tr>
<tr><td colspan="2" rowspan="1">Total Respondents</td>
  <td>364</td></tr>
<tr><td colspan="2" rowspan="1">(skipped this question)</td>
  <td>0</td></tr>
</tbody>
</table>
<p>13. What is your position?</p>
<table border="1" cellpadding="1" cellspacing="1" style="width: 500px; height: 153px;">
<tbody>
<tr>
  <td>&nbsp;</td>
  <td>Response Percent</td>
  <td>Response Total</td></tr>
<tr>
  <td>Student</td>
  <td>32%</td>
  <td>116</td></tr>
<tr>
  <td>Post Doctoral Fellow</td>
  <td>25.3%</td>
  <td>92</td></tr>
<tr>
  <td>Database Curator</td>
  <td>1.4%</td>
  <td>5</td></tr>
<tr>
  <td>Research Investigator</td>
  <td>8.8%</td>
  <td>32</td></tr>
<tr>
  <td>Principal Investigator</td>
  <td>29.5%</td>
  <td>107</td></tr>
<tr>
  <td>Other</td>
  <td>3%</td>
  <td>11</td></tr>
<tr><td colspan="2" rowspan="1">Total Respondents</td>
  <td>364</td></tr>
<tr><td colspan="2" rowspan="1">(skipped this question)</td>
  <td>0</td></tr>
</tbody>
</table>
<p>14. What kind of computer operating system do you primarily use to access GeneDB?</p>
<table border="1" cellpadding="1" cellspacing="1" style="width: 500px; height: 153px;">
<tbody>
<tr>
  <td>&nbsp;</td>
  <td>Response Percent</td>
  <td>Response Total</td></tr>
<tr>
  <td>PC/Windows</td>
  <td>49.3%</td>
  <td>179</td></tr>
<tr>
  <td>Mac</td>
  <td>47.1%</td>
  <td>171</td></tr>
<tr>
  <td>Linux/UNIX</td>
  <td>3.6%</td>
  <td>13</td></tr>
<tr>
  <td>Other</td>
  <td>0%</td>
  <td>0</td></tr>
<tr><td colspan="2" rowspan="1">Total Respondents</td>
  <td>364</td></tr>
<tr><td colspan="2" rowspan="1">(skipped this question)</td>
  <td>0</td></tr>
</tbody>
</table>
<p>15. Which Internet browser do you prefer?</p>
<table border="1" cellpadding="1" cellspacing="1" style="width: 500px; height: 153px;">
<tbody>
<tr>
  <td>&nbsp;</td>
  <td>Response Percent</td>
  <td>Response Total</td></tr>
<tr>
  <td>Internet Explorer</td>
  <td>41%</td>
  <td>149</td></tr>
<tr>
  <td>Netscape</td>
  <td>5%</td>
  <td>18</td></tr>
<tr>
  <td>Mozilla/Firefox</td>
  <td>27.5%</td>
  <td>100</td></tr>
<tr>
  <td>Safari</td>
  <td>25.6%</td>
  <td>93</td></tr>
<tr>
  <td>Other</td>
  <td>0.8%</td>
  <td>3</td></tr>
<tr><td colspan="2" rowspan="1">Total Respondents</td>
  <td>364</td></tr>
<tr><td colspan="2" rowspan="1">(skipped this question)</td>
  <td>0</td></tr>
</tbody>
</table>
<p>16. If you could add one feature to the database immediately, or improve one feature&nbsp; immediately, what would it be?</p>
<p>Total Respondents: <a href="/community/2006-survey-q16">135</a></p>
<p>(skipped this question) 228</p>
<p>17. How have the data and curation helped your research? Please provide specific examples and publications if possible. These examples will be helpful to secure future funding.</p>
<p>Total Respondents: <a href="/community/2006-survey-q17">144</a></p>
<p>(skipped this question) 219</p>
<p>18. Do you work experimentally on any other organisms in addition to fission yeast for your area of research, and if so which?&nbsp; (list all, including fission yeast, in order of relative importance to your work/degree of use)</p>
<p>Total Respondents: <a href="/community/2006-survey-q18">157</a></p>
<p>(skipped this question) 206</p>
<p>19. Please provide any additional comments and information about the current resource which you feel were not covered by this survey</p>
<p>Total Respondents: <a href="/community/2006-survey-q19">58</a></p>
<p>(skipped this question) 305</p>

  </div>
  </div>
</div>
<div *ngIf="section == 'datasets'">
<app-page-contents-menu title="Datasets" titleRoute="/datasets">

<div class="left-menu-part left-menu-item"><a routerLink="//downloads/genome-datasets">Genome datasets</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="//downloads/genome-datasets#Annotation_datasets">Annotation datasets</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/downloads/go-annotations">GO annotations</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/downloads/phenotype-annotations">Phenotype annotations</a></div>
<div class="left-menu-part left-sub-menu-item"><a href="ftp://ftp.pombase.org/pombe/annotations/modifications/">Modifications</a></div>
<div class="left-menu-part left-sub-menu-item"><a href="ftp://ftp.pombase.org/pombe/exports/">HCPIN datasets</a></div>
<div class="left-menu-part left-sub-menu-item"><a href="ftp://ftp.pombase.org/pombe/orthologs/">Orthologs</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="//downloads/protein-datasets">Protein datasets</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/downloads/names-and-identifiers">Names and IDs</a></div>
<div class="left-menu-part left-menu-item">GO slims</div>
<div class="left-menu-part left-sub-menu-item"><a href="ftp://ftp.pombase.org/nightly_update/misc/mf_goslim_pombe_ids_and_names.tsv">GO molecular function slim</a></div>
<div class="left-menu-part left-sub-menu-item"><a href="ftp://ftp.pombase.org/nightly_update/misc/bp_goslim_pombe_ids_and_names.tsv">GO biological process slim</a></div>
<div class="left-menu-part left-sub-menu-item"><a href="ftp://ftp.pombase.org/nightly_update/misc/cc_goslim_pombe_ids_and_names.tsv">GO cellular component slim</a></div>
<div class="left-menu-part left-menu-item"><a href="ftp://ftp.pombase.org/nightly_update/misc/disease_ontology_slim_ids_and_names.tsv">Mondo Disease Ontology slim</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documents">Documents</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/downloads/chado-database-dumps">Chado database dumps</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'index'">
<h2 id="datasets">Datasets</h2>
<div style="color: red">
<p><strong>Warning:</strong> Files in the old FTP site within the EBI domain are no longer updated. If you have a link that contains <strong>ftp.ebi.ac.uk</strong>, please check these pages for a link using <strong>ftp.pombase.org</strong>. Please <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a> if you need help finding a file or directory.</p>
</div>
<p>Links below go to web pages, except where “ftp” indicates that they go directly to ftp site directories. Note that the ftp subdirectories are organised slightly differently from the links on this page. If you have trouble finding anything, please ask the <a href="mailto:helpdesk@pombase.org">helpdesk</a>.</p>
<hr />
<h3 id="ftp-site">FTP site</h3>
<p><a href="ftp://ftp.pombase.org/pombe"><em>S. pombe</em> ftp site root directory</a></p>
<p>For FTP downloads, connect as “guest” if prompted; see <a href="faq/do-i-need-password-download-ftp-site">this FAQ</a> for more information.</p>
<hr />
<h3 id="genome-sequence-and-features"><a routerLink="/downloads/genome-datasets">Genome sequence and features</a></h3>
<ul>
<li>Genome sequence (FASTA or enriched EMBL format for Artemis)</li>
<li>Feature coordinates (coding regions, introns, exons, UTRs, etc.)</li>
<li>Genomic regions (centromeres, telomeres, mating type)</li>
</ul>
<hr />
<h3 id="annotation-datasets">Annotation datasets</h3>
<table>
<thead>
<tr class="header">
<th>Annotation type</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a routerLink="/downloads/protein-datasets">Protein datasets</a></td>
<td>Protein sequence FASTA database, peptide features, properties, etc.</td>
</tr>
<tr class="even">
<td><a routerLink="/downloads/go-annotations">GO annotations</a></td>
<td>Gene Ontology annotation files</td>
</tr>
<tr class="odd">
<td><a href="ftp://ftp.pombase.org/pombe/annotations/Gene_ontology/GO_complexes/">Macromolecular complexes</a></td>
<td>Subunits of protein and ribonucleoprotein complexes (GO cellular component terms and annotated genes; ftp)</td>
</tr>
<tr class="even">
<td><a routerLink="/downloads/phenotype-annotations">Phenotype annotations</a></td>
<td>FYPO phenotype annotation files - complete annotation set (PHAF) or viability summary</td>
</tr>
<tr class="odd">
<td><a href="ftp://ftp.pombase.org/pombe/high_confidence_physical_interactions/">HCPIN datasets</a></td>
<td>Physical interaction and GO substrate data that make up the High Confidence Physical Interaction Network datasets; also see <a routerLink="/documentation/high-confidence-physical-interaction-network">documentation</a> (ftp)</td>
</tr>
<tr class="even">
<td><a href="ftp://ftp.pombase.org/pombe/annotations/modifications/">Modifications</a></td>
<td>Protein modification data file (RNA modifications to be added in future; ftp)</td>
</tr>
<tr class="odd">
<td><a href="ftp://ftp.pombase.org/pombe/orthologs/">Orthologs</a></td>
<td>Manually curated ortholog sets for <a routerLink="/faq/how-can-i-find-s.-pombe-ortholog-s-human-gene">human</a> and <a routerLink="/faq/how-can-i-search-s.-cerevisiae-ortholog-s-homolog-s-s.-pombe-gene"><em>S. cerevisiae</em></a>; also see <a routerLink="/documentation/orthologs">documentation</a> (ftp)</td>
</tr>
</tbody>
</table>
<hr />
<h3 id="gene-names-and-ids"><a routerLink="/downloads/names-and-identifiers">Gene names and IDs</a></h3>
<p>Mappings between PomBase systematic IDs, gene names, product descriptions, and UniProt accession numbers</p>
<hr />
<h3 id="go-slim">GO slim</h3>
<p>Current GO slim IDs and term names <a href="ftp://ftp.pombase.org/nightly_update/misc/bp_goslim_pombe_ids_and_names.tsv">GO biological process slim</a> (ftp) <a href="ftp://ftp.pombase.org/nightly_update/misc/mf_goslim_pombe_ids_and_names.tsv">GO molecular function slim</a> (ftp) <a href="ftp://ftp.pombase.org/nightly_update/misc/cc_goslim_pombe_ids_and_names.tsv">GO cellular component slim</a> (ftp)</p>
<hr />
<h3 id="mondo-slim">Mondo slim</h3>
<p>Current <a href="ftp://ftp.pombase.org/nightly_update/misc/disease_ontology_slim_ids_and_names.tsv">fission yeast Mondo Disease Ontology slim IDs and term names</a> (ftp)</p>
<hr />
<h3 id="curated-inventories">Curated inventories</h3>
<ul>
<li><a routerLink="/browse-curation/dna-binding-sites">DNA binding sites</a></li>
<li><a routerLink="/browse-curation/drugs-known-pombe-targets">Drug targets</a></li>
</ul>

  </div>
  </div>
</div>
<div *ngIf="section == 'documentation'">
<app-page-contents-menu title="Documentation" titleRoute="/documentation">

<div class="left-menu-part left-menu-item">Gene pages</div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-basic-information">Basic information</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-gene-ontology">Gene Ontology</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-phenotypes">Phenotypes</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-target">Target of</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-protein-features">Protein features</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-transcript">Transcript</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-modifications">Modifications</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-gene-expression">Gene expression</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/genetic-and-physical-interactions">Genetic and physical interactions</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/orthologs">Ortholog curation</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/taxonomic-conservation">Taxonomic conservation</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/disease-association">Disease association</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-sequence">Sequence</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/controlled-curation">Controlled curation</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-external-references">External references</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/gene-page-literature">Literature</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/pombase-go-slim-documentation">GO Slim</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/JBrowse_quick_start">PomBase JBrowse</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/quick-little-tool">QuiLT visualisation</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/ontology-term-page">Ontology term pages</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/publication-page">Publication pages</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/genotype-page">Genotype pages</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/browse-curation/fission-yeast-phenotype-ontology">Fission Yeast Phenotype Ontology</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/simple-search-documentation">Simple search</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/advanced-search">Advanced search</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/motif-search">Motif search</a></div>
<div class="left-menu-part left-menu-item">Data submission file formats</div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/phenotype-data-bulk-upload-format">Phenotype data</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/modification-data-bulk-upload-format">Modification data</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/qualitative-gene-expression-data-bulk-upload-format">Qualitative gene expression data</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/quantitative-gene-expression-data-bulk-upload-format">Quantitative gene expression data</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/documentation/data-submission-form-for-HTP-sequence-linked-data">HTP data for browser tracks</a></div>


</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'Broad_dataset_email'">
<h3 id="email-import-of-the-fission-yeast-revisions-from-the-broad-institute-comparative-genome-paper-into-genedb">Email: Import of the fission yeast revisions from the Broad Institute comparative genome paper into GeneDB</h3>
<p>Originally sent by Val Wood to pombelist Sat Apr 23 16:00:37 BST 2011</p>
<p>The changes reported in PMID:21511999 are visible in GeneDB.</p>
<p>Gene Structure Changes</p>
<p>We have imported 306 of the gene structure changes. Some reported<br />
changes were rejected because :<br />
i) The change was already incorporated<br />
ii) The gene is known to be frame-shifted (the correct, albeit gene<br />
structure is currently represented) the confirmed frame-shifts will be<br />
fixed in the next round of sequence changes.<br />
iii) A different start site was reported, but homology or experimental<br />
evidence supports the existing coding sequence start site (if this was<br />
inconclusive I used the Broad start site)<br />
iv) The existing structure was already experimentally supported (in<br />
these cases alternative transcript features will be created once we move<br />
to the new PomBase)</p>
<p>Most of the changes are small, alterations to gene structures and<br />
include mainly:<br />
i) Additional in-frame introns<br />
ii) Use of an alternative starting methionine<br />
iii) Use of an alternative, but proximal acceptor (in these cases it<br />
is possible that both variants occur, but we have used the new<br />
experimentally supported one)<br />
iv) A couple of genes were “un-merged” and one merged.<br />
A small number of changes were larger, and involved additional N or C<br />
terminal exons, and occasionally gross changes to coding sequence.</p>
<p>As there are a large number of changes it is possible that your gene(s)<br />
of interest could be affected.<br />
All of the updated genes are annotated with:<br />
“warning, gene structure updated”PMID:xxxxxxxx" (this will change to<br />
PMID:21511999 when GeneDB is next updated)<br />
in the “controlled curation” section of the gene page</p>
<p>New Genes<br />
New gene structures have been created for the new genes reported<br />
All new genes identified in this analysis are annotated with:<br />
“warning, new gene PMID:xxxxxxxx” (this will change to PMID:21511999<br />
when GeneDB is next updated)<br />
in the “controlled curation” section of the gene page</p>
<p>ncRNAs<br />
All non coding RNAs were imported.<br />
Some are clearly coincident with existing predicted RNAs (However, some<br />
of the previously annotated RNA’s may appear on the incorrect strand as<br />
they were derived from Affymetrix data),<br />
Over time, these will be assessed and merged where appropriate.</p>
<p>UTRs<br />
The UTRs derived from the Broad data have been used for the consensus<br />
UTR set, the precedence for the current annotated consensus set is now :<br />
Individual published sources&gt; PMID:21511999 &gt;PMID:20118936<br />
We hope to make the entire datasets available in the new PomBase.</p>
<p>If you have any questions, please direct them to the new helpdesk,<br />
helpdesk at pombase.ac.uk, or send your query directly using the curator<br />
feedback on the GeneDB web page.</p>

  </div>
  <div *ngIf="pageName == 'JBrowse_quick_start'">
<h2 id="getting-started-with-pombase-jbrowse">Getting started with PomBase JBrowse</h2>
<p>The PomBase genome browser can be accessed in multiple ways:</p>
<ul>
<li>On any PomBase page, click the “Search” tab in the header menu, and select “Genome browser”</li>
<li>From a gene page, click the “View region in JBrowse …” link just above the map image</li>
<li>Go directly to https://www.pombase.org/jbrowse/</li>
</ul>
<h3 id="dna-sequence-forward-features-and-reverse-features">DNA sequence, forward features and reverse features</h3>
<p>Tracks for annotated sequence features — genes, repeats, etc. — on the forward and reverse strands are displayed by default.</p>
<p>To show the track containing the reference DNA sequence, use the “Genome sequence and features” filter and enable the “DNA sequence” track as described below.</p>
<h3 id="enabling-data-tracks">Enabling data tracks</h3>
<p>To show data tracks:</p>
<ol type="1">
<li><p>Click on the “Select tracks” button in the top left corner.</p>
<figure>
<img src="assets/jbrowse_main_top_track_button.png" width="700" alt="" /><figcaption>JBrowse top, with “select tracks” button highlighted</figcaption>
</figure></li>
<li><p>Locate the track(s) of interest (see below), and then click the tickbox for any individual track to toggle it on/off, or use the tickbox in the header to enable/disable all of the listed tracks at once.</p></li>
<li><p>Click the “Back to browser” button at the top.</p></li>
<li><p>The tracks should now be displayed. For example, the sequence feature and DNA sequence tracks look like this:</p>
<figure>
<img src="assets/jbrowse_seq_feature_tracks.png" width="700" alt="" /><figcaption>JBrowse display of pombe sequence and features</figcaption>
</figure></li>
</ol>
<p>To hide a track, click the (X) button to the left of the track label, or go to the “Select tracks” page and clear its tickbox.</p>
<p>Drag and drop tracks to change the order in which they appear.</p>
<h4 id="changing-the-track-scale">Changing the track scale</h4>
<p>By default, the scale fits the range of the data being displayed in the current viewing window. The scale can be manually defined.</p>
<p>Click the downarrow to the right of the track label, and click on edit config:</p>
<figure>
<img src="assets/jbrowse_track_label_config.png" width="700" alt="" /><figcaption>Jbrowse edit config</figcaption>
</figure>
<p>Change the scale by defining a max_score and/or a min score:</p>
<figure>
<img src="assets/jbrowse_max_score.png" width="400" alt="" /><figcaption>JBrowse_max_score defined</figcaption>
</figure>
<p>The scale can be made consistent across tracks:</p>
<p><img src="assets/jbrowse_rescaled.png" width="700" alt="rescaled tracks" /> jbrowse_rescaled.png</p>
<p>More wiggle track configuration option in the official <a href="http://gmod.org/wiki/JBrowse_Configuration_Guide#Wiggle_track_configuration_options">JBrowse documentation</a></p>
<h4 id="locating-data">Locating data</h4>
<p>Each entry in the left-hand column can be used to filter the track list. Filter criteria fall into several categories, corresponding to some of the track metadata (see below):</p>
<figure>
<img src="assets/jbrowse_track_selector.png" width="450" alt="" /><figcaption>JBrowse track selection interface</figcaption>
</figure>
<p>As you select filters, the screen will update to show only relevant criteria on the left, and matching tracks on the right.</p>
<p>You can also search for names, keywords, etc. using the “Contains text” box.</p>
<p>Any applied filters, including text, can be cleared by clicking the red X button that appears next to a selected filter (in the search box or the left-hand column), or by using the “Clear All Filters” button.</p>
<p>For a list of recently enabled tracks, click “Recently Used”.</p>
<h4 id="track-metadata">Track metadata</h4>
<p>Information about each track is captured as part of the curation process. Metadata includes:</p>
<ul>
<li>Data type: what the experiment measured</li>
<li>Track label (name): Brief description that appears with the track in the browser display</li>
<li>Alleles: data from wild type or mutant cells</li>
<li>Growth phase or response: data from cells assayed a specific stress response or growth phase (e.g. nitrogen starvation, oxidative stress, vegetative growth)</li>
<li>Localized gene product: the protein assayed in a chromatin binding experiment</li>
<li>Strain background mutations: any background mutations present in the strain for experimental purposes only and not considered as influencing the results in themselves, such as auxotrophic markers (e.g. <em>leu1-1</em>) or alleles utilized for block and release experiments (e.g. <em>cdc25-22</em>)</li>
<li>Mating type and ploidy</li>
<li>Conditions: experimental conditions, e.g. whether cells were grown in YES or EMM, or if H<sub>2</sub>O<sub>2</sub> or t-BOOH was used to induce oxidative stress</li>
<li>Study ID: accession for the study encompassing the data in external databases (e.g. GEO, ArrayExpress)</li>
<li>Sample ID: accession for the sample used to generate the data in external databases (e.g. GEO, ArrayExpress)</li>
<li>First author name</li>
<li>PubMed ID: paper in which the data were published</li>
<li>Publication year</li>
</ul>
<p>The metadata can be viewed in two ways:</p>
<ol type="1">
<li><p>In the track selector, tracks are listed with associated metadata on the right hand side:</p>
<figure>
<img src="assets/jbrowse_track_selection_top.png" width="750" alt="" /><figcaption>JBrowse track selector with metadata headers highlighted</figcaption>
</figure></li>
<li><p>In the browser, hover over a track label to display a down arrow:</p>
<figure>
<img src="assets/jbrowse_track_label_hover.png" width="300" alt="" /><figcaption>JBrowse track label</figcaption>
</figure>
<p>Click the arrow, then click “About this track” in the dropdown menu:</p>
<figure>
<img src="assets/jbrowse_track_menu_about.png" width="400" alt="" /><figcaption>JBrowse track dropdown menu</figcaption>
</figure>
<p>The metadata and stats are displayed:</p>
<figure>
<img src="assets/jbrowse_metadata_popup.png" width="400" alt="" /><figcaption>JBrowse track metadata popup</figcaption>
</figure>
<p>To dismiss the popup, click the ‘X’ in the upper right corner or the ‘OK’ button at the bottom.</p></li>
</ol>
<h3 id="navigating-the-browser">Navigating the browser</h3>
<h4 id="moving-around">Moving around</h4>
<p>There are several ways to navigate the genome.</p>
<ul>
<li><p>Use the box at the top that displays the chromosomal coordinates to navigate to a specified location: <img src="assets/jbrowse_main_top_coords.png" width="600" alt="JBrowse top of main display" /></p>
<p>In the coordinates box,</p>
<ul>
<li>Type a gene name, or</li>
<li>Select a chromosome in the pulldown, then type or paste a range of coordinates (two numbers separated by two dots, e.g. 2323258..2336458).
<ul>
<li>If you select one of the pulldown options but omit coordinates, the browser goes to a default location spanning most of the selected chromosome.</li>
<li>You can also type both a chromosome ID and coordinates directly in the location box, e.g II:2323258..2336458. Use Roman numerals (I, II, III) for the three nuclear chromosomes.</li>
</ul></li>
<li>Click the “Go” button or hit return to go to the gene or region.</li>
<li>Coordinates for some regions of interest:
<ul>
<li>Centromeres
<ul>
<li>I:3753687..3789421</li>
<li>II:1602264..1644747</li>
<li>III:1070904..1137003</li>
</ul></li>
<li>Mating type region
<ul>
<li>II:2129208..2137121</li>
</ul></li>
<li>rDNA (link to one complete repeat)
<ul>
<li>III:5542..13722</li>
</ul></li>
</ul></li>
<li>For more information on the chromosome selection options, see PomBase documentation on <a routerLink="/status/sequencing-status">genome sequencing status</a>, the <a routerLink="/status/mating-type-region">mating type region</a>, and <a routerLink="/faq/there-any-rdna-repeat-sequences-pombase">rDNA</a>.</li>
</ul></li>
<li><p>In the uppermost bar, a red box indicates the size and position of the currently displayed region, in the context of the whole chromosome. Click in the bar or drag the red box to change position.</p>
<figure>
<img src="assets/jbrowse_position_bar.png" width="500" alt="" /><figcaption>JBrowse position bar</figcaption>
</figure></li>
<li><p>Use the arrows next to the bar with the box for chromosomal coordinates to move right or left in increments.</p></li>
<li><p>Drag or swipe (depending on device) in the main browser window to scroll.</p></li>
</ul>
<h4 id="zooming">Zooming</h4>
<p>To zoom:</p>
<ul>
<li><p>Use the + and - buttons, or</p></li>
<li><p>In the main browser window, double-click to zoom in, and shift-double-click to zoom out, or</p></li>
<li><p>Use the narrow bar below the chromosomal coordinates:</p>
<figure>
<img src="assets/jbrowse_region_selection_bar.png" width="300" alt="" /><figcaption>JBrowse region selection bar</figcaption>
</figure>
<p>Click and drag to highlight a region, then click within the highlighted region to zoom so the region fills the browser window:</p>
<figure>
<img src="assets//jbrowse_region_zoom.png" width="500" alt="" /><figcaption>JBrowse zoom in on selected region</figcaption>
</figure></li>
</ul>
<p>To see the DNA sequence, zoom in until it appears, first as color-coded blocks, and then labeled with letters.</p>
<p>More <em>S. pombe</em>-specific information can be found in the <a href="https://www.pombase.org/faq/genome-browser">Genome browser FAQ list</a>.</p>
<h3 id="exporting-data">Exporting data</h3>
<p>To export the data for any track, hover over the track label to display a down arrow:</p>
<figure>
<img src="assets/jbrowse_track_label_hover.png" width="300" alt="" /><figcaption>JBrowse track label</figcaption>
</figure>
<p>Click the arrow, then click “Save track data” in the dropdown menu:</p>
<figure>
<img src="assets/jbrowse_track_menu_save.png" width="400" alt="" /><figcaption>JBrowse track dropdown menu save data</figcaption>
</figure>
<p>In the popup, choose a data format. GFF3 is available for most tracks; other options depend on the data type. You can also edit the name of the file where the data will be saved.</p>
<figure>
<img src="assets/jbrowse_metadata_popup.png" width="400" alt="" /><figcaption>JBrowse track metadata popup</figcaption>
</figure>
<p>Click ‘View’ to see how the data will appear in the saved file. To download the data file click ‘Save’ (available in the original popup or the ‘View’ panel). To dismiss the popup withous saving, click the ‘X’ in the upper right corner or the ‘Close’ button at the bottom.</p>

  </div>
  <div *ngIf="pageName == 'Lanterman_Dutrow_dataset_email'">
<h3 id="email-annotated-transcription-start-and-termination-sites-for-fission-yeast.">Email: Annotated transcription start and termination sites for fission yeast.</h3>
<p>Originally sent by Val Wood to pombelist Fri Feb 4 12:08:34 GMT 2011</p>
<p>A set of genome wide transcription start and termination sites manually<br />
annotated by Lantermann et. al in PMID: 20118936, based on<br />
transcriptome data from Dutrow et. al. PMID: 18641648 have been used to<br />
create 5’ and 3’UTR features for the reference genome, for genes with no<br />
previously annotated transcript from another source.</p>
<p>This increases the number of manually annotated UTRs from<br />
298 to 3605 for 5’<br />
778 to 3693 for 3’</p>
<p>Because it appears that transcription start and end can be fuzzy,<br />
these features should be considered as an “indicator” of the likely<br />
extent of transcription. We will continue to revise the reference<br />
transcript set as updates are reported, to represent the longest<br />
discrete transcript for a region as the consensus.</p>
<p>Data Access<br />
You can access the transcription start and end from individual GeneDB<br />
gene pages for genes of interest using the sequence viewers GBrowse or<br />
Artemis. The links to these 2 browsers are to the right and left of the<br />
“context map” on the “Location” section of the gene page.<br />
(Artemis tips: Use the “Artemis applet” option selected; If you are<br />
interested in a specific gene, reduce the upstream and downstream range<br />
of to 1000 before clicking “submit”; the page will take a few seconds<br />
to render the first time in a session)</p>
<p>You can download this data as part of the complete annotation in the “rich EMBL files” from here: ftp://ftp.sanger.ac.uk/pub/yeast/pombe/Chromosome_contigs/ (the files chromosome*.contig) and view them in Artemis on your desktop (see the <a routerLink="/faq/there-equivalent-artemis-java-applet-pombase">Artemis FAQ</a> and the <a href="ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/artemis.pdf">Artemis manual</a> (pdf; Sanger site) for additional information).</p>
<p>Hosting Further Transcriptome Datasets<br />
Clearly, as more transcriptome data becomes available the community will<br />
require access to the different datasets, in addition to the consensus<br />
transcripts. We intend to provide this in the future through the new<br />
PomBase using the Ensembl viewer.</p>

  </div>
  <div *ngIf="pageName == 'advanced-search'">
<h2 id="advanced-search">Advanced search</h2>
<p><strong>Note: This documentation is still a work in progress. Please contact the <a href="mailto:helpdesk@pombase.org">helpdesk</a> if you have any questions.</strong></p>
<p>The PomBase advanced search allows you to construct complex queries for <em>S. pombe</em> genes. At present all advanced search results are lists of genes that match the query specifications. Hints for specific searches are available in the <a routerLink="/faq">PomBase FAQ</a>, and linked <a routerLink="/#search-tips">below</a>.</p>
<h3 id="overview">Overview</h3>
<p>In the Search menu, choose Advanced (or bookmark http://www.pombase.org/query).</p>
<p>In the Query panel (top), click one of the links in the list on the left to choose a <a routerLink="/query-types">query type</a>. For each query, the interface guides your input.</p>
<figure>
<img src="assets/advanced_search_main.png" width="682" alt="" /><figcaption>advanced search page with new query</figcaption>
</figure>
<p>In the History panel (bottom), queries are listed in reverse chronological order (most recently run first). Results are linked in the Count column; the link goes to a page that displays the gene name, systematic ID, and product description for matching genes. The Download button offers additional options, including nucleotide and (where applicable) protein sequence. To return to the search page, use the blue “&lt;- Advanced search” button just above the result list, <strong>not</strong> your browser’s “Back” button.</p>
<p>You can select queries in the list to combine or delete them. To combine queries, you must select two or more from the list, and then click one of the operator buttons:</p>
<figure>
<img src="assets/advanced_search_combination.png" width="400" alt="" /><figcaption>advanced search page with new query</figcaption>
</figure>
<p>Results are added to the history list:</p>
<figure>
<img src="assets/advanced_search_combination_result.png" width="400" alt="" /><figcaption>advanced search page with new query</figcaption>
</figure>
<p>At present, there is only an awkward kludge to specify the order for a query using the Subtract/NOT operator. It always places the more recently run query first (newer NOT/Subtract older), but you can move any query to the top of the history list by clicking its “Results” link.</p>
<p>Query results can remain available in the history list for several days. If changes in annotation cause results to change since a query was last run, the entry in the history list will be labeled “out of date”:</p>
<figure>
<img src="assets/advanced_search_out-of-date_result.png" width="400" alt="" /><figcaption>advanced search page with out-of-date query</figcaption>
</figure>
<p>Click on the “Results” link to re-run the query and retrieve the up-to-date result.</p>
<p>If the parameters for a query are not all visible in the history, a “[details]” toggle (1) to show or hide additional details will appear. You can also edit the name that appears in the history list (2). The name will be used as part of the description when the query is used in any combined query (which can, in turn, be given a new name).</p>
<figure>
<img src="assets/query_name_details.png" width="400" alt="" /><figcaption>advanced search query name and details</figcaption>
</figure>
<h3 id="query-results-page">Query results page</h3>
<p>Clicking on any up-to-date count in the “Results” column goes to a page that shows the count, query details, and a list of matching genes.</p>
<figure>
<img src="assets/advanced_search_results.png" width="400" alt="" /><figcaption>advanced search results page</figcaption>
</figure>
<ol type="1">
<li><p>Return to the main Advanced search page (you can also use the browser “back” button)</p></li>
<li><p>Click on a header to sort by the column</p></li>
<li><p>Select columns to display</p></li>
<li><p>Send the gene list to <a routerLink="/documentation/quick-little-tool">QuiLT</a> for visualisation</p></li>
<li><p>Show <a routerLink="/documentation/pombase-go-slim-documentation">GO slim</a> annotations for genes in the list. Click “Slim with” to reveal a dropdown, and choose a slim. Note: when you return to the main Advanced search page, the query history will include an entry showing the query as a gene list. This is harmless.</p></li>
<li><p>Narrow the list: click the button to reveal checkboxes</p></li>
<li><p>Download selected data for genes in the list. In the popup, the default “Tab delimited” view offers a set of details that can be included in a downloaded text file. Click “Sequence” to retrieve amino acid or nucleotide sequences in FASTA format. Use the checkboxes to select which items are included in the headers. When “Nucleotide” is selected, flanking sequence options similar to those on the <a href="documentation/gene-page-sequence">gene page</a> are available.</p></li>
</ol>
<h4 id="sharing-search-results">Sharing search results</h4>
<p>Each results page has a stable, unique URL that you can bookmark, copy/paste, and share. Anyone who follows a shared link will see the same results page, and the query will be added to their query history.</p>
<p>The “Visualise” and “Slim” options also generate stable, unique, sharable URLs.</p>
<h3 id="query-types">Query types</h3>
<p>Ontology queries (GO, phenotype, protein modification, and protein sequence feature) retrieve genes that are annotated to the selected term(s), and to their descendants via specified relations (see the GO documentation on <a href="http://geneontology.org/docs/ontology-relations/">Ontology Structure and Ontology Relations</a> for more information on relationships between terms in ontologies, and see the descriptions below of specific ontology filters for lists of which relations are followed to retrieve annotations). Ontologies can be searched by ID or term name. Type or paste a complete ontology ID, including the prefix (e.g. GO:0005634, FYPO:0002059), or simply start typing to search for a term name. Choose a term from the list of options offered by the autocomplete, and click “Submit”.</p>
<h4 id="commonly-used-queries">Commonly used queries</h4>
<p>This item offers convenient links to perform frequently used queries easily. Click on the query description to add the results to the query history.</p>
<p>Please contact the <a href="mailto:helpdesk@pombase.org">helpdesk</a> if you would like any queries added to the selection.</p>
<h4 id="gene-names-and-ids">Gene names and IDs</h4>
<p>Type or paste a list of gene names or systematic IDs (or a mixture), or click the “Browse” button to select a file to upload. Click “Lookup” to add the gene list to the history (useful for combining the list with other queries).</p>
<h4 id="go">GO</h4>
<p>The Gene Ontology (GO) query retrieves gene products annotated to a GO term and to any of its child terms, following the <em>is_a</em>, <em>part_of</em>, <em>regulates</em>, <em>positively_regulates</em>,and <em>negatively_regulates</em> relationships in the ontology. You may also find it helpful to search or browse in <a href="http://www.ebi.ac.uk/QuickGO/">QuickGO</a> or <a href="http://amigo.geneontology.org/">AmiGO</a> to find GO terms of interest. If one search does not seem to retrieve as many results as you expect, try again using a less specific term. Note: prior to the November 2014 PomBase release, the <em>regulates</em> relations were not followed, and PomBase GO search results therefore did not match those in AmiGO.</p>
<h4 id="phenotype">Phenotype</h4>
<p>The phenotype (Fission Yeast Phenotype Ontology) query retrieves genes annotated to a FYPO term and to any of its child terms, following the <em>is_a</em>, <em>part_of</em>, <em>output_of</em>, <em>has_output</em>, and <em>has_part</em> relationships in the ontology. This query also offers three additional sets of options:</p>
<figure>
<img src="assets/fypo_search_options.png" width="651" alt="" /><figcaption>phenotype search options</figcaption>
</figure>
<p><strong>Alleles:</strong> By default, the phenotype search retrieves genes from single-allele genotypes annotated to the searched FYPO term. You can alter the selected options to add genes from multi-allele genotypes. See the <a href="documentation/gene-page-phenotypes">gene page phenotype documentation</a> and the <a href="documentation/genotype-page">genotype page documentation</a> for more information.</p>
<p><strong>Expression:</strong> Different alleles of one gene may have different phenotypes, and one allele may give rise to different phenotypes under different experimental conditions. At present, you can retrieve annotations for all alleles of a gene, or use the “Expression level” options restrict the query to null alleles (covers deletions and any other sequence changes, such as most disruptions, that completely abolish expression of the gene) or overexpression of the wild type allele.</p>
<p><strong>Conditions:</strong> The “Constrain condition” option restricts the results to include only genes that have phenotype annotations including the specified condition. The search uses the same condition descriptors as Canto and the PomBase web pages. Start typing, then choose from the autocomplete options.</p>
<p>Note that the results will include any gene that has phenotype annotations including the specified condition <em>for any allele</em>. Queries that include conditions can be combined using the AND, NOT, or OR operators like any other, but the result of any combination of phenotype queries will likely include annotations for different alleles. There may not be any individual annotation in which both/all of multiple conditions co-occur.</p>
<p>The search does not yet support querying for multiple conditions on the same annotation, nor for queries that <em>exclude</em> a given condition; both are planned for future development.</p>
<h4 id="product-type">Product type</h4>
<p>Choose a gene product type (e.g. protein coding, tRNA, etc.) from the pulldown menu.</p>
<h4 id="protein-modification">Protein modification</h4>
<p>Search for terms or IDs in the <a href="http://obofoundry.org/ontology/mod.html">PSI-MOD</a> ontology.</p>
<h4 id="protein-domain">Protein domain</h4>
<p>Search for an ID from Pfam, PRINTs, PROFILE, ProSite, or InterPro. Type or paste an accession and click “Submit”.</p>
<h4 id="protein-feature">Protein feature</h4>
<p>This query searches for terms or IDs in the <a href="http://sequenceontology.org/">Sequence Ontology</a> and retrieves protein-coding genes where the protein has the feature represented by the SO term (e.g. <a routerLink="/term/SO:0001807">KEN box SO:0001807</a> ).</p>
<h4 id="protein-length">Protein length</h4>
<p>Find protein-coding genes with products in a specified length range. Enter the desired minimum and maximum length and click “Search”.</p>
<h4 id="protein-mol.-weight">Protein mol. weight</h4>
<p>Find protein-coding genes with products in a specified mass range. Enter the desired minimum and maximum mass in kiloDaltons (kDa) and click “Search”.</p>
<h4 id="disease">Disease</h4>
<p>Search the Monarch Disease Ontology to find <em>S. pombe</em> genes whose human orthologs have been implicated in disease. Start typing ‘disease’ or the name of a specific disease, and choose from the autocomplete options. To retrieve all disease-associated genes, type or paste “MONDO:0000001”.</p>
<h4 id="number-of-tm-domains">Number of TM domains</h4>
<p>Find protein-coding genes with products that have a specified number of transmembrane domains. Enter the desired minimum and maximum number and click “Search”.</p>
<h4 id="genome-location">Genome location</h4>
<p>Find genes in a specified region of a chromosome. Select a chromosome from the pulldown, and click “Search” to retrieve all genes on the chromosome. Optional: enter start and end coordinates in the boxes to retrieve only genes in the indicated region.</p>
<h4 id="coding-exons">Coding exons</h4>
<p>Find protein-coding genes with a specified number of coding exons. Enter the desired minimum and maximum number and click “Search”.</p>
<h4 id="taxonomic-conservation">Taxonomic conservation</h4>
<p>Choose one of the descriptions from the pulldown menu. See the <a routerLink="/documentation/taxonomic-conservation">gene page documentation</a> for more information.</p>
<h4 id="characterisation-status">Characterisation status</h4>
<p>Choose one of the descriptions from the pulldown menu. See the <a routerLink="/status/gene-characterisation">gene characterisation page</a> for more information.</p>
<h3 id="search-tips">Search tips</h3>
<p><a routerLink="/faq">FAQ</a> entries relevant to using the advanced search are organised here by topic. Several of the topics also correspond to gene page sections.</p>
<h4 id="phenotype-searches">Phenotype searches</h4>
<ul>
<li><a routerLink="/faq/can-i-get-list-essential-pombe-genes">Can I get a list of essential pombe genes?</a></li>
<li><a routerLink="/faq/how-can-i-find-all-genes-have-given-mutant-phenotype">How can I find all of the genes that have a given mutant phenotype?</a></li>
<li><a routerLink="/faq/why-are-some-genes-annotated-both-viable-and-inviable-phenotypes">Why are some genes annotated to both viable and inviable phenotypes?</a></li>
<li><a routerLink="/faq/how-can-i-browse-phenotype-ontology-fypo">How can I browse the phenotype ontology (FYPO)?</a></li>
</ul>
<h4 id="gene-ontology-searches">Gene Ontology searches</h4>
<ul>
<li><a routerLink="/faq/how-can-i-search-or-browse-go-annotations">How can I search or browse GO annotations?</a></li>
<li><a routerLink="/faq/how-can-i-find-genes-specific-activity">How can I find genes with a specific activity?</a></li>
<li><a routerLink="/faq/how-can-i-find-protein-localization-data">How can I find protein localization data?</a></li>
<li><a routerLink="/faq/how-can-i-identify-all-genes-affect-process">How can I identify all of the genes that affect a process?</a> (also relevant to phenotype searches)</li>
<li><a routerLink="/faq/can-i-retrieve-functional-annotations-genes-list">Can I retrieve functional annotations for genes in a list?</a></li>
<li><a routerLink="/faq/how-can-i-find-transcription-factors-and-their-targets-pombase">How can I find transcription factors and their targets in PomBase?</a></li>
</ul>
<h4 id="protein-searches">Protein searches</h4>
<ul>
<li><a routerLink="/faq/how-can-i-find-modifications-my-protein-interest">How can I find modifications for my protein of interest?</a></li>
<li><a routerLink="/faq/how-can-i-find-all-s.-pombe-proteins-particular-protein-family">How can I find all S. pombe proteins in a particular protein family?</a></li>
<li><a routerLink="/faq/how-can-i-find-proteins-have-transmembrane-domains">How can I find proteins that have transmembrane domains?</a></li>
</ul>
<h4 id="taxonomic-conservation-1">Taxonomic conservation</h4>
<ul>
<li><a routerLink="/faq/how-can-i-find-s.-pombe-genes-associated-human-disease">How can I find S. pombe genes associated with human disease?</a></li>
<li><a routerLink="/faq/how-can-i-find-all-s.-pombe-genes-are-conserved-human">How can I find all S. pombe genes that are conserved in human?</a></li>
<li><a routerLink="/faq/can-i-search-genes-based-on-conservation-different-taxa">Can I search for genes based on conservation in different taxa?</a></li>
<li><a routerLink="/faq/can-i-find-all-unconserved-orphan-genes-fission-yeast">Can I find all of the unconserved (orphan) genes in fission yeast?</a></li>
</ul>
<h4 id="gene-and-genome-searches">Gene and genome searches</h4>
<ul>
<li><a routerLink="/faq/how-can-i-retrieve-all-s.-pombe-genes">How can I retrieve all S. pombe genes?</a></li>
<li><a routerLink="/faq/how-can-i-retrieve-all-s.-pombe-protein-coding-genes">How can I retrieve all S. pombe protein-coding genes?</a></li>
<li><a routerLink="/faq/can-i-provide-list-genes-search-on">Can I provide a list of genes to search on?</a></li>
<li><a routerLink="/faq/can-i-search-gene-list-and-retrieve-results-same-order-as-input-list">Can I search for a gene list and retrieve results in the same order as in the input list?</a> (answer: no)</li>
<li><a routerLink="/faq/how-can-i-find-snorna-genes">How can I find snoRNA genes?</a></li>
<li><a routerLink="/faq/how-can-i-find-transposons-s.-pombe-genome">How can I find transposons in the S. pombe genome?</a></li>
<li><a routerLink="/faq/how-can-i-find-genes-region-using-chromosome-coordinates">How can I find genes in a region using chromosome coordinates?</a></li>
<li><a routerLink="/faq/how-can-i-retrieve-intron-coordinates-or-sequences">How can I retrieve intron coordinates or sequences?</a></li>
<li><a routerLink="/faq/how-can-i-find-all-sequence-features-region-using-chromosome-coordinates">How can I find all sequence features in a region using chromosome coordinates?</a></li>
<li><a routerLink="/faq/how-can-i-find-genes-region-using-chromosome-coordinates">How can I find genes in a region using chromosome coordinates?</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'annotation-extension-relation-display'">
<h2 id="annotation-extension-relation-display">Annotation extension relation display</h2>
<p>Annotation extensions are used to provide additional specificity for annotations to GO, FYPO, and protein modification annotations. Each extension consists of a relation and another “entity”, which may be a gene in PomBase or another database, or another ontology term. (The <a href="http://wiki.geneontology.org/index.php/Annotation_Extension">GO Consortium wiki page on annotation extensions</a> contains useful information on relations.) Because some relations have unwieldy names, the PomBase gene page display substitutes more readable text.</p>
<p>This table shows the underlying relation name and corresponding display text for the affected relations, and the reciprocal relation displayed in “Target of” annotations (derived from extension data):</p>
<table>
<thead>
<tr class="header">
<th>Relation name</th>
<th>Display name</th>
<th>Reciprocal</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>activated_by</td>
<td>activated by</td>
<td>N/A</td>
</tr>
<tr class="even">
<td>coincident_with</td>
<td>at</td>
<td>N/A</td>
</tr>
<tr class="odd">
<td>during</td>
<td>during</td>
<td>N/A</td>
</tr>
<tr class="even">
<td>exists_during</td>
<td>during</td>
<td>N/A</td>
</tr>
<tr class="odd">
<td>happens_during</td>
<td>during</td>
<td>N/A</td>
</tr>
<tr class="even">
<td>has_regulation_target</td>
<td>regulates</td>
<td>regulated by</td>
</tr>
<tr class="odd">
<td>in_absence_of</td>
<td>in absence of</td>
<td>N/A</td>
</tr>
<tr class="even">
<td>in_presence_of</td>
<td>in presence of</td>
<td>N/A</td>
</tr>
<tr class="odd">
<td>inhibited_by</td>
<td>inhibited by</td>
<td>N/A</td>
</tr>
<tr class="even">
<td>not_exists_during</td>
<td>except during</td>
<td>N/A</td>
</tr>
<tr class="odd">
<td>not_happens_during</td>
<td>except during</td>
<td>N/A</td>
</tr>
<tr class="even">
<td>occurs_at</td>
<td>at</td>
<td>N/A</td>
</tr>
<tr class="odd">
<td>occurs_in</td>
<td>in</td>
<td>N/A</td>
</tr>
<tr class="even">
<td>required_for</td>
<td>required for</td>
<td>N/A</td>
</tr>
</tbody>
</table>
<p>For some relations, the best display string depends on term and its ancestry, as indicated in these tables:</p>
<p>The display for FYPO extensions indicating genes used in assays depends on whether the phenotype is normal or abnormal:</p>
<table>
<thead>
<tr class="header">
<th>Relation name</th>
<th>If descendant of</th>
<th>Display name</th>
<th>Reciprocal</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>assayed_using</td>
<td>FYPO:0001985 abnormal phenotype</td>
<td>affecting</td>
<td>affected by mutation in</td>
</tr>
<tr class="even">
<td>assayed_enzyme</td>
<td>FYPO:0001985 abnormal phenotype</td>
<td>affecting activity of</td>
<td>activity affected by mutation in</td>
</tr>
<tr class="odd">
<td>assayed_substrate</td>
<td>FYPO:0001985 abnormal phenotype</td>
<td>affecting substrate</td>
<td>affected by mutation in</td>
</tr>
<tr class="even">
<td>assayed_using</td>
<td>FYPO:0000257 normal phenotype</td>
<td>affecting</td>
<td>N/A (normal means no effect to report in “Target of” section)</td>
</tr>
<tr class="odd">
<td>assayed_enzyme</td>
<td>FYPO:0000257 normal phenotype</td>
<td>affecting activity of</td>
<td>N/A (normal means no effect to report in “Target of” section)</td>
</tr>
<tr class="even">
<td>assayed_substrate</td>
<td>FYPO:0000257 normal phenotype</td>
<td>affecting substrate</td>
<td>N/A (normal means no effect to report in “Target of” section)</td>
</tr>
</tbody>
</table>
<p>GO annotation extensions display for part_of:</p>
<table>
<thead>
<tr class="header">
<th>If descendant of</th>
<th>Display name</th>
<th>Reciprocal</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>GO:0003674 molecular_function</td>
<td>involved in</td>
<td>N/A</td>
</tr>
<tr class="even">
<td>GO:0008150 biological process</td>
<td>part of</td>
<td>N/A</td>
</tr>
</tbody>
</table>
<p>The display for the “input” relations (has_input or has_direct_input) follows this pattern:</p>
<table>
<thead>
<tr class="header">
<th>If descendant of</th>
<th>Display name</th>
<th>Reciprocal</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>GO:0003824 catalytic activity</td>
<td>has substrate</td>
<td>substrate of</td>
</tr>
<tr class="even">
<td>GO:0006810 transport</td>
<td>transports</td>
<td>transported by</td>
</tr>
<tr class="odd">
<td>GO:0005215 transporter activity</td>
<td>transports</td>
<td>transported by</td>
</tr>
<tr class="even">
<td>GO:0030234 enzyme regulator activity</td>
<td>regulates</td>
<td>regulated by</td>
</tr>
<tr class="odd">
<td>GO:0008047 enzyme activator activity</td>
<td>positively regulates</td>
<td>positively regulated by</td>
</tr>
<tr class="even">
<td>GO:0004857 enzyme inhibitor activity</td>
<td>negatively regulates</td>
<td>negatively regulated by</td>
</tr>
<tr class="odd">
<td>GO:0051179 localization</td>
<td>localizes</td>
<td>localized by</td>
</tr>
<tr class="even">
<td>GO:0005488 binding</td>
<td>binds</td>
<td>binds</td>
</tr>
<tr class="odd">
<td>(anything not listed above)</td>
<td>has input</td>
<td>input for</td>
</tr>
</tbody>
</table>

  </div>
  <div *ngIf="pageName == 'controlled-curation'">
<h2 id="pombase-controlled-curation">PomBase controlled curation</h2>
<p>Gene and publication pages include manually curated annotations of several types that cannot be represented using available ontologies.</p>
<p>For each annotation, the summary view shows a text description, which corresponds to an entry in the internal <!-- <a routerLink="/PBO">/faq/what-pbo-option-advanced-search</a> --> PB term set. The detailed view adds the PBO ID, evidence (if available), reference, and count. The count links to the <a routerLink="/documentation/ontology-term-page">ontology term page</a> for the description.</p>
<p>Links in the left-hand navigation menu include:</p>
<ul>
<li><p><strong>Complementation</strong> - Indicates that an <em>S. pombe</em> gene complements, or is complemented by, a gene from another species</p></li>
<li><p><strong>Miscellaneous</strong></p></li>
<li><p><strong>Subunit composition</strong> - Brief description of homo- or heteromeric complex(es) in which the gene product is found</p></li>
<li><p><strong>Warnings</strong> - Alerts to changes or anomalies that can affect interpretation of experimental results, such as changes to the sequence or feature annotation</p></li>
</ul>
<p>In the Miscellaneous category, annotations fall under one of these sub-headings:</p>
<ul>
<li><p><strong>Catalytic activity attributes:</strong> Features of a gene product’s catalytic activity, such as K<sub>M</sub>, k<sub>cat</sub>, substrate specificity</p></li>
<li><p><strong>Experimental tools:</strong> Indicates that the gene or its product is used as an experimental tool (e.g. reporter gene construct, selectable marker, antibody)</p></li>
<li><p><strong>Genome organization:</strong> Attributes of the genomic region around the gene, e.g. duplications, repeats, alternative transcripts</p></li>
<li><p><strong>Miscellaneous functional group:</strong> Function-related descriptions that are not covered by GO molecular function or biological process annotations</p></li>
<li><p><strong>Comment:</strong> Experimental observations and other notes that do not fit into ontologies or any of the above controlled curation categories; truly miscellaneous but interesting information</p></li>
</ul>

  </div>
  <div *ngIf="pageName == 'data-submission-form-for-HTP-sequence-linked-data'">
<h2 id="htp-sequence-linked-data-submission-instructions">HTP sequence-linked data submission instructions</h2>
<p>PomBase welcomes submissions of published HTP sequence-linked data, suitable for viewing in a genome browser. We require JBrowse compliant data files and associated metadata descriptions. Please see the <a href="/faq/what-file-formats-can-i-use-submit-high-throughput-data">FAQ on file formats</a> for links to the file format descriptions, and other entries in the <a href="https://www.pombase.org/faq/data-submission-and-formats">data submission FAQ category</a> for more information.</p>
<p>We have devised a file format for the metadata we need, with downloadable spreadsheet templates available in <a href="ftp://ftp.pombase.org/pombe/documents/HTP_submission_template.xlt">Excel</a> and <a href="ftp://ftp.pombase.org/pombe/documents/HTP_submission_template.ots">Open Document Format</a> (FTP site links). Letters in the table below refer to spreadsheet columns.</p>
<p>If you prefer, you can prepare a file without using the template. Create a tab-delimited text file, and include a header line that labels the columns, using the entry in the Contents column below as the column header text.</p>
<h3 id="file-columns">File columns</h3>
<table>
<thead>
<tr class="header">
<th>Column</th>
<th>Contents</th>
<th>Example</th>
<th>Mandatory?</th>
<th>Multiple entries allowed?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1 (A)</td>
<td>Data type</td>
<td>Transcripts, Chromatin binding, Nucleosome positioning, Poly(A) sites, Replication origins</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>2 (B)</td>
<td>Track label</td>
<td>see below for format and examples***</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>3 (C)</td>
<td>Assayed gene product</td>
<td>Fkh2</td>
<td>Only required for chromatin binding data, to specify the protein binding to chromatin</td>
<td>No</td>
</tr>
<tr class="even">
<td>4 (D)</td>
<td>Strain background</td>
<td>h- cdc25-22, leu1-1</td>
<td>Yes</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>5 (E)</td>
<td>WT or mutant (strains with only background mutations are considered WT)</td>
<td>WT</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>6 (F)</td>
<td>Mutant alleles</td>
<td>clr4delta, dfp1-3A</td>
<td>No</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>7 (G)</td>
<td>Conditions</td>
<td>YES, high temperature; glucose MM, standard temperature + HU</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="even">
<td>8 (H)</td>
<td>Comment</td>
<td>free-text field for additional information</td>
<td>No</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>9 (I)</td>
<td>Growth phase or response</td>
<td>vegetative growth, meiosis, quiescence, glucose starvation, oxidative stress, heat shock</td>
<td>Yes</td>
<td>Yes if the track combines data</td>
</tr>
<tr class="even">
<td>10 (J)</td>
<td>Strand</td>
<td>forward, reverse</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="odd">
<td>11 (K)</td>
<td>Assay type</td>
<td>tiling microarray, RNA-seq, HT sequencing</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>12 (L)</td>
<td>First author (surname)</td>
<td>Soriano</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>13 (M)</td>
<td>Publication year</td>
<td>2020</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>14 (N)</td>
<td>PubMed ID</td>
<td>31077324</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>15 (O)</td>
<td>Database</td>
<td>GEO, ArrayExpress</td>
<td>Required for data available in public repository databases</td>
<td>No</td>
</tr>
<tr class="even">
<td>16 (P)</td>
<td>Study ID</td>
<td>GSE110976, PRJEB7403</td>
<td>Required if the Database column has an entry</td>
<td>No</td>
</tr>
<tr class="odd">
<td>17 (Q)</td>
<td>Sample ID</td>
<td>GSM3019628, ERS555567</td>
<td>No</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>18 (R)</td>
<td>Data file type</td>
<td>bigwig, bed</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>19 (S)</td>
<td>File name</td>
<td>name given to submitted data file relevant to the track</td>
<td>Yes</td>
<td>No</td>
</tr>
</tbody>
</table>
<h3 id="track-label-format">Track label format</h3>
<p>The track label must uniquely describe each track. For consistency in track label descriptions, please follow the recommended format as closely as possible:</p>
<p>“Assayed gene product” “Data type” “in mutant” “during Growth phase or response” “additional experimental detail of importance (Conditions, Strain background)” “; repeat” “(strand)” “- First author (Publication year)”</p>
<p>Examples:</p>
<ul>
<li>Nucleosome positioning during vegetative growth; repeat 1 - Gonzalez et al. (2016)</li>
<li>Poly(A) sites during meiosis (forward strand) - Schlackow et al. (2013)</li>
<li>Transcripts in atf1delta (forward strand) - Soriano et al. (2013)</li>
<li>Replication origins in cells synchronized by cdc25-ts block/release and hydroxyurea arrest - Xu et al. (2012)</li>
<li>Replication origins in cells synchronized by elutriation and hydroxyurea arrest - Xu et al. (2012)</li>
<li>Replication origins in dfp1-3A synchronized by elutriation and hydroxyurea arrest - Xu et al. (2012)</li>
<li>Transcription start sites during oxidative stress (reverse strand) - Thodberg et al. (2018)</li>
</ul>
<h3 id="submit">Submit</h3>
<p>To submit the files, or if you have any questions, please <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a>.</p>

  </div>
  <div *ngIf="pageName == 'disease-association'">
<h2 id="gene-page-disease-associations">Gene page: Disease associations</h2>
<p>The Disease association section lists disease descriptions from the Monarch Initiative’s <a href="https://mondo.monarchinitiative.org/">Mondo Disease Ontology</a> (Mondo) for fission yeast orthologs of human disease-causing genes.</p>
<p>The summary view shows slim terms (see below) and the names of the most specific terms used to annotate the gene.</p>
<p>The detailed view shows more information for each annotation, and may display additional terms:</p>
<figure>
<img src="assets/gene_page_disease_full.png" title="Disease associations" width="800" alt="" /><figcaption>full gene page disease association annotation</figcaption>
</figure>
<ol type="1">
<li><p><a routerLink="/browse-curation/disease-slim">Mondo slim</a> terms applicable to the gene.</p></li>
<li><p>The Mondo term ID and name, which link to a page with additional information, including the term definition, any synonyms, relationships to other Mondo terms, and annotations to the term or its descendants.</p></li>
<li><p>Indicates whether the annotation was created by PomBase curators or obtained from an external source (usually <a href="https://www.malacards.org/">Malacards</a></p></li>
<li><p>The paper from which the annotation comes.</p></li>
<li><p>The number of genes annotated to the term, linked to an ontology term page as described above.</p></li>
</ol>

  </div>
  <div *ngIf="pageName == 'gene-page-basic-information'">
<h2 id="gene-page-basic-information">Gene page: Basic information</h2>
<p>At the top of each gene page is a set of basic information about the gene:</p>
<figure>
<img src="assets/basic_info_gene_page.png" title="Gene page basic information" width="800" alt="" /><figcaption>top of gene page</figcaption>
</figure>
<ol type="1">
<li>See the <a routerLink="/submit-data/gene-name-registry">Gene Name Registry</a> for more information about <em>S. pombe</em> gene names. Synonyms include all names other than the standard name that have been published for a gene, and thus are not guaranteed to be unique.</li>
<li>Free text description of the gene product</li>
<li>Chromosome coordinates for the gene. For protein-coding genes, the “coding start to stop” numbers give the start, end, and length excluding UTRs but including introns (other than introns located entirely within UTRs). The second set of numbers indicate the gene start, end, and length including the default UTRs chosen by PomBase curators (see <a routerLink="/faq/how-do-you-determine-genes-full-length-transcript-utr-coordinates-transcription-start-and-end-s">this FAQ</a> for more information).</li>
<li>What the three-letter acronym stands for; where available, name descriptions are provided for synonyms as well as primary names.</li>
<li>Category describing how well-studied a gene is; see the <a routerLink="/status/gene-characterisation">Gene Characterisation</a> page for details</li>
<li>Indicates whether a gene encodes protein, specifies a non-coding RNA, or is a pseudogene</li>
<li>The size of the mature gene product. For protein-coding genes, the length (number of amino acid residues) and molecular weight are shown.</li>
<li>Interactive graphic from PomBase JBrowse, centred on the location of the gene. Drag to scroll left and right, double-click to zoom in, shift-double-click to zoom out, and click a feature to see details in a popup. The “Full-screen view” link in the corner opens the fully functional JBrowse in a new tab or window. Reloading a gene page restores the display to the default location and zoom level.</li>
<li>Links to gene pages for nearby genes</li>
</ol>

  </div>
  <div *ngIf="pageName == 'gene-page-external-references'">
<h2 id="gene-page-external-references">Gene page: External references</h2>
<p>The External references section of a gene page provides links to information on a gene or its product in each of several external databases. For each database, a category, the name and a brief description are shown, and the identifier used by the database is displayed and linked.</p>
<p>The resources linked in this section cover a wide range of databases and data types. Because all are external, PomBase staff cannot provide assistance with any of the resources other than problems with the actual links.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-gene-expression'">
<h2 id="gene-page-gene-expression">Gene page: Gene expression</h2>
<p>The Gene expression section of a gene page displays curated qualitative and quantitative information about the level and timing of the gene’s expression. In each subsection, a simple summary appears by default (see explanations below the detail view illustration):</p>
<figure>
<img src="assets/gene_page_expression_summary.png" title="Gene Page: Gene Expression Summary" width="800" alt="" /><figcaption>gene page expression section, summary view</figcaption>
</figure>
<p>Click “Show details” to reveal additional information:</p>
<figure>
<img src="assets/gene_page_expression_details.png" title="Gene Page: Gene Expression Details" width="800" alt="" /><figcaption>gene page expression section, detail view</figcaption>
</figure>
<ol type="1">
<li>Qualitative descriptions are drawn from a small internal controlled vocabulary, with supporting extensions. Each description links to a list of all genes annotated to the same term.
<ul>
<li>a) What was measured, i.e. RNA or protein.</li>
<li>b) A qualitative description of the expression level (one of: present, absent, increased, decreased, unchanged, constant, fluctuates).</li>
<li>c) An optional extension indicating that expression is observed during a particular process or phase (using GO terms) or in the presence of a particular chemical (using ChEBI terms).</li>
<li>d) A brief descriptor for the type of evidence that supports the annotation. The evidence categories come from the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO).</li>
<li>e) The paper from which the annotation comes.</li>
</ul></li>
<li>Quantitative data include the number of RNA or protein molecules measured per cell, and may optionally include a copy number range. Quantitative expression annotations also indicate whether the experiment measures expression in a population or a single cell, and usually include a brief description of experimental conditions, drawn from a small ontology maintained by PomBase curators.
<ul>
<li>f) Molecules per cell (RNA or protein, as indicated in the sub-header). For population-scale experiment types this is an average, whereas for single-cell scale experiments it is an absolute number.</li>
<li>g) The life cycle stage or cell cycle phase in which expression was measured.</li>
<li>h) Information about experimental conditions, such as temperature, type of medium used, etc. Descriptions come from a small ontology maintained by PomBase curators.</li>
<li>i) Whether the experiment was done at the level of a single cell or a population of cells.</li>
<li>j) A brief descriptor for the type of evidence that supports the annotation. The evidence categories come from the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO). If a more specific description is not available, the general term “experimental evidence” is shown.</li>
<li>k) The paper from which the annotation comes.</li>
</ul></li>
</ol>
<p><strong>Important:</strong> Note that the condition descriptions are rather broad, and therefore do not necessarily capture all details of every aspect that may affect gene expression. We are also unable to capture strain details at present. We therefore recommend consulting the papers cited for the data before comparing or combining data from different publications.</p>
<p>Several of the <a routerLink="/documentation/gene-page-external-references">External References</a> also link to gene expression data.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-gene-ontology'">
<h2 id="gene-page-gene-ontology">Gene page: Gene Ontology</h2>
<p>PomBase uses the <a href="http://www.geneontology.org/">Gene Ontology</a> (GO) to describe the biological context of genes.</p>
<p>GO consists of three distinct <a href="http://www.geneontology.org/docs/ontology-documentation/">ontologies</a> (or sets of vocabularies) that describe a gene’s:</p>
<ol type="1">
<li>Molecular Function (MF) — single-step activities such as <em>kinase activity</em> or <em>transporter activity</em></li>
<li>Biological Process (BP) — a series of functions with a common goal such as <em>MAPK cascade</em> or <em>cytokinesis</em></li>
<li>Cellular component (CC) — the location where the gene product can be found, and what complexes it is part of, for instance <em>nucleus</em> or <em>fatty acid synthase complex</em></li>
</ol>
<p>A gene product may be <a href="http://www.geneontology.org/docs/go-annotations/">annotated</a> to several GO terms from each of the three ontologies; <a routerLink="/gene/SPCC1682.02c">mcm3</a>, for instance, is annotated to ATP-dependent DNA helicase activity, ATP binding, and DNA replication origin binding (MFs), it acts in mitotic DNA replication initiation and negatively regulates the MCM helicase activity (BPs), and is found locations including the replication fork and the pre-replicative complex (CC).</p>
<p>Each table includes ontology term details and supporting data. The GO annotation display on PomBase gene pages includes ontology term details and supporting data. The summary view shows just the essentials: The list of terms is filtered, using the ontology structure, so that it shows only the most specific terms used to annotate a genotype, and each unique combination of gene, GO term, and extension(s) is shown once:</p>
<figure>
<img src="assets/go_gene_page_summary.png" width="600" alt="" /><figcaption>summary gene page GO annotations</figcaption>
</figure>
<ol type="1">
<li>The GO term name, which links to a page with additional information, including the term definition, any synonyms, relationships to other GO terms, and annotations to the term or its descendants. (See the PomBase <a routerLink="/documentation/ontology-term-page">ontology term page documentation</a> and the GO documentation on the <a href="http://geneontology.org/docs/ontology-documentation/">GO graph</a> and <a href="http://geneontology.org/docs/ontology-relations/">Relations in GO</a> for more information.)</li>
<li><a routerLink="/documentation/pombase-go-slim-documentation">GO Slim</a> terms applicable to the gene.</li>
<li>GO annotations may have <a routerLink="/#annotation-extensions">extensions</a> to capture any of several types of additional detail. <em>S. pombe</em> genes link to PomBase gene pages, and ontology term names link to ontology term pages.</li>
</ol>
<p>The detailed view shows annotations to all GO terms, and includes more details for each annotation. It shows separate entries for repeat determinations of a given gene/term/extension combination (if supported by more than one line of evidence and/or reported in more than one paper), and annotations to terms hidden in the summary view:</p>
<figure>
<img src="assets/go_gene_page_full.png" width="800" alt="" /><figcaption>full gene page GO annotations</figcaption>
</figure>
<ol type="1">
<li><p>The unique ID and name for a GO term, linked to an ontology term page as described above.</p></li>
<li><p>An abbreviation (code) for the type of (see “Evidence codes” below) that supports the annotation. The evidence categories come from the set of <a href="http://geneontology.org/docs/guide-go-evidence-codes/">evidence codes</a> defined by the GO Consortium.</p></li>
<li><p>An additional ontology term or identifier that provides supporting details for annotations using certain evidence codes (see below and <a href="http://geneontology.org/docs/go-annotation-file-gaf-format-2.1/">GO documentation</a>).</p></li>
<li><p>An optional qualifier that modifies the connection between the gene product and the GO term. Entries come from the set of allowed qualifiers described in <a href="http://geneontology.org/docs/go-annotations/">GO’s annotation overview</a> or internal PomBase usage.</p></li>
<li><p>The paper from which the annotation comes.</p></li>
<li><p>The number of genes annotated to the term, linked to an ontology term page as described above.</p></li>
<li><p><a routerLink="/browse-curation/fission-yeast-go-slim-terms">GO Slim</a> terms applicable to the gene.</p></li>
<li><p>GO annotations may have extensions(see “Annotation extensions” below) to capture any of several types of additional detail. <em>S. pombe</em> genes link to PomBase gene pages.</p></li>
</ol>
<h4 id="term-id-and-name">Term ID and name</h4>
<p>Annotations are made to the definition of a term, not the term name itself, so we recommend that users always read the term definition. The definition of a term can be found on the ontology term page linked to the term name and ID.</p>
<h4 id="go-structure">GO Structure</h4>
<p>GO is structured in a hierarchal order with less specific terms being parents of more specific child terms. A child term may have multiple parents and multiple children; the BP <em>mitotic sister chromatid segregation</em>, for instance, has child terms representing each part of the segregation process (sister chromatid cohesion, separation, etc.) as well as parent terms connecting it to both the mitotic cell cycle and chromosome segregation. Crucially, whenever an annotation is made to a term, the gene product is automatically annotated to all the parent terms. The ancestry of a term can be viewed in browsers such as AmiGO or QuickGO, accessible via links on the ontology term page. For more information, see the <a href="http://geneontology.org/docs/ontology-documentation/">GO graph</a> documentation.</p>
<p>Multiple relationships exist to describe the links within the ontologies. A child term can have an <em>is_a</em> relation to the parent term, where the child is a more specific type of the parent, or a <em>part_of</em> relationship where the child makes up a part of the parent. For instance, the mitochondrion <em>is_a</em> intracellular organelle and is part_of a cell. Additionally, GO also include regulatory relationships. For more information on relationships in GO, see the <a href="http://geneontology.org/docs/ontology-relations/">Relations in GO</a> documentation.</p>
<h4 id="evidence-codes">Evidence codes</h4>
<p>In PomBase, every annotation is supported by a reference that states where the annotation comes from, and an evidence code that describes the type of data that supports the annotation. An annotation may be inferred from experimental ‘wet lab’ data, backed by a literature reference and citing experimental evidence such as <a href="http://wiki.geneontology.org/index.php/Inferred_from_Direct_Assay_(IDA">IDA</a>) (Inferred from Direct Assay) or <a href="http://wiki.geneontology.org/index.php/Inferred_from_Mutant_Phenotype_(IMP">IMP</a>) (Inferred from Mutant Phenotype). Further information on evidence codes is available in the <a href="http://geneontology.org/docs/guide-go-evidence-codes/">GO Evidence Codes</a> documentation.</p>
<p>Another source of annotations come from computational methods. Please note that all computational annotations are based on predictions. In cases where a sequence model has been used to annotate genes, but the genes annotated based on the model have not been manually checked, the <a href="http://wiki.geneontology.org/index.php/Inferred_from_Electronic_Annotation_(IEA">IEA</a>) (Inferred from Electronic Annotation) evidence code is assigned. If the annotations have been manually checked other evidence codes may be used, for instance <a href="http://wiki.geneontology.org/index.php/Inferred_from_Sequence_Orthology_(ISO">ISO</a>) (Inferred from Sequence Orthology) or <a href="http://wiki.geneontology.org/index.php/Inferred_from_Sequence_Model_(ISM">ISM</a>) (Inferred from Sequence Model). PomBase uses ISO to cross-reference to the roles of known <em>S. cerevisiae</em> genes, and uses ISM when domains present in a gene product can give clues to its biological role.</p>
<p>For some types of evidence, such as sequence comparisons or interaction data, it is important to note what gene or gene product was used in the comparison or detected in the interaction. In these cases the <a href="http://geneontology.org/docs/go-annotation-file-gaf-format-2.1/">With/From column</a> provides more information regarding the source of the information.</p>
<h4 id="annotation-extensions">Annotation extensions</h4>
<p>Where available, annotation extensions are displayed underneath the GO term name. The extensions provide additional specificity to the annotation by linking the term to another ontology term or a gene product via a relationship. Examples include specifying substrates of molecular functions or specifying the cellular localization during a process (for instance, <a routerLink="/gene/SPBC106.10">pka1</a> has protein serine/threonine kinase activity and has the substrates <a routerLink="/gene/SPBC119.04">mei3</a> and <a routerLink="/gene/SPAC6F12.02">rst2</a>. It is a cellular component of the nucleus during nitrogen starvation, but found in the vacuole during glucose starvation).</p>
<p>The GO Consortium provides further information on annotation extensions in its <a href="http://geneontology.org/docs/go-annotation-file-gaf-format-2.1/">file format guide</a>, on a <a href="http://wiki.geneontology.org/index.php/Annotation_Extension">wiki page</a>, and in publications from <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885854">2014</a> and <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27812947">2017</a>. PomBase converts many extension names to more human-friendly text, as described <a routerLink="/documentation/annotation-extension-relation-display">here</a>.</p>
<p>If an extension mentions another <em>S. pombe</em> gene, the extension data will also be displayed as an annotation in the “<a routerLink="/documentation/gene-page-target">Target of</a>” section of the page for that gene.</p>
<h4 id="finding-other-genes-annotated-to-a-specific-go-term">Finding other genes annotated to a specific GO term</h4>
<p>From a gene page, all <em>S. pombe</em> genes annotated to a term (or its children) can be found by clicking on a term name or ID to reach the ontology term page.</p>
<p>Additionally, the <a routerLink="/query">advanced search</a> can be used to search for all genes annotated to a particular GO term (see the <a routerLink="/documentation/advanced-search">advanced search documentation</a> for more information). To find annotations to specific GO terms in organisms other than <em>S. pombe</em> we recommend using <a href="http://amigo.geneontology.org">AmiGO</a> or <a href="http://www.ebi.ac.uk/QuickGO">QuickGO</a>.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-literature'">
<h2 id="gene-page-literature">Gene page: Literature</h2>
<p>The Literature section of a gene page lists the papers that have curated data for the gene or its product.</p>
<p>At the top of the section, there is a link to search PubMed for any papers on the gene. This may retrieve papers not on the gene page Literature list, if there are papers not yet curated (or spurious matches to the search criteria).</p>
<p>For each paper, the full citation and the number of genes curated from the paper are shown, and a “Details” link goes to the <a routerLink="/documentation/publication-page">publication page</a>.</p>
<p>By default, the list is sorted alphabetically by first author. Use the “Sort by” links to change the order.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-modifications'">
<h2 id="gene-page-modifications">Gene page: Modifications</h2>
<p>The Modifications section lists protein modifications that have been manually curated, using terms from the <a href="http://obofoundry.org/ontology/mod.html">PSI-MOD</a> ontology, for protein-coding genes. PomBase will add RNA modifications to this section in the future, when relevant data are curated.</p>
<h4 id="ontology-annotations-for-protein-features">Ontology Annotations for Protein Features</h4>
<p>The summary view shows only the names of the most specific terms used to annotate the gene:</p>
<figure>
<img src="assets/gene_page_modification_summary.png" title="Protein modifications" width="250" alt="" /><figcaption>summary gene page modification ontology annotation</figcaption>
</figure>
<p>The detailed view shows more information for each annotation, and may display additional terms:</p>
<figure>
<img src="assets/gene_page_modification_full.png" title="Protein modifications" width="800" alt="" /><figcaption>full gene page modification ontology annotation</figcaption>
</figure>
<ol type="1">
<li>Name and ID of the ontology term</li>
<li>Modification annotations may have extensions (see below) to capture any of several types of additional detail. <em>S. pombe</em> genes link to PomBase gene pages, and GO term names link to PomBase ontology term summary pages.</li>
<li>The summary view is filtered, using the ontology structure, so that it shows only the most specific terms used to annotate a gene. Modification annotations curated for a gene using less specific terms (“superclasses” or “ancestors” in the ontology) appear only in the detailed view.</li>
<li>An abbreviation (code) or brief description for the type of evidence that supports the annotation. Modification annotations use some of the <a href="http://geneontology.org/docs/guide-go-evidence-codes/">codes</a> defined by the GO Consortium, plus a small subset of the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO).</li>
<li>The residue(s) modified (where available). Note: for histones, residue numbering assumes that the initiator methionine is removed.</li>
<li>The published source of the annotation (where available)</li>
<li>The total number of genes annotated to this term or any of its descendants, linked to a list of those genes. The gene list page includes basic information about the ontology term, and a link to send the gene list to the advanced search.</li>
</ol>
<h4 id="ann_ext">Annotation extensions</h4>
<p>Where available, annotation extensions are displayed underneath the ontology term name. The extensions provide additional specificity to the annotation, often by linking the term to another ontology term or a gene product via a relationship. Examples include specifying the gene product that adds or removes a modification, specifying modified residues, or specifying that the modification is observed during a phase or process. For example, Lys4 (shown above) is phosphorylated on a serine residue during M phase of the mitotic cell cycle. Likewise, <a routerLink="/gene/SPBC11B10.09">Cdc2</a> is phosphorylated during G2, and phosphorylated on Tyrosine 15 by Wee1; it is phosphorylated during G2 but not M phase of the mitotic cell cycle; other extensions are also available for Cdc2.</p>
<p>The annotation extension field can also be used to indicate modification site occupancy, for experiments that measure the proportion of copies of the protein (or RNA) that have the modification.</p>
<p>Note: unlike GO annotations, for modification annotations extensions are not considered for display filtering. Annotations to less specific terms are hidden in the summary view whether they have extensions or not.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-phenotypes'">
<h2 id="gene-page-phenotypes">Gene page: Phenotypes</h2>
<p>PomBase defines a phenotype as an observable characteristic, or set of characteristics, of an organism that results from the interaction of its genotype with a given environment. In PomBase, phenotypes are annotated using terms from the Fission Yeast Phenotype Ontology (FYPO). FYPO uses several existing ontologies from the <a href="http://obofoundry.org/">Open Biological and Biomedical Ontologies</a> (OBO) collection to construct formal definitions. Basic documentation for FYPO is available at the <a href="http://obofoundry.org/ontology/fypo.html">OBO Foundry</a>, and further information is available on the <a href="http://curation.pombase.org/pombase-trac/wiki/FissionYeastPhenotypeOntology">PomBase wiki</a>.</p>
<p>In the phenotype annotation display on PomBase gene pages, the first item shown is a brief summary indicating whether cells with a null (deletion) allele of the gene are viable or inviable, or either depending on experimental conditions. Next, single-allele phenotypes are shown in two tables. The first table lists phenotypes observed at the population level, such as viability in culture, and the second shows cell-level phenotypes. Note that the viable/inviable population terms describe whether a gene is essential or not, and see the wiki <a href="http://curation.pombase.org/pombase-trac/wiki/FYPOContentStructure">FYPO Content and Structure</a> documentation for more information on cell and population phenotypes. Finally, two more tables list population-level and cell-level multi-allele phenotypes, i.e. phenotypes associated with double mutants, triple mutants, etc, and the relevant genotype details.</p>
<p>Each table includes ontology term details and supporting data. The summary view shows just the essentials: The list of terms is filtered, using the ontology structure, so that it shows only the most specific terms used to annotate a genotype, and each unique combination of gene, FYPO term, allele(s) and extension(s) is shown once. For single-allele phenotypes, the display includes:</p>
<figure>
<img src="assets/single_fypo_gene_page_summary.png" width="800" alt="" /><figcaption>single-allele FYPO summary on gene page</figcaption>
</figure>
<ol type="1">
<li>The FYPO term name, which links to a page with additional information, including the term definition, any synonyms, relationships to other FYPO terms, and annotations to the term or its descendants. (See the <a routerLink="/documentation/ontology-term-page">ontology term page documentation</a> for more.)</li>
<li>Allele details, including a name (if one is used in the literature) and description (where known). Mouse over the allele name to show the allele type, which indicates whether partial deletions or altered residues refer to amino acids or nucleotides, and expression level. If you can provide a description for any allele shown as “unknown”, please <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a>.</li>
<li>Phenotype annotations may have extensions to capture penetrance (proportion of a population that shows the phenotype) or severity (previously designated “expressivity”), or to document which gene or protein used in an assay for level, localisation, etc. <em>S. pombe</em> genes link to PomBase gene pages. Severity and penetrance use the relations <em>has_severity</em> and <em>has_penetrance</em> respectively, and can have values such as “high”, “medium”, or “low”. Penetrance can also use numerical values. A gene or gene product used in an assay is stored using the appropriate PomBase systematic ID and the relation <em>assayed_using</em>; the relation is converted to <em>affecting</em> in the gene page display. If a mutation affects an activity that modifies another gene product, extensions may capture the affected enzyme, the affected substrate, or both. The relation <em>assayed_enzyme</em> is displayed as <em>affecting activity of</em> on gene pages, and <em>assayed_substrate</em> is shown as <em>affecting substrate</em>.</li>
<li>The phenotype display can be filtered to show subsets of the total set of annotations. The term filter, available in the summary or detailed view, lists several broad phenotypic categories derived from high-level FYPO terms:</li>
</ol>
<figure>
<img src="assets/fypo_term_filter_pulldown.png" width="160" alt="" /><figcaption>FYPO annotation term filter</figcaption>
</figure>
<p>Choose one to restrict the annotation display to terms in the selected branch of FYPO. When term filtering is active, a message appears to indicate that not all annotations are shown:</p>
<figure>
<img src="assets/fypo_showing_n_annotations.png" width="600" alt="" /><figcaption>filtered annotation display message</figcaption>
</figure>
<p>Change the selection back to “No filter” to see annotations to all terms.</p>
<p>The detailed single-allele phenotype view shows annotations to all FYPO terms, and includes more details for each annotation. It shows separate entries for repeat determinations of a given gene/term/allele/extension combination (if supported by more than one line of evidence and/or reported in more than one paper), and annotations to terms hidden in the summary view:</p>
<figure>
<img src="assets/single_fypo_gene_page_full.png" width="800" alt="" /><figcaption>Single-allele FYPO detailed view on gene page</figcaption>
</figure>
<ol type="1">
<li>The unique ID and name for a term in the phenotype ontology. The ID links to a page with additional information, including the term definition, any synonyms, relationships to other FYPO terms, and annotations to the term or its descendants. (See the <a routerLink="/documentation/ontology-term-page">ontology term page documentation</a> for more.)</li>
<li>Allele details, including a name (if one is used in the literature) and description (where known). The column is headed “Genotypes” for consistency between the single- and multi-allele displays. Each genotype name links to a page with full details (type, description, and expression) for its allele(s), links to gene pages, and a list of all phenotype annotations for the genotype. If you can provide a description for any allele shown as “unknown”, please <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a>.</li>
<li>Mouse over the allele name to show the allele type, which indicates whether partial deletions or altered residues refer to amino acids or nucleotides, and expression level.</li>
<li>The phenotype display can be filtered to show subsets of the total set of annotations. The term filter, available in the summary or detailed view, lists several broad phenotypic categories derived from high-level FYPO terms. Choose one to restrict the annotation display to terms in the selected branch of FYPO. Change the selection back to “No filter” to see annotations to all terms. In the detailed view, evidence can also be filtered by choosing an evidence description from a pulldown. Change the selection back to “No filter” to see annotations using any evidence type. Term and evidence filters can be combined (note that some term/evidence combinations have no matching annotations). When either term or evidence filtering is active, a message appears to indicate that not all annotations are shown.</li>
<li>A brief descriptor for the type of evidence that supports the annotation. The evidence categories come from the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO).</li>
<li>Information about experimental conditions, such as temperature, type of medium used, etc. Descriptions come from a small ontology maintained by PomBase curators.</li>
<li>The paper from which the annotation comes.</li>
<li>Phenotype annotations may have extensions to capture penetrance (proportion of a population that shows the phenotype) or severity (previously designated “expressivity”), or to document which gene or protein used in an assay for level, localisation, etc. <em>S. pombe</em> genes link to PomBase gene pages. Severity and penetrance use the relations <em>has_severity</em> and <em>has_penetrance</em> respectively, and can have values such as “high”, “medium”, or “low”. Penetrance can also use numerical values. A gene or gene product used in an assay is stored using the appropriate PomBase systematic ID and the relation <em>assayed_using</em>; the relation is converted to <em>affecting</em> in the gene page display. If a mutation affects an activity that modifies another gene product, extensions may capture the affected enzyme, the affected substrate, or both. The relation <em>assayed_enzyme</em> is displayed as <em>affecting activity of</em> on gene pages, and <em>assayed_substrate</em> is shown as <em>affecting substrate</em>.</li>
</ol>
<p>Similar displays are used for multi-allele phenotypes. Unless otherwise noted below, all items are as described above for single-allele phenotype displays.</p>
<p>The multi-allele summary shows FYPO terms and genotype descriptions:</p>
<figure>
<img src="assets/multi_fypo_gene_page_summary.png" width="800" alt="" /><figcaption>multi-allele FYPO summary view on gene page</figcaption>
</figure>
<ol type="1">
<li>The FYPO term name, which links to the page for the term.</li>
<li>Genotype details, including a name (if one is used in the literature) and descriptions for the relevant alleles (where known). Mouse over any genotype name to show more allele details, or click to go to the genotype detail page. Multiple genotypes annotated to the same term are separated by commas.</li>
<li>Display filter for FYPO terms.</li>
</ol>
<p>The detailed view includes:</p>
<figure>
<img src="assets/multi_fypo_gene_page_full.png" width="800" alt="" /><figcaption>multi-allele FYPO full view on gene page</figcaption>
</figure>
<ol type="1">
<li>The unique ID and name for a term in the phenotype ontology, with links to the FYPO term page.</li>
<li>Genotype details, including a name (if one is used in the literature) and descriptions for the relevant alleles (where known). Mouse over any genotype name to show more allele details, or click to go to the genotype detail page. Multiple genotypes annotated to the same term are listed one per line.</li>
<li>Display filter for FYPO terms and evidence (see above).</li>
<li>Evidence description (see above).</li>
<li>Experimental conditions (see above).</li>
<li>The paper from which the annotation comes.</li>
</ol>
<p>Annotation extensions are also included in the multi-allele Summary and Full views (although none are included for the annotations in the above illustrations), using the same display as for single-allele phenotypes.</p>
<p>Also see the <a routerLink="/documentation/advanced-search">Advanced search documentation</a> for information on finding genes annotated to phenotype terms.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-protein-features'">
<h2 id="gene-page-protein-features">Gene page: Protein features</h2>
<p>Gene pages for protein-coding genes have a section describing protein features. Also see the <a routerLink="/documentation/gene-page-modifications">Modifications</a> documentation.</p>
<figure>
<img src="assets/gene_page_protein_features.png" width="800" alt="" /><figcaption>gene page protein features</figcaption>
</figure>
<ol type="1">
<li>The graphical view is a static graphic from the Ensembl genome browser for <em>S. pombe</em>, and shows the positions of domains and other features. Below the graphic is a link to the Ensembl browser’s protein view.</li>
<li>The table of protein families and domains is described in more detail below.</li>
<li>This section provides manual curation that ensures that large protein families (e.g. the WD family) include all known members. These manual annotations are provided because many families in protein family databases are rebuilt during the release cycle to include new sequences, and as a result gene products are sometimes lost or gained from protein families, and false negatives are common. This section is also used to capture published subfamily members which may be collected into a single family in protein family databases.</li>
<li>Any motifs or features annotated using <a href="http://sequenceontology.org/">Sequence Ontology</a> (SO) terms.</li>
<li>Table of the protein’s physical properties</li>
</ol>
<h4 id="protein-families-and-domains">Protein Families and Domains</h4>
<figure>
<img src="assets/gene_page_protein_domains.png" width="800" alt="" /><figcaption>gene page protein domains</figcaption>
</figure>
<ol type="a">
<li>Feature ID of this family or domain in the originating database. Where feasible, the ID links to the source database. Databases include Pfam, SMART, Prosite, Gene3D, SUPERFAMILY, TMHMM and Panther.</li>
<li>Source database name</li>
<li>Name of the matching feature in the contributing database</li>
<li>ID of the <a href="http://www.ebi.ac.uk/interpro/">InterPro</a> entry that includes the feature. InterPro classifies proteins into families using predictive models provided by several different databases that make up the InterPro consortium. InterPro is useful for assessing the species distribution of a particular family or domains.</li>
<li>The description of the domain in InterPro, from the contributing database</li>
<li>Start and end coordinates, showing the location of the feature within the protein</li>
<li>Link to a list of other fission yeast proteins which are members of the family or have the domain</li>
<li>Link to the InterPro entry for the feature<br />
</li>
<li>Link to the Pfam entry for the protein, which shows domain organization</li>
<li>Transmembrane domain coordinates, where applicable</li>
</ol>
<p>Note that some of the listed features (e.g. transmembrane domains from TMHMM) are predictions. Consult the contributing databases for further information, or <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a> if you notice any problems with annotated or predicted features.</p>

  </div>
  <div *ngIf="pageName == 'gene-page-sequence'">
<h2 id="gene-page-sequence">Gene page: Sequence</h2>
<p>The Sequence section of a gene page provides a widget to download DNA sequences for any gene, and amino acid sequences for protein-coding genes, as well as links to send sequences to BLAST. For protein-coding genes, the DNA sequence of the coding exon(s) is displayed by default.</p>
<figure>
<img src="assets/gene_page_sequence.png" width="800" alt="" /><figcaption>DNA sequence section</figcaption>
</figure>
<ol type="1">
<li><p>Toggle to show the amino acid sequence (not available for non-coding RNA genes).</p></li>
<li><p>Controls to add UTRs, introns, or flanking sequences to the displayed sequence. For upstream or downstream sequence, use the up/down control to increment by one base, or type or paste a number in the box.</p></li>
<li><p>The sequence header indicates what is included.</p></li>
<li><p>Link to save the displayed sequence to a file.</p></li>
<li><p>Links to send the displayed sequence to BLAST at <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">NCBI</a> or <a href="http://fungi.ensembl.org/Multi/Tools/Blast?db=core">Ensembl</a>. DNA sequences link to BLASTN by default.</p></li>
</ol>
<p>When the “Show translation” option is selected, irrelevant controls are hidden and BLASTP links are shown:</p>
<figure>
<img src="assets/gene_page_sequence_translated.png" width="800" alt="" /><figcaption>sequence section showing translation</figcaption>
</figure>

  </div>
  <div *ngIf="pageName == 'gene-page-target'">
<h2 id="gene-page-target-of">Gene page: Target of</h2>
<p>The “Target of” section describes genes that affect the gene of interest. Genes listed under “Target of” may include upstream regulators or enzymes that modify the product of the gene of interest. For example, the “Target of” annotations for <a routerLink="/gene/SPBC11B10.09">cdc2</a> indicate that it is a substrate of, and regulated by, the kinase Wee1 and the phosphatase Cdc25 (among others).</p>
<p>Target of data are derived by taking the reciprocal of <a href="/faq/what-annotation-extension">ontology annotation extensions</a> (using <a routerLink="/documentation/gene-page-gene-ontology#ann_ext">GO annotation extensions</a> and <a routerLink="/documentation/gene-page-phenotypes">phenotype</a> annotation extensions). The table includes:</p>
<ul>
<li>Ontology: GO or FYPO, i.e. the ontology in which the annotation was made</li>
<li>Relationship: indicates how the genes are connected. The relationship is the reciprocal of the relation used in the originating annotation extension, as noted in the <a routerLink="/documentation/annotation-extension-relation-display">Annotation extension relation display documentation</a>.</li>
<li>Gene: the gene whose product affects the gene of interest.</li>
<li>Product: the targeting gene’s product description.</li>
<li>Reference (hidden in “summary” view): the source of the annotation.</li>
</ul>

  </div>
  <div *ngIf="pageName == 'gene-page-transcript'">
<h2 id="gene-page-transcript">Gene page: Transcript</h2>
<p>The Transcript section of a gene page shows the positions of introns and exons in each transcript graphical and table formats.</p>
<figure>
<img src="assets/gene_page_transcript.png" width="800" alt="" /><figcaption>Transcript section</figcaption>
</figure>
<ol type="1">
<li><p>ID and genomic location. The transcript systematic ID is the gene systematic ID with “.1” appended. (When PomBase adds support for alternative transcripts, they will receive sequentially numbered suffixes.)</p></li>
<li><p>In the default view, intron and exon positions are shown in the table as genomic coordinates. When the “Show transcript coordinates” button is checked, positions in the table are numbered using the annotated transcription start site as 1.</p></li>
<li><p>Mouse over any exon or intron to highlight its coordinates in the table, and bring up a box with position details. Genomic coordinates and within-transcript positions appear in the pop-up no matter which is selected in the main display.</p></li>
<li><p>Mouse over any entry in the exon/intron tables to show the box with position details.</p></li>
</ol>

  </div>
  <div *ngIf="pageName == 'genetic-and-physical-interactions'">
<h2 id="genetic-and-physical-interactions">Genetic and physical interactions</h2>
<p>The Interactions section of a gene page displays genetic and physical interactions for a gene (or its product), in <a href="http://thebiogrid.org/">BioGRID</a> format. All interactions are curated manually by PomBase or BioGRID curators.</p>
<p>Each table has five columns:</p>
<p>The first column (with no header) describes the interaction. The text reflects the type of evidence listed in the Evidence column and, where applicable, the directionality of the interaction.</p>
<p><strong>Interacting gene:</strong> The gene that interacts with the gene of interest, linked to its gene page</p>
<p><strong>Interacting product:</strong> The description of the interacting gene’s product, from its gene page</p>
<p><strong>Evidence:</strong> The type of genetic interaction observed, or the type of experiment performed to detect a physical interaction. The evidence categories come from BioGRID, and are described on their <a href="http://wiki.thebiogrid.org/doku.php/experimental_systems">Experimental Evidence Codes</a> documentation page.</p>
<p><strong>Reference:</strong> The paper cited to support the interaction</p>

  </div>
  <div *ngIf="pageName == 'genotype-page'">
<h2 id="genotype-pages">Genotype pages</h2>
<p>Genotype pages are analogous to gene pages, but describe the alleles that make up a genotype, and show phenotypes associated with the genotype.</p>
<figure>
<img src="assets/genotype_page.png" width="800" alt="" /><figcaption>genotype page</figcaption>
</figure>
<ol type="1">
<li><p>The genotype name. A short and/or memorable name can be assigned, but otherwise the default name is constructed by concatenating the names of the alleles that make up the genotype.</p></li>
<li><p>Each genotype in composed of one or more alleles. The table lists details for each allele of interest in the genotype. Any curated background details will be shown above the table of alleles.</p>
<ol type="a">
<li>Gene name, linked to its gene page</li>
<li>Allele name and description. For point mutations and partial deletions, the description specifies the positions, and indicates altered residues for point mutations.</li>
<li>The allele type specifies what kind of change the allele description represents, e.g. amino acid mutation, deletion, other.</li>
<li>The expression level indicates the amount of gene product present in the assayed cells, or “Not assayed” if the level was not directly measured.</li>
</ol></li>
<li><p>Phenotypes annotated to the genotype are displayed as described in the <a routerLink="/documentation/gene-page-phenotypes">gene page documentation</a>.</p></li>
</ol>

  </div>
  <div *ngIf="pageName == 'high-confidence-physical-interaction-network'">
<h2 id="high-confidence-physical-interaction-network-hcpin">High Confidence Physical Interaction Network (HCPIN)</h2>
<p>The PomBase High Confidence Physical Interaction Network (HCPIN) is a collection of physical interaction data based on three subsets of Gene Ontology (GO) annotations:</p>
<ol type="1">
<li>Manually curated protein binding annotations. PomBase curators annotate genes to the GO “protein binding” molecular function term (GO:0005515) when proteins are shown experimentally to interact in a pairwise, and presumably direct, manner, often by more than one method</li>
<li>Targets or substrates of biochemical activity (e.g. protein kinases or phosphatases, protein methyltransferases, ubiquitin-protein ligases). When one protein is shown experimentally to modify another, a GO molecular function annotation represents the activity, and the target(s) can be captured using <a routerLink="/documentation/gene-page-gene-ontology#ann_ext">annotation extensions</a>. For example, <a routerLink="/gene/SPBC11B10.09">Cdc2</a> phosphorylates <a routerLink="/gene/SPBC146.03c">Cut3</a>, so it is annotated to ‘protein serine/threonine kinase activity’ with the extension ‘has substrate cut3’.</li>
<li>GO protein complex annotations. The HCPIN set includes both annotations supported directly by fission yeast experiments and those inferred from an experimentally characterised ortholog.</li>
</ol>
<p>The data can be downloaded in tab-delimited format from the <a href="/datasets">Download Datasets</a> page.</p>

  </div>
  <div *ngIf="pageName == 'index'">
<h1 id="pombase-web-site-help-documentation">PomBase Web Site Help &amp; Documentation</h1>
<div *ngIf="!deployConfigService.productionMode()"  class="doc-search">
<div class="doc-search-title">
Search documentation, news archive and FAQ:
</div>
<app-faceted-search [scope]="'doc'"></app-faceted-search>
</div>
<h3 id="DocsGenePage">Gene Pages</h3>
<ul>
<li><a routerLink="/documentation/gene-page-basic-information">Basic information</a></li>
<li><a routerLink="/documentation/gene-page-gene-ontology">Gene Ontology</a></li>
<li><a routerLink="/documentation/gene-page-phenotypes">Phenotypes</a></li>
<li><a routerLink="/documentation/gene-page-target">Target of</a></li>
<li><a routerLink="/documentation/gene-page-transcript">Transcript</a></li>
<li><a routerLink="/documentation/gene-page-protein-features">Protein features</a></li>
<li><a routerLink="/documentation/gene-page-modifications">Modifications</a></li>
<li><a routerLink="/documentation/gene-page-gene-expression">Gene expression</a></li>
<li><a routerLink="/documentation/genetic-and-physical-interactions">Interactions</a></li>
<li><a routerLink="/documentation/orthologs">Orthologs</a> and paralogs</li>
<li><a routerLink="/documentation/gene-page-sequence">Sequence</a></li>
<li><a routerLink="/documentation/taxonomic-conservation">Taxonomic conservation</a></li>
<li><a routerLink="/documentation/disease-association">Disease association</a></li>
<li><a routerLink="/documentation/controlled-curation">Controlled curation</a></li>
<li><a routerLink="/documentation/gene-page-external-references">External references</a></li>
<li><a routerLink="/documentation/gene-page-literature">Literature</a></li>
</ul>
<h3 id="DocsSearch">Searching</h3>
<ul>
<li><a routerLink="/documentation/simple-search-documentation">Simple search</a></li>
<li><a routerLink="/documentation/advanced-search">Advanced search</a></li>
<li><a routerLink="/documentation/motif-search">Peptide motif search</a></li>
</ul>
<h3 id="jbrowse-genome-browser">JBrowse Genome Browser</h3>
<ul>
<li><a routerLink="/documentation/JBrowse_quick_start">Getting started with PomBase JBrowse</a> - an introductory guide to browser navigation, track loading, metadata, etc. (by Antonia Lock)</li>
<li>Also see the General entry in the JBrowse Help menu</li>
<li>See “File Formats” below to submit data</li>
</ul>
<h3 id="quilt">QuiLT</h3>
<ul>
<li><a routerLink="/documentation/quick-little-tool">Documentation</a> for the Quick Little Tool for gene list visualisation</li>
</ul>
<h3 id="other-pombase-pages">Other PomBase Pages</h3>
<ul>
<li>GO Slim <a routerLink="/documentation/pombase-go-slim-documentation">Documentation</a> and <a routerLink="/browse-curation/fission-yeast-go-slimming-tips">Usage tips</a></li>
<li><a routerLink="/documentation/ontology-term-page">Ontology term pages</a></li>
<li><a routerLink="/documentation/publication-page">Publication pages</a></li>
<li><a routerLink="/documentation/genotype-page">Genotype pages</a></li>
<li><a routerLink="/browse-curation/fission-yeast-phenotype-ontology">Fission Yeast Phenotype Ontology (FYPO) page</a></li>
<li><a routerLink="/documentation/high-confidence-physical-interaction-network">High Confidence Physical Interaction Network (HCPIN)</a></li>
<li><a routerLink="/about/version-history">Data versions</a></li>
</ul>
<h3 id="how-to-cite-pombase">How to Cite PomBase</h3>
<ul>
<li>See the <a routerLink="/about/citing-pombase">Citing PomBase</a> page</li>
</ul>
<h3 id="file-formats-for-data-submission">File Formats for Data Submission</h3>
<p>PomBase accepts batch submissions of certain types of data that appear on PomBase gene pages. For these data types, we use dedicated PomBase-specific formats:</p>
<ul>
<li><a routerLink="/documentation/phenotype-data-bulk-upload-format">Phenotypes</a></li>
<li><a routerLink="/documentation/modification-data-bulk-upload-format">Modifications</a></li>
<li><a routerLink="/documentation/qualitative-gene-expression-data-bulk-upload-format">Qualitative gene expression</a></li>
<li><a routerLink="/documentation/quantitative-gene-expression-data-bulk-upload-format">Quantitative gene expression</a></li>
<li><a routerLink="/documentation/data-submission-form-for-HTP-sequence-linked-data">HTP data for JBrowse tracks</a></li>
</ul>
<p>For data that can be connected with sequence features or coordinates, and displayed as tracks in the genome browser, see the <a href="faq/what-file-formats-can-i-use-submit-high-throughput-data">data format FAQ</a> and further details linked there.</p>
<h3 id="linking-to-and-from-pombase">Linking to and from PomBase</h3>
<ul>
<li>Linking to PomBase: To link to any PomBase gene page, use the systematic ID for the gene in a URL with the syntax http://www.pombase.org/gene/[systematic ID]. For example, <a routerLink="/gene/SPBC11B10.09">http://www.pombase.org/gene/SPBC11B10.09</a> links to the gene page for <a routerLink="/spombe/result/SPBC11B10.09">cdc2</a>.</li>
<li>Linking from PomBase to external resources: We can provide links from PomBase gene pages to gene- or gene product-specific <em>S. pombe</em> data for any resource that uses URLs with PomBase systematic IDs . Please contact the PomBase Curators for more information.</li>
</ul>
<h3 id="comparative-genomics">Comparative Genomics</h3>
<p>We plan to add specific documentation about comparative genomics using <em>S. pombe</em> here. In the meantime, you can look at the documentation for <a href="http://www.ensembl.org/info/genome/compara/index.html">Ensembl Compara</a>.</p>
<h3 id="more-information">More information …</h3>
<p>For help with anything in PomBase not covered here, you can <a routerLink="/helpdesk@pombase.org">contact the curators</a>.</p>

  </div>
  <div *ngIf="pageName == 'modification-data-bulk-upload-format'">
<h2 id="modification-data-bulk-upload-format">Modification data bulk upload format</h2>
<p>PomBase welcomes submissions of published large-scale modification data sets. We have devised a tab-delimited text file format for bulk modification data.</p>
<p>Include a header line that labels the columns – use the entry in the Contents column below as the column header text.</p>
<table>
<thead>
<tr class="header">
<th>Column</th>
<th>Contents</th>
<th>Example</th>
<th>Mandatory?</th>
<th>Multiple entries allowed?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1</td>
<td>Gene systematic ID</td>
<td>SPBC11B10.09</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>2</td>
<td>Gene name</td>
<td>cdc2</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="odd">
<td>3</td>
<td>Ontology ID</td>
<td>MOD:0000001</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>4</td>
<td>Evidence</td>
<td>ECO:0000006</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>5</td>
<td>Residue</td>
<td>S72</td>
<td>No</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>6</td>
<td>Extension</td>
<td>added_by(PomBase:SPBC11B10.09)</td>
<td>No</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>7</td>
<td>Reference</td>
<td>PMID:24763107</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>8</td>
<td>Taxon</td>
<td>4896</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>9</td>
<td>Date</td>
<td><span class="no-break">2014-05-01</span></td>
<td>Yes</td>
<td>No</td>
</tr>
</tbody>
</table>
<p><strong>Notes:</strong></p>
<p>At present we accept protein modification data, using PSI-MOD IDs in the Ontology ID column. We plan to accept RNA modification data in the future. More information is available in the <a routerLink="/documentation/gene-page-modifications">gene page modifications documentation</a>.</p>
<p>Use one line per modified position (multiple entries are allowed only in the Extension column).</p>
<p>File columns:</p>
<ol type="1">
<li>Include the systematic ID for each gene. You can look up systematic IDs on gene pages, or refer to the file of all gene names from the dataset download page.</li>
<li>Gene names are optional. If you include them, use standard names in column 2 (see gene pages or the file of all gene names from the dataset download page.</li>
<li>For help finding suitable ontology (PSI-MOD) terms to describe your phenotypes, see the <a href="https://curation.pombase.org/pombe/docs/modification_annotation">Canto protein modification documentation</a>. If you can’t find a term you need, email the <a href="mailto:helpdesk@pombase.org">helpdesk</a> for assistance; we can advise you, and request new terms as needed.</li>
<li>For the Evidence column, we use a small selection from the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO). You are welcome to enquire with us in advance to find out which ECO terms/IDs fit your experiments, but we can accept files with brief descriptions, which curators will convert to ECO IDs.</li>
<li>The Residue column indicates the position modified. For protein modifications, use one-letter amino acid code. Multiple entries are allowed, but only for cases where two or more of the same modification are known to be present at the same time. Separate entries with commas (e.g. S72,T85). Position numbering should reflect the current sequence data in PomBase. Please refer to the <a routerLink="/status/gene-coordinate-changes">Gene Coordinate Changes</a> page to ensure that your residue position entries are up to date. Also note that histones are conventionally numbered assuming the initiator methionine is removed (i.e. every position in the mature protein is numbered, and is 1 less than the apparent numbering predicted by translating the ORF).</li>
<li>See the table below for allowed annotation extensions. Multiple extensions can be included for a modification. Separate extensions with a comma (,) if they combine to form a “compound” extension (all parts apply together), or with a pipe (|) if they are independent. You can also use separate rows for annotations with independent extensions.</li>
<li>The Reference column has the publication’s PubMed ID (PMID).</li>
<li>The taxon will usually be 4896 (the NCBI taxon ID for <em>Schizosaccharomyces pombe</em>), although if you have an NCBI taxon ID for a specific <em>S. pombe</em> strain you are welcome to use it</li>
<li>The date is the date on which the annotations are created; you may use the paper publication date or the date on which you prepare your data file. Format: YYYY-MM-DD</li>
</ol>
<p><strong>Allowed annotation extensions:</strong></p>
<p>Annotation extensions can be used to provide additional information, such as a gene whose product adds or removes a modification, a process or cell cycle phase during which a modification is present, or modification site occupancy (see the <a routerLink="/documentation/gene-page-modifications">gene page modifications documentation</a> for more information). Each annotation extension consists of a relation and either an identifier or a number.</p>
<table>
<thead>
<tr class="header">
<th>Extension relation</th>
<th>Meaning</th>
<th>Example</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>added_by</td>
<td>identifies a gene product that adds the modification</td>
<td>added_by(PomBase:SPBC11B10.09)</td>
</tr>
<tr class="even">
<td>affected_by</td>
<td>identifies a gene product that has some influence on the modification, but has not been conclusively shown to add or remove it</td>
<td>affected_by(PomBase:SPBC11B10.09)</td>
</tr>
<tr class="odd">
<td>removed_by</td>
<td>identifies a gene product that removes the modification</td>
<td>added_by(PomBase:SPAC24H6.05)</td>
</tr>
<tr class="even">
<td>added_during</td>
<td>identifies a biological process or cell cycle phase during which the modification is actively added</td>
<td>added_by(GO:0000085)</td>
</tr>
<tr class="odd">
<td>removed_during</td>
<td>identifies a biological process or cell cycle phase during which the modification is removed</td>
<td>removed_by(GO:0000087)</td>
</tr>
<tr class="even">
<td>present_during</td>
<td>identifies a biological process or cell cycle phase during which the modification is observed (note that the modification may have been added during this process – if this is known use the added_during relation – or during a preceding process)</td>
<td>present_during(GO:0000085)</td>
</tr>
<tr class="odd">
<td>absent_during</td>
<td>identifies a biological process or cell cycle phase during which the modification is not observed (note that the modification may have been removed during this process – if this is known use the removed_during relation – or during a preceding process)</td>
<td>absent_during(GO:0000087)</td>
</tr>
<tr class="even">
<td>required_for</td>
<td>indicates that a modification is required for a GO function or process</td>
<td>required_for(GO:0000086)</td>
</tr>
<tr class="odd">
<td>occupancy</td>
<td>percent representing what proportion of copies of the protein have the modification</td>
<td>occupancy(51.5%)</td>
</tr>
<tr class="even">
<td>level_fluctuates_during</td>
<td>identifies a biological process (e.g. the cell cycle or one or more of its phases) during which the modification site occupancy is observed to vary</td>
<td>level_fluctuates_during(GO:0000278)</td>
</tr>
</tbody>
</table>
<p>Please <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a> if you have any questions about what to use for modification IDs, Evidence, annotation extensions, or anything else you need to represent your data in this format.</p>

  </div>
  <div *ngIf="pageName == 'motif-search'">
<h2 id="peptide-motif-search">Peptide motif search</h2>
<p>The peptide motif search finds short peptide sequence matches in the predicted amino acid sequences of <em>S. pombe</em> proteins.</p>
<p>Type or paste a motif using the single-letter amino acid code. As you type, matches will appear. The results table shows each matching sequence in the context of a longer stretch of the protein sequence. The list of genes with one or more matches can be sent to the advanced search.</p>
<p>The search can also use wildcards, codes for groups of amino acids (e.g. any charged amino acid), and syntax to specify motifs at the beginning or end of a protein:</p>
<table>
<thead>
<tr class="header">
<th>Sequence</th>
<th>will find</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>CADR</td>
<td>any protein containing CADR</td>
</tr>
<tr class="even">
<td>CA[DE]R</td>
<td>any protein containing CADR or CAER</td>
</tr>
<tr class="odd">
<td>CA[DE]+LQ</td>
<td>CA(any sequence of D and E)LQ</td>
</tr>
<tr class="even">
<td>CA…R</td>
<td>CA(any three amino acids)R</td>
</tr>
<tr class="odd">
<td>CA.+R</td>
<td>CA(one or more amino acids)R</td>
</tr>
<tr class="even">
<td>SPR.</td>
<td>SP.R</td>
</tr>
<tr class="odd">
<td>^ME</td>
<td>proteins beginning with ME</td>
</tr>
<tr class="even">
<td>LAA$</td>
<td>proteins terminating LAA</td>
</tr>
<tr class="odd">
<td>^.{{ "{1,20}MCA" }}</td>
<td>proteins with MCA in the first 20 amino acids</td>
</tr>
</tbody>
</table>
<p>Amino acid group codes</p>
<table>
<thead>
<tr class="header">
<th>AA group</th>
<th>Code</th>
<th>Amino acids</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>acidic</td>
<td>0</td>
<td>DE</td>
</tr>
<tr class="even">
<td>alcohol</td>
<td>1</td>
<td>ST</td>
</tr>
<tr class="odd">
<td>aliphatic</td>
<td>2</td>
<td>AGILV</td>
</tr>
<tr class="even">
<td>aromatic</td>
<td>3</td>
<td>FHWY</td>
</tr>
<tr class="odd">
<td>basic</td>
<td>4</td>
<td>KRH</td>
</tr>
<tr class="even">
<td>charged</td>
<td>5</td>
<td>DEHKR</td>
</tr>
<tr class="odd">
<td>hydrophobic</td>
<td>6</td>
<td>AVILMFYW</td>
</tr>
<tr class="even">
<td>hydrophilic</td>
<td>7</td>
<td>KRHDENQ</td>
</tr>
<tr class="odd">
<td>polar</td>
<td>8</td>
<td>CDEHKNQRST</td>
</tr>
<tr class="even">
<td>small</td>
<td>9</td>
<td>ACDGNPSTV</td>
</tr>
<tr class="odd">
<td>tiny</td>
<td>B</td>
<td>AGS</td>
</tr>
<tr class="even">
<td>turnlike</td>
<td>Z</td>
<td>ACDEGHKNQRST</td>
</tr>
</tbody>
</table>
<p>Examples:</p>
<ul>
<li>CA6AC will find CALAC and CAFAC</li>
<li>CA0+LQ will find CA(any sequence of D or E)LQ</li>
</ul>

  </div>
  <div *ngIf="pageName == 'ontology-term-page'">
<h2 id="ontology-term-pages">Ontology term pages</h2>
<p>For each ontology term loaded into PomBase, a page summarizes essential details about the term, and shows any annotations to it or its descendants via <em>is_a</em>, <em>part_of</em>, and the <em>regulates</em> relations (the GO documentation on <a href="http://geneontology.org/docs/ontology-relations/">Ontology Structure and Ontology Relations</a> has some useful information on this topic). For FYPO terms only, the has_part relation is also used.</p>
<p>The main illustrations use a GO term page. Note that not all ontology terms have all of the depicted features.</p>
<figure>
<img src="assets/ontology_term_page_summary.png" width="800" alt="" /><figcaption>ontology term page</figcaption>
</figure>
<ol type="1">
<li>Ontology term details. The term’s name, ID, and definition are shown.</li>
<li>The term’s immediate parent(s), linked to the corresponding ontology term page(s).</li>
<li>Links to external browsers are available for some ontologies. GO, FYPO and PSI-MOD terms have links to <a href="http://bioportal.bioontology.org/">BioPortal</a>, and GO term pages also link to <a href="http://amigo.geneontology.org/amigo">AmiGO</a> and <a href="http://www.ebi.ac.uk/QuickGO/">QuickGO</a>. SO IDs link to <a href="http://sequenceontology.org/browser/obob.cgi">MISO</a>.</li>
<li>Link to a page that lists genes annotated to the term or its descendants. From the linked page, the gene list can be sent to the advanced search for use in queries.</li>
<li>In the annotation summary view, the term of interest is listed first, followed by any descendant term that has direct annotations. For each term, genes (or genotypes) are listed in a compact display, with separate lines only for unique combinations of gene, term, and extension. As in the <a routerLink="/documentation/index#DocsGenePage">gene page summary display</a>, most details are hidden.</li>
</ol>
<p>As on gene pages, a <a routerLink="/documentation/index#DocsGenePage">more detailed annotation display</a> is available:</p>
<figure>
<img src="assets/ontology_term_page_detail.png" width="800" alt="" /><figcaption>ontology term page</figcaption>
</figure>
<ol type="1">
<li>The detailed view shows one line per annotation, with separate entries for repeat determinations of a given gene/term/extension combination (if supported by more than one line of evidence and/or reported in more than one paper). The annotated gene (or genotype) is listed in the first column.</li>
<li>The rest of the detailed annotation display (term name, extensions, evidence, reference, etc.) is as described in the gene page documentation for each annotation type.</li>
<li>The table is organized into sections, one per descendant term, and each section includes genes annotated directly to the indicated term. An icon near each descendant term ID shows the relationship type that links the terms. For example, a meiotic spindle astral microtubule (GO:1990574) is a type of astral microtubule, and a mitotic spindle astral microtubule is part of an astral microtubule.</li>
</ol>
<p>FYPO term pages have a few distinctive features:</p>
<figure>
<img src="assets/ontology_term_page_fypo.png" width="800" alt="" /><figcaption>ontology term page</figcaption>
</figure>
<ol type="1">
<li>Links near the top of the page go to lists of genes and genotypes.</li>
<li>The annotation display is split into single-allele and multi-allele sections, as on gene pages.</li>
<li>Annotated genotypes (rather than genes) are listed for each term.</li>
<li>In the detailed view, evidence filtering is available as described in the <a routerLink="/documentation/gene-page-phenotypes">gene page documentation</a>.</li>
</ol>
<p>Ontology term pages are available for <a href="http://geneontology.org/">GO</a>, <a routerLink="/browse-curation/fission-yeast-phenotype-ontology">FYPO</a>, <a href="http://sequenceontology.org/">SO</a>, <a href="http://obofoundry.org/ontology/mod.html">PSI-MOD</a>, the internal PBO description collection, and PECO, the internal ontology we use for phenotype experimental conditions.</p>

  </div>
  <div *ngIf="pageName == 'orthologs'">
<h2 id="ortholog-curation">Ortholog curation</h2>
<p>The “Orthologs” section of a gene page includes two subsections.</p>
<p>The first lists any orthologous genes in budding yeast (<em>Saccharomyces cerevisiae</em>) or human (<em>Homo sapiens</em>) that have been assessed and recorded manually, as described below, by curators. Budding yeast entries link to the <a href="http://www.yeastgenome.org/"><em>Saccharomyces</em> Genome Database</a> (SGD), and human genes link to <a href="http://www.genenames.org/">HGNC</a>; the text descriptions come from these databases. For more information, including how to search the curated orthologs, see the FAQs on <a routerLink="/faq/how-can-i-search-s.-cerevisiae-ortholog-s-homolog-s-s.-pombe-gene">budding yeast</a> and <a routerLink="/faq/how-can-i-find-s.-pombe-ortholog-s-human-gene">human</a> orthologs.</p>
<p>The second subsection provides links to several resources that predict orthologs in all fungi or all species (PomBase cannot manually curate orthologs in species other than budding yeast and human). For further information, see the <a href="/faq/how-can-i-find-s.-pombe-orthologs-species-other-than-human-and-s.-cerevisiae">FAQ on orthologs in additional species</a>. (Note: these links also appear in the External references section.)</p>
<p>Manual ortholog prediction methods: The human and budding yeast orthologs are manually predicted based on a variety of sources. In some cases the consensus ortholog from the major ortholog predictors (<a href="http://www.ensembl.org/info/docs/compara/index.html">Compara</a>, <a href="http://inparanoid.sbc.su.se/">Inparanoid</a>, <a href="http://www.orthomcl.org/">OrthoMCL</a>) is used. Many distant orthologs have also been identified by <span class="st">PSI-BLAST</span> matches that are uniformly one-to one across species; these alignments have been submitted to the <a href="http://pfam.xfam.org/">Pfam</a> protein family database. Other ortholog predictions come from experimental data demonstrating functional correspondence or involving membership of corresponding complexes. These predictions are also aligned and submitted to Pfam before inclusion. PomBase’s approach ensures that the breadth of coverage is greater than any individual prediction method, and includes many ortholog calls which are not detected by any automated method. Gradually, we will add and display supporting references for all orthology calls.</p>
<h2 id="paralogs">Paralogs</h2>
<p>The Paralogs section lists any curated paralogous fission yeast genes.</p>

  </div>
  <div *ngIf="pageName == 'phenotype-data-bulk-upload-format'">
<h2 id="phenotype-data-bulk-upload-format">Phenotype data bulk upload format</h2>
<p>PomBase welcomes submissions of published large-scale phenotype data sets. We have devised a tab-delimited text file format for bulk phenotype data. A similar format is used for the downloadable file of single-allele phenotype data (with one more column at the start of each line to identify PomBase as the source; note that, because Database is column 1 in the downloadable file, column numbers differ by 1 between the download and upload formats).</p>
<p>Include a header line that labels the columns – use the entry in the Contents column below as the column header text.</p>
<table>
<thead>
<tr class="header">
<th>Column</th>
<th>Contents</th>
<th>Example</th>
<th>Mandatory?</th>
<th>Multiple entries allowed?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1</td>
<td>Gene systematic ID</td>
<td>SPBC11B10.09</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>2</td>
<td>FYPO ID</td>
<td>FYPO:0000001</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>3</td>
<td>Allele description</td>
<td>G146D</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>4</td>
<td>Expression</td>
<td>overexpression</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>5</td>
<td>Parental strain</td>
<td>975 h+</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>6</td>
<td>Background strain name</td>
<td>SP286</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="odd">
<td>7</td>
<td>Background genotype description</td>
<td>h+ ura4-D18 leu1-32 ade6-M210</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="even">
<td>8</td>
<td>Gene name</td>
<td>cdc2</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="odd">
<td>9</td>
<td>Allele name</td>
<td>cdc2-1w</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="even">
<td>10</td>
<td>Allele synonym</td>
<td>wee2-1</td>
<td>No</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>11</td>
<td>Allele type</td>
<td>amino acid mutation</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>12</td>
<td>Evidence</td>
<td>ECO:0000336</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>13</td>
<td>Condition</td>
<td>at high temperature</td>
<td>Yes</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>14</td>
<td>Penetrance</td>
<td>85%</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="odd">
<td>15</td>
<td>Severity</td>
<td>medium</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="even">
<td>16</td>
<td>Extension</td>
<td>assayed_using(PomBase:SPBC582.03)</td>
<td>No</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>17</td>
<td>Reference</td>
<td>PMID:23697806</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>18</td>
<td>taxon</td>
<td>taxon:4896</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>19</td>
<td>Date</td>
<td><span class="no-break">2012-01-01</span></td>
<td>Yes</td>
<td>No</td>
</tr>
</tbody>
</table>
<p><strong>Notes:</strong></p>
<p>Please include all 19 columns. If you have nothing to put in one of the non-mandatory columns, include the header and leave the column blank in the rest of the rows.</p>
<ol type="1">
<li>Include the systematic ID for each gene. You can look up systematic IDs on gene pages, or refer to the file of all gene names from the dataset download page.</li>
<li>For help finding suitable ontology (FYPO) terms to describe your phenotypes, see the <a routerLink="/browse-curation/fission-yeast-phenotype-ontology">FYPO summary page</a> and the <a routerLink="/faq/how-can-i-browse-phenotype-ontology-fypo">FAQ on browsing FYPO</a>. If you can’t find a term you need, email the <a href="mailto:helpdesk@pombase.org">helpdesk</a> for assistance; we can add new FYPO terms as needed.</li>
<li>The allele description specifically describes the change; see table below.</li>
<li>In the Expression column, use one of these values: ‘overexpression’, ‘knockdown’, ‘endogenous’, ‘null’, ‘not specified’. Deletions should always have ‘null’ expression.</li>
<li>The Parental strain column is for the parental strain designation, such as 972 h-, 975 h+, etc. This column must be filled in, but you can use “unknown” if you don’t know the original background.</li>
<li>Use the Strain name (background) column for a lab’s in-house name/ID/designation for the background strain (i.e. the derivative of the parental strain that has selectable marker alleles etc.). The description in the Genotype column should match this background strain.</li>
<li>The Genotype description column is for alleles in the background, such as selectable markers; these details are optional. To avoid redundancy, do not repeat the allele of interest (from column 3 or
<ol start="9" type="1">
<li>in the genotype column.</li>
</ol></li>
<li>Gene names are optional. If you include them, use standard names in column 8 (see gene pages or the file of All Gene Names from the dataset download page).</li>
<li>Allele names are optional. If you include them, use column 9 for the preferred allele name, and put any alternative names in column
<ol start="10" type="1">
<li></li>
</ol></li>
<li>See note 9 above. Separate multiple entries with pipes (|).</li>
<li>Allowed allele types, example descriptions, etc. are shown in the table below.</li>
<li>For the Evidence column, we use a small selection from the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO). You are welcome to enquire with us in advance to find out which ECO terms/IDs fit your experiments, but we can accept files with brief descriptions (such as those in the Canto phenotype evidence option list), which curators will convert to ECO IDs.</li>
<li>Similarly, Conditions use a small ontology maintained in-house by PomBase curators, and we can either advise you about which terms/IDs to use, or convert from text to IDs when we receive your file. Use multiple entries in cases where more than one condition detail applies at the same time (e.g. high temperature, minimal medium). Separate multiple entries with commas (,). Use separate lines if a phenotype is observed under more than one set of conditions (e.g. high and low temperature).</li>
<li>Penetrance describes the proportion of a population that shows a cell-level phenotype. Use decimals, percents, or “high” (above 80%), “medium” (20-80%), or “low” (less than 20%). We will convert to suitable IDs for loading. Penetrance data will be displayed as annotation extensions on gene pages.</li>
<li>Severity (formerly designated “expressivity) uses”high" (synonym: strong), “medium”, or “low” (synonym: weak). We will convert to suitable IDs for loading. Severity data will be displayed as annotation extensions on gene pages.</li>
<li>The Extension column can be used to record when a mutation in one gene affects another gene or its product. For example, if a mutation in gene A decreases its ability to phosphorylate protein B, you can use the phenotype “decreased protein kinase activity” and put the ID for gene B in an extension. Multiple extensions can be included for a phenotype annotation. Separate extensions with a comma (,) if they combine to form a “compound” extension (two or more genes assayed together), or with a pipe (|) if they are independent. Most phenotype extensions will be independent and pipe-separated.</li>
<li>The Reference column has the publication’s PubMed ID (PMID).</li>
<li>The taxon will usually be 4896 (the NCBI taxon ID for <em>Schizosaccharomyces pombe</em>), although if you have an NCBI taxon ID for a specific <em>S. pombe</em> strain you are welcome to use it</li>
<li>The date is the date on which the annotations are created; you may use the paper publication date or the date on which you prepare your data file. Format: YYYY-MM-DD</li>
</ol>
<p><strong>Details for allele types and descriptions:</strong></p>
<p>General note: Nucleotide and amino acid positions should reflect the current sequence data in PomBase. Please refer to the <a routerLink="/status/gene-coordinate-changes">Gene Coordinate Changes</a> page to ensure that your residue position entries are up to date.</p>
<p>For protein-coding genes, number nucleotide residues from 1 starting with the A of the initiator ATG.</p>
<table>
<thead>
<tr class="header">
<th>Allele type (col. 11)</th>
<th>Example allele description (col. 3)</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>amino acid mutation</td>
<td>S123A</td>
<td>use one-letter code; if more than one change, separate with comma(s)</td>
</tr>
<tr class="even">
<td>deletion</td>
<td>deletion</td>
<td>use this description for complete deletions</td>
</tr>
<tr class="odd">
<td>nonsense mutation</td>
<td>K111-&gt;stop</td>
<td></td>
</tr>
<tr class="even">
<td>nucleotide mutation</td>
<td>C123A</td>
<td>if more than one change, separate with comma(s)</td>
</tr>
<tr class="odd">
<td>disruption</td>
<td>pab1::ura4+</td>
<td>expression will usually, but not always, be null</td>
</tr>
<tr class="even">
<td>other</td>
<td>RGTPI inserted after I254</td>
<td>include a brief text description</td>
</tr>
<tr class="odd">
<td>partial amino acid deletion</td>
<td>1-100</td>
<td>indicate deleted residues; use comma-separated ranges for discontinuous deleted segments</td>
</tr>
<tr class="even">
<td>partial nucleotide deletion</td>
<td>500-800</td>
<td>indicate deleted residues; use comma-separated ranges for discontinuous deleted segments</td>
</tr>
<tr class="odd">
<td>unknown</td>
<td>unknown</td>
<td>an allele name is required if the type and description are unknown</td>
</tr>
<tr class="even">
<td>wild type</td>
<td>wild type</td>
<td>use with altered expression (overexpression or knockdown) for single-allele phenotypes</td>
</tr>
</tbody>
</table>
<p>Please <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a> if you have any questions about what to use for Evidence, Conditions, etc., or anything else you need to represent your data in this format.</p>
<p>Return to the <a routerLink="/browse-curation/fission-yeast-phenotype-ontology">Fission Yeast Phenotype Ontology</a> page</p>

  </div>
  <div *ngIf="pageName == 'pombase-go-slim-documentation'">
<h2 id="go-slims---gene-ontology-subsets">GO Slims - Gene Ontology subsets</h2>
<p>“GO slims” are subsets of the Gene Ontology (GO) that provide a broad overview of annotation distribution. Slims can offer a useful overview of a genome or the results of a large-scale experiment. For more information on GO slims, please see PomBase <a href="/browse-curation/fission-yeast-go-slimming-tips">GO Slimming Tips</a> and the <a href="http://geneontology.org/docs/go-subset-guide/">GO Subset Guide</a> at the Gene Ontology website.</p>
<p>PomBase provides a GO slim term set for each major branch of GO:</p>
<ul>
<li><a routerLink="/browse-curation/fission-yeast-mf-go-slim-terms">GO molecular function slim term set</a></li>
<li><a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">GO biological process slim term set</a></li>
<li><a routerLink="/browse-curation/fission-yeast-cc-go-slim-terms">GO cellular component slim term set</a></li>
</ul>
<p>Each fission yeast GO slim has been constructed to optimise coverage of gene products annotated to terms in the branch of GO. For the biological process and cellular component branches, coverage is almost complete; the molecular function branch has somewhat lower coverage. These slims provide good starting points for users who wish to identify terms of biological interest to create a “custom slim”, or to become familiar with the genome contents.</p>
<p>In the GO slim tables, GO IDs link to the QuickGO browser, where you can explore the ontology and annotations further. The annotation totals include annotations to the slim term and to descendants following the <em>is_a</em>, <em>part_of</em>, <em>regulates</em>, <em>positively_regulates,</em> and <em>negatively_regulates</em> relationships, and link to gene lists. (Note: the cellular component ontology does not contain any “regulates” links.)</p>
<p>The GO biological process slim table also includes links to visualisations in <a href="http://www.esyn.org/">esyN</a> for the physical interaction network of genes annotated to each term.</p>
<p>The bottom of each GO slim page lists some simple statistics: the total number of gene products annotated to slim terms, and the number of protein-coding gene products not covered by the slim, either because they are not annotated to any term in the GO branch, or because they have annotations to terms not covered by the slim.</p>
<p>Current GO slim IDs and term names can be downloaded from the PomBase ftp site:</p>
<ul>
<li><a href="ftp://ftp.pombase.org/nightly_update/misc/mf_goslim_pombe_ids_and_names.tsv">GO molecular function slim</a></li>
<li><a href="ftp://ftp.pombase.org/nightly_update/misc/bp_goslim_pombe_ids_and_names.tsv">GO biological process slim</a></li>
<li><a href="ftp://ftp.pombase.org/nightly_update/misc/cc_goslim_pombe_ids_and_names.tsv">GO cellular component slim</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'publication-page'">
<h2 id="publication-pages">Publication pages</h2>
<p>For each publication loaded into PomBase, a page summarizes essential details about paper, and shows any annotations that cite it.</p>
<figure>
<img src="assets/publication_page_top.png" width="800" alt="" /><figcaption>publication page top details</figcaption>
</figure>
<ol type="1">
<li><p>Publication details, including links to <a href="https://www.ncbi.nlm.nih.gov/pubmed/">PubMed</a> and <a href="http://europepmc.org/">EuropePMC</a></p></li>
<li><p>Any community curators who have contributed annotations via Canto are acknowledged.</p></li>
<li><p>Summary views are identical to those on <a routerLink="/documentation/index#DocsGenePage">gene pages</a>.</p></li>
<li><p>In detailed views annotations are displayed as on <a routerLink="/documentation/index#DocsGenePage">gene pages</a>, except that a column for the annotated gene is included and the Reference column is omitted.</p></li>
<li><p>Term and evidence filtering are available for phenotype annotations as described in the <a routerLink="/documentation/gene-page-phenotypes">gene page documentation</a>.</p></li>
</ol>
<p>Like gene pages, publication pages include annotations of all types, with summary and detailed views available for most types:</p>
<figure>
<img src="assets/publication_page_overview.png" width="800" alt="" /><figcaption>publication page</figcaption>
</figure>

  </div>
  <div *ngIf="pageName == 'qualitative-gene-expression-data-bulk-upload-format'">
<h2 id="qualitative-gene-expression-bulk-upload-file-format">Qualitative gene expression bulk upload file format</h2>
<p>PomBase welcomes submissions of published large-scale gene expression data sets. We have devised a tab-delimited text file format for bulk data representing qualitative assessments of protein or RNA levels.</p>
<p>Note: Please use this format only for data that you want to appear in the “Qualitative gene expression” section of PomBase gene pages. If you have gene expression data that should be included as a data track in the PomBase genome browser (microarray, RNASeq, etc.), please use the <a routerLink="/submit-data/data-submission-form-for-HTP-sequence-linked-data">data submission form for HTP sequence-linked data</a>.</p>
<p>Include a header line that labels the columns – use the entry in the Contents column below as the column header text.</p>
<table>
<thead>
<tr class="header">
<th>Column</th>
<th>Contents</th>
<th>Example</th>
<th>Mandatory?</th>
<th>Multiple entries allowed?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1</td>
<td>Gene systematic ID</td>
<td>SPBC11B10.09</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>2</td>
<td>Gene name</td>
<td>cdc2</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="odd">
<td>3</td>
<td>Type</td>
<td>protein</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>4</td>
<td>Evidence</td>
<td>ECO:0000006</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>5</td>
<td>Qualifier</td>
<td>increased</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>6</td>
<td>Extension</td>
<td>during(GO:0000084)</td>
<td>Yes</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>7</td>
<td>Reference</td>
<td>PMID:18203864</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>8</td>
<td>Taxon</td>
<td>taxon:4896</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>9</td>
<td>Date</td>
<td><span class="no-break">2014-05-01</span></td>
<td>Yes</td>
<td>No</td>
</tr>
</tbody>
</table>
<ol type="1">
<li>Include the systematic ID for each gene. You can look up systematic IDs on gene pages, or refer to the file of all gene names from the dataset download page.</li>
<li>Gene names are optional. If you include them, use standard names in column 2 (see gene pages or the file of all gene names from the dataset download page.</li>
<li>Type: what was measured; use “RNA” or “protein”</li>
<li>For the Evidence column, we use a small selection from the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO). You are welcome to enquire with us in advance to find out which ECO terms/IDs fit your experiments, but we can accept files with brief descriptions, which curators will convert to ECO IDs.</li>
<li>The Qualifier column uses a small vocabulary to describe what is observed about the RNA or protein level. Use one of: present, absent, unchanged, increased, decreased, constant, fluctuates</li>
<li>The Extension column documents circumstances under which the RNA or protein level is observed (or changes). Follow one of these patterns: “during (GO:0000084)” - when a specific biological process, such as a response to a stimulus, is taking place, or during a cell cycle phase; “in_presence_of(CHEBI:18420)” - when a specific chemical substance is present; “occurs_in(GO:0005634)” when the level is observed in part, rather than all, of a cell.</li>
<li>The Reference column has the publication’s PubMed ID (PMID).</li>
<li>The taxon will usually be 4896 (the NCBI taxon ID for <em>Schizosaccharomyces pombe</em>), although if you have an NCBI taxon ID for a specific <em>S. pombe</em> strain you are welcome to use it</li>
<li>The date is the date on which the annotations are created; you may use the paper publication date or the date on which you prepare your data file. Format: YYYY-MM-DD</li>
</ol>

  </div>
  <div *ngIf="pageName == 'quantitative-gene-expression-data-bulk-upload-format'">
<h2 id="quantitative-gene-expression-bulk-upload-file-format">Quantitative gene expression bulk upload file format</h2>
<p>PomBase welcomes submissions of published large-scale gene expression data sets. We have devised a tab-delimited text file format for bulk data representing representing protein or RNA copy number per cell.</p>
<p>Note: Please use this format only for data that you want to appear in the “Quantitative gene expression” section of PomBase gene pages. If you have gene expression data that should be included as a data track in the PomBase genome browser (microarray, RNASeq, etc.), please use the <a routerLink="/submit-data/data-submission-form">data submission form for HTP sequence-linked data</a>.</p>
<p>Include a header line that labels the columns – use the entry in the Contents column below as the column header text.</p>
<table>
<thead>
<tr class="header">
<th>Column</th>
<th>Contents</th>
<th>Example</th>
<th>Mandatory?</th>
<th>Multiple entries allowed?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1</td>
<td>Gene systematic ID</td>
<td>SPBC11B10.09</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>2</td>
<td>Gene name</td>
<td>cdc2</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="odd">
<td>3</td>
<td>Type</td>
<td>protein</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>4</td>
<td>Extension</td>
<td>during(GO:0051329)</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="odd">
<td>5</td>
<td>Copies per cell</td>
<td>1234</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>6</td>
<td>Range</td>
<td>1100-1350</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="odd">
<td>7</td>
<td>Evidence</td>
<td>ECO:0000006</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>8</td>
<td>Scale</td>
<td>single_cell</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>9</td>
<td>Condition</td>
<td>minimal medium, high temperature</td>
<td>Yes</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>10</td>
<td>Reference</td>
<td>PMID:23101633</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>11</td>
<td>Taxon</td>
<td>4896</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>12</td>
<td>Date</td>
<td><span class="no-break">2014-05-01</span></td>
<td>Yes</td>
<td>No</td>
</tr>
</tbody>
</table>
<ol type="1">
<li>Include the systematic ID for each gene. You can look up systematic IDs on gene pages, or refer to the file of all gene names from the dataset download page.</li>
<li>Gene names are optional. If you include them, use standard names in column 2 (see gene pages or the file of all gene names from the dataset download page.</li>
<li>Type: what was measured; use “RNA” or “protein”</li>
<li>The Extension column documents circumstances under which the RNA or protein level is observed (or changes). Follow one of these patterns: “during (GO:0000084)” - when a specific biological process, such as a response to a stimulus, is taking place, or during a cell cycle phase; “in_presence_of(CHEBI:18420)” - when a specific chemical substance is present; “occurs_in(GO:0005634)” when the level is observed in part, rather than all, of a cell.</li>
<li>Copies per cell: can be an average</li>
<li>Range: optional; use if the experimental method yields a range of copies/cell</li>
<li>For the Evidence column, we use a small selection from the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO). You are welcome to enquire with us in advance to find out which ECO terms/IDs fit your experiments, but we can accept files with brief descriptions, which curators will convert to ECO IDs.</li>
<li>Scale: whether levels are determined for a single cell or a cell population as a whole; use “single_cell” or “population”</li>
<li>Conditions use a small ontology maintained in-house by PomBase curators, and we can either advise you about which terms/IDs to use, or convert from text to IDs when we receive your file.</li>
<li>The Reference column has the publication’s PubMed ID (PMID).</li>
<li>The taxon will usually be 4896 (the NCBI taxon ID for <em>Schizosaccharomyces pombe</em>), although if you have an NCBI taxon ID for a specific <em>S. pombe</em> strain you are welcome to use it</li>
<li>The date is the date on which the annotations are created; you may use the paper publication date or the date on which you prepare your data file. Format: YYYY-MM-DD</li>
</ol>

  </div>
  <div *ngIf="pageName == 'quick-little-tool'">
<h2 id="quick-little-tool">Quick Little Tool</h2>
<p>PomBase’s Quick Little Tool (QuiLT) allows you to view multiple types of annotation for genes in a list in a single graphical display.</p>
<figure>
<img src="assets/quilt_display.png" width="600" alt="" /><figcaption>Quick Little Tool display</figcaption>
</figure>
<h3 id="finding-quilt">Finding QuiLT</h3>
<p>On any gene list page, including Advanced Search results and the gene lists linked to ontology term pages, click the “Visualise” button to reach QuiLT. To return to the previous page, use the “Finish visualisation” button.</p>
<h3 id="adding-and-sorting-annotations">Adding and sorting annotations</h3>
<p>In the QuiLT display, each row represents a single gene, and each column is an annotation type.</p>
<p>Use the checkboxes (1) to toggle an annotation type on and off. When annotations are shown, a link appears to sort on that annotation type. You may want to try different combinations of which types are shown, and which used to sort the list, until you get a display you like. You can download the graph (with key) in SVG format at any time (2). The key (4) shows only terms used to annotate genes in the list, so for small gene lists not all terms may appear.</p>
<p>Click on a row to select a single gene, or click on a box in any column (3) to select all genes in it. In the displayed gene list, you can clear the selection (5) or use the “Filter” button (6) to create a new gene list from the selection (which you can then visualise in QuiLT).</p>
<h3 id="sharing-results">Sharing results</h3>
<p>Each QuiLT visualisation page has a stable, unique URL that you can bookmark, copy/paste, and share. Anyone who follows a shared link will see QuiLT displaying the same gene list. Clicking the “Advanced Search” button adds a query that would produce the gene list to the query history.</p>
<h3 id="available-annotation-types">Available annotation types</h3>
<p>At present, QuiLT includes:</p>
<ul>
<li>Deletion viability</li>
<li>Presence or absence of budding yeast orthologs</li>
<li>Presence or absence of human orthologs</li>
<li>GO biological process</li>
<li>GO cellular component</li>
<li>GO molecular function</li>
<li><a routerLink="/status/gene-characterisation">Characterization status</a> for protein-coding genes</li>
<li><a routerLink="/documentation/taxonomic-conservation">Taxonomic distribution</a></li>
<li>Protein length</li>
</ul>
<h4 id="go-annotation-precedence">GO annotation precedence</h4>
<p>In each branch of GO, only one term can be included for display. If a gene is annotated to more than one GO term, one is selected for the QuiLT display according to a set order of precedence:</p>
<ul>
<li>GO biological process
<ul>
<li>signaling (GO:0023052)</li>
<li>gene expression (GO:0010467)</li>
<li>chromatin remodeling (GO:0006338)</li>
<li>protein folding (GO:0006457)</li>
<li>cellular component biogenesis (GO:0044085)</li>
<li>DNA metabolic process (GO:0006259)</li>
<li>cell cycle (GO:0007049)</li>
<li>cytoskeleton organization (GO:0007010)</li>
<li>membrane organization (GO:0061024)</li>
<li>organelle localization (GO:0051640)</li>
<li>lipid metabolic process (GO:0006629)</li>
<li>small molecule metabolic process (GO:0044281)</li>
<li>generation of precursor metabolites and energy (GO:0006091)</li>
<li>transport (GO:0006810)</li>
<li>cellular catabolic process (GO:0044248)</li>
<li>detoxification (GO:0098754)</li>
<li>other (i.e none of the above)</li>
<li>unknown</li>
</ul></li>
<li>GO cellular component
<ul>
<li>nucleus (GO:0005634)</li>
<li>endoplasmic reticulum (ER; GO:0005783)</li>
<li>mitochondrion (GO:0005739)</li>
<li>cytoplasm (GO:0005737)</li>
<li>other (i.e none of the above)</li>
<li>unknown</li>
</ul></li>
<li>GO molecular function
<ul>
<li>oxidoreductase activity (GO:0016491)</li>
<li>hydrolase activity (GO:0016787)</li>
<li>transferase activity (GO:0016740)</li>
<li>ligase activity (GO:0016874)</li>
<li>cyclase activity (GO:0009975)</li>
<li>lyase activity (GO:0016829)</li>
<li>unknown catalytic activity (GO:0003824)</li>
<li>enzyme regulator activity (GO:0030234)</li>
<li>DNA binding (GO:0003677)</li>
<li>transcription regulator activity (GO:0140110)</li>
<li>transporter activity (GO:0005215)</li>
<li>cytoskeletal protein binding (GO:0008092)</li>
<li>unfolded protein binding (GO:0051082)</li>
<li>RNA binding (GO:0003723)</li>
<li>metal ion binding (GO:0046872)</li>
<li>lipid binding (GO:0008289)</li>
<li>cell adhesion mediator activity (GO:0098631)</li>
<li>structural molecule activity (GO:0005198)</li>
<li>other (i.e none of the above)</li>
<li>unknown</li>
</ul></li>
</ul>

  </div>
  <div *ngIf="pageName == 'simple-search-documentation'">
<h2 id="simple-search">Simple search</h2>
<p>The simple search is available via the search box in the header on every PomBase page:</p>
<figure>
<img src="assets/simple_search_mouseover_help.png" width="600" alt="" /><figcaption>Simple search mouseover help</figcaption>
</figure>
<p>The simple search works for:</p>
<ul>
<li>Fission yeast gene names, synonyms, and systematic IDs</li>
<li>Fission yeast gene product descriptions</li>
<li>Human orthologs of fission yeast genes — search by standard gene symbols</li>
<li>Budding yeast orthologs of fission yeast genes — search by standard gene names or ORF names</li>
<li>Publications — search by PubMed ID</li>
<li>Ontology terms — search by term ID</li>
</ul>
<p>Start typing to see a pulldown of autocomplete matches, then choose one, or if there is a single match simply hit return, to go to the matching gene page.</p>
<figure>
<img src="assets/simple_search_dropdown.png" width="170" alt="" /><figcaption>Simple search autocomplete matches</figcaption>
</figure>
<p>For publications or ontology terms, type or paste an ID as directed.</p>

  </div>
  <div *ngIf="pageName == 'taxonomic-conservation'">
<h2 id="taxonomic-conservation">Taxonomic conservation</h2>
<p>The Taxonomic conservation section provides manually assigned classifiers of taxon distribution (at the domain/kingdom level) of the product of a protein-coding gene. The classifiers come from a small controlled vocabulary maintained by PomBase curators. All applicable terms are assigned to a protein.</p>
<p>For each annotation, the summary view shows a text description, which corresponds to an entry in the internal PBO <!-- <a routerLink="/faq/what-pbo-option-advanced-search">PBO</a> --> term set. The detailed view adds the PBO ID and a count that links to the <a routerLink="/documentation/ontology-term-page">ontology term page</a> for the description.</p>
<h3 id="taxonomic-conservation-terms">Taxonomic conservation terms</h3>
<ul>
<li>These terms identify the taxonomic groups in which gene product ortholog(s) have been identified:
<ul>
<li>conserved in archaea</li>
<li>conserved in bacteria</li>
<li>conserved in eukaryotes</li>
<li>conserved in fungi</li>
<li>conserved in metazoa</li>
<li>conserved in vertebrates</li>
</ul></li>
<li>These two terms represent taxon restrictions, and indicate that a gene product has not been detected outside these taxa:
<ul>
<li>conserved in fungi only</li>
<li>conserved in eukaryotes only</li>
</ul></li>
<li>Other terms
<ul>
<li><em>Schizosaccharomyces</em> specific - indicates that a gene product is only identified in the genus <em>Schizosaccharomyces</em> at present, but is present in more than one species in this genus</li>
<li><em>Schizosaccharomyces pombe</em> specific - indicates that a gene product is identified only in <em>S. pombe</em>, not in any other species even in the <em>Schizosaccharomyces</em> genus</li>
<li>no apparent <em>S. cerevisiae</em> ortholog - indicates a gene product which is conserved at least outside the <em>Schizosaccharomyces</em> but is absent from <em>S. cerevisiae</em></li>
<li>faster evolving duplicate - indicates that the copy of a gene product which has duplicated after the divergence of <em>Schizosaccharomyces</em> from other yeasts is evolving faster than its duplicate partner (judged because the fission yeast duplicate is more closely related to the orthologs in other species, but is the best hit of this gene product)</li>
<li>predominantly single copy (one to one) - the gene product is conserved largely one-to one across species with minor exceptions (for example, frequently duplicated in the vertebrate lineage)</li>
<li>orthologs cannot be distinguished - the gene product appears to be conserved, but it is not possible to identify orthologs unambiguously</li>
</ul></li>
</ul>
<h3 id="additional-notes">Additional notes:</h3>
<ul>
<li>Any term which has “conserved in fungi” but does <em>not</em> have “no apparent <em>S. cerevisiae</em> ortholog” will have a manually annotated <em>S. cerevisiae</em> ortholog.</li>
<li>Any gene product which is annotated as “conserved in vertebrates” will have a manually annotated human ortholog.</li>
</ul>

  </div>
  </div>
</div>
<div *ngIf="section == 'documents'">
<app-page-contents-menu title="Documents" titleRoute="/documents">

<div class="left-menu-part left-menu-item"><a routerLink="/documents/pombe2017">pombe2017</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documents/pombe2015">pombe2015</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documents/pombe2013">pombe2013</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documents/pombe2011">pombe2011</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'index'">
<h2 id="downloadable-documents">Downloadable Documents</h2>
<p>For FTP downloads, connect as “guest” if prompted; see <a href="faq/do-i-need-password-download-ftp-site">this FAQ</a> for more information.</p>
<h3 id="conferences-and-workshops">Conferences and Workshops</h3>
<h4 id="international-fission-yeast-meeting-series">International Fission Yeast Meeting series</h4>
<p><a routerLink="/documents/pombe2017">pombe2017</a> Banff, Alberta, Canada 14-19 May 2017</p>
<p><a routerLink="/documents/pombe2015">pombe2015</a> Kobe, Japan 21-25 June 2015</p>
<p><a routerLink="/documents/pombe2013">pombe2013</a> London, UK 24-29 June 2013</p>
<p><a routerLink="/documents/pombe2011">pombe2011</a> Boston, MA, USA 25-30 June 2011</p>
<h4 id="other-conferences-and-workshops">Other Conferences and Workshops</h4>
<ul>
<li><a href="ftp://ftp.pombase.org/archive/Presentations_workshops_and_meeting_abstracts/EMBO_workshop/">2009 EMBO Workshop</a>, Manchester, UK (FTP directory)</li>
<li><a href="ftp://ftp.pombase.org/archive/Presentations_workshops_and_meeting_abstracts/Computing_Workshop/">Computing workshops, 2006 &amp; 2007</a>, various dates and locations (FTP directory)</li>
<li><a href="ftp://ftp.pombase.org/archive/Presentations_workshops_and_meeting_abstracts/Computing_Workshop/Fission_yeast_computing_workshop_man_Mar2006.ppt">2006 Fission yeast genome analysis workshop</a>, Hinxton, UK - course manual (file at FTP site)</li>
<li><a href="ftp://ftp.pombase.org/archive/Presentations_workshops_and_meeting_abstracts/Pombe_meeting_abstract_books/Wellcome_Euro_Yeast_27_2_06.pdf">2006 European fission yeast meeting</a>, Hinxton, UK - abstracts (file at FTP site)</li>
</ul>
<h2 id="other-documents">Other documents</h2>
<ul>
<li><p><a href="ftp://ftp.pombase.org/pombe/documents/2019_pombase_survey_summary_no_freetext_responses.pdf">2019 PomBase user survey results</a> (PDF; file at FTP site)</p></li>
<li><p><a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2017/pombe2017_infographic_p132.pdf">2017 PomBase infographic</a> (PDF; file at FTP site)</p></li>
<li><p><a href="ftp://ftp.pombase.org/pombe/documents/2013_pombase_survey_summary.pdf">2013 PomBase user survey results</a> (PDF; file at FTP site)</p></li>
</ul>

  </div>
  <div *ngIf="pageName == 'pombe2011'">
<h2 id="pombe2011---6th-international-fission-yeast-meeting">pombe2011 - 6th International Fission Yeast Meeting</h2>
<p>Boston, MA, USA, 25-30 June 2011</p>
<h3 id="pombase-posters">PomBase Posters</h3>
<ul>
<li>P50: <a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2011/phenotype_pombe2011.pdf">Phenotypes in PomBase and Beyond</a></li>
<li>P96: <a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2011/pombe2011_pombase.pdf">PomBase.org</a></li>
<li>P159: <a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2011/pombe2011_p159.pdf">PomBase Communty Curation: A fast track to capture expert knowledge</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'pombe2013'">
<h2 id="pombe2013---seventh-interational-fission-yeast-meeting">pombe2013 - Seventh Interational Fission Yeast Meeting</h2>
<p>London, UK, 24-29 June 2013</p>
<h3 id="abstracts">Abstracts</h3>
<ul>
<li><a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2013/pombe2013_abstractbook-final2.pdf">Abstract book</a> (PDF)</li>
</ul>
<h3 id="pombase-presentations">PomBase presentations</h3>
<h4 id="talks">Talks</h4>
<ul>
<li>Tuesday Lunchtime: Paper Curation - Midori Harris and Antonia Lock</li>
<li>Thursday Lunchtime: PomBase Gene Pages and the Genome Browser - Val Wood and Mark McDowall</li>
</ul>
<h4 id="posters">Posters</h4>
<ul>
<li>219: <a href="ftp://ftp.pombase.org/pombeConferences/pombe2013/pombe2013_p219.pdf">A configurable tool for community literature curation</a></li>
<li>220: <a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2013/PomBase.org.pdf">PomBase.org</a></li>
<li>221: <a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2013/pombe2013_p221.pdf">Phenotype curation in PomBase</a></li>
<li>222: Using Gene Ontology biological process co-annotation for ontology and annotation quality control</li>
</ul>

  </div>
  <div *ngIf="pageName == 'pombe2015'">
<h2 id="pombe2015---eighth-international-fission-yeast-meeting">pombe2015 - Eighth International Fission Yeast Meeting</h2>
<p>Kobe, Japan, 21-26 June 2015</p>
<h3 id="pombase-presentations">PomBase presentations</h3>
<h4 id="talks">Talks</h4>
<ul>
<li>PomBase Workshop
<ul>
<li><a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2015/workshop_pombase_tour_vw2015.pdf">How to get more from PomBase</a> - a brief tour of PomBase gene pages, curated data types, querying, tools, and external links</li>
<li><a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2015/pombe2015_UsingGenomeBrowser.pdf">Using the PomBase Ensembl Browser</a> - a demo of the PomBase genome browser features and functions (PDF; large file)</li>
<li><a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2015/pombe2015_curation_workshop.pdf">Community curation using Canto</a> - how to create, edit and submit annotations based on published experiments</li>
</ul></li>
<li>81: Hidden in plain sight: The eukaryotically conserved unstudied proteins and a framework for their classification and characterisation</li>
<li>82: <a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2015/multi-phenotype_pombe2015.pdf">Curating multi-allele phenotypes in PomBase</a> (PDF; poster version)</li>
</ul>
<h4 id="posters">Posters</h4>
<ul>
<li>33: <a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2015/pombe2015_p33.pdf">Putting it all together with fission yeast: Data connections within and beyond PomBase</a></li>
<li>73: <a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2015/PomBaseGenomeBrowser.pdf">Using the PomBase Genome Browser</a> (PDF)</li>
</ul>

  </div>
  <div *ngIf="pageName == 'pombe2017'">
<h2 id="pombe2017---9th-international-fission-yeast-meeting">pombe2017 - 9th International Fission Yeast Meeting</h2>
<p>Banff, Alberta, Canada, 14-19 May 2017</p>
<h3 id="pombase-posters">PomBase Posters</h3>
<ul>
<li>132: <a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2017/pombe2017_infographic_p132.pdf">Fission Yeast People, Papers and PomBase</a> (The PomBase Infographic; PDF)</li>
<li>133: <a href="ftp://ftp.pombase.org/pombe/Conferences/pombe2017/pombe2017_p133.pdf">PomBase Community Literature Curation</a> (PDF)</li>
<li>164: Using fission yeast QC methods to correct and improve GO annotation across eukaryotic species</li>
</ul>
<h3 id="talk">Talk</h3>
<p>The new, improved PomBase</p>
<p>External link to <a href="http://people.ucalgary.ca/~young/pombe2017_program.pdf">conference program</a></p>

  </div>
  </div>
</div>
<div *ngIf="section == 'downloads'">
<app-page-contents-menu title="Downloads" titleRoute="/downloads">

<div class="left-menu-part left-menu-item"><a routerLink="/datasets">Datasets</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="//downloads/genome-datasets">Genome datasets</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="//downloads/genome-datasets#Annotation_datasets">Annotation datasets</a></div>
<div class="left-menu-part left-sub-sub-menu-item"><a routerLink="/downloads/go-annotations">GO annotations</a></div>
<div class="left-menu-part left-sub-sub-menu-item"><a routerLink="/downloads/phenotype-annotations">Phenotype annotations</a></div>
<div class="left-menu-part left-sub-sub-menu-item"><a href="ftp://ftp.pombase.org/pombe/annotations/modifications/">Modifications</a></div>
<div class="left-menu-part left-sub-sub-menu-item"><a href="ftp://ftp.pombase.org/pombe/exports/">HCPIN datasets</a></div>
<div class="left-menu-part left-sub-sub-menu-item"><a href="ftp://ftp.pombase.org/pombe/orthologs/">Orthologs</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="//downloads/protein-datasets">Protein datasets</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/downloads/names-and-identifiers">Names and IDs</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documents">Documents</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/downloads/chado-database-dumps">Chado database dumps</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'chado-database-dumps'">
<h2 id="chado-database-dumps">Chado database dumps</h2>
<p>The <a href="ftp://ftp.pombase.org/nightly_update/">nightly release</a> of PomBase data includes a dump from the PomBase <a href="http://gmod.org/wiki/Chado">Chado</a> database.</p>
<p>Archived Chado data dumps are also available for <a href="http://curation.pombase.org/releases/">download</a>. From January 2018 onwards, monthly data snapshots are provided. Earlier archived dumps correspond to the intermittent PomBase releases made between July 2012 and January 2017. For these dumps, the file names include release numbers that correspond to the “Annotation Version” numbers in the <a routerLink="/about/version-history">Data version history</a> table.</p>
<p>Please <a href="mailto:helpdesk@pombase.org">contact the curators</a> if you need any data not available at the links above.</p>
<p>The nightly release is on the PomBase FTP site. The other Chado dumps can also be retrieved by <a href="ftp://ftp.pombase.org/releases/">FTP</a> as well as HTTP. For FTP downloads, connect as “guest” if prompted; see <a routerLink="/faq/do-i-need-password-download-ftp-site">this FAQ</a> for more information.</p>

  </div>
  <div *ngIf="pageName == 'genome-datasets'">
<h2 id="genome-sequence-and-features">Genome sequence and features</h2>
<div style="color: red">
<p><strong>Warning:</strong> Files in the old FTP site within the EBI domain are no longer updated. If you have a link that contains <strong>ftp.ebi.ac.uk</strong>, please check these pages for a link using <strong>ftp.pombase.org</strong>. Please <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a> if you need help finding a file or directory.</p>
</div>
<p>Links below go to web pages or directly to directories within the PomBase FTP site. When you reach the FTP site, choose the file(s) you need and click to download (connect as “guest” if prompted; see <a href="faq/do-i-need-password-download-ftp-site">this FAQ</a> for more information).</p>
<p>Note that most of the datasets available here are compressed (.gz), and can be uncompressed by utilities available in all common operating systems.</p>
<h3 id="genome-sequence">Genome sequence</h3>
<p>The current <a href="ftp://ftp.pombase.org/pombe/genome_sequence_and_features/genome_sequence/">genome sequence</a> (ftp) is available in FASTA format. The linked directory contains a file for the whole genome sequence as well as separate files for each chromosome.</p>
<h4 id="feature-coordinates-only">Feature coordinates only</h4>
<p>These files contain coordinates, but no sequence data:</p>
<ul>
<li><a href="ftp://ftp.pombase.org/pombe/genome_sequence_and_features/gff3/">GFF3 format</a> (ftp) files contain coordinates for features of all types.</li>
<li><a href="ftp://ftp.pombase.org/pombe/genome_sequence_and_features/CDS_Coordinates/">CDS coordinates</a> (ftp) are in a tab-delimited file with columns for systematic ID, start coordinate, end coordinate, and strand.</li>
<li><a href="ftp://ftp.pombase.org/pombe/genome_sequence_and_features/Exon_Coordinates/">Exon coordinates</a> (ftp) are in the same tab-delimited file format as CDS coordinates.</li>
</ul>
<h4 id="feature-coordinates-and-sequence-data">Feature coordinates and sequence data</h4>
<ul>
<li><a href="ftp://ftp.pombase.org/pombe/genome_sequence_and_features/artemis_files/">Chromosome contigs</a> (ftp) in enriched EMBL-based format for loading into Artemis (see the <a routerLink="/faq/there-equivalent-artemis-java-applet-pombase">Artemis FAQ</a></li>
<li>Older files in <a href="ftp://ftp.pombase.org/pombe/genome_sequence_and_features/OLD/20170906/embl/">EMBL format</a> and <a href="ftp://ftp.pombase.org/pombe/genome_sequence_and_features/OLD/20170906/genbank/">GenBank format</a> are available, and reflect the current genome sequence, with feature annotations from January 2017 (ftp).</li>
</ul>
<!-- put this between the two existing lines above:
-   [Manually curated LTRs]() in GFF3 format
-->
<h4 id="feature-sequences">Feature sequences</h4>
<p><a href="ftp://ftp.pombase.org/pombe/genome_sequence_and_features/feature_sequences/">Sequences in FASTA format</a> for:</p>
<ul>
<li>Coding sequences (CDS)</li>
<li>CDS + introns</li>
<li>CDS + introns + UTRs <!-- -   Introns --></li>
<li>5’ and 3’ UTRs</li>
<li>Peptide sequences <!-- -   Non-coding RNA genes --></li>
</ul>
<h4 id="genomic-regions">Genomic regions</h4>
<table>
<thead>
<tr class="header">
<th>Region</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a routerLink="/status/telomeres">Telomeres</a></td>
<td>Telomeric sequence</td>
</tr>
<tr class="even">
<td><a routerLink="/status/centromeres">Centromeres</a></td>
<td>Centromeric sequence and maps</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/mating-type-region">Mating Type Region</a></td>
<td>Links to genome browser for 972 h- (chromosome 2 coordinates) and h+ (mating region contig)</td>
</tr>
</tbody>
</table>
<h4 id="other-data">Other data</h4>
<ul>
<li>Additional information and historical data are available for <a routerLink="/downloads/intron-data">introns</a></li>
<li><a href="ftp://ftp.pombase.org/archive/Cosmid_sequences/">Cosmid sequences</a> and <a href="ftp://ftp.pombase.org/archive/Cosmid_assembly_data/">Chromosome tiling paths from genome sequence assembly</a></li>
<li>You can also browse the files available from the <a href="ftp://ftp.ensemblgenomes.org/pub/current/fungi/">Ensembl Genomes FTP site</a>.</li>
</ul>

  </div>
  <div *ngIf="pageName == 'go-annotations'">
<h3 id="go-annotations">GO annotations</h3>
<div style="color: red">
<p><strong>Warning:</strong> Files in the old FTP site within the EBI domain are no longer updated. If you have a link that contains <strong>ftp.ebi.ac.uk</strong>, please check these pages for a link using <strong>ftp.pombase.org</strong>. Please <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a> if you need help finding a file or directory.</p>
</div>
<p><em>S. pombe</em> GO annotations are available from the <a href="ftp://ftp.pombase.org/pombe/annotations/Gene_ontology/gene_association.pombase.gz">PomBase FTP site</a> or from the <a href="http://current.geneontology.org/annotations/pombase.gaf.gz">GO repository</a> (either link downloads a gzipped file). They include annotations made by manual literature curation, annotations inferred from keyword mappings based on curated descriptions, and annotations shared by the UniProt <a href="http://www.ebi.ac.uk/GOA">GOA</a> team. The file is tab-delimited text, so it can be opened in a spreadsheet application or parsed with a script; the <a href="http://geneontology.org/docs/go-annotation-file-gaf-format-21/">format</a> is described on the GO website.</p>
<p>Also see the <a href="ftp://ftp.pombase.org/pombe/annotations/Gene_ontology/GO_complexes/">list of protein complexes</a> in the FTP directory, which uses GO macromolecular complex terms and IDs.</p>
<p>Note that the contents of the file downloadable from PomBase may differ from files available elsewhere (e.g. see <a href="why-are-go-annotations-different-between-pombase-and-uniprot-goa">this FAQ</a>, and will not include annotations inferred by transitivity (see <a href="/faq/why-are-go-terms-missing-from-the-downloadable-annotation-file">this FAQ</a>).</p>
<p>Previous versions of the <em>S. pombe</em> GO annotation file can be retrieved from the <a href="ftp://ftp.pombase.org/pombe/annotations/Gene_ontology/OLD/">PomBase FTP archive</a>.</p>
<p>To cite the fission yeast GO data, please see <a routerLink="/about/citing-pombase">Citing PomBase</a>.</p>
<p>For FTP downloads, connect as “guest” if prompted; see <a href="faq/do-i-need-password-download-ftp-site">this FAQ</a> for more information.</p>

  </div>
  <div *ngIf="pageName == 'index'">
<h2 class="title block-title" id="downloads">Downloads</h2>
<div style="color: red">
<p><strong>Warning:</strong> Files in the old FTP site within the EBI domain are no longer updated. If you have a link that contains <strong>ftp.ebi.ac.uk</strong>, please check these pages for a link using <strong>ftp.pombase.org</strong>. Please <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a> if you need help finding a file or directory.</p>
</div>
<ul>
<li><a routerLink="/datasets">Datasets</a>: Genome sequence and features, annotations, etc.</li>
<li><a routerLink="/documents">Documents</a>: Conference materials, manuals, reports, etc.</li>
<li><a routerLink="/downloads/chado-database-dumps">Chado database dumps</a>: Dumps of curated data stored in the PomBase Chado database.</li>
</ul>
<p>For FTP downloads, connect as “guest” if prompted; see <a href="faq/do-i-need-password-download-ftp-site">this FAQ</a> for more information.</p>

  </div>
  <div *ngIf="pageName == 'intron-data'">
<h3 id="introns">Introns</h3>
<div style="color: red">
<p><strong>Warning:</strong> Files in the old FTP site within the EBI domain are no longer updated. If you have a link that contains <strong>ftp.ebi.ac.uk</strong>, please check these pages for a link using <strong>ftp.pombase.org</strong>. Please <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a> if you need help finding a file or directory.</p>
</div>
<p>Downloadable intron datasets are available in FASTA format from the <a href="ftp://ftp.pombase.org/pombe/genome_sequence_and_features/feature_sequences/">PomBase FTP site</a> (connect as “guest” if prompted; see <a href="faq/do-i-need-password-download-ftp-site">this FAQ</a> for more information).</p>
<p>Note that you can also use the Advanced Search to find all genes containing introns, as described in the <a routerLink="/gfaq/how-can-i-retrieve-intron-coordinates-or-sequences">FAQ on introns</a>.</p>
<h5 id="legacy-intron-data">Legacy intron data</h5>
<p>We also provide access to archived <a href="ftp://ftp.pombase.org/archive/Intron_Data/OLD/">intron data</a>. Please note that this set of intron data reflect the dataset at the time of publication of the <em>S. pombe</em> genome sequence, and does not include any new introns, or changes to introns, since then. The intron data archive includes:</p>
<ul>
<li>List of all <em>S. pombe</em> intron-containing genes</li>
<li>List of all <em>S. pombe</em> genes with exon numbers</li>
<li><em>S. pombe</em> intron sequences in FASTA format (donor sequence and final 35 bases of acceptor)</li>
<li>All introns with branch site aligned</li>
<li>Donor frequency</li>
<li>Branch frequency</li>
<li>A README file is available with additional descriptions of file contents</li>
</ul>

  </div>
  <div *ngIf="pageName == 'names-and-identifiers'">
<h3 id="pombase-name-and-identifier-mappings">PomBase name and identifier mappings</h3>
<div style="color: red">
<p><strong>Warning:</strong> Files in the old FTP site within the EBI domain are no longer updated. If you have a link that contains <strong>ftp.ebi.ac.uk</strong>, please check these pages for a link using <strong>ftp.pombase.org</strong>. Please <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a> if you need help finding a file or directory.</p>
</div>
<p>These files are available in the <a href="ftp://ftp.pombase.org/pombe/names_and_identifiers/">names and IDs FTP directory</a></p>
<h4 id="systematic-id-to-uniprot-accession-number">Systematic ID to UniProt accession number</h4>
<ul>
<li><a href="ftp://ftp.pombase.org/pombe/names_and_identifiers/PomBase2UniProt.tsv">PomBase2UniProt.tsv</a> tab-delimited file with the PomBase systematic identifier for each protein-coding gene mapped to the corresponding UniProt accession number</li>
</ul>
<h4 id="gene-names">Gene names</h4>
<ul>
<li><p><a href="ftp://ftp.pombase.org/pombe/names_and_identifiers/gene_IDs_names.tsv">gene_IDs_names.tsv</a> tab-delimited file of systematic ID, primary gene name (where assigned), and all synonyms for each gene</p></li>
<li><p><a href="ftp://ftp.pombase.org/pombe/names_and_identifiers/gene_IDs_names_products.tsv">gene_IDs_names_products.tsv</a> tab-delimited file of systematic ID, primary gene name (where assigned), chromosome, product description, UniProtKB accession, all synonyms, and product type (protein coding, ncRNA, etc.) for each gene</p></li>
</ul>
<h4 id="systematic-id-to-gene-product">Systematic ID to gene product</h4>
<p>Files include systematic name, primary name (where assigned), synonyms (where assigned), and gene product description</p>
<ul>
<li><a href="ftp://ftp.pombase.org/pombe/names_and_identifiers/sysID2product.tsv">sysID2product.tsv</a> protein-coding genes</li>
<li><a href="ftp://ftp.pombase.org/pombe/names_and_identifiers/sysID2product.rna.tsv">sysID2product.rna.tsv</a> non-coding RNA genes</li>
<li><a href="ftp://ftp.pombase.org/pombe/names_and_identifiers/pseudogeneIDs.tsv">pseudogeneIDs.tsv</a> pseudogenes</li>
</ul>
<p>Note: A tab-delimited file of systematic identifiers mapped to EC numbers was previously maintained, but has not been updated since March 2012. The most recent version of the <a href="ftp://ftp.pombase.org/pombe/names_and_identifiers/OLD/gp2EC.txt">gp2EC.txt</a> file is available in the FTP archive, but because it is out of date it may contain errors or omissions.</p>
<p>For FTP downloads, connect as “guest” if prompted; see <a href="faq/do-i-need-password-download-ftp-site">this FAQ</a> for more information.</p>

  </div>
  <div *ngIf="pageName == 'phenotype-annotations'">
<h3 id="phenotype-annotations">Phenotype annotations</h3>
<div style="color: red">
<p><strong>Warning:</strong> Files in the old FTP site within the EBI domain are no longer updated. If you have a link that contains <strong>ftp.ebi.ac.uk</strong>, please check these pages for a link using <strong>ftp.pombase.org</strong>. Please <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a> if you need help finding a file or directory.</p>
</div>
<p><a href="ftp://ftp.pombase.org/pombe/annotations/Phenotype_annotations/phenotype_annotations.pombase.phaf.gz">Phenotype annotations</a> (link downloads gzipped file from PomBase ftp) for alleles of <em>S. pombe</em> genes are manually curated from the literature using Fission Yeast Phenotype Ontology (FYPO) terms. Note: this file contains annotations for single allele phenotypes (single mutants) only.</p>
<p>The file is in a version of the PomBase phenotype data bulk annotation format (PHAF), detailed below. This format in nearly identical to the one that can be used to submit phenotype annotations to PomBase in bulk, as described on the <a routerLink="/documentation/phenotype-data-bulk-upload-format">Phenotype data bulk upload format</a> page, with the addition of the Database column. Note that, because Database is column 1 in the downloadable file, column numbers differ by 1 between the download and upload formats.</p>
<p>Propagating phenotype annotations: Note that the file contains only direct annotations to FYPO terms. It does not include annotations that can be inferred by propagating between terms within the ontology. To make full use of the FYPO annotation data, we strongly recommend also using the ontology structure and inferred annotations. Please contact the <a href="mailto:helpdesk@pombase.org">PomBase helpdesk</a> if you need assistance.</p>
<p>For FTP downloads, connect as “guest” if prompted; see <a href="faq/do-i-need-password-download-ftp-site">this FAQ</a> for more information.</p>
<h3 id="viability-summary">Viability summary</h3>
<p>A <a href="ftp://ftp.pombase.org/pombe/annotations/Phenotype_annotations/FYPOviability.tsv">set of “viability summary” data</a> as shown at the top of the FYPO table on each gene page, is available as a downloadable file. The file has two columns: the gene systematic ID and one of three values: “viable”, “inviable” or “condition-dependent”.</p>
<p>To cite the fission yeast phenotype data (complete or viability summary), please see <a routerLink="/about/citing-pombase">Citing PomBase</a>.</p>
<h3 id="phaf-download-format">PHAF download format</h3>
<p>A column marked “mandatory” will always have an entry; non-mandatory columns may be empty.</p>
<table>
<thead>
<tr class="header">
<th>Column</th>
<th>Contents</th>
<th>Example</th>
<th>Mandatory?</th>
<th>Multiple entries allowed?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1</td>
<td>Database</td>
<td>PomBase</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>2</td>
<td>Gene systematic ID</td>
<td>SPBC11B10.09</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>3</td>
<td>FYPO ID</td>
<td>FYPO:0000001</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>4</td>
<td>Allele description</td>
<td>G146D</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>5</td>
<td>Expression</td>
<td>overexpression</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>6</td>
<td>Parental strain</td>
<td>975 h+</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>7</td>
<td>Background strain name</td>
<td>SP286</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="even">
<td>8</td>
<td>Background genotype description</td>
<td>h+ ura4-D18 leu1-32 ade6-M210</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="odd">
<td>9</td>
<td>Gene name</td>
<td>cdc2</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="even">
<td>10</td>
<td>Allele name</td>
<td>cdc2-1w</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="odd">
<td>11</td>
<td>Allele synonym</td>
<td>wee2-1</td>
<td>No</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>12</td>
<td>Allele type</td>
<td>amino acid mutation</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>13</td>
<td>Evidence</td>
<td>ECO:0000336</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>14</td>
<td>Condition</td>
<td>at high temperature</td>
<td>Yes</td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>15</td>
<td>Penetrance</td>
<td>85%</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="even">
<td>16</td>
<td>Severity</td>
<td>medium</td>
<td>No</td>
<td>No</td>
</tr>
<tr class="odd">
<td>17</td>
<td>Extension</td>
<td>assayed_using(PomBase:SPBC582.03)</td>
<td>No</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>18</td>
<td>Reference</td>
<td>PMID:23697806</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="odd">
<td>19</td>
<td>taxon</td>
<td>taxon:4896</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr class="even">
<td>20</td>
<td>Date</td>
<td><span class="no-break">2012-01-01</span></td>
<td>Yes</td>
<td>No</td>
</tr>
</tbody>
</table>
<p><strong>Notes:</strong></p>
<ol type="1">
<li>The database that produced the file – included to facilitate data sharing with other databases. Obviously always PomBase for this file.</li>
<li>The systematic ID for each gene.</li>
<li>To use the FYPO IDs, see the <a routerLink="/browse-curation/fission-yeast-phenotype-ontology">FYPO summary page</a> and the <a routerLink="/faq/how-can-i-browse-phenotype-ontology-fypo">FAQ on browsing FYPO</a>.</li>
<li>The allele description specifically describes the change; see the <a routerLink="/documentation/phenotype-data-bulk-upload-format">Phenotype data bulk upload format</a> for details on allele types and descriptions.</li>
<li>The Expression column contains one of these values: ‘overexpression’, ‘knockdown’, ‘endogenous’, ‘null’, ‘not specified’. Deletions always have ‘null’ expression.</li>
<li>The Parental strain column is for the parental strain designation, such as 972 h-, 975 h+, etc. This column must be filled in, but “unknown” is allowed.</li>
<li>The Background strain name column is used for a lab’s in-house name/ID/designation for the background strain (i.e. the derivative of the parental strain that has selectable marker alleles etc.). The description in the Genotype column should match this background strain.</li>
<li>The Background genotype description column is for alleles in the background, such as selectable markers; these details are optional. To avoid redundancy, it does not repeat the allele of interest (from column 4 or 10) in the genotype column.</li>
<li>Gene names are optional for upload, but are included where available in the download.</li>
<li>Allele names are optional for upload, but are included where available in the download. Column 10 lists the preferred allele name, and any alternative names are in column 11.</li>
<li>See note 10 above. Multiple entries are separated with pipes (|).</li>
<li>Allowed allele types, example descriptions, etc. are shown in the table at the bottom of the <a routerLink="/documentation/phenotype-data-bulk-upload-format">Phenotype data bulk upload format</a> page.</li>
<li>For the Evidence column, we use a small selection from the <a href="http://www.evidenceontology.org/">Evidence Ontology</a> (ECO). Please contact the <a href="mailto:helpdesk@pombase.org">Helpdesk</a> if you need assistance using these IDs</li>
<li>Similarly, Conditions use a small ontology maintained in-house by PomBase curators, and we can help you interpret the IDs. Multiple entries are shown in cases where more than one condition detail applies at the same time (e.g. high temperature, minimal medium), and are separated with commas (,).</li>
<li>Penetrance describes the proportion of a population that shows a cell-level phenotype. Penetrance data are represented as percents or entries from the in-house FYPO_EXT ontology (FYPO_EXT:0000001 = high; FYPO_EXT:0000002 = medium; FYPO_EXT:0000003 = low; FYPO_EXT:0000004 = full).</li>
<li>Severity (formerly designated “expressivity”) uses the FYPO_EXT ontology described in note 15.</li>
<li>The Extension column is used to record when a mutation in one gene affects another gene or its product. For example, if a mutation in gene A decreases its ability to phosphorylate protein B, the A allele is annotated to the phenotype “decreased protein kinase activity” with the ID for gene B in an extension. In the downloadable file, an annotation can have multiple comma-separated extensions if they combine to form a “compound” extension (e.g. two or more genes assayed together).</li>
<li>The Reference column has the publication’s PubMed ID (PMID).</li>
<li>The taxon will usually be 4896 (the NCBI taxon ID for <em>Schizosaccharomyces pombe</em>), although an NCBI taxon ID for a specific <em>S. pombe</em> strain would be allowed.</li>
<li>The date is the date on which the annotations are created. Format: YYYY-MM-DD</li>
</ol>

  </div>
  <div *ngIf="pageName == 'protein-datasets'">
<h3 id="protein-datasets">Protein datasets</h3>
<div style="color: red">
<p><strong>Warning:</strong> Files in the old FTP site within the EBI domain are no longer updated. If you have a link that contains <strong>ftp.ebi.ac.uk</strong>, please check these pages for a link using <strong>ftp.pombase.org</strong>. Please <a href="mailto:helpdesk@pombase.org">contact the PomBase curators</a> if you need help finding a file or directory.</p>
</div>
<p>The full <a href="ftp://ftp.pombase.org/pombe/genome_sequence_and_features/feature_sequences/peptide.fa.gz">protein dataset</a> is available in FASTA format (link downloads gzipped file from PomBase ftp).</p>
<p>The <a href="ftp://ftp.pombase.org/pombe/Protein_data/">protein data FTP directory</a> contains assorted data (see the README for file formats):</p>
<ul>
<li><a href="ftp://ftp.pombase.org/pombe/Protein_data/PeptideStats.tsv">PeptideStats.tsv</a> - Predicted molecular weight (kDa), predicted pI, charge, length (residues), codon adaptation index (CAI)</li>
<li><a href="ftp://ftp.pombase.org/pombe/Protein_data/Protein_Features.tsv">Protein_Features.tsv</a> - Protein features such as domains and family assignments</li>
<li><a href="ftp://ftp.pombase.org/pombe/Protein_data/aa_composition.tsv">aa_composition.tsv</a> - Amino acid composition</li>
<li><a href="ftp://ftp.pombase.org/pombe/Protein_data/transmembrane_domain_coords_and_seqs.tsv">transmembrane_domain_coords_and_seqs.tsv</a> - Sequences and coordinates for transmembrane domains</li>
</ul>
<p>Also see the <a href="ftp://ftp.pombase.org/pombe/annotations/modifications/">protein modification annotations</a> directory.</p>
<p>For FTP downloads, connect as “guest” if prompted; see <a href="faq/do-i-need-password-download-ftp-site">this FAQ</a> for more information.</p>

  </div>
  </div>
</div>
<div *ngIf="section == 'faq'">
<app-page-contents-menu title="FAQ" titleRoute="/faq">

<div class="left-menu-part left-menu-item"><a routerLink="/faq/data-submission-and-formats">Data submission and formats</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/faq/finding-data">Finding data</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/faq/gene-page">Gene page</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/faq/genome-browser">Genome browser</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/faq/genome-statistics-and-lists">Genome statistics and lists</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/faq/orthology">Orthology</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/faq/tools-and-resources">Tools and resources</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/faq/using-ontologies">Using ontologies</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/faq/all-frequently-asked-questions">All Frequently Asked Questions</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'all-frequently-asked-questions'">
<h2 id="all-frequently-asked-questions">All Frequently Asked Questions</h2>

  </div>
  <div *ngIf="pageName == 'data-submission-and-formats'">
<h2 id="data-submission-and-formats">Data submission and formats</h2>

  </div>
  <div *ngIf="pageName == 'finding-data'">
<h2 id="finding-data">Finding data</h2>

  </div>
  <div *ngIf="pageName == 'gene-page'">
<h2 id="gene-page">Gene page</h2>

  </div>
  <div *ngIf="pageName == 'genome-browser'">
<h2 id="genome-browser">Genome browser</h2>

  </div>
  <div *ngIf="pageName == 'genome-statistics-and-lists'">
<h2 id="genome-statistics-and-lists">Genome statistics and lists</h2>

  </div>
  <div *ngIf="pageName == 'index'">
<h1 id="frequently-asked-questions">Frequently asked questions</h1>
<p>Welcome to the PomBase FAQ. Choose a link in the left-hand menu to see the questions and answers in that category.</p>
<p>If you can’t find the information you need here or in the PomBase documentation, please contact the helpdesk.</p>

  </div>
  <div *ngIf="pageName == 'orthology'">
<h2 id="orthology">Orthology</h2>

  </div>
  <div *ngIf="pageName == 'tools-and-resources'">
<h2 id="tools-and-resources">Tools and resources</h2>

  </div>
  <div *ngIf="pageName == 'using-ontologies'">
<h2 id="using-ontologies">Using ontologies</h2>

  </div>
<div (click)="faq_navigate($event, '/faq/can-i-access-pombase-via-api', 'Can I access PomBase via an API?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'tools-and-resources' || pageName == 'can-i-access-pombase-via-api' || pageName == 'all-faqs'">
<h3 id="can-i-access-pombase-via-api">Can I access PomBase via an API?</h3>
<!-- pombase_categories: Tools and resources -->
<p>PomBase does not have an API at present.</p>
<p>The Ensembl API can be used to query <em>S. pombe</em> data. Note, however, that EG is updated much less frequently than PomBase, so EG data will rarely be as up-to-date as the PomBase web site. Documentation is available:</p>
<ol type="1">
<li><a href="http://ensemblgenomes.org/info/access/eg_api">Ensembl Genomes Perl API installation and basic use instructions</a> (with links to additional Ensembl API documentation)</li>
<li><a href="http://www.ensembl.org/info/docs/api/core/core_tutorial.html">Tutorial for using the API with the core database</a> - Includes examples about connecting to the database, retrieving chromosomes, genes, transcript and translations along with the corresponding xrefs.</li>
</ol>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-access-pombase-via-sql', 'Can I access PomBase via SQL?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'can-i-access-pombase-via-sql' || pageName == 'all-faqs'">
<h3 id="can-i-access-pombase-via-sql">Can I access PomBase via SQL?</h3>
<!-- pombase_categories: Finding data,Tools and resources -->
<p>PomBase does not have a publicly accessible SQL query server, but <a routerLink="/downloads/chado-database-dumps">Chado database dumps</a> (PostgresQL) are available to download and query locally.</p>
<p>The <a href="http://ensemblgenomes.org/info/access/mysql">Ensembl Genomes (EG) MySQL database</a> server does provide access to query <em>S. pombe</em> data. Note, however, that EG is updated much less frequently than PomBase, so EG data will rarely be as up-to-date as the PomBase web site.</p>
<p>MySQL dumps of EG data, including <em>Schizosaccharomyces</em> species, are available from <a href="ftp://ftp.ensemblgenomes.org/pub/current/fungi/mysql/">EG’s FTP site</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-convert-file-gff-gtf', 'Can I convert a file from GFF to GTF?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'tools-and-resources' || pageName == 'can-i-convert-file-gff-gtf' || pageName == 'all-faqs'">
<h3 id="can-i-convert-file-gff-gtf">Can I convert a file from GFF to GTF?</h3>
<!-- pombase_categories: Tools and resources -->
<p>PomBase does not offer a GFF-to-GTF converter. There is a perl script on SEQanswers, which uses the module Bio::Tools::GFF from the BioPerl library, available from <a href="http://seqanswers.com/forums/showpost.php?p=22529&amp;postcount=4" class="uri">http://seqanswers.com/forums/showpost.php?p=22529&amp;postcount=4</a></p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-convert-file-gtf-gff', 'Can I convert a file from GTF to GFF3?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'tools-and-resources' || pageName == 'can-i-convert-file-gtf-gff' || pageName == 'all-faqs'">
<h3 id="can-i-convert-file-gtf-gff">Can I convert a file from GTF to GFF3?</h3>
<!-- pombase_categories: Tools and resources -->
<p>PomBase does not offer a converter. The</p>
<p><a href="http://www.sequenceontology.org/">Sequence Ontology</a> project offers a conversion tool available for download or usage via web form from their <a href="https://github.com/The-Sequence-Ontology/GTF-TO-GFF3-Converter">GitHub site</a></p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-convert-ids-other-databases-or-pombase-ids', 'Can I convert IDs from other databases to or from PomBase IDs?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'tools-and-resources' || pageName == 'can-i-convert-ids-other-databases-or-pombase-ids' || pageName == 'all-faqs'">
<h3 id="can-i-convert-ids-other-databases-or-pombase-ids">Can I convert IDs from other databases to or from PomBase IDs?</h3>
<!-- pombase_categories: Tools and resources -->
<p>PomBase does not have its own tool for ID conversion. We suggest you try the EBI’s PICR web service (<a href="http://www.ebi.ac.uk/Tools/picr/" class="uri">http://www.ebi.ac.uk/Tools/picr/</a>), which can convert between UniProtKB, RefSeq, Ensembl Genomes (including <em>S. pombe</em> systematic IDs), and many other common database IDs.</p>
<p>For UniProt IDs, we provide a static mapping file of PomBase systematic IDs and UniProtKB accessions, available on the <a routerLink="/downloads/names-and-identifiers">Data Mapping</a> page and by FTP from <a href="ftp://ftp.pombase.org/pombe/names_and_identifiers/PomBase2UniProt.tsv" class="uri">ftp://ftp.pombase.org/pombe/names_and_identifiers/PomBase2UniProt.tsv</a>.</p>
<p>For other <em>Schizosaccharomyces</em> species (<em>S. japonicus</em>, <em>S. octosporus</em>, <em>S. cryophilus</em>), the PICR service converts between UniProtKB accessions and the identifiers used in Rhind <em>et al.</em> Comparative functional genomics of the fission yeasts (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21511999">PMID:21511999</a>), but with one caveat: the IDs used by PICR, UniProtKB, and QuickGO contain underscores, whereas those in Rhind <em>et al.</em> do not (e.g. SJAG_00455 versus SJAG00455). The IDs with underscores are correct.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-do-enrichment-analysis-using-phenotypes', 'Can I do an enrichment analysis using phenotypes?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'can-i-do-enrichment-analysis-using-phenotypes' || pageName == 'all-faqs'">
<h3 id="can-i-do-enrichment-analysis-using-phenotypes">Can I do an enrichment analysis using phenotypes?</h3>
<!-- pombase_categories: Finding data,Tools and resources,Using ontologies -->
<p>A small number of enrichment tools use phenotype data. See the <a routerLink="/faq/how-can-i-find-significant-shared-fypo-annotations-genes-list">FAQ on FYPO term enrichment</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-do-go-term-enrichment-other-schizosaccharomyces-species-e.g.-s.-japonicus', 'Can I do GO term enrichment for other Schizosaccharomyces species (e.g. S. japonicus)?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'can-i-do-go-term-enrichment-other-schizosaccharomyces-species-e.g.-s.-japonicus' || pageName == 'all-faqs'">
<h3 id="can-i-do-go-term-enrichment-other-schizosaccharomyces-species-e.g.-s.-japonicus">Can I do GO term enrichment for other <em>Schizosaccharomyces</em> species (e.g. <em>S. japonicus</em>)?</h3>
<!-- pombase_categories: Finding data,Tools and resources,Using ontologies -->
<p>There are two possible approaches:</p>
<ol type="1">
<li>Retrieve a set of GO annotations in GAF format for <em>S. japonicus</em>, <em>S. octosporus</em> or <em>S. cryophilus</em>, as described in the <a routerLink="/faq/how-can-i-find-go-annotations-other-schizosaccharomyces-species-e.g.-s.-japonicus">relevant FAQ</a>. Use the GO annotation dataset and your gene list for enrichment.</li>
</ol>
<p>OR</p>
<ol start="2" type="1">
<li>In your gene list of interest, substitute the <em>Schizosaccharomyces</em> species gene IDs with the IDs of orthologous <em>S. pombe</em> genes. For ortholog IDs, see the <a routerLink="/faq/how-can-i-find-orthologs-between-s.-pombe-and-other-schizosaccharomyces-species">FAQ on <em>Schizosaccharomyces</em> orthologs</a>, and use the indicated table from Rhind <em>et al.</em> Comparative functional genomics of the fission yeasts (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21511999">PMID:21511999</a>).</li>
</ol>
<p>In either case, you can then proceed as described in the <a routerLink="/faq/how-can-i-find-significant-shared-go-annotations-genes-list">FAQ on <em>S. pombe</em> GO enrichment</a>. For the first option, use the Princeton GO Term Finder or another enrichment tool that allows you to use your own GAF, and include the GO Slim analysis using GO Term Mapper as recommended in the <a routerLink="/faq/how-can-i-find-significant-shared-go-annotations-genes-list">GO enrichment FAQ</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-download-all-s.-pombe-phenotype-data', 'Can I download all S. pombe phenotype data?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'can-i-download-all-s.-pombe-phenotype-data' || pageName == 'all-faqs'">
<h3 id="can-i-download-all-s.-pombe-phenotype-data">Can I download all <em>S. pombe</em> phenotype data?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>Yes, the <a routerLink="/downloads/phenotype-annotations">Phenotype annotations</a> page offers two options, a complete phenotype annotation file and a “viability summary” for deletion mutants. At present, the full file contains all manually curated single mutant phenotypes, and is in the same format as PomBase uses for bulk phenotype data submissions (see the <a routerLink="/faq/what-file-formats-can-i-use-submit-high-throughput-data">file formats FAQ</a>). Further information on the viability summary is available in the <a routerLink="/faq/can-i-get-list-essential-pombe-genes">essential genes FAQ</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-download-interaction-data-used-by-esyn', 'Can I download the interaction data used by esyN?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'genome-statistics-and-lists' || pageName == 'can-i-download-interaction-data-used-by-esyn' || pageName == 'all-faqs'">
<h3 id="can-i-download-interaction-data-used-by-esyn">Can I download the interaction data used by esyN?</h3>
<!-- pombase_categories: Finding data,Genome statistics and lists -->
<p>The <em>S. pombe</em> networks in <a href="http://www.esyn.org/">esyN</a> use the PomBase <a routerLink="/documentation/high-confidence-physical-interaction-network">High Confidence Physical Interaction Network</a> (HCPIN) data. The data can be downloaded in tab-delimited format from the <a routerLink="/datasets">Download Datasets</a> page, which links to an FTP directory with two files. One contains physical interaction data, and the other contains GO substrate data. Each file lists two gene systematic IDs and a reference identifier (usually a PubMed ID).</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-download-sequence-whole-chromosome', 'Can I download the sequence for a whole chromosome?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'can-i-download-sequence-whole-chromosome' || pageName == 'all-faqs'">
<h3 id="can-i-download-sequence-whole-chromosome">Can I download the sequence for a whole chromosome?</h3>
<!-- pombase_categories: Finding data -->
<p>Yes, sequence and feature annotations data files are available for each chromosome on the <a routerLink="/downloads/genome-datasets">Genome datasets</a> page (FTP download).</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-download-sequences-many-genes-once-including-flanking-regions', 'Can I download sequences for many genes at once, including flanking regions? ')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'can-i-download-sequences-many-genes-once-including-flanking-regions' || pageName == 'all-faqs'">
<h3 id="can-i-download-sequences-many-genes-once-including-flanking-regions">Can I download sequences for many genes at once, including flanking regions?</h3>
<!-- pombase_categories: Finding data -->
<p>Yes; first use the <a routerLink="/query">advanced search</a> to find the set of genes you want (see the <a routerLink="/documentation/advanced-search">documentation</a> as needed).</p>
<p>Click on a count in the query history to see the <a routerLink="/documentation/advanced-search">results</a>, with a button for “Download” options. In the popup, click on the Sequence tab, select Nucleotide, then tick boxes to add any of: 5’ UTR, introns, 3’ UTR. To include flanking sequences, type or use the arrow buttons to set the desired length of upstream and downstream sequence.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-export-data-sequence-region-genome-browser', 'Can I export data for a sequence region from the genome browser?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-browser' || pageName == 'finding-data' || pageName == 'can-i-export-data-sequence-region-genome-browser' || pageName == 'all-faqs'">
<h3 id="can-i-export-data-sequence-region-genome-browser">Can I export data for a sequence region from the genome browser?</h3>
<!-- pombase_categories: Genome browser,Finding data -->
<p>To download track data from the <a href="https://www.pombase.org/jbrowse/">genome browser</a> (or follow a link from a gene page), first, make sure the you have enabled the track(s) of interest (see the <a href="/faq/how-can-i-show-or-hide-tracks-genome-browser">FAQ on showing tracks</a> if necessary).</p>
<p>Next select a region by <a href="/faq/how-can-i-display-sequence-region-using-sequence-coordinates-genome-browser">entering coordinates</a> or by highlighting it. For any data track, click on the end of the track label to show the dropdown menu, and choose “Save track data”. In the popup that appears, choose a data format, edit the file name if desired, and click “Save”.</p>
<p>Most track data are available in GFF3 format; other options will depend on the data type (e.g. BED, Wiggle, etc.).</p>
<p>See the <a routerLink="/documentation/JBrowse_quick_start">PomBase JBrowse guide</a> for more information.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-find-all-unconserved-orphan-genes-fission-yeast', 'Can I find all of the unconserved (orphan) genes in fission yeast?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-statistics-and-lists' || pageName == 'finding-data' || pageName == 'can-i-find-all-unconserved-orphan-genes-fission-yeast' || pageName == 'all-faqs'">
<h3 id="can-i-find-all-unconserved-orphan-genes-fission-yeast">Can I find all of the unconserved (orphan) genes in fission yeast?</h3>
<!-- pombase_categories: Genome statistics and lists,Finding data -->
<p>Orphan genes are generally defined as genes without homologs in other organisms. In PomBase, protein-coding genes conserved in the <em>Schizosaccharomyces</em> genus are distinguished from genes conserved only in <em>S. pombe</em>.</p>
<p>To retrieve either set of genes, use the “Taxonomic conservation” query in the <a routerLink="/query">Advanced Search</a>. Choose “Schizosaccharomyces specific” for genes found in more than one <em>Schizosaccharomyces</em> species, or “Schizosaccharomyces pombe specific” for genes found only in <em>S. pombe.</em> See the <a routerLink="/documentation/advanced-search">Advanced Search documentation</a> for help with performing searches.</p>
<p>Historical note: Prior to August 2014, PomBase and its predecessor GeneDB referred to single-copy genes conserved within, but not outside, the <em>Schizosaccharomyces</em> genus as “sequence orphans”. See the <a routerLink="/status/gene-characterisation-statistics-history">Gene characterisation statistics history</a> page for more details (note that the gene characterisation classifications reflect whether a gene has been studied experimentally as well as the extent of its conservation).</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-find-promoters-pombase', 'Can I find promoters in PomBase?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'can-i-find-promoters-pombase' || pageName == 'all-faqs'">
<h3 id="can-i-find-promoters-pombase">Can I find promoters in PomBase?</h3>
<!-- pombase_categories: Finding data -->
<p>There are very few manually curated promoters in PomBase; they are included in the forward and reverse strand feature tracks, which are enabled by default. To search the manually curated promoters, we suggest that you use Artemis. Follow the instructions in <a routerLink="/faq/there-equivalent-artemis-java-applet-pombase">this FAQ</a>, and search for features with “Key” = “promoter”.</p>
<p>Computationally identified matches to the <a href="https://www.pombase.org/browse-curation/dna-binding-sites">DNA binding sites</a></p>
<p>found in <em>S. pombe</em> are avaliable as a data track in PomBase JBrowse. To launch JBrowse with the track loaded, use <a href="https://www.pombase.org/jbrowse/?loc=III%3A725762..736349&amp;tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CConsensus%20transcription%20factor%20DNA%20binding%20motifs&amp;highlight=">this link</a>.</p>
<p>The genome browser also includes transcription start site datasets, such as the CAGE-defined <a href="https://www.pombase.org/jbrowse/?loc=II%3A27586..65254&amp;tracks=DNA%20sequence%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CConsensus%20transcription%20start%20sites%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20heat%20shock%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20heat%20shock%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20nitrogen%20starvation%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20nitrogen%20starvation%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20oxidative%20stress%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20oxidative%20stress%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20YES%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20YES%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20glucose%20minimal%20medium%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20glucose%20minimal%20medium%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)&amp;highlight=">transcription start sites</a> across 5 different conditions from <a href="https://www.biorxiv.org/content/early/2018/03/13/281642">Thodberg <em>et al.</em> (2018)</a>.</p>
<p>We will also soon add the dataset from Li <em>et al.</em> (2015) Genome-wide analysis of core promoter structures in <em>Schizosaccharomyces pombe</em> with DeepCAGE. RNA Biol. 12:525-37 (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25747261">PMID:25747261</a>), which was available in the previous version of the PomBase genome browser.</p>
<p>We will also add any new promoter data sets that are submitted to us.</p>
<!-- edit & restore
The PomBase genome browser includes a data track of core promoter
locations from Li *et al.* (2015) Genome-wide analysis of core promoter
structures in *Schizosaccharomyces pombe* with DeepCAGE. RNA Biol.
12:525-37 (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25747261">PMID:25747261</a>).
-->

</div>
<div (click)="faq_navigate($event, '/faq/can-i-find-replication-origins-pombase', 'Can I find replication origins in PomBase?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'can-i-find-replication-origins-pombase' || pageName == 'all-faqs'">
<h3 id="can-i-find-replication-origins-pombase">Can I find replication origins in PomBase?</h3>
<!-- pombase_categories: Finding data -->
<p>Replication origins cannot yet be viewed or searched as features on PomBase pages, but they can be displayed on tracks in the genome browser. Select one or more of the tracks with data type “Replication Origins” (see the <a routerLink="/faq/how-can-i-show-or-hide-tracks-genome-browser">browser track FAQ</a> if necessary).</p>
<p>You can also use the data collated by Conrad Nieduszynski in <em>S. pombe</em> OriDB:</p>
<ul>
<li>Home page: <a href="http://pombe.oridb.org/index.php" class="uri">http://pombe.oridb.org/index.php</a></li>
<li>Search: <a href="http://pombe.oridb.org/search.php?chr=all&amp;confirmed=true&amp;likely=true&amp;dubious=true&amp;name=" class="uri">http://pombe.oridb.org/search.php?chr=all&amp;confirmed=true&amp;likely=true&amp;dubious=true&amp;name=</a></li>
</ul>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-find-unmapped-genes-pombase', 'Can I find unmapped genes in PomBase?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'genome-statistics-and-lists' || pageName == 'can-i-find-unmapped-genes-pombase' || pageName == 'all-faqs'">
<h3 id="can-i-find-unmapped-genes-pombase">Can I find unmapped genes in PomBase?</h3>
<!-- pombase_categories: Finding data,Genome statistics and lists -->
<p>PomBase maintains a <a routerLink="/status/unmapped_genes">list of genetic loci that have not been cloned or mapped</a>. The page lists the names published for the loci and the publications that use the names. In a few cases, unmapped loci have been shown to be allelic with each other.</p>
<p>Note that some locus names coincidentally match the standard names of cloned genes. In these cases PomBase curators have determined that the unmapped locus is not the same as the named gene.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-generate-comprehensive-restriction-enzyme-map-genome-pombase', 'Can I generate a comprehensive restriction enzyme map of the genome in PomBase?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'tools-and-resources' || pageName == 'can-i-generate-comprehensive-restriction-enzyme-map-genome-pombase' || pageName == 'all-faqs'">
<h3 id="can-i-generate-comprehensive-restriction-enzyme-map-genome-pombase">Can I generate a comprehensive restriction enzyme map of the genome in PomBase?</h3>
<!-- pombase_categories: Tools and resources -->
<p>No, but this can be done within Artemis.</p>
<p>Install Artemis (available from <a href="http://www.sanger.ac.uk/science/tools/artemis" class="uri">http://www.sanger.ac.uk/science/tools/artemis</a>; for <em>S. pombe</em> we recommend using <a href="ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/v16/">Artemis Version 16</a>. g You can then read in the EMBL format chromosome contig files of sequence and annotation (available from the <a routerLink="/downloads/genome-datasets">Genome datasets</a> page). To generate a restriction map:</p>
<ol type="1">
<li>Create a new entry using the “Create” menu item “New Entry”</li>
<li>Toggle off the main annotation by un-checking the chromsome contig file (this will make your new file “no name” the active entry).</li>
<li>Save your new file with your preferred name.</li>
<li>Use the Create menu option “Mark From Pattern” to create features for any restriction patterns of interest and save them into your file.</li>
<li>You can add “color” labels to distinguish the different restriction sites. See the <a routerLink="/faq/there-equivalent-artemis-java-applet-pombase">Artemis FAQ</a> and the <a href="ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/artemis.pdf">Artemis manual</a> (pdf; Sanger site) for additional information.</li>
</ol>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-get-cdna-sequences-s.-pombe', 'Can I get cDNA sequences for S. pombe?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'can-i-get-cdna-sequences-s.-pombe' || pageName == 'all-faqs'">
<h3 id="can-i-get-cdna-sequences-s.-pombe">Can I get cDNA sequences for <em>S. pombe</em>?</h3>
<!-- pombase_categories: Finding data -->
<p>A file of cDNA sequences in FASTA format is available on the <a routerLink="/downloads/genome-datasets">Genome datasets</a> page, under the “Feature sequences” heading.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-get-file-specific-set-genome-features', 'Can I get a file with a specific set of genome features?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'can-i-get-file-specific-set-genome-features' || pageName == 'all-faqs'">
<h3 id="can-i-get-file-specific-set-genome-features">Can I get a file with a specific set of genome features?</h3>
<!-- pombase_categories: Finding data,Tools and resources -->
<p>The genomic GFF3 file available on the <a routerLink="/downloads/genome-datasets">Genome datasets</a> page includes all gene features, but not LTRs or other repeats. The files in EMBL and GenBank formats contain all annotated sequence features.</p>
<p>Another option for extracting all annotated features (or if you need to specify which feature types to include) is to use the Ensembl API. See the FAQ “<a routerLink="/faq/can-i-access-pombase-via-api">Can I access PomBase via an API?</a>” for more information on using the API.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-get-list-essential-pombe-genes', 'Can I get a list of essential pombe genes?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'genome-statistics-and-lists' || pageName == 'using-ontologies' || pageName == 'can-i-get-list-essential-pombe-genes' || pageName == 'all-faqs'">
<h3 id="can-i-get-list-essential-pombe-genes">Can I get a list of essential pombe genes?</h3>
<!-- pombase_categories: Finding data,Genome statistics and lists,Using ontologies -->
<p>If an essential gene is deleted, the cell cannot survive under normal laboratory conditions. A search for deletion alleles annotated to the Fission Yeast Phenotype Ontology term “inviable vegetative cell population” (<a routerLink="/term/FYPO:0002061">FYPO:0002061</a>) would therefore identify essential fission yeast genes. Similarly, deletion alleles annotated to “viable vegetative cell population” (<a routerLink="/term/FYPO:0002177">FYPO:0002060</a>) represent non-essential genes.</p>
<p>Note that viability/inviability annotations are fairly complete for protein-coding genes, but very few non-coding RNA genes have been tested.</p>
<p><strong>Downloadable summary</strong></p>
<p>A <a href="ftp://ftp.pombase.org/pombe/annotations/Phenotype_annotations/FYPOviability.tsv">set of “viability summary” data</a>, as shown at the top of the FYPO table on each gene page, is available as a downloadable file. The file has two columns: the gene systematic ID and one of three values: “viable”, “inviable” or “condition-dependent”.</p>
<p><strong>Querying</strong></p>
<ul>
<li><p>To find genes annotated to “inviable vegetative cell population”, You can also set up the query manually: select the “phenotype” query and type or paste the ID, FYPO:0002061. Select the “Null” option for “Expresion level” and submit the query. The results include all genes that showed inviable phenotypes in the HTP deletion project as well some manually annotated genes. Similarly, there is a query for genes annotated to “viable vegetative cell population” (FYPO:0002060).</p></li>
<li><p>For some deletion mutants, viability depends on experimental conditions, which cannot yet be queried in PomBase. These genes are annotated to both viable (FYPO:0002060) and inviable (FYPO:0002061) at once. To find them, do both searches above, then click the boxes beside both in the query history table and click the “Intersect” button (equivalent to the “AND” operator).</p></li>
<li><p>See the <a routerLink="/documentation/advanced-search">advanced search documentation</a> for more information on performing the searches described here.</p></li>
</ul>
<p><strong>A brief note about FYPO terms</strong></p>
<p>At present, there are very few null mutants annotated as inviable in life cycle stages other than vegetative growth, and “inviable vegetative cell population” best fits the most common usage of “essential gene”. If you do want to include other stages (such as “inviable spore”), you can use the very generic term “inviable cell population” (FYPO:0002059) or “viable cell population” (FYPO:0002058) in your query. All of the caveats about alleles and conditions still apply.</p>
<p><strong>Query links</strong></p>
<ul>
<li><app-query-link [goToResults]="true" [linkText]="'Null alleles annotated to &quot;inviable vegetative cell population&quot; (FYPO:0002061)'" [predefinedQueryId]="'null_inviable'"></app-query-link></li>
<li><app-query-link [goToResults]="true" [linkText]="'Null alleles annotated to &quot;viable vegetative cell population&quot; (FYPO:0002060)'" [predefinedQueryId]="'null_viable'"></app-query-link></li>
<li><app-query-link [goToResults]="true" [linkText]="'Null alleles annotated to both FYPO:0002061 and FYPO:0002060'" [predefinedQueryId]="'null_viable_and_inviable'"></app-query-link></li>
</ul>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-get-list-systematic-ids-primary-names-synonyms-and-gene-products-s.-pombe', 'Can I get a list of systematic IDs, primary names, synonyms and gene products in S. pombe?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-statistics-and-lists' || pageName == 'can-i-get-list-systematic-ids-primary-names-synonyms-and-gene-products-s.-pombe' || pageName == 'all-faqs'">
<h3 id="can-i-get-list-systematic-ids-primary-names-synonyms-and-gene-products-s.-pombe">Can I get a list of systematic IDs, primary names, synonyms and gene products in <em>S. pombe</em>?</h3>
<!-- pombase_categories: Genome statistics and lists -->
<p>Yes: From the <a routerLink="/downloads/names-and-identifiers">PomBase name and identifier mappings</a> page, retrieve this file:</p>
<p>ftp://ftp.pombase.org/pombe/names_and_identifiers/sysID2product.tsv</p>
<p>Also consider other files listed on the page.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-obtain-dump-all-curated-data-pombase', 'Can I obtain a dump of all curated data in PomBase?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'can-i-obtain-dump-all-curated-data-pombase' || pageName == 'all-faqs'">
<h3 id="can-i-obtain-dump-all-curated-data-pombase">Can I obtain a dump of all curated data in PomBase?</h3>
<!-- pombase_categories: Finding data,Tools and resources -->
<p>Data that appear on gene pages – sequence feature annotations, ontology annotations, etc. – are stored in a database that uses the <a href="http://gmod.org/wiki/Chado">Chado</a> schema. Snapshots of the PomBase Chado database are available via the <a routerLink="/downloads">Downloads</a> page.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-provide-list-genes-search-on', 'Can I provide a list of genes to search on?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'can-i-provide-list-genes-search-on' || pageName == 'all-faqs'">
<h3 id="can-i-provide-list-genes-search-on">Can I provide a list of genes to search on?</h3>
<!-- pombase_categories: Finding data -->
<p>Yes, you can type or paste systematic IDs into the “Gene filter” query in the <a routerLink="/query">advanced search</a>. IDs can be separated by spaces, commas, or newlines. Click “Lookup” to retrieve the results. At present you can use the list in the query history to combine with other queries, or download a fixed set of details (systematic ID, gene name (if any), feature type, coordinates, and strand). We plan to offer more flexible options in the near future, and to allow you to upload a file with your gene list.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-retrieve-annotations-metabolic-databases-such-as-ec-metacyc-reactome-or-rhea-s.-pombe-genes', 'Can I retrieve annotations to metabolic databases such as EC, MetaCyc, Reactome, or Rhea for S. pombe genes?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'can-i-retrieve-annotations-metabolic-databases-such-as-ec-metacyc-reactome-or-rhea-s.-pombe-genes' || pageName == 'all-faqs'">
<h3 id="can-i-retrieve-annotations-metabolic-databases-such-as-ec-metacyc-reactome-or-rhea-s.-pombe-genes">Can I retrieve annotations to metabolic databases such as EC, MetaCyc, Reactome, or Rhea for <em>S. pombe</em> genes?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>The best way to find metabolism-related annotations for <em>S. pombe</em> genes is to use the GO annotation data available from PomBase in combination with mappings between GO terms and entries in the various metabolism-oriented databases.</p>
<p>For example, many GO molecular function (MF) terms representing enzymatic activities are mapped to the corresponding <a href="https://www.qmul.ac.uk/sbcs/iubmb/enzyme/">Enzyme Commission</a> (EC) number for the reaction, and some are also mapped to entries from <a href="http://www.genome.jp/kegg/">KEGG</a> or from the <a href="http://www.rhea-db.org/home">Rhea</a> database of annotated chemical reactions. GO MF and biological process (BP) terms may be annotated to reactions or pathways, respectively, in <a href="http://metacyc.org/">MetaCyc</a> or <a href="http://www.reactome.org/">Reactome</a>.</p>
<p>A complete list, with descriptions and links, is available on the <a href="http://geneontology.org/docs/download-mappings/">GO Consortium’s Cross-References</a> page.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-retrieve-functional-annotations-genes-list', 'Can I retrieve functional annotations for genes in a list?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'can-i-retrieve-functional-annotations-genes-list' || pageName == 'all-faqs'">
<h3 id="can-i-retrieve-functional-annotations-genes-list">Can I retrieve functional annotations for genes in a list?</h3>
<!-- pombase_categories: Finding data,Tools and resources,Using ontologies -->
<p>You can find the GO annotations for your genes corresponding to functional roles and localizations. Our recommended approach depends on how many specific topics you are interested in:</p>
<ul>
<li><p>For a small number of specific GO terms (e.g. localization to the nucleus or cytoplasm, or a role in signaling or DNA metabolism), you can use the <a routerLink="/query">advanced search</a>. Paste your gene list into the “Gene names and IDs” query, and then combine it with a GO query for each term of interest (see the <a routerLink="/documentation/advanced-search">search documentation</a> for more information).</p></li>
<li><p>If you are interested in many GO terms, or if you do not know in advance which terms may be relevant, we recommend that you use a “GO term enrichment” tool. Such tools are typically used to find terms overrepresented for a gene list, but can be used to retrieve all GO annotations if the p-value threshold is set artificially high.</p></li>
</ul>
<p>Both the Advanced Search and term enrichment tools take advantage of the hierarchical structure of GO, such that annotations to specific terms are propagated to “ancestor” terms via <em>is_a</em> and <em>part_of</em> relations. See the <a routerLink="/documentation/gene-page-gene-ontology">PomBase GO documentation</a>, and the GO Consortium documentation linked there, for more information. (These approaches also make it easier to maintain and update your data than storing individual GO annotations locally.)</p>
<p>Also see the <a routerLink="/faq/how-can-i-find-significant-shared-go-annotations-genes-list">FAQ on GO term enrichment</a> and the <a routerLink="/documentation/pombase-go-slim-documentation">PomBase GO Slim documentation</a> page.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-retrieve-list-s.-pombe-go-slim-terms', 'Can I retrieve a list of the S. pombe GO slim terms?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-statistics-and-lists' || pageName == 'using-ontologies' || pageName == 'can-i-retrieve-list-s.-pombe-go-slim-terms' || pageName == 'all-faqs'">
<h3 id="can-i-retrieve-list-s.-pombe-go-slim-terms">Can I retrieve a list of the S. pombe GO slim terms?</h3>
<!-- pombase_categories: Genome statistics and lists,Using ontologies -->
<p>Yes, you can download the term names and IDs for the each of the fission yeast GO slims, and the Mondo Disease Ontology slim, from the PomBase FTP site:</p>
<p>Current GO slim IDs and term names</p>
<ul>
<li><a href="ftp://ftp.pombase.org/nightly_update/misc/mf_goslim_pombe_ids_and_names.tsv">GO molecular function slim</a></li>
<li><a href="ftp://ftp.pombase.org/nightly_update/misc/bp_goslim_pombe_ids_and_names.tsv">GO biological process slim</a></li>
<li><a href="ftp://ftp.pombase.org/nightly_update/misc/cc_goslim_pombe_ids_and_names.tsv">GO cellular component slim</a></li>
</ul>
<p>Current <a href="ftp://ftp.pombase.org/nightly_update/misc/disease_ontology_slim_ids_and_names.tsv">fission yeast Mondo Disease Ontology slim IDs and term names</a></p>
<p>For further information on using the <em>S. pombe</em> slims, please see the PomBase GO slim pages (<a href="/browse-curation/fission-yeast-bp-go-slim-terms">biological process</a>, <a href="/browse-curation/fission-yeast-mf-go-slim-terms">molecular function</a>, <a href="/browse-curation/fission-yeast-cc-go-slim-terms">cellular component</a>), <a href="/browse-curation/fission-yeast-go-slimming-tips">GO slimming tips</a>, and <a routerLink="/documentation/pombase-go-slim-documentation">GO slim documentation</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-retrieve-sequence-any-cosmids-used-build-s.-pombe-genome-sequence', 'Can I retrieve the sequence for any of the cosmids used to build the S. pombe genome sequence?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'can-i-retrieve-sequence-any-cosmids-used-build-s.-pombe-genome-sequence' || pageName == 'all-faqs'">
<h3 id="can-i-retrieve-sequence-any-cosmids-used-build-s.-pombe-genome-sequence">Can I retrieve the sequence for any of the cosmids used to build the <em>S. pombe</em> genome sequence?</h3>
<!-- pombase_categories: Finding data,Tools and resources -->
<p>Although old cosmid sequences used in the reference assembly are not available in PomBase directly, they are all stored in the International Nucleotide Sequence Database Collaboration database (ENA, GenBank, DDBJ) archives. For ease of searching, PomBase curators recommend finding the accession, e.g. AL137130, for a cosmid, and using GenBank to retrieve the sequence:</p>
<p>Go to <a href="http://www.ncbi.nlm.nih.gov/nucleotide/" class="uri">http://www.ncbi.nlm.nih.gov/nucleotide/</a> (or choose “Nucleotide” in the search pull-down menu on any NCBI search page). Enter the accession. The resulting page will inform you that the sequence has been replaced by one of the whole-chromosome entries, but offers links to both the current chromosome entry and the obsolete contig entry.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-search-gene-list-and-retrieve-results-same-order-as-input-list', 'Can I search for a gene list and retrieve results in the same order as in the input list?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'can-i-search-gene-list-and-retrieve-results-same-order-as-input-list' || pageName == 'all-faqs'">
<h3 id="can-i-search-gene-list-and-retrieve-results-same-order-as-input-list">Can I search for a gene list and retrieve results in the same order as in the input list?</h3>
<!-- pombase_categories: Finding data -->
<p>At present this is not possible; all advanced search results are ordered alphabetically by gene name, or by systematic ID for unnamed genes, by default. Click the “Product” column header to sort alphabetically by product description. We plan to add an option to preserve input order soon.</p>
<p>In the meantime, we suggest that you download the results using the “Details” link, and import them into a spreadsheet. You can then combine the results with other data in the spreadsheet, and use the spreadsheet software to sort on any column.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-search-genes-based-on-conservation-different-taxa', 'Can I search for genes based on conservation in different taxa?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'orthology' || pageName == 'finding-data' || pageName == 'can-i-search-genes-based-on-conservation-different-taxa' || pageName == 'all-faqs'">
<h3 id="can-i-search-genes-based-on-conservation-different-taxa">Can I search for genes based on conservation in different taxa?</h3>
<!-- pombase_categories: Orthology,Finding data -->
<p>Yes, use the “Taxonomic conservation” query in the <a routerLink="/query">advanced search</a>.</p>
<p>Example query: <app-query-link [goToResults]="true" [linkText]="'Genes conserved in vertebrates'" [predefinedQueryId]="'canned_query:conserved_in_vertebrates'"> </app-query-link></p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-search-pombase-protein-features', 'Can I search PomBase for protein features?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'can-i-search-pombase-protein-features' || pageName == 'all-faqs'">
<h3 id="can-i-search-pombase-protein-features">Can I search PomBase for protein features?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>A selection of protein sequence motifs and features have been manually curated using terms from the <a href="http://sequenceontology.org/">Sequence Ontology</a> (SO). For example, <a routerLink="/gene/SPAC15A10.03c">Rad54</a> has a KEN box (a motif recognized by the anaphase-promoting complex; <a routerLink="/term/SO:0001807">SO:0001807</a>), and <a routerLink="/gene/SPAC31A2.11c">Cuf1</a> and <a routerLink="/gene/SPAC1D4.10">Trz1</a> have nuclear localization signals (NLS; <a routerLink="/term/SO:0001528">SO:0001528</a>). These annotations are included in the Protein Features section of the gene page.</p>
<p>To search for these features, use the “Protein feature” query in the <a routerLink="/query">advanced search</a> (enter a SO ID or description; see the <a routerLink="/documentation/advanced-search">documentation</a> for help with searching).</p>
<p>Also see the FAQs on <a routerLink="/faq/how-can-i-find-proteins-have-transmembrane-domains">transmembrane domains</a> and <a routerLink="/faq/how-can-i-find-all-s.-pombe-proteins-particular-protein-family">protein families</a>, and the section of the search documentation on protein feature and protein domain queries.</p>
<p>Example query: <app-query-link [goToResults]="true" [linkText]="'nuclear localization signal (SO:0001528)'" [predefinedQueryId]="'SO_1528'"> </app-query-link></p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-share-my-search-results', 'Can I share my search results?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'can-i-share-my-search-results' || pageName == 'all-faqs'">
<h3 id="can-i-share-my-search-results">Can I share my search results?</h3>
<!-- pombase_categories: Finding data -->
<p>Yes. In the advanced search, each results page has a stable, unique URL that you can bookmark, copy/paste, and share. Anyone who follows a shared link will see the same results page, and the query will be added to their query history.</p>
<p>The “Visualise” and “Slim” options also generate stable, unique, sharable URLs, and QuiLT visualisations have the stable URLs no matter how you reach the QuiLT page.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-use-blast-find-s.-pombe-sequences-similar-my-query-sequence', 'Can I use BLAST to find S. pombe sequences similar to my query sequence?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'tools-and-resources' || pageName == 'can-i-use-blast-find-s.-pombe-sequences-similar-my-query-sequence' || pageName == 'all-faqs'">
<h3 id="can-i-use-blast-find-s.-pombe-sequences-similar-my-query-sequence">Can I use BLAST to find <em>S. pombe</em> sequences similar to my query sequence?</h3>
<!-- pombase_categories: Tools and resources -->
<p>PomBase does not maintain a BLAST server, but the BLAST tools available at <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">NCBI</a> and <a href="https://fungi.ensembl.org/Multi/Tools/Blast">Ensembl</a> allow a target species genome to be selected.</p>
<p>Note, however, that we recommend alternate approaches for some search purposes, especially for proteins:</p>
<ul>
<li>If you have a short peptide sequence, our <a routerLink="/motif_search">peptide motif search</a> finds matches in <em>S. pombe</em> proteins quickly.</li>
<li>If you are specifically looking for <em>S. pombe</em> orthologs of a protein-coding gene, simple BLAST results can be misleading. Instead, you can look up manually curated orthologs in PomBase for <a routerLink="/faq/how-can-i-find-s.-pombe-ortholog-s-human-gene">human</a> and <a routerLink="/faq/how-can-i-search-s.-cerevisiae-ortholog-s-homolog-s-s.-pombe-gene">budding yeast</a>, or use the methods described in the FAQ on <a routerLink="/faq/how-can-i-find-s.-pombe-orthologs-species-other-than-human-and-s.-cerevisiae">finding orthologs in other species</a>.</li>
</ul>
<p>Also see other FAQs in the <a routerLink="/faq/orthology">Orthology</a> category and the <a routerLink="/documentation/orthologs">Ortholog curation</a> documentation.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-use-blast-search-short-sequences', 'Can I use BLAST to search for short sequences?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'tools-and-resources' || pageName == 'can-i-use-blast-search-short-sequences' || pageName == 'all-faqs'">
<h3 id="can-i-use-blast-search-short-sequences">Can I use BLAST to search for short sequences?</h3>
<!-- pombase_categories: Tools and resources -->
<p>Yes, you can search for short nucleotide sequences, such as primers or other oligomers. We recommend NCBI BLAST, where you can search for short nucleotide sequences, such as primers or other oligomers (PomBase does not maintain its own BLAST server).</p>
<p>For short peptide sequences, we recommend the PomBase <a routerLink="/motif_search">peptide motif search</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-use-wild-card-beginning-search-string', 'Can I use a wild card at the beginning of a search string?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'can-i-use-wild-card-beginning-search-string' || pageName == 'all-faqs'">
<h3 id="can-i-use-wild-card-beginning-search-string">Can I use a wild card at the beginning of a search string?</h3>
<!-- pombase_categories: Finding data -->
<p>The simple search finds search strings anywhere in IDs, gene names, and product descriptions. It does not require any wildcard character, and shows autocomplete suggestions as you type.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-view-my-data-privately-genome-browser', 'Can I view my data privately in the genome browser?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-browser' || pageName == 'can-i-view-my-data-privately-genome-browser' || pageName == 'all-faqs'">
<h3 id="can-i-view-my-data-privately-genome-browser">Can I view my data privately in the genome browser?</h3>
<!-- pombase_categories: Genome browser -->
<p>Yes, if you have locally stored data that you want to see in the context of the genome browser, you can add it as a custom track. In the Track menu, select “Open track file or URL”, and enter the location in the popup.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-view-my-protein-interest-multiple-alignment-related-proteins-other-species', 'Can I view my protein of interest in a multiple alignment with related proteins in other species?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'orthology' || pageName == 'tools-and-resources' || pageName == 'can-i-view-my-protein-interest-multiple-alignment-related-proteins-other-species' || pageName == 'all-faqs'">
<h3 id="can-i-view-my-protein-interest-multiple-alignment-related-proteins-other-species">Can I view my protein of interest in a multiple alignment with related proteins in other species?</h3>
<!-- pombase_categories: Orthology,Tools and resources -->
<p>PomBase does not have its own alignment viewer. DIOPT, Compara, and PANTHER, three of the tools we recommend for finding orthologs, have alignment display options. See the <a href="faq/how-can-i-find-s.-pombe-orthologs-species-other-than-human-and-s.-cerevisiae">FAQ on orthologs in other species</a> for more information.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-view-nucleotide-sequence-genome-browser', 'Can I view nucleotide sequence in the genome browser?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-browser' || pageName == 'tools-and-resources' || pageName == 'can-i-view-nucleotide-sequence-genome-browser' || pageName == 'all-faqs'">
<h3 id="can-i-view-nucleotide-sequence-genome-browser">Can I view nucleotide sequence in the genome browser?</h3>
<!-- pombase_categories: Genome browser,Tools and resources -->
<p>Yes. First, make sure the DNA sequence track is enabled (see the <a routerLink="/faq/how-can-i-show-or-hide-tracks-genome-browser">FAQ on showing tracks</a> if necessary).</p>
<p>Then simply zoom in until sequence appears. The graphic will first display colored blocks representing color-coded nucleotides, and then legible sequence.</p>
<p>At any zoom level, use the arrows to scroll along the sequence.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-view-variation-data-pombase', 'Can I view variation data in PomBase?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'genome-browser' || pageName == 'can-i-view-variation-data-pombase' || pageName == 'all-faqs'">
<h3 id="can-i-view-variation-data-pombase">Can I view variation data in PomBase?</h3>
<!-- pombase_categories: Finding data,Genome browser -->
<p>The PomBase genome browser includes variation data from natural <em>S. pombe</em> isolates, published in:</p>
<p>Jeffares DC <em>et al.</em> 2015. The genomic and phenotypic diversity of Schizosaccharomyces pombe. <em>Nat Genet.</em> <strong>47(3):</strong>235-241. doi: <a href="http://dx.doi.org/10.1038/ng.3215">10.1038/ng.3215</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25665008">PMID:25665008</a></p>
<p>To view the variation data, enable one or more of the tracks under “Variation” (see the <a routerLink="/faq/how-can-i-show-or-hide-tracks-genome-browser">FAQ on showing tracks</a> if necessary). Insertions/deletions (indels) and SNPs can be enabled as separate tracks.</p>
<p>Clicking on any variation feature in either the track brings up a pop-up box with further details about the variation.</p>

</div>
<div (click)="faq_navigate($event, '/faq/can-i-visualise-networks-pombase', 'Can I visualise networks in PomBase?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'tools-and-resources' || pageName == 'can-i-visualise-networks-pombase' || pageName == 'all-faqs'">
<h3 id="can-i-visualise-networks-pombase">Can I visualise networks in PomBase?</h3>
<!-- pombase_categories: Tools and resources -->
<p>This is an area of active development at PomBase. At present, PomBase links GO terms to the web-based network tool <a href="http://www.esyn.org/">esyN</a> <sup>1</sup>: on the <a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">GO biological process slim page</a> and on ontology term pages for GO biological process slim terms, each GO slim term links to the <a routerLink="/documentation/high-confidence-physical-interaction-network">HCPIN physical interaction network</a> in esyN. For example, the GO process slim page and the ontology term page for “regulation of mitotic cell cycle phase transition” (<a routerLink="/term/GO:1901990">GO:1901990</a>) link to <a href="http://www.esyn.org//builder.php?type=Graph&amp;term=GO:1901990&amp;interactionType=physical&amp;source=pombase&amp;includeInteractors=false" class="uri">http://www.esyn.org//builder.php?type=Graph&amp;term=GO:1901990&amp;interactionType=physical&amp;source=pombase&amp;includeInteractors=false</a>.</p>
<p>We plan to provide gene-centred network displays on gene pages in the near future.</p>
<!--
On gene pages, we have links to gene-specific interaction networks in
esyN in the table headers of the Interactions sections:

-   The Genetic Interactions section links to all interactions centred
    on the gene and curated in BioGRID
-   The Physical interactions section has links to two datasets:
    -   All physical interactions curated in BioGRID for the gene
    -   Interactions for the gene in the PomBase <a routerLink="/documentation/high-confidence-physical-interaction-network">High Confidence         Physical Interaction Network         (HCPIN)</a> 

For example, the Genetic Interactions header for cdc2 links to
<http://www.esyn.org/builder.php?type=Graph&query=SPBC11B10.09&organism=4896&interactionType=genetic&source=biogrid>

-->
<p>Using the esyN network display:</p>
<ul>
<li>A brief simulation is used to position the nodes initially. This layout can often be improved by continuing the simulation: click “Layouts” in the left-hand panel, then click “Force-Directed (improve)”. Repeat until you like the network arrangement.</li>
<li>You can extend any network in esyN:</li>
</ul>
<ol type="1">
<li>Click on a node in the display.</li>
<li>Click “Extend Network” in the right-hand bar.</li>
<li>In the Advanced Tools box under the network display, first click “Get Interactions”.</li>
<li>In the table that appears, click “Add” buttons to add individual interactions, or click “Add all”.</li>
</ol>
<p>You can also use EsyN to:</p>
<p>Visualize interactions for a user-defined list of genes. To do this, visit <a href="http://www.esyn.org/builder.php?type=Graph" class="uri">http://www.esyn.org/builder.php?type=Graph</a>, click on the “Network from list” option in the left-hand panel, and follow the instructions in the pop-up.</p>
<p>Build your own network – either an Interactome graph or a Petri net – from scratch (see the tutorial at <a href="http://www.esyn.org/tutorial.html" class="uri">http://www.esyn.org/tutorial.html</a>). In both cases you can use the Advanced Tools to retrieve the interactions for a number of model organisms from several databases (see <a href="http://www.esyn.org/builder.php?type=Graph#interactions" class="uri">http://www.esyn.org/builder.php?type=Graph#interactions</a>).</p>
<p>Save and share your networks. By logging in via the “My esyN” link (at the top of every esyN page), any user can save, share privately with collaborators, or publish any network.</p>
<p>Browse, view and modify, previously published, models (both graphs and Petri nets) at <a href="http://www.esyn.org/browse.php" class="uri">http://www.esyn.org/browse.php</a>. We describe these networks are “public” in the sense of the open source movement, so that they are not only free to be copied, modified and (when possible) re-published, but we also actively encourage any collaborative effort to build and improve these biological networks.</p>
<p><sup>1</sup>esyN reference: Bean DM, Heimbach J, Ficorella L, Micklem G, Oliver SG, Favrin G. 2014. esyN: network building, sharing and publishing. <em>PLoS One</em>. 2014 Sep 2; <strong>9(9)</strong>:e106035. doi: <a href="http://dx.doi.org/10.1371/journal.pone.0106035">10.1371/journal.pone.0106035</a>. eCollection 2014. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25181461">PMID:25181461</a></p>

</div>
<div (click)="faq_navigate($event, '/faq/do-i-need-password-download-ftp-site', 'Do I need a password to download from the FTP site?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'genome-statistics-and-lists' || pageName == 'tools-and-resources' || pageName == 'do-i-need-password-download-ftp-site' || pageName == 'all-faqs'">
<h3 id="do-i-need-password-download-ftp-site">Do I need a password to download from the FTP site?</h3>
<!-- pombase_categories: Finding data,Genome statistics and lists,Tools and resources -->
<p>You do not need a password to download anything from the PomBase FTP site. In some web browsers, clicking a link to an FTP directory takes you to a page listing the contents, and clicking a file link produces a dialog box asking whether to open or save the file. Other browsers require a separate connection step:</p>
<ul>
<li><p>If your browser asks “Do you want to allow this page to …?”, answer “yes” or “allow”.</p></li>
<li><p>If you’re prompted to “connect as guest or registered user”, select “guest”.</p></li>
</ul>
<p>You should then be able to connect and download files (note: we have observed that this step can be sensitive to slow network connections).</p>
<p>If these steps don’t work, please let us know via the <a href="mailto:helpdesk@pombase.org">Helpdesk</a>, and we will try to assist you.</p>

</div>
<div (click)="faq_navigate($event, '/faq/do-you-have-polyadenylation-data-fission-yeast', 'Do you have polyadenylation data for fission yeast?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'do-you-have-polyadenylation-data-fission-yeast' || pageName == 'all-faqs'">
<h3 id="do-you-have-polyadenylation-data-fission-yeast">Do you have polyadenylation data for fission yeast?</h3>
<!-- pombase_categories: Finding data -->
<p>Fission yeast polyadenylation data are available in the genome browser. To display the data, enable one or more of the tracks with the data type “Poly(A) sites”.</p>
<p><a routerLink="/faq/how-can-i-show-or-hide-tracks-genome-browser">Help with configuring browser tracks</a> is available.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-are-mating-type-specific-gene-pages-organized', 'How are the mating type specific gene pages organized?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'gene-page' || pageName == 'finding-data' || pageName == 'how-are-mating-type-specific-gene-pages-organized' || pageName == 'all-faqs'">
<h3 id="how-are-mating-type-specific-gene-pages-organized">How are the mating type specific gene pages organized?</h3>
<!-- pombase_categories: Gene page, Finding data -->
<p>As described in detail in this <a href="https://www1.bio.ku.dk/english/research/fg/cellecyklus_genomintegritet/mating/">online tutorial</a> (external link) provided by the Nielsen lab, <em>S. pombe</em> switch between the two mating types M and P.</p>
<p>Genetic information encoded by the mat1 locus determines the mating type: if this locus contains the Pc and Pi genes, the cell is mating type P, and if it contains the Mc and Mi genes the cell is mating type M. Additionally, <em>S. pombe</em> contains two silent loci: mat2 and mat3. These loci are not expressed but host the information needed for each mating type configuration. Mat2 contains the two genes Pc and Pi, and Mat3 contains the two genes Mc and Mi. Recombinational DNA repair during mitotic cell division ensures production of one daughter cell of parental mating type and one daughter of the opposite mating type.</p>
<p>A wild type S. pombe cell thus contains 6 mating type specific genes: 1. mat1-P/Mc - expressed 2. mat1-P/Mi - expressed 3. mat2-Pc - silent 4. mat2-Pi - silent 5. mat3-Mc - silent 6. mat3-Mi - silent</p>
<p>The sequenced <em>S. pombe</em> reference strain (972 h-) is in the M mating type configuration (encodes the Mc and Mi genes at the mat1 locus). The mat2 genes are deleted in this strain for technical reasons, whereas the mat3 genes are intact.</p>
<p>The DNA seqence of the WT <a href="https://www.pombase.org/status/mating-type-region">silent mating type region</a> containing the mat2 and mat3 genes was reconstructed yielding an extra <a href="https://www.pombase.org/downloads/genome-datasets">mating type region contig</a> (see ‘current genome’ ftp site link). Only the P genes from this contig have gene pages in PomBase.</p>
<p>The systematic IDs and contig source of the mating type specific genes are: 1. mat1-Mc - SPBC23G7.09 (from the chromosome 2 contig) 2. mat1-Mi - SPBC23G7.17c (from the chromosome 2 contig) 3. mat2-Pc - SPMTR.01 (from the mating type contig) 4. mat2-Pi - SPMTR.02 (from the mating type contig) 5. mat3-Mc - SPBC1711.02 (from the chromosome 2 contig) 6. mat3-Mi - SPBC1711.01c (from the chromosome 2 contig)</p>
<p>The duplicate M genes without gene pages are: 7. mat3-Mc - SPMTR.04 (extra copy from the mating type contig) 8. mat3-Mi - SPMTR.03 (extra copy from the mating type contig)</p>
<p>For the M specific genes, functional annotation (GO, phenotypes…) is attached to the mat1-Mc and mat1-Mi genes. For the P specific genes, functional annotation is attached to mat2-Pc and mat2-Pi out of necessity (the reference strain is in the M configuration).</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-are-non-coding-rna-genes-identified', 'How are non-coding RNA genes identified?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-statistics-and-lists' || pageName == 'how-are-non-coding-rna-genes-identified' || pageName == 'all-faqs'">
<h3 id="how-are-non-coding-rna-genes-identified">How are non-coding RNA genes identified?</h3>
<!-- pombase_categories: Genome statistics and lists -->
<p>Most non-coding RNAs in PomBase are based on transcriptome data, either from Jürg Bähler’s lab (Solexa/deep sequencing; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>) or Nick Rhind’s lab (RNA sequencing; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21511999">PMID:21511999</a>). For any ncRNA, the source should be linked as a publication in the “Literature” section at the bottom of the PomBase gene page. To get an idea of the transcription in a region, you can look at the Bähler Lab Transcriptome Viewer, which is linked from most gene pages, e.g. <a href="http://www.bahlerlab.info/cgi-bin/pombetv/pombetv?condition=13&genename=SPNCRNA.200&action=display">SPNCRNA.200</a>. Unfortunately some genes, such as SPNCRNA.1115, post-date the viewer and therefore do not have entries, but you can look at transcription in this region by accessing a neighboring gene.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-are-pombase-systematic-ids-determined', 'How are PomBase systematic IDs determined?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-statistics-and-lists' || pageName == 'how-are-pombase-systematic-ids-determined' || pageName == 'all-faqs'">
<h3 id="how-are-pombase-systematic-ids-determined">How are PomBase systematic IDs determined?</h3>
<!-- pombase_categories: Genome statistics and lists -->
<p>Systematic IDs follow patterns based on the feature type, and in some cases the chromosome, as shown in the table below.</p>
<p>Open reading frame (ORF) IDs also indicate which cosmid or plasmid they were found on in genome sequencing. In most cases, ORF IDs that end with a digit indicate that the ORF is on the forward (Watson) strand, and an ORF with an ID that ends with ‘c’ is on the reverse (Crick) stand. There are a few exceptions, however, because some cosmids were moved and their orientation reversed late in the sequence assembly procedure</p>
<p>IDs with ‘.1’ appended are transcript IDs; the dot-and-digit IDs follow Ensembl’s standard. At present, PomBase has only one transcript annotated for any given feature, but in the future when alternative transcripts are annotated the digit will be incremented (.2, .3, etc.).</p>
<p><strong>Systematic ID patterns</strong></p>
<table>
<thead>
<tr class="header">
<th>ID pattern</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>SPAC*</td>
<td>features, usually ORFs, on chromosome 1, sequenced on cosmids</td>
</tr>
<tr class="even">
<td>SPBC*</td>
<td>features, usually ORFs, on chromosome 2, sequenced on cosmids</td>
</tr>
<tr class="odd">
<td>SPCC*</td>
<td>features, usually ORFs, on chromosome 3, sequenced on cosmids</td>
</tr>
<tr class="even">
<td>SPAP*</td>
<td>features, usually ORFs, on chromosome 1, sequenced on plasmids</td>
</tr>
<tr class="odd">
<td>SPBP*</td>
<td>features, usually ORFs, on chromosome 2, sequenced on plasmids</td>
</tr>
<tr class="even">
<td>SBCP*</td>
<td>features, usually ORFs, on chromosome 3, sequenced on plasmids</td>
</tr>
<tr class="odd">
<td>SPATRNA*</td>
<td>tRNA genes on chromosome 1</td>
</tr>
<tr class="even">
<td>SPBTRNA*</td>
<td>tRNA genes on chromosome 2</td>
</tr>
<tr class="odd">
<td>SPCTRNA*</td>
<td>tRNA genes on chromosome 3</td>
</tr>
<tr class="even">
<td>SPLTRA*</td>
<td>LTRs on chromosome 1</td>
</tr>
<tr class="odd">
<td>SPLTRB*</td>
<td>LTRs on chromosome 2</td>
</tr>
<tr class="even">
<td>SPLTRC*</td>
<td>LTRs on chromosome 3</td>
</tr>
<tr class="odd">
<td>SPNCRNA*</td>
<td>non-coding RNA genes (no chromosome info in ID)</td>
</tr>
<tr class="even">
<td>SPRPTA.*</td>
<td>repeats (other than LTRs or centromeric repeats) on chromosome 1</td>
</tr>
<tr class="odd">
<td>SPRPTB.*</td>
<td>repeats (other than LTRs or centromeric repeats) on chromosome 2</td>
</tr>
<tr class="even">
<td>SPRPTC.*</td>
<td>repeats (other than LTRs or centromeric repeats) on chromosome 3</td>
</tr>
<tr class="odd">
<td>SPRPTCENA*</td>
<td>centromeric repeats on chromosome 1</td>
</tr>
<tr class="even">
<td>SPRPTCENB*</td>
<td>centromeric repeats on chromosome 2</td>
</tr>
<tr class="odd">
<td>SPRPTCENC*</td>
<td>centromeric repeats on chromosome 3</td>
</tr>
<tr class="even">
<td>SPRRNA*</td>
<td>rRNA genes (no chromosome info in ID)</td>
</tr>
<tr class="odd">
<td>SPSNORNA*</td>
<td>snoRNA genes (no chromosome info in ID)</td>
</tr>
<tr class="even">
<td>SPSNRNA*</td>
<td>snRNA genes (no chromosome info in ID)</td>
</tr>
<tr class="odd">
<td>SPTF*</td>
<td>transposons (no chromosome info in ID)</td>
</tr>
<tr class="even">
<td>SPMTR*</td>
<td>features on the separately sequenced mating type region contig</td>
</tr>
<tr class="odd">
<td>SPMIT*</td>
<td>features on the mitochondrial chromosome</td>
</tr>
<tr class="even">
<td>SPMITTRNA*</td>
<td>subset of SPMIT*; tRNA genes on mitochondrial chromosome</td>
</tr>
<tr class="odd">
<td>SPNUMT*</td>
<td>NUMTs (nuclear mitochondrial pseudogenes) (no chromosome info in ID)</td>
</tr>
</tbody>
</table>

</div>
<div (click)="faq_navigate($event, '/faq/how-are-sequence-feature-types-defined-pombase', 'How are sequence feature types defined in PomBase?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'genome-statistics-and-lists' || pageName == 'how-are-sequence-feature-types-defined-pombase' || pageName == 'all-faqs'">
<h3 id="how-are-sequence-feature-types-defined-pombase">How are sequence feature types defined in PomBase?</h3>
<!-- pombase_categories: Finding data,Genome statistics and lists -->
<p>S. pombe genome features were originally annotated using <a href="http://www.sanger.ac.uk/science/tools/artemis">Artemis</a>. As noted in the manual (ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/artemis.pdf - see p. 9), Artemis draws from a list of feature keys that is documented at EBI: (<a href="ftp://ftp.ebi.ac.uk/pub/databases/embl/doc/FT\_current.html\#7.2\" class="uri">ftp://ftp.ebi.ac.uk/pub/databases/embl/doc/FT\_current.html\#7.2\</a>)</p>
<p>In the genome sequence data files, features are defined using <a href="http://sequenceontology.org/">Sequence Ontology</a> terms. Gene pages use a selection of human-friendly text descriptions for feature types.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-browse-phenotype-ontology-fypo', 'How can I browse the phenotype ontology (FYPO)?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'how-can-i-browse-phenotype-ontology-fypo' || pageName == 'all-faqs'">
<h3 id="how-can-i-browse-phenotype-ontology-fypo">How can I browse the phenotype ontology (FYPO)?</h3>
<!-- pombase_categories: Finding data,Tools and resources,Using ontologies -->
<p>In the future, we plan to make Fission Yeast Phenotype Ontology (FYPO) terms and annotations available in a browser analogous to AmiGO or QuickGO. Until such a browser becomes available, FYPO is accessible in these external resources:</p>
<p>NCBO BioPortal - search on the <a href="http://bioportal.bioontology.org/">BioPortal home page</a>, go to the <a href="http://bioportal.bioontology.org/ontologies/FYPO">FYPO summary page</a>, or go to the <a href="http://bioportal.bioontology.org/ontologies/FYPO/?p=classes&conceptid=root">FYPO terms page</a>. For assistance, see the “User Interface” part of the <a href="http://bioportal.bioontology.org/help">BioPortal Help</a>.</p>
<p>EBI’s Ontology Lookup Service (OLS) - search on the <a href="http://www.ebi.ac.uk/ontology-lookup/">OLS home page</a> or go to the <a href="http://www.ebi.ac.uk/ontology-lookup/browse.do?ontName=FYPO">FYPO page</a>. Help is provided on each page.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-display-sequence-region-using-sequence-coordinates-genome-browser', 'How can I display a sequence region using sequence coordinates in the genome browser?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-browser' || pageName == 'finding-data' || pageName == 'how-can-i-display-sequence-region-using-sequence-coordinates-genome-browser' || pageName == 'all-faqs'">
<h3 id="how-can-i-display-sequence-region-using-sequence-coordinates-genome-browser">How can I display a sequence region using sequence coordinates in the genome browser?</h3>
<!-- pombase_categories: Genome browser,Finding data -->
<p>Go to the <a href="https://www.pombase.org/jbrowse/">genome browser</a>, and enter coordinates in the location box. The format is ‘I:100000..200000’ (i.e. use Roman numerals to specify the chromosome, and don’t include the word “chromosome”; use ‘..’ between the start and end coordinates.)</p>
<p>You can also choose a chromosome from the pulldown, and enter just the numbers specifying position in the location box.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-all-genes-have-given-mutant-phenotype', 'How can I find all of the genes that have a given mutant phenotype?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-all-genes-have-given-mutant-phenotype' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-all-genes-have-given-mutant-phenotype">How can I find all of the genes that have a given mutant phenotype?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>You can search for genes annotated to a Fission Yeast Phenotype Ontology term in the <a routerLink="/query">advanced search</a>. In the “Phenotype” query, if you know the ID (for example, “inviable cell” is <a routerLink="/term/FYPO:0000049">FYPO:0000049</a>, and “elongated cell” is <a routerLink="/term/FYPO:0002177">FYPO:0000017</a>) you can type or paste the ID into the box. Otherwise, start typing a term name or description; the autocomplete feature will suggest phenotypes. Choose one to retrieve annotated genes.</p>
<p>Note that the FYPO search retrieves annotations by following the <em>is_a</em>, <em>part_of</em>, <em>output_of</em>, <em>has_output</em>, and <em>has_part</em> relationships in the ontology. For example, FYPO includes the relation “inviable swollen elongated cell with enlarged nucleus” (<a routerLink="/term/FYPO:0002177">FYPO:0002083</a>) <em>has_part</em> “swollen cell” (<a routerLink="/term/FYPO:0000025">FYPO:0000025</a>). Genes annotated to FYPO:0002083 will therefore be retrieved in a search for FYPO:0000025. See the <a routerLink="/documentation/advanced-search">advanced search documentation</a> for more information.</p>
<p>Example query: <app-query-link [goToResults]="true" [linkText]="'Genes annotated to &quot;elongated cell&quot; (FYPO:0000017), all alleles, any expression level'" [predefinedQueryId]="'FYPO_17_all'"> </app-query-link></p>
<p>Also see the FAQ on <a routerLink="/faq/can-i-get-list-essential-pombe-genes">finding essential genes</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-all-s.-pombe-genes-are-conserved-human', 'How can I find all S. pombe genes that are conserved in human?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'genome-statistics-and-lists' || pageName == 'orthology' || pageName == 'how-can-i-find-all-s.-pombe-genes-are-conserved-human' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-all-s.-pombe-genes-are-conserved-human">How can I find all <em>S. pombe</em> genes that are conserved in human?</h3>
<!-- pombase_categories: Finding data,Genome statistics and lists,Orthology -->
<p>Almost all genes that are conserved between fission yeast and human are also conserved in other vertebrates (there are two exceptions, both of which are genes encoding amino acid biosynthesis proteins that have become pseudogenes in human). To retreive these genes, go to the <a routerLink="/query">advanced search</a>, click the “Taxonomic conservation” query option, and choose the description “Conserved in vertebrates”.</p>
<p>Also see the FAQs on <a routerLink="/faq/how-can-i-find-s.-pombe-genes-associated-human-disease">finding disease gene orthologs</a>, <a routerLink="/faq/how-can-i-find-s.-pombe-ortholog-s-human-gene">finding the ortholog of a specific gene</a>, and on <a routerLink="/faq/how-can-i-obtain-list-human-and-s.-pombe-orthologs">downloading the full set of curated orthologs</a>.</p>
<p>Query link: <app-query-link [goToResults]="true" [linkText]="'Genes conserved in vertebrates'" [predefinedQueryId]="'canned_query:conserved_in_vertebrates'"></app-query-link></p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-all-s.-pombe-proteins-particular-protein-family-or-have-particular-domain', 'How can I find all S. pombe proteins in a particular protein family, or that have a particular domain?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'orthology' || pageName == 'finding-data' || pageName == 'how-can-i-find-all-s.-pombe-proteins-particular-protein-family-or-have-particular-domain' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-all-s.-pombe-proteins-particular-protein-family-or-have-particular-domain">How can I find all <em>S. pombe</em> proteins in a particular protein family, or that have a particular domain?</h3>
<!-- pombase_categories: Orthology,Finding data -->
<p>There are various ways you can find protein family members or domains.</p>
<ol type="1">
<li>If you know the Pfam, PRINTs, PROSITE, or InterPro accession for the family or domain you want, you can use the <a routerLink="/query">advanced search</a>). Click “Protein domain” and then enter the accession.</li>
<li>If you don’t have an accession, but do know any member of the family, go directly to its gene page. In the “Protein Features” section of the gene page there is a table of protein domains and families. The “Count” column links to a list of all family members in <em>S. pombe</em>.</li>
<li>If you know neither accessions nor family members, you can search for keywords in the <a href="http://www.ebi.ac.uk/interpro/">InterPro</a> database, which combines signatures from a number of member databases, including Pfam. Record the accession number(s) of the family, and use them in the PomBase advanced search as described in item 1 above. (If necessary, you can use the query history to combine the results of several queries.)</li>
</ol>
<p>Example query: <app-query-link [goToResults]="true" [linkText]="'Proteins matching &quot;ATPase, AAA-type, core&quot; (Pfam:PF00004)'" [predefinedQueryId]="'Pfam_PF00004'"> </app-query-link></p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-all-sequence-features-region-using-chromosome-coordinates', 'How can I find all sequence features in a region using chromosome coordinates?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-browser' || pageName == 'finding-data' || pageName == 'how-can-i-find-all-sequence-features-region-using-chromosome-coordinates' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-all-sequence-features-region-using-chromosome-coordinates">How can I find all sequence features in a region using chromosome coordinates?</h3>
<!-- pombase_categories: Genome browser,Finding data -->
<p>You can do this in the <a href="https://www.pombase.org/jbrowse/">genome browser</a> (or follow a link from a gene page). First <a routerLink="/faq/how-can-i-display-sequence-region-using-sequence-coordinates-genome-browser">enter the coordinates</a>.</p>
<p>Ensure that the PomBase forward and reverse strand feature tracks are visible (they are enabled by default). For each strand, click the small triangle at the right-hand end of the track label to reveal the popup menu. Select “Save track data”, then choose from the options in the popup.</p>
<p>You can also use the <a href="https://www.pombase.org/query">advanced search</a> to retrieve genes in a region, but at present other sequence feature types are not included. Use the “Genome location” query. First, select a chromosome in the pulldown. You can then either add start and end coordinates to specify a region, or leave the “Restrict to region” boxes blank to retrieve all genes on the chromosome.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-genes-region-using-chromosome-coordinates', 'How can I find genes in a region using chromosome coordinates?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'how-can-i-find-genes-region-using-chromosome-coordinates' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-genes-region-using-chromosome-coordinates">How can I find genes in a region using chromosome coordinates?</h3>
<!-- pombase_categories: Finding data -->
<p>To retrieve genes in a region in the <a href="https://www.pombase.org/query">advanced search</a>, use the “Genome location” query. First, select a chromosome in the pulldown. You can then either add start and end coordinates to specify a region, or leave the “Restrict to region” boxes blank to retrieve all genes on the chromosome.</p>
<p>You can also use the genome browser to find all features (not only genes), <a routerLink="/faq/how-can-i-retrieve-sequence-region-using-sequence-coordinates">as described here</a>.</p>
<p>Example query: <app-query-link [goToResults]="true" [linkText]="'Genes between coordinates 1000000-2000000 on chromosome 2'" [predefinedQueryId]="'chr2_1000000-2000000'"> </app-query-link></p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-genes-specific-activity', 'How can I find genes with a specific activity?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-genes-specific-activity' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-genes-specific-activity">How can I find genes with a specific activity?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>PomBase uses Gene Ontology (GO) molecular function terms to capture the activities – including enzymatic activities, binding, transporters, etc. – of gene products. You can therefore use the GO term query in the <a routerLink="/query">advanced search</a> to retrieve genes whose products have a given activity.</p>
<p>In the “GO” query, if you know the ID (for example, “histone acetyltransferase activity” is <a routerLink="/term/GO:0004402">GO:0004402</a>, and “calcium ion transmembrane transporter activity” is <a routerLink="/term/GO:0015085">GO:0015085</a>), type or paste it into the box. Otherwise, and start typing a name or description; the autocomplete feature will suggest terms. Choose one to retrieve annotated genes. You can try using more specific or less specific terms to retrieve the results that best fit your expectations and needs. See the <a routerLink="/documentation/advanced-search">advanced search documentation</a> and the <a routerLink="/documentation/gene-page-gene-ontology">Gene Page GO documentation</a> for more information, including how [ontology searches retrieve annotations] to general terms.</p>
<p>Example query: <app-query-link [goToResults]="true" [linkText]="'phosphoprotein phosphatase activity (GO:0004721)'" [predefinedQueryId]="'GO_4721'"> </app-query-link></p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-go-annotations-other-schizosaccharomyces-species-e.g.-s.-japonicus', 'How can I find GO annotations for other Schizosaccharomyces species (e.g. S. japonicus)?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-go-annotations-other-schizosaccharomyces-species-e.g.-s.-japonicus' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-go-annotations-other-schizosaccharomyces-species-e.g.-s.-japonicus">How can I find GO annotations for other <em>Schizosaccharomyces</em> species (e.g. <em>S. japonicus</em>)?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>For the sequenced strains of <em>S. japonicus</em>, <em>S. octosporus</em> and <em>S. cryophilus</em>, the Ensembl group has generated GO annotation data sets for protein-coding genes by transferring experiment-based annotations from <em>S. pombe</em> orthologs. You can use the <a href="http://www.ebi.ac.uk/QuickGO/">QuickGO</a> browser to retrieve the data for each species – follow the “Search and Filter GO annotation sets” link, then click “Filter” to set a taxon filter for the taxon ID:</p>
<ul>
<li><em>S. japonicus</em> (strain yFS275): 402676</li>
<li><em>S. octosporus</em> (strain yFS286): 483514</li>
<li><em>S. cryophilus</em> (strain OY26): 653667</li>
</ul>
<p>Because these automated annotations are inferred only from experimentally-derived <em>S. pombe</em> annotations, coverage will not be complete.</p>
<p>Note that the GAF downloaded from QuickGO uses UniProtKB accessions in the gene product ID column (column 2). To use the GAF in any further analysis, such as term enrichment, you will have to convert the accessions to systematic IDs. See the <a routerLink="/faq/can-i-convert-ids-other-databases-or-pombase-ids">FAQ on ID mapping</a> for hints.</p>
<p>One feasible approach to improve annotation coverage is to download the <em>S. pombe</em> GO annotations (see the <a routerLink="/downloads/go-annotations">GO Annotations download</a></p>
<p>page), and then substitute the <em>S. pombe</em> IDs with the IDs of orthologous genes from the other <em>Schizosaccharomyces</em> species of interest. For ortholog IDs, see the <a routerLink="/faq/how-can-i-find-orthologs-between-s.-pombe-and-other-schizosaccharomyces-species">FAQ on <em>Schizosaccharomyces</em> orthologs</a>, and use the indicated table from Rhind <em>et al.</em> Comparative functional genomics of the fission yeasts (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21511999">PMID:21511999</a>).</p>
<p>Note that some genes are present in <em>S. japonicus</em>, <em>S. octosporus</em> or <em>S. cryophilus</em> but absent from <em>S. pombe</em>. For some of these gene products, GO annotations can be transferred from other species. If you wish to include annotations for these genes in your analysis you will need to use this option, and extend your GAF with the relevant annotation lines (contact the <a href="mailto:helpdesk@pombase.org">Helpdesk</a> if you need assistance). Combining all approaches gives the best coverage possible at present. You can use a “GO Slim” tool such as Princeton’s GO Term Mapper to see if there are any gaps in coverage, as described in the <a routerLink="/faq/how-can-i-find-significant-shared-go-annotations-genes-list">FAQ on enrichment in <em>S. pombe</em></a>. Also see the FAQs on GO term enrichment in <a routerLink="/faq/can-i-do-go-term-enrichment-other-schizosaccharomyces-species-e.g.-s.-japonicus">other <em>Schizosaccharomyces</em> species</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-intron-branch-sites', 'How can I find intron branch sites?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'how-can-i-find-intron-branch-sites' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-intron-branch-sites">How can I find intron branch sites?</h3>
<!-- pombase_categories: Finding data -->
<p>The PomBase genome browser includes a dataset of intron branch sites as a track. Follow <a href="https://www.pombase.org/jbrowse/?loc=I%3A704338..724338&tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CDNA%20sequence%2CIntron%20Branch%20Points%20-%20Bitton%20et%20al.%20&#40;2014&#41;&tracklist=1&nav=1&overview=1&highlight=">this link</a> to view the data.</p>
<p>The data were published in: Bitton DA et al. (2014), <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24709818">PMID:24709818</a>; <a href="http://dx.doi.org/10.1101/gr.166819.113">DOI:10.1101/gr.166819.113</a></p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-modifications-my-protein-interest', 'How can I find modifications for my protein of interest?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-modifications-my-protein-interest' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-modifications-my-protein-interest">How can I find modifications for my protein of interest?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>Protein modifications (where curated) are included in the Modifications section on gene pages. (We plan to include RNA modifications later.) The <a routerLink="/documentation/gene-page-modifications">gene page modifications documentation</a> describes the display.</p>
<p>To retrieve all genes whose products have a given modification, use the “Protein modifications” query in the [Advanced Search]. If you know the ID (for example, “phosphorylated residue” is MOD:00696), type or paste it into the box. Otherwise, start typing a name or description; the autocomplete feature will suggest terms. Choose one to retrieve annotated genes. See the <a routerLink="/documentation/advanced-search">advanced search documentation</a> for more information, including how ontology searches retrieve annotations to general terms.</p>
<p>Example query: <app-query-link [goToResults]="true" [linkText]="'phosphorylated residue (MOD:00696)'" [predefinedQueryId]="'MOD_696'"> </app-query-link></p>
<p>There is also an ontology term page for each modification term used to annotate <em>S. pombe</em> proteins, e.g. <a routerLink="/term/MOD:00696">MOD:00696</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-mutants-increased-or-decreased-sensitivity-chemicals', 'How can I find mutants with increased or decreased sensitivity to chemicals')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-mutants-increased-or-decreased-sensitivity-chemicals' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-mutants-increased-or-decreased-sensitivity-chemicals">How can I find mutants with increased or decreased sensitivity to chemicals</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>In FYPO, terms for increased chemical sensitivity are grouped under <a href="https://www.pombase.org/term/FYPO:0002683">increased sensitivity to chemical (FYPO:0002683)</a>. Similarly, decreased chemical sensitivity (increased resistance) terms are grouped under <a href="https://www.pombase.org/term/FYPO:0002682">increased resistance to chemical (FYPO:0002682)</a>.</p>
<p>The ontology term pages for each term shows more specific terms, most of which identify specific chemicals (e.g. <a href="https://www.pombase.org/term/FYPO:0000088">sensitive to hydroxyurea (FYPO:000088)</a>, and the genotypes annotated to them.</p>
<p>In any phenotype annotation display (on gene pages, ontology term pages, or publication pages), phenotype annotations can be <a routerLink="/documentation/gene-page-phenotypes">filtered</a> to show only “sensitive to chemical” annotations.</p>
<p>You can also use the advanced search Phenotype query to search for <app-query-link [goToResults]="true" [linkText]="'FYPO:0002682'" [predefinedQueryId]="'single_allele_chemical_resistance'"></app-query-link>, <app-query-link [goToResults]="true" [linkText]="'FYPO:0002683'" [predefinedQueryId]="'single_allele_chemical_sensitivity'"></app-query-link> or any of the more specific terms. (Links go to results lists of single-allele genotypes annotated to each term.)</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-orthologs-between-s.-pombe-and-other-schizosaccharomyces-species', 'How can I find orthologs between S. pombe and other Schizosaccharomyces species?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'orthology' || pageName == 'finding-data' || pageName == 'how-can-i-find-orthologs-between-s.-pombe-and-other-schizosaccharomyces-species' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-orthologs-between-s.-pombe-and-other-schizosaccharomyces-species">How can I find orthologs between <em>S. pombe</em> and other <em>Schizosaccharomyces</em> species?</h3>
<!-- pombase_categories: Orthology,Finding data -->
<p>On a gene-by-gene basis, you can use the Ensembl genome browser link to reach Fungal Compara as described in the <a routerLink="/faq/how-can-i-find-s.-pombe-orthologs-species-other-than-human-and-s.-cerevisiae">FAQ on orthologs in other species</a>.</p>
<p>For a full set of orthologous genes in <em>S. pombe</em>, <em>S. cryophilus</em>, <em>S.japonicus</em> and <em>S. octosporus</em>, see Table S12, columns AD-AG, in Rhind <em>et al.</em> Comparative functional genomics of the fission yeasts (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21511999">PMID:21511999</a>).</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-protein-localization-data', 'How can I find protein localization data?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-protein-localization-data' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-protein-localization-data">How can I find protein localization data?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>Gene Ontology (GO) cellular component annotations capture the localizations of gene products to subcellular structures such as organelles or complexes. GO Cellular Component annotations are displayed on PomBase gene pages as described in the <a routerLink="/documentation/gene-page-gene-ontology">PomBase GO documentation</a>. The GO Consortium’ ontology <a href="http://geneontology.org/docs/ontology-documentation/">overview</a> describes what the Cellular Component ontology includes. To search for proteins (or functional RNAs) with a particular localization, use the GO query in the <a routerLink="/query">advanced search</a> to find genes annotated to the relevant GO Cellular Component term(s).</p>
<p>Pombase GO Cellular Component annotations include data from the whole-genome localization study (Matsuyama et al. 2006) as well as manually curated data from papers on small-scale experiments, and inferences from ortholog annotations. Macromolecular complex annotations are also available in a file (see <a routerLink="/faq/there-list-protein-complexes-s.-pombe-and-their-subunits">FAQ</a>).</p>
<p>Example query: <app-query-link [goToResults]="true" [linkText]="'nucleus (GO:0005634)'" [predefinedQueryId]="'GO_5634'"> </app-query-link></p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-proteins-have-transmembrane-domains', 'How can I find proteins that have transmembrane domains?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'how-can-i-find-proteins-have-transmembrane-domains' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-proteins-have-transmembrane-domains">How can I find proteins that have transmembrane domains?</h3>
<!-- pombase_categories: Finding data -->
<p>In the <a routerLink="/query">advanced search</a>, use the “Number of TM domains” query. Enter the minimum and maximum number of domains (use the same number as minimum and maximum to retrieve proteins with, e.g. exactly 7 transmembrane domains).</p>
<p>Example query: <app-query-link [goToResults]="true" [linkText]="'Genes whose products have 7 transmembrane domains'" [predefinedQueryId]="'7TM'"> </app-query-link></p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-rrna-genes', 'How can I find rRNA genes?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'how-can-i-find-rrna-genes' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-rrna-genes">How can I find rRNA genes?</h3>
<!-- pombase_categories: Finding data -->
<p>In the <a routerLink="/query">advanced search</a>, click “Product type”, then select “rRNA” from the pulldown.</p>
<p>Also see the FAQ on <a routerLink="/faq/there-any-rdna-repeat-sequences-pombase">rDNA sequences</a>.</p>
<p>Query: <app-query-link [goToResults]="true" [linkText]="'rRNA genes'" [predefinedQueryId]="'rrna_genes'"> </app-query-link></p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-s.-pombe-genes-associated-human-disease', 'How can I find S. pombe genes associated with human disease?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'orthology' || pageName == 'finding-data' || pageName == 'how-can-i-find-s.-pombe-genes-associated-human-disease' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-s.-pombe-genes-associated-human-disease">How can I find <em>S. pombe</em> genes associated with human disease?</h3>
<!-- pombase_categories: Orthology,Finding data -->
<p><em>S. pombe</em> genes whose human orthologs have been implicated in disease are annotated with terms from the <a href="https://mondo.monarchinitiative.org/">Monarch Disease Ontology</a>. To retrieve all of these genes, you can use the most general “disease” term in a query. In the <a routerLink="/query">advanced search</a>, select “Disease”, and copy or paste the ID “MONDO:0000001”.</p>
<p>Also see the FAQs on <a routerLink="/faq/how-can-i-find-all-s.-pombe-genes-are-conserved-human">finding genes conserved in human</a>, <a routerLink="/faq/how-can-i-find-s.-pombe-ortholog-s-human-gene">finding the ortholog of a specific gene</a>, and on <a routerLink="/faq/how-can-i-obtain-list-human-and-s.-pombe-orthologs">downloading the full set of curated orthologs</a>.</p>
<p>Example queries:</p>
<ul>
<li><p><app-query-link [goToResults]="true" [linkText]="'disease or disorder (MONDO:0000001)'" [predefinedQueryId]="'canned_query:all_diseases'"> </app-query-link></p></li>
<li><p><app-query-link [goToResults]="true" [linkText]="'cancer (MONDO:0004992)'" [predefinedQueryId]="'cancer'"> </app-query-link></p></li>
</ul>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-s.-pombe-genes-complement-or-are-complemented-by-gene-another-species', 'How can I find S. pombe genes that complement or are complemented by a gene from another species?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-s.-pombe-genes-complement-or-are-complemented-by-gene-another-species' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-s.-pombe-genes-complement-or-are-complemented-by-gene-another-species">How can I find <em>S. pombe</em> genes that complement or are complemented by a gene from another species?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>If there is complementation data available for an <em>S. pombe</em> gene, it will be displayed in the Complementation section of the gene page. For example, <a routerLink="/spombe/result/SPAC57A10.12c"><em>ura3</em></a> can be complemented by <em>S. cerevisiae</em> <a href="http://www.yeastgenome.org/locus/S000001699/overview"><em>URA1</em></a>, and itself complements human <a href="http://www.genenames.org/cgi-bin/gene_symbol_report?hgnc_id=HGNC:2867">DHODH</a>.</p>
<!-- To search for complementation annotations, use one of the "PBO"
filters in the <a routerLink="/query">advanced search</a> (see the
<a routerLink="/documentation/advanced-search">documentation</a> for help with
searching). The complementation descriptions are stored as entries in
the PBO internal ontology, so a search for PBO term names that match
"complements" or "complemented by" will retrieve genes with
complementation data curated. The most general term, "complementation"
(PBO:2000000) retrieves all genes that have any complementation
annotation.

Example queries:

-   <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2223%22,%22query%22:%22PBO:2000000%22%7D%7D,%22filter_count%22:%221%22%7D%5D">Genes with any complementation annotation     (PBO:2000000)</a> 
-   <a routerLink="/spombe/query/builder?filter=37&value=%5B%7B%22param%22:%7B%22filter_1%22:%7B%22filter%22:%2223%22,%22query%22:%22PBO:0011087%22%7D%7D,%22filter_count%22:%221%22%7D%5D">functionally complements S. cerevisiae URA1     (PBO:0011087)</a> 
-->

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-s.-pombe-ortholog-s-human-gene', 'How can I find S. pombe ortholog(s) of a human gene?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'orthology' || pageName == 'finding-data' || pageName == 'how-can-i-find-s.-pombe-ortholog-s-human-gene' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-s.-pombe-ortholog-s-human-gene">How can I find <em>S. pombe</em> ortholog(s) of a human gene?</h3>
<!-- pombase_categories: Orthology,Finding data -->
<p>In PomBase, human orthologs are curated for <em>S. pombe</em> genes as described in the <a routerLink="/documentation/orthologs">Orthologs documentation</a>.</p>
<p>To find <em>S. pombe</em> orthologs for a human gene, you can search for the standard human gene name in the simple search box in the page header. For example, searching for human <em>ABTB1</em> will retrieve the <em>S. pombe</em> gene <em>btb3</em>. To find standard human gene names, you can search <a href="http://www.genenames.org/">HGNC</a>. Note that in a few cases, a human gene name will coincidentally match a name or synonym of a non-orthologous <em>S. pombe</em> gene as well as the actual curated ortholog(s), so please check the gene pages carefully, especially if your search retrieves more than one result.</p>
<p>Also see the FAQs on on <a routerLink="/faq/how-can-i-find-all-s.-pombe-genes-are-conserved-human">finding genes conserved in human</a>, <a routerLink="/faq/how-can-i-find-s.-pombe-genes-associated-human-disease">finding disease gene orthologs</a>, and on <a routerLink="/faq/how-can-i-obtain-list-human-and-s.-pombe-orthologs">downloading the full set of curated orthologs</a>. You may also find the FAQ on <a routerLink="/faq/how-can-i-find-s.-pombe-orthologs-species-other-than-human-and-s.-cerevisiae">orthologs in other species</a> useful.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-s.-pombe-orthologs-species-other-than-human-and-s.-cerevisiae', 'How can I find S. pombe orthologs for species other than human and S. cerevisiae?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'orthology' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'how-can-i-find-s.-pombe-orthologs-species-other-than-human-and-s.-cerevisiae' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-s.-pombe-orthologs-species-other-than-human-and-s.-cerevisiae">How can I find <em>S. pombe</em> orthologs for species other than human and <em>S. cerevisiae</em>?</h3>
<!-- pombase_categories: Orthology,Finding data,Tools and resources -->
<p>For orthologs that are not manually curated by PomBase, we suggest a few approaches:</p>
<p><strong>DIOPT</strong></p>
<p>From any gene page, follow the link to DIOPT (in the Orthologs section and under External References).</p>
<p>The <a href="http://www.flyrnai.org/diopt">Drosophila RNAi Screening Center Integrative Ortholog Prediction Tool</a> (DIOPT) is a web-based tool that integrates several orthology prediction tools to identify orthologous proteins for nine species (<em>Caenorhabditis elegans, Danio rerio, Drosophila melanogaster, Homo sapiens, Mus musculus, Rattus norvegicus, Saccharomyces cerevisiae, Schizosaccharomyces pombe, and Xenopus tropicalis</em>).</p>
<p><strong>PANTHER</strong></p>
<p>The PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System classifies proteins (and their genes) according to family and subfamily, molecular function, biological process, and pathway.</p>
<p>From any gene page, follow the link to PANTHER (in the Orthologs section and under External References). The linked page includes a list of orthologs, and links to phylogenetic tree views that can also display alignments. <a href="http://www.pantherdb.org/help/PANTHERhelp.jsp">Help</a> is available.</p>
<p><strong>Compara</strong></p>
<p>You can search for orthologs/paralogs in Fungi, or in a pan-taxonomic comparison (eukaryotes), using Compara in the Ensembl browser.</p>
<ol type="1">
<li><p>On any gene page, follow the link to the Ensembl genome browser under Orthologs or External References.</p></li>
<li><p>Click the “Gene:” tab to show Compara links in the left-hand margin - for fungal alignments, choose “Fungal Compara”, or for all eukaryotic species choose “Pan-taxonomic Compara”.</p></li>
<li><p>You should see a “collapsed” gene tree highlighting your fission yeast gene of interest. From here you can click on any node to see a menu of options:</p>
<ol type="1">
<li>Expand or collapes specific sub-nodes of the tree, or expand the tree fully</li>
<li>View the alignment in FASTA format</li>
<li>Launch the jalview multiple alignment viewer to see the full alignment and colour by residue conservation, hydrophobicity, etc.</li>
</ol></li>
<li><p>To configure the protein entries visible in the alignment, select the most “inclusive” node you require. You can reduce the number of entries by collapsing individual sub-trees before you generate your alignment.</p></li>
</ol>
<p>Information about how the Compara trees are generated, homology types, and species is available from the <a href="http://ensemblgenomes.org/info/data/comparative_genomics">Ensembl comparative genomics documentation</a>.</p>
<p><strong>PSI-BLAST</strong></p>
<p>To search for putative orthologs not fund in DIOPT or Compara, we recomment PSI-BLAST (Position-Specific Iterated BLAST) at <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&PAGE=Proteins&PROGRAM=blastp&RUN_PSIBLAST=on">NCBI</a> or <a href="https://www.ebi.ac.uk/Tools/sss/psiblast/">EBI</a>. As described in the <a href="https://www.ncbi.nlm.nih.gov/books/NBK2590/">NCBI tutorial</a>, PSI-BLAST derives a position-specific scoring matrix (PSSM) or profile from the multiple sequence alignment of sequences detected above a given score threshold using protein–protein BLAST.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-significant-shared-fypo-annotations-genes-list', 'How can I find significant shared FYPO annotations for genes in a list?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-significant-shared-fypo-annotations-genes-list' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-significant-shared-fypo-annotations-genes-list">How can I find significant shared FYPO annotations for genes in a list?</h3>
<!-- pombase_categories: Finding data,Tools and resources,Using ontologies -->
<p>FYPO enrichment analysis is analogous to GO term enrichment, using phenotypes rather than GO annotations, i.e. analysing a gene list by finding FYPO terms that are significantly over- or under-represented among the annotations for the genes.</p>
<p>At present, PomBase does not have its own FYPO enrichment tool, and very few ontology enrichment tools can use phenotype data. One that does is <a href="http://bahlerweb.cs.ucl.ac.uk/cgi-bin/GLA/GLA_input">AnGeLi</a>, produced by Jürg Bähler’s lab.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-significant-shared-go-annotations-genes-list', 'How can I find significant shared GO annotations for genes in a list?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'how-can-i-find-significant-shared-go-annotations-genes-list' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-significant-shared-go-annotations-genes-list">How can I find significant shared GO annotations for genes in a list?</h3>
<!-- pombase_categories: Finding data,Tools and resources,Using ontologies -->
<p>GO term enrichment identifies GO terms that are significantly overrepresented (or underrepresented) among a set of genes.</p>
<p>At present PomBase does not have its own GO enrichment tool. We recommend using the <a href="http://go.princeton.edu/cgi-bin/GOTermFinder">Generic GO Term Finder at Princeton</a>, because it offers a simple interface and up-to-date ontology and annotation data, including the current PomBase GO annotation dataset (you can upload your own backgound set, GO annotation file, or both). You can also use the GO Term Finder to retrieve all annotations for your gene list by setting the p-value to 1.</p>
<p>Before you perform an enrichment analysis, we recommend that you do a GO slim analysis of your gene list, for a broad overview of the annotation set (for more information, see the <a href="/documentation/pombase-go-slim-documentation">Fission Yeast GO slim documentation</a> page and <a routerLink="/faq/how-can-i-use-go-slims-s.-pombe">FAQ</a>). You can use Princeton’s GO slimming tool, <a href="http://go.princeton.edu/cgi-bin/GOTermMapper">GO Term Mapper</a>, which is especially useful if you use your own GAF for the enrichment, because it will show:</p>
<ul>
<li>IDs in your gene list that are missing from the annotation set (the annotations in GO Term Mapper’s database or your uploaded GAF)</li>
<li>Genes in your list that have annotations but do not map to the slim</li>
<li>Genes in your list that have no annotations</li>
</ul>
<p>A few other enrichment tools are described on the GO Consortium’s <a href="https://geneontology.github.io/docs/go-enrichment-analysis/">GO Enrichment Analysis</a> page. If you choose one of these, we recommend that you only use an enrichment tool that allows you to upload a background set representing the genes used in your experiment. Even if you regard the whole genome as the relevant background, it is important to specify the background gene set explicitly to obtain meaningful results, especially if you have data only for protein-coding genes. For example, annotated tRNA genes can obscure enrichment of protein-coding genes annotated to translation.</p>
<p>For any GO analysis, we strongly recommend that you describe your approach fully in methods, and include the release details (number and/or date) for PomBase and the GO terms and annotations you use.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-snorna-genes', 'How can I find snoRNA genes?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'how-can-i-find-snorna-genes' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-snorna-genes">How can I find snoRNA genes?</h3>
<!-- pombase_categories: Finding data -->
<p>In the <a routerLink="/query">advanced search</a>, click “Product type”, then select “snoRNA” from the pulldown.</p>
<p>Note that there are likely a number of snoRNAs that have not yet been identified and annotated in <em>S. pombe</em>; we hope to investigate further in the future.</p>
<p>Query: <app-query-link [goToResults]="true" [linkText]="'snoRNA genes'" [predefinedQueryId]="'snorna_genes'"> </app-query-link></p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-transcription-factor-binding-sites-pombase', 'How can I find transcription factor binding sites in PomBase?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'how-can-i-find-transcription-factor-binding-sites-pombase' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-transcription-factor-binding-sites-pombase">How can I find transcription factor binding sites in PomBase?</h3>
<!-- pombase_categories: Finding data -->
<p>There is a data track available for transcription factor binding sites in the genome browser. To enable the track, follow the <a routerLink="/faq/how-can-i-show-or-hide-tracks-genome-browser">instructions for showing tracks</a>, and filter on the data type “Promoters”, or use this link:</p>
<p><a href="https://www.pombase.org/jbrowse/?loc=III%3A725765..736352&tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CConsensus%20transcription%20factor%20DNA%20binding%20motifs&highlight=">Transcription factor binding sitse in JBrowse</a></p>
<p>Also see the <a routerLink="/faq/how-can-i-find-transcription-factors-and-their-targets-pombase">FAQ on finding transcription factors</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-transcription-factors-and-their-targets-pombase', 'How can I find transcription factors and their targets in PomBase?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'how-can-i-find-transcription-factors-and-their-targets-pombase' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-transcription-factors-and-their-targets-pombase">How can I find transcription factors and their targets in PomBase?</h3>
<!-- pombase_categories: Finding data -->
<p>All sequence-specific DNA-binding transcription factors should be annotated to at least two GO Molecular Function terms, either directly or by transitivity (i.e. annotated to a more specific “descendant” term linked to one of these terms):</p>
<ul>
<li>GO:0000976 transcription regulatory region sequence-specific DNA binding (view in <a routerLink="/term/GO:0000976">PomBase</a>, <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000976">QuickGO</a> or <a href="http://amigo.geneontology.org/amigo/term/GO:0000976">AmiGO</a>)</li>
<li>GO:0003700 sequence-specific DNA binding transcription factor activity (view in <a routerLink="/term/GO:0003700">PomBase</a>, <a href="http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003700">QuickGO</a> or <a href="http://amigo.geneontology.org/amigo/term/GO:0003700">AmiGO</a>)</li>
</ul>
<p><a routerLink="/faq/what-annotation-extension">Annotation extensions</a> are used to capture two types of “target” data (where available):</p>
<ul>
<li>Annotations to GO:0000976 (or a descendant) may have extensions that capture DNA binding specificity using <a href="http://sequenceontology.org">Sequence Ontology</a> (SO) terms. A list of DNA binding sites identified in <em>S. pombe</em> is available on the <a routerLink="/browse-curation/dna-binding-sites">DNA Binding Sites</a> page.</li>
<li>Annotations to GO:0003700 (or a descendant) may have extensions identifying target genes.</li>
</ul>
<p>Because it is not yet possible to query annotation extensions in the PomBase advanced search, to identify target genes you must either inspect transcription factor gene pages manually, or search the GO annotation dataset. For the latter:</p>
<ol type="1">
<li>Download the GO annotation file (GAF) from the <a routerLink="/downloads/go-annotations">GO Annotations</a> page. The file is tab-delimited text, so it can be opened in a spreadsheet application or parsed with a script; the <a href="http://geneontology.org/docs/go-annotation-file-gaf-format-2.1/">format</a> is described on the GO website.</li>
<li>Look up the GO IDs for the specific terms to which genes are directly annotated – the “Child Terms” feature in QuickGO is good for this (for “transcription factor activity”, the most commonly used term is GO:0000978, RNA polymerase II core promoter proximal region sequence-specific DNA binding).</li>
<li>Find annotations to the GO IDs of interest (GO ID is column 5; gene ID column 2), and then look at the annotation extensions (column 16).</li>
<li>Contact the <a href="mailto:helpdesk@pombase.org">Helpdesk</a> if you have any problems or questions.</li>
</ol>
<p>Finally, note that not all <em>S. pombe</em> transcription factors have been extensively characterised with respect to target genes, and for those that have, target curation in PomBase may be incomplete. You may therefore wish to query for transcription factors that have been have been experimentally characterised, and therefore might have targets which are not yet curated. To do so, use the Advanced Search to find which of the genes annotated to the transcription factor-related GO terms above have the annotation status “published” (e.g. GO ID “GO:0000978” AND Annotation Status “published”; see the <a routerLink="/documentation/advanced-search">advanced search documentation</a> for more tips on setting up the query).</p>
<p>Query links:</p>
<ul>
<li><app-query-link [goToResults]="true" [linkText]="'Genes annotated to &quot;transcription regulatory region sequence-specific DNA binding&quot; (GO:0000976) or &quot;sequence-specific DNA binding transcription factor activity&quot; (GO:0003700)'" [predefinedQueryId]="'GO_976_or_GO_3700'"></app-query-link></li>
<li><app-query-link [goToResults]="true" [linkText]="'Genes annotated to &quot;RNA polymerase II core promoter proximal region sequence-specific DNA binding&quot; (GO:0000978) with characterisation status &quot;published&quot;'" [predefinedQueryId]="'GO_978_and_published'"></app-query-link></li>
</ul>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-find-transposons-s.-pombe-genome', 'How can I find transposons in the S. pombe genome?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'how-can-i-find-transposons-s.-pombe-genome' || pageName == 'all-faqs'">
<h3 id="how-can-i-find-transposons-s.-pombe-genome">How can I find transposons in the <em>S. pombe</em> genome?</h3>
<!-- pombase_categories: Finding data,Tools and resources -->
<p>In the <a routerLink="/query">advanced search</a>, click “Characterisation status”, then select “transposon” from the pulldown.</p>
<p>At present, there are 11 full-length transposons annotated, and two frameshifted copies.</p>
<p>Query link: <app-query-link [goToResults]="true" [linkText]="'Transposons'" [predefinedQueryId]="'transposons'"></app-query-link></p>
<p>Lone LTRs are also annotated as sequence features. They cannot yet be retrieved by the simple or advanced searches, but they are included in the forward and reverse strand sequence feature tracks in the genome browser.</p>
<p>Finally, if you wish to install Artemis (available from <a href="http://www.sanger.ac.uk/science/tools/artemis">The Sanger Institute Artemis page</a>; for <em>S. pombe</em> we recommend using <a href="ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/v16/">Artemis Version 16</a>), you can use it to view LTRs in more detail. Read in the EMBL format files of sequence and annotation (available from the <a routerLink="/downloads/genome-datasets">Genome sequences</a> page). To see LTRs,</p>
<ol type="1">
<li>In the Select menu, choose “By Key”.</li>
<li>In the pulldown that pops up, choose “LTR”.</li>
</ol>
<p>See the <a routerLink="/faq/there-equivalent-artemis-java-applet-pombase">Artemis FAQ</a> and the <a href="ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/artemis.pdf">Artemis manual</a> (pdf; Sanger site) for additional information.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-get-cosmids-my-favorite-gene', 'How can I get cosmids for my favorite gene?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'how-can-i-get-cosmids-my-favorite-gene' || pageName == 'all-faqs'">
<h3 id="how-can-i-get-cosmids-my-favorite-gene">How can I get cosmids for my favorite gene?</h3>
<p>Please enquire with the Sanger Institute <a href="mailto:archives@sanger.ac.uk">archives@sanger.ac.uk</a> about clones included in the final genome sequence.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-get-my-data-into-pombase', 'How can I get my data into PomBase?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'data-submission-and-formats' || pageName == 'genome-browser' || pageName == 'how-can-i-get-my-data-into-pombase' || pageName == 'all-faqs'">
<h3 id="how-can-i-get-my-data-into-pombase">How can I get my data into PomBase?</h3>
<!-- pombase_categories: Data submission and formats,Genome browser -->
<p>The best route depends on the amount and type of data you have.</p>
<ul>
<li>If you have a published paper with “small-scale” data (a few genes, several different data types, etc.), you can curate it in <a href="http://curation.pombase.org/pombe">Canto</a>, PomBase’s online <a routerLink="/community/fission-yeast-community-curation-project">community curation</a> tool. The <a href="http://curation.pombase.org/pombe/docs/index/">Canto documentation</a> describes the supported data types (Gene Ontology, phenotypes, interactions, modifications) and how to use the curation interface.</li>
<li>If you have data from large-scale experiments that is associated with sequence coordinates, it can be displayed as a track in PomBase JBrowse. See the <a routerLink="/submit-data">data submission page</a> for instructions. <!-- Use the <a routerLink="/submit-data/data-submission-form">HTP data submission form</a>
    to send details.
--></li>
<li>If you have a large set of phenotype data (e.g. from a screen of the deletion collection), especially if there are many annotations to one or a few <a routerLink="/browse-curation/fission-yeast-phenotype-ontology">FYPO</a> term(s), you can send them to the helpdesk in the <a routerLink="/documentation/phenotype-data-bulk-upload-format">phenotype annotation spreadsheet format</a>.</li>
<li>Similarly, we can accept sets of GO, modification, or gene expression data as described on the <a href="/submit-data/bulk-annotation">dataset submission page</a>.</li>
<li>For large sets of genetic or physical interaction data, we recommend that you prepare a spreadsheet using the <a href="http://wiki.thebiogrid.org/doku.php/contribute">template provided by BioGRID</a> (see “Step 1. Send Us Your Interaction Data”). You are welcome to send the spreadsheet either to BioGRID or to PomBase (via the helpdesk); PomBase and BioGRID exchange data regularly so all interactions will appear in both databases.</li>
</ul>
<p>You can do both Canto curation and large-scale data submission for a single publication, if it reports both large- and small-scale experimental results.</p>
<p>For any data types not listed above, or if you have any questions, please contact the <a href="mailto:helpdesk@pombase.org">PomBase helpdesk</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-identify-all-genes-affect-process', 'How can I identify all of the genes that affect a process?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-identify-all-genes-affect-process' || pageName == 'all-faqs'">
<h3 id="how-can-i-identify-all-genes-affect-process">How can I identify all of the genes that affect a process?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>The best way to find genes that have any effect on a process, we recommend searching for both GO and FYPO terms relevant to the process.</p>
<p>As described in the <a routerLink="/faq/why-are-some-genes-abnormal-phenotype-annotated-corresponding-go-process-while-others-are-not">FAQ on GO and FYPO annotations</a>, PomBase curators annotate all genes with phenotypes that affect a process, whereas GO annotations are restricted to genes whose products act directly in a process or its regulation. By querying for genes annotated to either a GO term or a FYPO term, you can find genes with relevant phenotypes (including “downstream effects”) as well as genes involved in a process (with or without mutant phenotypes affecting the process).</p>
<p>Use the union (OR operator) button in the PomBase <a routerLink="/query">advanced search</a>, available in the query history, as described in the <a routerLink="/documentation/advanced-search">advanced search documentation</a>. For example, to find genes that affect cellular respiration, search for “<a routerLink="/term/FYPO:0000078">FYPO:0000078</a> (abnormal cellular respiration)”; search for “<a routerLink="/term/GO:0045333">GO:0045333</a> (cellular respiration)”; and then combine them in a third query. For any process, you can try using more specific or less specific terms to retrieve the results that best fit your expectations and needs.</p>
<p>Example query: <app-query-link [goToResults]="true" [linkText]="'genes annotated to &quot;abnormal cellular respiration&quot; (FYPO:0000078) or &quot;cellular respiration&quot; (GO:0045333)'" [predefinedQueryId]="'FYPO_78_or_GO_45333'"> </app-query-link></p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-locate-centromeres', 'How can I locate centromeres?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-browser' || pageName == 'finding-data' || pageName == 'how-can-i-locate-centromeres' || pageName == 'all-faqs'">
<h3 id="how-can-i-locate-centromeres">How can I locate centromeres?</h3>
<!-- pombase_categories: Genome browser,Finding data -->
<p>Centromeres can be retrieved in the PomBase genome browser; the coordinates are:</p>
<p>Chromosome I: <a href="https://www.pombase.org/jbrowse/?loc=I%3A3753680..3789414&amp;tracks=DNA%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features&amp;highlight=">3753687-3789421</a> Chromosome II: <a href="https://www.pombase.org/jbrowse/?loc=II%3A1602261..1644744&amp;tracks=DNA%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features&amp;highlight=">1602264-1644747</a> Chromosome III: <a href="https://www.pombase.org/jbrowse/?loc=III%3A1070899..1136998&amp;tracks=DNA%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features&amp;highlight=">1070904-1137003</a></p>
<p>Sequence features within the centromeres, such as repeats, are annotated with Sequence Ontology terms. For more details, see the <a routerLink="/status/centromeres">Centromeres</a> page.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-locate-mating-type-region', 'How can I locate the mating type region?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-browser' || pageName == 'finding-data' || pageName == 'how-can-i-locate-mating-type-region' || pageName == 'all-faqs'">
<h3 id="how-can-i-locate-mating-type-region">How can I locate the mating type region?</h3>
<!-- pombase_categories: Genome browser,Finding data -->
<p>Browse for Chromosome <a href="https://www.pombase.org/jbrowse/?loc=II%3A2129210..2137123&amp;tracks=DNA%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features&amp;highlight=">II:2129208-2137121</a>, and see the <a routerLink="/status/mating-type-region">Mating type region</a> page.</p>
<p>The mating type region will soon be annotated as a feature, and refer to a Sequence Ontology term.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-locate-telomeres-and-subtelomeric-regions', 'How can I locate telomeres and subtelomeric regions?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'how-can-i-locate-telomeres-and-subtelomeric-regions' || pageName == 'all-faqs'">
<h3 id="how-can-i-locate-telomeres-and-subtelomeric-regions">How can I locate telomeres and subtelomeric regions?</h3>
<!-- pombase_categories: Finding data -->
<p>The current <em>S. pombe</em> genome assembly does not include the complete telomeric regions or the telomeric short repeats. These omissions are beyond the control of PomBase curators. Subtelomeric repeats are also not explicitly defined at present, although we hope to provide this information in the future. Additional information about <em>S. pombe</em> telomeres is available at on the <a routerLink="/status/telomeres">Telomeres</a> page.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-obtain-list-human-and-s.-pombe-orthologs', 'How can I obtain the list of human and S. pombe orthologs?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'genome-statistics-and-lists' || pageName == 'orthology' || pageName == 'how-can-i-obtain-list-human-and-s.-pombe-orthologs' || pageName == 'all-faqs'">
<h3 id="how-can-i-obtain-list-human-and-s.-pombe-orthologs">How can I obtain the list of human and <em>S. pombe</em> orthologs?</h3>
<!-- pombase_categories: Finding data,Genome statistics and lists,Orthology -->
<p>The current version of the manually curated list of orthologs and orthologous groups identified between fission yeast and human is available for download from the <a href="ftp://ftp.pombase.org/pombe/orthologs/">orthologs</a> FTP directory (linked from the <a routerLink="/datasets">Datasets</a> page). Also see the FAQs on <a routerLink="/faq/how-can-i-find-all-s.-pombe-genes-are-conserved-human">finding genes conserved in human</a>, <a routerLink="/faq/how-can-i-find-s.-pombe-genes-associated-human-disease">finding disease gene orthologs</a>, and <a routerLink="/faq/how-can-i-find-s.-pombe-ortholog-s-human-gene">finding the ortholog of a specific gene</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-retrieve-all-s.-pombe-genes', 'How can I retrieve all S. pombe genes?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-statistics-and-lists' || pageName == 'finding-data' || pageName == 'how-can-i-retrieve-all-s.-pombe-genes' || pageName == 'all-faqs'">
<h3 id="how-can-i-retrieve-all-s.-pombe-genes">How can I retrieve all <em>S. pombe</em> genes?</h3>
<!-- pombase_categories: Genome statistics and lists,Finding data -->
<p>The <a routerLink="/downloads/names-and-identifiers">Names and identifiers page</a> provides lists of all genes, including non-coding RNA genes. The gene_IDs_names.tsv file contains the systematic IDs and names for both protein-coding and non-coding RNA genes, whereas two separate files are available that also include product descriptions.</p>
<p>The <a routerLink="/query">advanced search</a> includes a query that retrieves all protein-coding genes at once, as decribed in <a routerLink="/faq/how-can-i-retrieve-all-s.-pombe-protein-coding-genes">this FAQ</a>.</p>
<p>This query retrieves genes of all types (protein-coding, non-coding RNA, and pseudogenes) by combining the “product type” options with the OR operator:</p>
<p><app-query-link [goToResults]="true" [linkText]="'All genes'" [predefinedQueryId]="'canned_query:all_genes'"></app-query-link></p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-retrieve-all-s.-pombe-protein-coding-genes', 'How can I retrieve all S. pombe protein-coding genes?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'genome-statistics-and-lists' || pageName == 'how-can-i-retrieve-all-s.-pombe-protein-coding-genes' || pageName == 'all-faqs'">
<h3 id="how-can-i-retrieve-all-s.-pombe-protein-coding-genes">How can I retrieve all <em>S. pombe</em> protein-coding genes?</h3>
<!-- pombase_categories: Finding data,Genome statistics and lists -->
<p>The <a routerLink="/query">advanced search</a> includes a query that retrieves all protein-coding genes at once. Click “Canned queries”, then click “All protein coding genes (ex. dubious and transposon)”.</p>
<p>Further explanation: All protein coding genes have the type “protein coding”, but this type also includes a few transposon genes and several genes that are dubious (i.e. predicted by automated methods considered unlikely to actually encode protein), which you will presumably want to exclude from the set. The canned query does this.</p>
<p>Query link: <app-query-link [goToResults]="true" [linkText]="'Protein-coding genes (excluding &quot;dubious&quot; and &quot;transposon&quot; status)'" [predefinedQueryId]="'canned_query:all_protein_coding_ex_dubious_and_transposon'"></app-query-link></p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-retrieve-gene-sequence-including-upstream-and-downstream-sequences', 'How can I retrieve a gene sequence, including upstream and downstream sequences?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'gene-page' || pageName == 'genome-browser' || pageName == 'finding-data' || pageName == 'how-can-i-retrieve-gene-sequence-including-upstream-and-downstream-sequences' || pageName == 'all-faqs'">
<h3 id="how-can-i-retrieve-gene-sequence-including-upstream-and-downstream-sequences">How can I retrieve a gene sequence, including upstream and downstream sequences?</h3>
<!-- pombase_categories: Gene page,Genome browser,Finding data -->
<p>You can retrieve sequences from a gene page or in the genome browser, although flanking sequences are only available from gene pages at present.</p>
<p><strong>On the gene page</strong>: Scroll down or click the left-hand menu link to the Sequence section of the page, where a DNA sequence is displayed by default. For protein-coding genes, the CDS is shown, and you can click boxes to add the sequences of any annotated introns or 5’ or 3’ UTRs. To include flanking sequences, enter a number of nucleotides in the “upstream” and/or “downstream” boxes. Only applicable options appear for non-coding RNA genes. The “Download sequence” button saves a FASTA file with your selection.</p>
<p><strong>To use the Genome Browser</strong>: Click the “View in JBrowse” link under the map graphic on a gene page. In the browser, click on any feature to show a popup with details, including the sequence. Use controls in the popup to save FASTA sequence.</p>
<p>To retrieve flanking regions for more than one gene at a time, see <a routerLink="/faq/can-i-download-sequences-many-genes-once-including-flanking-regions">this FAQ</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-retrieve-intron-coordinates-or-sequences', 'How can I retrieve intron coordinates or sequences?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'how-can-i-retrieve-intron-coordinates-or-sequences' || pageName == 'all-faqs'">
<h3 id="how-can-i-retrieve-intron-coordinates-or-sequences">How can I retrieve intron coordinates or sequences?</h3>
<!-- pombase_categories: Finding data -->
<p>Downloadable intron datasets are available in FASTA format from the <a routerLink="/downloads/intron-data">Intron Data</a> page.</p>
<p>You can also find genes with introns using the PomBase <a routerLink="/query">advanced search</a>. To find all genes with introns, search for genes with a specified number of exons, and use the range 2 (i.e. at least one intron) to 20 (more than the maximum known, 16 introns). You can also restrict the search to protein-coding genes. Note that the PomBase count includes introns in UTRs.</p>
<p>Instructions for searching PomBase</p>
<ol type="1">
<li>Go to the <a routerLink="/query">advanced search</a>.</li>
<li>Select the “Coding exons” query.</li>
<li>Enter values: min 2, max 20. Click “Search”.</li>
<li>Optional: to restrict to protein-coding genes, select the “Product type” query, then choose “protein”. Use the “Intersect” (AND operator") button to combine the queries.</li>
</ol>
<p>Click on a count in the query history to see the results, with a button for “Download” options including coordinates and sequence.</p>
<p>Also see the <a routerLink="/faq/how-can-i-find-all-sequence-features-region-using-chromosome-coordinates">FAQ on finding sequence features in a region</a>.</p>
<p>Query link: <app-query-link [goToResults]="true" [linkText]="'protein-coding genes with 2-20 exons'" [predefinedQueryId]="'2-20_exons'"></app-query-link></p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-retrieve-sequence-coordinates-all-features-particular-type', 'How can I retrieve sequence coordinates for all features of a particular type?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'how-can-i-retrieve-sequence-coordinates-all-features-particular-type' || pageName == 'all-faqs'">
<h3 id="how-can-i-retrieve-sequence-coordinates-all-features-particular-type">How can I retrieve sequence coordinates for all features of a particular type?</h3>
<!-- pombase_categories: Finding data -->
<p>Available options:</p>
<ol type="1">
<li><p>Download one of the files available via the <a routerLink="/downloads/genome-datasets">Genome Datasets</a> page. The GFF3 files contain coordinates, and you can parse the files for the feature type you need. For example, to find all non-coding RNAs, search for “ncRNA_gene”; for coding sequences, use “CDS”, etc. There are also separate files available for CDS and UTR data.</p></li>
<li><p>If you only need genes, you can use the <a routerLink="/query">advanced search</a> to find all genes of a given type. (Note that non-gene features such as repeats cannot be retrieved by this method.) Select the “Product type” query, then choose a type from the pulldown menu. The “Download” options include coordinates and sequences in FASTA format. If you need more than one feature type, query for each type and then use Query Management to combine the individual queries with the “Union” (OR operator) button. See the <a routerLink="/documentation/advanced-search">advanced search documentation</a> for more information. Click on a count in the query history to see the results, with a button for “Download” options including coordinates and sequence.</p></li>
<li><p>The bioinformatically inclined can also use the Ensembl Genomes REST API to retrieve transcript feature coordinates, as described in the <a routerLink="/faq/s.-pombe-transcriptome-available-fasta-format">FAQ on pombe transcriptome sequences</a>. Select the desired feature type(s) from the output file of stable IDs (bear in mind that Ensembl idiosyncratically uses “biotype” to mean feature type). Note, however, that EG is updated much less frequently than PomBase, so EG data will rarely be as up-to-date as the PomBase web site. Documentation is available:</p></li>
</ol>
<p>Query: <app-query-link [goToResults]="true" [linkText]="'snoRNA genes'" [predefinedQueryId]="'snorna_genes'"> </app-query-link></p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-retrieve-sequence-region-using-sequence-coordinates', 'How can I retrieve the sequence of a region using sequence coordinates?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-browser' || pageName == 'finding-data' || pageName == 'how-can-i-retrieve-sequence-region-using-sequence-coordinates' || pageName == 'all-faqs'">
<h3 id="how-can-i-retrieve-sequence-region-using-sequence-coordinates">How can I retrieve the sequence of a region using sequence coordinates?</h3>
<!-- pombase_categories: Genome browser,Finding data -->
<p>Go to the <a href="https://www.pombase.org/jbrowse/">genome browser</a>, and make sure the DNA sequence track is enabled (see the <a routerLink="/faq/how-can-i-show-or-hide-tracks-genome-browser">FAQ on showing tracks</a> if necessary). Enter region coordinates (as described in <a href="https://www.pombase.org/faq/how-can-i-display-sequence-region-using-sequence-coordinates-genome-browser">this FAQ</a>.</p>
<p>In the “DNA sequence” track label, click on the small triangle at the right-hand end of the label box to reveal the popup menu, then select “Save track data”.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-retrieve-sequences-fasta-file-custom-headers', 'How can I retrieve sequences in a FASTA file with custom headers?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'how-can-i-retrieve-sequences-fasta-file-custom-headers' || pageName == 'all-faqs'">
<h3 id="how-can-i-retrieve-sequences-fasta-file-custom-headers">How can I retrieve sequences in a FASTA file with custom headers?</h3>
<!-- pombase_categories: Finding data -->
<p>You can customize the headers for FASTA files of downloaded results from the advanced search (i.e. for any list of genes). The Download dialog offers several options for items to include in the headers. See the <a routerLink="/documentation/advanced-search">advanced search documentation</a> for more information.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-retrieve-sequences-non-protein-coding-genes', 'How can I retrieve sequences for non-protein-coding genes?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'how-can-i-retrieve-sequences-non-protein-coding-genes' || pageName == 'all-faqs'">
<h3 id="how-can-i-retrieve-sequences-non-protein-coding-genes">How can I retrieve sequences for non-protein-coding genes?</h3>
<!-- pombase_categories: Finding data -->
<p>A file of all non-coding RNA gene sequences is available on the <a routerLink="/downloads/genome-datasets">Genome sequences page</a>.</p>
<p>If you need sequences for all genes of a single type (tRNAs, rRNAs, other ncRNAs, etc.) we recommend using the <a routerLink="/query">advanced search</a> and results download as described in the <a routerLink="/faq/how-can-i-retrieve-sequence-coordinates-all-features-particular-type">FAQ on retrieving sequence coordinates for all features of a particular type</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-retrieve-utr-sequences', 'How can I retrieve UTR sequences?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'how-can-i-retrieve-utr-sequences' || pageName == 'all-faqs'">
<h3 id="how-can-i-retrieve-utr-sequences">How can I retrieve UTR sequences?</h3>
<!-- pombase_categories: Finding data -->
<p>Transcript start and end coordinates from all sources will be available as individual data tracks in the genome browser in the near future, which will allow you to view, evaluate and download them. We also provide downloadable UTR data sets that are updated periodically, available on the <a routerLink="/downloads/genome-datasets">Genome sequence and features</a> page.</p>
<p>Also see the <a routerLink="/faq/how-do-you-determine-gene-s-full-length-transcript-utr-coordinates-transcription-start-and-end-sites">precedence criteria</a> used to choose default UTR features to display on gene pages.</p>
<p>To retrieve UTRs for a specified list of genes, see the FAQ on <a routerLink="/faq/can-i-download-sequences-many-genes-once-including-flanking-regions">downloading sequences for multiple genes</a> (choose 5’ UTR and/or 3’ UTR in the popup).</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-search-genes-involved-both-go-process-and-regulation-process', 'How can I search for genes involved in both a GO process and regulation of the process?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-search-genes-involved-both-go-process-and-regulation-process' || pageName == 'all-faqs'">
<h3 id="how-can-i-search-genes-involved-both-go-process-and-regulation-process">How can I search for genes involved in both a GO process and regulation of the process?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>The <a routerLink="/query">advanced search</a> GO query retrieves gene products annotated to a GO term and to any of its child terms, following the <em>is_a</em>, <em>part_of</em>, and <em>regulates</em> relationships in the ontology (also see the <a routerLink="/documentation/gene-page-gene-ontology">PomBase GO documentation</a>. GO search results for Biological Process terms therefore include genes involved in the process and its regulation.</p>
<p>Genes annotated directly to the process can be distinguished from those annotated to regulation on the <a routerLink="/documentation/ontology-term-page">ontology term page</a> for the term.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-search-or-browse-go-annotations', 'How can I search or browse GO annotations?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'how-can-i-search-or-browse-go-annotations' || pageName == 'all-faqs'">
<h3 id="how-can-i-search-or-browse-go-annotations">How can I search or browse GO annotations?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>You can search for GO terms by name or ID in the PomBase <a href="/query">advanced search</a>, and retrieve a list of all genes annotated to the term and its descendants via the relations <em>is_a</em>, <em>part_of</em>, <em>regulates</em>, <em>positively_regulates,</em> and <em>negatively_regulates</em>. For example, a search for “cytokinesis” will include genes annotated to “regulation of cytokinesis”. (See the GO documentation on <a href="http://geneontology.org/docs/ontology-relations/">Ontology Structure and Ontology Relations</a> for more information.)</p>
<p><em>S. pombe</em> GO annotations are also available in browsers that use the GO repository, notably <a href="http://amigo.geneontology.org/">AmiGO</a> and <a href="http://www.ebi.ac.uk/QuickGO/">QuickGO</a>. Both browsers have extensive documentation available:</p>
<ul>
<li><a href="http://wiki.geneontology.org/index.php/AmiGO_2_Manual:_Overview">AmiGO Manual</a></li>
<li><a href="http://www.ebi.ac.uk/QuickGO/help">QuickGO Help page</a></li>
</ul>
<p>Hint: to find <em>S. pombe</em> annotations, use Organism: Schizosaccharomyces pombe, Taxon: 4896 (<em>Schizosaccharomyces pombe</em>) or Source: PomBase. You can download the results in GAF format.</p>
<p>In PomBase, GO term names and IDs on gene pages link to ontology term pages for GO terms, which in turn offer links to AmiGO, QuickGO and BioPortal.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-search-s.-cerevisiae-ortholog-s-homolog-s-s.-pombe-gene', 'How can I search for the S. cerevisiae ortholog(s)/homolog(s) of an S. pombe gene?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'orthology' || pageName == 'finding-data' || pageName == 'how-can-i-search-s.-cerevisiae-ortholog-s-homolog-s-s.-pombe-gene' || pageName == 'all-faqs'">
<h3 id="how-can-i-search-s.-cerevisiae-ortholog-s-homolog-s-s.-pombe-gene">How can I search for the <em>S. cerevisiae</em> ortholog(s)/homolog(s) of an <em>S. pombe</em> gene?</h3>
<!-- pombase_categories: Orthology,Finding data -->
<p>In PomBase, <em>S. cerevisiae</em> orthologs are curated for <em>S. pombe</em> genes as described in the <a routerLink="/documentation/orthologs">Orthologs documentation</a>.</p>
<p>To find <em>S. pombe</em> orthologs for a budding yeast gene, you can search for the systematic name (ORF name) of the <em>S. cerevisiae</em> gene in the simple search box in the page header. For example, <em>S. cerevisiae</em> <em>LRP1</em> has the systematic name YHR081W, and a search on this in PomBase will retrieve the <em>S. pombe</em> gene <em>cti1</em>. Note that only systematic names can be searched for <em>S. cerevisiae</em>, to avoid confusion in cases where unrelated genes coincidentally have the same name in <em>S. pombe</em> and <em>S. cerevisiae</em>. To find systematic names of <em>S. cerevisiae</em> genes, you can search <a href="https://www.yeastgenome.org/">SGD</a>.</p>
<p>Also see the <a routerLink="/faq/how-do-i-receive-updates-list-s.-cerevisiae-and-s.-pombe-orthologs">FAQ on downloading the full set of orthologs</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-see-nucleotide-level-similarity-between-s.-pombe-and-other-schizosaccharomyces-species', 'How can I see nucleotide-level similarity between S. pombe and other Schizosaccharomyces species?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'orthology' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'how-can-i-see-nucleotide-level-similarity-between-s.-pombe-and-other-schizosaccharomyces-species' || pageName == 'all-faqs'">
<h3 id="how-can-i-see-nucleotide-level-similarity-between-s.-pombe-and-other-schizosaccharomyces-species">How can I see nucleotide-level similarity between <em>S. pombe</em> and other <em>Schizosaccharomyces</em> species?</h3>
<!-- pombase_categories: Orthology,Finding data,Tools and resources -->
<p>At present there are two ways to view nucleotide-level similarity between <em>S. pombe</em> and <em>S. japonicus</em>, <em>S. octosporus</em>, or <em>S. cryophilus</em>. Both use the Ensembl genome browser.</p>
<ol type="1">
<li>To view nucleotide similarity data tracks in the browser, Go to your region of interest in the browser (e.g. follow the link from the External references section of a gene page). Click “Configure this page” button in the left-hand bar. A popup box will appear. Select the data type “Comparative Genomics” on the left of the popup, then choose one or more of the “LASTz net” tracks.</li>
<li>Display syntenic regions as follows:
<ol type="1">
<li>Again, go to the region of interest. Make sure the “Location” tab is selected in the horizontal set of tabs along the top.</li>
<li>In the left-hand menu, find the “Comparative Genomics” heading, and click on “Region Comparison”.</li>
<li>To select a species for comparison, go to the bottom of the left-hand menu, and click the “Select species or regions” link (it may appear to be subtly blinking; we apologise for this anomaly).</li>
<li>In the popup, click the “+” beside any species in the “Unselected species or regions” list to move it to the “Selected species or regions” list. Note: “lastz” is the nucleotide alignment algorithm used. Close the popup - click the tick/check mark in the upper right corner, or click outside the popup.</li>
<li>Synteny views will now be visible in the bottom-most graphical display (scroll down if necessary). For any region in the <em>S. pombe</em> genome, pink tracks show the region in the second genome with the best nucleotide alignment. Green bands connect the best-aligned regions to highlight synteny.</li>
</ol></li>
</ol>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-show-or-hide-tracks-genome-browser', 'How can I show or hide tracks in the genome browser?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'genome-browser' || pageName == 'how-can-i-show-or-hide-tracks-genome-browser' || pageName == 'all-faqs'">
<h3 id="how-can-i-show-or-hide-tracks-genome-browser">How can I show or hide tracks in the genome browser?</h3>
<!-- pombase_categories: Finding data,Genome browser -->
<p>In JBrowse, click the “Select tracks” box in the upper left corner to display the list of available tracks. The list can be filtered using the items in the left-hand column, or searched. Tick boxes to enable tracks, then click the “Back to browser” button.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-submit-high-throughput-data-pombase', 'How can I submit high-throughput data to PomBase?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'data-submission-and-formats' || pageName == 'genome-browser' || pageName == 'how-can-i-submit-high-throughput-data-pombase' || pageName == 'all-faqs'">
<h3 id="how-can-i-submit-high-throughput-data-pombase">How can I submit high-throughput data to PomBase?</h3>
<!-- pombase_categories: Data submission and formats,Genome browser -->
<p>PomBase welcomes large sets of published data. The recommended submission route depends on the data type:</p>
<ul>
<li>For any data that can be associated with genome sequence coordinates (e.g. gene expression, ChIP-seq protein localisation, variation, etc.), we can add data tracks to JBrowse. Please see the <a href="/documentation/data-submission-form-for-HTP-sequence-linked-data">HTP data submission page</a> for instructions and submission templates.</li>
<li>Several types of data associated with genes can be displayed on gene pages. PomBase has developed bulk upload formats for phenotype, modification, and gene expression data – see the <a href="/submit-data/bulk-annotation">Annotation data submission page</a> and the <a href="/faq/what-file-formats-can-i-use-submit-high-throughput-data">FAQ on file formats</a> for links to the file format descriptions and data submission forms.</li>
</ul>
<p>If you have any other type of large-scale data – or if you have problems or questions regarding the available submission forms – please contact the <a href="mailto:helpdesk@pombase.org">helpdesk</a>.</p>
<!-- use the <a routerLink="/submit-data/data-submission-form">data submission form for HTP sequence-linked data</a>. -->

</div>
<div (click)="faq_navigate($event, '/faq/how-can-i-use-go-slims-s.-pombe', 'How can I use GO slims with S. pombe?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-statistics-and-lists' || pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'how-can-i-use-go-slims-s.-pombe' || pageName == 'all-faqs'">
<h3 id="how-can-i-use-go-slims-s.-pombe">How can I use GO slims with S. pombe?</h3>
<!-- pombase_categories: Genome statistics and lists,Tools and resources,Using ontologies -->
<p>The <a routerLink="/documentation/pombase-go-slim-documentation">Fission Yeast GO slim pages</a> provide generic GO slims for <em>S. pombe</em>, and show total genes annotated to each term directly or to any of its descendants.</p>
<p>If you want GO slim annotations for your own list of <em>S. pombe</em> genes, use the advanced search “Gene names and IDs” option, and then use the “Slim” button on the search results page. See the <a href="documentation/advanced-search">advanced search documentation</a> for more information.</p>
<p>If you want to use a different slim, we recommend using the <a href="http://go.princeton.edu/cgi-bin/GOTermMapper">GO Term Mapper</a> at Princeton. Upload your list of genes, and select “Schizosaccharomyces pombe (PomBase)” in the Organism pulldown. For the slim, select one of the radio buttons, or scroll down to Advanced Options to choose GO terms to make up a custom slim. GO Term Mapper’s interface and documentation should make the rest straightforward, but let PomBase staff know if you have any problems.</p>
<p>For further information on using the generic <em>S. pombe</em> slim, or on creating your own GO slim, please see the <a routerLink="/browse-curation/fission-yeast-go-slimming-tips">Fission Yeast GO slimming tips</a> page.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-do-i-cite-data-pombase-how-do-i-cite-genome-sequence-comparison-data-etc.', 'How do I cite data from PomBase? How do I cite the genome sequence, comparison data, etc.?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'tools-and-resources' || pageName == 'how-do-i-cite-data-pombase-how-do-i-cite-genome-sequence-comparison-data-etc.' || pageName == 'all-faqs'">
<h3 id="how-do-i-cite-data-pombase-how-do-i-cite-genome-sequence-comparison-data-etc.">How do I cite data from PomBase? How do I cite the genome sequence, comparison data, etc.?</h3>
<!-- pombase_categories: Tools and resources -->
<p>See the <a routerLink="/about/citing-pombase">Citing PomBase</a> page, which lists papers to cite for PomBase, the <em>S. pombe</em> genome sequence, Canto, FYPO, annotations and Compara. Additional key papers may be added as needed.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-do-i-receive-updates-list-s.-cerevisiae-and-s.-pombe-orthologs', 'How do I receive updates to the list of S. cerevisiae and S. pombe orthologs?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-statistics-and-lists' || pageName == 'orthology' || pageName == 'how-do-i-receive-updates-list-s.-cerevisiae-and-s.-pombe-orthologs' || pageName == 'all-faqs'">
<h3 id="how-do-i-receive-updates-list-s.-cerevisiae-and-s.-pombe-orthologs">How do I receive updates to the list of <em>S. cerevisiae</em> and <em>S. pombe</em> orthologs?</h3>
<!-- pombase_categories: Genome statistics and lists,Orthology -->
<p>The current version of the manually curated list of orthologs and orthologous groups identified between fission and budding yeast is available for download from the <a href="ftp://ftp.pombase.org/pombe/orthologs/">orthologs</a> FTP directory (linked from the <a routerLink="/datasets">Datasets</a> page).</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-do-you-determine-gene-s-full-length-transcript-utr-coordinates-transcription-start-and-end-sites', 'How do you determine a genes full-length transcript? / UTR coordinates? / transcription start and end sites?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-statistics-and-lists' || pageName == 'finding-data' || pageName == 'how-do-you-determine-gene-s-full-length-transcript-utr-coordinates-transcription-start-and-end-sites' || pageName == 'all-faqs'">
<h3 id="how-do-you-determine-gene-s-full-length-transcript-utr-coordinates-transcription-start-and-end-sites">How do you determine a gene’s full-length transcript? / UTR coordinates? / transcription start and end sites?</h3>
<!-- pombase_categories: Genome statistics and lists,Finding data -->
<p>PomBase curators have chosen default UTR features using three data sources and a set of precedence criteria:</p>
<ol type="1">
<li>Highest priority is given to data from low-throughput “conventional” experiments performed on individual mRNAs and reported in publications or submitted to EMBL. Where low-throughput data are not available, one of three high-throughput datasets is used.</li>
<li>The Broad data published in 2011 by Rhind <em>et al.</em> (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21511999">PMID:21511999</a>) is given precedence because it is the most recent, is higher resolution and detected splicing within the UTRs. <strong>Note</strong>: This study used a “greedy” algorithm to determine the longest possible transcript from transcriptome reads, which may result in the prediction of longer UTRs than are actually present. Use these data with caution, and refer to the transcript profiling data in the genome browser for genes of interest.</li>
<li>For genes not covered by (1) or (2), start/end data from Lantermann <em>et al.</em> (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=20118936">PMID:20118936</a>) based on transcriptome data from Dutrow <em>et al.</em> (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a>) are used where available.</li>
<li>For genes not covered by (1), (2) or (3), we use data from Wilhelm <em>et al.</em> (<a href="http://www.ncbi.nlm.nih.gov/pubmed/18488015">PMID:18488015</a>).</li>
</ol>
<p>More information is available in the descriptions of two HTP datasets originally sent to pombelist (<a routerLink="/documentation/Broad_dataset_email">Broad</a> and <a routerLink="/documentation/Lanterman_Dutrow_dataset_email">Lanterman/Dutrow</a>.</p>
<p>Transcript start and end coordinates from all sources will be available as individual data tracks in the Ensembl genome browser in the near future, which will allow you to view and evaluate them. PomBase will also curate splice and transcript variants as data become available.</p>
<!-- remove this when ticket #350 done -->
<p>Note: exon and CDS coordinates are temporarily unavailable on PomBase gene pages.</p>

</div>
<div (click)="faq_navigate($event, '/faq/how-many-conserved-unstudied-proteins-are-there-how-list-generated', 'How many conserved unstudied proteins are there? How is the list generated?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-statistics-and-lists' || pageName == 'how-many-conserved-unstudied-proteins-are-there-how-list-generated' || pageName == 'all-faqs'">
<h3 id="how-many-conserved-unstudied-proteins-are-there-how-list-generated">How many conserved unstudied proteins are there? How is the list generated?</h3>
<!-- pombase_categories: Genome statistics and lists -->
<p>As genes are annotated, each is assigned a status, as described on the <a routerLink="/status/gene-characterisation">Gene characterisation status</a> page. Taxonomic conservation of a gene is assigned manually on a case-by-case basis, taking into account multiple criteria. Additional information is available from PomBase curators upon request.</p>
<p>Genes listed on the <a routerLink="/status/priority-unstudied-genes">Priority unstudied genes</a> page are those that have “conserved unknown” characterisation status and the “conserved in vertebrates” taxonomic distribution.</p>
<p>You can also use the <a routerLink="/query">advanced search</a> to find conserved unstudied genes as described in the FAQs on <a routerLink="/faq/what-does-characterisation-status-mean-gene">characterisation status</a> and <a routerLink="/faq/can-i-search-genes-based-on-conservation-different-taxa">taxonomic conservation</a>. Start by searching for Characterisation status “conserved unknown”, and refine the search by adding a Taxonomic conservation query if you wish.</p>

</div>
<div (click)="faq_navigate($event, '/faq/i-found-discrepancy-between-genbank-embl-ena-or-ddbj-entry-and-sequence-pombase.-what-should-i-do', 'I found a discrepancy between a GenBank (EMBL/ENA or DDBJ) entry and a sequence in Pombase. What should I do?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'i-found-discrepancy-between-genbank-embl-ena-or-ddbj-entry-and-sequence-pombase.-what-should-i-do' || pageName == 'all-faqs'">
<h3 id="i-found-discrepancy-between-genbank-embl-ena-or-ddbj-entry-and-sequence-pombase.-what-should-i-do">I found a discrepancy between a GenBank (EMBL/ENA or DDBJ) entry and a sequence in Pombase. What should I do?</h3>
<!-- pombase_categories: Finding data -->
<p>We recommend using only the genome sequence, either from PomBase <a routerLink="/downloads/genome-datasets">downloadable files</a> or from the <a routerLink="/faq/how-can-i-retrieve-gene-sequence-including-upstream-and-downstream-sequences">sequence retrieval tools</a> on the gene pages and in the genome browser. Although there are some sequence updates still pending, the genome sequence is more accurate than individual gene sequences that predate the genome.</p>
<p>Many older <em>S. pombe</em> sequence submissions to the DNA databases (International Nucleotide Sequence Database Collaboration databases, i.e. ENA, GenBank, DDBJ) contain one or more errors (sometimes with an error rate as high as 20%), and we do not have the resources to maintain past sequences or flag every error in PomBase.</p>

</div>
<div (click)="faq_navigate($event, '/faq/s.-pombe-transcriptome-available-fasta-format', 'Is the S. pombe transcriptome available in FASTA format?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 's.-pombe-transcriptome-available-fasta-format' || pageName == 'all-faqs'">
<h3 id="s.-pombe-transcriptome-available-fasta-format">Is the <em>S. pombe</em> transcriptome available in FASTA format?</h3>
<!-- pombase_categories: Finding data,Tools and resources -->
<p>There is no single transcriptome sequence file available from PomBase at present. Several transcriptomic data sets are available as <a routerLink="/faq/how-can-i-show-or-hide-tracks-genome-browser">tracks</a> in the PomBase <a href="https://www.pombase.org/jbrowse/">genome browser</a>. The GFF3 genome feature files available from the <a routerLink="/downloads/genome-datasets">Genome Datasets</a> page include the coordinates of the <a routerLink="/faq/how-do-you-determine-gene-s-full-length-transcript-utr-coordinates-transcription-start-and-end-sites">annotated full-length transcript features</a>.</p>
<p>The bioinformatically inclined can also use the Ensembl Genomes REST API to retrieve transcript feature coordinates. (Note that, while the sequence has not changed recently, annotation data are likely to be out of date relative to the PomBase web site Ensembl Genomes is updated much less frequently than PomBase.) The <a routerLink="/faq/there-any-programmatic-access-pombase-data">FAQ on programmatic access to PomBase</a> provides an introduction to using the API, some pombe-specific examples, and links to additional documentation.</p>
<ul>
<li>For transcript coordinates, first retrieve all stable IDs using this URL:
<ul>
<li><a href="http://rest.ensemblgenomes.org/lookup/genome/schizosaccharomyces\_pombe?content-type=application/json" class="uri">http://rest.ensemblgenomes.org/lookup/genome/schizosaccharomyces\_pombe?content-type=application/json</a></li>
</ul></li>
<li>For each ID in the list, retrieve details via URLs in this format:
<ul>
<li><a href="http://rest.ensemblgenomes.org/lookup/id/SPBC11B10.09?content-type=application/json;expand=1" class="uri">http://rest.ensemblgenomes.org/lookup/id/SPBC11B10.09?content-type=application/json;expand=1</a>;</li>
</ul></li>
<li>Use your preferred scripting method to iterate through the ID list and parse the desired coordinates from the data obtained via the second URL.</li>
</ul>
<p>The Broad Institute has <a href="http://www.broadinstitute.org/ftp/pub/annotation/fungi/schizosaccharomyces/">archived genomic data files</a> for the <em>Schizosaccharomyces</em> species, including transcript files.</p>

</div>
<div (click)="faq_navigate($event, '/faq/there-any-programmatic-access-pombase-data', 'Is there any programmatic access to PomBase data?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'tools-and-resources' || pageName == 'there-any-programmatic-access-pombase-data' || pageName == 'all-faqs'">
<h3 id="there-any-programmatic-access-pombase-data">Is there any programmatic access to PomBase data?</h3>
<!-- pombase_categories: Tools and resources -->
<p>The Ensembl Genomes REST API Endpoints page provides a REST-ful interface allows language-independent programmatic access to all genomes accessible through Ensembl Genomes, including the same Schizosaccharomyces pombe genome sequence data available in PomBase. (Note that, while the sequence has not changed recently, annotation data are likely to be out of date relative to the PomBase web site Ensembl Genomes is updated much less frequently than PomBase.) The REST interface provides data in a variety of formats including GFF3, FASTA and JSON. Data types accessible via this interface include:</p>
<ul>
<li>genomic features, including genes and CDSs</li>
<li>genomic and protein sequences</li>
<li>cross-references including ontologies</li>
<li>gene trees and orthologues</li>
</ul>
<p>In addition, the interface also provides access to the Variant Effect Predictor tool and a tool for mapping genomic coordinates between different versions of genome assemblies.</p>
<p>The <a href="https://github.com/Ensembl/ensembl-rest/wiki">user guide</a> provides comprehensive descriptions of interface functionality, plus examples using a variety of languages and interfaces. The following URLs are examples specific to <em>S. pombe</em>:</p>
<ul>
<li>retrieve the cDNA sequence of the specified gene in FASTA: <a href="http://rest.ensembl.org/sequence/id/SPAC2F7.03c?content-type=text/x-fasta;type=cdna" class="uri">http://rest.ensembl.org/sequence/id/SPAC2F7.03c?content-type=text/x-fasta;type=cdna</a></li>
<li>retrieve list of homologues from the pan-taxonomic comparative &lt;database for the specified gene in JSON format: http://rest.ensembl.org/homology/id/SPAC2F7.03c?content-type=application/json&amp;format=condensed&amp;compara=pan_homology&gt;</li>
</ul>

</div>
<div (click)="faq_navigate($event, '/faq/there-any-rdna-repeat-sequences-pombase', 'Are there any rDNA repeat sequences in PomBase?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-browser' || pageName == 'finding-data' || pageName == 'there-any-rdna-repeat-sequences-pombase' || pageName == 'all-faqs'">
<h3 id="there-any-rdna-repeat-sequences-pombase">Are there any rDNA repeat sequences in PomBase?</h3>
<!-- pombase_categories: Genome browser,Finding data -->
<p>The reference genome sequence excludes most of the ribosomal DNA (rDNA) repeats, which are present in two tandem arrays on chromosome III. These arrays are estimated to be 1225 kb and 240 kb in size for the sequenced strain (972 h-). The reference sequence includes two partial and one complete representative rDNA repeats:</p>
<ul>
<li>At the left-hand end a partial repeat contains <a routerLink="/gene/SPRRNA.42">SPRRNA.42</a> (partial 18S gene)</li>
<li>A complete repeat contains <a routerLink="/gene/SPRRNA.47">SPRRNA.47</a> (28S), <a routerLink="/gene/SPRRNA.50">SPRRNA.50</a> (5.8S), and <a routerLink="/gene/SPRRNA.43">SPRRNA.43</a> (18S)</li>
<li>A partial repeat at the right-hand end contains <a routerLink="/gene/SPRRNA.45">SPRRNA.45</a></li>
</ul>
<p>The complete repeat sequence coordinates are <a href="https://www.pombase.org/jbrowse/?loc=III%3A5540..13720&amp;tracks=DNA%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features&amp;highlight=">Chromosome 3:5542-13722</a> (note that the reverse strand is transcribed). The link goes to the PomBase genome browser, where you can view and <a routerLink="/faq/how-can-i-retrieve-sequence-region-using-sequence-coordinates">download</a> the sequence. Because the reverse strand is transcribed, you may want to choose “-1” in the location settings.</p>
<p>Also see the FAQ on <a routerLink="/faq/how-can-i-find-rrna-genes">finding rRNA genes</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/there-equivalent-artemis-java-applet-pombase', 'Is there an equivalent to the Artemis java applet in PomBase?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'there-equivalent-artemis-java-applet-pombase' || pageName == 'all-faqs'">
<h3 id="there-equivalent-artemis-java-applet-pombase">Is there an equivalent to the Artemis java applet in PomBase?</h3>
<!-- pombase_categories: Finding data,Tools and resources -->
<p>No; at present only the JBrowse and links to the Ensembl genome browser are available via the PomBase web site. (In the future we may investigate the possibility of adding an Artemis applet to PomBase, and will update this FAQ accordingly when it becomes available.)</p>
<p>If you want to browse the <em>S. pombe</em> genome in the Artemis environment, it is fairly easy to download and run locally:</p>
<ul>
<li><p><em>Important:</em> For <em>S. pombe</em> we recommend using <a href="ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/v16/">Artemis Version 16</a>, not the most recent version available from the <a href="http://www.sanger.ac.uk/science/tools/artemis">Artemis home page</a>. Newer versions of Artemis may be incompatible with the default Java installed on some computers, especially Macs.</p></li>
<li><p>Read in the chromosome contig files available from the <a routerLink="/downloads/genome-datasets">Genome sequences</a> page. (You can read in one or more files in a session.)</p></li>
</ul>
<p>Once you have loaded the file(s), you can do many different things, e.g.:</p>
<ul>
<li>Find features by name or ID</li>
<li>Find all features of a given type (e.g. see the “<a routerLink="/faq/how-can-i-find-transposons-s.-pombe-genome">can I find transposons</a>” FAQ)</li>
<li>Find matches to a specific nucleotide sequence (e.g. see the “<a routerLink="/faq/can-i-generate-comprehensive-restriction-enzyme-map-genome-pombase">restriction enzyme map</a>” FAQ)</li>
<li>View the nucleotide or amino acid sequence of a region or feature</li>
<li>Export selected sequences</li>
</ul>
<p>See the <a href="ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/artemis.pdf">Artemis manual</a> (pdf; Sanger site) for additional information.</p>

</div>
<div (click)="faq_navigate($event, '/faq/there-list-drug-targets-s.-pombe', 'Is there a list of drug targets in S. pombe?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-statistics-and-lists' || pageName == 'finding-data' || pageName == 'there-list-drug-targets-s.-pombe' || pageName == 'all-faqs'">
<h3 id="there-list-drug-targets-s.-pombe">Is there a list of drug targets in <em>S. pombe</em>?</h3>
<!-- pombase_categories: Genome statistics and lists,Finding data -->
<p>The “<a routerLink="/browse-curation/drugs-known-pombe-targets">Drugs with knowns S. pombe targets</a>” page lists drugs that have been shown to affect <em>S. pombe</em>, with brief summaries of their targets.</p>
<p>If you notice any errors or omissions on this page, or can provide any supporting references, please email the helpdesk.</p>

</div>
<div (click)="faq_navigate($event, '/faq/there-list-protein-complexes-s.-pombe-and-their-subunits', 'Is there a list of protein complexes in S. pombe, and their subunits?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'genome-statistics-and-lists' || pageName == 'using-ontologies' || pageName == 'there-list-protein-complexes-s.-pombe-and-their-subunits' || pageName == 'all-faqs'">
<h3 id="there-list-protein-complexes-s.-pombe-and-their-subunits">Is there a list of protein complexes in <em>S. pombe</em>, and their subunits?</h3>
<!-- pombase_categories: Finding data,Genome statistics and lists,Using ontologies -->
<p>Yes, there is a file that lists GO macromolecular complex assignments for fission yeast gene products in the FTP directory:</p>
<p>ftp://ftp.pombase.org/pombe/annotations/Gene_ontology/GO_complexes/</p>
<p>Note that the complex inventory includes the RNA subunits of ribonucleoprotein complexes. There is some redundancy in the list, because some gene products are annotated to both complexes and subcomplexes. For example, subunits of the DASH complex (<a routerLink="/term/GO:0042729">GO:0042729</a>) are annotated to ‘condensed chromosome outer kinetochore’ (<a routerLink="/term/GO:0000940">GO:0000940</a>) as well as GO:0042729. Additional notes are available in a README file: ftp://ftp.pombase.org/pombe/annotations/Gene_ontology/GO_complexes/README</p>
<p>Also see the <a routerLink="/faq/how-can-i-find-protein-localization-data">FAQ on localization</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/what-annotation-extension', 'What is an annotation extension?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'using-ontologies' || pageName == 'what-annotation-extension' || pageName == 'all-faqs'">
<h3 id="what-annotation-extension">What is an annotation extension?</h3>
<!-- pombase_categories: Using ontologies -->
<p>Annotation extensions can be used with annotations to terms from various ontologies, such as GO, FYPO, modifications, etc. Extensions provide additional specificity to the annotation by linking the term to another ontology term or a gene product via a relationship.</p>
<p>Extensions are most commonly used with GO annotations, where they can be used to capture details such as substrates of molecular functions or cell cycle phases during which a localization is observed. More information is available in the <a routerLink="/documentation/gene-page-gene-ontology">gene page GO annotation documentation</a>.</p>
<p>The GO Consortium provides further information on annotation extensions in its <a href="http://geneontology.org/docs/go-annotation-file-gaf-format-2.1/">file format guide</a>, on a <a href="http://wiki.geneontology.org/index.php/Annotation_Extension">wiki page</a>, and in publications from <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885854">2014</a> and <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27812947">2017</a>. PomBase converts many extension names to more human-friendly text, as described <a routerLink="/documentation/annotation-extension-relation-display">here</a>.</p>
<p>Phenotype annotations using FYPO may have extensions that capture severity or penetrance, or identify a gene or gene product used in an assay, as described in the <a routerLink="/documentation/gene-page-phenotypes">gene page phenotype documentation</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/what-bam-format', 'What is BAM format?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'data-submission-and-formats' || pageName == 'what-bam-format' || pageName == 'all-faqs'">
<h3 id="what-bam-format">What is BAM format?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>BAM is a binary file format used for nucleotide sequence alignment data.</p>
<p>The <a href="http://samtools.github.io/hts-specs/SAMv1.pdf">file format specification</a> (PDF) is available from the <a href="http://samtools.sourceforge.net/">SAMtools</a> web site.</p>
<p>The <a href="http://genome.ucsc.edu/FAQ/FAQformat.html#format5.1">UCSC Genome Bioinformatics FAQ</a> and the <a href="http://www.broadinstitute.org/software/igv/BAM">Broad Institute file format guide</a> provide additional information.</p>

</div>
<div (click)="faq_navigate($event, '/faq/what-bed-format', 'What is BED format?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'data-submission-and-formats' || pageName == 'what-bed-format' || pageName == 'all-faqs'">
<h3 id="what-bed-format">What is BED format?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>BED is a tab-delimited text format that defines a feature track for a genome browser.</p>
<p>BED format is described in the <a href="http://genome.ucsc.edu/FAQ/FAQformat.html#format1">UCSC Genome Bioinformatics FAQ</a>, and the <a href="http://www.broadinstitute.org/software/igv/BED">Broad Institute file format guide</a> provides additional information.</p>

</div>
<div (click)="faq_navigate($event, '/faq/what-bedgraph-format', 'What is bedGraph format?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'data-submission-and-formats' || pageName == 'what-bedgraph-format' || pageName == 'all-faqs'">
<h3 id="what-bedgraph-format">What is bedGraph format?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>BedGraph is a file format that allows display of continuous-valued data in a track in genome browsers that support the format.</p>
<p>At present, JBrowse does not support bedGraph, so we cannot use data in this format for PomBase. If you have data in bedGraph format, we recommend converting to WIG or bigWig format.</p>
<p>BedGraph format is described at the <a href="http://genome.ucsc.edu/goldenPath/help/bedgraph.html">UCSC Genome Bioinformatics web site</a>, and the <a href="http://www.broadinstitute.org/software/igv/bedgraph">Broad Institute file format guide</a> provides additional information.</p>

</div>
<div (click)="faq_navigate($event, '/faq/what-bigbed-format', 'What is bigBed format?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'data-submission-and-formats' || pageName == 'what-bigbed-format' || pageName == 'all-faqs'">
<h3 id="what-bigbed-format">What is bigBed format?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>BigBed is a binary file format that is created by conversion from <a routerLink="/faq/what-bed-format">BED</a>, and thus stores similar types of data for display in a genome browser track.</p>
<p>BigBed format is described at the <a href="http://genome.ucsc.edu/goldenPath/help/bigBed.html">UCSC Genome Bioinformatics web site</a>, and the <a href="http://www.broadinstitute.org/software/igv/bigbed">Broad Institute file format guide</a> provides additional information.</p>

</div>
<div (click)="faq_navigate($event, '/faq/what-bigwig-format', 'What is bigWig format?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'data-submission-and-formats' || pageName == 'what-bigwig-format' || pageName == 'all-faqs'">
<h3 id="what-bigwig-format">What is bigWig format?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>BigWig is a file format for display of dense, continuous data in a genome browser track, created by conversion from Wiggle (<a routerLink="/faq/what-wig-format">WIG</a>) format.</p>
<p>BigWig format is described at the <a href="http://genome.ucsc.edu/goldenPath/help/bigWig.html">UCSC Genome Bioinformatics web site</a>, and the <a href="http://www.broadinstitute.org/software/igv/bigwig">Broad Institute file format guide</a> provides additional information.</p>

</div>
<div (click)="faq_navigate($event, '/faq/what-does-characterisation-status-mean-gene', 'What does the &quot;characterisation status&quot; mean for a gene?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-statistics-and-lists' || pageName == 'what-does-characterisation-status-mean-gene' || pageName == 'all-faqs'">
<h3 id="what-does-characterisation-status-mean-gene">What does the “characterisation status” mean for a gene?</h3>
<!-- pombase_categories: Genome statistics and lists -->
<p>Each gene is assigned exactly one characterisation status that reflects how much is known about the gene, whether it is conserved, etc. Specific status descriptions:</p>
<ul>
<li><strong>Published</strong>: Completely or partially characterised in a small scale experiment, with some published information about the biological role (corresponding to any of the <a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">fission yeast GO biological process slim</a> biological process terms)</li>
<li><strong>Biological role inferred</strong>: A biological role (as above, a <a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">fission yeast GO biological process slim</a> term) is inferred from homology to an experimentally characterised gene product</li>
<li><strong>Conserved unknown</strong>: Conserved outside the <em>Schizosaccharomyces</em>, but nothing known about the biological role in any organism</li>
<li><strong><em>Schizosaccharomyces</em> specific protein, uncharacterised</strong>: Unpublished and found only in fission yeast (<em>S. pombe</em>, <em>S. octosporus</em>, <em>S. japonicus</em>, <em>S. cryophilus</em>); nothing known about biological role. May be single copy or a member of a multi-member family.</li>
<li><strong><em>S. pombe</em> specific protein, uncharacterised</strong>: Unpublished and found only in <em>S. pombe</em> (not detected in other <em>Schizosaccharomyces</em> species); nothing known about biological role</li>
<li><strong>Transposon</strong>: Known or predicted transposable element</li>
<li><strong>Dubious</strong>: Unlikely to be protein coding</li>
</ul>
<p>A current summary of <a routerLink="/status/gene-characterisation">gene characterisation status for the <em>S. pombe</em> genome</a> is available, as well as a table of <a routerLink="/status/gene-characterisation-statistics-history">historical characterisation status</a> counts.</p>
<p>You can also retrieve current lists of genes with each characterisation status using the <a routerLink="/query">advanced search</a>. Select the Characterisation status query, then choose a status from the pulldown menu, and submit.</p>

</div>
<div (click)="faq_navigate($event, '/faq/what-file-formats-can-i-use-submit-high-throughput-data', 'What file formats can I use to submit high-throughput data?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'data-submission-and-formats' || pageName == 'what-file-formats-can-i-use-submit-high-throughput-data' || pageName == 'all-faqs'">
<h3 id="what-file-formats-can-i-use-submit-high-throughput-data">What file formats can I use to submit high-throughput data?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>At present, PomBase can host any types of data that can be connected with sequence features or coordinates, and can display the data as tracks in the genome browser. We accept data in any of several formats. To choose a file format for your data, consult the table below and the linked FAQs.</p>
<p>Also please see the <a href="/documentation/data-submission-form-for-HTP-sequence-linked-data">HTP data submission page</a> for instructions and submission templates, and <a href="mailto:helpdesk@pombase.org">consult the helpdesk</a> to submit a file or if you need further assistance.</p>
<!-- <a routerLink="/submit-data/data-submission-form">accept data</a> in any of several formats. -->
<table>
<thead>
<tr class="header">
<th>File format</th>
<th>Recommended for</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><a routerLink="/faq/what-bam-format">BAM</a></td>
<td>sequence alignments, especially from high-throughput experiments such as RNAseq</td>
</tr>
<tr class="even">
<td><a routerLink="/faq/what-bed-format">BED</a></td>
<td>sequence features with coordinates</td>
</tr>
<tr class="odd">
<td><a routerLink="/faq/what-bigbed-format">bigBed</a></td>
<td>sequence features with coordinates</td>
</tr>
<tr class="even">
<td><a routerLink="/faq/what-bigwig-format">bigWig</a></td>
<td>values attached to genome locations/regions</td>
</tr>
<tr class="odd">
<td><a routerLink="/faq/what-gff">GFF3</a></td>
<td>sequence features with coordinates</td>
</tr>
<tr class="even">
<td><a routerLink="/faq/what-vcf">VCF</a></td>
<td>structural variations, such as SNPs, insertions, deletions, or copy number variants</td>
</tr>
<tr class="odd">
<td><a routerLink="/faq/what-wig-format">WIG</a></td>
<td>values attached to genome locations/regions</td>
</tr>
</tbody>
</table>
<!-- <a routerLink="/faq/what-psl-format">PSL</a> | sequence alignments -->
<p>We can also accept batch submissions of certain types of data that appear on PomBase gene pages. For these data types, we use dedicated PomBase-specific formats as shown in the table:</p>
<table>
<thead>
<tr class="header">
<th>Data type</th>
<th>File format description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Phenotypes</td>
<td><a routerLink="/documentation/phenotype-data-bulk-upload-format">phenotype file format</a></td>
</tr>
<tr class="even">
<td>Modifications</td>
<td><a routerLink="/documentation/modification-data-bulk-upload-format">modification file format</a></td>
</tr>
<tr class="odd">
<td>Qualitative gene expression</td>
<td><a routerLink="/documentation/qualitative-data-gene-expression-bulk-upload-format">qualitative gene expression file format</a></td>
</tr>
<tr class="even">
<td>Quantitative gene expression</td>
<td><a routerLink="/documentation/quantitative-data-gene-expression-bulk-upload-format">quantitative gene expression file format</a></td>
</tr>
</tbody>
</table>
<p>We may be able to accept data in other text formats. Please enquire via the PomBase <a href="mailto:helpdesk@pombase.org">helpdesk</a> if you have any questions about your data format.</p>

</div>
<div (click)="faq_navigate($event, '/faq/what-gff', 'What is GFF3?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'data-submission-and-formats' || pageName == 'what-gff' || pageName == 'all-faqs'">
<h3 id="what-gff">What is GFF3?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>Generic Feature Format Version 3 (GFF3) is a tab-delimited text file format used to represent genomic sequence features.</p>
<p>PomBase produces GFF3 <a href="ftp://ftp.pombase.org/pombe/genome_sequence_and_features/gff3/">files of <em>S. pombe</em> sequence features</a>, and accepts high-throughput data submissions in this format.</p>
<p>The <a href="https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3">file format specification</a> is available from the <a href="http://www.sequenceontology.org/">Sequence Ontology</a> GitHub site. Validation tools are available from various online providers.</p>

</div>
<div (click)="faq_navigate($event, '/faq/what-go-term-enrichment-how-can-i-do-it-my-genes', 'What is GO term enrichment? How can I do it for my genes?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'tools-and-resources' || pageName == 'using-ontologies' || pageName == 'what-go-term-enrichment-how-can-i-do-it-my-genes' || pageName == 'all-faqs'">
<h3 id="what-go-term-enrichment-how-can-i-do-it-my-genes">What is GO term enrichment? How can I do it for my genes?</h3>
<!-- pombase_categories: Finding data,Tools and resources,Using ontologies -->
<p>“GO term enrichment” refers to analysing a gene list by finding GO terms that are significantly over- or under-represented among the annotations for the genes. Finding GO terms that are shared by genes in your list can help you find out what they have in common biologically.</p>
<p>PomBase does not have its own GO enrichment tool, but we recommend one, and provide a bit more information, in the <a routerLink="/faq/how-can-i-find-significant-shared-go-annotations-genes-list">FAQ on GO term enrichment</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/what-psl-format', 'What is PSL format?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'data-submission-and-formats' || pageName == 'what-psl-format' || pageName == 'all-faqs'">
<h3 id="what-psl-format">What is PSL format?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>PSL is a tab-delimited text format that represents sequence alignments.</p>
<p>PSL format is described in the <a href="http://genome.ucsc.edu/FAQ/FAQformat.html#format2">UCSC Genome Bioinformatics FAQ</a>, and the <a href="http://www.broadinstitute.org/software/igv/node/171">Broad Institute file format guide</a> provides additional information.</p>

</div>
<div (click)="faq_navigate($event, '/faq/what-vcf', 'What is VCF?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'data-submission-and-formats' || pageName == 'what-vcf' || pageName == 'all-faqs'">
<h3 id="what-vcf">What is VCF?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>Variant Call Format (VCF) is a text file format used to describe structural variations, such as SNPs, insertions, deletions, or copy number variants.</p>
<p>The <a href="https://github.com/samtools/hts-specs">file format specification</a> is available from GitHub (see entries with “VCF” in the name).</p>
<p>The <a href="http://www.internationalgenome.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41">1000 Genomes site</a>, <a href="http://genome.ucsc.edu/FAQ/FAQformat.html#format10.1">UCSC Genome Bioinformatics FAQ</a> and the <a href="http://www.broadinstitute.org/software/igv/VCF">Broad Institute file format guide</a> provide additional information.</p>

</div>
<div (click)="faq_navigate($event, '/faq/what-wig-format', 'What is WIG format?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'data-submission-and-formats' || pageName == 'what-wig-format' || pageName == 'all-faqs'">
<h3 id="what-wig-format">What is WIG format?</h3>
<!-- pombase_categories: Data submission and formats -->
<p>Wiggle (WIG) is a file format for display of continuous-value data in a genome browser track.</p>
<p>BigWig format is described at the <a href="http://genome.ucsc.edu/goldenPath/help/wiggle.html">UCSC Genome Bioinformatics web site</a>, and the <a href="http://www.broadinstitute.org/software/igv/WIG">Broad Institute file format guide</a> provides additional information.</p>

</div>
<div (click)="faq_navigate($event, '/faq/when-was-genomic-sequence-last-updated', 'When was the genomic sequence last updated?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-statistics-and-lists' || pageName == 'when-was-genomic-sequence-last-updated' || pageName == 'all-faqs'">
<h3 id="when-was-genomic-sequence-last-updated">When was the genomic sequence last updated?</h3>
<!-- pombase_categories: Genome statistics and lists -->
<p>The reference sequence was last updated in January 2007; only feature coordinates and annotation have changed since then. See <a routerLink="/status/sequence-updates">Sequence updates</a> and <a routerLink="/status/sequence-updates-pending">Pending sequence updates</a> for more information.</p>

</div>
<div (click)="faq_navigate($event, '/faq/where-can-i-download-genome-sequence', 'Where can I download the genome sequence?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'where-can-i-download-genome-sequence' || pageName == 'all-faqs'">
<h3 id="where-can-i-download-genome-sequence">Where can I download the genome sequence?</h3>
<!-- pombase_categories: Finding data -->
<p>Genome sequence files can be downloaded from the <a routerLink="/downloads/genome-datasets">Genome sequence and features</a> page in several different formats.</p>

</div>
<div (click)="faq_navigate($event, '/faq/where-can-i-find-basic-statistics-on-s.-pombe-genome-such-as-genome-size-total-number-genes-mean-intergenic-distance-etc.', 'Where can I find basic statistics on the S. pombe genome, such as genome size, total number of genes, mean intergenic distance, etc.?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-statistics-and-lists' || pageName == 'where-can-i-find-basic-statistics-on-s.-pombe-genome-such-as-genome-size-total-number-genes-mean-intergenic-distance-etc.' || pageName == 'all-faqs'">
<h3 id="where-can-i-find-basic-statistics-on-s.-pombe-genome-such-as-genome-size-total-number-genes-mean-intergenic-distance-etc.">Where can I find basic statistics on the <em>S. pombe</em> genome, such as genome size, total number of genes, mean intergenic distance, etc.?</h3>
<!-- pombase_categories: Genome statistics and lists -->
<p>A <a routerLink="/status/statistics">page of statistics</a> is available, but note that it was last updated in January 2017.</p>

</div>
<div (click)="faq_navigate($event, '/faq/where-can-i-find-information-about-pombase-data-versions', 'Where can I find information about PomBase data versions?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'genome-statistics-and-lists' || pageName == 'where-can-i-find-information-about-pombase-data-versions' || pageName == 'all-faqs'">
<h3 id="where-can-i-find-information-about-pombase-data-versions">Where can I find information about PomBase data versions?</h3>
<!-- pombase_categories: Genome statistics and lists -->
<p>PomBase annotations are updated each time the daily upload to the preview site (behind the scenes) succeeds, with snapshots archived on a monthly basis. To refer to PomBase in your publications, we recommend citing the date on which you view or download data. If you download an archived snapshot, cite its date.</p>
<p>Prior to the September 2017 PomBase upgrade, periodic data releases were flagged with version numbers and release dates. Each legacy version number has two parts, of which the first is the Ensembl Genomes (EG) version and the second is the version of curated PomBase annotations (sequence features, ontology annotations, etc.). For example, PomBase version 20_39 used EG version 20 and PomBase annotation data version 39. The <a routerLink="/about/version-history">Data version history</a> page shows additional information about the versions of various data and software portions of the legacy PomBase releases.</p>

</div>
<div (click)="faq_navigate($event, '/faq/why-are-go-annotations-different-between-pombase-and-uniprot-goa', 'Why are GO annotations different between PomBase and UniProt/GOA?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'why-are-go-annotations-different-between-pombase-and-uniprot-goa' || pageName == 'all-faqs'">
<h3 id="why-are-go-annotations-different-between-pombase-and-uniprot-goa">Why are GO annotations different between PomBase and UniProt/GOA?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>The GO annotations available from PomBase (gene pages, advanced search, etc.) and the GO Consortium site (AmiGO; GO downloads) differ from those available from the UniProt GOA site (including QuickGO) for three main reasons:</p>
<ol type="1">
<li>RNA - PomBase provides GO annotations for functional RNAs (e.g. rRNA, tRNA, snRNA), but at present the UniProt GOA dataset only includes annotations for protein-coding genes.</li>
<li>Time lag - <em>S. pombe</em> GO data are updated at the same time on the PomBase and GO Consortium sites, but the UniProt GOA site may be up to a few weeks behind.</li>
<li>Filtering - PomBase does not include automated annotations that are redundant with manual annotations (contact the <a href="mailto:helpdesk@pombase.org">helpdesk</a> for further details). The GO Consortium site uses the same filtered annotation dataset as PomBase, whereas the UniProt GOA site includes the automated annotations.</li>
</ol>

</div>
<div (click)="faq_navigate($event, '/faq/why-are-go-terms-missing-downloadable-annotation-file', 'Why are GO terms missing from the downloadable annotation file?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'finding-data' || pageName == 'using-ontologies' || pageName == 'why-are-go-terms-missing-downloadable-annotation-file' || pageName == 'all-faqs'">
<h3 id="why-are-go-terms-missing-downloadable-annotation-file">Why are GO terms missing from the downloadable annotation file?</h3>
<!-- pombase_categories: Finding data,Using ontologies -->
<p>The downloadable file of <a href="/downloads/go-annotations">PomBase GO annotations</a> is in the GO Consortium’s <a href="http://geneontology.org/docs/go-annotation-file-gaf-format-2.1/">GAF format</a>, and only includes “direct” annotations, i.e. the actual term-gene product connections made by manual curation or computational transfer. When an annotation is made to a term, the gene product is automatically inferred to be annotated to all the “ancestor” terms in the ontology. These inferred annotations are used in PomBase web pages and searches, but are not included in the GAF file. More information on ontology structure and annotation inference is available in documentation at <a routerLink="/documentation/gene-page-gene-ontology">PomBase</a> and GO )<a href="http://geneontology.org/docs/ontology-documentation/">ontology</a> and <a href="http://geneontology.org/docs/go-annotations/">annotation</a>.</p>
<p>When you use GO annotations in any analysis, we strongly recommend using tools that take ontology structure and transitive inference of annotations into account.</p>
<!-- uncomment when GAF download goes live: -->
<!-- Annotations downloadable for search results are also in GAF format, so the same considerations apply. -->

</div>
<div (click)="faq_navigate($event, '/faq/why-are-some-genes-abnormal-phenotype-annotated-corresponding-go-process-while-others-are-not', 'Why are some genes with an abnormal phenotype annotated to the corresponding GO process while others are not?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'using-ontologies' || pageName == 'why-are-some-genes-abnormal-phenotype-annotated-corresponding-go-process-while-others-are-not' || pageName == 'all-faqs'">
<h3 id="why-are-some-genes-abnormal-phenotype-annotated-corresponding-go-process-while-others-are-not">Why are some genes with an abnormal phenotype annotated to the corresponding GO process while others are not?</h3>
<!-- pombase_categories: Using ontologies -->
<p>PomBase curators use GO Biological Process annotations to indicate that a gene product is directly involved in a process or its regulation. FYPO annotations indicate when a mutation in a gene causes a change in a process, but do not say whether the effect is direct or indirect.</p>
<p>Many mutant phenotypes reflect downstream effects of compromising an upstream process. In these cases, we annotate the phenotypes using FYPO terms, but do not annotate to the GO corresponding biological process term. We use “regulation of biological process” GO terms in cases where there is evidence for a gene playing a regulatory role in wild-type cells, but not where defects in an upstream process affect a downstream process (even though the latter is sometimes described as “regulating” or “modulating” the downstream process).</p>
<p>For example, a defect in cellular respiration may arise from mutations in genes directly involved in respiration, but also as a downstream effect of mutations in genes involved in mitochondrial translation, respiratory chain complex assembly, or ubiquinone biosynthesis. Similarly, DNA replication defects often also lead to defects in chromosome segregation; for the genes involved we annotate both replication and segregation phenotypes, but only replication in GO biological process.</p>
<p>The cell cycle offers an even more dramatic example of why we restrict usage of GO annotations. Over 750 genes can be mutated to give an elongated vegetative cell phenotype, which is traditionally interpreted as indicating that cell cycle progression is blocked in interphase. Most of these genes, however, are involved in transcription, translation, transport or splicing, and cell cycle delays seen in mutants are due to activation of cell cycle checkpoints by the abnormal processes. To annotate all 750 genes to “regulation of mitotic cell cycle” would obscure the genes that actually are part of the cell cycle regulatory network, greatly reducing the usefulness and precision of GO annotations.</p>
<p>Also see the <a routerLink="/faq/how-can-i-identify-all-genes-affect-process">FAQ on finding genes that affect a process</a>.</p>

</div>
<div (click)="faq_navigate($event, '/faq/why-are-some-genes-annotated-both-viable-and-inviable-phenotypes', 'Why are some genes annotated to both viable and inviable phenotypes?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'gene-page' || pageName == 'using-ontologies' || pageName == 'why-are-some-genes-annotated-both-viable-and-inviable-phenotypes' || pageName == 'all-faqs'">
<h3 id="why-are-some-genes-annotated-both-viable-and-inviable-phenotypes">Why are some genes annotated to both viable and inviable phenotypes?</h3>
<!-- pombase_categories: Gene page,Using ontologies -->
<p>One gene can be correctly annotated to both a “viable” term and an “inviable” term from FYPO, under certain circumstances:</p>
<ul>
<li>Different alleles may have different phenotypes; e.g., a deletion may be inviable, but a point mutation may be fully viable or conditionally lethal.</li>
<li>One allele may cause death under some, but not all, conditions.</li>
<li>An allele may cause only some cells in a population to die (this would be annotated using an “inviable cell” term, with an extension to indicate incomplete penetrance (“low” or “medium”), plus an annotation to a “viable cell population” term).</li>
<li>Cells that can divide for a few generations but then die are annotated as inviable, but can acquire suppressor mutations at a high enough frequency for populations to appear viable.</li>
</ul>
<p>At present, alleles cannot be queried directly in the PomBase advanced search, but the FYPO phenotype filters do allow you to retrieve annotations for all alleles, or to restrict to null expression (deletions etc.) or overexpression of the wild-type allele. Comparing results with and without the allele restrictions may help resolve apparent discrepancies.</p>
<p>Note that it not yet possible to search for specific conditions, or for penetrance, but we plan to add these features to the Advanced Search.</p>
<p>If, however, the allele and condition details are identical, annotation to both viable and inviable terms is probably an error (either one of the terms is wrong, or there are missing or incorrect details for the alleles and/or conditions). Please let us know via the <a href="mailto:helpdesk@pombase.org">helpdesk</a> if you notice any potential errors.</p>

</div>
<div (click)="faq_navigate($event, '/faq/why-do-gene-pages-only-show-curated-orthologs-human-and-s.-cerevisiae', 'Why do gene pages only show curated orthologs for human and S. cerevisiae?')" *ngIf="pageName == 'all-frequently-asked-questions' || pageName == 'gene-page' || pageName == 'genome-statistics-and-lists' || pageName == 'orthology' || pageName == 'why-do-gene-pages-only-show-curated-orthologs-human-and-s.-cerevisiae' || pageName == 'all-faqs'">
<h3 id="why-do-gene-pages-only-show-curated-orthologs-human-and-s.-cerevisiae">Why do gene pages only show curated orthologs for human and <em>S. cerevisiae</em>?</h3>
<!-- pombase_categories: Gene page,Genome statistics and lists,Orthology -->
<p>In PomBase, human and <em>S. cerevisiae</em> orthologs are manually curated for <em>S. pombe</em> genes as described in the <a routerLink="/documentation/orthologs">Orthologs documentation</a>. Because manual ortholog curation is extremely time-consuming, it is not done for any species other than human and <em>S. cerevisiae</em>. For automated ortholog prediction of orthologs in other species please see the <a routerLink="/faq/how-can-i-find-s.-pombe-orthologs-species-other-than-human-and-s.-cerevisiae">relevant FAQ</a>.</p>
<p>In the future we will add a tree view of consensus orthologs in key species to the gene pages.</p>

</div>
  </div>
</div>
<div *ngIf="section == 'help'">
<app-page-contents-menu title="Help" titleRoute="/help">

<div class="left-menu-part left-menu-item"><a routerLink="/faq">Frequently Asked Questions (FAQ)</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation">PomBase Documentation</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/about/contacts">Contact Curators</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'index'">
<h1 id="help">Help</h1>
<h3 id="frequently-asked-questions-faq"><a routerLink="/faq">Frequently Asked Questions (FAQ)</a></h3>
<h3 id="pombase-documentation"><a routerLink="/documentation">PomBase Documentation</a></h3>
<h3 id="contact-curators"><a routerLink="/about/contacts">Contact Curators</a></h3>

  </div>
  </div>
</div>
<div *ngIf="section == 'news'">
<app-page-contents-menu title="News" titleRoute="/news">

<div class="left-menu-part left-menu-item"><a pageScroll href="#published-go-term-matrix-annotation-qc">Published: GO Term Matrix for annotation QC</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombetalks-september-nd">PombeTalks September 2nd</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#th-pombe-meeting-postponed">11th Pombe meeting postponed to 2022</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombetalks-august-th">PombeTalks August 19th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombetalks-august-th">PombeTalks August 5th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#fission-yeast-micropublications">Fission yeast microPublications</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombetalks-july-nd">PombeTalks July 22nd</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombetalks-july-th">PombeTalks July 8th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombetalks-june-th">PombeTalks June 24th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#editable-pombase-query-names">Editable PomBase query names</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombetalks-postponed-june-th">PombeTalks POSTPONED to June 17th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombetalks-may-th">PombeTalks May 27th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#new-fission-yeast-go-slims">New fission yeast GO slims</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#new-ontology-slimming-options-advanced-search-results">New ontology slimming options for advanced search results</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-adopts-mondo-disease-gene-curation">PomBase adopts MONDO for disease gene curation</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#improved-gene-disease-curation">Improved gene-disease curation</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#published-community-curation-pombase">Published: Community curation in PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombetalks-may-th">PombeTalks May 13th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#chromatin-silencing-ontology-amp-annotation-overhaul">Chromatin silencing ontology &amp; annotation overhaul</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-now-uses-interpro-version-.">PomBase now uses InterPro Version 79.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#mitochondrial-genome-update">Mitochondrial genome update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#midori-harris-receives-biocuration-career-award">Midori Harris receives 2020 Biocuration Career Award</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombetalks-start-on-april-th">PombeTalks start on April 29th</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#angeli-update">AnGeLi update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#new-search-result-download-options">New search result download options</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-now-uses-interpro-version-.">PomBase now uses InterPro Version 77.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#community-curation-response-rate-reaches">Community curation response rate reaches 50%</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#a-quarter-million-annotations">A quarter of a million annotations</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#query-phenotypes-conditions">Query phenotypes for conditions</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#levures-mod-les-et-outils-th-international-conference">Levures, Modèles et Outils 14th International Conference</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-now-elixir-node-service">PomBase is now an ELIXIR Node Service</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#new-vectors-simple-reliable-s.-pombe-molecular-biology">New vectors for simple, reliable *S. pombe* molecular biology</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#unique-permanent-urls-search-results">Unique permanent URLs for search results</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#browser-tracks-now-loadable-publication-pages">Browser tracks now loadable from publication pages</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#new-pombase-funding-wellcome-trust">New PomBase funding from the Wellcome Trust</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-now-uses-interpro-version-.">PomBase now uses InterPro Version 76.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#replication-origin-data-loaded-into-jbrowse">Replication origin data loaded into JBrowse</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#join-conversations-on-slack">Join the conversations on Slack</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#fitness-landscape-fission-yeast-genome-dataset-loaded-into-jbrowse">"Fitness Landscape of the Fission Yeast Genome" dataset loaded into JBrowse</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#trna-metabolism-go-annotation-update">tRNA metabolism GO annotation update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#customisable-fasta-download">Customisable FASTA download</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#new-homology-modelling-and-ortholog-links-on-gene-pages">New homology modelling and ortholog links on gene pages</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-interpro-update">PomBase InterPro Update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#s.-pombe-included-gene-info-browser-extension">*S. pombe* included in Gene Info browser extension</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#integrated-motif-search">Integrated motif search</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#icygmb-registration-open">ICYGMB 2019  - registration open</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#go-slim-any-s.-pombe-gene-list">GO slim for any *S. pombe* gene list</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#south-eastern-regional-yeast-meeting-serym-registration-open">South Eastern Regional Yeast Meeting (SERYM) - registration open</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#international-cell-cycle-meeting-registration-open">International Cell Cycle meeting - registration open</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#published-hidden-plain-sight-what-remains-be-discovered-eukaryotic-proteome">Published: Hidden in plain sight: What remains to be discovered in the eukaryotic proteome?</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombe-registration-open">Pombe 2019 - registration open</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#val-wood-wins-biocuration-society-award">Val Wood wins Biocuration society award</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#improved-disease-association-dataset-released">Improved disease association dataset released</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#mitochondrial-go-annotation-update">Mitochondrial GO annotation update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#new-nucleosome-occupancy-maps-loaded">New nucleosome occupancy maps loaded</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#see-your-genes-quilt">See your genes in a QuiLT</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#fission-yeast-transmembrane-transport-overhaul">Fission yeast transmembrane transport overhaul</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#hidden-plain-sight-what-remains-be-discovered-eukaryotic-proteome">Hidden in plain sight: What remains to be discovered in the eukaryotic proteome?</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-your-pocket">PomBase in your pocket</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#celebrating-years-go">Celebrating 20 years of GO</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#rna-central-and-pombase">RNA central and PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#new-pombase-genomic-region-graphics">New PomBase genomic region graphics</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-nar-database-issue">PomBase NAR Database Issue</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#fungal-pathogen-genomics-course-registration-open">Fungal Pathogen Genomics Course 2019 - registration open</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#transcript-tracks-atkinson-et-al.-loaded">Transcript tracks from Atkinson et al. (2018) loaded</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombelist-changes">pombelist changes</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#new-genome-browser-tracks">New genome browser tracks</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#new-book-chapter-about-pombase">New book chapter about PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-releases-jbrowse">PomBase releases JBrowse</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#in-memory-andr-eacute-goffeau">In memory of Andr&eacute; Goffeau</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#congratulations-gsa-award-winners">Congratulations to GSA award winners</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#new-improved-pombase-goes-live">New, improved PomBase goes live</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#th-international-fission-yeast-meeting-early-registration-closes-soon">9th International Fission Yeast Meeting - early registration closes soon</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#th-international-fission-yeast-meeting-registration-open">9th International Fission Yeast Meeting registration open</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#show-your-support-database-funding">Show your support for database funding</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#our-model-organism-database-commentary">Our model organism database commentary</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#new-advanced-search-features">New Advanced Search features</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#new-genetics-primer-fission-yeast">New genetics primer for fission yeast</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update-viability-summary-correction-alert">PomBase data update; viability summary correction alert</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update-multi-allele-phenotypes">PomBase data update with multi-allele phenotypes</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#canto-downtime-new-version">Canto downtime & new version</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombe-travel-fellowships">Pombe 2015 travel fellowships</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombe-poster-abstract-deadline-extended">Pombe 2015 poster abstract deadline extended</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombe-abstract-deadline-approaching">Pombe 2015 abstract deadline approaching</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombe-registration-now-open">Pombe 2015 registration now open</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#new-compact-go-annotation-display">New compact GO annotation display</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#esyn-network-visualizations-pombase">esyN network visualizations in PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#gene-ontology-publication-on-annotation-extensions">Gene Ontology publication on annotation extensions</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#human-ortholog-data-correction-coming-next-month">Human ortholog data correction coming next month</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#meeting-mini-reviews-published">2013 meeting mini-reviews published</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-survey-results-available">PomBase survey results available</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#new-target-gene-page-section">New "Target Of" gene page section</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-user-survey-open">PomBase User Survey open</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#send-htp-data-pombase">Send HTP data to PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#connecting-pombase">Connecting With PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombelist-has-moved">pombelist has moved</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-website-update">PomBase website update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-launches-community-curation">PomBase launches community curation</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-data-update">PomBase data update</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#genedb-s.-pombe-decommissioned">GeneDB S. pombe decommissioned</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#quantitative-gene-expression-data-available-pombase">Quantitative gene expression data available in PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#carl-singer-foundation-established">Carl Singer Foundation Established</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombe-registration-abstracts-by-mon-th-april">Pombe 2013: registration & abstracts by Mon 8th April</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#data-update-on-pombase-web-site">Data update on PomBase web site</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#speed-improvements-and-new-data-on-pombase-web-site">Speed improvements and new data on PomBase web site</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#new-data-and-new-features-on-pombase-web-site">New data and new features on PomBase web site</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-web-site-fully-live">PomBase web site fully live</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-preview-launch">PomBase preview launch</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#pombase-nar-paper-published-online">PomBase NAR paper published online</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#genedb-s.-pombe-now-uses-latest-release-pfam-protein-family-database-.-.">GeneDB (S. pombe) now uses the latest release of the Pfam protein family database(25.0).</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#schizosaccharomyces-comparative-genome-paper-published">Schizosaccharomyces Comparative Genome Paper Published</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#genome-reappraisal-reveals-new-genes-and-revised-gene-structures">Genome reappraisal reveals new genes and revised gene structures</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#annotated-transcription-start-and-termination-sites-fission-yeast">Annotated transcription start and termination sites for fission yeast</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#analysis-fission-yeast-deletion-publication">Analysis of Fission Yeast Deletion Publication</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#funding-pombase">Funding for PomBase</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#fission-yeast-one-key-organisms-reference-genomes-project">Fission yeast is one of the 12 key organisms of the reference genomes project</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#genedb-s.-pombe-now-uses-latest-update-pfam-release-.">GeneDB (S. pombe) now uses the latest update to Pfam, release 24.0</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#fission-yeast-ensembl-fungi">Fission yeast in Ensembl Fungi</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#genedb-now-using-version-pfam-protein-family-database.">GeneDB is now using Version 23 of the Pfam protein family database.</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#s.-pombe-genedb-now-includes-deep-links-biological-general-repository-interaction-datasets-biogrid">S. pombe GeneDB now includes "deep links" to the Biological General Repository for Interaction Datasets (BioGRID)</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#global-sequence-and-chip-study-examines-eukaryotic-transcription">Global sequence and chip study examines eukaryotic transcription</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#the-h-mating-type-region-has-been-provided">The h- mating type region has been provided</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#baumann-and-zakian-labs-identify-telomerase-rna">Baumann and Zakian labs identify telomerase RNA</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#wellcome-trust-advanced-course-genome-wide-approaches-fission-yeast">Wellcome Trust Advanced Course 'Genome-wide approaches with fission yeast</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#th-international-fission-yeast-meeting">4th International Fission Yeast Meeting</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#genedb-representation-fission-yeast-data-moved-contigs-chromosomes">GeneDB representation of the fission yeast data moved from contigs to chromosomes</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#yeast-special-issue-european-fission-yeast-meeting">Yeast Special Issue from the 2006 European Fission Yeast Meeting</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#the-first-fission-yeast-whole-proteome-localization-study-now-published">The first fission yeast whole proteome localization study is now published</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#fission-yeast-database-survey">Fission yeast database survey</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#european-fission-yeast-meeting">European Fission Yeast Meeting</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#comparative-genomics-eukaryotic-microorganisms">Comparative Genomics of Eukaryotic Microorganisms</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#second-east-coast-regional-pombe-meeting">Second East Coast Regional pombe Meeting</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#general-repository-interaction-datasets">General Repository for Interaction Datasets</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#the-third-international-fission-yeast-meeting">The Third International Fission Yeast Meeting</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#methods-volume-issue">Methods Volume 33 Issue 3</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#recent-genome-wide-surveys">Recent Genome wide surveys</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#the-new-fission-yeast-book-now-published">The 'new' fission yeast book is now published</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#schizosaccharomyces-pombe-essential-genes-pilot-study">Schizosaccharomyces pombe Essential Genes: A pilot Study</a></div>
<div class="left-menu-part left-menu-item"><a pageScroll href="#global-transcriptional-responses-fission-yeast-environmental-stress">Global transcriptional responses of fission yeast to environmental stress</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'global-transcriptional-responses-fission-yeast-environmental-stress'">
<h3 id="global-transcriptional-responses-of-fission-yeast-to-environmental-stress">Global transcriptional responses of fission yeast to environmental stress</h3>
<p>Chen D, Toone WM, Mata J, Lyne R, Burns G, Kivinen K, Brazma A, Jones N, Bähler J. Mol Biol Cell. 2003 Jan;14(1):214-29. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12529438">PMID:12529438</a></p>

  </div>
  <div *ngIf="pageName == 'schizosaccharomyces-pombe-essential-genes-pilot-study'">
<h3 id="schizosaccharomyces-pombe-essential-genes-a-pilot-study">Schizosaccharomyces pombe Essential Genes: A pilot Study</h3>
<p>Decottignies A, Sanchez-Perez I, Nurse P Genome Res. 2003 Mar;13(3):399-406. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12618370">PMID:12618370</a></p>

  </div>
  <div *ngIf="pageName == 'new-fission-yeast-book-now-published'">
<h3 id="the-new-fission-yeast-book-is-now-published">The ‘new’ fission yeast book is now published</h3>
<p>Egel, R., Copenhagen, Denmark (Ed.) The Molecular Biology of Schizosaccharomyces pombe Genetics, Genomics and Beyond ISBN:3-540-00693-1</p>

  </div>
  <div *ngIf="pageName == 'recent-genome-wide-surveys'">
<h3 id="recent-genome-wide-surveys">Recent Genome wide surveys</h3>
<p>Correlations Between Gene Expression and Gene Conservation in Fission Yeast. Mata J, Bahler J. Genome Res. 2003 Nov 12 <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14613978">PMID:14613978</a><br />
<br />
FELINES: a utility for extracting and examining EST-defined introns and exons. Drabenstot SD et al Nucleic Acids Res. 2003 Nov 15;31(22):e141. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14602934">PMID:14602934</a><br />
<br />
Genome-wide distribution of DNA replication origins at A+T-rich islands in Schizosaccharomyces pombe. Segurado M, De Luis A, Antequera F. EMBO Rep. 2003 Nov;4(11):1048-53. Epub 2003 Oct 17. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14566325">PMID:14566325</a><br />
<br />
Retrotransposons and their recognition of pol II promoters: a comprehensive survey… Bowen NJ et al Genome Res. 2003 Sep;13(9):1984-97. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12952871">PMID:12952871</a></p>

  </div>
  <div *ngIf="pageName == 'methods-volume-33-issue-3'">
<h3 id="methods-volume-33-issue-3">Methods Volume 33 Issue 3</h3>
<p>This issue of Methods includes 11 papers for fission yeast protocols including DNA damage checkpoint assays, cell wall analysis, TAP, nuclear envelope integrity assays, GFP imaging, TS mutant creation and plasmid use and construction. See the <a href="http://www.sciencedirect.com/science/journal/10462023/33/3">Methods site</a> or the <a routerLink="/fission-yeast-papers-methods-volume-33-issue-3">Table of Contents</a> (at PomBase)<a href="/fission-yeast-papers-methods-volume-33-issue-3"></a> for details of the papers including PMIDs.</p>

  </div>
  <div *ngIf="pageName == 'third-international-fission-yeast-meeting'">
<h3 id="the-third-international-fission-yeast-meeting">The Third International Fission Yeast Meeting</h3>
<p>The meeting was held at UC San Diego on August 24-29, 2004.</p>

  </div>
  <div *ngIf="pageName == 'general-repository-interaction-datasets'">
<h3 id="general-repository-for-interaction-datasets">General Repository for Interaction Datasets</h3>
<p>A project to record published genetic and physical interactions is underway with Mike Tyers and the GRID group at Toronto.</p>

  </div>
  <div *ngIf="pageName == 'second-east-coast-regional-pombe-meeting'">
<h3 id="second-east-coast-regional-pombe-meeting">Second East Coast Regional pombe Meeting</h3>
<p>Second East Coast Regional pombe Meeting<br />
<br />
This meeting took place from November 11-13, 2005 in Miami Beach, Florida.</p>

  </div>
  <div *ngIf="pageName == 'comparative-genomics-eukaryotic-microorganisms'">
<h3 id="comparative-genomics-of-eukaryotic-microorganisms">Comparative Genomics of Eukaryotic Microorganisms</h3>
<p>Comparative Genomics of Eukaryotic Microorganisms:<br />
Eukaryotic Genome Evolution, Approaches with Yeasts and Fungi</p>
<p><br />
This conference took place from 12th-17th November 2005 in Sant Feliu de Guixols, Spain. Full details can be found <a href="http://www.esf.org/index.php?id=940">here</a>.</p>

  </div>
  <div *ngIf="pageName == 'european-fission-yeast-meeting'">
<h3 id="european-fission-yeast-meeting">European Fission Yeast Meeting</h3>
<p>The European Fission Yeast Meeting (16th-18th March 2006) and The Fission Yeast Bioinformatics workshop (15th - 16th Mar 2006) both took place at the Wellcome Trust Genome Campus in Hinxton (Cambridge, UK).</p>

  </div>
  <div *ngIf="pageName == 'fission-yeast-database-survey'">
<h3 id="fission-yeast-database-survey">Fission yeast database survey</h3>
<p>The fission yeast database survey is now closed. You can view the survey results <a routerLink="/community/pombase_2006_survey_summary">here</a>.</p>

  </div>
  <div *ngIf="pageName == 'first-fission-yeast-whole-proteome-localization-study-now-published'">
<h3 id="the-first-fission-yeast-whole-proteome-localization-study-is-now-published">The first fission yeast whole proteome localization study is now published</h3>
<p>The first fission yeast whole proteome localization study is now published: <a href="http://www.nature.com/nbt/journal/v24/n7/abs/nbt1222.html">Matsuyama A. et al (2006): ORFeome cloning and global analysis of protein localization in the fission yeast Schizosaccharomyces pombe. Nat Biotech 24, 841-7.</a><br />
 </p>

  </div>
  <div *ngIf="pageName == 'yeast-special-issue-2006-european-fission-yeast-meeting'">
<h3 id="yeast-special-issue-from-the-2006-european-fission-yeast-meeting">Yeast Special Issue from the 2006 European Fission Yeast Meeting</h3>
<p>The October issue of the journal Yeast is a <a href="http://onlinelibrary.wiley.com/doi/10.1002/yea.v23:13/issuetoc">fission yeast special issue</a> containing 13 articles and reviews commissioned as a result of the European Fission Yeast Meeting, which are FREE to download.</p>

  </div>
  <div *ngIf="pageName == 'genedb-representation-fission-yeast-data-moved-contigs-chromosomes'">
<h3 id="genedb-representation-of-the-fission-yeast-data-moved-from-contigs-to-chromosomes">GeneDB representation of the fission yeast data moved from contigs to chromosomes</h3>
<p>GeneDB representation of the fission yeast data moved from contigs to chromosomes. See the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2007/000128.html">pombelist archive</a> for details.</p>

  </div>
  <div *ngIf="pageName == '4th-international-fission-yeast-meeting'">
<h3 id="th-international-fission-yeast-meeting">4th International Fission Yeast Meeting</h3>
<p>4th International Fission Yeast Meeting held in Copenhagen.</p>

  </div>
  <div *ngIf="pageName == 'wellcome-trust-advanced-course-genome-wide-approaches-fission-yeast'">
<h3 id="wellcome-trust-advanced-course-genome-wide-approaches-with-fission-yeast">Wellcome Trust Advanced Course ’Genome-wide approaches with fission yeast</h3>
<p>Wellcome Trust Advanced Course ‘Genome-wide approaches with fission yeast’ held in Hinxton.</p>

  </div>
  <div *ngIf="pageName == 'baumann-and-zakian-labs-identify-telomerase-rna'">
<h3 id="baumann-and-zakian-labs-identify-telomerase-rna">Baumann and Zakian labs identify telomerase RNA</h3>
<p>Baumann and Zakian labs identify elusive <a href="/spombe/result/SPNCRNA.214">telomerase RNA</a> (PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18157152">18157152</a> and PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18157149">18157149</a>)</p>

  </div>
  <div *ngIf="pageName == 'h-mating-type-region-has-been-provided'">
<h3 id="the-h--mating-type-region-has-been-provided">The h- mating type region has been provided</h3>
<p>The <a routerLink="/status/mating-type-region">h- mating type region</a> has been provided by Xavier Marsellach and Lorena Aguilar.</p>

  </div>
  <div *ngIf="pageName == 'global-sequence-and-chip-study-examines-eukaryotic-transcription'">
<h3 id="global-sequence-and-chip-study-examines-eukaryotic-transcription">Global sequence and chip study examines eukaryotic transcription</h3>
<p>Dynamic repertoire of the fission yeast transcriptome reveals: 94% of the genome is transcribed; extensive variation in different stages and conditions; global and condition-specific coupling between splicing efficiency and transcription; confirms the majority of introns; refines ~75 gene structures; identifies 453 new transcripts 26 of which were predicted to code for proteins.</p>

  </div>
  <div *ngIf="pageName == 's-pombe-genedb-now-includes-deep-links-biological-general-repository-interaction-datasets-biogr'">
<h3 id="s.-pombe-genedb-now-includes-deep-links-to-the-biological-general-repository-for-interaction-datasets-biogrid">S. pombe GeneDB now includes “deep links” to the Biological General Repository for Interaction Datasets (BioGRID)</h3>
<p><em>S. pombe</em> GeneDB now includes “deep links” to the Biological General Repository for Interaction Datasets (BioGRID) interaction datasets from the ‘Database Cross References’ section of the individual Gene Pages.</p>

  </div>
  <div *ngIf="pageName == 'genedb-now-using-version-23-pfam-protein-family-database'">
<h3 id="genedb-is-now-using-version-23-of-the-pfam-protein-family-database.">GeneDB is now using Version 23 of the Pfam protein family database.</h3>
<p>GeneDB is now using Version 23 of the Pfam protein family database. A total of 4154 (83%) S. pombe proteins now have at least one Pfam domain or family assignment (compared to 76% for S. cerevisiae), the highest percentage coverage for any eukaryote.</p>

  </div>
  <div *ngIf="pageName == 'fission-yeast-ensembl-fungi'">
<h3 id="fission-yeast-in-ensembl-fungi">Fission yeast in Ensembl Fungi</h3>
<p>The fission yeast genome and annotation dataset is now available as part of <a href="http://fungi.ensembl.org/index.html">Ensembl Fungi</a>.</p>

  </div>
  <div *ngIf="pageName == 'genedb-s-pombe-now-uses-latest-update-pfam-release-240'">
<h3 id="genedb-s.-pombe-now-uses-the-latest-update-to-pfam-release-24.0">GeneDB (S. pombe) now uses the latest update to Pfam, release 24.0</h3>
<p>GeneDB (S. pombe) now uses the latest update to Pfam, release 24.0 and 88.5% of fission yeast proteins now contain a match to at least one Pfam domain (increased from 83% in version 23).</p>

  </div>
  <div *ngIf="pageName == 'fission-yeast-one-12-key-organisms-reference-genomes-project'">
<h3 id="fission-yeast-is-one-of-the-12-key-organisms-of-the-reference-genomes-project">Fission yeast is one of the 12 key organisms of the reference genomes project</h3>
<p>Fission yeast is one of the 12 key organisms of the <a href="http://geneontology.org/GO.refgenome.shtml?all">reference genomes project</a>. The goal of this project is to completely annotate twelve reference genomes so that those annotations may be used to effectively seed the automatic annotation efforts of other genome.</p>

  </div>
  <div *ngIf="pageName == 'funding-pombase'">
<h3 id="funding-for-pombase">Funding for PomBase</h3>
<p>Funding was awarded by the Wellcome Trust for a fission yeast Model Organism Database, <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2010-March/002731.html">PomBase</a>.</p>

  </div>
  <div *ngIf="pageName == 'analysis-fission-yeast-deletion-publication'">
<h3 id="analysis-of-fission-yeast-deletion-publication">Analysis of Fission Yeast Deletion Publication</h3>
<p>The analysis of the fission yeast deletion collection is now published online in <a href="http://www.nature.com/nbt/journal/v28/n6/abs/nbt.1628.html">Nature Biotechnology</a>.</p>

  </div>
  <div *ngIf="pageName == 'annotated-transcription-start-and-termination-sites-fission-yeast'">
<h3 id="annotated-transcription-start-and-termination-sites-for-fission-yeast">Annotated transcription start and termination sites for fission yeast</h3>
<p>Further details are available on the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000814.html">pombe mailing list</a>.</p>

  </div>
  <div *ngIf="pageName == 'genome-reappraisal-reveals-new-genes-and-revised-gene-structures'">
<h3 id="genome-reappraisal-reveals-new-genes-and-revised-gene-structures">Genome reappraisal reveals new genes and revised gene structures</h3>
<p>Further information on the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000815.html">pombe mailing list</a>.</p>

  </div>
  <div *ngIf="pageName == 'schizosaccharomyces-comparative-genome-paper-published'">
<h3 id="schizosaccharomyces-comparative-genome-paper-published">Schizosaccharomyces Comparative Genome Paper Published</h3>
<p>A paper describing the major findings of the <em>Schizosaccharomyces</em> Comparative Genome Project was <a href="http://www.sciencemag.org/content/early/2011/04/20/science.1203357">published today</a> in Science Express and reported changes are included in GeneDB.</p>
<p>Further details are described in the pombe mailing list posts:</p>
<ul>
<li><a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000855.html">Schizosaccharomyces Comparative Genome Paper Published</a></li>
<li><a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000856.html">Import of the fission yeast revisions from the Broad Institute comparative genome paper into GeneDB</a>.</li>
</ul>

  </div>
  <div *ngIf="pageName == 'genedb-s-pombe-now-uses-latest-release-pfam-protein-family-database250'">
<h3 id="genedb-s.-pombe-now-uses-the-latest-release-of-the-pfam-protein-family-database25.0.">GeneDB (S. pombe) now uses the latest release of the Pfam protein family database(25.0).</h3>

  </div>
  <div *ngIf="pageName == 'pombase-nar-paper-published-online'">
<h3 id="pombase-nar-paper-published-online">PomBase NAR paper published online</h3>
<p>A paper describing <a href="http://www.pombase.org">PomBase</a> has been published online will be included in the 2012 Database Issue of <em>Nucleic Acids Research</em>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153">Abstract</a> and <a href="http://nar.oxfordjournals.org/content/early/2011/10/28/nar.gkr853">open access full text</a> are available.</p>

  </div>
  <div *ngIf="pageName == 'pombase-preview-launch'">
<h3 id="pombase-preview-launch">PomBase preview launch</h3>
<p>A preview of <a href="http://www.pombase.org">PomBase</a>, the new model organism database for the fission yeast <em>Schizosaccharomyces pombe</em>, has been <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000970.html">announced to the <em>S. pombe</em> community</a> for testing and feedback. For more on PomBase, see the NAR Database Issue paper (<a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153">PubMed abstract</a>) or <a href="/about/contacts">contact the PomBase staff</a>.</p>

  </div>
  <div *ngIf="pageName == 'pombase-web-site-fully-live'">
<h3 id="pombase-web-site-fully-live">PomBase web site fully live</h3>
<p>We are pleased to announce that the PomBase web site, <a href="http://www.pombase.org">www.pombase.org</a>, is now fully live; the preview phase has ended. The site has been updated with an assortment of new features, datatypes, and bug fixes.</p>
<p>More recent data, reflecting additions and changes through March 20, 2012, are now available on gene pages and in search results.</p>
<p>The updated site features a Gene List Search that provides behavior equivalent to GeneDB’s List Download. You can now type or paste lists into the Gene Systematic IDs and Gene Names filters, and use the Query History to combine a gene list search with other search options. For convenience, there is a direct link to a search page pre-configured to accept a list of systematic IDs available in the Find menu, on the Find page, and here: <a routerLink="/spombe/query/builder?filter=12">http://www.pombase.org/spombe/query/builder?filter=12</a></p>
<p>The Advanced Search also now offers:</p>
<ul>
<li>options to search GO, FYPO, and other ontologies by term name or ID;</li>
<li>autocomplete for ontology term name search;</li>
<li>ability to search for genes in a region, such as centromeres or telomeres;</li>
<li>improved organization of filter selections.</li>
</ul>
<p>We have also fixed a Sequence Download error reported by some users, so that the “CDS”, CDS + UTRs“, and”CDS + UTRs + Introns" options now retrieve the correct sequences.<br />
<br />
In addition, numerous minor improvements have been made. Please send any questions or comments on the PomBase web site to us at &lt;<a href="mailto:helpdesk@pombase.org" class="email">helpdesk@pombase.org</a>&gt;.</p>

  </div>
  <div *ngIf="pageName == 'new-data-and-new-features-pombase-web-site'">
<h3 id="new-data-and-new-features-on-pombase-web-site">New data and new features on PomBase web site</h3>
<p>We are pleased to announce that we have updated both data and web site features for PomBase.</p>
<p>Most importantly, we have added new data types, and upgraded the gene pages to display them.</p>
<p>We have also added more annotations of existing data types, bringing the web site content up to September 11, 2012. The new annotations include the first contributions to come in via the new community curation system, and we thank the researchers who are participating in the initial phase of community curation.</p>
<p><strong>New annotation types:</strong></p>
<ul>
<li>Phenotype annotations now use the Fission Yeast Phenotype Ontology (FYPO), and include allele details, expression levels, and experimental conditions. With FYPO, more detailed phenotypes can be described, and links between terms for related phenotypes support improved phenotype searches.</li>
<li>Many GO annotations now include “annotation extensions” that provide additional specificity. For example, extensions may capture the substrate of a catalytic activity, the cell cycle phase during which a function or process occurs, or any of several other types of supporting information for the annotation. Annotation extensions are described in more detail below.</li>
</ul>
<p>You can see these new data types on many gene pages, such as <a routerLink="/spombe/result/SPBC11B10.09">cdc2</a> or <a routerLink="/spombe/result/SPBC106.10">pka1</a>.</p>
<p><strong>New web site features:</strong></p>
<ul>
<li>Annotation display - Gene page GO and phenotype displays have been revamped to show new annotation types described above.</li>
<li>Ontology term pages - Each ontology term ID now links to pages with information about the term and lists of genes annotated to it.</li>
<li>Ontology graph links - GO and phenotype annotation sections now include links to graphical ontology displays in the genome browser.</li>
<li>Sequence highlighting - Sequence download now offers an option to show colour highlighting of regions such as UTRs, introns and exons.</li>
<li>Versions - Each gene page now shows the current data version in the format PomBase:x.y, where <em>x</em> is the Ensembl Genomes (EG) version, and <em>y</em> is the Chado version. The sequence, and sequence feature locations, remain stable within any EG version, whereas annotations change with each Chado update.</li>
<li>Protein family information is now included in the Protein Features gene page section.</li>
<li>The Protein Feature section includes a link to the Pfam entry for a protein.</li>
<li>Transcript source data (e.g. for UTR coordinates) is now displayed in the Transcript Features section.</li>
<li>A Documentation page contains links to relevant portions of the Ensembl Genomes documentation. (More documentation will be added over the coming months.)</li>
</ul>
<p><strong>What are annotation extensions?</strong></p>
<p>Annotation extensions are a form of supporting data that can be added GO annotations (or other ontology annotations) to capture additional details not provided by the ontology term itself.</p>
<p>The information in GO annotation extensions encompasses several effector-target relationships, such as</p>
<ul>
<li>localisation dependencies</li>
<li>substrates of functions, e.g. targets of a protein kinase – see the has_substrate extensions on Cdc2’s “protein serine/threonine kinase” (GO:0004674) annotations</li>
<li>activators and inhibitors</li>
<li>regulation targets of signalling pathways or transcription factors</li>
</ul>
<p>Additional extensions describe spatial and temporal aspects of processes. For example, several S. pombe annotations now include extensions that indicate in which phase of the cell cycle a gene product is found in a cellular component or involved in a process – see the pka1 annotations to “nucleus” (GO:0005634) and “cytoplasm” (GO:0005737).</p>
<p>You may also find the <a href="http://wiki.geneontology.org/index.php/Annotation_Extension">GO wiki page</a> on annotation extensions informative, although it is primarily aimed at curators.</p>
<p>Annotation extensions can also be used with phenotype annotations. The most common usage of phenotype annotation extensions is to capture which gene, protein, etc. was used in an assay. For example, the sam5 (G441E) mutation of <em>pka1</em> causes nuclear accumulation of Ste11. This is represented by annotation to the ontology term “nuclear protein accumulation” (FYPO:0000255), with the extension “assayed_using(PomBase:SPBC32C12.02)”. Extensions can also indicate expressivity or penetrance for a phenotype.</p>

  </div>
  <div *ngIf="pageName == 'speed-improvements-and-new-data-pombase-web-site'">
<h3 id="speed-improvements-and-new-data-on-pombase-web-site">Speed improvements and new data on PomBase web site</h3>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through December 17, 2012, and reflects complete curation of an additional 70 papers.<br />
<br />
We have also made some improvements “under the hood” that should make gene page loading much faster. Please let us know if you have any problems with gene pages loading slowly or incompletely, whether or not you have reported issues in the past.<br />
<br />
We are aware that there is an intermittent problem with the “Reference” column display in the data tables – sometimes a PubMed ID appears instead of an author name and year. This problem will be fixed as soon as possible. Please alert us if you notice anything else odd or wrong.</p>

  </div>
  <div *ngIf="pageName == 'data-update-pombase-web-site'">
<h3 id="data-update-on-pombase-web-site">Data update on PomBase web site</h3>
<p>We have once again updated the data available on the PomBase web site. The data now includes manual curation through March 6, 2013.<br />
<br />
We now expect to be able to update PomBase data every month, and will soon have an automated pipeline in place. We thank all of you for your patience during the long months when updates were infrequent.<br />
<br />
You should also see a few small improvements in the site:</p>
<ul>
<li>Ontology term pages now display the text definition for each term.</li>
<li>FASTA sequence retrieval should be quicker, and less likely to time out, for large gene lists.</li>
<li>There has been some tidying of the display of “extension” data for GO and phenotype annotations.</li>
</ul>
<p>Last month we noted an intermittent problem with the “Reference” column display in the data tables. The occurrence of this problem should now be greatly reduced, so please let us know if you see it recurring.<br />
<br />
As usual, please don’t hesitate to alert us of any other problems with data or site performance, or if you have any questions.<br />
<br />
</p>

  </div>
  <div *ngIf="pageName == 'pombe-2013-registration-abstracts-mon-8th-april'">
<h3 id="pombe-2013-registration-abstracts-by-mon-8th-april">Pombe 2013: registration &amp; abstracts by Mon 8th April</h3>
<pre><code>Dear Pombe Fans,
      Please remember the imminent deadline (Monday 8th April) to register and submit abstracts for Pombe 2013: http://events.embo.org/13-pombe
      Abstracts are also required from all who have already been invited to talk.
      And do book your accommodation if you haven&#39;t yet done so.
      More details are in previous email forwarded below.
      Cheers,
      -Jürg &amp; Jacky
      From: On Behalf Of Bahler, Jurg
      Sent: 18 March 2013 17:49
      To: pombelist at sanger.ac.uk
      Subject: [Pombelist] Pombe 2013: Accommodation, registration &amp; abstracts
      Dear Pombe Afficionados,
      Only three weeks left to register and submit abstracts for Pombe 2013, by Monday 8th April: http://events.embo.org/13-pombe
      Speakers for 10 plenary talks and all workshop talks will be selected from abstracts, and there will be attractive poster prizes.
      Payment is only requested after registration, by 10th May.
      Important: if you require accommodation, please do book this real soon now. Especially the most cost-effective student accommodation (comfortable, with private bathrooms) may not be available much longer, as it will be put on general sale shortly. Both hotels and student accommodation will sell out in June, so you have to arrange it now. Information on accommodation is available here: http://events.embo.org/13-pombe/application.html
      We will provide a number of free registrations for which you can apply during online registration (a few of which are reserved for student members of The Genetics Society: you become eligible if you join them now). The meeting is also supported by the Biochemical Society, so if you are, or become, a member you can apply to them for student bursaries or, if you have been a member for at least 1 year, also for travel grants.
      We highly appreciate all the generous contributions from our sponsors so far:
      Platinum: EMBO
      Gold: Biochemical Society, Genetics Society, Formedium, Sunrise Science Products, Singer Instruments, F1000Research, PomBase/Wellcome Trust
      Silver: MDPI - Open Access Publishing, Hybrigenics, Infors, Life Technologies, Bioneer
      Bronze: Nature Communications, m2p labs, Imsol, Open Biology
      We look very much forward to welcoming you in London this June!
      All the best,
      -Jürg &amp; Jacky</code></pre>

  </div>
  <div *ngIf="pageName == 'carl-singer-foundation-established'">
<h3 id="carl-singer-foundation-established">Carl Singer Foundation Established</h3>
<p>Carl Singer, who was an integral part of the yeast research community for many years, passed away on February 8, 2013. Throughout his career, Carl supported yeast research both with his engineering expertise and with his good cheer. In tribute to Carl, the Singer family has now set up <a href="http://www.thecarlsingerfoundation.org/">The Carl Singer Foundation</a>, a charitable foundation dedicated to supporting scientific education in the field of yeast genetics. Questions about the foundation may be directed to Harry Singer at harry [at] thecarlsingerfoundation.org.<br />
<br />
Carl’s family would be happy to receive memories of Carl’s life at regards [at] singerinstruments.com.</p>
<p>H/T <a href="http://www.yeastgenome.org/carl-singer-foundation-established">SGD</a></p>

  </div>
  <div *ngIf="pageName == 'quantitative-gene-expression-data-available-pombase'">
<h3 id="quantitative-gene-expression-data-available-in-pombase">Quantitative gene expression data available in PomBase</h3>
<p>We have extended the Gene Expression section of each gene page to support the display of quantitative expression data, and are now showing data from two publications:</p>
<ul>
<li>Marguerat S, Schmidt A, Codlin S, Chen W, Aebersold R, BählerJ. 2012. “Quantitative analysis of fission yeast transcriptomes and proteomes in proliferating and quiescent cells.” <em>Cell</em> <strong>151:</strong>671-683.</li>
<li>Wu JQ, Pollard TD. 2005. “Counting cytokinesis proteins globally and locally in fission yeast.” <em>Science</em> <strong>310:</strong>310-314.</li>
</ul>
<p>We will also soon refine the display of the new expression data, and can add more datasets upon request. We thank Sam Marguerat for preparing the data from both papers for inclusion in PomBase.</p>
<p>We have also updated the PomBase site to include manual curation through April 4, 2013, and we have updated the “all gene names” file on the PomBase ftp site. The new file is available at<br />
<a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/gene_IDs_names.tsv" class="uri">ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/gene_IDs_names.tsv</a></p>

  </div>
  <div *ngIf="pageName == 'genedb-s-pombe-decommissioned'">
<h3 id="genedb-s.-pombe-decommissioned">GeneDB S. pombe decommissioned</h3>
<p>As of 14 May 2013, the old GeneDB database for <em>S. pombe</em> is no longer available. This resource consisted of static web pages, was not updated after March 2012, and not supported by an underlying relational database. The PomBase site fully supersedes GeneDB <em>S. pombe</em>, and provides improved infrastructure that will meet the current and future needs of the fission yeast community. Please e-mail the <a href="mailto:helpdesk@pombase.org">helpdesk</a> if you cannot find a replacement for any GeneDB functionality in PomBase.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through 13 May, 2013.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-0'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>PomBase data now includes manual curation through June 9, 2013, and represents complete annotation for 664 publications (as well as partial curation of many more). A highlight of this month’s literature curation update is the addition of over 9400 phenotype annotations, representing about 95% of the phenotype data from the recently published genome-wide study of cell cycle and cell morphology (<a href="http://rsob.royalsocietypublishing.org/content/3/5/130053.abstract">Hayles <em>et al.</em> Open Biology May 2013</a>; <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23697806">PMID:23697806</a>). We have also improved the display of allele details for phenotype annotations. Other changes include better support for gene synonyms in the simple search, regular updates to the UTR data files, and a number of minor adjustments to external links in the annotation data tables and the external references section.</p>

  </div>
  <div *ngIf="pageName == 'pombase-launches-community-curation'">
<h3 id="pombase-launches-community-curation">PomBase launches community curation</h3>
<p>At the pombe 2013 conference in London, PomBase officially launched its community curation initiative, which allows researchers to contribute publication-based annotations directly to the database. PomBase curators invite lab heads by individual email to curate newly published papers, providing links to the online curation system and its documentation. Researchers can also initiate curation of any older fission yeast publication in PubMed. Community curation uses the open-source online tool <a href="http://curation.pombase.org/pombe/">Canto</a>.</p>

  </div>
  <div *ngIf="pageName == 'pombase-website-update'">
<h3 id="pombase-website-update">PomBase website update</h3>
<p>We’d like to highlight a few improvements we’ve just made to the PomBase website. Most of the changes affect the gene pages:</p>
<ul>
<li>The basic information display at the top of each gene page is more compact.</li>
<li>For ontology annotations, the number of genes annotated is now shown, in a column labeled “Count” (also, changes behind the scenes involving this data mean that pages should load faster).</li>
<li>Annotation extensions for GO are displayed using human-friendly text instead of internal “relation” labels.</li>
<li>The Quick Links box can now be collapsed and expanded by clicking its header.</li>
<li>Display of modification annotations using PSI-MOD is improved.</li>
</ul>
<p>In addition, the Motif Search output now includes standard gene names and product descriptions. As we noted in a separate message, CDS coordinate files are once again available from the Downloads, with accurate and up-to-date data.<br />
<br />
</p>

  </div>
  <div *ngIf="pageName == 'pombelist-has-moved'">
<h3 id="pombelist-has-moved">pombelist has moved</h3>
<p><strong>Update: This item dates from July 2013, and the links in it no longer work.</strong> \ Please see the <a routerLink="/community">Fission Yeast Community</a> page for the current mailing list link. \ (<span class="no-break">2020-02-18</span>)</p>
<p>The pombe community mailing list, pombelist, has migrated from the Wellcome Trust Sanger Institute and is now hosted by EBI. The new address is <a href="mailto:pombelist@ebi.ac.uk" class="email">pombelist@ebi.ac.uk</a> (please note that the old address no longer works, and will generate an automatic notification including the new address). The link to <a href="http://listserver.ebi.ac.uk/mailman/listinfo/pombelist">subscribe</a> has also been updated, and the entire <a href="http://listserver.ebi.ac.uk/mailman/private/pombelist/">archive</a> is available at the new location.</p>

  </div>
  <div *ngIf="pageName == 'connecting-pombase'">
<h3 id="connecting-with-pombase">Connecting With PomBase</h3>
<p>To complement the <a routerLink="/community/mailing-lists">mailing list</a> and twitter (<a href="http://twitter.com/PomBase"><span class="citation" data-cites="PomBase">@PomBase</span></a>) it is now possible to follow the activities of PomBase and interact with other members of the pombe community via the new <a href="http://www.linkedin.com/groups/PomBase-5122686">LinkedIn Group</a> and <a href="https://plus.google.com/114842152049074799331">Google+</a>.</p>
<p>Links to these are also available from the front page of the PomBase.org site.</p>

  </div>
  <div *ngIf="pageName == 'send-htp-data-pombase'">
<h3 id="send-htp-data-to-pombase">Send HTP data to PomBase</h3>
<p>At the pombe 2013 meeting in London, PomBase staff received numerous requests display various published data, such as gene expression, histone modifications, etc. in the genome browser. To provide this, we now invite pombe researchers to send data: If you have published any high-throughput experiments that produced data  that can be associated with genome sequence coordinates, and thereby displayed as tracks on the PomBase genome browser, please fill out the <a href="/submit-data/data-submission-form">HTP Data Submission Form</a>. We can also accept large sets of phenotype data via the Phenotype Data Submission Form. If you have any problems or questions, contact us via the <a href="mailto:helpdesk@pombase.org">PomBase Helpdesk</a> at any time.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-1'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have once again updated the data available on the PomBase web site. The data now includes manual curation through August 11, 2013. We are particularly pleased to note that this update includes annotations from several dozen papers curated by the <em>S. pombe</em> community. Many thanks to all who have done, or are doing, paper curation in <a href="http://curation.pombase.org/pombe/">Canto</a>.<br />
<br />
We also have an updated version of the <em>S. pombe</em>/human ortholog table available upon <a href="mailto:helpdesk@pombase.org">request</a>.</p>

  </div>
  <div *ngIf="pageName == 'pombase-user-survey-open'">
<h3 id="pombase-user-survey-open">PomBase User Survey open</h3>
<p>To guide current and future development, PomBase is now conducting a <a href="https://www.surveymonkey.com/s/NDM2BQX">user survey</a>, where we invite the fission yeast research community and any other PomBase users to evaluate the resources provided so far and comment on future priorities. The survey should take about 10 minutes to complete. Thank you for your participation!</p>
<p><a href="https://www.surveymonkey.com/s/NDM2BQX" class="uri">https://www.surveymonkey.com/s/NDM2BQX</a></p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-2'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>The PomBase web site has been updated and now includes manually curated data through October 6, 2013. The number of community-curated papers continues to increase, ensuring that PomBase gene pages contain complete and up-to-date information. We are also pleased to announce that data tracks are now available in the genome browser for data from these two publications:</p>
<ul>
<li>Woolcock KJ, Gaidatzis D, Punga T, Bühler M. 2010. Dicer associates with chromatin to repress genome activity in Schizosaccharomyces pombe. Nat Struct Mol Biol. 2011 Jan;18(1):94-9. doi: 10.1038/nsmb.1935 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21151114">PMID:21151114</a></li>
<li>Mata J. 2013. Genome-wide mapping of polyadenylation sites in fission yeast reveals widespread alternative polyadenylation. RNA Biol. 2013 Aug 1;10(8):1407-14. doi: 10.4161/rna.25758 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23900342">PMID:23900342</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'new-target-gene-page-section'">
<h3 id="new-target-of-gene-page-section">New “Target Of” gene page section</h3>
<p>With the October 2013 update, gene pages now include “<a href="/documentation/gene-page-target">Target Of</a>” annotations, which describe genes that affect the gene of interest. These annotations are essentially the reciprocal of <a href="/faqs/what-annotation-extension">ontology annotation extensions</a>. Each “Target Of” annotation includes a relationship that indicates how the genes are connected, the name and product of the second gene, and a reference. Genes listed under “Target Of” may include upstream regulators or enzymes that modify the product of the gene of interest. For example, the “Target Of” annotations for <a routerLink="/spombe/result/SPBC11B10.09#recipext">cdc2</a> indicate that it is a substrate of, and regulated by, the kinase Wee1 and the phosphatase Cdc25 (among others). At present, “Target Of” data includes annotations derived from <a href="/documentation/gene-page-gene-ontology#ann_ext">GO annotation extensions</a>. We will soon extend it to include data from <a routerLink="/documentation/gene-page-phenotypes">phenotype</a> annotation extensions.</p>

  </div>
  <div *ngIf="pageName == 'pombase-survey-results-available'">
<h3 id="pombase-survey-results-available">PomBase survey results available</h3>
<p>The 2013 PomBase user survey closed at the end of October, and the results are available <a href="ftp://ftp.pombase.org/pombe/documents/2013_pombase_survey_summary.pdf">here</a> (PDF at FTP site). Some highlights have been sent to the pombe mailing list. Many thanks to all who completed the survey.</p>

  </div>
  <div *ngIf="pageName == '2013-meeting-mini-reviews-published'">
<h3 id="meeting-mini-reviews-published">2013 meeting mini-reviews published</h3>
<p>A series of mini-reviews, which were invited in association with the International Fission Yeast Meeting in London, have now been published in Biochemical Society Transactions: <a href="http://www.biochemsoctrans.org/bst/041/6/default.htm#c" class="uri">http://www.biochemsoctrans.org/bst/041/6/default.htm#c</a></p>
<p>(Thanks to Jürg Bahler for this item)</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-3'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through November 11, 2013. We now have future meetings available as a <a routerLink="/calendar">calendar</a> or a <a routerLink="/events">list</a>. The <a routerLink="/faq">FAQ</a> and some documentation pages have also been updated.<br />
<br />
</p>

  </div>
  <div *ngIf="pageName == 'human-ortholog-data-correction-coming-next-month'">
<h3 id="human-ortholog-data-correction-coming-next-month">Human ortholog data correction coming next month</h3>
<p>We are about to release a data update for PomBase. Please note that there is still a problem with the human orthologs, as originally described on this list in mid-December (see archived message at <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2013/003926.html" class="uri">http://listserver.ebi.ac.uk/pipermail/pombelist/2013/003926.html</a>). We will correct this problem in the next PomBase release, and apologise for any inconvenience in the meantime.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-4'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have once again updated the data available on the PomBase web site. The data now includes manual curation through January 10, 2014, and covers over 100 papers that have been curated in Canto by community members. We again thank all who have contributed curation via Canto.</p>
<p>We have made some improvements to the gene pages. Highlights:</p>
<ul>
<li>The Sequence section now has links to NCBI BLAST as well as Ensembl BLAST.</li>
<li>The External References section now links to the Pomb(A) polyadenylation viewer.</li>
</ul>
<p>In the genome browser, new data tracks are now available for data from these publications:</p>
<ul>
<li>Rhind N, [and many more]. 2011. Comparative functional genomics of the fission yeasts. Science 332(6032):930-6. doi: 10.1126/science.1203357. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:21511999</a></li>
<li>Schlackow M, Marguerat S, Proudfoot NJ, Bähler J, Erban R, Gullerova M. 2013. Genome-wide analysis of poly(A) site selection in <em>Schizosaccharomyces pombe</em>. RNA. 19(12):1617-31. doi:10.1261/rna.040675.113. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24152550">PMID:24152550</a></li>
<li>Soriano I, Quintales L, Antequera F. 2013. Clustered regulatory elements at nucleosome-depleted regions punctuate a constant nucleosomal landscape in Schizosaccharomyces pombe. BMC Genomics. 14:813. doi:10.1186/1471-2164-14-813. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24256300">PMID:24256300</a> (partial data;  remainder coming in the next update)</li>
<li>Xu J, Yanagisawa Y, Tsankov AM, Hart C, Aoki K, Kommajosyula N, Steinmann KE, Bochicchio J, Russ C, Regev A, Rando OJ, Nusbaum C, Niki H, Milos P, Weng Z, Rhind N. 2012. Genome-wide identification and characterization of replication origins by deep sequencing. Genome Biol. 13(4):R27. doi:10.1186/gb-2012-13-4-r27. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22531001">PMID:22531001</a></li>
</ul>
<p>Now that more data tracks are available, we have added some categories to the track configuration section to improve organization. Additional documentation is in preparation, and will be announced here when available.<br />
<br />
Genome sequences for additional <em>Schizosaccharomyces</em> species (<em>S. japonicus</em>, <em>S. octosporus</em>, and <em>S. cryophilus</em>) have recently become available in Ensembl Fungi, and the PomBase genome browser now includes comparative genomics data, with a view of region comparisons between each new genome and <em>S. pombe</em>.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-5'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>Data on the PomBase web site now includes manual curation through February 24, 2014. Human orthologs that went missing from gene pages have been restored, and other small improvements have been made to gene pages. Community curation now covers over 130 publications.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-6'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through April 28, 2014. Transcriptome data from <a href="http://europepmc.org/abstract/MED/23101633">Margeurat <em>et al</em> (2012)</a> is now available as Ensembl Browser tracks.</p>
<p>Thank you to all who have done, or are doing, paper curation in Canto. Over 159 community-curated papers are now included in PomBase.</p>
<p>There are a number of routes to accelerate your data into PomBase, (either through community curation, or by supplying HTP sequence, modification or phenotype data in one of our specified formats), see <a href="http://www.pombase.org/submit-data" class="uri">http://www.pombase.org/submit-data</a> for more details.</p>
<p>As usual, please don’t hesitate to alert us of any other problems with data or site performance, or if you have any questions.</p>
<p>Sincerely yours,<br />
The PomBase Staff</p>

  </div>
  <div *ngIf="pageName == 'gene-ontology-publication-annotation-extensions'">
<h3 id="gene-ontology-publication-on-annotation-extensions">Gene Ontology publication on annotation extensions</h3>
<p>PomBase was an early adopter of <a routerLink="/documentation/gene-page-gene-ontology#ann_ext">annotation extensions</a>, which add spatial, temporal, or substrate/target details to GO annotations. The GO Consortium has now published a paper describing its implementation of annotation extensions, in which PomBase examples and its gene page display figure prominently:</p>
<p>Huntley, R.P. <em>et al.</em> (2014) A method for increasing expressivity of Gene Ontology annotations using a compositional approach. <em>BMC Bioinformatics</em> 2014, <strong>15</strong>:155. <a href="http://www.biomedcentral.com/1471-2105/15/155/abstract">doi:10.1186/1471-2105-15-155</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24885854">PMID:24885854</a></p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-7'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through June 6, 2014. In other improvements, a downloadable file of <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/FASTA/cdna_introns_utrs.fa.gz">intron sequence data</a> (FASTA format) is now available, and phenotypes are now included in the Target Of section on gene pages.</p>
<p>The gene pages also now display protein modification data from two large-scale datasets:</p>
<ul>
<li>Wilson-Grady JT, Villén J, Gygi SP. 2008 .Phosphoproteome analysis of fission yeast. <em>J Proteome Res</em>. 2008 Mar;<strong>7</strong>(3):1088-97. doi:10.1021/pr7006335. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18257517">PMID:18257517</a></li>
<li>Carpy A, Krug K, Graf S, Koch A, Popic S, Hauf S, Macek B. 2014. Absolute proteome and phosphoproteome dynamics during the cell cycle of fission yeast. <em>Mol Cell Proteomics</em>. 2014 Apr 23. [Epub ahead of print] <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24763107">PMID:24763107</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-8'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through July 8, 2014. The gene pages also now display protein modification data from an additional large-scale dataset:</p>
<p>Koch A, Krug K, Pengelley S, Macek B, Hauf S. 2011. Mitotic substrates of the kinase aurora with roles in chromatin regulation identified through quantitative phosphoproteomics of fission yeast. <em>Sci Signal</em>. <strong>4</strong>(179): rs6 doi: 10.1126/scisignal.2001588 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21712547">PMID:21712547</a><br />
<br />
We have also made corrections to some residue positions affected by sequence updates in one of the modification datasets we added last month:</p>
<p>Carpy A, Krug K, Graf S, Koch A, Popic S, Hauf S, Macek B. 2014. Absolute proteome and phosphoproteome dynamics during the cell cycle of fission yeast. <em>Mol Cell Proteomics</em>. 2014 Apr 23. [Epub ahead of print] <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24763107">PMID:24763107</a></p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-9'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through August 8, 2014. Community curation now covers over 190 papers. Gene pages now include links to the <a href="https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/buildDetails?atlas_build_id=355"><em>S. pombe</em> PeptideAtlas</a>, a database of peptides identified in tandem mass spectrometry proteomics experiments.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-10'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through August 30, 2014. Community curation now covers over 200 papers.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-11'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through October 27, 2014, including 225 community-curated publications. The gene page Phenotype section now includes data from the high-throughput microscopy analysis of viable deletion mutants reported in:<br />
<br />
Graml V, Studera X, Lawson JL, Chessel A, Geymonat M, Bortfeld-Miller M, Walter T, Wagstaff L, Piddini E, Carazo-Salas RE. A Genomic Multiprocess Survey of Machineries that Control and Link Cell Shape, Microtubule Organization, and Cell-Cycle Progression. Dev Cell. 2014 Oct 27;31(2):227-39. doi: <a href="http://dx.doi.org/10.1016/j.devcel.2014.09.005">10.1016/j.devcel.2014.09.005</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25373780">PMID:25373780</a>. Links to the accompanying SYSGRO resource have been added to the External References section of the gene pages.</p>
<p>The genome browser now includes tracks for intron branch point data from:</p>
<p>Bitton DA, Rallis C, Jeffares DC, Smith GC, Chen YY, Codlin S, Marguerat  S, Bähler J. LaSSO, a strategy for genome-wide mapping of intronic  lariats and branch points using RNA-seq. Genome Res. 2014 Jul;24(7):1169-79. doi: <a href="http://dx.doi.org/10.1101/gr.166819.113">10.1101/gr.166819.113</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24709818">PMID:24709818</a>.</p>
<p>We have greatly improved search results for GO and FYPO annotations: both now follow more relationship types within the ontology to retrieve genes annotated to a term. The PomBase GO search now includes the <em>regulates</em> relationships, so its search results are consistent with those in the GO Consortium’s AmiGO browser. The FYPO search now uses <em>has_part,</em> <em>has_output</em>, and <em>output_of</em> as well as <em>is_a</em> and <em>part_of</em>. The Phenotype section now includes a highlighted sub-header that indicates whether a deletion mutant is viable or inviable. A file of protein complex subunits is available for download, and numerous smaller improvements have been made in the gene pages and static pages.</p>

  </div>
  <div *ngIf="pageName == 'esyn-network-visualizations-pombase'">
<h3 id="esyn-network-visualizations-in-pombase">esyN network visualizations in PomBase</h3>
<p>PomBase has implemented network visualisations for fission yeast in <a href="http://www.esyn.org/">esyN</a>, using data curated by BioGRID and PomBase. esyN is a web-based tool for building, sharing, and viewing network data developed by Dan Bean and Giorgio Favrin in the <a href="http://www.sysbiol.cam.ac.uk">Cambridge Systems Biology Centre</a>, University of Cambridge, UK.<br />
<br />
On gene pages, we have links to gene-specific interaction networks in esyN in the table headers of the Interactions sections:</p>
<ul>
<li>The Genetic Interactions section links to all interactions centred on the gene and curated in BioGRID</li>
<li>The Physical interactions section has links to two datasets:
<ul>
<li>All physical interactions curated in BioGRID for the gene</li>
<li>Interactions for the gene in the PomBase <a href="/documentation/high-confidence-physical-interaction-network">High Confidence Physical Interaction Network (HCPIN)</a></li>
</ul></li>
</ul>
<p>We also have esyN links on the <a href="/browse-curation/fission-yeast-bp-go-slim-terms">GO Slim page</a> and on ontology term pages for GO Slim biological process terms. Each GO Slim term links to the HCPIN physical interaction network in esyN (for example, see the “regulation of mitotic cell cycle” <a href="http://www.esyn.org//builder.php?type=Graph&term=GO:0007346&interactionType=physical&source=pombase&includeInteractors=false">network</a>).</p>

  </div>
  <div *ngIf="pageName == 'new-compact-go-annotation-display'">
<h3 id="new-compact-go-annotation-display">New compact GO annotation display</h3>
<p>To make the Gene Ontology (GO) annotations easier to read on PomBase gene pages, we have introduced a new, streamlined display that presents just the essentials. The summary shows the term name (hyperlinked to the ontology term page), the count of genes annotated to the term, and any annotation extensions. All of the previously visible annotation details are still available – simply click the “Summary” button to switch to the “Full” view. Or click the “+” and “-” icons to expand or collapse the annotation to a single term.<br />
 <br />
In addition, the top of the Biological Process table now lists any GO slim terms applicable to the gene.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-12'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through January 12, 2015, including 240 community-curated publications. The gene page Phenotype section now features a compact default display. A downloadable “viability summary” data file is now available. The PomBase BLAST server has incorporated interface changes made Ensembl-wide.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-13'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through February 2, 2015, including 245 community-curated publications. On the gene pages, the interaction tables now provides a bit of descriptive text for each annotation, indicating the nature and direction of the interaction.</p>

  </div>
  <div *ngIf="pageName == 'pombe-2015-registration-now-open'">
<h3 id="pombe-2015-registration-now-open">Pombe 2015 registration now open</h3>
<p>Registration for Pombe 2015: 8th International Fission Yeast Meeting is now open at the conference web site, <a href="https://amarys-jtb.jp/web/Pombe2015/index.html" class="uri">https://amarys-jtb.jp/web/Pombe2015/index.html</a></p>
<p>The registration deadline is 17 May 2015.</p>
<p>Thanks to Yasushi Hiraoka for this item.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-14'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through March7, 2015, including 250 community-curated publications.The autocomplete feature of the Advanced Search ontology term filter has been improved with respect to response time and relevance of suggested terms.</p>

  </div>
  <div *ngIf="pageName == 'pombe-2015-abstract-deadline-approaching'">
<h3 id="pombe-2015-abstract-deadline-approaching">Pombe 2015 abstract deadline approaching</h3>
<p>Abstracts are due on Sunday, April 19, 2015 for the <a href="https://amarys-jtb.jp/web/Pombe2015/index.html">8th International Fission Yeast Meeting</a> in Kobe, Japan. Registration will remain open until May 17, but the abstract submission deadline cannot be extended.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-15'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through April 7, 2015, including 260 community-curated publications.The Advanced Search now supports queries for proteins with a specified number of transmembrane domains.</p>

  </div>
  <div *ngIf="pageName == 'pombe-2015-poster-abstract-deadline-extended'">
<h3 id="pombe-2015-poster-abstract-deadline-extended">Pombe 2015 poster abstract deadline extended</h3>
<p>The abstract submission deadline for the <a href="https://amarys-jtb.jp/web/Pombe2015/index.html">8th International Fission Yeast Meeting</a> in Kobe, Japan has been extended until midnight Friday, April 24 for <strong>posters only</strong>. Registration is open until May 17.</p>

  </div>
  <div *ngIf="pageName == 'pombe-2015-travel-fellowships'">
<h3 id="pombe-2015-travel-fellowships">Pombe 2015 travel fellowships</h3>
<p>Applications are now being accepted for fellowships to provide financial support for students and postdocs attending the 8th International Fission Yeast Meeting in Kobe, Japan. To apply, follow the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2015/004246.html">instructions sent to the pombase mailing list</a>. The deadline is may 17, 2015 (same as the registration deadline).</p>

  </div>
  <div *ngIf="pageName == 'canto-downtime-new-version'">
<h3 id="canto-downtime-new-version">Canto downtime &amp; new version</h3>
<p>Canto, PomBase’s literature curation tool, will be unavailable for approximately 3 weeks starting at 12:00 midnight UK time (BST) tonight, 27 May 2015, while we deploy an upgraded version.<br />
<br />
The upgraded Canto will feature an entirely new interface for annotating multi-allele phenotypes and the corresponding genotypes, as well as improved workflows for single-allele phenotypes, GO, etc. All existing annotations will be retained, and users can resume curation using the new and improved features in any unfinished sessions when Canto is back online.<br />
<br />
We will announce when the new version of Canto is released to the public.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-16'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through May 8, 2015, including 265 community-curated publications.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-17'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through May 26, 2015, including 270 community-curated publications. See you at Pombe 2015 in Kobe!</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-multi-allele-phenotypes'">
<h3 id="pombase-data-update-with-multi-allele-phenotypes">PomBase data update with multi-allele phenotypes</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through August 13, 2015, including 300 community-curated publications.</p>
<p>PomBase gene pages now include multi-allele phenotype annotations (i.e. phenotypes of double mutants, triple mutants, etc.). New sub-sections of the gene pages display multi-allele phenotypes at the population and individual cell level, paralleling the organisation of the single allele phenotype display. Compact and full views are available; both show phenotypes with the relevant genotypes and the alleles that make them up, and the full view adds details for evidence, expression, conditions, and references.</p>
<p>The genome browser now includes data tracks for two more publications:<br />
<br />
DNA polymerase usage from:<br />
Daigaku Y, Keszthelyi A, Müller CA, Miyabe I, Brooks T, Retkute R, Hubank M, Nieduszynski CA, Carr AM. 2015. A global profile of replicative polymerase usage. <em>Nat Struct Mol Biol.</em> 2015 Mar;<strong>22(3):</strong>192-8. doi: <a href="http://dx.doi.org/10.1038/nsmb.2962">10.1038/nsmb.2962</a> PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25664722">25664722</a><br />
<br />
Promoters and transcription start sites from:<br />
Li H, Hou J, Bai L, Hu C, Tong P, Kang Y, Zhao X, Shao Z. 2015. Genome-wide analysis of core promoter structures in Schizosaccharomyces pombe with DeepCAGE. <em>RNA Biol.</em> 2015;<strong>12(5):</strong>525-37. doi: <a href="http://dx.doi.org/10.1080/15476286.2015.1022704">10.1080/15476286.2015</a>.1022704 PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25747261">25747261</a></p>
<p>Codon adaptation index (CAI) values are now included in the Protein Properties section of the gene pages and in the downloadable PeptideStats.tsv file. A file of amino acid composition data is also available from the FTP site and the <a href="/downloads/protein-datasets">Protein Datasets</a> page.</p>
<p>The gene page section that was formerly misnamed “species distribution” is now called “taxonomic conservation”.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-viability-summary-correction-alert'">
<h3 id="pombase-data-update-viability-summary-correction-alert">PomBase data update; viability summary correction alert</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through September 6, 2015.</p>
<p>Errors in the previous FYPOviability.tsv file have been corrected, and we recommend that all users update this file, especially those who downloaded it earlier in September 2015.</p>

  </div>
  <div *ngIf="pageName == 'new-genetics-primer-fission-yeast'">
<h3 id="new-genetics-primer-for-fission-yeast">New genetics primer for fission yeast</h3>
<p>A new genetics primer, aimed at researchers interested in using fission yeast as a model system, has recently been published. The primer includes a brief history of fission yeast research, an introduction to available genetic tools, and the use of PomBase for data analysis</p>
<p>Hoffman CS, Wood V, Fantes PA. (2015) An Ancient Yeast for Young Geneticists: A Primer on the <em>Schizosaccharomyces pombe</em> Model System. <em>Genetics</em> <strong>201</strong>:403-423. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26447128">PMID:26447128</a> <a href="http://dx.doi.org/10.1534/genetics.115.181503">DOI:10.1534/genetics.115.181503</a></p>

  </div>
  <div *ngIf="pageName == 'new-advanced-search-features'">
<h3 id="new-advanced-search-features">New Advanced Search features</h3>
<p>We have introduced new features to the <a href="/spombe/query/builder">Advanced Search</a>:</p>
<ul>
<li>There are now two query reuse options: store a query in your web browser cache, or download a JSON file that can be uploaded later to re-run.</li>
<li>You can now query for genes that interact genetically or physically with a specified gene.</li>
<li>The FYPO query now offers options to choose nulls (deletions or disruptions), wild-type overexpression, or all alleles. The search results will include any genes that have an allele that matches the allele criteria and the chosen phenotype.</li>
</ul>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-18'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through November 9, 2015, including 340 community-curated publications.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-19'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through January 25, 2016.<br />
The genome browser includes variation data, in tracks under “Variation”, from natural <em>S. pombe</em> isolates, published in:<br />
<br />
Jeffares DC <em>et al.</em> 2015. The genomic and phenotypic diversity of Schizosaccharomyces pombe. <em>Nat Genet.</em> <strong>47(3):</strong> 235-241. doi:<a href="http://dx.doi.org/10.1038/ng.3215">10.1038/ng.3215</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25665008">PMID:25665008</a></p>
<p>New files are now available from the PomBase FTP site, and are linked from pages in the Download Datasets area:</p>
<ul>
<li>Non-coding RNA sequence feature coordinates (available via the <a href="/downloads/data-mappings">Data Mappings</a> page);</li>
<li>Protein features, such as domains and family assignments (available via the <a routerLink="/downloads/protein-datasets">Protein Datasets</a> page);</li>
<li>Protein modification annotations (also in Protein Datasets).</li>
</ul>
<p>The <a routerLink="/status/new-and-removed-genes">New and Removed Genes page</a> has been updated to reflect recent deletions and merges.<br />
<br />
Note: Ontology graph views are no longer available in the genome browser, so links have been removed from the GO, FYPO, and modification tables on the gene pages. For GO and FYPO, links to external ontology browsers that offer graphical views are available on the <a href="/documentation/ontology-term-page">Ontology Term pages</a>.<br />
<br />
</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-20'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through March 9, 2016.</p>
<p><strong>Important:</strong> We have corrected a problem that made erroneous interaction data and literature appear on some gene pages.</p>
<p>The gene pages now include interaction data from the Vo et al. proteome-wide study (curated by BioGRID and imported into PomBase):<br />
Vo TV <em>et al.</em> 2016. A Proteome-wide Fission Yeast Interactome Reveals Network Evolution Principles from Yeasts to Human. <em>Cell</em> <strong>164(1-2)</strong>: 310-23. doi: <a href="http://dx.doi.org/10.1016/j.cell.2015.11.037">10.1016/j.cell.2015.11.037</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=26771498">PMID:26771498</a>.</p>
<p>The genome browser now includes transcriptome data published in:<br />
<br />
Eser P, Wachutka L, Maier KC, Demel C, Boroni M, Iyer S, Cramer P, Gagneur J. 2016. Determinants of RNA metabolism in the Schizosaccharomyces pombe genome. <em>Mol Syst Biol</em>. <strong>12(2):</strong> 857. doi: <a href="http://dx.doi.org/10.15252/msb.20156526">10.15252/msb.20156526</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=26883383">PMID:26883383</a>.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-21'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through April 8, 2016.</p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-22'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through May 12, 2016.</p>

  </div>
  <div *ngIf="pageName == 'our-model-organism-database-commentary'">
<h3 id="our-model-organism-database-commentary">Our model organism database commentary</h3>
<p>Several of the PomBase staff, joined by our advisor Sir Paul Nurse, have published a Comment in <em>BMC Biology</em> briefly describing the importance of model organism databases to the success of modern biomedical research:<br />
<br />
Oliver SG, Lock A, Harris MA, Nurse P, Wood V. 2016. Model organism databases: essential resources that need the support of both funders and users.<br />
<em>BMC Biol.</em> 2016 <strong>14(1)</strong>: 49. doi: <a href="http://dx.doi.org/10.1186/s12915-016-0276-z">10.1186/s12915-016-0276-z</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27334346">PMID:27334346</a></p>

  </div>
  <div *ngIf="pageName == 'show-your-support-database-funding'">
<h3 id="show-your-support-for-database-funding">Show your support for database funding</h3>
<p>In response to planned cuts to database funding, leading model organism researchers have prepared an open letter to NIH Director Dr. Francis Collins to demonstrate support for the independent community-focused databases that are essential to their work. Although PomBase is not directly funded by NIH, we collaborate extensively with those that are, including the GO Consortium and several model organism databases.<br />
<br />
The Genetics Society of America website where the letter can be viewed and signed is at <a href="http://www.genetics-gsa.org/MODsupport" class="uri">http://www.genetics-gsa.org/MODsupport</a><br />
<br />
Please sign the letter to add your voice in support of the databases that help make your research possible. For more information, we recommend an email that Mike Cherry sent to the GO-Friends mailing list, archived at <a href="https://mailman.stanford.edu/pipermail/go-friends/2016-June/002355.html" class="uri">https://mailman.stanford.edu/pipermail/go-friends/2016-June/002355.html</a></p>

  </div>
  <div *ngIf="pageName == 'pombase-data-update-23'">
<h3 id="pombase-data-update">PomBase data update</h3>
<p>We have updated the data available on the PomBase web site to include manual curation through September 11, 2016.</p>

  </div>
  <div *ngIf="pageName == '9th-international-fission-yeast-meeting-registration-open'">
<h3 id="th-international-fission-yeast-meeting-registration-open">9th International Fission Yeast Meeting registration open</h3>
<p>Registration for the 9th International Fission Yeast Meeting is now open. The meeting will be held in Banff, Canada from May 14-19, 2017. Early registration closes Dec 1, 2016! Please see our website at <a href="http://www.pombe2017.com">www.pombe2017.com</a> for details. We look forward to seeing you in Banff!<br />
<br />
- Conference Organizers: Dallan Young, Gordon Chua, Paul Young</p>

  </div>
  <div *ngIf="pageName == '9th-international-fission-yeast-meeting-early-registration-closes-soon'">
<h3 id="th-international-fission-yeast-meeting---early-registration-closes-soon">9th International Fission Yeast Meeting - early registration closes soon</h3>
<p>Reminder: early registration for the 9th International Fission Yeast Meeting in Banff closes <strong>Dec. 31, 2016</strong>. Please see the conference website at <a href="http://www.pombe2017.com">www.pombe2017.com</a> for details.</p>

  </div>
  <div *ngIf="pageName == 'new-improved-pombase-goes-live'">
<h3 id="new-improved-pombase-goes-live">New, improved PomBase goes live</h3>
<p>The new PomBase web site, which has been under development during 2017, has been released. The new site features:</p>
<ul>
<li>Nightly data updates</li>
<li>New publication pages</li>
<li>New genotype pages</li>
<li>Improved ontology term pages</li>
<li>Improved summary views for annotation displays</li>
<li>Phenotype annotation display filtering</li>
<li>Faster querying in the advanced search</li>
<li>Front page research and community curation highlights</li>
<li>Streamlined back-end data storage and retrieval</li>
</ul>
<p>We thank the members of the fission yeast research community who have followed its progress via the preview site, and welcome feedback from all users.</p>

  </div>
  <div *ngIf="pageName == 'congratulations-to-gsa-award-winners'">
<h3 id="congratulations-to-gsa-award-winners">Congratulations to GSA award winners</h3>
<p>The <a href="http://www.genetics-gsa.org">Genetics Society of America</a> (GSA) has announced two award winners familiar to the model organism database world:</p>
<ul>
<li>Ira Herskowitz Award: Mike Cherry, Stanford University</li>
<li>Lifetime Achievement Award: Steve Oliver, University of Cambridge</li>
</ul>
<p>The awards will be presented at the next <a href="http://conferences.genetics-gsa.org/yeast/2018/index">Yeast Genetics Meeting</a>, at Stanford University in August 2018. Congratulations and thanks to Mike and Steve!</p>

  </div>
  <div *ngIf="pageName == 'andre-goffeau'">
<h3 id="in-memory-of-andré-goffeau">In memory of André Goffeau</h3>
<p>Sadly, PomBase staff and the fission yeast community note the death of André Goffeau on April 2, 2018. In addition to initiating and coordinating the sequencing of the budding yeast genome, Prof. Goffeau will be remembered for his contributions to the fission yeast genome project and for his knowledge, leadership, and friendship.</p>

  </div>
  <div *ngIf="pageName == 'pombase-deploys-jbrowse'">
<h3 id="pombase-releases-jbrowse">PomBase releases JBrowse</h3>
<!-- newsfeed_thumbnail: browser.png -->
<p>PomBase has now implemented <a href="https://www.pombase.org/jbrowse/?loc=II%3A564641..602220&amp;tracks=DNA%2CPomBase%20forward%20strand%20features%2CPomBase%20%20reverse%20strand%20features&amp;highlight=">JBrowse</a>, from the <a href="http://gmod.org/wiki/Main_Page">GMOD</a> project, as its genome browser. The new browser offers a number of improvements over the old:</p>
<ul>
<li>Quick, responsive scrolling and zooming</li>
<li>Simple track selection interface</li>
<li>Intuitive controls</li>
<li>Simplified data submission pipeline behind the scenes</li>
<li>More informative track metadata</li>
</ul>

  </div>
  <div *ngIf="pageName == 'book-chapter'">
<h3 id="new-book-chapter-about-pombase">New book chapter about PomBase</h3>
<!-- newsfeed_thumbnail: book_chapter.png -->
<p>PomBase has a <a href="https://link.springer.com/protocol/10.1007/978-1-4939-7737-6_4">new book chapter</a> in <a href="https://link.springer.com/book/10.1007/978-1-4939-7737-6"><em>Eukaryotic Genomic Databases</em> (Methods and Protocols)</a>. This chapter provides insight into the curation philosophy and data organization at PomBase, and provides a guide to using PomBase tailored for infrequent visitors and anyone considering extending their research to include <em>S. pombe</em>. The chapter is <a href="https://link.springer.com/protocol/10.1007/978-1-4939-7737-6_4">free to download</a> courtesy of the <a href="https://wellcome.ac.uk/">Wellcome Trust</a>.</p>

  </div>
  <div *ngIf="pageName == 'new-browse-tracks'">
<h3 id="new-genome-browser-tracks">New genome browser tracks</h3>
<!-- newsfeed_thumbnail: new_tracks.png -->
<p>We are very pleased to announce that we have loaded a number of new datasets into the PomBase [JBrowse genome browser (https://www.pombase.org/jbrowse/). These include:</p>
<ul>
<li><a href="https://www.biorxiv.org/content/early/2018/03/13/281642">Thodberg <em>et al.</em> (2018)</a> - CAGE-defined <a href="https://www.pombase.org/jbrowse/?loc=II%3A27586..65254&amp;tracks=DNA%20sequence%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CConsensus%20transcription%20start%20sites%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20heat%20shock%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20heat%20shock%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20nitrogen%20starvation%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20nitrogen%20starvation%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20oxidative%20stress%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20oxidative%20stress%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20YES%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20YES%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20glucose%20minimal%20medium%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20glucose%20minimal%20medium%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)&amp;highlight=">transcription start sites</a> across 5 different conditions</li>
<li><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23163955">Yadav <em>et al.</em> (2012)</a> - <a href="https://www.pombase.org/jbrowse/?loc=I%3A2364901..2436900&amp;tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CPosition%20specific%20G(x)%20scores%20(free%20energy%20needed%20to%20separate%20a%20base%20pair)%20-%20Yadav%20et%20al.%20(2012)&amp;highlight=">G(x) scores</a> specifying the amount of free energy needed to melt base pairs in the DNA duplex at different genomic locations</li>
<li>A PomBase-generated <a href="https://www.pombase.org/jbrowse/?loc=III%3A725762..736349&amp;tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CConsensus%20transcription%20factor%20DNA%20binding%20motifs&amp;highlight=">dataset of promoter elements</a> across the genome (computational matching of the <a routerLink="/browse-curation/dna-binding-sites">consensus promoter sequences</a> to the reference DNA sequence)</li>
<li><a href="https://www.pombase.org/jbrowse/?loc=I%3A2178501..2207300&amp;tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CIntron%20Branch%20Points%20-%20Bitton%20et%20al.%20(2014)&amp;highlight=">Intron branch points</a> from <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24709818">Bitton <em>et al.</em> (2014)</a> which was also available in the Ensembl browser.</li>
</ul>
<p>For anyone wanting a quick introduction to our genome browser, Antonia Lock has written <a routerLink="/documentation/JBrowse_quick_start">“Getting started with PomBase JBrowse”</a>, a basic guide that covers loading tracks, navigating the browser, what metadata we provide, and more.</p>

  </div>
  <div *ngIf="pageName == 'mailing-list-move-to-cam'">
<h3 id="pombelist-changes">pombelist changes</h3>
<!-- newsfeed_thumbnail: pombelist.png -->
<p>The pombe community mailing list, “pombelist”, is now hosted by the University of Cambridge. The new address for posting messages is <a href="mailto:pombelist@pombase.org" class="email">pombelist@pombase.org</a>. The link to <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">subscribe</a> has also changed.</p>

  </div>
  <div *ngIf="pageName == 'transcript-tracks-from-Atkinson-et-al-(2018)-loaded'">
<h3 id="transcript-tracks-from-atkinson-et-al.-2018-loaded">Transcript tracks from Atkinson et al. (2018) loaded</h3>
<!-- newsfeed_thumbnail: Atkinson.png -->
<p>We are very pleased to announce that we have loaded the transcript tracks from <a href="https://www.ncbi.nlm.nih.gov/pubmed/29914874">Atkinson et al. (2018)</a> into the PomBase JBrowse genome browser. For a brief introduction to getting started with PomBase JBrowse, please see our <a href="https://www.pombase.org/documentation/JBrowse_quick_start">documentation page</a>. If you have published data that you would like to see hosted, please get in touch.</p>

  </div>
  <div *ngIf="pageName == '2019-fungal-pathogen-genomics-course-registration-open'">
<h3 id="fungal-pathogen-genomics-course-2019---registration-open">Fungal Pathogen Genomics Course 2019 - registration open</h3>
<!-- newsfeed_thumbnail: WT_course.png -->
<p>Registration for the <a href="https://coursesandconferences.wellcomegenomecampus.org/our-events/fungal-pathogen-genomics-2019/">2019 Fungal Pathogen Genomics Course</a> is now open. The course is hosted by Wellcome Genome Advanced Courses and Scientific Conferences, and will take place May 7-12, 2019, at the Wellcome Genome Campus, Hinxton, UK. Course content provides hands-on training on how to: - Take advantage of unique tools offered by FungiDB, EnsemblFungi, PomBase, SGD/CGD, and MycoCosm/JGI; - Develop testable hypotheses; - Investigate transcriptomics, proteomics and genomics datasets across multiple databases and different user interfaces. Please see the <a href="https://coursesandconferences.wellcomegenomecampus.org/our-events/fungal-pathogen-genomics-2019/">course website</a> for more information, including how to apply, costs (limited bursaries are available), programme, and logistics.</p>

  </div>
  <div *ngIf="pageName == 'nar-database-issue'">
<h3 id="pombase-nar-database-issue">PomBase NAR Database Issue</h3>
<!-- newsfeed_thumbnail: NAR_paper.png -->
<p>Our NAR database update “PomBase 2018: user-driven reimplementation of the fission yeast database provides rapid and intuitive access to diverse, interconnected information” is <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky961/5128936">now available</a>. We have updated the <a routerLink="/about/citing-pombase">Citing PomBase</a> to recommend citing this new paper. Thank you all for guiding the development of the new, improved PomBase, and for your continued usage, curation contributions, and suggestions!</p>

  </div>
  <div *ngIf="pageName == 'jbrowse-graphics-gene-page'">
<h3 id="new-pombase-genomic-region-graphics">New PomBase genomic region graphics</h3>
<!-- newsfeed_thumbnail: browser.png -->
<p>PomBase gene pages now use interactive graphics from PomBase JBrowse to depict the genomic region around the gene. Drag to scroll left and right, double-click to zoom in, shift-double-click to zoom out, and click a feature to see details in a popup. The “Full-screen view” link in the corner opens the fully functional JBrowse in a new tab or window. Reloading a gene page restores the display to the default location and zoom level.</p>

  </div>
  <div *ngIf="pageName == 'rna-central'">
<h3 id="rna-central-and-pombase">RNA central and PomBase</h3>
<!-- newsfeed_thumbnail: rna_central_icon.png -->
<p>RNAcentral is a comprehensive database of non-coding RNA sequences. PomBase is an RNAcentral Consortium member, and all of the curated non-coding RNAs from PomBase will be available in RNAcentral soon. For more information, see their recent <a href="https://doi.org/10.1093/nar/gky1034">NAR Database Issue paper</a>, as well as current <a href="https://rnacentral.org/search?q=pombe">search results for <em>S. pombe</em> RNAs</a>.</p>

  </div>
  <div *ngIf="pageName == '20-years-GO'">
<h3 id="celebrating-20-years-of-go">Celebrating 20 years of GO</h3>
<!-- newsfeed_thumbnail: go-logo-icon.mini.png -->
<p>PomBase curators are major contributors to the <a href="http://geneontology.org/">Gene Ontology</a> (GO) project — ontology content, annotations, and QC procedures — and co-authors on the new <a href="https://doi.org/10.1093/nar/gky1055">GO NAR Database Issue paper</a>.</p>
<p>We recommend citing the GO and PomBase NAR papers when you use GO data in your analyses.</p>

  </div>
  <div *ngIf="pageName == 'S.-pombe-in-your-pocket'">
<h3 id="pombase-in-your-pocket">PomBase in your pocket</h3>
<!-- newsfeed_thumbnail: mobile_thumb.png -->
<p>Our usage statistics informed us that over 20% of devices accessing PomBase are smartphones or tablets. We therefore spent some time optimizing the display for small screens. We hope that you will continue to enjoy PomBase on the go!</p>

  </div>
  <div *ngIf="pageName == 'conserved-unknowns-pub'">
<h3 id="hidden-in-plain-sight-what-remains-to-be-discovered-in-the-eukaryotic-proteome">Hidden in plain sight: What remains to be discovered in the eukaryotic proteome?</h3>
<!-- newsfeed_thumbnail: unknowns.png -->
<p>In a new publication, PomBase curators consider the challenges and opportunities that conserved, but persistently unstudied, proteins pose for diverse areas of basic and applied biology. To draw attention to these proteins, we develop metrics to define unknown lists, provide unknown inventories for human and yeast, classify <em>S. pombe</em> unknowns by numerous orthogonal attributes, and speculate about reasons for their neglect.</p>
<p>A <a href="https://www.biorxiv.org/content/early/2018/11/16/469569">pre-publication manuscript</a> is available at bioRxiv.</p>

  </div>
  <div *ngIf="pageName == 'tm-transport-overhaul'">
<h3 id="fission-yeast-transmembrane-transport-overhaul">Fission yeast transmembrane transport overhaul</h3>
<!-- newsfeed_thumbnail: antiporter.png -->
<p>The Gene Ontology “<a href="https://www.pombase.org/term_genes/GO:0055085">transmembrane transport</a>” branch has recently been substantially revised. In line with these revisions, PomBase has standardised gene product descriptions for transporters, and overhauled GO annotations to be as complete and comprehensive as possible based on current knowledge.</p>
<p>Icon courtesy of <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29077811">Reactome</a>.</p>

  </div>
  <div *ngIf="pageName == 'quick-little-tool'">
<h3 id="see-your-genes-in-a-quilt">See your genes in a QuiLT</h3>
<!-- newsfeed_thumbnail: QLT.png -->
<p>PomBase now offers a new way to display gene lists graphically based on multiple orthogonal annotation types — the Quick Little Tool (QuiLT) for visualisation.</p>
<p>Inspired by our recent analysis of conserved unstudied proteins (see figures 4 and S1 in the <a href="https://www.biorxiv.org/content/early/2018/11/16/469569">manuscript at bioRxiv</a>), QuiLT allows you to create a similar figure for any gene list you create or import using the advanced search. To use QuiLT, follow the link to your search results, then click the “Visualise” button. QuiLT visualisation is also available from the PomBase pages that list genes annotated to an ontology term, and on the Priority unstudied genes page.</p>
<p>To see the Unknowns dataset in QuiLT, visit the <a routerLink="/results/from/predefined/priority-unstudied-genes:conserved_unknown">unknowns results page</a> and click “Visualise”.</p>
<p>The QuiLT display is interactive, and you can:</p>
<ul>
<li>Highlight subsets of the list, and filter the display</li>
<li>Toggle annotation types on and off</li>
<li>Reorder the list to focus on features of most interest</li>
<li>Download the image</li>
</ul>
<p>See the <a routerLink="/documentation/quick-little-tool">QuiLT documentation</a> for more information, and <a href="mailto:helpdesk@pombase.org">contact the curators</a> if you have comments, questions or suggestions.</p>
<p>Many thanks to our star (and only) programmer, Kim Rutherford, for developing QuiLT.</p>

  </div>
  <div *ngIf="pageName == 'new-nucleosome-occupancy-browser-maps-loaded'">
<h3 id="new-nucleosome-occupancy-maps-loaded">New nucleosome occupancy maps loaded</h3>
<!-- newsfeed_thumbnail: browser.png -->
<p>We have loaded the nucleosome occupancy maps as described in González et al. (2016) <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27662899">PMID: 27662899</a>. This dataset was generated using the paired-end sequencing protocol of Illumina and thus those maps are of higher resolution than those made with single-end (SE) sequencing hosted in the browser since before.</p>
<p>Here is a link that <a href="https://www.pombase.org/jbrowse/?loc=II%3A2751621..2767510&tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2Cnucleosome%20positioning%20during%20vegetative%20growth%3B%20repeat%201%20-%20Gonz%C3%A1lez%20et%20al.%20%282016%29%2Cnucleosome%20positioning%20during%20vegetative%20growth%3B%20repeat%202%20-%20Gonz%C3%A1lez%20et%20al.%20%282016%29%2CDNA%20sequence&highlight=">loads the tracks in PomBase JBrowse</a>. And here is a link to our <a href="https://www.pombase.org/documentation/JBrowse_quick_start">JBrowse quickstart guide</a>.</p>
<p>Many thanks to Paco Antequera for sending us the bigwig files! If you would like us to load any datasets then please get in touch.</p>

  </div>
  <div *ngIf="pageName == 'mitochondrial-protein-review'">
<h3 id="mitochondrial-go-annotation-update">Mitochondrial GO annotation update</h3>
<!-- newsfeed_thumbnail: mito_news.png -->
<p>Responding to increasing interest in mitochondrial biology, especially relating to ageing, neurogenerative diseases, and processes at the ER-mitochondrion interface, we have reviewed <em>S. pombe</em> mitochondrial GO annotations. Although there is still relatively little fission yeast-derived experimental data in this area, we have refined many inferred annotations for mitochondrial complexes and sub-components as well as some for processes.</p>
<p>You can see all 753 <em>S. pombe</em> mitochondrial annotations on the ontology term page for <a href="https://www.pombase.org/term/GO:0005739">mitochondrion (GO:0005739)</a>.</p>
<p>Icon courtesy of Reactome.</p>

  </div>
  <div *ngIf="pageName == 'improved-disease-association-dataset-released'">
<h3 id="improved-disease-association-dataset-released">Improved disease association dataset released</h3>
<!-- newsfeed_thumbnail: human disease.png -->
<p>We are pleased to announce the release of our improved human disease mappings dataset. This dataset connects human disease causing genes to their <em>S. pombe</em> orthologs.</p>
<p>Diseases are now mapped to the Disease Ontology (DO) and the dataset has been extended by data from Malacards. All disease associations can be accessed from the top level disease page. A disease slim has been created to facilitate browsing of disease categories. Currently, 907 S. pombe genes are associated with disease (up from 610 in the original dataset). This number is due to increase as mappings are still in progress.</p>
<p>Many thanks to DO and Malacards for help in improving this annotation set. Icon courtesy of <a href="https://pixabay.com/en/users/mcmurryjulie-2375405/">Julie McMurry</a>.</p>

  </div>
  <div *ngIf="pageName == 'wood-isb-award'">
<h3 id="val-wood-wins-biocuration-society-award">Val Wood wins Biocuration society award</h3>
<!-- newsfeed_thumbnail: isb_logo_32px.png -->
<p>Congratulations to PomBase project leader Val Wood, who has received the 2019 Exceptional Contributions to Biocuration Award from the International Society for Biocuration. Read more at the <a href="https://www.biocuration.org/anne-morgat-and-val-wood-recipients-of-the-2019-biocuration-awards/">ISB site</a></p>

  </div>
  <div *ngIf="pageName == 'pombe-2019-registration-open'">
<h3 id="pombe-2019---registration-open">Pombe 2019 - registration open</h3>
<!-- newsfeed_thumbnail: pombe2019tiny.png -->
<p>Registration for the <a href="http://www.pombe2019.com">10th International Fission Yeast Meeting</a> is now open!</p>
<p>The conference will take place July 14-19, 2019, in Barcelona, Spain. Early registration closes on April 15th — or when capacity is reached. Please see the <a href="http://www.pombe2019.com">conference website</a> for more information, including registration final deadline and costs (some travel grants are available), abstract submission, programme, accommodation, and logistics.</p>

  </div>
  <div *ngIf="pageName == 'conserved-unknowns-open-biol'">
<h3 id="published-hidden-in-plain-sight-what-remains-to-be-discovered-in-the-eukaryotic-proteome">Published: Hidden in plain sight: What remains to be discovered in the eukaryotic proteome?</h3>
<!-- newsfeed_thumbnail: unknowns.png -->
<p>Our analysis of conserved unknown proteins has now been published in <a href="https://royalsocietypublishing.org/doi/10.1098/rsob.180241">Open Biology</a>. In it, PomBase curators consider the challenges and opportunities that conserved, but persistently unstudied, proteins pose for diverse areas of basic and applied biology. We develop metrics to define unknown lists, provide unknown inventories for human and yeast, and classify <em>S. pombe</em> unknowns by numerous orthogonal attributes, all with a view to drawing attention to the unknowns to alleviate their neglect.</p>

  </div>
  <div *ngIf="pageName == 'trieste-cell-cycle-meeting'">
<h3 id="international-cell-cycle-meeting---registration-open">International Cell Cycle meeting - registration open</h3>
<!-- newsfeed_thumbnail: chantal_roubinet_tiny.png -->
<p>Registration is now open for the <a href="https://www.ucl.ac.uk/lmcb/cell-cycle-meeting-2019">Inaugural Trieste Cell Cycle Meeting</a>, which will be held June 3-6, 2019, in Trieste, Italy.</p>
<p>This is the first of a planned series of biennial cell cycle meetings that will take place in Europe, and will alternate with the Salk Cell Cycle meetings held on the US west coast.</p>
<p>Organisers Rob de Bruin, Snezhana Oliferenko, Rosella Visintin and Peter Thorpe hope to see you there!</p>
<p>Icon derived from meeting image; credit: Chantal Roubinet, Baum lab</p>

  </div>
  <div *ngIf="pageName == 'serym2019'">
<h3 id="south-eastern-regional-yeast-meeting-serym---registration-open">South Eastern Regional Yeast Meeting (SERYM) - registration open</h3>
<!-- newsfeed_thumbnail: serym2019tiny.png -->
<p>Registration is now open for the <a href="https://petitinstitute.gatech.edu/serym-2019">26th annual South Eastern Regional Yeast Meeting (SERYM)</a>, which will be held April 12-14, 2019, in Atlanta, GA, USA.</p>
<p>Fission yeast’s own Susan Forsburg is the keynote speaker. The meeting brings together researchers who use any type of yeast as a model system, covering diverse, interdisciplinary topics from strategies for treatment of fungal disease to modeling human disease in yeast.</p>
<p>Icon: SERYM 2019</p>

  </div>
  <div *ngIf="pageName == 'list-go-slim-tool'">
<h3 id="go-slim-for-any-s.-pombe-gene-list">GO slim for any <em>S. pombe</em> gene list</h3>
<!-- newsfeed_thumbnail: slim_tiny.png -->
<p>PomBase’s advanced search now allows you to retrieve <a href="documentation/pombase-go-slim-documentation">GO slim</a> annotations for any set of search results. To find GO slim annotations for your own list of <em>S. pombe</em> genes, use the advanced search “Gene names and IDs” option, and then use the “Slim” button on the search results page.</p>
<p>See the <a href="browse-curation/fission-yeast-bp-go-slim-terms">fission yeast GO slim page</a> and the <a href="documentation/advanced-search">advanced search documentation</a> for more information.</p>

  </div>
  <div *ngIf="pageName == 'icygmb2019'">
<h3 id="icygmb-2019---registration-open">ICYGMB 2019 - registration open</h3>
<!-- newsfeed_thumbnail: icygmb_tiny.png -->
<p>Registration is now open for the <a href="https://yeast2019.org">29th International Conference on Yeast Genetics and Molecular Biology (ICYGMB)</a>, which returns to Gothenburg, Sweden, August 18-22, 2019.</p>
<p>Yeast2019 is the meeting of the international yeast research community where the latest, and even unpublished results are exchanged, and new projects, alliances, and collaborations are founded. Featuring 55 confirmed speakers including keynote lectures by Susan Gasser, Roger Kornberg and Frederick Roth, this conference will contain important news and information for all yeast researchers. A do-not-miss-event.</p>

  </div>
  <div *ngIf="pageName == 'motif-search-integrated'">
<h3 id="integrated-motif-search">Integrated motif search</h3>
<!-- newsfeed_thumbnail: motif_search_cropped.png -->
<p>The PomBase <a routerLink="/motif_search">motif search</a> has been fully integrated into the website, and allows users to find protein motifs and send them directly to the PomBase <a routerLink="/query">advanced search</a>.</p>

  </div>
  <div *ngIf="pageName == 'pombe-gene-info-extension'">
<h3 id="s.-pombe-included-in-gene-info-browser-extension"><em>S. pombe</em> included in Gene Info browser extension</h3>
<!-- newsfeed_thumbnail: gix_icon.png -->
<p><em>S. pombe</em> gene information is now included in the <a href="https://gene-info.org/">Gene Info</a> extension (GIX) for the Chrome and Firefox web browsers. GIX allows you to retrieve information about a gene product directly on any webpage simply by double clicking an official gene name, synonym or supported accession. Searching or double-clicking on text terms retrieves gene function annotation, GO terms, external database links, and interaction data drawn from BioGRID and IntAct. Retrieved gene names are automatically hyperlinked for rapid recursive searches.</p>
<p>GeneInfo is fully open source, available online at <a href="https://github.com/knightjdr/gene-info/">GitHub</a>. Tutorial videos, a step-by-step guide, and download links for Firefox Add-ons and the Chrome web storeare available <a href="https://gene-info.org/">online</a>. GeneInfo was developed by James Knight in the <a href="https://gingraslab.lunenfeld.ca/">Gingras Lab</a> at the <a href="http://www.lunenfeld.ca/">Lunenfeld-Tanenbaum Research Institute</a> in Toronto, Canada.</p>

  </div>
  <div *ngIf="pageName == 'pombase-interpro-update'">
<h3 id="pombase-interpro-update">PomBase InterPro Update</h3>
<!-- newsfeed_thumbnail: interpro_32px.png -->
<p>PomBase now uses InterPro Version 73.0, which integrates 1,531 new methods from the CATH-Gene3D (122), CDD (330), PANTHER (1075), Pfam (2), PROSITE profiles (1) and TIGRFAMs (1) databases, and covers 81.2% of UniProt Knowledgebase release 2019_02.</p>
<p>See <a href="https://www.ebi.ac.uk/about/news/service-news/InterPro-73.0">the news item at InterPro</a> for additional information, including release notes.</p>

  </div>
  <div *ngIf="pageName == 'customisable-fasta'">
<h3 id="customisable-fasta-download">Customisable FASTA download</h3>
<!-- newsfeed_thumbnail: pombelist.png -->
<p>You can now download nucleotide or peptide sequences for genes in Advanced search results in FASTA format, and customise what is included in the FASTA headers (e.g. gene names, product descriptions, sequence coordinates, or various IDs can be included).</p>

  </div>
  <div *ngIf="pageName == 'prediction-links'">
<h3 id="new-homology-modelling-and-ortholog-links-on-gene-pages">New homology modelling and ortholog links on gene pages</h3>
<!-- newsfeed_thumbnail: tree_for_news.png -->
<p>We have added new external links to PomBase gene pages for structure and ortholog predictions:</p>
<ul>
<li><p>Protein-specific links to <a href="https://swissmodel.expasy.org/">SWISS-MODEL</a>, a fully automated protein structure homology-modelling server, accessible via the ExPASy web server, lead to a SWISS-MODEL Repository page for each sequence and present results. If no structure or model is available, you can either trigger adding an entry to the repository with a single click or easily interactively search for templates and build models in your own SWISS-MODEL workspace.</p></li>
<li><p>Ensembl Fungi Compara and Ensembl Pan-taxonomic Compara links lead to orthology predictions from the Ensembl Compara pipeline for fungi and all species, respectively.</p></li>
<li><p><a href="http://www.pantherdb.org/">PANTHER</a> links retrieve gene information, classification, and predicted orthologs.</p></li>
</ul>

  </div>
  <div *ngIf="pageName == 'tRNA_metabolism_update'">
<h3 id="trna-metabolism-go-annotation-update">tRNA metabolism GO annotation update</h3>
<!-- newsfeed_thumbnail: rna_metabolism_32px.png -->
<p>The process of tRNA metabolism, and the associated molecular functions have recently been reviewed.</p>
<ul>
<li><a href="https://www.pombase.org/term/GO:0006399">“tRNA metabolic process” annotation</a></li>
<li><a href="https://www.pombase.org/term_genes/GO:0006399">genes for “tRNA metabolic process”</a></li>
</ul>
<p>Please let us know if the annotation can be further improved.</p>

  </div>
  <div *ngIf="pageName == 'Fitness_Landscape_of_the_Fission_Yeast_Genome-data-loaded'">
<h3 id="fitness-landscape-of-the-fission-yeast-genome-dataset-loaded-into-jbrowse">“Fitness Landscape of the Fission Yeast Genome” dataset loaded into JBrowse</h3>
<!-- newsfeed_thumbnail: browser.png -->
<p>We have loaded the <a href="https://www.pombase.org/reference/PMID:31077324">Grech et al. (2019) “Fitness Landscape of the Fission Yeast Genome”</a> dataset into JBrowse. In this study, transposon mutagenesis libraries were created to map transposon insertion sites in the S. pombe genome. From this data, functional elements of the genome were inferred. The tracks from this study can be loaded by a single click from the linked publication page above</p>
<p>Thanks Dan Jeffares for sending us the data.</p>
<p>For anyone new to JBrowse we have a <a href="https://www.pombase.org/documentation/JBrowse_quick_start">quick start guide</a>.</p>

  </div>
  <div *ngIf="pageName == 'join-the-conversations-on-slack'">
<h3 id="join-the-conversations-on-slack">Join the conversations on Slack</h3>
<!-- newsfeed_thumbnail: slack_icon-32px.png -->
<p>The vibrant fission yeast community now has a Slack channel. Slack provides a forum for the research community. Follow conversations you care about, message colleagues privately, or in groups, ask questions, post responses. All archived and searchable.</p>
<ul>
<li>See the <a href="/assets/external_media/PombeSlack-flyer.pdf">PombeSlack flyer</a> for more information, including how to join.</li>
</ul>
<!-- included with original news item; commented out after expiry:
 - <a href="https://bit.ly/2XYkcEi">Join PombeSlack</a> -->

  </div>
  <div *ngIf="pageName == 'replication-origin-tracks-loaded'">
<h3 id="replication-origin-data-loaded-into-jbrowse">Replication origin data loaded into JBrowse</h3>
<!-- newsfeed_thumbnail: browser.png -->
<p>We have loaded data from: <a href="https://www.pombase.org/reference/PMID:14566325">Segurado et al. (2003) “A+T-rich islands”</a>, <a href="https://www.pombase.org/reference/PMID:17304213">Hayashi et al. (2007) “Pre-replicative complex localization; early and late firing origins”</a>, and <a href="https://www.pombase.org/reference/PMID:18093330">Mickle et al. (2007) “Replication origins with functional classification”</a>.</p>
<p>To view the tracks, either follow the hyperlinks above to the respective PomBase publication pages, and click on the “view” link after “Datasets from this publication are available in the PomBase JBrowse genome browser”, or go directly to <a href="http://www.pombase.org/jbrowse/">the browser</a> and click on the “select tracks” button to find the tracks manually.</p>
<p>For anyone new to JBrowse we have a <a href="https://www.pombase.org/documentation/JBrowse_quick_start">quick start guide</a>.</p>

  </div>
  <div *ngIf="pageName == 'interpro-v76-loaded'">
<h3 id="pombase-now-uses-interpro-version-76.0">PomBase now uses InterPro Version 76.0</h3>
<!-- newsfeed_thumbnail: interpro_32px.png -->
<p>PomBase now uses InterPro Version 76.0, which integrates 277 new methods from the CATH-Gene3D (1), PANTHER (178) and CDD (98) databases. InterPro cites 59846 publications in PubMed. See the InterPro <a href="https://www.ebi.ac.uk/interpro/release_notes/">release notes</a> for further information.</p>

  </div>
  <div *ngIf="pageName == 'new-wt-pombase-funding'">
<h3 id="new-pombase-funding-from-the-wellcome-trust">New PomBase funding from the Wellcome Trust</h3>
<!-- newsfeed_thumbnail: pombase-logo-32x32px.png -->
<p>We are pleased to announce that the recent PomBase application for continued Wellcome Trust funding was successful. Although the grant was not fully funded, we are confident that we can cover the shortfall by small grants for stand-alone projects and collaborations. We would like to thank the pombe community for their support with the application, and the Wellcome Trust for their continued funding. We look forward to supporting your research until 2025 (and beyond).</p>

  </div>
  <div *ngIf="pageName == 'JBrowse-tracks-loadable-from-publication-pages'">
<h3 id="browser-tracks-now-loadable-from-publication-pages">Browser tracks now loadable from publication pages</h3>
<!-- newsfeed_thumbnail: new_tracks.png -->
<p>Data tracks from datasets hosted in the PomBase genome browser can now be browsed and loaded from their respective publication pages. For an example, see <a href="https://www.pombase.org/reference/PMID:29914874">Atkinson et al. (2018)</a>. Data tracks are now also downloadable from the publication pages.</p>

  </div>
  <div *ngIf="pageName == 'shareable-result-links'">
<h3 id="unique-permanent-urls-for-search-results">Unique permanent URLs for search results</h3>
<!-- newsfeed_thumbnail: advanced_search.png -->
<p>All result pages from the <a routerLink="/query">Advanced search</a> now have a unique permanent URL that can be bookmarked and shared with your colleagues.</p>
<p>The <a routerLink="/documentation/quick-little-tool">QuiLT</a> and <a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">GO slim</a> pages also now have permanent URLs.</p>
<p>Examples:</p>
<ul>
<li><a routerLink="/results/from/id/7474e390-ddc0-4f70-8c2e-d6a69d471a50">All disease associated genes</a></li>
<li><a routerLink="/vis/from/id/7474e390-ddc0-4f70-8c2e-d6a69d471a50">QuiLT example for all disease associated genes</a></li>
<li><a routerLink="/slim:bp_goslim_pombe/from/id/7474e390-ddc0-4f70-8c2e-d6a69d471a50">GO slim for disease associated genes</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'new-stable-integration-vectors'">
<h3 id="new-vectors-for-simple-reliable-s.-pombe-molecular-biology">New vectors for simple, reliable <em>S. pombe</em> molecular biology</h3>
<!-- newsfeed_thumbnail: stable_pAde6_vector.png -->
<p>To enable fission yeast researchers to manipulate <em>S. pombe</em> molecular biology reproducibly and easily, Aleks Vještica and Magdalena Marek in Sophie Martin’s lab have designed and constructed a series of simple, fully characterized plasmids.</p>
<p>The Stable Integration Vector (SIV) series provides a highly modular toolbox to introduce heterologous sequences more stably was possible with than previously available vectors. The toolkit includes antibiotic resistance markers, promoters, fluorescent tags, and terminators, as well as large set of ready-to-use fluorescent probes to mark organelles and visualize cellular processes.</p>
<p>The work is published in the <a href="https://jcs.biologists.org/content/early/2019/11/29/jcs.240754.long">Journal of Cell Science</a>, and a PomBase <a href="https://www.pombase.org/reference/PMID:31801797">publication page</a> is available.</p>

  </div>
  <div *ngIf="pageName == 'pombase-elixir-node-service'">
<h3 id="pombase-is-now-an-elixir-node-service">PomBase is now an ELIXIR Node Service</h3>
<!-- newsfeed_thumbnail: elixir_uk_thumb.png -->
<p>PomBase has been awarded Node Service status by the <a href="https://elixiruknode.org/">UK node</a> of <a href="https://elixir-europe.org/">ELIXIR</a>. ELIXIR-UK Node Services support the bioinformatics and broader biological research communities by providing training and resources that help researchers to find and share data, exchange expertise, and agree on best practices at national, European and international levels. The review panel describes PomBase as a “mature, leading model organism database which is popular, unique, well used, and has a strong user community.”</p>

  </div>
  <div *ngIf="pageName == 'lmo14-strasbourg'">
<h3 id="levures-modèles-et-outils-14th-international-conference">Levures, Modèles et Outils 14th International Conference</h3>
<!-- newsfeed_thumbnail: poster_LMO14_32px.png -->
<p>The 14th edition of the <a href="https://lmo-14-2020.sciencesconf.org/">“Levures, Modèles et Outils” meeting (LMO14)</a> will be held in July 9-11, 2020, at the University of Strasbourg in France. Registration is open February 3rd to June 30th, and abstracts can be submitted from February 3rd to April 10th. Authors will be notified in early May and the final program will be available in early June.</p>
<p>The sessions will be diverse and present the latest findings using yeast as a model organism on the following topics:</p>
<ol type="1">
<li>Cell biology, cell cycle, cytoskeleton</li>
<li>Gene expression regulation</li>
<li>Population, functional and evolutionary genomics</li>
<li>Replication, repair and recombination</li>
<li>Transport, sensing and signaling</li>
<li>Pathogenic yeast and filamentous fungi</li>
<li>Tools, resources and databases</li>
<li>New technologies, yeast and industry</li>
</ol>

  </div>
  <div *ngIf="pageName == 'advanced-search-phenotype-conditions'">
<h3 id="query-phenotypes-for-conditions">Query phenotypes for conditions</h3>
<!-- newsfeed_thumbnail: advanced_search.png -->
<p>The PomBase advanced search <a routerLink="/query">Advanced search</a> now supports using experimental conditions as search criteria for phenotype annotations. For example, you can now query for genes that show abnormal chromosome segregation mutant phenotypes specifically at high or low temperatures. The search uses the same condition descriptors as Canto and the PomBase web pages.</p>
<p>Note that phenotype queries that have condition constraints can be combined, but pay careful attention to the annotations for the results. Future work will add support for querying for multiple conditions on the same annotation, and for specifying conditions to exclude from results.</p>

  </div>
  <div *ngIf="pageName == 'quarter_million'">
<h3 id="a-quarter-of-a-million-annotations">A quarter of a million annotations</h3>
<!-- newsfeed_thumbnail: quarter_mill.png -->
<p>PomBase recently reached 250,000 annotations to controlled vocabularies and ontologies. The majority (over 90%) are assigned manually from fission yeast experimental data derived from 3776 publications, most of which report low-throughput, hypothesis-driven experiments.</p>
<p>You can query and combine any of these data types in the <a href="https://www.pombase.org/query">Advanced search</a>.</p>
<p>Thank you to everyone who contributed to this significant achievement through community curation.</p>

  </div>
  <div *ngIf="pageName == '50_percent'">
<h3 id="community-curation-response-rate-reaches-50">Community curation response rate reaches 50%</h3>
<!-- newsfeed_thumbnail: canto.png -->
<p>789/1587 publications assigned to community members for curation are finished. A big thank you to everyone who has participated so far. For more details, and all our curation metrics, see https://curation.pombase.org/pombe/stats/annotation</p>

  </div>
  <div *ngIf="pageName == 'interpro-v77'">
<h3 id="pombase-now-uses-interpro-version-77.0">PomBase now uses InterPro Version 77.0</h3>
<!-- newsfeed_thumbnail: interpro_32px.png -->
<p>PomBase now uses InterPro Version 77.0, which integrates 145 new methods from the CATH-Gene3D (134), and SUPERFAMILY (11) databases. InterPro cites 59894 publications in PubMed. See the InterPro <a href="https://www.ebi.ac.uk/interpro/release_notes/">release notes</a> for further information.</p>

  </div>
  <div *ngIf="pageName == 'angeli'">
<h3 id="angeli-update">AnGeLi update</h3>
<!-- newsfeed_thumbnail: angeli.png -->
<p>AnGeLi (developed by Danny Bitton) is a tool that allows you to perform enrichments over gene lists.</p>
<p>AnGeLi has recently been updated to provide 9320 lists, including ontology-based annotations from PomBase (as of <span class="no-break">2020-03-04</span>) as well as many additional datasets from the Bähler laboratory.</p>

  </div>
  <div *ngIf="pageName == 'new-search-result-download-options'">
<h3 id="new-search-result-download-options">New search result download options</h3>
<!-- newsfeed_thumbnail: advanced_search.png -->
<p>The PomBase <a routerLink="/query">Advanced search</a> has added new options to the data you can download for your query results:</p>
<ul>
<li>All physical interactors of a gene product</li>
<li>Deletion viability</li>
<li>Protein length</li>
</ul>

  </div>
  <div *ngIf="pageName == 'pombetalks'">
<h3 id="pombetalks-start-on-april-29th">PombeTalks start on April 29th</h3>
<!-- newsfeed_thumbnail: PombeTalks32px.png -->
<p>The first in the new series of online PombeTalks will take place on Wednesday 29 April 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>. Speakers:</p>
<ul>
<li><p><strong>Aleksandar Vjeṧtica</strong>, Sophie Martin’s lab, University of Lausanne: <em>Cycling for reproductive fidelity: Coupling the cell cycle and re-fertilisation blocks ensures ploidy maintenance during sexual lifecycle</em></p></li>
<li><p><strong>Haitong Hou</strong>, Julia Cooper’s lab, NCI &amp; University of Colorado: <em>Centromeres are dismantled by foundational meiotic proteins Spo11 and Rec8</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also mark your calendars for the next two sessions on <strong>May 13</strong> and <strong>May 27</strong>, and if you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>

  </div>
  <div *ngIf="pageName == 'midori-biocuration-career-award'">
<h3 id="midori-harris-receives-2020-biocuration-career-award">Midori Harris receives 2020 Biocuration Career Award</h3>
<!-- newsfeed_thumbnail: isb_logo_32px.png -->
<p>Midori Harris, ontology developer and curator at PomBase, has been awarded the <a href="https://www.biocuration.org/midori-harris-recipient-of-the-2020-biocuration-award/">2020 Biocuration Career Award</a>.</p>
<p>Congratulations to Midori and a huge thanks for all that you do for PomBase.</p>

  </div>
  <div *ngIf="pageName == 'mitochondrial-genome-update'">
<h3 id="mitochondrial-genome-update">Mitochondrial genome update</h3>
<!-- newsfeed_thumbnail: mitogenome.png -->
<p>The mitochondrial genome sequence in PomBase has been updated to reflect corrections made in <a href="https://www.ncbi.nlm.nih.gov/pubmed/31364709">Tao et al. (2019) “Intraspecific Diversity of Fission Yeast Mitochondrial Genomes”</a>.</p>

  </div>
  <div *ngIf="pageName == 'interpro-v79'">
<h3 id="pombase-now-uses-interpro-version-79.0">PomBase now uses InterPro Version 79.0</h3>
<!-- newsfeed_thumbnail: interpro_32px.png -->
<p>PomBase now uses InterPro Version 79.0, which integrates:</p>
<ul>
<li>128 new InterPro entries</li>
<li>An update to PIRSF [3.10]</li>
<li>151 new methods from the SUPERFAMILY (4), CATH-Gene3D (6), PIRSF (9), PANTHER (106), and CDD (26) databases.</li>
</ul>
<p>InterPro cites 48466 publications in PubMed. See the <a href="https://www.ebi.ac.uk/interpro/release_notes/">InterPro release notes</a> for further information.</p>

  </div>
  <div *ngIf="pageName == 'chromatin-silencing'">
<h3 id="chromatin-silencing-ontology-annotation-overhaul">Chromatin silencing ontology &amp; annotation overhaul</h3>
<!-- newsfeed_thumbnail: nucleosome.png -->
<p>PomBase curators have collaborated with the GO Consortium to improve the representation of chromatin silencing and the underlying heterochromatin organization processes in the GO biological process ontology and annotations.</p>
<p>Notably, “chromatin silencing” terms have been removed from GO on the grounds that they conflated various heterochromatin assembly, formation, and maintenance pathways with processes that affect chromatin-mediated repression more indirectly (e.g. tethering to the nuclear envelope). Chromatin silencing is a phenotype resulting from the cumulative effects of these processes, and the Fission Yeast Phenotype Ontology (FYPO) accordingly retains a full suite of “chromatin silencing” terms.</p>
<p>Annotations using the GO chromatin silencing terms were reviewed, and either removed or reannotated based on what could be inferred from the phenotypes, resulting in a substantially revised set of <a href="https://www.pombase.org/term/GO:0031507">heterochromatin assembly annotations</a>. Further work is required, so please <a href="mailto:helpdesk@pombase.org">send us</a> any corrections.</p>

  </div>
  <div *ngIf="pageName == 'pombetalks-may-13'">
<h3 id="pombetalks-may-13th">PombeTalks May 13th</h3>
<!-- newsfeed_thumbnail: PombeTalks32px.png -->
<p>The next online PombeTalks will take place on Wednesday 13 May 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>. Speakers:</p>
<ul>
<li><p><strong>Sarah Lambert</strong>, Institut Curie, Paris, France: <em>Resolution of replication stress in space and time for maintaining genome stability</em></p></li>
<li><p><strong>Cornelia Kilchert</strong>, Justus-Liebig-University, Giessen, Germany: <em>RNA-binding proteins in fission yeast - a global perspective</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also mark your calendars for the next two sessions on <strong>May 27</strong> and <strong>June 10</strong>, and if you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>

  </div>
  <div *ngIf="pageName == 'community-curation-publication'">
<h3 id="published-community-curation-in-pombase">Published: Community curation in PomBase</h3>
<!-- newsfeed_thumbnail: canto.png -->
<p>The PomBase team has published an overview of our experience with community curation for fission yeast. In the article, out this week in <a href="https://academic.oup.com/database/article/doi/10.1093/database/baaa028/5827230">Database</a>, we reflect on the factors that have made our community’s remarkable, standard-setting achievements possible, and on the benefits we and PomBase users derive from this effort. We highlight the collaboration between authors and professional curators that arises via community curation, and how annotation quality improves as a result.</p>
<p>Watch for invitations to curate your new papers, or see our <a href="https://www.pombase.org/community/fission-yeast-community-curation-project">community curation page</a> for more information.</p>

  </div>
  <div *ngIf="pageName == 'increased-disease-annotation-coverage'">
<h3 id="improved-gene-disease-curation">Improved gene-disease curation</h3>
<!-- newsfeed_thumbnail: human disease.png -->
<p>PomBase disease gene curation associates disease descriptors with fission yeast orthologs of human disease-causing genes. We have now increased coverage by adding new gene–disease term connections, with 3954 individual annotations to 1195 genes (up from 2588 and 905 respectively in January 2019). Disease associations now cover 24.5% of all fission yeast protein-coding genes, and over one third of those with human orthologs.</p>

  </div>
  <div *ngIf="pageName == 'pombase-uses-mondo'">
<h3 id="pombase-adopts-mondo-for-disease-gene-curation">PomBase adopts MONDO for disease gene curation</h3>
<!-- newsfeed_thumbnail: human disease.png -->
<p>PomBase has switched from the <a href="https://disease-ontology.org/">Disease Ontology</a> (DO) to the Monarch Initiative’s <a href="https://mondo.monarchinitiative.org/">Mondo Disease Ontology</a> (Mondo) for disease gene curation. Mondo covers the same set of disease descriptions as DO, but has a richer hierarchical structure that classifies more specific descriptions into broad categories (e.g. anemia, cancer, kidney disease) suitable for a disease “slim” term set.</p>
<p>PomBase curators are collaborating with Mondo to improve its disease classification, especially in areas that will support inferences that improve fission yeast disease annotation coverage in the new <a href="https://www.pombase.org/browse-curation/disease-slim">PomBase Mondo slim</a>. The new disease slim is a work in progress, so if there is a particular disease grouping that you would find useful, please <a href="mailto:helpdesk@pombase.org">let us know</a>.</p>

  </div>
  <div *ngIf="pageName == 'advanced-search-slim-options'">
<h3 id="new-ontology-slimming-options-for-advanced-search-results">New ontology slimming options for advanced search results</h3>
<!-- newsfeed_thumbnail: dropdown_menu.png -->
<p>The PomBase <a routerLink="/query">advanced search</a> results panel now allows you to retrieve annotations to any of the fission yeast <a href="documentation/pombase-go-slim-documentation">GO slims</a> or the <a href="https://www.pombase.org/browse-curation/disease-slim">Mondo disease slim</a> for genes in the results list. For example, you can query for all genes involved in a process and slim the resulting list by molecular function or disease association.</p>

  </div>
  <div *ngIf="pageName == 'new-fission-yeast-go-slims'">
<h3 id="new-fission-yeast-go-slims">New fission yeast GO slims</h3>
<!-- newsfeed_thumbnail: dropdown_menu.png -->
<p>To complement the overview provided by the <a href="/browse-curation/fission-yeast-bp-go-slim-terms">fission yeast GO biological process slim</a>, we have created GO slims for the <a href="/browse-curation/fission-yeast-mf-go-slim-terms">molecular function</a> and <a routerLink="/browse-curation/fission-yeast-cc-go-slim-terms">cellular component</a> branches of GO. Each slim page provides links to ontology term pages, annotated genes, and to download slim terms and IDs from the PomBase ftp site.</p>

  </div>
  <div *ngIf="pageName == 'pombetalks-may-27'">
<h3 id="pombetalks-may-27th">PombeTalks May 27th</h3>
<!-- newsfeed_thumbnail: PombeTalks32px.png -->
<p>The next online PombeTalks will take place on Wednesday 27 May 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>. This time, in addition to the usual pair of research talks, our own Val Wood will show a few of PomBase’s lesser-known features.</p>
<ul>
<li><p><strong>Angad Garg</strong>, Stewart Schuman’s lab, Memorial Sloan Kettering Cancer Center: <em>Long non-coding RNA control of phosphate homeostasis</em></p></li>
<li><p><strong>José López Hernández</strong>, Sarah Zander’s lab, Stowers Institute for Medical Research: <em>Diverse mating strategies in S. pombe affect the spread of wtf meiotic drivers</em></p></li>
<li><p><strong>Val Wood</strong>, PomBase: <em>Hidden corners of PomBase: Ten features you might not have seen</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also mark your calendars for the next two sessions on and <strong>June 10</strong> and <strong>24</strong>, and if you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>

  </div>
  <div *ngIf="pageName == 'pombetalks-june-10'">
<h3 id="pombetalks-postponed-to-june-17th">PombeTalks POSTPONED to June 17th</h3>
<!-- newsfeed_thumbnail: PombeTalks32px.png -->
<p>Please note that the next online PombeTalks will take place one week later than originally planned, to support the <a href="https://www.shutdownstem.com/">STEM Strike for Black Lives</a> on 10th June.</p>
<p>In the meantime, please complete this brief <a href="https://forms.gle/ANVsCcfaqf2p4Q9fA">survey</a> of the audience.</p>
<p>On Wednesday <strong>17th June</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>, the speakers will be:</p>
<ul>
<li><p><strong>Gautam Dey</strong>, Baum lab, UCL / EMBL Heidelberg: <em>Closed mitosis requires local disassembly of the nuclear envelope</em></p></li>
<li><p><strong>Meredith Betterton</strong>, UC Boulder: <em>Computational modeling of fission yeast mitosis: what we can learn about pombe from computer simulations</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. The next two sessions will b on <strong>June 27</strong> and <strong>July 8</strong>. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>

  </div>
  <div *ngIf="pageName == 'name-queries'">
<h3 id="editable-pombase-query-names">Editable PomBase query names</h3>
<!-- pombase_flags: frontpage -->
<!-- newsfeed_thumbnail: advanced_search.png -->
<p>Entries in the <a routerLink="/query">PomBase advanced search</a> query history now show brief, user-editable query descriptions, and a toggle to show or hide additional details.</p>

  </div>
  <div *ngIf="pageName == 'pombetalks-june-24'">
<h3 id="pombetalks-june-24th">PombeTalks June 24th</h3>
<!-- newsfeed_thumbnail: PombeTalks32px.png -->
<p>The next online PombeTalks will take place on Wednesday <strong>24th June</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Sito Torres-Garcia</strong>, Allshire lab, University of Edinburgh: <em>Epigenetic gene silencing by heterochromatin primes fungal resistance</em></p></li>
<li><p><strong>Julie Rich-Robinson</strong>, Das lab, University of Tennessee: <em>Cell-cycle-dependent cues temporally regulate Cdc42 activity at growth sites in fission yeast</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up.</p>
<p>A <a href="https://researchseminars.org/seminar/pombeTalks">schedule</a> is now available for the rest of the summer, including the next talks on <strong>July 8th</strong>. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>

  </div>
  <div *ngIf="pageName == 'pombetalks-july-8'">
<h3 id="pombetalks-july-8th">PombeTalks July 8th</h3>
<!-- newsfeed_thumbnail: PombeTalks32px.png -->
<p>The next online PombeTalks will take place on Wednesday <strong>8th July</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Sahana Holla</strong>, Grewal lab, NIH: <em>Positioning heterochromatin at the nuclear periphery promotes epigenetic inheritance</em></p></li>
<li><p><strong>Nick Rhind</strong>, UMass Medical School: <em>Cell size is controlled by size-dependent expression of mitotic activators</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>July 22nd</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">summer schedule</a> is available. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>

  </div>
  <div *ngIf="pageName == 'pombetalks-july-22'">
<h3 id="pombetalks-july-22nd">PombeTalks July 22nd</h3>
<!-- newsfeed_thumbnail: PombeTalks32px.png -->
<p>The next online PombeTalks will take place on Wednesday <strong>22nd July</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Prof. Dr. Ann Ehrenhofer-Murray</strong>, Institut für Biologie, Humboldt-Universität zu Berlin: <em>Queuosine and m5c modification of RNA: Nutritional control of translation in S. pombe homestasis</em></p></li>
<li><p><strong>Dr. Sarah Sabatinos</strong>, Department of Chemistry and Biology, Ryerson University: <em>Long-term effects of surviving replication instability</em></p></li>
<li><p><strong>PomBase</strong> microPublications announcement (Midori Harris)</p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>August 5</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">summer schedule</a> is available. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>

  </div>
  <div *ngIf="pageName == 'fission-yeast-micropublications'">
<h3 id="fission-yeast-micropublications">Fission yeast microPublications</h3>
<!-- pombase_flags: frontpage -->
<!-- newsfeed_thumbnail: micropubs32px.png -->
<p>PomBase has recently joined <a href="https://www.micropublication.org/">microPublication.org</a>, which “publishes brief, novel findings, negative and/or reproduced results, and results which may lack a broader scientific narrative”, as a Partner Database. Fission yeast researchers can thus now make any results available to the community, even those that don’t fit neatly into traditional publications.</p>
<p>Visit the <a href="https://www.micropublication.org/">microPublications website</a> to learn more, to <a href="https://www.micropublication.org/submit-new-article/">register</a> and submit your data, or sign up to review. Send questions to the <a href="mailto:helpdesk@pombase.org">PomBase helpdesk</a>.</p>

  </div>
  <div *ngIf="pageName == 'pombetalks-aug-5'">
<h3 id="pombetalks-august-5th">PombeTalks August 5th</h3>
<!-- newsfeed_thumbnail: PombeTalks32px.png -->
<p>The next online PombeTalks will take place on Wednesday <strong>5th August</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Feng Li</strong>, Levin Lab NICHD/NIH, USA: <em>Identification of an integrase-independent pathway of retrotransposition</em></p></li>
<li><p><strong>Ivan Surovtsev</strong>, King lab, Yale University, USA: <em>Liquid-liquid phase separation, heterochromatin domains and nuclear mechanics</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>August 19</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">summer schedule</a> is available. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>

  </div>
  <div *ngIf="pageName == 'pombetalks-aug-19'">
<h3 id="pombetalks-august-19th">PombeTalks August 19th</h3>
<!-- newsfeed_thumbnail: PombeTalks32px.png -->
<p>The next online PombeTalks will take place on Wednesday <strong>19th August</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Joe Magliozzi</strong>, Moseley Lab, Dartmouth: <em>Cell polarity kinases regulate RNA-binding protein Sts5 to control cell shape</em></p></li>
<li><p><strong>Ramakanth Neeli</strong>, Minc Lab, Institute Jacques Monod: <em>Mechanisms and Functions of Cell Wall Mechanosensing in Fission Yeast</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>September 2</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">schedule</a> is available for the next few weeks. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>

  </div>
  <div *ngIf="pageName == '11th-pombe-conference-postponed'">
<h3 id="th-pombe-meeting-postponed-to-2022">11th Pombe meeting postponed to 2022</h3>
<!-- pombase_flags: frontpage -->
<!-- newsfeed_thumbnail: pombe2022-32px.png -->
<p>Due to the ongoing Covid-19 pandemic, the <a href="http://www.fission-yeast2022.org/">11th International Fission Yeast Meeting</a>, due to take place in Hiroshima, Japan, has been postponed.</p>
<p>The new dates will be <strong>12th (Sun -17th (Fri) June, 2022</strong>.</p>
<p>Please see the <a href="http://www.fission-yeast2022.org/">conference website</a> and pombelist for further announcements.</p>

  </div>
  <div *ngIf="pageName == 'pombetalks-sept-2'">
<h3 id="pombetalks-september-2nd">PombeTalks September 2nd</h3>
<!-- pombase_flags: frontpage -->
<!-- newsfeed_thumbnail: PombeTalks32px.png -->
<p>The next online PombeTalks will take place on Wednesday <strong>2nd September</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>François Bachand</strong>, USherbrooke, Canada: <em>Proximity-dependent biotinylation assays in fission yeast and a tale about slow RNA polymerase II transcription</em></p></li>
<li><p><strong>Scott Curran</strong>, Nurse Lab, The Crick Institute, UK: <em>A quantitative and spatial analysis of the cell cycle control network</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>September 16</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">schedule</a> is available for the next few weeks. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>

  </div>
  <div *ngIf="pageName == 'GO-term-matrix-publication'">
<h3 id="published-go-term-matrix-for-annotation-qc">Published: GO Term Matrix for annotation QC</h3>
<!-- pombase_flags: frontpage -->
<!-- newsfeed_thumbnail: go-term-matrix.png -->
<p>In collaboration with the GO Consortium, the PomBase team has published a report on the <strong>Term Matrix</strong> approach to GO annotation quality control. The article, out this week in <a href="https://royalsocietypublishing.org/doi/10.1098/rsob.200149">Open Biology</a>, describes biological processes that do, or don’t, share annotated gene products, and how we use co-annotation patterns to build rules to detect, correct, and prevent errors.</p>

  </div>
  <div *ngIf="pageName == 'index'">
<h2 id="news-archive">News archive</h2>
<div class="news-archive">
<div id="2020-09-02-GO-term-matrix-publication" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/go-term-matrix.png"/></p>
<h3 id="published-go-term-matrix-annotation-qc">Published: GO Term Matrix for annotation QC</h3>
<p><em>2020-09-02</em></p>
<p>In collaboration with the GO Consortium, the PomBase team has published a report on the <strong>Term Matrix</strong> approach to GO annotation quality control. The article, out this week in <a href="https://royalsocietypublishing.org/doi/10.1098/rsob.200149">Open Biology</a>, describes biological processes that do, or don’t, share annotated gene products, and how we use co-annotation patterns to build rules to detect, correct, and prevent errors.</p>
</div>
<div id="2020-09-01-pombetalks-sept-2" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-september-nd">PombeTalks September 2nd</h3>
<p><em>2020-09-01</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>2nd September</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>François Bachand</strong>, USherbrooke, Canada: <em>Proximity-dependent biotinylation assays in fission yeast and a tale about slow RNA polymerase II transcription</em></p></li>
<li><p><strong>Scott Curran</strong>, Nurse Lab, The Crick Institute, UK: <em>A quantitative and spatial analysis of the cell cycle control network</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>September 16</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">schedule</a> is available for the next few weeks. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-08-19-11th-pombe-conference-postponed" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/pombe2022-32px.png"/></p>
<h3 id="th-pombe-meeting-postponed">11th Pombe meeting postponed to 2022</h3>
<p><em>2020-08-19</em></p>
<p>Due to the ongoing Covid-19 pandemic, the <a href="http://www.fission-yeast2022.org/">11th International Fission Yeast Meeting</a>, due to take place in Hiroshima, Japan, has been postponed.</p>
<p>The new dates will be <strong>12th (Sun -17th (Fri) June, 2022</strong>.</p>
<p>Please see the <a href="http://www.fission-yeast2022.org/">conference website</a> and pombelist for further announcements.</p>
</div>
<div id="2020-08-12-pombetalks-aug-19" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-august-th">PombeTalks August 19th</h3>
<p><em>2020-08-12</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>19th August</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Joe Magliozzi</strong>, Moseley Lab, Dartmouth: <em>Cell polarity kinases regulate RNA-binding protein Sts5 to control cell shape</em></p></li>
<li><p><strong>Ramakanth Neeli</strong>, Minc Lab, Institute Jacques Monod: <em>Mechanisms and Functions of Cell Wall Mechanosensing in Fission Yeast</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>September 2</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">schedule</a> is available for the next few weeks. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-07-31-pombetalks-aug-5" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-august-th">PombeTalks August 5th</h3>
<p><em>2020-07-31</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>5th August</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Feng Li</strong>, Levin Lab NICHD/NIH, USA: <em>Identification of an integrase-independent pathway of retrotransposition</em></p></li>
<li><p><strong>Ivan Surovtsev</strong>, King lab, Yale University, USA: <em>Liquid-liquid phase separation, heterochromatin domains and nuclear mechanics</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>August 19</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">summer schedule</a> is available. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-07-22-fission-yeast-micropublications" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/micropubs32px.png"/></p>
<h3 id="fission-yeast-micropublications">Fission yeast microPublications</h3>
<p><em>2020-07-22</em></p>
<p>PomBase has recently joined <a href="https://www.micropublication.org/">microPublication.org</a>, which “publishes brief, novel findings, negative and/or reproduced results, and results which may lack a broader scientific narrative”, as a Partner Database. Fission yeast researchers can thus now make any results available to the community, even those that don’t fit neatly into traditional publications.</p>
<p>Visit the <a href="https://www.micropublication.org/">microPublications website</a> to learn more, to <a href="https://www.micropublication.org/submit-new-article/">register</a> and submit your data, or sign up to review. Send questions to the <a href="mailto:helpdesk@pombase.org">PomBase helpdesk</a>.</p>
</div>
<div id="2020-07-21-pombetalks-july-22" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-july-nd">PombeTalks July 22nd</h3>
<p><em>2020-07-21</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>22nd July</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Prof. Dr. Ann Ehrenhofer-Murray</strong>, Institut für Biologie, Humboldt-Universität zu Berlin: <em>Queuosine and m5c modification of RNA: Nutritional control of translation in S. pombe homestasis</em></p></li>
<li><p><strong>Dr. Sarah Sabatinos</strong>, Department of Chemistry and Biology, Ryerson University: <em>Long-term effects of surviving replication instability</em></p></li>
<li><p><strong>PomBase</strong> microPublications announcement (Midori Harris)</p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>August 5</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">summer schedule</a> is available. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-07-02-pombetalks-july-8" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-july-th">PombeTalks July 8th</h3>
<p><em>2020-07-02</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>8th July</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Sahana Holla</strong>, Grewal lab, NIH: <em>Positioning heterochromatin at the nuclear periphery promotes epigenetic inheritance</em></p></li>
<li><p><strong>Nick Rhind</strong>, UMass Medical School: <em>Cell size is controlled by size-dependent expression of mitotic activators</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also see PombeSlack for Q&amp;A after the talks, and for recordings of previous sessions.</p>
<p>The next talks are on <strong>July 22nd</strong>, and the <a href="https://researchseminars.org/seminar/pombeTalks">summer schedule</a> is available. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-06-19-pombetalks-june-24" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-june-th">PombeTalks June 24th</h3>
<p><em>2020-06-19</em></p>
<p>The next online PombeTalks will take place on Wednesday <strong>24th June</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>:</p>
<ul>
<li><p><strong>Sito Torres-Garcia</strong>, Allshire lab, University of Edinburgh: <em>Epigenetic gene silencing by heterochromatin primes fungal resistance</em></p></li>
<li><p><strong>Julie Rich-Robinson</strong>, Das lab, University of Tennessee: <em>Cell-cycle-dependent cues temporally regulate Cdc42 activity at growth sites in fission yeast</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up.</p>
<p>A <a href="https://researchseminars.org/seminar/pombeTalks">schedule</a> is now available for the rest of the summer, including the next talks on <strong>July 8th</strong>. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-06-11-name-queries" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/advanced_search.png"/></p>
<h3 id="editable-pombase-query-names">Editable PomBase query names</h3>
<p><em>2020-06-11</em></p>
<p>Entries in the <a routerLink="/query">PomBase advanced search</a> query history now show brief, user-editable query descriptions, and a toggle to show or hide additional details.</p>
</div>
<div id="2020-06-05-pombetalks-june-10" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-postponed-june-th">PombeTalks POSTPONED to June 17th</h3>
<p><em>2020-06-05</em></p>
<p>Please note that the next online PombeTalks will take place one week later than originally planned, to support the <a href="https://www.shutdownstem.com/">STEM Strike for Black Lives</a> on 10th June.</p>
<p>In the meantime, please complete this brief <a href="https://forms.gle/ANVsCcfaqf2p4Q9fA">survey</a> of the audience.</p>
<p>On Wednesday <strong>17th June</strong> 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>, the speakers will be:</p>
<ul>
<li><p><strong>Gautam Dey</strong>, Baum lab, UCL / EMBL Heidelberg: <em>Closed mitosis requires local disassembly of the nuclear envelope</em></p></li>
<li><p><strong>Meredith Betterton</strong>, UC Boulder: <em>Computational modeling of fission yeast mitosis: what we can learn about pombe from computer simulations</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. The next two sessions will b on <strong>June 27</strong> and <strong>July 8</strong>. If you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-05-21-pombetalks-may-27" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-may-th">PombeTalks May 27th</h3>
<p><em>2020-05-21</em></p>
<p>The next online PombeTalks will take place on Wednesday 27 May 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>. This time, in addition to the usual pair of research talks, our own Val Wood will show a few of PomBase’s lesser-known features.</p>
<ul>
<li><p><strong>Angad Garg</strong>, Stewart Schuman’s lab, Memorial Sloan Kettering Cancer Center: <em>Long non-coding RNA control of phosphate homeostasis</em></p></li>
<li><p><strong>José López Hernández</strong>, Sarah Zander’s lab, Stowers Institute for Medical Research: <em>Diverse mating strategies in S. pombe affect the spread of wtf meiotic drivers</em></p></li>
<li><p><strong>Val Wood</strong>, PomBase: <em>Hidden corners of PomBase: Ten features you might not have seen</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also mark your calendars for the next two sessions on and <strong>June 10</strong> and <strong>24</strong>, and if you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-05-20-new-fission-yeast-go-slims" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/dropdown_menu.png"/></p>
<h3 id="new-fission-yeast-go-slims">New fission yeast GO slims</h3>
<p><em>2020-05-20</em></p>
<p>To complement the overview provided by the <a href="/browse-curation/fission-yeast-bp-go-slim-terms">fission yeast GO biological process slim</a>, we have created GO slims for the <a href="/browse-curation/fission-yeast-mf-go-slim-terms">molecular function</a> and <a routerLink="/browse-curation/fission-yeast-cc-go-slim-terms">cellular component</a> branches of GO. Each slim page provides links to ontology term pages, annotated genes, and to download slim terms and IDs from the PomBase ftp site.</p>
</div>
<div id="2020-05-20-advanced-search-slim-options" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/dropdown_menu.png"/></p>
<h3 id="new-ontology-slimming-options-advanced-search-results">New ontology slimming options for advanced search results</h3>
<p><em>2020-05-20</em></p>
<p>The PomBase <a routerLink="/query">advanced search</a> results panel now allows you to retrieve annotations to any of the fission yeast <a href="documentation/pombase-go-slim-documentation">GO slims</a> or the <a href="https://www.pombase.org/browse-curation/disease-slim">Mondo disease slim</a> for genes in the results list. For example, you can query for all genes involved in a process and slim the resulting list by molecular function or disease association.</p>
</div>
<div id="2020-05-18-pombase-uses-mondo" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/human disease.png"/></p>
<h3 id="pombase-adopts-mondo-disease-gene-curation">PomBase adopts MONDO for disease gene curation</h3>
<p><em>2020-05-18</em></p>
<p>PomBase has switched from the <a href="https://disease-ontology.org/">Disease Ontology</a> (DO) to the Monarch Initiative’s <a href="https://mondo.monarchinitiative.org/">Mondo Disease Ontology</a> (Mondo) for disease gene curation. Mondo covers the same set of disease descriptions as DO, but has a richer hierarchical structure that classifies more specific descriptions into broad categories (e.g. anemia, cancer, kidney disease) suitable for a disease “slim” term set.</p>
<p>PomBase curators are collaborating with Mondo to improve its disease classification, especially in areas that will support inferences that improve fission yeast disease annotation coverage in the new <a href="https://www.pombase.org/browse-curation/disease-slim">PomBase Mondo slim</a>. The new disease slim is a work in progress, so if there is a particular disease grouping that you would find useful, please <a href="mailto:helpdesk@pombase.org">let us know</a>.</p>
</div>
<div id="2020-05-18-increased-disease-annotation-coverage" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/human disease.png"/></p>
<h3 id="improved-gene-disease-curation">Improved gene-disease curation</h3>
<p><em>2020-05-18</em></p>
<p>PomBase disease gene curation associates disease descriptors with fission yeast orthologs of human disease-causing genes. We have now increased coverage by adding new gene–disease term connections, with 3954 individual annotations to 1195 genes (up from 2588 and 905 respectively in January 2019). Disease associations now cover 24.5% of all fission yeast protein-coding genes, and over one third of those with human orthologs.</p>
</div>
<div id="2020-05-11-community-curation-publication" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/canto.png"/></p>
<h3 id="published-community-curation-pombase">Published: Community curation in PomBase</h3>
<p><em>2020-05-11</em></p>
<p>The PomBase team has published an overview of our experience with community curation for fission yeast. In the article, out this week in <a href="https://academic.oup.com/database/article/doi/10.1093/database/baaa028/5827230">Database</a>, we reflect on the factors that have made our community’s remarkable, standard-setting achievements possible, and on the benefits we and PomBase users derive from this effort. We highlight the collaboration between authors and professional curators that arises via community curation, and how annotation quality improves as a result.</p>
<p>Watch for invitations to curate your new papers, or see our <a href="https://www.pombase.org/community/fission-yeast-community-curation-project">community curation page</a> for more information.</p>
</div>
<div id="2020-05-07-pombetalks-may-13" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-may-th">PombeTalks May 13th</h3>
<p><em>2020-05-07</em></p>
<p>The next online PombeTalks will take place on Wednesday 13 May 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>. Speakers:</p>
<ul>
<li><p><strong>Sarah Lambert</strong>, Institut Curie, Paris, France: <em>Resolution of replication stress in space and time for maintaining genome stability</em></p></li>
<li><p><strong>Cornelia Kilchert</strong>, Justus-Liebig-University, Giessen, Germany: <em>RNA-binding proteins in fission yeast - a global perspective</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also mark your calendars for the next two sessions on <strong>May 27</strong> and <strong>June 10</strong>, and if you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-05-07-chromatin-silencing" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/nucleosome.png"/></p>
<h3 id="chromatin-silencing-ontology-amp-annotation-overhaul">Chromatin silencing ontology &amp; annotation overhaul</h3>
<p><em>2020-05-07</em></p>
<p>PomBase curators have collaborated with the GO Consortium to improve the representation of chromatin silencing and the underlying heterochromatin organization processes in the GO biological process ontology and annotations.</p>
<p>Notably, “chromatin silencing” terms have been removed from GO on the grounds that they conflated various heterochromatin assembly, formation, and maintenance pathways with processes that affect chromatin-mediated repression more indirectly (e.g. tethering to the nuclear envelope). Chromatin silencing is a phenotype resulting from the cumulative effects of these processes, and the Fission Yeast Phenotype Ontology (FYPO) accordingly retains a full suite of “chromatin silencing” terms.</p>
<p>Annotations using the GO chromatin silencing terms were reviewed, and either removed or reannotated based on what could be inferred from the phenotypes, resulting in a substantially revised set of <a href="https://www.pombase.org/term/GO:0031507">heterochromatin assembly annotations</a>. Further work is required, so please <a href="mailto:helpdesk@pombase.org">send us</a> any corrections.</p>
</div>
<div id="2020-05-05-interpro-v79" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-now-uses-interpro-version-.">PomBase now uses InterPro Version 79.0</h3>
<p><em>2020-05-05</em></p>
<p>PomBase now uses InterPro Version 79.0, which integrates:</p>
<ul>
<li>128 new InterPro entries</li>
<li>An update to PIRSF [3.10]</li>
<li>151 new methods from the SUPERFAMILY (4), CATH-Gene3D (6), PIRSF (9), PANTHER (106), and CDD (26) databases.</li>
</ul>
<p>InterPro cites 48466 publications in PubMed. See the <a href="https://www.ebi.ac.uk/interpro/release_notes/">InterPro release notes</a> for further information.</p>
</div>
<div id="2020-05-01-mitochondrial-genome-update" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/mitogenome.png"/></p>
<h3 id="mitochondrial-genome-update">Mitochondrial genome update</h3>
<p><em>2020-05-01</em></p>
<p>The mitochondrial genome sequence in PomBase has been updated to reflect corrections made in <a href="https://www.ncbi.nlm.nih.gov/pubmed/31364709">Tao et al. (2019) “Intraspecific Diversity of Fission Yeast Mitochondrial Genomes”</a>.</p>
</div>
<div id="2020-04-28-midori-biocuration-career-award" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/isb_logo_32px.png"/></p>
<h3 id="midori-harris-receives-biocuration-career-award">Midori Harris receives 2020 Biocuration Career Award</h3>
<p><em>2020-04-28</em></p>
<p>Midori Harris, ontology developer and curator at PomBase, has been awarded the <a href="https://www.biocuration.org/midori-harris-recipient-of-the-2020-biocuration-award/">2020 Biocuration Career Award</a>.</p>
<p>Congratulations to Midori and a huge thanks for all that you do for PomBase.</p>
</div>
<div id="2020-04-22-pombetalks" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/PombeTalks32px.png"/></p>
<h3 id="pombetalks-start-on-april-th">PombeTalks start on April 29th</h3>
<p><em>2020-04-22</em></p>
<p>The first in the new series of online PombeTalks will take place on Wednesday 29 April 2020 at 17:00 <a href="https://greenwichmeantime.com/time-zone/europe/european-union/central-european-time/">Central European Time</a>. Speakers:</p>
<ul>
<li><p><strong>Aleksandar Vjeṧtica</strong>, Sophie Martin’s lab, University of Lausanne: <em>Cycling for reproductive fidelity: Coupling the cell cycle and re-fertilisation blocks ensures ploidy maintenance during sexual lifecycle</em></p></li>
<li><p><strong>Haitong Hou</strong>, Julia Cooper’s lab, NCI &amp; University of Colorado: <em>Centromeres are dismantled by foundational meiotic proteins Spo11 and Rec8</em></p></li>
</ul>
<p>Talks will be streamed live via <a href="https://zoom.us/">Zoom</a>. The link will be circulated via <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">pombelist</a> and PombeSlack one day in advance, so make sure you’ve signed up. Also mark your calendars for the next two sessions on <strong>May 13</strong> and <strong>May 27</strong>, and if you’re interested in speaking, please fill out <a href="https://docs.google.com/forms/d/e/1FAIpQLSdjnkJfadUwM2eKIBJBQXeLt3aOfzrQEb3D8lvNym1g93DIRQ/viewform">this form</a>.</p>
<p>Thanks to Gautam Dey and the rest of the PombeTalks team for organizing. See you online!</p>
</div>
<div id="2020-03-23-angeli" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/angeli.png"/></p>
<h3 id="angeli-update">AnGeLi update</h3>
<p><em>2020-03-23</em></p>
<p>AnGeLi (developed by Danny Bitton) is a tool that allows you to perform enrichments over gene lists.</p>
<p>AnGeLi has recently been updated to provide 9320 lists, including ontology-based annotations from PomBase (as of <span class="no-break">2020-03-04</span>) as well as many additional datasets from the Bähler laboratory.</p>
</div>
<div id="2020-03-23-new-search-result-download-options" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/advanced_search.png"/></p>
<h3 id="new-search-result-download-options">New search result download options</h3>
<p><em>2020-03-23</em></p>
<p>The PomBase <a routerLink="/query">Advanced search</a> has added new options to the data you can download for your query results:</p>
<ul>
<li>All physical interactors of a gene product</li>
<li>Deletion viability</li>
<li>Protein length</li>
</ul>
</div>
<div id="2020-03-01-interpro-v77" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-now-uses-interpro-version-.">PomBase now uses InterPro Version 77.0</h3>
<p><em>2020-03-01</em></p>
<p>PomBase now uses InterPro Version 77.0, which integrates 145 new methods from the CATH-Gene3D (134), and SUPERFAMILY (11) databases. InterPro cites 59894 publications in PubMed. See the InterPro <a href="https://www.ebi.ac.uk/interpro/release_notes/">release notes</a> for further information.</p>
</div>
<div id="2020-02-28-50_percent" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/canto.png"/></p>
<h3 id="community-curation-response-rate-reaches">Community curation response rate reaches 50%</h3>
<p><em>2020-02-28</em></p>
<p>789/1587 publications assigned to community members for curation are finished. A big thank you to everyone who has participated so far. For more details, and all our curation metrics, see https://curation.pombase.org/pombe/stats/annotation</p>
</div>
<div id="2020-02-20-quarter_million" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/quarter_mill.png"/></p>
<h3 id="a-quarter-million-annotations">A quarter of a million annotations</h3>
<p><em>2020-02-20</em></p>
<p>PomBase recently reached 250,000 annotations to controlled vocabularies and ontologies. The majority (over 90%) are assigned manually from fission yeast experimental data derived from 3776 publications, most of which report low-throughput, hypothesis-driven experiments.</p>
<p>You can query and combine any of these data types in the <a href="https://www.pombase.org/query">Advanced search</a>.</p>
<p>Thank you to everyone who contributed to this significant achievement through community curation.</p>
</div>
<div id="2020-02-05-advanced-search-phenotype-conditions" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/advanced_search.png"/></p>
<h3 id="query-phenotypes-conditions">Query phenotypes for conditions</h3>
<p><em>2020-02-05</em></p>
<p>The PomBase advanced search <a routerLink="/query">Advanced search</a> now supports using experimental conditions as search criteria for phenotype annotations. For example, you can now query for genes that show abnormal chromosome segregation mutant phenotypes specifically at high or low temperatures. The search uses the same condition descriptors as Canto and the PomBase web pages.</p>
<p>Note that phenotype queries that have condition constraints can be combined, but pay careful attention to the annotations for the results. Future work will add support for querying for multiple conditions on the same annotation, and for specifying conditions to exclude from results.</p>
</div>
<div id="2020-02-04-lmo14-strasbourg" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/poster_LMO14_32px.png"/></p>
<h3 id="levures-mod-les-et-outils-th-international-conference">Levures, Modèles et Outils 14th International Conference</h3>
<p><em>2020-02-04</em></p>
<p>The 14th edition of the <a href="https://lmo-14-2020.sciencesconf.org/">“Levures, Modèles et Outils” meeting (LMO14)</a> will be held in July 9-11, 2020, at the University of Strasbourg in France. Registration is open February 3rd to June 30th, and abstracts can be submitted from February 3rd to April 10th. Authors will be notified in early May and the final program will be available in early June.</p>
<p>The sessions will be diverse and present the latest findings using yeast as a model organism on the following topics:</p>
<ol type="1">
<li>Cell biology, cell cycle, cytoskeleton</li>
<li>Gene expression regulation</li>
<li>Population, functional and evolutionary genomics</li>
<li>Replication, repair and recombination</li>
<li>Transport, sensing and signaling</li>
<li>Pathogenic yeast and filamentous fungi</li>
<li>Tools, resources and databases</li>
<li>New technologies, yeast and industry</li>
</ol>
</div>
<div id="2020-01-17-pombase-elixir-node-service" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/elixir_uk_thumb.png"/></p>
<h3 id="pombase-now-elixir-node-service">PomBase is now an ELIXIR Node Service</h3>
<p><em>2020-01-17</em></p>
<p>PomBase has been awarded Node Service status by the <a href="https://elixiruknode.org/">UK node</a> of <a href="https://elixir-europe.org/">ELIXIR</a>. ELIXIR-UK Node Services support the bioinformatics and broader biological research communities by providing training and resources that help researchers to find and share data, exchange expertise, and agree on best practices at national, European and international levels. The review panel describes PomBase as a “mature, leading model organism database which is popular, unique, well used, and has a strong user community.”</p>
</div>
<div id="2020-01-07-new-stable-integration-vectors" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/stable_pAde6_vector.png"/></p>
<h3 id="new-vectors-simple-reliable-s.-pombe-molecular-biology">New vectors for simple, reliable <em>S. pombe</em> molecular biology</h3>
<p><em>2020-01-07</em></p>
<p>To enable fission yeast researchers to manipulate <em>S. pombe</em> molecular biology reproducibly and easily, Aleks Vještica and Magdalena Marek in Sophie Martin’s lab have designed and constructed a series of simple, fully characterized plasmids.</p>
<p>The Stable Integration Vector (SIV) series provides a highly modular toolbox to introduce heterologous sequences more stably was possible with than previously available vectors. The toolkit includes antibiotic resistance markers, promoters, fluorescent tags, and terminators, as well as large set of ready-to-use fluorescent probes to mark organelles and visualize cellular processes.</p>
<p>The work is published in the <a href="https://jcs.biologists.org/content/early/2019/11/29/jcs.240754.long">Journal of Cell Science</a>, and a PomBase <a href="https://www.pombase.org/reference/PMID:31801797">publication page</a> is available.</p>
</div>
<div id="2019-11-28-shareable-result-links" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/advanced_search.png"/></p>
<h3 id="unique-permanent-urls-search-results">Unique permanent URLs for search results</h3>
<p><em>2019-11-28</em></p>
<p>All result pages from the <a routerLink="/query">Advanced search</a> now have a unique permanent URL that can be bookmarked and shared with your colleagues.</p>
<p>The <a routerLink="/documentation/quick-little-tool">QuiLT</a> and <a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">GO slim</a> pages also now have permanent URLs.</p>
<p>Examples:</p>
<ul>
<li><a routerLink="/results/from/id/7474e390-ddc0-4f70-8c2e-d6a69d471a50">All disease associated genes</a></li>
<li><a routerLink="/vis/from/id/7474e390-ddc0-4f70-8c2e-d6a69d471a50">QuiLT example for all disease associated genes</a></li>
<li><a routerLink="/slim:bp_goslim_pombe/from/id/7474e390-ddc0-4f70-8c2e-d6a69d471a50">GO slim for disease associated genes</a></li>
</ul>
</div>
<div id="2019-10-16-JBrowse-tracks-loadable-from-publication-pages" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/new_tracks.png"/></p>
<h3 id="browser-tracks-now-loadable-publication-pages">Browser tracks now loadable from publication pages</h3>
<p><em>2019-10-16</em></p>
<p>Data tracks from datasets hosted in the PomBase genome browser can now be browsed and loaded from their respective publication pages. For an example, see <a href="https://www.pombase.org/reference/PMID:29914874">Atkinson et al. (2018)</a>. Data tracks are now also downloadable from the publication pages.</p>
</div>
<div id="2019-10-07-new-wt-pombase-funding" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/pombase-logo-32x32px.png"/></p>
<h3 id="new-pombase-funding-wellcome-trust">New PomBase funding from the Wellcome Trust</h3>
<p><em>2019-10-07</em></p>
<p>We are pleased to announce that the recent PomBase application for continued Wellcome Trust funding was successful. Although the grant was not fully funded, we are confident that we can cover the shortfall by small grants for stand-alone projects and collaborations. We would like to thank the pombe community for their support with the application, and the Wellcome Trust for their continued funding. We look forward to supporting your research until 2025 (and beyond).</p>
</div>
<div id="2019-10-06-interpro-v76-loaded" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-now-uses-interpro-version-.">PomBase now uses InterPro Version 76.0</h3>
<p><em>2019-10-06</em></p>
<p>PomBase now uses InterPro Version 76.0, which integrates 277 new methods from the CATH-Gene3D (1), PANTHER (178) and CDD (98) databases. InterPro cites 59846 publications in PubMed. See the InterPro <a href="https://www.ebi.ac.uk/interpro/release_notes/">release notes</a> for further information.</p>
</div>
<div id="2019-08-30-replication-origin-tracks-loaded" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/></p>
<h3 id="replication-origin-data-loaded-into-jbrowse">Replication origin data loaded into JBrowse</h3>
<p><em>2019-08-30</em></p>
<p>We have loaded data from: <a href="https://www.pombase.org/reference/PMID:14566325">Segurado et al. (2003) “A+T-rich islands”</a>, <a href="https://www.pombase.org/reference/PMID:17304213">Hayashi et al. (2007) “Pre-replicative complex localization; early and late firing origins”</a>, and <a href="https://www.pombase.org/reference/PMID:18093330">Mickle et al. (2007) “Replication origins with functional classification”</a>.</p>
<p>To view the tracks, either follow the hyperlinks above to the respective PomBase publication pages, and click on the “view” link after “Datasets from this publication are available in the PomBase JBrowse genome browser”, or go directly to <a href="http://www.pombase.org/jbrowse/">the browser</a> and click on the “select tracks” button to find the tracks manually.</p>
<p>For anyone new to JBrowse we have a <a href="https://www.pombase.org/documentation/JBrowse_quick_start">quick start guide</a>.</p>
</div>
<div id="2019-07-19-join-the-conversations-on-slack" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/slack_icon-32px.png"/></p>
<h3 id="join-conversations-on-slack">Join the conversations on Slack</h3>
<p><em>2019-07-19</em></p>
<p>The vibrant fission yeast community now has a Slack channel. Slack provides a forum for the research community. Follow conversations you care about, message colleagues privately, or in groups, ask questions, post responses. All archived and searchable.</p>
<ul>
<li>See the <a href="/assets/external_media/PombeSlack-flyer.pdf">PombeSlack flyer</a> for more information, including how to join.</li>
</ul>
<!-- included with original news item; commented out after expiry:
 - <a href="https://bit.ly/2XYkcEi">Join PombeSlack</a> -->
</div>
<div id="2019-07-11-Fitness_Landscape_of_the_Fission_Yeast_Genome-data-loaded" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/></p>
<h3 id="fitness-landscape-fission-yeast-genome-dataset-loaded-into-jbrowse">“Fitness Landscape of the Fission Yeast Genome” dataset loaded into JBrowse</h3>
<p><em>2019-07-11</em></p>
<p>We have loaded the <a href="https://www.pombase.org/reference/PMID:31077324">Grech et al. (2019) “Fitness Landscape of the Fission Yeast Genome”</a> dataset into JBrowse. In this study, transposon mutagenesis libraries were created to map transposon insertion sites in the S. pombe genome. From this data, functional elements of the genome were inferred. The tracks from this study can be loaded by a single click from the linked publication page above</p>
<p>Thanks Dan Jeffares for sending us the data.</p>
<p>For anyone new to JBrowse we have a <a href="https://www.pombase.org/documentation/JBrowse_quick_start">quick start guide</a>.</p>
</div>
<div id="2019-05-19-tRNA_metabolism_update" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/rna_metabolism_32px.png"/></p>
<h3 id="trna-metabolism-go-annotation-update">tRNA metabolism GO annotation update</h3>
<p><em>2019-05-19</em></p>
<p>The process of tRNA metabolism, and the associated molecular functions have recently been reviewed.</p>
<ul>
<li><a href="https://www.pombase.org/term/GO:0006399">“tRNA metabolic process” annotation</a></li>
<li><a href="https://www.pombase.org/term_genes/GO:0006399">genes for “tRNA metabolic process”</a></li>
</ul>
<p>Please let us know if the annotation can be further improved.</p>
</div>
<div id="2019-04-18-customisable-fasta" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/pombelist.png"/></p>
<h3 id="customisable-fasta-download">Customisable FASTA download</h3>
<p><em>2019-04-18</em></p>
<p>You can now download nucleotide or peptide sequences for genes in Advanced search results in FASTA format, and customise what is included in the FASTA headers (e.g. gene names, product descriptions, sequence coordinates, or various IDs can be included).</p>
</div>
<div id="2019-04-18-prediction-links" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/tree_for_news.png"/></p>
<h3 id="new-homology-modelling-and-ortholog-links-on-gene-pages">New homology modelling and ortholog links on gene pages</h3>
<p><em>2019-04-18</em></p>
<p>We have added new external links to PomBase gene pages for structure and ortholog predictions:</p>
<ul>
<li><p>Protein-specific links to <a href="https://swissmodel.expasy.org/">SWISS-MODEL</a>, a fully automated protein structure homology-modelling server, accessible via the ExPASy web server, lead to a SWISS-MODEL Repository page for each sequence and present results. If no structure or model is available, you can either trigger adding an entry to the repository with a single click or easily interactively search for templates and build models in your own SWISS-MODEL workspace.</p></li>
<li><p>Ensembl Fungi Compara and Ensembl Pan-taxonomic Compara links lead to orthology predictions from the Ensembl Compara pipeline for fungi and all species, respectively.</p></li>
<li><p><a href="http://www.pantherdb.org/">PANTHER</a> links retrieve gene information, classification, and predicted orthologs.</p></li>
</ul>
</div>
<div id="2019-04-17-pombase-interpro-update" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/interpro_32px.png"/></p>
<h3 id="pombase-interpro-update">PomBase InterPro Update</h3>
<p><em>2019-04-17</em></p>
<p>PomBase now uses InterPro Version 73.0, which integrates 1,531 new methods from the CATH-Gene3D (122), CDD (330), PANTHER (1075), Pfam (2), PROSITE profiles (1) and TIGRFAMs (1) databases, and covers 81.2% of UniProt Knowledgebase release 2019_02.</p>
<p>See <a href="https://www.ebi.ac.uk/about/news/service-news/InterPro-73.0">the news item at InterPro</a> for additional information, including release notes.</p>
</div>
<div id="2019-04-16-pombe-gene-info-extension" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/gix_icon.png"/></p>
<h3 id="s.-pombe-included-gene-info-browser-extension"><em>S. pombe</em> included in Gene Info browser extension</h3>
<p><em>2019-04-16</em></p>
<p><em>S. pombe</em> gene information is now included in the <a href="https://gene-info.org/">Gene Info</a> extension (GIX) for the Chrome and Firefox web browsers. GIX allows you to retrieve information about a gene product directly on any webpage simply by double clicking an official gene name, synonym or supported accession. Searching or double-clicking on text terms retrieves gene function annotation, GO terms, external database links, and interaction data drawn from BioGRID and IntAct. Retrieved gene names are automatically hyperlinked for rapid recursive searches.</p>
<p>GeneInfo is fully open source, available online at <a href="https://github.com/knightjdr/gene-info/">GitHub</a>. Tutorial videos, a step-by-step guide, and download links for Firefox Add-ons and the Chrome web storeare available <a href="https://gene-info.org/">online</a>. GeneInfo was developed by James Knight in the <a href="https://gingraslab.lunenfeld.ca/">Gingras Lab</a> at the <a href="http://www.lunenfeld.ca/">Lunenfeld-Tanenbaum Research Institute</a> in Toronto, Canada.</p>
</div>
<div id="2019-03-19-motif-search-integrated" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/motif_search_cropped.png"/></p>
<h3 id="integrated-motif-search">Integrated motif search</h3>
<p><em>2019-03-19</em></p>
<p>The PomBase <a routerLink="/motif_search">motif search</a> has been fully integrated into the website, and allows users to find protein motifs and send them directly to the PomBase <a routerLink="/query">advanced search</a>.</p>
</div>
<div id="2019-03-13-icygmb2019" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/icygmb_tiny.png"/></p>
<h3 id="icygmb-registration-open">ICYGMB 2019 - registration open</h3>
<p><em>2019-03-13</em></p>
<p>Registration is now open for the <a href="https://yeast2019.org">29th International Conference on Yeast Genetics and Molecular Biology (ICYGMB)</a>, which returns to Gothenburg, Sweden, August 18-22, 2019.</p>
<p>Yeast2019 is the meeting of the international yeast research community where the latest, and even unpublished results are exchanged, and new projects, alliances, and collaborations are founded. Featuring 55 confirmed speakers including keynote lectures by Susan Gasser, Roger Kornberg and Frederick Roth, this conference will contain important news and information for all yeast researchers. A do-not-miss-event.</p>
</div>
<div id="2019-03-05-list-go-slim-tool" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/slim_tiny.png"/></p>
<h3 id="go-slim-any-s.-pombe-gene-list">GO slim for any <em>S. pombe</em> gene list</h3>
<p><em>2019-03-05</em></p>
<p>PomBase’s advanced search now allows you to retrieve <a href="documentation/pombase-go-slim-documentation">GO slim</a> annotations for any set of search results. To find GO slim annotations for your own list of <em>S. pombe</em> genes, use the advanced search “Gene names and IDs” option, and then use the “Slim” button on the search results page.</p>
<p>See the <a href="browse-curation/fission-yeast-bp-go-slim-terms">fission yeast GO slim page</a> and the <a href="documentation/advanced-search">advanced search documentation</a> for more information.</p>
</div>
<div id="2019-03-04-serym2019" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/serym2019tiny.png"/></p>
<h3 id="south-eastern-regional-yeast-meeting-serym-registration-open">South Eastern Regional Yeast Meeting (SERYM) - registration open</h3>
<p><em>2019-03-04</em></p>
<p>Registration is now open for the <a href="https://petitinstitute.gatech.edu/serym-2019">26th annual South Eastern Regional Yeast Meeting (SERYM)</a>, which will be held April 12-14, 2019, in Atlanta, GA, USA.</p>
<p>Fission yeast’s own Susan Forsburg is the keynote speaker. The meeting brings together researchers who use any type of yeast as a model system, covering diverse, interdisciplinary topics from strategies for treatment of fungal disease to modeling human disease in yeast.</p>
<p>Icon: SERYM 2019</p>
</div>
<div id="2019-02-27-trieste-cell-cycle-meeting" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/chantal_roubinet_tiny.png"/></p>
<h3 id="international-cell-cycle-meeting-registration-open">International Cell Cycle meeting - registration open</h3>
<p><em>2019-02-27</em></p>
<p>Registration is now open for the <a href="https://www.ucl.ac.uk/lmcb/cell-cycle-meeting-2019">Inaugural Trieste Cell Cycle Meeting</a>, which will be held June 3-6, 2019, in Trieste, Italy.</p>
<p>This is the first of a planned series of biennial cell cycle meetings that will take place in Europe, and will alternate with the Salk Cell Cycle meetings held on the US west coast.</p>
<p>Organisers Rob de Bruin, Snezhana Oliferenko, Rosella Visintin and Peter Thorpe hope to see you there!</p>
<p>Icon derived from meeting image; credit: Chantal Roubinet, Baum lab</p>
</div>
<div id="2019-02-20-conserved-unknowns-open-biol" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/unknowns.png"/></p>
<h3 id="published-hidden-plain-sight-what-remains-be-discovered-eukaryotic-proteome">Published: Hidden in plain sight: What remains to be discovered in the eukaryotic proteome?</h3>
<p><em>2019-02-20</em></p>
<p>Our analysis of conserved unknown proteins has now been published in <a href="https://royalsocietypublishing.org/doi/10.1098/rsob.180241">Open Biology</a>. In it, PomBase curators consider the challenges and opportunities that conserved, but persistently unstudied, proteins pose for diverse areas of basic and applied biology. We develop metrics to define unknown lists, provide unknown inventories for human and yeast, and classify <em>S. pombe</em> unknowns by numerous orthogonal attributes, all with a view to drawing attention to the unknowns to alleviate their neglect.</p>
</div>
<div id="2019-02-15-pombe-2019-registration-open" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/pombe2019tiny.png"/></p>
<h3 id="pombe-registration-open">Pombe 2019 - registration open</h3>
<p><em>2019-02-15</em></p>
<p>Registration for the <a href="http://www.pombe2019.com">10th International Fission Yeast Meeting</a> is now open!</p>
<p>The conference will take place July 14-19, 2019, in Barcelona, Spain. Early registration closes on April 15th — or when capacity is reached. Please see the <a href="http://www.pombe2019.com">conference website</a> for more information, including registration final deadline and costs (some travel grants are available), abstract submission, programme, accommodation, and logistics.</p>
</div>
<div id="2019-02-12-wood-isb-award" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/isb_logo_32px.png"/></p>
<h3 id="val-wood-wins-biocuration-society-award">Val Wood wins Biocuration society award</h3>
<p><em>2019-02-12</em></p>
<p>Congratulations to PomBase project leader Val Wood, who has received the 2019 Exceptional Contributions to Biocuration Award from the International Society for Biocuration. Read more at the <a href="https://www.biocuration.org/anne-morgat-and-val-wood-recipients-of-the-2019-biocuration-awards/">ISB site</a></p>
</div>
<div id="2019-01-09-improved-disease-association-dataset-released" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/human disease.png"/></p>
<h3 id="improved-disease-association-dataset-released">Improved disease association dataset released</h3>
<p><em>2019-01-09</em></p>
<p>We are pleased to announce the release of our improved human disease mappings dataset. This dataset connects human disease causing genes to their <em>S. pombe</em> orthologs.</p>
<p>Diseases are now mapped to the Disease Ontology (DO) and the dataset has been extended by data from Malacards. All disease associations can be accessed from the top level disease page. A disease slim has been created to facilitate browsing of disease categories. Currently, 907 S. pombe genes are associated with disease (up from 610 in the original dataset). This number is due to increase as mappings are still in progress.</p>
<p>Many thanks to DO and Malacards for help in improving this annotation set. Icon courtesy of <a href="https://pixabay.com/en/users/mcmurryjulie-2375405/">Julie McMurry</a>.</p>
</div>
<div id="2018-12-17-mitochondrial-protein-review" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/mito_news.png"/></p>
<h3 id="mitochondrial-go-annotation-update">Mitochondrial GO annotation update</h3>
<p><em>2018-12-17</em></p>
<p>Responding to increasing interest in mitochondrial biology, especially relating to ageing, neurogenerative diseases, and processes at the ER-mitochondrion interface, we have reviewed <em>S. pombe</em> mitochondrial GO annotations. Although there is still relatively little fission yeast-derived experimental data in this area, we have refined many inferred annotations for mitochondrial complexes and sub-components as well as some for processes.</p>
<p>You can see all 753 <em>S. pombe</em> mitochondrial annotations on the ontology term page for <a href="https://www.pombase.org/term/GO:0005739">mitochondrion (GO:0005739)</a>.</p>
<p>Icon courtesy of Reactome.</p>
</div>
<div id="2018-12-03-new-nucleosome-occupancy-browser-maps-loaded" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/></p>
<h3 id="new-nucleosome-occupancy-maps-loaded">New nucleosome occupancy maps loaded</h3>
<p><em>2018-12-03</em></p>
<p>We have loaded the nucleosome occupancy maps as described in González et al. (2016) <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27662899">PMID: 27662899</a>. This dataset was generated using the paired-end sequencing protocol of Illumina and thus those maps are of higher resolution than those made with single-end (SE) sequencing hosted in the browser since before.</p>
<p>Here is a link that <a href="https://www.pombase.org/jbrowse/?loc=II%3A2751621..2767510&tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2Cnucleosome%20positioning%20during%20vegetative%20growth%3B%20repeat%201%20-%20Gonz%C3%A1lez%20et%20al.%20%282016%29%2Cnucleosome%20positioning%20during%20vegetative%20growth%3B%20repeat%202%20-%20Gonz%C3%A1lez%20et%20al.%20%282016%29%2CDNA%20sequence&highlight=">loads the tracks in PomBase JBrowse</a>. And here is a link to our <a href="https://www.pombase.org/documentation/JBrowse_quick_start">JBrowse quickstart guide</a>.</p>
<p>Many thanks to Paco Antequera for sending us the bigwig files! If you would like us to load any datasets then please get in touch.</p>
</div>
<div id="2018-11-21-quick-little-tool" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/QLT.png"/></p>
<h3 id="see-your-genes-quilt">See your genes in a QuiLT</h3>
<p><em>2018-11-21</em></p>
<p>PomBase now offers a new way to display gene lists graphically based on multiple orthogonal annotation types — the Quick Little Tool (QuiLT) for visualisation.</p>
<p>Inspired by our recent analysis of conserved unstudied proteins (see figures 4 and S1 in the <a href="https://www.biorxiv.org/content/early/2018/11/16/469569">manuscript at bioRxiv</a>), QuiLT allows you to create a similar figure for any gene list you create or import using the advanced search. To use QuiLT, follow the link to your search results, then click the “Visualise” button. QuiLT visualisation is also available from the PomBase pages that list genes annotated to an ontology term, and on the Priority unstudied genes page.</p>
<p>To see the Unknowns dataset in QuiLT, visit the <a routerLink="/results/from/predefined/priority-unstudied-genes:conserved_unknown">unknowns results page</a> and click “Visualise”.</p>
<p>The QuiLT display is interactive, and you can:</p>
<ul>
<li>Highlight subsets of the list, and filter the display</li>
<li>Toggle annotation types on and off</li>
<li>Reorder the list to focus on features of most interest</li>
<li>Download the image</li>
</ul>
<p>See the <a routerLink="/documentation/quick-little-tool">QuiLT documentation</a> for more information, and <a href="mailto:helpdesk@pombase.org">contact the curators</a> if you have comments, questions or suggestions.</p>
<p>Many thanks to our star (and only) programmer, Kim Rutherford, for developing QuiLT.</p>
</div>
<div id="2018-11-20-tm-transport-overhaul" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/antiporter.png"/></p>
<h3 id="fission-yeast-transmembrane-transport-overhaul">Fission yeast transmembrane transport overhaul</h3>
<p><em>2018-11-20</em></p>
<p>The Gene Ontology “<a href="https://www.pombase.org/term_genes/GO:0055085">transmembrane transport</a>” branch has recently been substantially revised. In line with these revisions, PomBase has standardised gene product descriptions for transporters, and overhauled GO annotations to be as complete and comprehensive as possible based on current knowledge.</p>
<p>Icon courtesy of <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29077811">Reactome</a>.</p>
</div>
<div id="2018-11-17-conserved-unknowns-pub" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/unknowns.png"/></p>
<h3 id="hidden-plain-sight-what-remains-be-discovered-eukaryotic-proteome">Hidden in plain sight: What remains to be discovered in the eukaryotic proteome?</h3>
<p><em>2018-11-17</em></p>
<p>In a new publication, PomBase curators consider the challenges and opportunities that conserved, but persistently unstudied, proteins pose for diverse areas of basic and applied biology. To draw attention to these proteins, we develop metrics to define unknown lists, provide unknown inventories for human and yeast, classify <em>S. pombe</em> unknowns by numerous orthogonal attributes, and speculate about reasons for their neglect.</p>
<p>A <a href="https://www.biorxiv.org/content/early/2018/11/16/469569">pre-publication manuscript</a> is available at bioRxiv.</p>
</div>
<div id="2018-11-14-S.-pombe-in-your-pocket" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/mobile_thumb.png"/></p>
<h3 id="pombase-your-pocket">PomBase in your pocket</h3>
<p><em>2018-11-14</em></p>
<p>Our usage statistics informed us that over 20% of devices accessing PomBase are smartphones or tablets. We therefore spent some time optimizing the display for small screens. We hope that you will continue to enjoy PomBase on the go!</p>
</div>
<div id="2018-11-08-20-years-GO" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/go-logo-icon.mini.png"/></p>
<h3 id="celebrating-years-go">Celebrating 20 years of GO</h3>
<p><em>2018-11-08</em></p>
<p>PomBase curators are major contributors to the <a href="http://geneontology.org/">Gene Ontology</a> (GO) project — ontology content, annotations, and QC procedures — and co-authors on the new <a href="https://doi.org/10.1093/nar/gky1055">GO NAR Database Issue paper</a>.</p>
<p>We recommend citing the GO and PomBase NAR papers when you use GO data in your analyses.</p>
</div>
<div id="2018-11-06-rna-central" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/rna_central_icon.png"/></p>
<h3 id="rna-central-and-pombase">RNA central and PomBase</h3>
<p><em>2018-11-06</em></p>
<p>RNAcentral is a comprehensive database of non-coding RNA sequences. PomBase is an RNAcentral Consortium member, and all of the curated non-coding RNAs from PomBase will be available in RNAcentral soon. For more information, see their recent <a href="https://doi.org/10.1093/nar/gky1034">NAR Database Issue paper</a>, as well as current <a href="https://rnacentral.org/search?q=pombe">search results for <em>S. pombe</em> RNAs</a>.</p>
</div>
<div id="2018-10-23-jbrowse-graphics-gene-page" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/></p>
<h3 id="new-pombase-genomic-region-graphics">New PomBase genomic region graphics</h3>
<p><em>2018-10-23</em></p>
<p>PomBase gene pages now use interactive graphics from PomBase JBrowse to depict the genomic region around the gene. Drag to scroll left and right, double-click to zoom in, shift-double-click to zoom out, and click a feature to see details in a popup. The “Full-screen view” link in the corner opens the fully functional JBrowse in a new tab or window. Reloading a gene page restores the display to the default location and zoom level.</p>
</div>
<div id="2018-10-15-nar-database-issue" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/NAR_paper.png"/></p>
<h3 id="pombase-nar-database-issue">PomBase NAR Database Issue</h3>
<p><em>2018-10-15</em></p>
<p>Our NAR database update “PomBase 2018: user-driven reimplementation of the fission yeast database provides rapid and intuitive access to diverse, interconnected information” is <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky961/5128936">now available</a>. We have updated the <a routerLink="/about/citing-pombase">Citing PomBase</a> to recommend citing this new paper. Thank you all for guiding the development of the new, improved PomBase, and for your continued usage, curation contributions, and suggestions!</p>
</div>
<div id="2018-10-10-2019-fungal-pathogen-genomics-course-registration-open" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/WT_course.png"/></p>
<h3 id="fungal-pathogen-genomics-course-registration-open">Fungal Pathogen Genomics Course 2019 - registration open</h3>
<p><em>2018-10-10</em></p>
<p>Registration for the <a href="https://coursesandconferences.wellcomegenomecampus.org/our-events/fungal-pathogen-genomics-2019/">2019 Fungal Pathogen Genomics Course</a> is now open. The course is hosted by Wellcome Genome Advanced Courses and Scientific Conferences, and will take place May 7-12, 2019, at the Wellcome Genome Campus, Hinxton, UK. Course content provides hands-on training on how to: - Take advantage of unique tools offered by FungiDB, EnsemblFungi, PomBase, SGD/CGD, and MycoCosm/JGI; - Develop testable hypotheses; - Investigate transcriptomics, proteomics and genomics datasets across multiple databases and different user interfaces. Please see the <a href="https://coursesandconferences.wellcomegenomecampus.org/our-events/fungal-pathogen-genomics-2019/">course website</a> for more information, including how to apply, costs (limited bursaries are available), programme, and logistics.</p>
</div>
<div id="2018-10-06-transcript-tracks-from-Atkinson-et-al-(2018)-loaded" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/Atkinson.png"/></p>
<h3 id="transcript-tracks-atkinson-et-al.-loaded">Transcript tracks from Atkinson et al. (2018) loaded</h3>
<p><em>2018-10-06</em></p>
<p>We are very pleased to announce that we have loaded the transcript tracks from <a href="https://www.ncbi.nlm.nih.gov/pubmed/29914874">Atkinson et al. (2018)</a> into the PomBase JBrowse genome browser. For a brief introduction to getting started with PomBase JBrowse, please see our <a href="https://www.pombase.org/documentation/JBrowse_quick_start">documentation page</a>. If you have published data that you would like to see hosted, please get in touch.</p>
</div>
<div id="2018-08-31-mailing-list-move-to-cam" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/pombelist.png"/></p>
<h3 id="pombelist-changes">pombelist changes</h3>
<p><em>2018-08-31</em></p>
<p>The pombe community mailing list, “pombelist”, is now hosted by the University of Cambridge. The new address for posting messages is <a href="mailto:pombelist@pombase.org" class="email">pombelist@pombase.org</a>. The link to <a href="https://lists.cam.ac.uk/mailman/listinfo/ucam-pombelist">subscribe</a> has also changed.</p>
</div>
<div id="2018-05-28-new-browse-tracks" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/new_tracks.png"/></p>
<h3 id="new-genome-browser-tracks">New genome browser tracks</h3>
<p><em>2018-05-28</em></p>
<p>We are very pleased to announce that we have loaded a number of new datasets into the PomBase [JBrowse genome browser (https://www.pombase.org/jbrowse/). These include:</p>
<ul>
<li><a href="https://www.biorxiv.org/content/early/2018/03/13/281642">Thodberg <em>et al.</em> (2018)</a> - CAGE-defined <a href="https://www.pombase.org/jbrowse/?loc=II%3A27586..65254&amp;tracks=DNA%20sequence%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CConsensus%20transcription%20start%20sites%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20heat%20shock%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20heat%20shock%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20nitrogen%20starvation%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20nitrogen%20starvation%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20oxidative%20stress%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20during%20oxidative%20stress%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20YES%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20YES%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20glucose%20minimal%20medium%20(forward%20strand)%20-%20Thodberg%20et%20al.%20(2018)%2CTranscription%20start%20sites%20in%20glucose%20minimal%20medium%20(reverse%20strand)%20-%20Thodberg%20et%20al.%20(2018)&amp;highlight=">transcription start sites</a> across 5 different conditions</li>
<li><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23163955">Yadav <em>et al.</em> (2012)</a> - <a href="https://www.pombase.org/jbrowse/?loc=I%3A2364901..2436900&amp;tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CPosition%20specific%20G(x)%20scores%20(free%20energy%20needed%20to%20separate%20a%20base%20pair)%20-%20Yadav%20et%20al.%20(2012)&amp;highlight=">G(x) scores</a> specifying the amount of free energy needed to melt base pairs in the DNA duplex at different genomic locations</li>
<li>A PomBase-generated <a href="https://www.pombase.org/jbrowse/?loc=III%3A725762..736349&amp;tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CConsensus%20transcription%20factor%20DNA%20binding%20motifs&amp;highlight=">dataset of promoter elements</a> across the genome (computational matching of the <a routerLink="/browse-curation/dna-binding-sites">consensus promoter sequences</a> to the reference DNA sequence)</li>
<li><a href="https://www.pombase.org/jbrowse/?loc=I%3A2178501..2207300&amp;tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features%2CIntron%20Branch%20Points%20-%20Bitton%20et%20al.%20(2014)&amp;highlight=">Intron branch points</a> from <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24709818">Bitton <em>et al.</em> (2014)</a> which was also available in the Ensembl browser.</li>
</ul>
<p>For anyone wanting a quick introduction to our genome browser, Antonia Lock has written <a routerLink="/documentation/JBrowse_quick_start">“Getting started with PomBase JBrowse”</a>, a basic guide that covers loading tracks, navigating the browser, what metadata we provide, and more.</p>
</div>
<div id="2018-05-22-book-chapter" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/book_chapter.png"/></p>
<h3 id="new-book-chapter-about-pombase">New book chapter about PomBase</h3>
<p><em>2018-05-22</em></p>
<p>PomBase has a <a href="https://link.springer.com/protocol/10.1007/978-1-4939-7737-6_4">new book chapter</a> in <a href="https://link.springer.com/book/10.1007/978-1-4939-7737-6"><em>Eukaryotic Genomic Databases</em> (Methods and Protocols)</a>. This chapter provides insight into the curation philosophy and data organization at PomBase, and provides a guide to using PomBase tailored for infrequent visitors and anyone considering extending their research to include <em>S. pombe</em>. The chapter is <a href="https://link.springer.com/protocol/10.1007/978-1-4939-7737-6_4">free to download</a> courtesy of the <a href="https://wellcome.ac.uk/">Wellcome Trust</a>.</p>
</div>
<div id="2018-04-16-pombase-deploys-jbrowse" class="news-item">
<p><img class="newsfeed-thumbnail" src="/assets/newsfeed/browser.png"/></p>
<h3 id="pombase-releases-jbrowse">PomBase releases JBrowse</h3>
<p><em>2018-04-16</em></p>
<p>PomBase has now implemented <a href="https://www.pombase.org/jbrowse/?loc=II%3A564641..602220&amp;tracks=DNA%2CPomBase%20forward%20strand%20features%2CPomBase%20%20reverse%20strand%20features&amp;highlight=">JBrowse</a>, from the <a href="http://gmod.org/wiki/Main_Page">GMOD</a> project, as its genome browser. The new browser offers a number of improvements over the old:</p>
<ul>
<li>Quick, responsive scrolling and zooming</li>
<li>Simple track selection interface</li>
<li>Intuitive controls</li>
<li>Simplified data submission pipeline behind the scenes</li>
<li>More informative track metadata</li>
</ul>
</div>
<section id="2018-04-05-andre-goffeau" class="news-item">
<h3 id="in-memory-andr-eacute-goffeau">In memory of André Goffeau</h3>
<p><em>2018-04-05</em></p>
<p>Sadly, PomBase staff and the fission yeast community note the death of André Goffeau on April 2, 2018. In addition to initiating and coordinating the sequencing of the budding yeast genome, Prof. Goffeau will be remembered for his contributions to the fission yeast genome project and for his knowledge, leadership, and friendship.</p>
</section>
<section id="2017-11-24-congratulations-to-gsa-award-winners" class="news-item">
<h3 id="congratulations-gsa-award-winners">Congratulations to GSA award winners</h3>
<p><em>2017-11-24</em></p>
<p>The <a href="http://www.genetics-gsa.org">Genetics Society of America</a> (GSA) has announced two award winners familiar to the model organism database world:</p>
<ul>
<li>Ira Herskowitz Award: Mike Cherry, Stanford University</li>
<li>Lifetime Achievement Award: Steve Oliver, University of Cambridge</li>
</ul>
<p>The awards will be presented at the next <a href="http://conferences.genetics-gsa.org/yeast/2018/index">Yeast Genetics Meeting</a>, at Stanford University in August 2018. Congratulations and thanks to Mike and Steve!</p>
</section>
<section id="2017-10-24-new-improved-pombase-goes-live" class="news-item">
<h3 id="new-improved-pombase-goes-live">New, improved PomBase goes live</h3>
<p><em>2017-10-24</em></p>
<p>The new PomBase web site, which has been under development during 2017, has been released. The new site features:</p>
<ul>
<li>Nightly data updates</li>
<li>New publication pages</li>
<li>New genotype pages</li>
<li>Improved ontology term pages</li>
<li>Improved summary views for annotation displays</li>
<li>Phenotype annotation display filtering</li>
<li>Faster querying in the advanced search</li>
<li>Front page research and community curation highlights</li>
<li>Streamlined back-end data storage and retrieval</li>
</ul>
<p>We thank the members of the fission yeast research community who have followed its progress via the preview site, and welcome feedback from all users.</p>
</section>
<section id="2016-12-11-9th-international-fission-yeast-meeting-early-registration-closes-soon" class="news-item">
<h3 id="th-international-fission-yeast-meeting-early-registration-closes-soon">9th International Fission Yeast Meeting - early registration closes soon</h3>
<p><em>2016-12-11</em></p>
<p>Reminder: early registration for the 9th International Fission Yeast Meeting in Banff closes <strong>Dec. 31, 2016</strong>. Please see the conference website at <a href="http://www.pombe2017.com">www.pombe2017.com</a> for details.</p>
</section>
<section id="2016-10-31-9th-international-fission-yeast-meeting-registration-open" class="news-item">
<h3 id="th-international-fission-yeast-meeting-registration-open">9th International Fission Yeast Meeting registration open</h3>
<p><em>2016-10-31</em></p>
<p>Registration for the 9th International Fission Yeast Meeting is now open. The meeting will be held in Banff, Canada from May 14-19, 2017. Early registration closes Dec 1, 2016! Please see our website at <a href="http://www.pombe2017.com">www.pombe2017.com</a> for details. We look forward to seeing you in Banff!<br />
<br />
- Conference Organizers: Dallan Young, Gordon Chua, Paul Young</p>
</section>
<section id="2016-10-19-pombase-data-update-23" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2016-10-19</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through September 11, 2016.</p>
</section>
<section id="2016-06-27-show-your-support-database-funding" class="news-item">
<h3 id="show-your-support-database-funding">Show your support for database funding</h3>
<p><em>2016-06-27</em></p>
<p>In response to planned cuts to database funding, leading model organism researchers have prepared an open letter to NIH Director Dr. Francis Collins to demonstrate support for the independent community-focused databases that are essential to their work. Although PomBase is not directly funded by NIH, we collaborate extensively with those that are, including the GO Consortium and several model organism databases.<br />
<br />
The Genetics Society of America website where the letter can be viewed and signed is at <a href="http://www.genetics-gsa.org/MODsupport" class="uri">http://www.genetics-gsa.org/MODsupport</a><br />
<br />
Please sign the letter to add your voice in support of the databases that help make your research possible. For more information, we recommend an email that Mike Cherry sent to the GO-Friends mailing list, archived at <a href="https://mailman.stanford.edu/pipermail/go-friends/2016-June/002355.html" class="uri">https://mailman.stanford.edu/pipermail/go-friends/2016-June/002355.html</a></p>
</section>
<section id="2016-06-15-our-model-organism-database-commentary" class="news-item">
<h3 id="our-model-organism-database-commentary">Our model organism database commentary</h3>
<p><em>2016-06-15</em></p>
<p>Several of the PomBase staff, joined by our advisor Sir Paul Nurse, have published a Comment in <em>BMC Biology</em> briefly describing the importance of model organism databases to the success of modern biomedical research:<br />
<br />
Oliver SG, Lock A, Harris MA, Nurse P, Wood V. 2016. Model organism databases: essential resources that need the support of both funders and users.<br />
<em>BMC Biol.</em> 2016 <strong>14(1)</strong>: 49. doi: <a href="http://dx.doi.org/10.1186/s12915-016-0276-z">10.1186/s12915-016-0276-z</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/27334346">PMID:27334346</a></p>
</section>
<section id="2016-05-31-pombase-data-update-22" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2016-05-31</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through May 12, 2016.</p>
</section>
<section id="2016-05-09-pombase-data-update-21" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2016-05-09</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through April 8, 2016.</p>
</section>
<section id="2016-04-11-pombase-data-update-20" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2016-04-11</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through March 9, 2016.</p>
<p><strong>Important:</strong> We have corrected a problem that made erroneous interaction data and literature appear on some gene pages.</p>
<p>The gene pages now include interaction data from the Vo et al. proteome-wide study (curated by BioGRID and imported into PomBase):<br />
Vo TV <em>et al.</em> 2016. A Proteome-wide Fission Yeast Interactome Reveals Network Evolution Principles from Yeasts to Human. <em>Cell</em> <strong>164(1-2)</strong>: 310-23. doi: <a href="http://dx.doi.org/10.1016/j.cell.2015.11.037">10.1016/j.cell.2015.11.037</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=26771498">PMID:26771498</a>.</p>
<p>The genome browser now includes transcriptome data published in:<br />
<br />
Eser P, Wachutka L, Maier KC, Demel C, Boroni M, Iyer S, Cramer P, Gagneur J. 2016. Determinants of RNA metabolism in the Schizosaccharomyces pombe genome. <em>Mol Syst Biol</em>. <strong>12(2):</strong> 857. doi: <a href="http://dx.doi.org/10.15252/msb.20156526">10.15252/msb.20156526</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=26883383">PMID:26883383</a>.</p>
</section>
<section id="2016-02-11-pombase-data-update-19" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2016-02-11</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through January 25, 2016.<br />
The genome browser includes variation data, in tracks under “Variation”, from natural <em>S. pombe</em> isolates, published in:<br />
<br />
Jeffares DC <em>et al.</em> 2015. The genomic and phenotypic diversity of Schizosaccharomyces pombe. <em>Nat Genet.</em> <strong>47(3):</strong> 235-241. doi:<a href="http://dx.doi.org/10.1038/ng.3215">10.1038/ng.3215</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25665008">PMID:25665008</a></p>
<p>New files are now available from the PomBase FTP site, and are linked from pages in the Download Datasets area:</p>
<ul>
<li>Non-coding RNA sequence feature coordinates (available via the <a href="/downloads/data-mappings">Data Mappings</a> page);</li>
<li>Protein features, such as domains and family assignments (available via the <a routerLink="/downloads/protein-datasets">Protein Datasets</a> page);</li>
<li>Protein modification annotations (also in Protein Datasets).</li>
</ul>
<p>The <a routerLink="/status/new-and-removed-genes">New and Removed Genes page</a> has been updated to reflect recent deletions and merges.<br />
<br />
Note: Ontology graph views are no longer available in the genome browser, so links have been removed from the GO, FYPO, and modification tables on the gene pages. For GO and FYPO, links to external ontology browsers that offer graphical views are available on the <a href="/documentation/ontology-term-page">Ontology Term pages</a>.<br />
<br />
</p>
</section>
<section id="2015-12-02-pombase-data-update-18" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2015-12-02</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through November 9, 2015, including 340 community-curated publications.</p>
</section>
<section id="2015-12-02-new-advanced-search-features" class="news-item">
<h3 id="new-advanced-search-features">New Advanced Search features</h3>
<p><em>2015-12-02</em></p>
<p>We have introduced new features to the <a href="/spombe/query/builder">Advanced Search</a>:</p>
<ul>
<li>There are now two query reuse options: store a query in your web browser cache, or download a JSON file that can be uploaded later to re-run.</li>
<li>You can now query for genes that interact genetically or physically with a specified gene.</li>
<li>The FYPO query now offers options to choose nulls (deletions or disruptions), wild-type overexpression, or all alleles. The search results will include any genes that have an allele that matches the allele criteria and the chosen phenotype.</li>
</ul>
</section>
<section id="2015-10-19-new-genetics-primer-fission-yeast" class="news-item">
<h3 id="new-genetics-primer-fission-yeast">New genetics primer for fission yeast</h3>
<p><em>2015-10-19</em></p>
<p>A new genetics primer, aimed at researchers interested in using fission yeast as a model system, has recently been published. The primer includes a brief history of fission yeast research, an introduction to available genetic tools, and the use of PomBase for data analysis</p>
<p>Hoffman CS, Wood V, Fantes PA. (2015) An Ancient Yeast for Young Geneticists: A Primer on the <em>Schizosaccharomyces pombe</em> Model System. <em>Genetics</em> <strong>201</strong>:403-423. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26447128">PMID:26447128</a> <a href="http://dx.doi.org/10.1534/genetics.115.181503">DOI:10.1534/genetics.115.181503</a></p>
</section>
<section id="2015-09-28-pombase-data-update-viability-summary-correction-alert" class="news-item">
<h3 id="pombase-data-update-viability-summary-correction-alert">PomBase data update; viability summary correction alert</h3>
<p><em>2015-09-28</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through September 6, 2015.</p>
<p>Errors in the previous FYPOviability.tsv file have been corrected, and we recommend that all users update this file, especially those who downloaded it earlier in September 2015.</p>
</section>
<section id="2015-09-03-pombase-data-update-multi-allele-phenotypes" class="news-item">
<h3 id="pombase-data-update-multi-allele-phenotypes">PomBase data update with multi-allele phenotypes</h3>
<p><em>2015-09-03</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through August 13, 2015, including 300 community-curated publications.</p>
<p>PomBase gene pages now include multi-allele phenotype annotations (i.e. phenotypes of double mutants, triple mutants, etc.). New sub-sections of the gene pages display multi-allele phenotypes at the population and individual cell level, paralleling the organisation of the single allele phenotype display. Compact and full views are available; both show phenotypes with the relevant genotypes and the alleles that make them up, and the full view adds details for evidence, expression, conditions, and references.</p>
<p>The genome browser now includes data tracks for two more publications:<br />
<br />
DNA polymerase usage from:<br />
Daigaku Y, Keszthelyi A, Müller CA, Miyabe I, Brooks T, Retkute R, Hubank M, Nieduszynski CA, Carr AM. 2015. A global profile of replicative polymerase usage. <em>Nat Struct Mol Biol.</em> 2015 Mar;<strong>22(3):</strong>192-8. doi: <a href="http://dx.doi.org/10.1038/nsmb.2962">10.1038/nsmb.2962</a> PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25664722">25664722</a><br />
<br />
Promoters and transcription start sites from:<br />
Li H, Hou J, Bai L, Hu C, Tong P, Kang Y, Zhao X, Shao Z. 2015. Genome-wide analysis of core promoter structures in Schizosaccharomyces pombe with DeepCAGE. <em>RNA Biol.</em> 2015;<strong>12(5):</strong>525-37. doi: <a href="http://dx.doi.org/10.1080/15476286.2015.1022704">10.1080/15476286.2015</a>.1022704 PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25747261">25747261</a></p>
<p>Codon adaptation index (CAI) values are now included in the Protein Properties section of the gene pages and in the downloadable PeptideStats.tsv file. A file of amino acid composition data is also available from the FTP site and the <a href="/downloads/protein-datasets">Protein Datasets</a> page.</p>
<p>The gene page section that was formerly misnamed “species distribution” is now called “taxonomic conservation”.</p>
</section>
<section id="2015-06-16-pombase-data-update-17" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2015-06-16</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through May 26, 2015, including 270 community-curated publications. See you at Pombe 2015 in Kobe!</p>
</section>
<section id="2015-05-26-canto-downtime-new-version" class="news-item">
<h3 id="canto-downtime-new-version">Canto downtime &amp; new version</h3>
<p><em>2015-05-26</em></p>
<p>Canto, PomBase’s literature curation tool, will be unavailable for approximately 3 weeks starting at 12:00 midnight UK time (BST) tonight, 27 May 2015, while we deploy an upgraded version.<br />
<br />
The upgraded Canto will feature an entirely new interface for annotating multi-allele phenotypes and the corresponding genotypes, as well as improved workflows for single-allele phenotypes, GO, etc. All existing annotations will be retained, and users can resume curation using the new and improved features in any unfinished sessions when Canto is back online.<br />
<br />
We will announce when the new version of Canto is released to the public.</p>
</section>
<section id="2015-05-26-pombase-data-update-16" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2015-05-26</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through May 8, 2015, including 265 community-curated publications.</p>
</section>
<section id="2015-04-23-pombe-2015-travel-fellowships" class="news-item">
<h3 id="pombe-travel-fellowships">Pombe 2015 travel fellowships</h3>
<p><em>2015-04-23</em></p>
<p>Applications are now being accepted for fellowships to provide financial support for students and postdocs attending the 8th International Fission Yeast Meeting in Kobe, Japan. To apply, follow the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2015/004246.html">instructions sent to the pombase mailing list</a>. The deadline is may 17, 2015 (same as the registration deadline).</p>
</section>
<section id="2015-04-19-pombase-data-update-15" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2015-04-19</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through April 7, 2015, including 260 community-curated publications.The Advanced Search now supports queries for proteins with a specified number of transmembrane domains.</p>
</section>
<section id="2015-04-19-pombe-2015-poster-abstract-deadline-extended" class="news-item">
<h3 id="pombe-poster-abstract-deadline-extended">Pombe 2015 poster abstract deadline extended</h3>
<p><em>2015-04-19</em></p>
<p>The abstract submission deadline for the <a href="https://amarys-jtb.jp/web/Pombe2015/index.html">8th International Fission Yeast Meeting</a> in Kobe, Japan has been extended until midnight Friday, April 24 for <strong>posters only</strong>. Registration is open until May 17.</p>
</section>
<section id="2015-04-09-pombe-2015-abstract-deadline-approaching" class="news-item">
<h3 id="pombe-abstract-deadline-approaching">Pombe 2015 abstract deadline approaching</h3>
<p><em>2015-04-09</em></p>
<p>Abstracts are due on Sunday, April 19, 2015 for the <a href="https://amarys-jtb.jp/web/Pombe2015/index.html">8th International Fission Yeast Meeting</a> in Kobe, Japan. Registration will remain open until May 17, but the abstract submission deadline cannot be extended.</p>
</section>
<section id="2015-03-23-pombase-data-update-14" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2015-03-23</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through March7, 2015, including 250 community-curated publications.The autocomplete feature of the Advanced Search ontology term filter has been improved with respect to response time and relevance of suggested terms.</p>
</section>
<section id="2015-02-26-pombe-2015-registration-now-open" class="news-item">
<h3 id="pombe-registration-now-open">Pombe 2015 registration now open</h3>
<p><em>2015-02-26</em></p>
<p>Registration for Pombe 2015: 8th International Fission Yeast Meeting is now open at the conference web site, <a href="https://amarys-jtb.jp/web/Pombe2015/index.html" class="uri">https://amarys-jtb.jp/web/Pombe2015/index.html</a></p>
<p>The registration deadline is 17 May 2015.</p>
<p>Thanks to Yasushi Hiraoka for this item.</p>
</section>
<section id="2015-02-16-pombase-data-update-13" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2015-02-16</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through February 2, 2015, including 245 community-curated publications. On the gene pages, the interaction tables now provides a bit of descriptive text for each annotation, indicating the nature and direction of the interaction.</p>
</section>
<section id="2015-01-26-pombase-data-update-12" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2015-01-26</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through January 12, 2015, including 240 community-curated publications. The gene page Phenotype section now features a compact default display. A downloadable “viability summary” data file is now available. The PomBase BLAST server has incorporated interface changes made Ensembl-wide.</p>
</section>
<section id="2014-12-10-new-compact-go-annotation-display" class="news-item">
<h3 id="new-compact-go-annotation-display">New compact GO annotation display</h3>
<p><em>2014-12-10</em></p>
<p>To make the Gene Ontology (GO) annotations easier to read on PomBase gene pages, we have introduced a new, streamlined display that presents just the essentials. The summary shows the term name (hyperlinked to the ontology term page), the count of genes annotated to the term, and any annotation extensions. All of the previously visible annotation details are still available – simply click the “Summary” button to switch to the “Full” view. Or click the “+” and “-” icons to expand or collapse the annotation to a single term.<br />
 <br />
In addition, the top of the Biological Process table now lists any GO slim terms applicable to the gene.</p>
</section>
<section id="2014-12-10-esyn-network-visualizations-pombase" class="news-item">
<h3 id="esyn-network-visualizations-pombase">esyN network visualizations in PomBase</h3>
<p><em>2014-12-10</em></p>
<p>PomBase has implemented network visualisations for fission yeast in <a href="http://www.esyn.org/">esyN</a>, using data curated by BioGRID and PomBase. esyN is a web-based tool for building, sharing, and viewing network data developed by Dan Bean and Giorgio Favrin in the <a href="http://www.sysbiol.cam.ac.uk">Cambridge Systems Biology Centre</a>, University of Cambridge, UK.<br />
<br />
On gene pages, we have links to gene-specific interaction networks in esyN in the table headers of the Interactions sections:</p>
<ul>
<li>The Genetic Interactions section links to all interactions centred on the gene and curated in BioGRID</li>
<li>The Physical interactions section has links to two datasets:
<ul>
<li>All physical interactions curated in BioGRID for the gene</li>
<li>Interactions for the gene in the PomBase <a href="/documentation/high-confidence-physical-interaction-network">High Confidence Physical Interaction Network (HCPIN)</a></li>
</ul></li>
</ul>
<p>We also have esyN links on the <a href="/browse-curation/fission-yeast-bp-go-slim-terms">GO Slim page</a> and on ontology term pages for GO Slim biological process terms. Each GO Slim term links to the HCPIN physical interaction network in esyN (for example, see the “regulation of mitotic cell cycle” <a href="http://www.esyn.org//builder.php?type=Graph&term=GO:0007346&interactionType=physical&source=pombase&includeInteractors=false">network</a>).</p>
</section>
<section id="2014-11-12-pombase-data-update-11" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2014-11-12</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through October 27, 2014, including 225 community-curated publications. The gene page Phenotype section now includes data from the high-throughput microscopy analysis of viable deletion mutants reported in:<br />
<br />
Graml V, Studera X, Lawson JL, Chessel A, Geymonat M, Bortfeld-Miller M, Walter T, Wagstaff L, Piddini E, Carazo-Salas RE. A Genomic Multiprocess Survey of Machineries that Control and Link Cell Shape, Microtubule Organization, and Cell-Cycle Progression. Dev Cell. 2014 Oct 27;31(2):227-39. doi: <a href="http://dx.doi.org/10.1016/j.devcel.2014.09.005">10.1016/j.devcel.2014.09.005</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25373780">PMID:25373780</a>. Links to the accompanying SYSGRO resource have been added to the External References section of the gene pages.</p>
<p>The genome browser now includes tracks for intron branch point data from:</p>
<p>Bitton DA, Rallis C, Jeffares DC, Smith GC, Chen YY, Codlin S, Marguerat  S, Bähler J. LaSSO, a strategy for genome-wide mapping of intronic  lariats and branch points using RNA-seq. Genome Res. 2014 Jul;24(7):1169-79. doi: <a href="http://dx.doi.org/10.1101/gr.166819.113">10.1101/gr.166819.113</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24709818">PMID:24709818</a>.</p>
<p>We have greatly improved search results for GO and FYPO annotations: both now follow more relationship types within the ontology to retrieve genes annotated to a term. The PomBase GO search now includes the <em>regulates</em> relationships, so its search results are consistent with those in the GO Consortium’s AmiGO browser. The FYPO search now uses <em>has_part,</em> <em>has_output</em>, and <em>output_of</em> as well as <em>is_a</em> and <em>part_of</em>. The Phenotype section now includes a highlighted sub-header that indicates whether a deletion mutant is viable or inviable. A file of protein complex subunits is available for download, and numerous smaller improvements have been made in the gene pages and static pages.</p>
</section>
<section id="2014-09-16-pombase-data-update-10" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2014-09-16</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through August 30, 2014. Community curation now covers over 200 papers.</p>
</section>
<section id="2014-08-18-pombase-data-update-9" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2014-08-18</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through August 8, 2014. Community curation now covers over 190 papers. Gene pages now include links to the <a href="https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/buildDetails?atlas_build_id=355"><em>S. pombe</em> PeptideAtlas</a>, a database of peptides identified in tandem mass spectrometry proteomics experiments.</p>
</section>
<section id="2014-07-17-pombase-data-update-8" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2014-07-17</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through July 8, 2014. The gene pages also now display protein modification data from an additional large-scale dataset:</p>
<p>Koch A, Krug K, Pengelley S, Macek B, Hauf S. 2011. Mitotic substrates of the kinase aurora with roles in chromatin regulation identified through quantitative phosphoproteomics of fission yeast. <em>Sci Signal</em>. <strong>4</strong>(179): rs6 doi: 10.1126/scisignal.2001588 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21712547">PMID:21712547</a><br />
<br />
We have also made corrections to some residue positions affected by sequence updates in one of the modification datasets we added last month:</p>
<p>Carpy A, Krug K, Graf S, Koch A, Popic S, Hauf S, Macek B. 2014. Absolute proteome and phosphoproteome dynamics during the cell cycle of fission yeast. <em>Mol Cell Proteomics</em>. 2014 Apr 23. [Epub ahead of print] <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24763107">PMID:24763107</a></p>
</section>
<section id="2014-07-08-pombase-data-update-7" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2014-07-08</em></p>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through June 6, 2014. In other improvements, a downloadable file of <a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/FASTA/cdna_introns_utrs.fa.gz">intron sequence data</a> (FASTA format) is now available, and phenotypes are now included in the Target Of section on gene pages.</p>
<p>The gene pages also now display protein modification data from two large-scale datasets:</p>
<ul>
<li>Wilson-Grady JT, Villén J, Gygi SP. 2008 .Phosphoproteome analysis of fission yeast. <em>J Proteome Res</em>. 2008 Mar;<strong>7</strong>(3):1088-97. doi:10.1021/pr7006335. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18257517">PMID:18257517</a></li>
<li>Carpy A, Krug K, Graf S, Koch A, Popic S, Hauf S, Macek B. 2014. Absolute proteome and phosphoproteome dynamics during the cell cycle of fission yeast. <em>Mol Cell Proteomics</em>. 2014 Apr 23. [Epub ahead of print] <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24763107">PMID:24763107</a></li>
</ul>
</section>
<section id="2014-06-29-gene-ontology-publication-annotation-extensions" class="news-item">
<h3 id="gene-ontology-publication-on-annotation-extensions">Gene Ontology publication on annotation extensions</h3>
<p><em>2014-06-29</em></p>
<p>PomBase was an early adopter of <a routerLink="/documentation/gene-page-gene-ontology#ann_ext">annotation extensions</a>, which add spatial, temporal, or substrate/target details to GO annotations. The GO Consortium has now published a paper describing its implementation of annotation extensions, in which PomBase examples and its gene page display figure prominently:</p>
<p>Huntley, R.P. <em>et al.</em> (2014) A method for increasing expressivity of Gene Ontology annotations using a compositional approach. <em>BMC Bioinformatics</em> 2014, <strong>15</strong>:155. <a href="http://www.biomedcentral.com/1471-2105/15/155/abstract">doi:10.1186/1471-2105-15-155</a> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24885854">PMID:24885854</a></p>
</section>
<section id="2014-05-15-pombase-data-update-6" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2014-05-15</em></p>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through April 28, 2014. Transcriptome data from <a href="http://europepmc.org/abstract/MED/23101633">Margeurat <em>et al</em> (2012)</a> is now available as Ensembl Browser tracks.</p>
<p>Thank you to all who have done, or are doing, paper curation in Canto. Over 159 community-curated papers are now included in PomBase.</p>
<p>There are a number of routes to accelerate your data into PomBase, (either through community curation, or by supplying HTP sequence, modification or phenotype data in one of our specified formats), see <a href="http://www.pombase.org/submit-data" class="uri">http://www.pombase.org/submit-data</a> for more details.</p>
<p>As usual, please don’t hesitate to alert us of any other problems with data or site performance, or if you have any questions.</p>
<p>Sincerely yours,<br />
The PomBase Staff</p>
</section>
<section id="2014-03-20-pombase-data-update-5" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2014-03-20</em></p>
<p>Data on the PomBase web site now includes manual curation through February 24, 2014. Human orthologs that went missing from gene pages have been restored, and other small improvements have been made to gene pages. Community curation now covers over 130 publications.</p>
</section>
<section id="2014-02-20-pombase-data-update-4" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2014-02-20</em></p>
<p>We have once again updated the data available on the PomBase web site. The data now includes manual curation through January 10, 2014, and covers over 100 papers that have been curated in Canto by community members. We again thank all who have contributed curation via Canto.</p>
<p>We have made some improvements to the gene pages. Highlights:</p>
<ul>
<li>The Sequence section now has links to NCBI BLAST as well as Ensembl BLAST.</li>
<li>The External References section now links to the Pomb(A) polyadenylation viewer.</li>
</ul>
<p>In the genome browser, new data tracks are now available for data from these publications:</p>
<ul>
<li>Rhind N, [and many more]. 2011. Comparative functional genomics of the fission yeasts. Science 332(6032):930-6. doi: 10.1126/science.1203357. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:21511999</a></li>
<li>Schlackow M, Marguerat S, Proudfoot NJ, Bähler J, Erban R, Gullerova M. 2013. Genome-wide analysis of poly(A) site selection in <em>Schizosaccharomyces pombe</em>. RNA. 19(12):1617-31. doi:10.1261/rna.040675.113. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24152550">PMID:24152550</a></li>
<li>Soriano I, Quintales L, Antequera F. 2013. Clustered regulatory elements at nucleosome-depleted regions punctuate a constant nucleosomal landscape in Schizosaccharomyces pombe. BMC Genomics. 14:813. doi:10.1186/1471-2164-14-813. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24256300">PMID:24256300</a> (partial data;  remainder coming in the next update)</li>
<li>Xu J, Yanagisawa Y, Tsankov AM, Hart C, Aoki K, Kommajosyula N, Steinmann KE, Bochicchio J, Russ C, Regev A, Rando OJ, Nusbaum C, Niki H, Milos P, Weng Z, Rhind N. 2012. Genome-wide identification and characterization of replication origins by deep sequencing. Genome Biol. 13(4):R27. doi:10.1186/gb-2012-13-4-r27. <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22531001">PMID:22531001</a></li>
</ul>
<p>Now that more data tracks are available, we have added some categories to the track configuration section to improve organization. Additional documentation is in preparation, and will be announced here when available.<br />
<br />
Genome sequences for additional <em>Schizosaccharomyces</em> species (<em>S. japonicus</em>, <em>S. octosporus</em>, and <em>S. cryophilus</em>) have recently become available in Ensembl Fungi, and the PomBase genome browser now includes comparative genomics data, with a view of region comparisons between each new genome and <em>S. pombe</em>.</p>
</section>
<section id="2014-02-19-human-ortholog-data-correction-coming-next-month" class="news-item">
<h3 id="human-ortholog-data-correction-coming-next-month">Human ortholog data correction coming next month</h3>
<p><em>2014-02-19</em></p>
<p>We are about to release a data update for PomBase. Please note that there is still a problem with the human orthologs, as originally described on this list in mid-December (see archived message at <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2013/003926.html" class="uri">http://listserver.ebi.ac.uk/pipermail/pombelist/2013/003926.html</a>). We will correct this problem in the next PomBase release, and apologise for any inconvenience in the meantime.</p>
</section>
<section id="2013-12-08-pombase-data-update-3" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2013-12-08</em></p>
<p>We have updated the data available on the PomBase web site to include manual curation through November 11, 2013. We now have future meetings available as a <a routerLink="/calendar">calendar</a> or a <a routerLink="/events">list</a>. The <a routerLink="/faq">FAQ</a> and some documentation pages have also been updated.<br />
<br />
</p>
</section>
<section id="2013-11-24-2013-meeting-mini-reviews-published" class="news-item">
<h3 id="meeting-mini-reviews-published">2013 meeting mini-reviews published</h3>
<p><em>2013-11-24</em></p>
<p>A series of mini-reviews, which were invited in association with the International Fission Yeast Meeting in London, have now been published in Biochemical Society Transactions: <a href="http://www.biochemsoctrans.org/bst/041/6/default.htm#c" class="uri">http://www.biochemsoctrans.org/bst/041/6/default.htm#c</a></p>
<p>(Thanks to Jürg Bahler for this item)</p>
</section>
<section id="2013-11-20-pombase-survey-results-available" class="news-item">
<h3 id="pombase-survey-results-available">PomBase survey results available</h3>
<p><em>2013-11-20</em></p>
<p>The 2013 PomBase user survey closed at the end of October, and the results are available <a href="ftp://ftp.pombase.org/pombe/documents/2013_pombase_survey_summary.pdf">here</a> (PDF at FTP site). Some highlights have been sent to the pombe mailing list. Many thanks to all who completed the survey.</p>
</section>
<section id="2013-10-27-new-target-gene-page-section" class="news-item">
<h3 id="new-target-gene-page-section">New “Target Of” gene page section</h3>
<p><em>2013-10-27</em></p>
<p>With the October 2013 update, gene pages now include “<a href="/documentation/gene-page-target">Target Of</a>” annotations, which describe genes that affect the gene of interest. These annotations are essentially the reciprocal of <a href="/faqs/what-annotation-extension">ontology annotation extensions</a>. Each “Target Of” annotation includes a relationship that indicates how the genes are connected, the name and product of the second gene, and a reference. Genes listed under “Target Of” may include upstream regulators or enzymes that modify the product of the gene of interest. For example, the “Target Of” annotations for <a routerLink="/spombe/result/SPBC11B10.09#recipext">cdc2</a> indicate that it is a substrate of, and regulated by, the kinase Wee1 and the phosphatase Cdc25 (among others). At present, “Target Of” data includes annotations derived from <a href="/documentation/gene-page-gene-ontology#ann_ext">GO annotation extensions</a>. We will soon extend it to include data from <a routerLink="/documentation/gene-page-phenotypes">phenotype</a> annotation extensions.</p>
</section>
<section id="2013-10-21-pombase-data-update-2" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2013-10-21</em></p>
<p>The PomBase web site has been updated and now includes manually curated data through October 6, 2013. The number of community-curated papers continues to increase, ensuring that PomBase gene pages contain complete and up-to-date information. We are also pleased to announce that data tracks are now available in the genome browser for data from these two publications:</p>
<ul>
<li>Woolcock KJ, Gaidatzis D, Punga T, Bühler M. 2010. Dicer associates with chromatin to repress genome activity in Schizosaccharomyces pombe. Nat Struct Mol Biol. 2011 Jan;18(1):94-9. doi: 10.1038/nsmb.1935 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21151114">PMID:21151114</a></li>
<li>Mata J. 2013. Genome-wide mapping of polyadenylation sites in fission yeast reveals widespread alternative polyadenylation. RNA Biol. 2013 Aug 1;10(8):1407-14. doi: 10.4161/rna.25758 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23900342">PMID:23900342</a></li>
</ul>
</section>
<section id="2013-09-18-pombase-user-survey-open" class="news-item">
<h3 id="pombase-user-survey-open">PomBase User Survey open</h3>
<p><em>2013-09-18</em></p>
<p>To guide current and future development, PomBase is now conducting a <a href="https://www.surveymonkey.com/s/NDM2BQX">user survey</a>, where we invite the fission yeast research community and any other PomBase users to evaluate the resources provided so far and comment on future priorities. The survey should take about 10 minutes to complete. Thank you for your participation!</p>
<p><a href="https://www.surveymonkey.com/s/NDM2BQX" class="uri">https://www.surveymonkey.com/s/NDM2BQX</a></p>
</section>
<section id="2013-09-15-pombase-data-update-1" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2013-09-15</em></p>
<p>We have once again updated the data available on the PomBase web site. The data now includes manual curation through August 11, 2013. We are particularly pleased to note that this update includes annotations from several dozen papers curated by the <em>S. pombe</em> community. Many thanks to all who have done, or are doing, paper curation in <a href="http://curation.pombase.org/pombe/">Canto</a>.<br />
<br />
We also have an updated version of the <em>S. pombe</em>/human ortholog table available upon <a href="mailto:helpdesk@pombase.org">request</a>.</p>
</section>
<section id="2013-08-18-send-htp-data-pombase" class="news-item">
<h3 id="send-htp-data-pombase">Send HTP data to PomBase</h3>
<p><em>2013-08-18</em></p>
<p>At the pombe 2013 meeting in London, PomBase staff received numerous requests display various published data, such as gene expression, histone modifications, etc. in the genome browser. To provide this, we now invite pombe researchers to send data: If you have published any high-throughput experiments that produced data  that can be associated with genome sequence coordinates, and thereby displayed as tracks on the PomBase genome browser, please fill out the <a href="/submit-data/data-submission-form">HTP Data Submission Form</a>. We can also accept large sets of phenotype data via the Phenotype Data Submission Form. If you have any problems or questions, contact us via the <a href="mailto:helpdesk@pombase.org">PomBase Helpdesk</a> at any time.</p>
</section>
<section id="2013-07-29-connecting-pombase" class="news-item">
<h3 id="connecting-pombase">Connecting With PomBase</h3>
<p><em>2013-07-29</em></p>
<p>To complement the <a routerLink="/community/mailing-lists">mailing list</a> and twitter (<a href="http://twitter.com/PomBase"><span class="citation" data-cites="PomBase">@PomBase</span></a>) it is now possible to follow the activities of PomBase and interact with other members of the pombe community via the new <a href="http://www.linkedin.com/groups/PomBase-5122686">LinkedIn Group</a> and <a href="https://plus.google.com/114842152049074799331">Google+</a>.</p>
<p>Links to these are also available from the front page of the PomBase.org site.</p>
</section>
<section id="2013-07-21-pombelist-has-moved" class="news-item">
<h3 id="pombelist-has-moved">pombelist has moved</h3>
<p><em>2013-07-21</em></p>
<p><strong>Update: This item dates from July 2013, and the links in it no longer work.</strong> \ Please see the <a routerLink="/community">Fission Yeast Community</a> page for the current mailing list link. \ (<span class="no-break">2020-02-18</span>)</p>
<p>The pombe community mailing list, pombelist, has migrated from the Wellcome Trust Sanger Institute and is now hosted by EBI. The new address is <a href="mailto:pombelist@ebi.ac.uk" class="email">pombelist@ebi.ac.uk</a> (please note that the old address no longer works, and will generate an automatic notification including the new address). The link to <a href="http://listserver.ebi.ac.uk/mailman/listinfo/pombelist">subscribe</a> has also been updated, and the entire <a href="http://listserver.ebi.ac.uk/mailman/private/pombelist/">archive</a> is available at the new location.</p>
</section>
<section id="2013-07-18-pombase-website-update" class="news-item">
<h3 id="pombase-website-update">PomBase website update</h3>
<p><em>2013-07-18</em></p>
<p>We’d like to highlight a few improvements we’ve just made to the PomBase website. Most of the changes affect the gene pages:</p>
<ul>
<li>The basic information display at the top of each gene page is more compact.</li>
<li>For ontology annotations, the number of genes annotated is now shown, in a column labeled “Count” (also, changes behind the scenes involving this data mean that pages should load faster).</li>
<li>Annotation extensions for GO are displayed using human-friendly text instead of internal “relation” labels.</li>
<li>The Quick Links box can now be collapsed and expanded by clicking its header.</li>
<li>Display of modification annotations using PSI-MOD is improved.</li>
</ul>
<p>In addition, the Motif Search output now includes standard gene names and product descriptions. As we noted in a separate message, CDS coordinate files are once again available from the Downloads, with accurate and up-to-date data.<br />
<br />
</p>
</section>
<section id="2013-06-23-pombase-launches-community-curation" class="news-item">
<h3 id="pombase-launches-community-curation">PomBase launches community curation</h3>
<p><em>2013-06-23</em></p>
<p>At the pombe 2013 conference in London, PomBase officially launched its community curation initiative, which allows researchers to contribute publication-based annotations directly to the database. PomBase curators invite lab heads by individual email to curate newly published papers, providing links to the online curation system and its documentation. Researchers can also initiate curation of any older fission yeast publication in PubMed. Community curation uses the open-source online tool <a href="http://curation.pombase.org/pombe/">Canto</a>.</p>
</section>
<section id="2013-06-20-pombase-data-update-0" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2013-06-20</em></p>
<p>PomBase data now includes manual curation through June 9, 2013, and represents complete annotation for 664 publications (as well as partial curation of many more). A highlight of this month’s literature curation update is the addition of over 9400 phenotype annotations, representing about 95% of the phenotype data from the recently published genome-wide study of cell cycle and cell morphology (<a href="http://rsob.royalsocietypublishing.org/content/3/5/130053.abstract">Hayles <em>et al.</em> Open Biology May 2013</a>; <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23697806">PMID:23697806</a>). We have also improved the display of allele details for phenotype annotations. Other changes include better support for gene synonyms in the simple search, regular updates to the UTR data files, and a number of minor adjustments to external links in the annotation data tables and the external references section.</p>
</section>
<section id="2013-05-20-pombase-data-update" class="news-item">
<h3 id="pombase-data-update">PomBase data update</h3>
<p><em>2013-05-20</em></p>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through 13 May, 2013.</p>
</section>
<section id="2013-05-13-genedb-s-pombe-decommissioned" class="news-item">
<h3 id="genedb-s.-pombe-decommissioned">GeneDB S. pombe decommissioned</h3>
<p><em>2013-05-13</em></p>
<p>As of 14 May 2013, the old GeneDB database for <em>S. pombe</em> is no longer available. This resource consisted of static web pages, was not updated after March 2012, and not supported by an underlying relational database. The PomBase site fully supersedes GeneDB <em>S. pombe</em>, and provides improved infrastructure that will meet the current and future needs of the fission yeast community. Please e-mail the <a href="mailto:helpdesk@pombase.org">helpdesk</a> if you cannot find a replacement for any GeneDB functionality in PomBase.</p>
</section>
<section id="2013-05-07-quantitative-gene-expression-data-available-pombase" class="news-item">
<h3 id="quantitative-gene-expression-data-available-pombase">Quantitative gene expression data available in PomBase</h3>
<p><em>2013-05-07</em></p>
<p>We have extended the Gene Expression section of each gene page to support the display of quantitative expression data, and are now showing data from two publications:</p>
<ul>
<li>Marguerat S, Schmidt A, Codlin S, Chen W, Aebersold R, BählerJ. 2012. “Quantitative analysis of fission yeast transcriptomes and proteomes in proliferating and quiescent cells.” <em>Cell</em> <strong>151:</strong>671-683.</li>
<li>Wu JQ, Pollard TD. 2005. “Counting cytokinesis proteins globally and locally in fission yeast.” <em>Science</em> <strong>310:</strong>310-314.</li>
</ul>
<p>We will also soon refine the display of the new expression data, and can add more datasets upon request. We thank Sam Marguerat for preparing the data from both papers for inclusion in PomBase.</p>
<p>We have also updated the PomBase site to include manual curation through April 4, 2013, and we have updated the “all gene names” file on the PomBase ftp site. The new file is available at<br />
<a href="ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/gene_IDs_names.tsv" class="uri">ftp://ftp.ebi.ac.uk/pub/databases/pombase/pombe/Mappings/gene_IDs_names.tsv</a></p>
</section>
<section id="2013-04-11-carl-singer-foundation-established" class="news-item">
<h3 id="carl-singer-foundation-established">Carl Singer Foundation Established</h3>
<p><em>2013-04-11</em></p>
<p>Carl Singer, who was an integral part of the yeast research community for many years, passed away on February 8, 2013. Throughout his career, Carl supported yeast research both with his engineering expertise and with his good cheer. In tribute to Carl, the Singer family has now set up <a href="http://www.thecarlsingerfoundation.org/">The Carl Singer Foundation</a>, a charitable foundation dedicated to supporting scientific education in the field of yeast genetics. Questions about the foundation may be directed to Harry Singer at harry [at] thecarlsingerfoundation.org.<br />
<br />
Carl’s family would be happy to receive memories of Carl’s life at regards [at] singerinstruments.com.</p>
<p>H/T <a href="http://www.yeastgenome.org/carl-singer-foundation-established">SGD</a></p>
</section>
<section id="2013-04-02-pombe-2013-registration-abstracts-mon-8th-april" class="news-item">
<h3 id="pombe-registration-abstracts-by-mon-th-april">Pombe 2013: registration &amp; abstracts by Mon 8th April</h3>
<p><em>2013-04-02</em></p>
<pre><code>Dear Pombe Fans,
      Please remember the imminent deadline (Monday 8th April) to register and submit abstracts for Pombe 2013: http://events.embo.org/13-pombe
      Abstracts are also required from all who have already been invited to talk.
      And do book your accommodation if you haven&#39;t yet done so.
      More details are in previous email forwarded below.
      Cheers,
      -Jürg &amp; Jacky
      From: On Behalf Of Bahler, Jurg
      Sent: 18 March 2013 17:49
      To: pombelist at sanger.ac.uk
      Subject: [Pombelist] Pombe 2013: Accommodation, registration &amp; abstracts
      Dear Pombe Afficionados,
      Only three weeks left to register and submit abstracts for Pombe 2013, by Monday 8th April: http://events.embo.org/13-pombe
      Speakers for 10 plenary talks and all workshop talks will be selected from abstracts, and there will be attractive poster prizes.
      Payment is only requested after registration, by 10th May.
      Important: if you require accommodation, please do book this real soon now. Especially the most cost-effective student accommodation (comfortable, with private bathrooms) may not be available much longer, as it will be put on general sale shortly. Both hotels and student accommodation will sell out in June, so you have to arrange it now. Information on accommodation is available here: http://events.embo.org/13-pombe/application.html
      We will provide a number of free registrations for which you can apply during online registration (a few of which are reserved for student members of The Genetics Society: you become eligible if you join them now). The meeting is also supported by the Biochemical Society, so if you are, or become, a member you can apply to them for student bursaries or, if you have been a member for at least 1 year, also for travel grants.
      We highly appreciate all the generous contributions from our sponsors so far:
      Platinum: EMBO
      Gold: Biochemical Society, Genetics Society, Formedium, Sunrise Science Products, Singer Instruments, F1000Research, PomBase/Wellcome Trust
      Silver: MDPI - Open Access Publishing, Hybrigenics, Infors, Life Technologies, Bioneer
      Bronze: Nature Communications, m2p labs, Imsol, Open Biology
      We look very much forward to welcoming you in London this June!
      All the best,
      -Jürg &amp; Jacky</code></pre>
</section>
<section id="2013-04-01-data-update-pombase-web-site" class="news-item">
<h3 id="data-update-on-pombase-web-site">Data update on PomBase web site</h3>
<p><em>2013-04-01</em></p>
<p>We have once again updated the data available on the PomBase web site. The data now includes manual curation through March 6, 2013.<br />
<br />
We now expect to be able to update PomBase data every month, and will soon have an automated pipeline in place. We thank all of you for your patience during the long months when updates were infrequent.<br />
<br />
You should also see a few small improvements in the site:</p>
<ul>
<li>Ontology term pages now display the text definition for each term.</li>
<li>FASTA sequence retrieval should be quicker, and less likely to time out, for large gene lists.</li>
<li>There has been some tidying of the display of “extension” data for GO and phenotype annotations.</li>
</ul>
<p>Last month we noted an intermittent problem with the “Reference” column display in the data tables. The occurrence of this problem should now be greatly reduced, so please let us know if you see it recurring.<br />
<br />
As usual, please don’t hesitate to alert us of any other problems with data or site performance, or if you have any questions.<br />
<br />
</p>
</section>
<section id="2013-03-01-speed-improvements-and-new-data-pombase-web-site" class="news-item">
<h3 id="speed-improvements-and-new-data-on-pombase-web-site">Speed improvements and new data on PomBase web site</h3>
<p><em>2013-03-01</em></p>
<p>We have updated the data available on the PomBase web site. The data now includes manual curation through December 17, 2012, and reflects complete curation of an additional 70 papers.<br />
<br />
We have also made some improvements “under the hood” that should make gene page loading much faster. Please let us know if you have any problems with gene pages loading slowly or incompletely, whether or not you have reported issues in the past.<br />
<br />
We are aware that there is an intermittent problem with the “Reference” column display in the data tables – sometimes a PubMed ID appears instead of an author name and year. This problem will be fixed as soon as possible. Please alert us if you notice anything else odd or wrong.</p>
</section>
<section id="2012-11-06-new-data-and-new-features-pombase-web-site" class="news-item">
<h3 id="new-data-and-new-features-on-pombase-web-site">New data and new features on PomBase web site</h3>
<p><em>2012-11-06</em></p>
<p>We are pleased to announce that we have updated both data and web site features for PomBase.</p>
<p>Most importantly, we have added new data types, and upgraded the gene pages to display them.</p>
<p>We have also added more annotations of existing data types, bringing the web site content up to September 11, 2012. The new annotations include the first contributions to come in via the new community curation system, and we thank the researchers who are participating in the initial phase of community curation.</p>
<p><strong>New annotation types:</strong></p>
<ul>
<li>Phenotype annotations now use the Fission Yeast Phenotype Ontology (FYPO), and include allele details, expression levels, and experimental conditions. With FYPO, more detailed phenotypes can be described, and links between terms for related phenotypes support improved phenotype searches.</li>
<li>Many GO annotations now include “annotation extensions” that provide additional specificity. For example, extensions may capture the substrate of a catalytic activity, the cell cycle phase during which a function or process occurs, or any of several other types of supporting information for the annotation. Annotation extensions are described in more detail below.</li>
</ul>
<p>You can see these new data types on many gene pages, such as <a routerLink="/spombe/result/SPBC11B10.09">cdc2</a> or <a routerLink="/spombe/result/SPBC106.10">pka1</a>.</p>
<p><strong>New web site features:</strong></p>
<ul>
<li>Annotation display - Gene page GO and phenotype displays have been revamped to show new annotation types described above.</li>
<li>Ontology term pages - Each ontology term ID now links to pages with information about the term and lists of genes annotated to it.</li>
<li>Ontology graph links - GO and phenotype annotation sections now include links to graphical ontology displays in the genome browser.</li>
<li>Sequence highlighting - Sequence download now offers an option to show colour highlighting of regions such as UTRs, introns and exons.</li>
<li>Versions - Each gene page now shows the current data version in the format PomBase:x.y, where <em>x</em> is the Ensembl Genomes (EG) version, and <em>y</em> is the Chado version. The sequence, and sequence feature locations, remain stable within any EG version, whereas annotations change with each Chado update.</li>
<li>Protein family information is now included in the Protein Features gene page section.</li>
<li>The Protein Feature section includes a link to the Pfam entry for a protein.</li>
<li>Transcript source data (e.g. for UTR coordinates) is now displayed in the Transcript Features section.</li>
<li>A Documentation page contains links to relevant portions of the Ensembl Genomes documentation. (More documentation will be added over the coming months.)</li>
</ul>
<p><strong>What are annotation extensions?</strong></p>
<p>Annotation extensions are a form of supporting data that can be added GO annotations (or other ontology annotations) to capture additional details not provided by the ontology term itself.</p>
<p>The information in GO annotation extensions encompasses several effector-target relationships, such as</p>
<ul>
<li>localisation dependencies</li>
<li>substrates of functions, e.g. targets of a protein kinase – see the has_substrate extensions on Cdc2’s “protein serine/threonine kinase” (GO:0004674) annotations</li>
<li>activators and inhibitors</li>
<li>regulation targets of signalling pathways or transcription factors</li>
</ul>
<p>Additional extensions describe spatial and temporal aspects of processes. For example, several S. pombe annotations now include extensions that indicate in which phase of the cell cycle a gene product is found in a cellular component or involved in a process – see the pka1 annotations to “nucleus” (GO:0005634) and “cytoplasm” (GO:0005737).</p>
<p>You may also find the <a href="http://wiki.geneontology.org/index.php/Annotation_Extension">GO wiki page</a> on annotation extensions informative, although it is primarily aimed at curators.</p>
<p>Annotation extensions can also be used with phenotype annotations. The most common usage of phenotype annotation extensions is to capture which gene, protein, etc. was used in an assay. For example, the sam5 (G441E) mutation of <em>pka1</em> causes nuclear accumulation of Ste11. This is represented by annotation to the ontology term “nuclear protein accumulation” (FYPO:0000255), with the extension “assayed_using(PomBase:SPBC32C12.02)”. Extensions can also indicate expressivity or penetrance for a phenotype.</p>
</section>
<section id="2012-07-01-pombase-web-site-fully-live" class="news-item">
<h3 id="pombase-web-site-fully-live">PomBase web site fully live</h3>
<p><em>2012-07-01</em></p>
<p>We are pleased to announce that the PomBase web site, <a href="http://www.pombase.org">www.pombase.org</a>, is now fully live; the preview phase has ended. The site has been updated with an assortment of new features, datatypes, and bug fixes.</p>
<p>More recent data, reflecting additions and changes through March 20, 2012, are now available on gene pages and in search results.</p>
<p>The updated site features a Gene List Search that provides behavior equivalent to GeneDB’s List Download. You can now type or paste lists into the Gene Systematic IDs and Gene Names filters, and use the Query History to combine a gene list search with other search options. For convenience, there is a direct link to a search page pre-configured to accept a list of systematic IDs available in the Find menu, on the Find page, and here: <a routerLink="/spombe/query/builder?filter=12">http://www.pombase.org/spombe/query/builder?filter=12</a></p>
<p>The Advanced Search also now offers:</p>
<ul>
<li>options to search GO, FYPO, and other ontologies by term name or ID;</li>
<li>autocomplete for ontology term name search;</li>
<li>ability to search for genes in a region, such as centromeres or telomeres;</li>
<li>improved organization of filter selections.</li>
</ul>
<p>We have also fixed a Sequence Download error reported by some users, so that the “CDS”, CDS + UTRs“, and”CDS + UTRs + Introns" options now retrieve the correct sequences.<br />
<br />
In addition, numerous minor improvements have been made. Please send any questions or comments on the PomBase web site to us at &lt;<a href="mailto:helpdesk@pombase.org" class="email">helpdesk@pombase.org</a>&gt;.</p>
</section>
<section id="2011-11-27-pombase-preview-launch" class="news-item">
<h3 id="pombase-preview-launch">PomBase preview launch</h3>
<p><em>2011-11-27</em></p>
<p>A preview of <a href="http://www.pombase.org">PomBase</a>, the new model organism database for the fission yeast <em>Schizosaccharomyces pombe</em>, has been <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000970.html">announced to the <em>S. pombe</em> community</a> for testing and feedback. For more on PomBase, see the NAR Database Issue paper (<a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153">PubMed abstract</a>) or <a href="/about/contacts">contact the PomBase staff</a>.</p>
</section>
<section id="2011-10-27-pombase-nar-paper-published-online" class="news-item">
<h3 id="pombase-nar-paper-published-online">PomBase NAR paper published online</h3>
<p><em>2011-10-27</em></p>
<p>A paper describing <a href="http://www.pombase.org">PomBase</a> has been published online will be included in the 2012 Database Issue of <em>Nucleic Acids Research</em>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153">Abstract</a> and <a href="http://nar.oxfordjournals.org/content/early/2011/10/28/nar.gkr853">open access full text</a> are available.</p>
</section>
<section id="2011-04-28-genedb-s-pombe-now-uses-latest-release-pfam-protein-family-database250" class="news-item">
<h3 id="genedb-s.-pombe-now-uses-latest-release-pfam-protein-family-database-.-.">GeneDB (S. pombe) now uses the latest release of the Pfam protein family database(25.0).</h3>
<p><em>2011-04-28</em></p>
</section>
<section id="2011-04-21-schizosaccharomyces-comparative-genome-paper-published" class="news-item">
<h3 id="schizosaccharomyces-comparative-genome-paper-published">Schizosaccharomyces Comparative Genome Paper Published</h3>
<p><em>2011-04-21</em></p>
<p>A paper describing the major findings of the <em>Schizosaccharomyces</em> Comparative Genome Project was <a href="http://www.sciencemag.org/content/early/2011/04/20/science.1203357">published today</a> in Science Express and reported changes are included in GeneDB.</p>
<p>Further details are described in the pombe mailing list posts:</p>
<ul>
<li><a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000855.html">Schizosaccharomyces Comparative Genome Paper Published</a></li>
<li><a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000856.html">Import of the fission yeast revisions from the Broad Institute comparative genome paper into GeneDB</a>.</li>
</ul>
</section>
<section id="2011-02-01-genome-reappraisal-reveals-new-genes-and-revised-gene-structures" class="news-item">
<h3 id="genome-reappraisal-reveals-new-genes-and-revised-gene-structures">Genome reappraisal reveals new genes and revised gene structures</h3>
<p><em>2011-02-01</em></p>
<p>Further information on the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000815.html">pombe mailing list</a>.</p>
</section>
<section id="2011-01-31-annotated-transcription-start-and-termination-sites-fission-yeast" class="news-item">
<h3 id="annotated-transcription-start-and-termination-sites-fission-yeast">Annotated transcription start and termination sites for fission yeast</h3>
<p><em>2011-01-31</em></p>
<p>Further details are available on the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2011/000814.html">pombe mailing list</a>.</p>
</section>
<section id="2010-05-15-analysis-fission-yeast-deletion-publication" class="news-item">
<h3 id="analysis-fission-yeast-deletion-publication">Analysis of Fission Yeast Deletion Publication</h3>
<p><em>2010-05-15</em></p>
<p>The analysis of the fission yeast deletion collection is now published online in <a href="http://www.nature.com/nbt/journal/v28/n6/abs/nbt.1628.html">Nature Biotechnology</a>.</p>
</section>
<section id="2010-02-28-funding-pombase" class="news-item">
<h3 id="funding-pombase">Funding for PomBase</h3>
<p><em>2010-02-28</em></p>
<p>Funding was awarded by the Wellcome Trust for a fission yeast Model Organism Database, <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2010-March/002731.html">PomBase</a>.</p>
</section>
<section id="2009-11-30-fission-yeast-one-12-key-organisms-reference-genomes-project" class="news-item">
<h3 id="fission-yeast-one-key-organisms-reference-genomes-project">Fission yeast is one of the 12 key organisms of the reference genomes project</h3>
<p><em>2009-11-30</em></p>
<p>Fission yeast is one of the 12 key organisms of the <a href="http://geneontology.org/GO.refgenome.shtml?all">reference genomes project</a>. The goal of this project is to completely annotate twelve reference genomes so that those annotations may be used to effectively seed the automatic annotation efforts of other genome.</p>
</section>
<section id="2009-10-31-genedb-s-pombe-now-uses-latest-update-pfam-release-240" class="news-item">
<h3 id="genedb-s.-pombe-now-uses-latest-update-pfam-release-.">GeneDB (S. pombe) now uses the latest update to Pfam, release 24.0</h3>
<p><em>2009-10-31</em></p>
<p>GeneDB (S. pombe) now uses the latest update to Pfam, release 24.0 and 88.5% of fission yeast proteins now contain a match to at least one Pfam domain (increased from 83% in version 23).</p>
</section>
<section id="2009-09-30-fission-yeast-ensembl-fungi" class="news-item">
<h3 id="fission-yeast-ensembl-fungi">Fission yeast in Ensembl Fungi</h3>
<p><em>2009-09-30</em></p>
<p>The fission yeast genome and annotation dataset is now available as part of <a href="http://fungi.ensembl.org/index.html">Ensembl Fungi</a>.</p>
</section>
<section id="2009-08-31-genedb-now-using-version-23-pfam-protein-family-database" class="news-item">
<h3 id="genedb-now-using-version-pfam-protein-family-database.">GeneDB is now using Version 23 of the Pfam protein family database.</h3>
<p><em>2009-08-31</em></p>
<p>GeneDB is now using Version 23 of the Pfam protein family database. A total of 4154 (83%) S. pombe proteins now have at least one Pfam domain or family assignment (compared to 76% for S. cerevisiae), the highest percentage coverage for any eukaryote.</p>
</section>
<section id="2008-11-30-s-pombe-genedb-now-includes-deep-links-biological-general-repository-interaction-datasets-biogr" class="news-item">
<h3 id="s.-pombe-genedb-now-includes-deep-links-biological-general-repository-interaction-datasets-biogrid">S. pombe GeneDB now includes “deep links” to the Biological General Repository for Interaction Datasets (BioGRID)</h3>
<p><em>2008-11-30</em></p>
<p><em>S. pombe</em> GeneDB now includes “deep links” to the Biological General Repository for Interaction Datasets (BioGRID) interaction datasets from the ‘Database Cross References’ section of the individual Gene Pages.</p>
</section>
<section id="2008-04-30-global-sequence-and-chip-study-examines-eukaryotic-transcription" class="news-item">
<h3 id="global-sequence-and-chip-study-examines-eukaryotic-transcription">Global sequence and chip study examines eukaryotic transcription</h3>
<p><em>2008-04-30</em></p>
<p>Dynamic repertoire of the fission yeast transcriptome reveals: 94% of the genome is transcribed; extensive variation in different stages and conditions; global and condition-specific coupling between splicing efficiency and transcription; confirms the majority of introns; refines ~75 gene structures; identifies 453 new transcripts 26 of which were predicted to code for proteins.</p>
</section>
<section id="2008-01-31-h-mating-type-region-has-been-provided" class="news-item">
<h3 id="the-h-mating-type-region-has-been-provided">The h- mating type region has been provided</h3>
<p><em>2008-01-31</em></p>
<p>The <a routerLink="/status/mating-type-region">h- mating type region</a> has been provided by Xavier Marsellach and Lorena Aguilar.</p>
</section>
<section id="2007-12-31-baumann-and-zakian-labs-identify-telomerase-rna" class="news-item">
<h3 id="baumann-and-zakian-labs-identify-telomerase-rna">Baumann and Zakian labs identify telomerase RNA</h3>
<p><em>2007-12-31</em></p>
<p>Baumann and Zakian labs identify elusive <a href="/spombe/result/SPNCRNA.214">telomerase RNA</a> (PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18157152">18157152</a> and PMID:<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18157149">18157149</a>)</p>
</section>
<section id="2007-09-30-wellcome-trust-advanced-course-genome-wide-approaches-fission-yeast" class="news-item">
<h3 id="wellcome-trust-advanced-course-genome-wide-approaches-fission-yeast">Wellcome Trust Advanced Course ’Genome-wide approaches with fission yeast</h3>
<p><em>2007-09-30</em></p>
<p>Wellcome Trust Advanced Course ‘Genome-wide approaches with fission yeast’ held in Hinxton.</p>
</section>
<section id="2007-05-31-4th-international-fission-yeast-meeting" class="news-item">
<h3 id="th-international-fission-yeast-meeting">4th International Fission Yeast Meeting</h3>
<p><em>2007-05-31</em></p>
<p>4th International Fission Yeast Meeting held in Copenhagen.</p>
</section>
<section id="2006-12-31-genedb-representation-fission-yeast-data-moved-contigs-chromosomes" class="news-item">
<h3 id="genedb-representation-fission-yeast-data-moved-contigs-chromosomes">GeneDB representation of the fission yeast data moved from contigs to chromosomes</h3>
<p><em>2006-12-31</em></p>
<p>GeneDB representation of the fission yeast data moved from contigs to chromosomes. See the <a href="http://listserver.ebi.ac.uk/pipermail/pombelist/2007/000128.html">pombelist archive</a> for details.</p>
</section>
<section id="2006-09-30-yeast-special-issue-2006-european-fission-yeast-meeting" class="news-item">
<h3 id="yeast-special-issue-european-fission-yeast-meeting">Yeast Special Issue from the 2006 European Fission Yeast Meeting</h3>
<p><em>2006-09-30</em></p>
<p>The October issue of the journal Yeast is a <a href="http://onlinelibrary.wiley.com/doi/10.1002/yea.v23:13/issuetoc">fission yeast special issue</a> containing 13 articles and reviews commissioned as a result of the European Fission Yeast Meeting, which are FREE to download.</p>
</section>
<section id="2006-06-30-first-fission-yeast-whole-proteome-localization-study-now-published" class="news-item">
<h3 id="the-first-fission-yeast-whole-proteome-localization-study-now-published">The first fission yeast whole proteome localization study is now published</h3>
<p><em>2006-06-30</em></p>
<p>The first fission yeast whole proteome localization study is now published: <a href="http://www.nature.com/nbt/journal/v24/n7/abs/nbt1222.html">Matsuyama A. et al (2006): ORFeome cloning and global analysis of protein localization in the fission yeast Schizosaccharomyces pombe. Nat Biotech 24, 841-7.</a><br />
 </p>
</section>
<section id="2006-04-30-fission-yeast-database-survey" class="news-item">
<h3 id="fission-yeast-database-survey">Fission yeast database survey</h3>
<p><em>2006-04-30</em></p>
<p>The fission yeast database survey is now closed. You can view the survey results <a routerLink="/community/pombase_2006_survey_summary">here</a>.</p>
</section>
<section id="2006-03-17-european-fission-yeast-meeting" class="news-item">
<h3 id="european-fission-yeast-meeting">European Fission Yeast Meeting</h3>
<p><em>2006-03-17</em></p>
<p>The European Fission Yeast Meeting (16th-18th March 2006) and The Fission Yeast Bioinformatics workshop (15th - 16th Mar 2006) both took place at the Wellcome Trust Genome Campus in Hinxton (Cambridge, UK).</p>
</section>
<section id="2005-11-16-comparative-genomics-eukaryotic-microorganisms" class="news-item">
<h3 id="comparative-genomics-eukaryotic-microorganisms">Comparative Genomics of Eukaryotic Microorganisms</h3>
<p><em>2005-11-16</em></p>
<p>Comparative Genomics of Eukaryotic Microorganisms:<br />
Eukaryotic Genome Evolution, Approaches with Yeasts and Fungi</p>
<p><br />
This conference took place from 12th-17th November 2005 in Sant Feliu de Guixols, Spain. Full details can be found <a href="http://www.esf.org/index.php?id=940">here</a>.</p>
</section>
<section id="2005-11-12-second-east-coast-regional-pombe-meeting" class="news-item">
<h3 id="second-east-coast-regional-pombe-meeting">Second East Coast Regional pombe Meeting</h3>
<p><em>2005-11-12</em></p>
<p>Second East Coast Regional pombe Meeting<br />
<br />
This meeting took place from November 11-13, 2005 in Miami Beach, Florida.</p>
</section>
<section id="2004-08-31-general-repository-interaction-datasets" class="news-item">
<h3 id="general-repository-interaction-datasets">General Repository for Interaction Datasets</h3>
<p><em>2004-08-31</em></p>
<p>A project to record published genetic and physical interactions is underway with Mike Tyers and the GRID group at Toronto.</p>
</section>
<section id="2004-08-29-third-international-fission-yeast-meeting" class="news-item">
<h3 id="the-third-international-fission-yeast-meeting">The Third International Fission Yeast Meeting</h3>
<p><em>2004-08-29</em></p>
<p>The meeting was held at UC San Diego on August 24-29, 2004.</p>
</section>
<section id="2004-04-30-methods-volume-33-issue-3" class="news-item">
<h3 id="methods-volume-issue">Methods Volume 33 Issue 3</h3>
<p><em>2004-04-30</em></p>
<p>This issue of Methods includes 11 papers for fission yeast protocols including DNA damage checkpoint assays, cell wall analysis, TAP, nuclear envelope integrity assays, GFP imaging, TS mutant creation and plasmid use and construction. See the <a href="http://www.sciencedirect.com/science/journal/10462023/33/3">Methods site</a> or the <a routerLink="/fission-yeast-papers-methods-volume-33-issue-3">Table of Contents</a> (at PomBase)<a href="/fission-yeast-papers-methods-volume-33-issue-3"></a> for details of the papers including PMIDs.</p>
</section>
<section id="2003-10-31-recent-genome-wide-surveys" class="news-item">
<h3 id="recent-genome-wide-surveys">Recent Genome wide surveys</h3>
<p><em>2003-10-31</em></p>
<p>Correlations Between Gene Expression and Gene Conservation in Fission Yeast. Mata J, Bahler J. Genome Res. 2003 Nov 12 <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14613978">PMID:14613978</a><br />
<br />
FELINES: a utility for extracting and examining EST-defined introns and exons. Drabenstot SD et al Nucleic Acids Res. 2003 Nov 15;31(22):e141. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14602934">PMID:14602934</a><br />
<br />
Genome-wide distribution of DNA replication origins at A+T-rich islands in Schizosaccharomyces pombe. Segurado M, De Luis A, Antequera F. EMBO Rep. 2003 Nov;4(11):1048-53. Epub 2003 Oct 17. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14566325">PMID:14566325</a><br />
<br />
Retrotransposons and their recognition of pol II promoters: a comprehensive survey… Bowen NJ et al Genome Res. 2003 Sep;13(9):1984-97. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12952871">PMID:12952871</a></p>
</section>
<section id="2003-08-31-new-fission-yeast-book-now-published" class="news-item">
<h3 id="the-new-fission-yeast-book-now-published">The ‘new’ fission yeast book is now published</h3>
<p><em>2003-08-31</em></p>
<p>Egel, R., Copenhagen, Denmark (Ed.) The Molecular Biology of Schizosaccharomyces pombe Genetics, Genomics and Beyond ISBN:3-540-00693-1</p>
</section>
<section id="2003-02-28-schizosaccharomyces-pombe-essential-genes-pilot-study" class="news-item">
<h3 id="schizosaccharomyces-pombe-essential-genes-pilot-study">Schizosaccharomyces pombe Essential Genes: A pilot Study</h3>
<p><em>2003-02-28</em></p>
<p>Decottignies A, Sanchez-Perez I, Nurse P Genome Res. 2003 Mar;13(3):399-406. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12618370">PMID:12618370</a></p>
</section>
<section id="2002-12-31-global-transcriptional-responses-fission-yeast-environmental-stress" class="news-item">
<h3 id="global-transcriptional-responses-fission-yeast-environmental-stress">Global transcriptional responses of fission yeast to environmental stress</h3>
<p><em>2002-12-31</em></p>
<p>Chen D, Toone WM, Mata J, Lyne R, Burns G, Kivinen K, Brazma A, Jones N, Bähler J. Mol Biol Cell. 2003 Jan;14(1):214-29. <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12529438">PMID:12529438</a></p>
</section>
</div>

  </div>
  </div>
</div>
<div *ngIf="section == 'status'">
<app-page-contents-menu title="Status" titleRoute="/status">

<div class="left-menu-part left-menu-item"><a routerLink="/status/statistics">Statistics</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/status/sequencing-status">Sequencing status</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/status/centromeres">Centromeres</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/status/mating-type-region">Mating type region</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/status/telomeres">Telomeres</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/status/sequencing-updates">Sequence updates</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/status/sequence-updates-pending">Sequence updates pending</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/status/gene-coordinate-changes">Gene coordinate changes</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/status/new-and-removed-genes">New and removed genes</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/status/gene-characterisation">Gene characterisation</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/status/priority-unstudied-genes">Priority unstudied genes</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/status/unmapped_genes">Unmapped genes</a></div>

</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'centromeres'">
<h2 id="centromeres">Centromeres</h2>
<p>At present, each centromere is annotated as a single sequence feature in PomBase, which can be viewed in and downloaded from the Ensembl genome browser.</p>
<p>Chromosome coordinates (links to JBrowse):</p>
<ul>
<li>CEN 1: <a href="https://www.pombase.org/jbrowse/?loc=I%3A3753680..3789414&amp;tracks=DNA%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features&amp;highlight=">I:3753687-3789421</a></li>
<li>CEN 2: <a href="https://www.pombase.org/jbrowse/?loc=II%3A1602261..1644744&amp;tracks=DNA%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features&amp;highlight=">II:1602264-1644747</a></li>
<li>CEN 3: <a href="https://www.pombase.org/jbrowse/?loc=III%3A1070899..1136998&amp;tracks=DNA%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features&amp;highlight=">III:1070904-1137003</a></li>
</ul>
<p>Note that specific repeats within centromeres cannot yet be viewed or searched as features on PomBase pages, but they are included in the forward and reverse strand feature tracks in the genome browser, which are enabled by default.</p>
<p>Repeats are also shown in the diagram below. To see the repeat sequences, download and unzip the <a href="ftp://ftp.pombase.org/pombe/genome_sequence_and_features/artemis_files/">contiguated sequence files</a> and view them in Artemis. (See <a routerLink="/faq/is-there-an-equivalent-to-the-artemis-java-applet-in-pombase">this FAQ</a> for more information.)</p>
<p><strong>Note:</strong> Recent work by Chad Ellermeier and Gerry Smith suggests that there are only 4 +/- 1 copies of the 6760 bp repeat missing from chromosome 3</p>
<p>Please note: This map is a schematic diagram. Distances and overlaps are approximate. Please refer to the sequence data to design experimental constructs.</p>
<figure>
<img src="assets/centromeremapping.gif" width="779" height="540" alt="" /><figcaption>Centromere map</figcaption>
</figure>
<p>Centromere map from: The genome sequence of <em>Schizosaccharomyces pombe</em>. Nature 2002 Feb 21;415(6874):871-80. created by Rhian Gwilliam</p>

  </div>
  <div *ngIf="pageName == 'gene-characterisation'">
<h3 id="protein-coding-gene-characterisation-status">Protein-coding gene characterisation status</h3>
<p><app-characterisation-status-table></app-characterisation-status-table></p>
<p>Protein-coding gene characterisation status descriptions</p>
<p><strong>Published:</strong> Completely or partially characterised in a small scale experiment, with some published information about the biological role (corresponding to any of the <a href="browse-curation/fission-yeast-bp-go-slim-terms">fission yeast GO biological process slim</a> terms)</p>
<p><strong>Biological role inferred:</strong> A biological role (as above, a <a href="browse-curation/fission-yeast-bp-go-slim-terms">fission yeast GO Process slim</a> term) is inferred from homology to an experimentally characterised gene product</p>
<p><strong>Conserved protein (unknown biological role):</strong> Conserved outside the <i>Schizosaccharomyces</i>, but nothing known about the biological role in any organism</p>
<p><strong><i>Schizosaccharomyces</i> specific protein, uncharacterized:</strong> Unpublished and found only in fission yeast (<i>S. pombe</i>, <i>S. octosporus</i>, <i>S. japonicus</i>, <i>S. cryophilus</i>); nothing known about biological role. May be single copy or a member of a multi-member family.</p>
<p><strong><i>S. pombe</i> specific protein, uncharacterized:</strong> Unpublished and found only in <i>S. pombe</i> (not detected in other <i>Schizosaccharomyces</i> species); nothing known about biological role</p>
<p><strong>Dubious:</strong> Unlikely to be protein coding</p>
<p><strong>Transposon:</strong> A predicted or experimentally verified transposable element.</p>
<p>Note: You can retrieve current lists of genes with each characterisation status using the <a routerLink="/query">advanced search</a>. Select the Characterisation status query, then choose a status from the pulldown menu, and submit. A set of <a routerLink="/status/gene-characterisation-statistics-history">historical data</a> recorded at various intervals is available.</p>

  </div>
  <div *ngIf="pageName == 'gene-characterisation-statistics-history'">
<h3 id="protein-coding-gene-characterisation-statistics-history">Protein-coding gene characterisation statistics history</h3>
<h4 id="history-of-characterisation-status-using-current-status-descriptions">History of characterisation status using current status descriptions</h4>
<h5 id="introduced-with-pombase-chado-release-46-august-2014">(introduced with PomBase Chado release 46, August 2014):</h5>
<table>
 <thead>
  <tr>
   <th style="width: 100px;">Date</th>
   <th style="width: 100px;">Published</th>
   <th style="width: 100px;">Role inferred</th>
   <th style="width: 100px;">Conserved unknown</th>
   <th style="width: 100px;"><em><em><strong>Schizo.</strong></em></em></th>
   <th style="width: 100px;"><em>S. pombe</em></th>
   <th style="width: 100px;">Dubious</th>
   <th style="width: 100px;">Transposon</th>
   <th style="width: 100px;">Total</th>
  </tr>
 </thead>
 <tbody>
  <tr>
  <td><span class="no-break">2019-12-31</span></td>
  <td>2368</td>
  <td>2010</td>
  <td>406</td>
  <td>162</td>
  <td>122</td>
  <td>55</td>
  <td>13</td>
  <td>5136</td>
 </tr>
 <tbody>
  <tr>
  <td><span class="no-break">2018-10-05</span></td>
  <td>2339</td>
  <td>2035</td>
  <td>410</td>
  <td>163</td>
  <td>122</td>
  <td>55</td>
  <td>13</td>
  <td>5137</td>
 </tr>
 <tbody>
  <tr>
  <td><span class="no-break">2017-01-30</span></td>
  <td>2235</td>
  <td>1996</td>
  <td>511</td>
  <td>180</td>
  <td>145</td>
  <td>54</td>
  <td>13</td>
  <td>5171</td>
 </tr>
  <tr>
  <td><span class="no-break">2016-09-12</span></td>
  <td>2227</td>
  <td>2034</td>
  <td>503</td>
  <td>190</td>
  <td>149</td>
  <td>55</td>
  <td>13</td>
  <td>5171</td>
 </tr>
 <tr>
  <td><span class="no-break">2015-01-19</span></td>
  <td>2154</td>
  <td>2050</td>
  <td>529</td>
  <td>178</td>
  <td>143</td>
  <td>76</td>
  <td>13</td>
  <td>5143</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-09-17</span></td>
  <td>2124</td>
  <td>2066</td>
  <td>537</td>
  <td>183</td>
  <td>143</td>
  <td>77</td>
  <td>&nbsp;13</td>
  <td>5143</td>
 </tr>
 </tbody>
</table>
<h4 id="history-of-characterisation-status-using-previous-status">History of characterisation status using previous status</h4>
<p>descriptions: ####</p>
<table>
 <thead>
  <tr>
   <th style="width: 100px;">Date</th>
   <th style="width: 100px;">Published</th>
   <th style="width: 100px;">Role inferred</th>
   <th style="width: 100px;">Conserved unknown</th>
   <th style="width: 100px;"><em>S. pombe</em></th>
   <th style="width: 100px;">Orphan</th>
   <th style="width: 100px;">Dubious</th>
   <th style="width: 100px;">Transposon</th>
   <th style="width: 100px;">Total</th>
  </tr>
 </thead>
 <tbody>
 <tr>
  <td><span class="no-break">2006-08-25</span></td>
  <td>1560</td>
  <td>2433</td>
  <td>458</td>
  <td>68</td>
  <td>403</td>
  <td>57</td>
  <td>&nbsp;</td>
  <td>4979</td>
 </tr>
 <tr>
  <td><span class="no-break">2007-02-27</span></td>
  <td>1607</td>
  <td>2329</td>
  <td>572</td>
  <td>47</td>
  <td>364</td>
  <td>60</td>
  <td>&nbsp;</td>
  <td>4979</td>
 </tr>
 <tr>
  <td><span class="no-break">2008-06-05</span></td>
  <td>1715</td>
  <td>2210</td>
  <td>634</td>
  <td>42</td>
  <td>344</td>
  <td>57</td>
  <td>&nbsp;</td>
  <td>5002</td>
 </tr>
 <tr>
  <td><span class="no-break">2009-01-14</span></td>
  <td>1812</td>
  <td>2132</td>
  <td>636</td>
  <td>56</td>
  <td>318</td>
  <td>57</td>
  <td>&nbsp;</td>
  <td>5011</td>
 </tr>
 <tr>
  <td><span class="no-break">2010-02-22</span></td>
  <td>1848</td>
  <td>2160</td>
  <td>614</td>
  <td>55</td>
  <td>286</td>
  <td>57</td>
  <td>&nbsp;</td>
  <td>5020</td>
 </tr>
 <tr>
  <td><span class="no-break">2010-06-30</span></td>
  <td>1916</td>
  <td>2163</td>
  <td>592</td>
  <td>54</td>
  <td>285</td>
  <td>58</td>
  <td>11</td>
  <td>5025</td>
 </tr>
 <tr>
  <td><span class="no-break">2011-04-27</span></td>
  <td>1916</td>
  <td>2169</td>
  <td>588</td>
  <td>64</td>
  <td>315</td>
  <td>79</td>
  <td>11</td>
  <td>5142</td>
 </tr>
 <tr>
  <td><span class="no-break">2011-12-16</span></td>
  <td>1936</td>
  <td>2165</td>
  <td>582</td>
  <td>66</td>
  <td>309</td>
  <td>73</td>
  <td>11</td>
  <td>5142</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-05-23</span></td>
  <td>2012</td>
  <td>2137</td>
  <td>541</td>
  <td>66</td>
  <td>303</td>
  <td>71</td>
  <td>13</td>
  <td>5143</td>
 </tr>
 <tr>
  <td><span class="no-break">2013-12-06</span></td>
  <td>2083</td>
  <td>2096</td>
  <td>522</td>
  <td>64</td>
  <td>294</td>
  <td>71</td>
  <td>13</td>
  <td>5143</td>
 </tr>
 <tr>
  <td><span class="no-break">2014-07-14</span></td>
  <td>2110</td>
  <td>2076</td>
  <td>517</td>
  <td>64</td>
  <td>292</td>
  <td>71</td>
  <td>13</td>
  <td>5143</td>
 </tr>
 </tbody>
</table>
<h3 id="status-descriptions">Status Descriptions</h3>
<table>
<thead>
 <tr>
  <th>Shorthand</th>
  <th>Full name</th>
  <th>Description</th>
  <th>Comment</th>
 </tr>
</thead>
<tbody>
 <tr>
  <td>Published</td>
  <td>Experimentally characterised (or published)</td>
  <td>Completely or partially characterised in a small scale experiment, with some published information about the biological role (corresponding to any of the fission yeast GO biological process slim terms)</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>Role inferred</td>
  <td>Role inferred from homology</td>
  <td>A biological role (as above, a fission yeast GO process slim term) is inferred from homology to an experimentally characterised gene product</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>Conserved unknown</td>
  <td>Conserved protein (unknown biological role)</td>
  <td>Conserved outside the <em>Schizosaccharomyces</em>, but nothing known about the biological role in any organism</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><em>Schizo.</em></td>
  <td><em><em>Schizosaccharomyces</em>&nbsp;</em>specific protein, uncharacterised</td>
  <td>Unpublished and found only in fission yeast (<em>S. pombe</em>,&nbsp;<em>S. octosporus,&nbsp;<em>S. japonicus, S. cryophilus</em></em><em></em>); nothing known about biological role. May be single copy or a member of a multi-member family.</td>
  <td>Introduced Sept. 2014</td>
 </tr>
 <tr>
  <td><em>S. pombe</em></td>
  <td><em><em>S. pombe</em>&nbsp;</em>specific protein, uncharacterised</td>
  <td>Unpublished and found only in&nbsp;<em>S. pombe&nbsp;</em>(not detected in other&nbsp;<em>Schizosaccharomyces</em>&nbsp;species); nothing known about biological role</td>
  <td>Introduced Sept. 2014&nbsp;</td>
 </tr>
 <tr>
  <td>Transposon</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>Dubious</td>
  <td>&nbsp;</td>
  <td>Unlikely to be protein coding</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td><em>S. pombe</em></td>
  <td>&nbsp;<em>S. pombe</em>&nbsp;specific families</td>
  <td>Unpublished and found only in fission yeast (<em>S. pombe</em>,&nbsp;<em>S. octosporus,&nbsp;</em><em>S. japonicus, S. cryophilus</em>); nothing known about biological role, but are not single copy (duplications in fission yeast)</td>
  <td>&nbsp;Used Aug. 2006-Aug. 2014</td>
 </tr>
 <tr>
  <td>&nbsp;Orphan</td>
  <td>Sequence orphan, uncharacterised</td>
  <td>Unpublished and found only in fission yeast (<em>S. pombe</em>,&nbsp;<em>S. octosporus</em>,&nbsp;<em>S. japonicus, S. cryophilus</em>); nothing known about biological role</td>
  <td>Used Aug. 2006-Aug. 2014</td>
 </tr>
 </tbody>
 </table>

  </div>
  <div *ngIf="pageName == 'gene-coordinate-changes'">
<h3 id="gene-coordinate-changes">Gene coordinate changes</h3>
<table>
<thead>
<tr class="header">
<th>Systematic ID</th>
<th>Standard name</th>
<th>Comments</th>
<th>Reference</th>
<th>Date</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>SPAC2E12.05</td>
<td>wtf1</td>
<td>pseudo-&gt;coding; coordinates changed from 5064305..5066231 to join(5064781..5064919,5065087..5065332,5065380..5065610,5065652..5065705,5065760..5065836)</td>
<td>PMID:28631610</td>
<td><span class="no-break">2020-05-01</span></td>
</tr>
<tr class="even">
<td>SPBC1706.02c</td>
<td>wtf2</td>
<td>Coordinates changed from complement(join(593188..593261,593311..593368,593437..593575) to complement(join(593185..593261,593393..593368,593437..593575)</td>
<td>PMID:28631610, PMID:30991417</td>
<td><span class="no-break">2020-05-01</span></td>
</tr>
<tr class="odd">
<td>SPCC548.02c</td>
<td>wtf3</td>
<td>Coordinates changed from complement(join(219185..219261,219294..219368,219409..219684,219734..219979,220148..220283) to complement(join(219185..219261,219315..219368,219409..219684,219734..219979,220148..220286)</td>
<td>PMID:28631610, PMID:30991417</td>
<td><span class="no-break">2020-05-01</span></td>
</tr>
<tr class="even">
<td>SPCC548.03c</td>
<td>wtf4</td>
<td>Coordinates changed from complement(join(221199..221335,221395..221457,221498..221689,221726..222016,222065..222337,222566..222701)) to complement(join(221199..221344,221395..221457,221498..221689,221726..222016,222065..222337,222566..222701))</td>
<td>PMID:28631610, PMID:30991417</td>
<td><span class="no-break">2020-05-01</span></td>
</tr>
<tr class="odd">
<td>SPCC553.05c</td>
<td>wtf6</td>
<td>Coordinates changed from complement(join(297354..297655,297705..297950,298119..298194)) to complement(join(297140..297216,297266..297307, 297349..297655,297705..297950,298119..298258)): introduces stop codon at position 25</td>
<td>PMID:28631610, PMID:30991417</td>
<td><span class="no-break">2020-05-01</span></td>
</tr>
<tr class="even">
<td>SPCC736.05</td>
<td>wtf7</td>
<td>Coordinates changed from join(320608..320755,320964..321335,321387..321523) to join(320617..320755,320964..321335,321387..321523)</td>
<td>PMID:28631610, PMID:30991417</td>
<td><span class="no-break">2020-05-01</span></td>
</tr>
<tr class="odd">
<td>SPCC306.09c</td>
<td>wtf8</td>
<td>pseudo-&gt;coding; coordinates changed from join(427445..427583,427752..427997,428072..428290,428336..428527,428568..428630,428681..428763) to join(427445..427583,427752..427997,428047..428290,428336..428527,428568..428630,428681..428763); introduces stop codon at position 138</td>
<td>PMID:28631610, PMID:30991417</td>
<td><span class="no-break">2020-05-01</span></td>
</tr>
<tr class="even">
<td>SPCC622.21</td>
<td>wtf12</td>
<td>Coordinates changed from join(1401530..1401668,1401900..1402172,1402490..1402551,1402553..1402672) to join(1401530..1401668,1401900..1402172,1402221..1402448,1402490..1402672); introduces stop codon at position 274</td>
<td>PMID:30991417</td>
<td><span class="no-break">2020-05-01</span></td>
</tr>
<tr class="odd">
<td>SPCC162.04c</td>
<td>wtf13</td>
<td>Coordinates changed from complement(join(1580123..1580613,1580650..1581006,1581055..1581327,1581559..1581694)) to complement(join(1580123..1580268,1580319..1580381,1580422..1580613,1580650..1581006,1581055..1581327,1581559..1581694))</td>
<td>PMID:28631610, PMID:30991417</td>
<td><span class="no-break">2020-05-01</span></td>
</tr>
<tr class="even">
<td>SPCC285.06c</td>
<td>wtf17</td>
<td>Coordinates changed from complement(join(1805166..1805242,1805299..1805352,1805393..1805614,1805664..1805909,1806076..1806225,1806411..1806450)) to complement(join(1805166..1805242,1805299..1805352,1805393..1805614,1805664..1805909,1806076..1806214))</td>
<td>PMID:28631610, PMID:30991417</td>
<td><span class="no-break">2020-05-01</span></td>
</tr>
<tr class="odd">
<td>SPCC285.07c</td>
<td>wtf18</td>
<td>Coordinates changed from complement(join(1807004..1807131,1807182..1807244,1807285..1807476,1807513..1807726,1807760..1807836,1807885..1808157,1808396..1808534)) to complement(join(1807004..1807131,1807182..1807244,1807285..1807476,1807513..1807836,1807885..1808157,1808396..1808534))</td>
<td>PMID:28631610, PMID:30991417</td>
<td><span class="no-break">2020-05-01</span></td>
</tr>
<tr class="even">
<td>SPCC576.16c</td>
<td>wtf22</td>
<td>Coordinates changed from complement(join(2109181..2109395,2109432..2109608,2109645..2109923,2109973..2110221,2110391..2110528)) to complement(join(2109178..2110528,2109178..2109281,2109333..2109395,2109436..2109608,2109645..2109923,2109973..2110221,2110391..2110528)); introduces stop codon at position 50</td>
<td>PMID:28631610</td>
<td><span class="no-break">2020-05-01</span></td>
</tr>
<tr class="odd">
<td>SPCC830.02</td>
<td>wtf24</td>
<td>Coordinates changed from join(2181204..2181342,2181510..2181758,2181808..2182031,2182077..2182371,2182422..2182504) to join(2181204..2181342,2181510..2181758,2181808..2182031,2182077..2182268,2182309..2182371,2182422..2182504); introduces stop codon at position 145</td>
<td>PMID:28631610, PMID:30991417</td>
<td><span class="no-break">2020-05-01</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPNCRNA.103">SPNCRNA.103</a></td>
<td>sme2</td>
<td>Coordinates changed from complement(339346..340012) to complement(339346..340907)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24920274">PMID:24920274</a></td>
<td><span class="no-break">2019-11-08</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC577.05c">SPBC577.05c</a></td>
<td>rec27</td>
<td>Coordinates changed from complement(join(757566..757746,757796..757951,758130..758197)) to complement(join(757566..757746,757796..757951,758002..758042))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/28469148">PMID:28469148</a></td>
<td><span class="no-break">2017-09-14</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPSNORNA.41">SPSNORNA.41</a></td>
<td>snR46</td>
<td>Switched from + strand to - strand</td>
<td>pers. comm. Francois Bachand</td>
<td><span class="no-break">2017-05-25</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC530.13">SPBC530.13</a></td>
<td>lsc1</td>
<td>Frameshifted by Chr_II:825012!G→A; coordinates changed from join(824374..824534,824579..824660,824874..825011,825065..825188,825245..825747) to join(824374..824534,824579..824660,824874..824999,825065..825188,825245..825747)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-17</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC3A11.06">SPAC3A11.06</a></td>
<td>mvp1</td>
<td>Frameshifted by Chr_I:3460318!T→T; coordinates changed from complement(join(3459233..3459436,3459492..3459665,3459705..3459897,3459941..3460185,3460234..3460364,3460425..3460442,3460508..3460560,3460642..3461263,3461323..3461583,3461766..3461859)) to complement(join(3459233..3459436,3459492..3459665,3459705..3459897,3459941..3460185,3460234..3460318,3460318..3460372,3460425..3460442,3460508..3460560,3460642..3461263,3461323..3461583,3461766..3461859))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-06</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC12B10.09">SPAC12B10.09</a></td>
<td>pet801</td>
<td>Coordinates changed from join(4588250..4588510,4588567..4588723,4588869..4589323,4589410..4589574) to join(4588487..4588510,4588567..4588723,4588869..4589323,4589410..4589574)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC16D10.10">SPBC16D10.10</a></td>
<td>tad2</td>
<td>Frameshifted by Chr_II:3619003!A→AG; coordinates changed from join(3618117..3618179,3618228..3618617,3618658..3618702,3618747..3618918,3618956..3619071,3619113..3619321,3619366..3619540) to join(3618117..3618179,3618228..3618617,3618658..3618702,3618747..3618920,3618968..3619003,3619003..3619071,3619113..3619321,3619366..3619540)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC13E7.01">SPBC13E7.01</a></td>
<td>cwf22</td>
<td>Frameshifted by Chr_II:3040332!C→CG; coordinates changed from join(3037988..3040331,3040334..3040650) to join(3037988..3040332,3040332..3040650)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC32F12.08c">SPBC32F12.08c</a></td>
<td>duo1</td>
<td>Frameshifted by Chr_II:2798040!CT→C; coordinates changed from complement(2798007..2798507) to complement(join(2797652..2798040,2798042..2798507))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC4F6.10">SPBC4F6.10</a></td>
<td>vps901</td>
<td>Frameshifted by Chr_II:2709414!G→GC; coordinates changed from join(2708076..2709415,2709418..2709793) to join(2708076..2709414,2709414..2709793)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC14C8.09c">SPBC14C8.09c</a></td>
<td></td>
<td>Frameshifted by Chr_II:2219928!A→AT; coordinates changed from complement(join(2219430..2219934,2219937..2220322)) to complement(join(2219430..2219928,2219928..2220322))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC23G7.06c">SPBC23G7.06c</a></td>
<td></td>
<td>Frameshifted by Chr_II:2108180!T→TA; coordinates changed from complement(join(2106448..2108178,2108181..2108786)) to complement(join(2106448..2108180,2108180..2108786))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC29A3.08">SPBC29A3.08</a></td>
<td>pof4</td>
<td>Frameshifted by Chr_II:2053516!G→GC; coordinates changed from join(2053033..2053517,2053519..2053600) to join(2053033..2053517,2053517..2053823)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC1A4.06c">SPBC1A4.06c</a></td>
<td>tam41</td>
<td>Frameshifted by Chr_II:1987101!CG→C and Chr_II:1987117!TG→T; coordinates changed from complement(join(1987044..1987099,1987102..1987675,1987737..1988015,1988081..1988353)) to complement(join(1987044..1987100,1987102..1987117,1987119..1987675,1987737..1988015,1988081..1988353))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC1E8.03c">SPBC1E8.03c</a></td>
<td></td>
<td>Frameshifted by Chr_II:1960392!A→AG; coordinates changed from complement(1960384..1961817) to complement(join(1960166..1960392,1960392..1961817))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC16E9.16c">SPBC16E9.16c</a></td>
<td>lsd90</td>
<td>Frameshifted by Chr_II:1948953!GA→G and Chr_II:1950050!A→AG; coordinates changed from complement(join(1947985..1948953,1948955..1950050,1950053..1950255)) to complement(join(1947985..1948952,1948954..1950048,1950048..1950255))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC29A4.03c">SPAC29A4.03c</a></td>
<td>mrps9</td>
<td>Frameshifted by Chr_I:5142627!A→AG; coordinates changed from join(5142407..5142628,5142631..5143224) to join(5142407..5142626,5142626..5143224)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC29E6.04">SPAC29E6.04</a></td>
<td>nnf1</td>
<td>Frameshifted by Chr_I:4410191!CG→C; coordinates changed from join(4409773..4410192, 4410194..4410391) to join(4409773..4410190,4410192..4410391)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC29E6.03c">SPAC29E6.03c</a></td>
<td>uso1</td>
<td>Frameshifted by Chr_I:4407494!T→TG; coordinates changed from complement(join(4405990..4407375,4407447..4409096,4409171..4409338)) to complement(join(4405990..4407494,4407494..4409096,4409171..4409338))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1071.01c">SPAC1071.01c</a></td>
<td>pta1</td>
<td>Frameshifted by Chr_I:3855790!GT→G; coordinates changed from complement(3855780..3857792) to complement(join(3855626..3855790,3855792..3857792))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC3A11.09">SPAC3A11.09</a></td>
<td>sod22</td>
<td>Frameshifted by Chr_I:3450130!GT→G; coordinates changed from complement(join(3450114..3452269,3452485..3452608)) to complement(join(3450056..3450130,3450132..3452269,3452485..3452608))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1486.05">SPAC1486.05</a></td>
<td>nup189</td>
<td>Frameshifted by Chr_I:3197528!A→AG; coordinates changed from join(3197028..3197529,3197529..3202450) to join(3197028..3197529,3197529..3202450)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC688.08">SPAC688.08</a></td>
<td>srb8</td>
<td>Frameshifted by Chr_I:3125118!A→AT; coordinates changed from join(3123163..3125057,3125099..3126310,3126356..3126653) to join(3123163..3125118,3125118..3126310,3126356..3126653)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC823.04">SPAC823.04</a></td>
<td>rrp36</td>
<td>Frameshifted by Chr_I:2588066!C→CA and Chr_I:2588021!C→CA; coordinates changed from join(2587729..2587897,2587952..2588028,2588031..2588143,2588182..2588620) to join(2587729..2587897,2587952..2588025,2588025..2588066,2588066..2588620)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC22F3.11c">SPAC22F3.11c</a></td>
<td>snu23</td>
<td>Frameshifted by Chr_I:682993!TC-&gt;T; coordinates changed to join(682874..682996,682998..683109,683160..683446)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC17G8.01c">SPAC17G8.01c</a></td>
<td>trl1</td>
<td>Frameshifted by Chr_I:2343703!G→GA; coordinates changed from complement(join(2341346..2343706,2343754..2343756)) to complement(join(2341346..2343703,2343703..2343756))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAP27G11.10c">SPAP27G11.10c</a></td>
<td>nup184</td>
<td>Frameshifted by Chr_I:1625092!T→TC; coordinates changed from complement(join(1625076..1629753,1629825..1629841)) to complement(join(1624885..1625092,1625092..1629753,1629825..1629841))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC3A12.04c">SPAC3A12.04c</a></td>
<td>rpp1</td>
<td>Frameshifted by Chr_I:1424708!CA→C; coordinates changed from complement(join(1424697..1425161,1425231..1425329,1425372..1425430,1425478..1425562)) to complement(join(1424660..1424708,1424710..1425161,1425231..1425329,1425372..1425430,1425478..1425562))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26615217">PMID:26615217</a> and pers. comm. Li-Lin Du</td>
<td><span class="no-break">2017-04-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC21B10.12">SPBC21B10.12</a></td>
<td>rec6</td>
<td>Coordinates changed from complement(join(1649038..1649472,1649558..1649671)) to complement(join(1649038..1649472,1649558..1649727,1649815..1649878,1650043..1650156))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26917764">PMID:26917764</a></td>
<td><span class="no-break">2016-04-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC22F3.11c">SPAC22F3.11c</a></td>
<td>snu23</td>
<td>Coordinates changed from complement(890220..890546) to join(682874..682996,682998..683109,683160..683446) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22365419">PMID:22365419</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2016-03-17</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC29A4.23">SPAC29A4.23</a></td>
<td></td>
<td>Frameshifted; intron/exon boundary changes; coordinates changed to join(5139024..5139190,5139192..5139424,5139427..5139575)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/26494834">PMID:26494834</a></td>
<td><span class="no-break">2015-11-28</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC13G1.04c">SPBC13G1.04c</a></td>
<td>abh1</td>
<td>N-terminal shortened to use downstream methionine</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22365419">PMID:22365419</a></td>
<td><span class="no-break">2015-10-13</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1486.05">SPAC1486.05</a></td>
<td>nup189</td>
<td>Coordinates changed from join(3197028..3197443,3197530..3202450) to join(3197028..3197528,3197528..3202450)</td>
<td>pers. comm. Y. Hiraoka</td>
<td><span class="no-break">2015-04-14</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC320.09">SPCC320.09</a></td>
<td>hem15</td>
<td>Coordinates changed from complement(join(149153..150163,150279..150323,150374..150472)) to complement(join(149153..150163,150279..150323,150374..150589)) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC417.03">SPCC417.03</a></td>
<td></td>
<td>Coordinates changed from join(1672003..1672051,1672111..1672364) to join(1672003..1672051,1672243..1672332) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC162.07">SPCC162.07</a></td>
<td>ent1</td>
<td>Coordinates changed from complement(join(1571648..1573733,1574016..1574050)) to complement(join(1571648..1573733,1574258..1574280)) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC27B12.10c">SPBC27B12.10c</a></td>
<td>tom7</td>
<td>Coordinates changed from complement(join(1344032..1344157,1344214..1344237)) to complement(join(1343913..1343934,1343989..1344025,1344082..1344157,1344214..1344237)) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC146.01">SPBC146.01</a></td>
<td>med15</td>
<td>Coordinates changed from join(996843..997019,997066..1000080) to join(996797..996870,996917..997019,997066..1000080) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC823.02">SPAC823.02</a></td>
<td></td>
<td>Coordinates changed from join(2583804..2583902,2583941..2583973) to join(2583863..2583902,2583941..2584047) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437"></a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC839.02">SPBC839.02</a></td>
<td></td>
<td>Coordinates changed from join(597611..599124,599256..599334) to 597611..599125 based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1D4.08">SPAC1D4.08</a></td>
<td>pis1</td>
<td>Coordinates changed from join(650814..650893,651173..651848) to join(650488..650545,651193..651848) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC22F3.04">SPAC22F3.04</a></td>
<td>mug62</td>
<td>Coordinates changed from complement(join(698033..698177, 698223..698441, 698480..698841, 698884..699104, 699148..699363, 699410..702533)) to complement(join(698033..698177, 698223..698441, 698480..698841, 698884..699104, 699148..699363, 699410..702557, 702603..702755)) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC110.05">SPAC110.05</a></td>
<td></td>
<td>Coordinates changed from complement(1905657..1905884) to complement(1905728..1905820) based on ribosome profiling</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437">PMID:24929437</a></td>
<td><span class="no-break">2014-07-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPSNRNA.07">SPSNRNA.07</a></td>
<td>snu32</td>
<td>Coordinates changed from 3959174-3958768 to 3959086-3958832, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/1560765">PMID:1560765</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPSNRNA.06">SPSNRNA.06</a></td>
<td>snu6</td>
<td>Coordinates changed from (2562559-2562374,2562323-2562085) to (2562276-2562323,2562374-2562427), to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/2909894">PMID:2909894</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPSNRNA.05">SPSNRNA.05</a></td>
<td>snu5</td>
<td>Coordinates changed from 3236617-3237051 to 3236867-3236986, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/2587274">PMID:2587274</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPSNRNA.04">SPSNRNA.04</a></td>
<td>snu4</td>
<td>Coordinates changed from 467025-467823 to 467233-4683361, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/2795654">PMID:2795654</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPSNRNA.03">SPSNRNA.03</a></td>
<td>snu3</td>
<td>Coordinates changed from 988266-987631 to 987825-987570, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/3194197">PMID:3194197</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPSNRNA.02">SPSNRNA.02</a></td>
<td>snu2</td>
<td>Coordinates changed from 959770-959305 to 959856-959370, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/3244367">PMID:3244367</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPSNRNA.01">SPSNRNA.01</a></td>
<td>snu1</td>
<td>Coordinates changed from 3019819-3020472 to 3020205-3020353, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/2188102">PMID:2188102</a></td>
<td><span class="no-break">2014-06-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPNCRNA.98">SPNCRNA.98</a></td>
<td>srp7</td>
<td>Coordinates changed from 4268566-4269064 to 4268662-4268916, to represent the actual functional RNA</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/2837764">PMID:2837764</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/16453822">PMID:16453822</a></td>
<td><span class="no-break">2014-06-12</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC5H10.06c">SPAC5H10.06c</a></td>
<td>adh4</td>
<td>N-terminal shortened to use downstream methionine; removed 43 amino acids; UTR exon annotated</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/24003116">PMID:24003116</a></td>
<td><span class="no-break">2014-01-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC3D6.04c">SPBC3D6.04c</a></td>
<td>mad1</td>
<td>N-terminal shortened to use downstream methionine; removed 13 amino acids</td>
<td>pers. comm. Silke Hauf</td>
<td><span class="no-break">2013-12-06</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC25B2.07c">SPBC25B2.07c</a></td>
<td>mmb1</td>
<td>N-terminal shortened to use downstream methionine</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/21856157">PMID:21856157</a></td>
<td><span class="no-break">2013-12-06</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC2F3.13c">SPAC9G1.15c</a></td>
<td>mzt1</td>
<td>N-terminal shortened to use downstream methionine</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/23885124">PMID:23885124</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/24006493">PMID:24006493</a></td>
<td><span class="no-break">2013-11-01</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC2F3.13c">SPAC2F3.13c</a></td>
<td></td>
<td>removed N terminal region overlapping with plp1</td>
<td></td>
<td><span class="no-break">2012-12-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC4B3.05c">SPCC4B3.05c</a></td>
<td>hem12</td>
<td>N terminal extended by 4 amino acids based on homology</td>
<td></td>
<td><span class="no-break">2012-01-05</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1281.07c">SPCC1281.07c</a></td>
<td></td>
<td>N terminal extended by 24 amino acids</td>
<td></td>
<td><span class="no-break">2012-01-05</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC29A3.06">SPBC29A3.06</a></td>
<td></td>
<td>N terminal extended</td>
<td></td>
<td><span class="no-break">2012-01-05</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1006.03c">SPAC1006.03c</a></td>
<td>red1</td>
<td>intron donor corrected</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC22F3.11c">SPAC22F3.11c</a></td>
<td>snu23</td>
<td>gene structure updated</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/21511999">PMID:21511999</a></td>
<td><span class="no-break">2011-03-11</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC947.10">SPBC947.10</a></td>
<td>dsc1</td>
<td>N-terminal extended to use upstream methionine</td>
<td>pers. comm. E. Stewart</td>
<td><span class="no-break">2011-02-04</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPACUNK4.14">SPACUNK4.14</a></td>
<td>mdb1</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC18B11.05">SPAC18B11.05</a></td>
<td>gpi18</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC18B11.05">SPAC18B11.05</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC13F5.07c">SPAC13F5.07c</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC1709.12">SPBC1709.12</a></td>
<td>rid1</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC1711.10c">SPBC1711.10c</a></td>
<td>npl4</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC3A11.03">SPAC3A11.03</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC16H5.12c">SPBC16H5.12c</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC4F6.10">SPBC4F6.10</a></td>
<td>vps901</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC13E7.01">SPBC13E7.01</a></td>
<td>cwf22</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1322.16">SPCC1322.16</a></td>
<td>phb2</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC4G3.12c">SPCC4G3.12c</a></td>
<td>not2</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC29A4.03c">SPAC29A4.03c</a></td>
<td>mrps9</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC25H2.11c">SPBC25H2.11c</a></td>
<td>spt7</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC651.06">SPBC651.06</a></td>
<td>mug166</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBP8B7.31">SPBP8B7.31</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC29A10.17">SPBC29A10.17</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB17E12.14c">SPAPB17E12.14c</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC688.04c">SPAC688.04c</a></td>
<td>gst3</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC29A4.03c">SPAC29A4.03c</a></td>
<td>mrps9</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC688.11">SPAC688.11</a></td>
<td>end4</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC823.04">SPAC823.04</a></td>
<td></td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
<td><span class="no-break">2011-02-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC622.17">SPCC622.17</a></td>
<td>apn1</td>
<td>added 2 5’ exons</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21193357">PMID:21193357</a></td>
<td><span class="no-break">2011-01-06</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC306.10">SPCC306.10</a></td>
<td></td>
<td>pseudo-&gt;coding</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBPB21E7.02c">SPBPB21E7.02c</a></td>
<td></td>
<td>pseudo-&gt;coding</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC8D2.07c">SPBC8D2.07c</a></td>
<td>sfc9</td>
<td></td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC3E7.04c">SPBC3E7.04c</a></td>
<td></td>
<td>pseudo-&gt;coding</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC1778.02">SPBC1778.02</a></td>
<td>rap1</td>
<td></td>
<td>pers. comm. Yasushi Hiraoka</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC1348.12">SPBC1348.12</a></td>
<td></td>
<td>pseudo-&gt;coding</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC12D12.09">SPBC12D12.09</a></td>
<td>rev7</td>
<td></td>
<td>pers. comm. Nicole Kosarek</td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC3H1.04c">SPAC3H1.04c</a></td>
<td>mdm31</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2010-10-21</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC22G7.04">SPAC22G7.04</a></td>
<td>ubp13</td>
<td>coordinates updated to join(732880..732897, 732942..733040, 733085..733251, 733369..733495, 733536..734113, 734151..735457, 735495..736465)</td>
<td>pers. comm. James Iben</td>
<td><span class="no-break">2010-10-20</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC27F1.09c">SPAC27F1.09c</a></td>
<td>prp10</td>
<td>added intron to match fully spliced</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9837997">PMID:9837997</a></td>
<td><span class="no-break">2010-09-27</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC26H5.02c">SPAC26H5.02c</a></td>
<td>nse2</td>
<td>second intron extended to 115 nt</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-08-20</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC5E4.10c">SPCC5E4.10c</a></td>
<td></td>
<td>coordinates updated from 651622..651905 , 651973..651994 to 651622..651905, 651973..652045, 652115..652225, 652280..652371, 652419..652455</td>
<td>pers. comm. K. Gould</td>
<td><span class="no-break">2010-03-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC458.04c">SPAC458.04c</a></td>
<td></td>
<td>coordinates updated from 4737605..4738147, 4738189..4738283, 4738330..4738714 to 4737605..4738147, 4738189..4738283, 4738330..4738740, 4738779..4738854</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2010-02-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC29B12.08">SPAC29B12.08</a></td>
<td></td>
<td>replaced exon 5428786..5428883 with 5429038.. 5429045</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2009-12-14</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPNCRNA.445">SPNCRNA.445</a></td>
<td>snoR61</td>
<td>updated to extend 3’ and added splice and identified as snR6</td>
<td>pers. comm. J. Matthews</td>
<td><span class="no-break">2008-08-08</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC2A9.07c">SPBC2A9.07c</a></td>
<td></td>
<td>removed intron</td>
<td>pers. comm. Cathrine Arnason Boe</td>
<td><span class="no-break">2009-09-10</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC800.02">SPBC800.02</a></td>
<td>whi5</td>
<td>increased intron by 42bp; old coordinates 253148..253231, 253310..253984, new coordinates 253148..253231, 253352..253984</td>
<td>pers. comm. Rob de Bruin</td>
<td><span class="no-break">2009-07-27</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC18H10.08c">SPBC18H10.08c</a></td>
<td>ubp4</td>
<td>missing 2 N-terminal exons; old - complement(1784219..1785535); new - complement(join(1784219..1785748, 1785937..1786080, 1786141..1786248))</td>
<td>pers. comm. I. Kouranti</td>
<td><span class="no-break">2009-09-10</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC800.02">SPBC800.02</a></td>
<td>whi5</td>
<td>increased intron by 42bp; old coordinates 253148..253231, 253310..253984, new coordinates 253148..253231, 253352..253984</td>
<td>pers. comm. Rob de Bruin</td>
<td><span class="no-break">2009-07-27</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC16E9.16c">SPBC16E9.16c</a></td>
<td>lsd90</td>
<td>gene structure revised to remove introns, genome sequence contains 2 frameshifts</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18079165">PMID:18079165</a></td>
<td><span class="no-break">2008-01-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBP23A10.07">SPBP23A10.07</a></td>
<td>rpa2</td>
<td>truncated N-term to agree with orthologs; 2007960 -&gt; 2008119</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2009-03-30</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC548.04">SPCC548.04</a></td>
<td></td>
<td>final exon 223687..223852 to 223687..223796, 223853.223860</td>
<td>pers. comm. Marc Feuermann, Uniprot (SIB)</td>
<td><span class="no-break">2009-02-23</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1442.04c">SPCC1442.04c</a></td>
<td></td>
<td>frameshifted; complement(1774200..1775429) to complement(join(1773925..1774239, 1774236..1775429))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1F8.07c">SPAC1F8.07c</a></td>
<td></td>
<td>frameshifted; complement(101760..103544) to complement(join(101836..101872, 101872..103544))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC22F3.11c">SPAC22F3.11c</a></td>
<td>snu23</td>
<td>frameshifted; join(682874..683034, 683102..683109, 683160..683446) to join(682874..682994, 682996..683109, 683160..683446)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC1A4.06c">SPBC1A4.06c</a></td>
<td></td>
<td>frameshifted; C term exon changed from 1987076..1987675 to 1987044….1987675</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC29A3.08">SPBC29A3.08</a></td>
<td></td>
<td>frameshifted; truncated to 2048336..2049892, 2049894..2049896</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC13E7.01">SPBC13E7.01</a></td>
<td></td>
<td>frameshifted</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC23G7.06c">SPBC23G7.06c</a></td>
<td></td>
<td>frameshifted; now a single exon</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC14C8.09c">SPBC14C8.09c</a></td>
<td></td>
<td>merged with SPBC14C8.08c; frameshifted</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
<td><span class="no-break">2009-02-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC725.12">SPBC725.12</a></td>
<td>nbl1</td>
<td>old join(1227762..1227876, 1227926..1228098); new join(1227762..1228094, 1228166..1228307)</td>
<td>pers. comm. Kathy Gould</td>
<td><span class="no-break">2009-01-30</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC31G5.12c">SPAC31G5.12c</a></td>
<td>maf1</td>
<td>old complement(join(3008184..3008795, 3008914..3008957, 3009044..3009089)); new complement(join(3008184..3008795, 3009160..3009258, 3009312..3009317))</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15590667">PMID:15590667</a></td>
<td><span class="no-break">2008-12-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB1E7.14">SPAPB1E7.14</a></td>
<td>iec5</td>
<td>old join(3322106..3322331, 3322379..3322512); new join(3321979..3322018, 3322071..3322331, 3322379..3322512)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19040720">PMID:19040720</a></td>
<td><span class="no-break">2008-12-18</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC962.05">SPCC962.05</a></td>
<td>ast1</td>
<td>intron position moved</td>
<td>pers comm. M. O’Connell</td>
<td><span class="no-break">2008-12-06</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC27D7.12c">SPAC27D7.12c</a></td>
<td>but1</td>
<td>extended N-terminal to use first MET as start</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=14623327">PMID:14623327</a></td>
<td><span class="no-break">2008-11-11</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC11B10.10c">SPBC11B10.10c</a></td>
<td>pht1</td>
<td>N terminal shortened to 2nd methionine; starting coordinate changed from 1503157 to 1503064</td>
<td>pers. comm. L Buchanan and F Stewart</td>
<td><span class="no-break">2008-09-05</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC959.05c">SPAC959.05c</a></td>
<td></td>
<td>merged with SPAC959.06c; splicing is tentative</td>
<td>pers. comm. Charley Chahwan</td>
<td><span class="no-break">2008-08-15</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC18E5.14c">SPBC18E5.14c</a></td>
<td></td>
<td>merged with SPBC18E5.09c</td>
<td>pers. comm. Charley Chahwan</td>
<td><span class="no-break">2008-07-31</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC651.06">SPBC651.06</a></td>
<td>mug166</td>
<td>merged with SPBC651.07</td>
<td>pers. comm. Charley Chahwan</td>
<td><span class="no-break">2008-07-26</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC31F10.08">SPBC31F10.08</a></td>
<td>mde2</td>
<td>old coordinates 3764820..3765449, new join(3764820..3765437, 3765491..3765709)</td>
<td>pers. comm. Charley Chahwan</td>
<td><span class="no-break">2008-07-18</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC2G2.09c">SPBC2G2.09c</a></td>
<td>crs1</td>
<td>added new C-terminal exon, deleted exon 3452822..3452877, extended intron 2 by 4 nucleotides</td>
<td>pers. comm. Charley Chahwan</td>
<td><span class="no-break">2008-07-09</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC338.08">SPCC338.08</a></td>
<td>ctp1</td>
<td>old coordinates complement(1358962..1359819), new complement(join(1358885..1359082, 1359121..1359819)); further altered to complement(join(1358891..1359082, 1359127..1359819)) in accordance with paper</td>
<td>pers. comm. Paul Russell; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18378696">PMID:18378696</a></td>
<td><span class="no-break">2008-01-11</span>;<span class="no-break">2008-06-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC962.05">SPCC962.05</a></td>
<td></td>
<td>old join(554521..554722, 554792..556200), new join(554521..554744, 554865..556200)</td>
<td>pers. comm. Matthew O’Connell</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC6F6.16c">SPAC6F6.16c</a></td>
<td></td>
<td>merged with SPAC6F6.18c and five introns added</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18535244">PMID:18535244</a></td>
<td><span class="no-break">2008-06-09</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC63.10c">SPCC63.10c</a></td>
<td></td>
<td>old complement(join(854651..855695, 855766..855800, 855839..855926, 855984..856213)), new complement(join(854651..855926, 855984..856213)); deleted 2nd and 3rd introns, were not required or supported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1827.02c">SPCC1827.02c</a></td>
<td></td>
<td>old complement(join(2377039..2378078, 2378197..2378221)), new complement(2377039..2378127); deleted first exon, unsupported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1450.14c">SPCC1450.14c</a></td>
<td></td>
<td>old complement(join(1759295..1760998, 1761117..1761128)), new complement(1759295..1760998); deleted first exon, unsupported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1393.06c">SPCC1393.06c</a></td>
<td></td>
<td>old complement(join(804627..805627, 805678..805800, 805907..806009)), new complement(join(804627..805627 , 805678..805800 , 805849..805921 , 805965..806009)); changed first intron and added a new 2nd exon, improved homology, also confirmed by transread data</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBP35G2.06c">SPBP35G2.06c</a></td>
<td></td>
<td>old complement(join(973049..974181, 974223..974577, 974617..975002, 975044..975137, 975172..976632)), new complement(join(973049..974181, 974223..974577, 974647..975002, 975044..975137, 975172..976632)); changed donor for intron 3</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC32H8.08c">SPBC32H8.08c</a></td>
<td></td>
<td>old complement(join(1465811..1466035, 1466138..1467057, 1467311..1467456, 1467514..1467539)), new complement(join(1465811..1466035 , 1466138..1467057 , 1467311..1467431)); removed 5’ intron/exon (unsupported) and trimmed to next met</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC21.03c">SPBC21.03c</a></td>
<td></td>
<td>old complement(join(3217985..3218424, 3218467..3218491, 3218693..3218947)), new complement(join(3217985..3218424, 3218462..3218491, 3218580..3218651, 3218704..3218947)); changed 2 internal introns, now agrees with transread data, but still potential to be incorrect</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC19G7.06">SPBC19G7.06</a></td>
<td></td>
<td>old join(2359429..2359479 , 2359547..2359693 , 2359845..2359928 , 2360001..2361092), new join(2359429..2359479, 2359547..2359693, 2359845..2359928, 2360064..2361092); altered 3rd intron branch acceptor to agree with published</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC16C6.03c">SPBC16C6.03c</a></td>
<td></td>
<td>old complement(join(4332356..4332454, 4332746..4333123)), new complement(4332698..4333123); final exon, unsupported removed</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC15D4.13c">SPBC15D4.13c</a></td>
<td></td>
<td>old complement(join(3032492..3032710, 3032761..3032822, 3032865..3033222)), new complement(join(3032492..3032710, 3032761..3032822, 3032889..3033222)); better donor for first intron</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB17E12.03">SPAPB17E12.03</a></td>
<td></td>
<td>old join(1271889..1271972, 1272015..1272386, 1272483..1272962), new join(1271889..1271972, 1272015..1272962); removed in-frame splice, unsupported, improved homology</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC9G1.13c">SPAC9G1.13c</a></td>
<td></td>
<td>old complement(join(1995208..1996148, 1996325..1996413, 1996478..1996683)), new complement(join(1995208..1996148, 1996191..1996268, 1996325..1996413, 1996478..1996683)); added a new 3rd exon, improved homology</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC8C9.19">SPAC8C9.19</a></td>
<td></td>
<td>old join(3660430..3660438, 3660491..3660556, 3660607..3660713, 3660784..3660886), new join(3660430..3660438, 3660491..3660556, 3660607..3660717); deleted final exon, unsupported, improved length and alignment to orthologs</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC6F12.08c">SPAC6F12.08c</a></td>
<td></td>
<td>old complement(join(1322439..1323955, 1323999..1324140, 1324195..1324234, 1324299..1324336)), new complement(join(1322439..1323955, 1323999..1324140, 1324195..1324252, 1324299..1324336)); improved branch acceptor for intron 1</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC589.02c">SPAC589.02c</a></td>
<td></td>
<td>old complement(join(3095075..3098590, 3098626..3098775, 3098878..3098883)), new complement(join(3095075..3098590, 3098626..3098781, 3098878..3098883)); improved branch/acceptor for 1st intron</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC3A12.05c">SPAC3A12.05c</a></td>
<td></td>
<td>old complement(join(1426213..1427148, 1427189..1429617, 1429709..1429836, 1429882..1429913)), new complement(join(1426213..1427148, 1427189..1429617, 1429709..1429830, 1429882..1429913)); improved distance beteen branch and acceptor</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC328.05">SPAC328.05</a></td>
<td></td>
<td>old join(3484695..3484698, 3484752..3485065, 3485241..3485342, 3485382..3485528, 3485582..3485807, 3485910..3485974, 3486056..3486502), new join(3484695..3484698, 3484752..3485065, 3485241..3485342, 3485382..3485528, 3485582..3485974, 3486056..3486502); removed in-frame splice (exon 5), unsupported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC2C4.06c">SPAC2C4.06c</a></td>
<td></td>
<td>old complement(join(4269367..4270554, 4270587..4270593, 4270638..4270674, 4270736..4270871)), new complement(join(4269367..4270554, 4270631..4270674, 4270736..4270871)); replaced 2 internal exons with a different singe exon</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC26H5.04">SPAC26H5.04</a></td>
<td></td>
<td>old join(4122149..4123354, 4123535..4124518), new 4122149..4124518; removed unsupported in-frame splice</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23H3.04">SPAC23H3.04</a></td>
<td></td>
<td>old join(2499035..2499164, 2499325..2499353, 2499443..2499824, 2499875..2500236), new join(2499035..2499164, 2499160..2499353, 2499443..2499824, 2499875..2500236); changed 5’ 1st and 2nd exon, improved homology</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1B3.10c">SPAC1B3.10c</a></td>
<td></td>
<td>old complement(join(4945933..4945984, 4946094..4948084)), new complement(4946090..4948084); removed 3’ exon, unsupported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1751.02c">SPAC1751.02c</a></td>
<td></td>
<td>old complement(join(385755..385887, 385938..386043, 386107..386179, 386296..386310)), new complement(join(385755..385887, 385938..386043, 386107..386149)); remove 5’ exon, trimmed to new Met, improved homology</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC11D3.07c">SPAC11D3.07c</a></td>
<td></td>
<td>old complement(join(118195..119827, 119930..120108)), new complement(118195..120108); removed in-frame splice, not required or supported</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1142.01">SPAC1142.01</a></td>
<td></td>
<td>old join(3625447..3625460, 3625555..3627544), new 3625574..3627544; removed 5’ exon and trimmed to new Met, improved homology</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1093.03">SPAC1093.03</a></td>
<td></td>
<td>old join(4613627..4613686, 4613723..4614081, 4614152..4614378, 4614438..4614954, 4614996..4616331), new join(4613627..4613686, 4613723..4614081, 4614164..4614378, 4614438..4614954, 4614996..4616331); improved distance between branch and acceptor, intron 2</td>
<td>Solexa intron data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC16D10.10">SPBC16D10.10</a></td>
<td></td>
<td>old join(3618117..3618179, 3618228..3618617, 3618658..3618702, 3618747..3618971, 3619077..3619321, 3619366..3619540), new join(3618117..3618179, 3618228..3618617, 3618658..3618702, 3618747..3618918, 3618956..3619071, 3619113..3619321, 3619366..3619540); changed 2 introns (existing 4 and 5) to create a new exon 5, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB18E9.01">SPAPB18E9.01</a></td>
<td></td>
<td>old join(3974310..3974485, 3974541..3974629, 3974674..3974775, 3974879..3975132, 3975193..3975429, 3975485..3975832, 3975882..3975983), new join(3974310..3974485, 3974541..3974629, 3974674..3974775, 3974879..3975132, 3975193..3975429, 3975485..3975832, 3975882..3975962, 3976014..3976079); added small 3’ exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC15A10.12c">SPAC15A10.12c</a></td>
<td></td>
<td>old complement(3709451..3709804), new complement(join(3709206..3709346, 3709499..3709804));</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1620.10">SPCC1620.10</a></td>
<td>cwf26</td>
<td>old 2163205..2164118, new join(2163205..2164118, 2164159..2164165); added 3’ exon encoding single methionine</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1620.09c">SPCC1620.09c</a></td>
<td>tfg1</td>
<td>old complement(join(2161527..2162363, 2162405..2162500, 2162584..2163123)), new complement(join(2161331..2161515, 2161565..2162363, 2162405..2162500, 2162584..2163123)); added 5’ exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1442.08c">SPCC1442.08c</a></td>
<td>cox17</td>
<td>old complement(join(1781505..1781563, 1781637..1781744, 1781830..1781914)), new complement(join(1781505..1781563, 1781637..1781744, 1781830..1781855, 1781920..1781935, 1781981..1782032)); added 2 small 5’ exons, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1020.11c">SPCC1020.11c</a></td>
<td></td>
<td>old join(757365..757424, 757506..757802), new join(757365..757424, 757506..757761, 757851..757861); added 3’intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC16C4.16c">SPCC16C4.16c</a></td>
<td></td>
<td>old complement(join(697862..698127 , 698180..698399)), new complement(join(697605..697752, 697875..698127, 698180..698399)); added 3’ intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC16C4.01">SPCC16C4.01</a></td>
<td>sif2</td>
<td>old join(658832..659139, 659223..659488, 659567..659628, 659709..660413), new join(658832..659139, 659223..659488, 659567..659628, 659709..660197, 660298..660322); added 3’ intron which changed frame of final exon, shorter product, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC736.12c">SPCC736.12c</a></td>
<td>mmi1</td>
<td>old complement(join(338026..338178, 338232..338347, 338395..339354, 339483..339567)), new complement(join(337829..337985, 338030..338178, 338232..338347, 338395..339354, 339483..339567); added 3’ exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBPB2B2.18">SPBPB2B2.18</a></td>
<td></td>
<td>old join(4502439..4502463, 4502712..4502857, 4503068..4503286, 4503527..4503664), new join(4502692..4502725, 4502787..4502857, 4503068..4503286, 4503527..4503664); changed 5’ intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC21C3.07c">SPBC21C3.07c</a></td>
<td></td>
<td>old complement(join(3807920..3808607, 3808665..3808795), new complement(join(3807714..3807783, 3807831..3807891, 3807946..3808607, 3808665..3808795)); added two 3’ exons</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC31F10.12">SPBC31F10.12</a></td>
<td></td>
<td>old join(3773353..3773525, 3773581..3773678, 3773795..3773999, 3774059..3774116, 3774192..3774212), new join(3773308..3773318, 3773373..3773525, 3773581..3773678, 3773795..3773999, 3774059..3774116, 3774192..3774212); added new 5’ intron and exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC13G1.04c">SPBC13G1.04c</a></td>
<td></td>
<td>old complement(3732941..3733762), new complement(join(3732806..3732930, 3732994..3733762)); added new 3’ intron and exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC16D10.02">SPBC16D10.02</a></td>
<td>trm11</td>
<td>old join(3588806..3589595, 3589639..3589803, 3589845..3590123, 3590163..3590287), new join(3588806..3589595, 3589639..3589803, 3589845..3590123, 3590163..3590278, 3590327..3590386); added new 3’ intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC2D10.16">SPBC2D10.16</a></td>
<td></td>
<td>old join(2996452..2996463, 2996509..2996904), new join(2996452..2996463, 2996509..2996687, 2996738..2996879); added new intron in second exon, which created a new 3rd exon, and identified homology with Cenp-S (was previously sequence orphan)</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC2D10.15c">SPBC2D10.15c</a></td>
<td></td>
<td>old complement(join(2995324..2995579, 2995619..2995923, 2995971..2996066)), new complement(join(2995324..2995497, 2995534..2995579, 2995619..2995923, 2995971..2996066)); altered first intron, added in-frame intron, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC2A9.07c">SPBC2A9.07c</a></td>
<td></td>
<td>old complement(2959078..2959902), new complement(join(2959078..2959129, 2959172..2959902)); added in-frame intron, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBP4H10.12">SPBP4H10.12</a></td>
<td></td>
<td>old join(2895855..2895863, 2896001..2896507), new join(2895855..2895863, 2896001..2896459, 2896516..2896587); added new 3’ intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC4.02c">SPBC4.02c</a></td>
<td></td>
<td>old complement(join(2369064..2369160, 2369246..2369416, 2369455..2371243, 2371351..2371451, 2371507..2371583)), new complement(join(2369064..2369190, 2369246..2369416, 2369455..2371243, 2371351..2371451, 2371507..2371583)); changed accceptor for final intron, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC8D2.17">SPBC8D2.17</a></td>
<td></td>
<td>old 1390183..1391238, new join(1390183..1391093, 1391147..1391222); added new 3’ intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC4.02c">SPBC4.02c</a></td>
<td></td>
<td>old complement(join(1185617..1186640, 1186738..1186846, 1187087..1187324)), new complement(join(1185617..1186640, 1186738..1186846, 1187087..1187162, 1187220..1187324)); added in-frame splice to first (existing) exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBPJ4664.05">SPBPJ4664.05</a></td>
<td></td>
<td>old 706175..706666, new join(705932..706077, 706141..706666); added new 5’ intron and exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC947.03c">SPBC947.03c</a></td>
<td></td>
<td>old 676629..676931, new join(676629..676908, 677057..677127); added 5’ inton/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC582.04c">SPBC582.04c</a></td>
<td></td>
<td>old complement(join(423269..423598, 423669..423738, 423782..423871, 423912..424373, 424424..424872, 424921..425160, 425223..425350, 425410..425446)), new complement(join(423269..423598, 423669..423738, 423782..423871, 423912..424373, 424424..424692, 424747..424872, 424921..425160, 425223..425350, 425410..425446)); added in-frame intron to (existing) exon 3</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC106.05c">SPBC106.05c</a></td>
<td>tim11</td>
<td>old complement(join(383527..383612, 383703..383874, 383980..384012)), new complement(join(383527..383612, 383703..383874, 383940..383960)); changed 5’ exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC22E12.10c">SPAC22E12.10c</a></td>
<td>etp1</td>
<td>old complement(5037208..5039103), new complement(join(5037208..5037446, 5037492..5039103)); added in-frame splice</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC890.07c">SPAC890.07c</a></td>
<td>rmt1</td>
<td>old complement(join(5016072..5016215, 5016262..5016596, 5016649..5016943, 5016994..5017059, 5017138..5017252, 5017421..5017433, 5017700..5017703, 5017782..5017829)), new complement(join(5016020..5016026, 5016076..5016215, 5016262..5016596, 5016649..5016943, 5016994..5017059, 5017138..5017252, 5017421..5017433, 5017700..5017703, 5017782..5017829); added small N terminal exon, which added a single conserved cysteine</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC4F10.08">SPAC4F10.08</a></td>
<td>mug126</td>
<td>old join(4844855..4845824, 4845863..4846203), new join(4844855..4845824, 4845863..4846063, 4846104..4846129); added new 3’ intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC144.17c">SPAC144.17c</a></td>
<td></td>
<td>old complement(join(4687483..4688462, 4688513..4688831)), new complement(join(4687388..4687477, 4687522..4688462, 4688513..4688831)); added 3’ intron and exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC30C2.05">SPAC30C2.05</a></td>
<td>erv14</td>
<td>old complement(join(4687483..4688462, 4688513..4688831)), new complement(join(4687388..4687477, 4687522..4688462, 4688513..4688831)); added new 5’ intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC30.03c">SPAC30.03c</a></td>
<td>tsn1</td>
<td>old join(4394707..4394741, 4394799..4394967, 4395010..4395092, 4395131..4395491, 4395530..4395592), new complement(join(4394707..4394741, 4394799..4394967, 4395010..4395092, 4395131..4395367, 4395416..4395491, 439 5530..4395592)); added in-frame intron to (previous) 2nd exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC2C4.11c">SPAC2C4.11c</a></td>
<td>rpb28</td>
<td>old complement(4278729..4279505), new complement(join(4278729..4279405, 4279457..4279505)); added small N term in-frame intron</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC2F3.18c">SPAC2F3.18c</a></td>
<td></td>
<td>old 3942398..3942730 new 3942299..3942364, 3942506..3942730; added small N term in-frame intron</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1F12.08">SPAC1F12.08</a></td>
<td></td>
<td>old join(3817464..3817803, 3817851..3817991, 3818035..3818543), new join(3817464..3817803, 3817851..3817991, 3818035..3818130, 3818170..3818543); added new intron, in-frame splice in final exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1556.03">SPAC1556.03</a></td>
<td>azr1</td>
<td>old join(3799244..3799680, 3799731..3800160), new join(3799244..3799680, 3799731..3800135, 3800184..3800241); added new 5’ intron and exon/improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1556.01c">SPAC1556.01c</a></td>
<td>rad50</td>
<td>old complement(join(3791855..3792229, 3792267..3795557, 3795601..3795684, 3795734..3795856)), new complement(join(3791831..3791851, 3791891..3792229, 3792267..3795557, 3795601..3795684, 3795734..3795856)); added small 3’ intron and exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC15A10.10">SPAC15A10.10</a></td>
<td></td>
<td>old join(3697508..3697752, 3697795..3698368, 3698421..3698527, 3698570..3699538, 3699582..3699837), new join(3697508..3697752, 3697795..3698368, 3698421..3698527, 3698570..3699310, 3699359..3699538, 3699582..3699837); added in-frame intron</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC16E8.12c">SPAC16E8.12c</a></td>
<td></td>
<td>old complement(join(3523136..3523644, 3524027..3524030)), new complement(join(3523136..3523644, 3523925..3523996, 3524044..3524098)); added 2 5’ exons</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB2B4.06">SPAPB2B4.06</a></td>
<td></td>
<td>old join(2722735..2723210, 2723273..2723300, 2723349..2723582), new join(2722735..2723210, 2723273..2723300, 2723349..2723559, 2723608..2723660); added new 3’ intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC823.09c">SPAC823.09c</a></td>
<td></td>
<td>old complement(join(2594922..2595693, 2595734..2595819)), new complement(join(2594777..2594913, 2594966..2595693, 2595734..2595819)); added new 3’ intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC823.04">SPAC823.04</a></td>
<td></td>
<td>old 2587729..2587897 , 2588085..2588620, new join(2587729..2587897, 2588106..2588620); altered to use valid acceptor</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAPB1A10.16">SPAPB1A10.16</a></td>
<td></td>
<td>old 1863972..1864388, new join(1863972..1864360, 1864415..1864457); added new 3’ intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC17A5.02c">SPAC17A5.02c</a></td>
<td>dbr1</td>
<td>old complement(join(1754109..1754310, 1754350..1755118, 1755164..1755285, 1755331..1755638)), new complement(join(1754033..1754145, 1754186..1754310, 1754350..1755118, 1755164..1755285, 1755331..1755638)); added new 5’intron/exon, now agrees with published date</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC9.06c">SPAC9.06c</a></td>
<td></td>
<td>old complement(join(1473692..1473885, 1473939..1474105, 1474153..1474212, 1474264..1474341, 1474460..1474539)), new complement(join(1473692..1473885, 1473939..1474105, 1474153..1474212, 1474264..1474341, 1474381..1474404, 1474460..1474539)); added new exon (2nd) improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB17E12.08">SPAPB17E12.08</a></td>
<td></td>
<td>old join(1279573..1279644, 1279820..1279869, 1279953..1280325), new join(1279573..1279779, 1279820..1279869, 1279953..1280325); changed first donor to make n-terminal exon longer, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC56F8.04c">SPAC56F8.04c</a></td>
<td>coq2</td>
<td>old complement(1133226..1134302), new complement(join(1133226..1134283, 1134332..1134356)); added 5’ intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1A6.03c">SPAC1A6.03c</a></td>
<td></td>
<td>old complement(1070139..1072013), new complement(join(1070061..1070138, 1070205..1072013)); new 3’ intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23C4.04c">SPAC23C4.04c</a></td>
<td></td>
<td>old 1036033..1036347, new join(1036033..1036342, 1036397..1036455); added intron</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC24H6.01c">SPAC24H6.01c</a></td>
<td></td>
<td>old join(490016..490262, 490338..490771, 490812..490931, 490979..491149, 491186..491980), new join(490016..490262, 490338..490771, 490812..490931, 490979..491149, 491186..491858, 491905..492011); added new 3’ intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC630.11">SPAC630.11</a></td>
<td>vps55</td>
<td>old join(366801..366817, 366873..366891, 366968..367040, 367123..367189, 367293..367348, 367395..367531), new join(366801..366817, 366873..366891, 366968..367040, 367105..367189, 367293..367348, 367395..367531); changed internal intron acceptor, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1639.01c">SPAC1639.01c</a></td>
<td></td>
<td>old complement(join(251726..252337, 252392..252752, 253590..253630)), new complement(join(251726..251745, 251863..252337, 252392..252752, 253590..253630)); added new 3’ intron/exon</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC13G6.04">SPAC13G6.04</a></td>
<td>tim8</td>
<td>old join(180423..180528, 180586..180776), new join(180423..180528, 180586..180721, 180890..180911); added new 3’ intron/exon, improved homology</td>
<td>Solexa transread data</td>
<td><span class="no-break">2008-05-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC16C6.02c">SPBC16C6.02c</a></td>
<td></td>
<td>complement(join(4322221..4322742, 4322970..4331555, 4331641..4331793, 4331856..4331968, 4332007..4332028)) was split into complement(join(4322829..4331555, 4331641..4331793, 4331856..4331968, 4332007..4332028)) and new gene SPBC16C6.14 - complement(4322221..4322622)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18367542">PMID:18367542</a></td>
<td><span class="no-break">2008-04-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC16E9.16c">SPBC16E9.16c</a></td>
<td></td>
<td>This CDS has 2 frameshifts in GeneDB at present; coordinates changed from 1947985..1949069 , 1949218..1950058 , 1950253..1950255 to 1947985..1948953, 1948955..1950050, 1950053..1950255</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18079165">PMID:18079165</a></td>
<td><span class="no-break">2007-01-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC16D10.10">SPBC16D10.10</a></td>
<td></td>
<td>exon 4 splice altered; coordinates changed from 3618117..3618179, 3618228..3618617, 3618658..3618702, 3618747..3618920, 3619065..3619321, 3619366..3619540 to 3618117..3618179, 3618228..3618617, 3618658..3618702, 3618747..3618971, 3619077..3619321, 3619366..3619540</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC19G7.06">SPBC19G7.06</a></td>
<td></td>
<td>coordinates changed from 2359429..2359479, 2359547..2359693, 2359845..2359928, 2360064..2361092 to 2359429..2359479, 2359547..2359693, 2359845..2359928, 2360001..2361092</td>
<td>Christopher J. McInerny</td>
<td><span class="no-break">2007-12-07</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC806.09c">SPAC806.09c</a></td>
<td></td>
<td>changed from 251726..252337, 252392..252752, 252945..252958 to 251726..252337, 252392..252752, 253590..253630</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-12-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC29E6.03c">SPAC29E6.03c</a></td>
<td></td>
<td>changed internal splice site, better intron, slightly improved homology; deleted final small exon, unsupported; coordinates changed from 4405871..4405889, 4405994..4407291, 4407447..4409096, 4409171..4409338 to 4405990..4407375, 4407447..4409096, 4409171..4409338</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC688.08">SPAC688.08</a></td>
<td></td>
<td>deleted final exon, not supported by Solexa data; coordinates changed from 3123163..3125057, 3125099..3126310, 3126356..3126631, 3126671..3126989 to 3123163..3125057, 3125099..3126310, 3126356..3126653</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC823.04">SPAC823.04</a></td>
<td></td>
<td>changed first intron acceptor, which gave a different n-term exon and improved homology; coordinates changed from 2587859..2587901, 2588085..2588620 to 2587729..2587897, 2588085..2588620</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC622.14">SPCC622.14</a></td>
<td></td>
<td>altered donor for intron 3, improved donor consensus; coordinates changed from 1427098..1427310, 1427632..1427709, 1427788..1428180, 1428277..1428522 to 1427098..1427310, 1427632..1427709, 1427788..1428216, 1428277..1428522</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1442.13c">SPCC1442.13c</a></td>
<td></td>
<td>deleted final exon, improved homology; coordinates changed from 1791196..1791285, 1791376..1791909 to 1791346..1791909</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1494.02">SPCC1494.02</a></td>
<td></td>
<td>2 C-terminal exons removed, not supported by homology; coordinates changed from 2317988..2318068, 2318200..2318244, 2318398..2318508, 2318579..2318676, 2318903..2318966 to 2318335..2318508, 2318579..2318676, 2318903..2318966</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAPB1A10.03">SPAPB1A10.03</a></td>
<td></td>
<td>SPAPB1A10.03 split to create SPAPB1A10.03 and SPAPB1A10.16; new coordinates 1863363..1863381, 1863434..1863762 and 1863972..1864388</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC11B10.03">SPBC11B10.03</a></td>
<td></td>
<td>removed N-teminal exon, poor donor, did not improve homology; coordinates changed from 1490376..1490423, 1490593..1491747 to 1490611..1491747</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC16H5.04">SPBC16H5.04</a></td>
<td></td>
<td>altered intron 2 donor and acceptor, improved homology; coordinates changed from 2293393..2293824, 2293938..2294039, 2294087..2294125 to 2293393..2293775, 2293877..2294039, 2294087..2294125</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC4C5.01">SPAC4C5.01</a></td>
<td></td>
<td>altered intron 5 to improve donor and acceptor, improves homology; coordinates changed from 1192545..1192592, 1192662..1192704, 1192779..1192841, 1192916..1193010, 1193062..1193175, 1193236..1193613 to 1192545..1192592, 1192662..1192704, 1192779..1192841, 1192916..1193010, 1193062..1193145, 1193197..1193613</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-03</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23D3.08">SPAC23D3.08</a></td>
<td>usp108</td>
<td>final exon deleted, not supported by Solexa intron data; coordinates changed from 4352514..4353359, 4353410..4353573, 4353623..4353761 to 4352514..4353359, 4353410..4353577</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC16A3.14">SPBC16A3.14</a></td>
<td></td>
<td>final exon deleted, not supported by Solexa intron data; coordinates changed from 4270907..4270939, 4271137..4271937 to 4271131..4271937</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC1604.17c">SPBC1604.17c</a></td>
<td></td>
<td>updated, better donor (GTAATA instead of GTTAGT); original intron prediction not supported by Solexa data; coordinates changed from 3900047..3900121, 3900166..3901146, 3901219..3901542 to 3900047..3900121, 3900166..3901170, 3901219..3901542</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC29A3.07c">SPBC29A3.07c</a></td>
<td></td>
<td>final exon deleted, not supported by Solexa intron data; homology improved; coordinates changed from 2050069..2050183, 2050304..2050358, 2050452..2050695 to 2050258..2050358, 2050452..2050695</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC21C3.07c">SPBC21C3.07c</a></td>
<td></td>
<td>removed final exon, not supported by Solexa transcript data; coordinates changed from 3807714..3807783, 3807963..3808607, 3808665..3808795 to 3807920..3808607, 3808665..3808795</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC21B10.02">SPBC21B10.02</a></td>
<td></td>
<td>intron unsupported by Solexa transcript data, removed final exon; coordinates changed from 1669624..1669695, 1669737..1670144 to 1669719..1670144</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC227.11c">SPAC227.11c</a></td>
<td></td>
<td>removed final exon, unsupported by Solexa data; coordinates changed from 514434..514489, 514557..515082, 515222..515482, 515534..515659 to 514552..515082, 515222..515482, 515534..515659</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC736.05">SPCC736.05</a></td>
<td>wtf7</td>
<td>removed final exon, unsupported by Solexa intron data; coordinates changed from 320617..320755, 320964..321335, 321387..321509, 321554..321582 to 320617..320755, 320964..321335, 321387..321523</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1450.09c">SPCC1450.09c</a></td>
<td></td>
<td>original intron absent from Solexa data; C terminal exon unsupported so exon deleted; coordinates changed from 1741451..1741481, 1741589..1743459 to 1741588..1743459</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC56F8.05c">SPAC56F8.05c</a></td>
<td></td>
<td>revised intron structure/original intron absent from Solexa data. N terminal exon unsupported and had no branch site so exon deleted; coordinates changed from 1134697..1135061, 1135195..1135402, 1135517..1135684, 1135729..1135776, 1135932..1135970, 1136111..1136170 to 1134697..1135061, 1135195..1135402, 1135517..1135684, 1135729..1135776, 1135932..1136003</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAPB1A10.15">SPAPB1A10.15</a></td>
<td></td>
<td>revised intron structure/original intron absent from Solexa data so small C-term exon deleted; coordinates changed from 1892554..1892640, 1892926..1893060, 1893098..1893504, 1893585..1893618 to 1892554..1892640, 1892926..1893060, 1893098..1893694</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB17E12.08">SPAPB17E12.08</a></td>
<td></td>
<td>revised intron structure/original intron absent from Solexa data so small N-term exon deleted; coordinates changed from 1279492..1279506, 1279582..1279644, 1279820..1279869, 1279953..1280325 to 1279573..1279644, 1279820..1279869, 1279953..1280325</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC17C9.08">SPAC17C9.08</a></td>
<td></td>
<td>revised intron structure/original intron absent from Solexa data and unsupported from mRNA data so final exon deleted; coordinates changed from 4484758..4484804, 4484843..4485274, 4485350..4485596, 4485713..4485994 to 4484835..4485274, 4485350..4485596, 4485713..4485994</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC16A10.06c">SPAC16A10.06c</a></td>
<td></td>
<td>revised intron structure/original intron absent from Solexa data; coordinates updated from 3089583..3089996, 3090112..3090285, 3090533..3090697 to 3089583..3090047, 3090112..3090285, 3090533..3090697</td>
<td>Solexa intron data</td>
<td><span class="no-break">2007-12-02</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC18.18c">SPCC18.18c</a></td>
<td></td>
<td>coordinates changed from 1988631..1990079 to 1988631..1990100, 1990630..1990722</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-11-29</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC631.02">SPAC631.02</a></td>
<td></td>
<td>coordinates updated from 2107940..2110123 to 2107470..2107515, 2107860..2110123</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-11-28</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC354.07c">SPBC354.07c</a></td>
<td></td>
<td>additional exon added; coordinates updated from 558335..559225, 559273..559474, 559942..560028, 560151..560170 to 558335..559225, 559273..559474, 559942..560028, 560151..560163, 560231..560246</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-11-28</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC6G9.16c">SPAC6G9.16c</a></td>
<td></td>
<td>coordinates changed from 3278951..3279349 to join(3278951..3279367, 3279420..3279539, 3279683..3279819, 3279865..3279985)</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-11-22</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBPB2B2.07c">SPBPB2B2.07c</a></td>
<td></td>
<td>coordinates updated from 4474434..4474931 to join(4474145..4474403, 4474438..4474931)</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-11-22</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1529.01">SPCC1529.01</a></td>
<td></td>
<td>new exon added; coordinates changed from join(237554..237923, 237975..238080, 238139..238533, 238588..239105) to join(237554..237923, 237975..238080, 238139..238533, 238588..239092, 239151..239250)</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-10-12</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1020.11c">SPCC1020.11c</a></td>
<td></td>
<td>coordinates updated from 757494..757802 to join(757365..757424, 757506..757802)</td>
<td>Solexa transcript data</td>
<td><span class="no-break">2007-10-12</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC22F8.07c">SPAC22F8.07c</a></td>
<td>rtf1</td>
<td>Two in-frame splices added</td>
<td><a href="http://www.ebi.ac.uk/ena/data/view/AJ627891">EMBL:AJ627891</a></td>
<td><span class="no-break">2007-08-28</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC9E9.02">SPAC9E9.02</a></td>
<td></td>
<td>altered to include stop codon</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2007-07-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC409.12">SPBC409.12</a></td>
<td>stn1</td>
<td>N-terminal extended by 2 exons</td>
<td>pers. comm. C. Chahwan</td>
<td><span class="no-break">2007-07-06</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23C4.04c">SPAC23C4.04c</a></td>
<td></td>
<td>altered to include stop codon and trimmed to methionine</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2007-06-19</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC29E6.04">SPAC29E6.04</a></td>
<td>nnf1</td>
<td>Sequence frameshifted at 4410192, new coordinates give an N terminal extension from 4410194..4410391 (NOTE: Still needs resequencing and underlying sequence changing)</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17035632">PMID:17035632</a></td>
<td><span class="no-break">2007-05-16</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC13G7.07">SPAC13G7.07</a></td>
<td></td>
<td>changed second exon 3’ boundary and added C-terminal exons</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17310250">PMID:17310250</a></td>
<td><span class="no-break">2007-03-23</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPNCRNA.82">SPNCRNA.82</a></td>
<td>mrp1</td>
<td>coordinates updated from 1234705..1234837 to 1234451..1234849, upon realisation that this corresponded to RNase MRP</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2007-02-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23D3.08">SPAC23D3.08</a></td>
<td>usp108</td>
<td>final exon removed</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2007-02-05</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBP22H7.09c">SPBP22H7.09c</a></td>
<td>mis15</td>
<td>updated sequence coordinates as per <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15369671">PMID:15369671</a></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15369671">PMID:15369671</a></td>
<td><span class="no-break">2007-01-05</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCC1223.10c">SPCC1223.10c</a></td>
<td>eaf1</td>
<td>Added exons as per <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17150956">PMID:17150956</a></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17150956">PMID:17150956</a></td>
<td><span class="no-break">2006-12-14</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAPB17E12.08">SPAPB17E12.08</a></td>
<td></td>
<td>coordinates updated from join(9968..9990, 10070..10442) to join(9609..9623, 9699..9761, 9937..9986, 10070..10442). Improved homology</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2006-12-12</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC30B4.08">SPBC30B4.08</a></td>
<td>eri1</td>
<td>gene structure updated as per <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16797182">PMID:16797182</a></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16797182">PMID:16797182</a></td>
<td><span class="no-break">2006-09-22</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC12D12.09">SPAC12D12.09</a></td>
<td>rev7</td>
<td>truncated penultimate exon to remove GIN</td>
<td>pers. comm. Nicole Kosarek</td>
<td><span class="no-break">2006-08-01</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC12D12.09">SPAC12D12.09</a></td>
<td>rev7</td>
<td>added new exon to insert KCIDIFGEF</td>
<td>pers. comm. Nicole Kosarek</td>
<td><span class="no-break">2006-08-01</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC4B3.05c">SPCC4B3.05c</a></td>
<td>hem12</td>
<td>Coordinates updated from join(28064..28097, 28155..28189, 28238..28816, 28866..29330) to join(28064..28097, 28155..28189, 28236..28260, 28308..28816, 28866..29330). Improves Homology</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2006-06-12</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPCPJ732.03">SPCPJ732.03</a></td>
<td>meu15</td>
<td>new coordinates as per publication</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=11376151">PMID:11376151</a></td>
<td><span class="no-break">2006-06-12</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC21E11.05c">SPAC21E11.05c</a></td>
<td>cyp8</td>
<td>Single base insertion allowed first 2 exons to be merged and extended (GAGT[G]GCA)</td>
<td>pers. comm. Trevor Pemberton (via Ivo Pedruzzi, UniProt)</td>
<td><span class="no-break">2006-05-22</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC6G9.13c">SPAC6G9.13c</a></td>
<td>bqt1</td>
<td>Additional C terminal exon</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16615890">PMID:16615890</a></td>
<td><span class="no-break">2006-05-03</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBCPT2R1.08c">SPBCPT2R1.08c</a></td>
<td>tlh2</td>
<td>truncated to 1919 a.a.; removed truncated version of a malate transporter fused to gene, not known whether these residues are included in translation</td>
<td>pers. comm. Liew Li Phing</td>
<td><span class="no-break">2006-04-13</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC1D4.11c">SPAC1D4.11c</a></td>
<td>lkh1</td>
<td>lengthened to 690 a.a.</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12565823">PMID:12565823</a></td>
<td><span class="no-break">2006-04-12</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC4G3.05c">SPCC4G3.05c</a></td>
<td>mus81</td>
<td>added N terminal exon</td>
<td>pers. comm. Fred Kippert</td>
<td><span class="no-break">2006-02-17</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC3B9.22c">SPBC3B9.22c</a></td>
<td>dad4</td>
<td>added new C-terminal exon based on Pfam protein family</td>
<td><a href="http://pfam.xfam.org/">Pfam</a></td>
<td><span class="no-break">2006-02-01</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC9.05">SPAC9.05</a></td>
<td>fml1</td>
<td>second intron acceptor extended by 15 bp</td>
<td>pers. comm. Anke Schürer</td>
<td><span class="no-break">2006-01-19</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC31G5.06">SPAC31G5.06</a></td>
<td></td>
<td>added N-teminal exon to extend prediction</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2005-11-01</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC17H9.20">SPAC17H9.20</a></td>
<td>psc3</td>
<td>added intron at 37250..37297</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16207082">PMID:16207082</a></td>
<td><span class="no-break">2005-10-10</span></td>
</tr>
<tr class="even">
<td>SPCC1223.15c</td>
<td>spc19</td>
<td>4 amino acid extension at N-terminus</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16079914">PMID:16079914</a></td>
<td><span class="no-break">2005-09-09</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC15D4.02">SPBC15D4.02</a></td>
<td></td>
<td>trimmed N terminal 28.10.03; looked overextended as large low complexity region in front in zinc finger which is usually N-term; alignment looks better</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2003-10-28</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC959.05c">SPAC959.05c</a></td>
<td></td>
<td>extended N-terminal to first methionine</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2003-09-14</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC29B12.08">SPAC29B12.08</a></td>
<td></td>
<td>new N-terminal exon</td>
<td>pers. comm. Aengus Stewart</td>
<td><span class="no-break">2003-09-09</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC2F3.13c">SPAC2F3.13c</a></td>
<td></td>
<td>annotated as SPAC2F3.13c putative tRNA-ribosyltransferase, but actually a gene merge. Split to create SPAC2F3.13c the tRNA-ribosyltransferase and new CDS SPAC2F3.17c was created from the C-terminal</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2003-06-30</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC19B12.06c">SPAC19B12.06c</a></td>
<td></td>
<td>added exon 18153..18218</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2003-04-22</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC31G5.07">SPAC31G5.07</a></td>
<td></td>
<td>N terminal extended to use longest ORF in the absence of homology</td>
<td>pers. comm. Henar Valdivieso Montero</td>
<td><span class="no-break">2003-01-08</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC29A4.14c">SPAC29A4.14c</a></td>
<td></td>
<td>N-terminal extended</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-09-11</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC22F3.03c">SPAC22F3.03c</a></td>
<td>rdh54</td>
<td>4th exon extended by altering acceptor in intron 3 cosmid coordinates 34092-33923</td>
<td>pers. comm. Mike Catlett</td>
<td><span class="no-break">2003-05-09</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC2F3.13c">SPAC2F3.13c</a></td>
<td></td>
<td>split to create SPAC2F3.13c and SPAC2F3.12c</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-06-30</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC2F3.12c">SPAC2F3.12c</a></td>
<td></td>
<td>created from the C-terminal of SPAC2F3.13c</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-06-30</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC1347.05c">SPBC1347.05c</a></td>
<td></td>
<td>new C-terminal exon</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-10-11</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC23C11.10">SPAC23C11.10</a></td>
<td></td>
<td>found 3 additional C-terminal exons by eye</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-06-18</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC1296.04">SPAC1296.04</a></td>
<td></td>
<td>has 2 additional N-terminal exons</td>
<td>pers. comm. Val Wood</td>
<td></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC1604.17c">SPBC1604.17c</a></td>
<td></td>
<td>previous coordinates (8945..8971, 9475..10455, 10528..10851)</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-04-29</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC1442.13c">SPCC1442.13c</a></td>
<td></td>
<td>C terminal exon added ; old coordinates (26947..27936); new exon extends G-patch domain</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-04-15</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBP23A10.04">SPBP23A10.04</a></td>
<td>apc2</td>
<td>coordinates updated additional in-frame splice at 5325, 5371</td>
<td>pers. comm. Hyun-Joo Yoon</td>
<td><span class="no-break">2003-02-04</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC23C4.08">SPAC23C4.08</a></td>
<td>rho3</td>
<td>N-terminal exon removed</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12653963">PMID:12653963</a></td>
<td><span class="no-break">2003-03-31</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC16A10.04">SPAC16A10.04</a></td>
<td>rho4</td>
<td>N-terminal exon replaced</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12653963">PMID:12653963</a></td>
<td><span class="no-break">2003-03-31</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPBC6B1.09c">SPBC6B1.09c</a></td>
<td>nbs1</td>
<td>additional intron 25664..25705, and exon extension 25974..25988</td>
<td>pers. comm. Charly Chahwan</td>
<td><span class="no-break">2003-03-07</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPBC6B1.09c">SPBC6B1.09c</a></td>
<td>nbs1</td>
<td>prediction extended by 4 N- terminal exons, this identified FHA domain</td>
<td>pers. comm. Val Wood</td>
<td><span class="no-break">2002-04-22</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPCC320.13c">SPCC320.13c</a></td>
<td>ark1</td>
<td>trimmed to 2nd met</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12676091">PMID:12676091</a></td>
<td><span class="no-break">2003-01-08</span></td>
</tr>
<tr class="even">
<td><a routerLink="/gene/SPAC11E3.03">SPAC11E3.03</a></td>
<td>pcs1</td>
<td>N terminal truncated, removed exon 9067..9103, truncated from 9149 to 9211 start</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12689592">PMID:12689592</a></td>
<td><span class="no-break">2003-04-13</span></td>
</tr>
<tr class="odd">
<td><a routerLink="/gene/SPAC19B12.05c">SPAC19B12.05c</a></td>
<td></td>
<td>additional exon</td>
<td><span class="no-break">pers. comm. Val Wood</span></td>
<td><span class="no-break">2003-04-22</span></td>
</tr>
</tbody>
</table>

  </div>
  <div *ngIf="pageName == 'index'">
<h2 id="genome-status">Genome Status</h2>
<table>
<tbody>
<tr class="odd">
<td><a routerLink="/status/statistics">Genome statistics</a></td>
<td>Latest information on the status of the genome</td>
</tr>
<tr class="even">
<td><a routerLink="/status/sequencing-status">Sequencing status</a></td>
<td>Contig size, gap status and progress, including <a routerLink="/status/centromeres">centromeres</a> and <a routerLink="/status/telomeres">telomeres</a></td>
</tr>
<tr class="odd">
<td><a routerLink="/status/sequencing-updates">Sequence updates</a></td>
<td>Changes to the genome sequence since July 2003</td>
</tr>
<tr class="even">
<td><a routerLink="/status/sequence-updates-pending">Sequence updates pending</a></td>
<td>Pending changes, mainly from the Broad Institute, some supported by data from other sources</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/gene-coordinate-changes">Gene coordinate changes</a></td>
<td>Changes to coordinates of individual genes since publication</td>
</tr>
<tr class="even">
<td><a routerLink="/status/new-and-removed-genes">New and removed genes</a></td>
<td>Genes identified or removed since publication</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/gene-characterisation">Gene characterisation</a></td>
<td>Current counts of protein coding gene status, as published in small scale experiments</td>
</tr>
<tr class="even">
<td><a routerLink="/status/gene-characterisation-statistics-history">Historical gene characterisation</a></td>
<td>Changes in gene characterisation statistics over time</td>
</tr>
<tr class="odd">
<td><a routerLink="/status/priority-unstudied-genes">Priority unstudied genes</a></td>
<td>Unstudied nuclear-encoded protein-coding genes conserved 1:1 in human</td>
</tr>
<tr class="even">
<td><a routerLink="/status/unmapped_genes">Unmapped genes</a></td>
<td>List of genes identified genetically but not cloned or physically mapped</td>
</tr>
</tbody>
</table>
<p>Note: Many older <em>S. pombe</em> sequence submissions to the DNA databases (International Nucleotide Sequence Database Collaboration databases, i.e. ENA, GenBank, DDBJ) contain one or more errors, and we do not have the resources to maintain past sequences or flag every error in PomBase.</p>

  </div>
  <div *ngIf="pageName == 'mating-type-region'">
<h2 id="mating-type-region">Mating type region</h2>
<p>The <em>S. pombe</em> mating type loci are located on Chromosome 2. The reference strain 972 h- encodes the M-specific mating genes <a href="https://www.pombase.org/jbrowse/?loc=II%3A2114008..2115135&amp;tracks=PomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features&amp;highlight=">II:2114008-2115135</a> at the expressed mat1 locus. The silent region mat3M is located at coordinates <a href="https://www.pombase.org/jbrowse/?loc=II%3A2129210..2137123&amp;tracks=DNA%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features&amp;highlight=">II:2129208-2137121</a>. Note that the silent mat2P region and cenH element are deleted in the reference strain.</p>
<p>A contig of the h90 configuration of the mat2P-mat3M region was created by Xavier Marsellach and Lorena Aguilar (Azorín lab) using available data and <em>S. pombe var. kambucha</em> as a scaffold. The contig can be viewed in the <a href="https://www.pombase.org/jbrowse/?loc=mating_type_region%3A2013..18114&amp;tracks=DNA%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features&amp;highlight=">genome browser</a>. Replacing the Chromosome 2 region spanning coordinates 2129208-2137121 with the separate contig sequence yields the Chromosome 2 contig of an h90 strain.</p>
<p>For a description of how the mating type specific genes are organized and annotated in PomBase, see this <a href="https://www.pombase.org/faq/how-are-mating-type-specific-gene-pages-organized">FAQ item</a>.</p>
<p>For a detailed description of the <em>S. pombe</em> mating type region, please see the <a href="http://www1.bio.ku.dk/english/research/fg/cellecyklus_genomintegritet/mating/">online tutorial</a> provided by the Nielsen lab (external link).</p>

  </div>
  <div *ngIf="pageName == 'new-and-removed-genes'">
<h3 id="new-and-removed-genes">New and removed genes</h3>
<h4 id="new-genes-since-publication">New genes since publication</h4>
<table>
 <thead>
  <tr>
   <th>Gene</th>
   <th>Description</th>
   <th>Date</th>
  </tr>
 </thead>
 <tbody>
 <tr>
  <td><a routerLink="/gene/SPBC1709.20" routerLinkActive="active" [attr.title]="'SPBC1709.20'"><span class="gene-link">SPBC1709.20</span></a></td>
  <td>RNase P and RNase MRP subunit; identified in targeted search for <em>S. cerevisiae</em> POP8 ortholog</td>
  <td>June 2002</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC7D4.15c" routerLinkActive="active" [attr.title]="'SPAC7D4.15c'"><span class="gene-link">SPAC7D4.15c</span></a></td>
  <td>Oligosaccharyltransferase subunit (predicted)</td>
  <td>June 2002</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC119.18" routerLinkActive="active" [attr.title]="'SPBC119.18'"><span class="gene-link">SPBC119.18</span></a></td>
  <td>conserved hypothetical protein</td>
  <td>June 2002</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1751.04" routerLinkActive="active" [attr.title]="'SPAC1751.04'"><span class="gene-link">SPAC1751.04</span></a></td>
  <td>hypothetical protein; sequence orphan</td>
  <td>June 2002</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC330.19c" routerLinkActive="active" [attr.title]="'SPCC330.19c'"><span class="gene-link">SPCC330.19c</span></a></td>
  <td>hypothetical protein; sequence orphan; confirmed by EST</td>
  <td>Sept 2002</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC959.10" routerLinkActive="active" [attr.title]="'SPAC959.10'"><span class="gene-link">SPAC959.10</span></a></td>
  <td>tRNA splicing endonuclease; similar to <em>S. cerevisiae</em> YMR059W</td>
  <td>Sept 2003</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC14C4.16" routerLinkActive="active" [attr.title]="'SPAC14C4.16'"><span class="gene-link">SPAC14C4.16</span></a></td>
  <td>hypothetical protein; sequence orphan; identified in targeted search for <em>S. cerevisiae</em> DAD3 ortholog</td>
  <td>Sept 2003</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC6B1.12c" routerLinkActive="active" [attr.title]="'SPBC6B1.12c'"><span class="gene-link">SPBC6B1.12c</span></a></td>
  <td>SAGA histone acetylase complex (predicted); involved in nuclear export (predicted); similar to <em>S. cerevisiae</em> YBR111W-A; identified in targeted search for <em>S. cerevisiae</em> SUS1 ortholog</td>
  <td>Feb 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC25H2.10c" routerLinkActive="active" [attr.title]="'SPBC25H2.10c'"><span class="gene-link">SPBC25H2.10c</span></a></td>
  <td>dihydropteroate synthase (predicted) (ISS); similar to <em>S. cerevisiae</em> YGL232W; updated from pseudogene to coding</td>
  <td>March 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC56F2.14" routerLinkActive="active" [attr.title]="'SPBC56F2.14'"><span class="gene-link">SPBC56F2.14</span></a></td>
  <td>ribosomal protein, mitochondrial L44 (predicted); similar to <em>S. cerevisiae</em> YMR225C; identified in targeted search for <em>S. cerevisiae</em> MRP144 ortholog</td>
  <td>March 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC106.05c" routerLinkActive="active" [attr.title]="'SPBC106.05c'"><span class="gene-link">SPBC106.05c</span></a></td>
  <td>proton-transporting ATP synthase complex, coupling factor F(o) (predicted); non-essential (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12618370">PMID:12618370</a>); similar to <em>S. cerevisiae</em> YDR322C-A; identified in targeted search for <em>S. cerevisiae</em> YDR322C-A ortholog</td>
  <td>March 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC19D5.11c" routerLinkActive="active" [attr.title]="'SPAC19D5.11c'"><span class="gene-link">SPAC19D5.11c</span></a></td>
  <td>similar to <em>S. cerevisiae</em> CTF8; pers. comm. Stuart MacNeil</td>
  <td>September 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC19D5.10c" routerLinkActive="active" [attr.title]="'SPAC19D5.10c'"><span class="gene-link">SPAC19D5.10c</span></a></td>
  <td>sequence orphan</td>
  <td>September 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC3A12.19" routerLinkActive="active" [attr.title]="'SPAC3A12.19'"><span class="gene-link">SPAC3A12.19</span></a></td>
  <td>mitochondrial ribosomal protein; identified in targeted search for <em>S. cerevisiae</em> YBR282W ortholog</td>
  <td>September 2004</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC57A10.14" routerLinkActive="active" [attr.title]="'SPAC57A10.14'"><span class="gene-link">SPAC57A10.14</span></a></td>
  <td>SAGA complex subunit (predicted); identified in targeted search for <em>S. cerevisiae</em> SGF11 ortholog</td>
  <td>January 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC8C9.19" routerLinkActive="active" [attr.title]="'SPAC8C9.19'"><span class="gene-link">SPAC8C9.19</span></a></td>
  <td>conserved protein; similar to <em>D. hansenii</em> DEHA0C10780g</td>
  <td>May 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC8C9.20" routerLinkActive="active" [attr.title]="'SPAC8C9.20'"><span class="gene-link">SPAC8C9.20</span></a></td>
  <td>sequence orphan</td>
  <td>May 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1223.15c" routerLinkActive="active" [attr.title]="'SPCC1223.15c'"><span class="gene-link">SPCC1223.15c</span></a></td>
  <td>DASH complex subunit Spc19; pers. comm. Xiangwei He</td>
  <td>May 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC3D6.16" routerLinkActive="active" [attr.title]="'SPBC3D6.16'"><span class="gene-link">SPBC3D6.16</span></a></td>
  <td>sequence orphan</td>
  <td>Added May 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC3B9.22c" routerLinkActive="active" [attr.title]="'SPBC3B9.22c'"><span class="gene-link">SPBC3B9.22c</span></a></td>
  <td>DASH complex subunit Dad4 (predicted); pers. comm. Xiangwei He</td>
  <td>May 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC16E9.16c" routerLinkActive="active" [attr.title]="'SPBC16E9.16c'"><span class="gene-link">SPBC16E9.16c</span></a></td>
  <td>sequence orphan; previously annotated as pseudogene; two peptides in mass spec analysis (pers. comm. Dieter Wolf)</td>
  <td>November 2005</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC27E2.12" routerLinkActive="active" [attr.title]="'SPAC27E2.12'"><span class="gene-link">SPAC27E2.12</span></a></td>
  <td>dubious</td>
  <td>Added February 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC1347.14c" routerLinkActive="active" [attr.title]="'SPBC1347.14c'"><span class="gene-link">SPBC1347.14c</span></a></td>
  <td>dubious</td>
  <td>February 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC16C4.21" routerLinkActive="active" [attr.title]="'SPAC16C4.21'"><span class="gene-link">SPAC16C4.21</span></a></td>
  <td>dubious</td>
  <td>June 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC6B12.18" routerLinkActive="active" [attr.title]="'SPAC6B12.18'"><span class="gene-link">SPAC6B12.18</span></a></td>
  <td>sequence orphan</td>
  <td>July 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC222.17c" routerLinkActive="active" [attr.title]="'SPAC222.17c'"><span class="gene-link">SPAC222.17c</span></a></td>
  <td>dubious</td>
  <td>July 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC27E2.13" routerLinkActive="active" [attr.title]="'SPAC27E2.13'"><span class="gene-link">SPAC27E2.13</span></a></td>
  <td>dubious</td>
  <td>July 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC417.15" routerLinkActive="active" [attr.title]="'SPCC417.15'"><span class="gene-link">SPCC417.15</span></a></td>
  <td>dubious</td>
  <td>July 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1250.07" routerLinkActive="active" [attr.title]="'SPAC1250.07'"><span class="gene-link">SPAC1250.07</span></a></td>
  <td>dubious</td>
  <td>July 2006</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1250.07" routerLinkActive="active" [attr.title]="'SPAC1250.07'"><span class="gene-link">SPAC1250.07</span></a></td>
  <td>TFIIIC subunit Tfc7; from PMID:17409385</td>
  <td>April 2007</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1393.14" routerLinkActive="active" [attr.title]="'SPCC1393.14'"><span class="gene-link">SPCC1393.14</span></a></td>
  <td>ten1; nuclear telomere cap complex subunit; pers. comm. Paul Russell</td>
  <td>July 2007</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAPB1A10.16" routerLinkActive="active" [attr.title]="'SPAPB1A10.16'"><span class="gene-link">SPAPB1A10.16</span></a></td>
  <td>conserved fungal protein; created from the C-term exon of SPAPB1A10.03</td>
  <td>December 2007</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC569.09" routerLinkActive="active" [attr.title]="'SPCC569.09'"><span class="gene-link">SPCC569.09</span></a></td>
  <td>sequence orphan; longest reading frome in solexa transcript SPNG2623</td>
  <td>December 2007</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.214" routerLinkActive="active" [attr.title]="'SPNCRNA.214'"><span class="gene-link">SPNCRNA.214</span></a></td>
  <td>telomerase RNA (predicted)</td>
  <td>January 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1183.12" routerLinkActive="active" [attr.title]="'SPCC1183.12'"><span class="gene-link">SPCC1183.12</span></a></td>
  <td>previously SPNCRNA.546; splice site not spotted because it was outside the coordinate range of the feature; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18367542">PMID:18367542</a></td>
  <td>April 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC16C6.14" routerLinkActive="active" [attr.title]="'SPBC16C6.14'"><span class="gene-link">SPBC16C6.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18367542">PMID:18367542</a></td>
  <td>April 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC70.12c" routerLinkActive="active" [attr.title]="'SPCC70.12c'"><span class="gene-link">SPCC70.12c</span></a></td>
  <td>extender of the chronological lifespan protein Ecl1; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18422613">PMID:18422613</a></td>
  <td>April 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1442.19" routerLinkActive="active" [attr.title]="'SPCC1442.19'"><span class="gene-link">SPCC1442.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC222.17" routerLinkActive="active" [attr.title]="'SPAC222.17'"><span class="gene-link">SPAC222.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC569.09" routerLinkActive="active" [attr.title]="'SPCC569.09'"><span class="gene-link">SPCC569.09</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAPB1E7.14" routerLinkActive="active" [attr.title]="'SPAPB1E7.14'"><span class="gene-link">SPAPB1E7.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC688.16" routerLinkActive="active" [attr.title]="'SPAC688.16'"><span class="gene-link">SPAC688.16</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAP4C9.02" routerLinkActive="active" [attr.title]="'SPAP4C9.02'"><span class="gene-link">SPAP4C9.02</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC16C4.22" routerLinkActive="active" [attr.title]="'SPCC16C4.22'"><span class="gene-link">SPCC16C4.22</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC23D3.16" routerLinkActive="active" [attr.title]="'SPAC23D3.16'"><span class="gene-link">SPAC23D3.16</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1183.12" routerLinkActive="active" [attr.title]="'SPCC1183.12'"><span class="gene-link">SPCC1183.12</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC25H1.10C" routerLinkActive="active" [attr.title]="'SPAC25H1.10C'"><span class="gene-link">SPAC25H1.10C</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC1604.25" routerLinkActive="active" [attr.title]="'SPBC1604.25'"><span class="gene-link">SPBC1604.25</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC23D3.17" routerLinkActive="active" [attr.title]="'SPAC23D3.17'"><span class="gene-link">SPAC23D3.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC4G9.22" routerLinkActive="active" [attr.title]="'SPAC4G9.22'"><span class="gene-link">SPAC4G9.22</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC757.15" routerLinkActive="active" [attr.title]="'SPCC757.15'"><span class="gene-link">SPCC757.15</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC227.19C" routerLinkActive="active" [attr.title]="'SPAC227.19C'"><span class="gene-link">SPAC227.19C</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC167.09" routerLinkActive="active" [attr.title]="'SPAC167.09'"><span class="gene-link">SPAC167.09</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC16C6.14" routerLinkActive="active" [attr.title]="'SPBC16C6.14'"><span class="gene-link">SPBC16C6.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC20G4.09" routerLinkActive="active" [attr.title]="'SPAC20G4.09'"><span class="gene-link">SPAC20G4.09</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC222.18" routerLinkActive="active" [attr.title]="'SPAC222.18'"><span class="gene-link">SPAC222.18</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC6F6.19" routerLinkActive="active" [attr.title]="'SPAC6F6.19'"><span class="gene-link">SPAC6F6.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC22F3.15" routerLinkActive="active" [attr.title]="'SPAC22F3.15'"><span class="gene-link">SPAC22F3.15</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC27E2.14" routerLinkActive="active" [attr.title]="'SPAC27E2.14'"><span class="gene-link">SPAC27E2.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC23H4.21" routerLinkActive="active" [attr.title]="'SPAC23H4.21'"><span class="gene-link">SPAC23H4.21</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC57A7.15C" routerLinkActive="active" [attr.title]="'SPAC57A7.15C'"><span class="gene-link">SPAC57A7.15C</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC12G12.17" routerLinkActive="active" [attr.title]="'SPAC12G12.17'"><span class="gene-link">SPAC12G12.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC29A10.17" routerLinkActive="active" [attr.title]="'SPBC29A10.17'"><span class="gene-link">SPBC29A10.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1556.06" routerLinkActive="active" [attr.title]="'SPAC1556.06'"><span class="gene-link">SPAC1556.06</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC26H8.16" routerLinkActive="active" [attr.title]="'SPBC26H8.16'"><span class="gene-link">SPBC26H8.16</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC70.12C" routerLinkActive="active" [attr.title]="'SPCC70.12C'"><span class="gene-link">SPCC70.12C</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18488015">PMID:18488015</a>; Solexa prediction</td>
  <td>May 2008</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBP35G2.16c" routerLinkActive="active" [attr.title]="'SPBP35G2.16c'"><span class="gene-link">SPBP35G2.16c</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19352039">PMID:19352039</a></td>
  <td>April 2009</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC8E4.12c" routerLinkActive="active" [attr.title]="'SPBC8E4.12c'"><span class="gene-link">SPBC8E4.12c</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19352039">PMID:19352039</a></td>
  <td>April 2009</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC1235.16" routerLinkActive="active" [attr.title]="'SPCC1235.16'"><span class="gene-link">SPCC1235.16</span></a></td>
  <td>pers. comm. Juan Mata</td>
  <td>April 2009</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC18.19C" routerLinkActive="active" [attr.title]="'SPCC18.19C'"><span class="gene-link">SPCC18.19C</span></a></td>
  <td>pers. comm. Val Wood; note first 2 exons may be incorrect, homology is within 3rd exon</td>
  <td>February 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC306.10" routerLinkActive="active" [attr.title]="'SPCC306.10'"><span class="gene-link">SPCC306.10</span></a></td>
  <td>pers. comm. Val Wood; pseudo-&gt; coding</td>
  <td>March 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.497" routerLinkActive="active" [attr.title]="'SPNCRNA.497'"><span class="gene-link">SPNCRNA.497</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.498" routerLinkActive="active" [attr.title]="'SPNCRNA.498'"><span class="gene-link">SPNCRNA.498</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.499" routerLinkActive="active" [attr.title]="'SPNCRNA.499'"><span class="gene-link">SPNCRNA.499</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.507" routerLinkActive="active" [attr.title]="'SPNCRNA.507'"><span class="gene-link">SPNCRNA.507</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.510" routerLinkActive="active" [attr.title]="'SPNCRNA.510'"><span class="gene-link">SPNCRNA.510</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.514" routerLinkActive="active" [attr.title]="'SPNCRNA.514'"><span class="gene-link">SPNCRNA.514</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.515" routerLinkActive="active" [attr.title]="'SPNCRNA.515'"><span class="gene-link">SPNCRNA.515</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.523" routerLinkActive="active" [attr.title]="'SPNCRNA.523'"><span class="gene-link">SPNCRNA.523</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.530" routerLinkActive="active" [attr.title]="'SPNCRNA.530'"><span class="gene-link">SPNCRNA.530</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18641648">PMID:18641648</a></td>
  <td>May 2010</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC222.19" routerLinkActive="active" [attr.title]="'SPAC222.19'"><span class="gene-link">SPAC222.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC17G8.15" routerLinkActive="active" [attr.title]="'SPAC17G8.15'"><span class="gene-link">SPAC17G8.15</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1805.18" routerLinkActive="active" [attr.title]="'SPAC1805.18'"><span class="gene-link">SPAC1805.18</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1486.11" routerLinkActive="active" [attr.title]="'SPAC1486.11'"><span class="gene-link">SPAC1486.11</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC3H5.13" routerLinkActive="active" [attr.title]="'SPAC3H5.13'"><span class="gene-link">SPAC3H5.13</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC15A10.17" routerLinkActive="active" [attr.title]="'SPAC15A10.17'"><span class="gene-link">SPAC15A10.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC4F10.22" routerLinkActive="active" [attr.title]="'SPAC4F10.22'"><span class="gene-link">SPAC4F10.22</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1B3.21" routerLinkActive="active" [attr.title]="'SPAC1B3.21'"><span class="gene-link">SPAC1B3.21</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPRRNA.56" routerLinkActive="active" [attr.title]="'SPRRNA.56'"><span class="gene-link">SPRRNA.56</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC9G1.15c" routerLinkActive="active" [attr.title]="'SPAC9G1.15c'"><span class="gene-link">SPAC9G1.15c</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC13F5.08" routerLinkActive="active" [attr.title]="'SPAC13F5.08'"><span class="gene-link">SPAC13F5.08</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC6B12.19" routerLinkActive="active" [attr.title]="'SPAC6B12.19'"><span class="gene-link">SPAC6B12.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC3G9.17" routerLinkActive="active" [attr.title]="'SPAC3G9.17'"><span class="gene-link">SPAC3G9.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC16E8.18c" routerLinkActive="active" [attr.title]="'SPAC16E8.18c'"><span class="gene-link">SPAC16E8.18c</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC926.10" routerLinkActive="active" [attr.title]="'SPAC926.10'"><span class="gene-link">SPAC926.10</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC19G12.17" routerLinkActive="active" [attr.title]="'SPAC19G12.17'"><span class="gene-link">SPAC19G12.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC23A1.20" routerLinkActive="active" [attr.title]="'SPAC23A1.20'"><span class="gene-link">SPAC23A1.20</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC1F7.14" routerLinkActive="active" [attr.title]="'SPAC1F7.14'"><span class="gene-link">SPAC1F7.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC4D7.15" routerLinkActive="active" [attr.title]="'SPAC4D7.15'"><span class="gene-link">SPAC4D7.15</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC4D7.14" routerLinkActive="active" [attr.title]="'SPAC4D7.14'"><span class="gene-link">SPAC4D7.14</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC839.20" routerLinkActive="active" [attr.title]="'SPBC839.20'"><span class="gene-link">SPBC839.20</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC409.23" routerLinkActive="active" [attr.title]="'SPBC409.23'"><span class="gene-link">SPBC409.23</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC8D2.23" routerLinkActive="active" [attr.title]="'SPBC8D2.23'"><span class="gene-link">SPBC8D2.23</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC3H7.18" routerLinkActive="active" [attr.title]="'SPBC3H7.18'"><span class="gene-link">SPBC3H7.18</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC1711.18" routerLinkActive="active" [attr.title]="'SPBC1711.18'"><span class="gene-link">SPBC1711.18</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC24C6.13" routerLinkActive="active" [attr.title]="'SPBC24C6.13'"><span class="gene-link">SPBC24C6.13</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC32F12.16" routerLinkActive="active" [attr.title]="'SPBC32F12.16'"><span class="gene-link">SPBC32F12.16</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC30D10.21" routerLinkActive="active" [attr.title]="'SPBC30D10.21'"><span class="gene-link">SPBC30D10.21</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC887.22" routerLinkActive="active" [attr.title]="'SPBC887.22'"><span class="gene-link">SPBC887.22</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC839.19" routerLinkActive="active" [attr.title]="'SPBC839.19'"><span class="gene-link">SPBC839.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC530.16" routerLinkActive="active" [attr.title]="'SPBC530.16'"><span class="gene-link">SPBC530.16</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC14C8.19" routerLinkActive="active" [attr.title]="'SPBC14C8.19'"><span class="gene-link">SPBC14C8.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC17D1.17" routerLinkActive="active" [attr.title]="'SPBC17D1.17'"><span class="gene-link">SPBC17D1.17</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC1105.19" routerLinkActive="active" [attr.title]="'SPBC1105.19'"><span class="gene-link">SPBC1105.19</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC56F2.15" routerLinkActive="active" [attr.title]="'SPBC56F2.15'"><span class="gene-link">SPBC56F2.15</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCP20C8.04" routerLinkActive="active" [attr.title]="'SPCP20C8.04'"><span class="gene-link">SPCP20C8.04</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC330.20" routerLinkActive="active" [attr.title]="'SPCC330.20'"><span class="gene-link">SPCC330.20</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC63.09" routerLinkActive="active" [attr.title]="'SPCC63.09'"><span class="gene-link">SPCC63.09</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC4B3.20" routerLinkActive="active" [attr.title]="'SPCC4B3.20'"><span class="gene-link">SPCC4B3.20</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPCC330.21" routerLinkActive="active" [attr.title]="'SPCC330.21'"><span class="gene-link">SPCC330.21</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21270388">PMID:21270388</a></td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC11D3.11c" routerLinkActive="active" [attr.title]="'SPAC11D3.11c'"><span class="gene-link">SPAC11D3.11c</span></a></td>
  <td>pseudo to coding</td>
  <td>Jan 2011</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPBC713.14c" routerLinkActive="active" [attr.title]="'SPBC713.14c'"><span class="gene-link">SPBC713.14c</span></a></td>
  <td>identified by ribosome profiling (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=24929437">PMID:24929437</a>)</td>
  <td>July 2014</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC823.17" routerLinkActive="active" [attr.title]="'SPAC823.17'"><span class="gene-link">SPAC823.17</span></a></td>
  <td>identified by ribosome profiling (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=24929437">PMID:24929437</a>)</td>
  <td>July 2014</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC110.06" routerLinkActive="active" [attr.title]="'SPAC110.06'"><span class="gene-link">SPAC110.06</span></a></td>
  <td>identified by ribosome profiling (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=24929437">PMID:24929437</a>)</td>
  <td>July 2014</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAPB1A11.06" routerLinkActive="active" [attr.title]="'SPAPB1A11.06'"><span class="gene-link">SPAPB1A11.06</span></a></td>
  <td>previously SPNCRNA.31 (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26494834">PMID:26494834</a>)</td>
  <td>November 2015</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC110.06" routerLinkActive="active" [attr.title]="'SPAC110.06'"><span class="gene-link">SPAC110.06</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26494834">PMID:26494834</a></td>
  <td>November 2015</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC29A4.23" routerLinkActive="active" [attr.title]="'SPAC29A4.23'"><span class="gene-link">SPAC29A4.23</span></a></td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26494834">PMID:26494834</a></td>
  <td>November 2015</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPAC110.06" routerLinkActive="active" [attr.title]="'SPAC110.06'"><span class="gene-link">SPAC110.06</span></a></td>
  <td>F1F0 ATPase synthase peripheral stalk assembly factor Ina17 (predicted) (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26494834">PMID:26494834</a>)</td>
  <td>November 2015</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1688" routerLinkActive="active" [attr.title]="'SPNCRNA.1688'"><span class="gene-link">SPNCRNA.1688</span></a></td>
  <td>intergenic RNA nc-tgp1 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=25428589">PMID:25428589</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1701" routerLinkActive="active" [attr.title]="'SPNCRNA.1701'"><span class="gene-link">SPNCRNA.1701</span></a></td>
  <td>name prl101; synonym prl1 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1702" routerLinkActive="active" [attr.title]="'SPNCRNA.1702'"><span class="gene-link">SPNCRNA.1702</span></a></td>
  <td>name prl104; synonym prl4 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1703" routerLinkActive="active" [attr.title]="'SPNCRNA.1703'"><span class="gene-link">SPNCRNA.1703</span></a></td>
  <td>name prl105; synonym prl5 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1704" routerLinkActive="active" [attr.title]="'SPNCRNA.1704'"><span class="gene-link">SPNCRNA.1704</span></a></td>
  <td>name prl107; synonym prl7 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1705" routerLinkActive="active" [attr.title]="'SPNCRNA.1705'"><span class="gene-link">SPNCRNA.1705</span></a></td>
  <td>name prl108; synonym prl8 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1706" routerLinkActive="active" [attr.title]="'SPNCRNA.1706'"><span class="gene-link">SPNCRNA.1706</span></a></td>
  <td>name prl106; synonym prl6 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1707" routerLinkActive="active" [attr.title]="'SPNCRNA.1707'"><span class="gene-link">SPNCRNA.1707</span></a></td>
  <td>name prl102; synonym prl2 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPNCRNA.1708" routerLinkActive="active" [attr.title]="'SPNCRNA.1708'"><span class="gene-link">SPNCRNA.1708</span></a></td>
  <td>name prl103; synonym prl3 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=28031482">PMID:28031482</a>)</td>
  <td>January 2017</td>
 </tr>
 <tr>
  <td><a routerLink="/gene/SPMITNCRNA.01" routerLinkActive="active" [attr.title]="'SPNCRNA.1708'"><span class="gene-link">SPNCRNA.1708</span></a></td>
  <td>name rnpB (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=29954949">PMID:29954949</a>)</td>
  <td>August 2018</td>
 </tr>
   <tr>
  <td><a routerLink="/gene/SPNCRNA.9001" routerLinkActive="active" [attr.title]="'SPNCRNA.9001'"><span class="gene-link">SPNCRNA.9001</span></a></td>
  <td>SPBC8E4.02c replaced by SPNCRNA.9001 (prt2) (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=29414789">PMID:29414789</a>)</td>
  <td>October 2018</td>
 </tr>
 </tbody>
</table>
<h4 id="genes-removed-since-publication">Genes removed since publication</h4>
<table>
 <tbody>
 <tr>
  <th>Gene</th>
  <th>Comment</th>
  <th>Date</th>
 </tr>
 <tr>
  <td>SPBC19G7.12c</td>
  <td>removed because sequence similarity supports SPBC19G7.11c</td>
  <td><span class="no-break">2004-09-14</span></td>
 </tr>
 <tr>
  <td>SPAC212.10</td>
  <td>removed as part of UTR of SPAC212.11</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC3F10.14</td>
  <td>reverse strand of SPAC3F10.15c</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC428.09c</td>
  <td>presumed non-coding as no methionine or consensus splice sites</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC222.17c</td>
  <td>presumed non-coding; was a very small ORF in ncRNA prl50</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPNCRNA.135</td>
  <td>Initially annotated as a non-coding RNA but subsequently determined to be the 5' UTR of SPBC16D10.06 (pers. comm S. Whitehall)</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBPB21E7.10</td>
  <td>partial ORF of 35 aa, no reason to believe this is coding</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC4G8.15c</td>
  <td>dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAPB18E9.03c</td>
  <td>dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC18H10.21c</td>
  <td>alias SPBC9B6.01c; dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC9B6.01c</td>
  <td>alias of SPBC18H10.21c; dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPCC2H8.03</td>
  <td>dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC22E12.12</td>
  <td>dubious; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPNCRNA.85</td>
  <td>replaced by protein coding gene SPAC227.19c</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPNCRNA.48</td>
  <td>replaced by protein coding gene SPAC2F3.18</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPNCRNA.97</td>
  <td>replaced by short protein coding gene SPAC27E2.14</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC8E11.09c</td>
  <td>previously annotated as dubious but no evidence for splicing, and non-canonical splice sites; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC22E12.15</td>
  <td>previously annotated as dubious but dubious splice prediction; unsupported by Solexa transcript data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC1002.21</td>
  <td>was annotated as dubious but based on position in uracil regulatable operon, as this CDS does not appear to be transcribed. t is possible that the localization data is an artifact of the combination of overexpression and hydrophobicity. The protein sequence is also a strange composition, and there is no ortholog in octosporus or japonicus genomes. On balance, taking into account all of the available features and contextual information, it is unlikely that this feature codes for a bona fide protein.</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC8C9.20</td>
  <td>non consensus branch sites, and no coverage with transcriptome data</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC22E12.12</td>
  <td>unsupported by Solexa transcript data and largely overlapping with 3' UTR of rpl24-3</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPCC1183.12</td>
  <td>previously SPNCRNA.546; splice site not spotted because it was outside the coordinate range of the feature; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18367542">PMID:18367542</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC16C6.14</td>
  <td>unsupported by Solexa transcript data and largely overlapping with 3' UTR of rpl24-3</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC6F6.18c</td>
  <td>merged with SPAC6F6.16c; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18535244">PMID:18535244</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC651.07</td>
  <td>merged with SPBC651.06; pers. comm. Charly Chahwan</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPNCRNA.109</td>
  <td>part of protein coding gene merge</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPBC18E5.09c</td>
  <td>merged with SPBC18E5.14c; pers. comm. Charly Chahwan</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPAC959.06c</td>
  <td>merged with SPAC959.05c; pers. comm. Charly Chahwan</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SPCC794.14c</td>
  <td>dubious</td>
  <td>2005-05</td>
 </tr>
 <tr>
  <td>SPNCRNA.523</td>
  <td>extended exons of adjacent protein coding gene</td>
  <td><span class="no-break">2008-12-19</span></td>
 </tr>
 <tr>
  <td>SPBC14C8.08c</td>
  <td>merged with SPBC14C8.09c; frameshifted; <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16823372">PMID:16823372</a></td>
  <td><span class="no-break">2009-02-02</span></td>
 </tr>
 <tr>
  <td>SPRRNA.25</td>
  <td>small fragment of 25S rRNA; <span class="no-break">2009-11-26</span></td>
 </tr>
 <tr>
  <td>SPAC4G9.22</td>
  <td>likely to be non-coding RNA, no supporting evidence for protein</td>
  <td><span class="no-break">2010-02-21</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.509</td>
  <td>dubious and partially overlapping with Ost5</td>
  <td><span class="no-break">2010-02-18</span></td>
 </tr>
 <tr>
  <td>SPBC11C11.12</td>
  <td>cob intron fragment currently annotated as pseudogene</td>
  <td><span class="no-break">2010-03-09</span></td>
 </tr>
 <tr>
  <td>SPBCPT2R1.09c</td>
  <td>no coding region</td>
  <td><span class="no-break">2010-09-19</span></td>
 </tr>
 <tr>
  <td>SPBPB2B2.04</td>
  <td>no coding region</td>
  <td><span class="no-break">2010-09-19</span></td>
 </tr>
 <tr>
  <td>SPBPB2B2.03c</td>
  <td>no coding region</td>
  <td><span class="no-break">2010-09-19</span></td>
 </tr>
 <tr>
  <td>SPAC17A2.15</td>
  <td>was on the reverse strand of the N-terminal extension of SPAC17A2.08</td>
  <td><span class="no-break">2011-03-10</span></td>
 </tr>
 <tr>
  <td>SPBC1348.13</td>
  <td>Coordinates updated; now annotated as nuclear mitochondrial pseudogene (SO:0001044)</td>
  <td><span class="no-break">2011-11-20</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.10</td>
  <td>merged with SPNCRNA.1234</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1098</td>
  <td>merged with SPNCRNA.07</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1196</td>
  <td>merged with SPNCRNA.67</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.125</td>
  <td>merged with SPCC16A11.14</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1254</td>
  <td>merged with SPNCRNA.511</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1291</td>
  <td>merged with SPNCRNA.519</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1486</td>
  <td>merged with SPNCRNA.388</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.18</td>
  <td>merged with SPNCRNA.928</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.489</td>
  <td>merged with SPNCRNA.1187</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.502</td>
  <td>merged with SPNCRNA.1228</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.582</td>
  <td>merged with SPNCRNA.1208</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.64</td>
  <td>merged with SPCC1393.13</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.884</td>
  <td>merged with SPNCRNA.214</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.958</td>
  <td>merged with SPAC23A1.20</td>
  <td><span class="no-break">2011-12-16</span></td>
 </tr>
 <tr>
  <td>SPAC1F12.03c</td>
  <td>removed; replaced by a nuclear mitochondrial pseudogene (NUMT) feature</td>
  <td><span class="no-break">2012-07-16</span></td>
 </tr>
 <tr>
  <td>SPAC1F8.09c</td>
  <td>annotated in error</td>
  <td><span class="no-break">2013-02-22</span></td>
 </tr>
 <tr>
  <td>SPBC713.13</td>
  <td>deleted; ID added as synonym to SPBC713.14c&nbsp;(<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>)</td>
  <td><span class="no-break">2014-07-04</span></td>
 </tr>
 <tr>
  <td>SPAC823.02</td>
  <td>deleted; replaced by SPAC823.17 at same locus, but in different frame&nbsp;(<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:24929437</a>)</td>
  <td><span class="no-break">2014-07-04</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.252</td>
  <td>changed to LTR</td>
  <td><span class="no-break">2015-01-22</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.76</td>
  <td>overlapped with correctly annotated NUMT (SPNUMT.9)</td>
  <td><span class="no-break">2015-10-12</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.83</td>
  <td>overlapped with correctly annotated NUMT (SPNUMT.8)</td>
  <td><span class="no-break">2015-10-12</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.906</td>
  <td>merged with snR30</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.814</td>
  <td>merged with snR98</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1436</td>
  <td>merged with snR95</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.9061</td>
  <td>merged with snR91</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1038</td>
  <td>merged with snR90</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.1394</td>
  <td>merged with snR3</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.727</td>
  <td>deleted; ID added as synonym to SPNCRNA.82 and SPSNORNA.10 (covers both)</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.727</td>
  <td>deleted; ID added as synonym to snR41/SPSNORNA.07, snR70/SPSNORNA.08 and snR51b/SPSNORNA.09 (covers all)</td>
  <td><span class="no-break">2015-11-16</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.866</td>
  <td>unsupported (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:21511999</a>)</td>
  <td><span class="no-break">2015-11-30</span></td>
 </tr>
 <tr>
  <td>SPNCRNA.867</td>
  <td>unsupported (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21511999">PMID:21511999</a>)</td>
  <td><span class="no-break">2015-11-30</span></td>
 </tr>
 <tr>
  <td>SPAC1D4.07c</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24929437">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26615217">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPAC27E2.13</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24929437">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26615217">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPAC4H3.12c</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24929437">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26615217">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPBC1685.12c</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24929437">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26615217">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPBC32F12.17</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24929437">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26615217">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPBC354.11c</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24929437">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26615217">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPBC36.13</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24929437">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26615217">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPCC1672.14</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24929437">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26615217">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPCC188.05</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24929437">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26615217">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPCC417.04</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24929437">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26615217">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPCC622.05</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24929437">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26615217">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
 <tr>
  <td>SPCC794.16</td>
  <td>not protein-coding (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24929437">PMID:24929437</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26615217">PMID:26615217</a>)</td>
  <td><span class="no-break">2015-12-11</span></td>
 </tr>
    <tr>
  <td>SPBC8E4.02c</td>
  <td>SPBC8E4.02c replaced by SPNCRNA.9001 (prt2) (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29414789">PMID:29414789</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29414789">PMID:29414789</a>)</td>
  <td><span class="no-break">2018-10-11</span></td>
 </tr>
 </tbody>
</table>

  </div>
  <div *ngIf="pageName == 'priority-unstudied-genes'">
<h3 id="priority-unstudied-genes">Priority unstudied genes</h3>
<p>Fission yeast genes are classed as “unknown” if there is no information about the broad cellular role (biological process) in which it participates (corresponding to any of the <a routerLink="/browse-curation/fission-yeast-bp-go-slim-terms">high level GO biological process slim classes</a>). For these, we have been unable to identify a broad biological role based on experimental data in fission yeast or any other organism. Note that, as all genes in fission yeast have been curated, these genes are documented as “unknown” (they are not “unannotated”).</p>
<p>In fission yeast the “unknown” inventory is <app-query-link [goToResults]="true" [predefinedQueryId]="'priority-unstudied-genes:unknown_genes'"></app-query-link> entries, many of which are apparently found only in the fission yeast clade. However, a large number (<app-query-link [goToResults]="true" [predefinedQueryId]="'priority-unstudied-genes:conserved_unknown'"></app-query-link>) are conserved, and a significant number of these (<app-query-link [goToResults]="true" [predefinedQueryId]="'priority-unstudied-genes:conserved_unknown_AND_conserved_in_vertebrates'"></app-query-link>) have orthologs in vertebrates. These genes are listed below.</p>
<p>You can recreate this query, or variations of it, using the Advanced Search. See the <a routerLink="/faq/how-can-i-find-all-s.-pombe-genes-are-conserved-human">relevant FAQ</a> and the <a routerLink="/query">Advanced Search</a> documentation for details. You can also send this list directly to the advanced search (For example to mine for phenotypes or locations of interest).</p>
<p><app-predefined-query-results [description]="'Unstudied genes'" [predefinedQueryId]="'priority-unstudied-genes:conserved_unknown_AND_conserved_in_vertebrates'"></app-predefined-query-results></p>

  </div>
  <div *ngIf="pageName == 'sequence-updates-pending'">
<h3 id="pending-sequence-updates">Pending sequence updates</h3>
<h4 id="individual-sequence-updates">Individual sequence updates</h4>
<ul>
<li>Chromosome III: The reference sequence is missing 4 bp – GATC to be inserted after reference position 17744235. Pers. comm. Nathan Lee (S. Grewal lab), reported <span class="no-break">2014-03-31</span>.</li>
</ul>
<h4 id="sequence-discrepancies-from-the-broad-institute">Sequence discrepancies from the Broad Institute</h4>
<p>As part of a technology development project, the Broad Institute generated 200-fold sequence coverage of the genome of <em>S. pombe</em> strain 972 using Illumina (Solexa) technology. This deep coverage was used to identify discrepancies between the Illumina read data and the current reference assembly of 972, employing a polymorphism discovery algorithm written specifically for Illumina data, and tuned to have a very low false positive call rate. 190 sequence discrepancies were observed, all but two of which were either single base insertions, deletions or substitutions. These discrepancies could be due to sequence errors in the reference 972 assembly, sequence polymorphisms between the two isolates of 972 sequenced, or systematic errors in the Illumina data or analysis. Of the 190 discrepancies, 25 occur in homopolymer runs of at least 7 nucleotides, which are both more likely to produce sequencing errors and inter-strain polymorphisms, and 39 are clustered within 10 nucleotides of another discrepancy, which is more common in cases of mis-alignment of the Illumina reads. Excluding these sites, the discrepancy rate is about 1 in 100,000, which is close to the estimated error rate in published reference sequence.</p>
<p>The data listed here were updated in August 2008; the original coordinates were based on an older assembly and did not match the current assembly of the right arm of Chromosome I. One discrepancy is a known artifact. At nucleotide 1774235 of Chromosome III there is an insertion of GACT caused by an assembly glitch in the published reference, which will be corrected in future releases. In addition, there are several clusters of discrepancies which are suspicious. These discrepancies will be investigated and the discrepancy list updated as appropriate. Beyond that analysis, no validation of other discrepancies is currently planned. However, we welcome any validation efforts by members of the community. If you generate any data pertinent to these discrepancies, or if you have any comments about them, please contact <a href="mailto:nick.rhind@umassmed.edu">Nick Rhind</a> or <a href="mailto:val@sanger.ac.uk">Valerie Wood</a>. These data can be cited as follows: “Data were generated at the Broad Institute. Chad Nusbaum, personal communication.”</p>
<p>The data can be downloaded as an Excel spreadsheet from the <a href="ftp://ftp.pombase.org/archive/972_discrepancies_from_Broad.xls">PomBase ftp site</a></p>
<table>
<thead>
<tr class="header">
<th>Chromosome</th>
<th>Location</th>
<th>Left</th>
<th>Sample</th>
<th>Reference</th>
<th>Right</th>
<th>Clustered?</th>
<th>Homopolymer</th>
<th>Confirmed in</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>I</td>
<td>94353</td>
<td>GTGACAATTGAATTTTTTTT</td>
<td>T</td>
<td></td>
<td>AAATGTTTCTATATTGTTAA</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>101872</td>
<td>GAGGACGGGCGTTGGCGGAG</td>
<td>G</td>
<td></td>
<td>CAACAGCCTTACCCCAGTCA</td>
<td>No</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="odd">
<td>I</td>
<td>470379</td>
<td>TGTGATAGACAAAAAAAGAC</td>
<td></td>
<td>C</td>
<td>GTTAAAAGAACCTAAACGAA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>470390</td>
<td>AAAAAAGACCGTTAAAAGAA</td>
<td></td>
<td>C</td>
<td>CTAAACGAATTGTAATCAGA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>482738</td>
<td>AGCTATTCGATCCACCGCAG</td>
<td>G</td>
<td></td>
<td>TTTGTCTGATTTGTTGCCCT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>523602</td>
<td>ACAATGATACTTACTCCACA</td>
<td>A</td>
<td>G</td>
<td>CGATTCCATTTTCCAAAGAC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>523882</td>
<td>GTTTCAGCCATTGAAAGATA</td>
<td>A</td>
<td>G</td>
<td>CGATCGTTCTCAGACAAAGA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>527155</td>
<td>AATCATGGATTTAATACATA</td>
<td>A</td>
<td>C</td>
<td>CTTTTAGTTTGTATGTGATA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>554538</td>
<td>GAAGATTTATTTTTTTTTTT</td>
<td>T</td>
<td></td>
<td>GGAATTTATTCTGCTCGGCC</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>670033</td>
<td>AGAGATTATTATTAAGATCA</td>
<td>A</td>
<td></td>
<td>CCTAAATATCGTAACGTACG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>682996</td>
<td>ATAAAACTACTATACTTCCC</td>
<td></td>
<td>C</td>
<td>GCTGGAGCTTCGGTAGGCCG</td>
<td>No</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="even">
<td>I</td>
<td>759333</td>
<td>TGTTCACAGTTTTTCCCAAA</td>
<td>T</td>
<td>G</td>
<td>TTGCTTTTTCTTTTGGCTTA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>862963</td>
<td>ATTAAATTCTCGAAGTTGCT</td>
<td>TT</td>
<td>G</td>
<td>ATTTTCCAAATCAATTCATG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>1048646</td>
<td>ACGGAAAAGAAAAAAAAAAA</td>
<td>A</td>
<td></td>
<td>TCCGCGTTCAGTAGAAGTCA</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>1274804</td>
<td>GTACTTGAAATTATTTTTTA</td>
<td>A</td>
<td></td>
<td>TAAAGTGAGAACGCCCAGTT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>1424708</td>
<td>GTGCTTCATCAGTGACAACC</td>
<td></td>
<td>A</td>
<td>TTTAGAATGGATCTGTATGT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>1625094</td>
<td>ATCAAATAGTCGATGCCTCC</td>
<td>C</td>
<td></td>
<td>AGCATCTCAATCTTCAATTC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>1783457</td>
<td>ACTCTGGGTTCTTCTTATGT</td>
<td>T</td>
<td>C</td>
<td>TGGGACGGTCTTCGGCATAA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>1936298</td>
<td>ACTCTTAAATTTTAGGAAAA</td>
<td>A</td>
<td></td>
<td>GACGCTCTTTTATAGTAAGG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>1952317</td>
<td>TATATGTACGTACATATTTT</td>
<td>T</td>
<td></td>
<td>GCGCACCAGTTAGTTTGCGA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>1999191</td>
<td>TTTACCGCTTGACTGTGACC</td>
<td>C</td>
<td>G</td>
<td>TACGACCTAACAATTTATTA</td>
<td>No</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/18088324">PMID:18088324</a></td>
</tr>
<tr class="even">
<td>I</td>
<td>2104090</td>
<td>AAACTGTTGTAAAAAAATTA</td>
<td>T</td>
<td>G</td>
<td>ATTATCGGTGTATTTATTTA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>2104096</td>
<td>TTGTAAAAAAATTAGATTAT</td>
<td>A</td>
<td>C</td>
<td>GGTGTATTTATTTATATTAG</td>
<td>Yes</td>
<td>-</td>
<td>6</td>
</tr>
<tr class="even">
<td>I</td>
<td>2154492</td>
<td>AATTTGTTAAAAAAGGAGTA</td>
<td>A</td>
<td></td>
<td>GTTTACCTCTCGCTTTATGC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>2288241</td>
<td>TCTTCATTGTTATTGTTACG</td>
<td></td>
<td>G</td>
<td>ATTCGTCACCCAGTGAAGAT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>2343704</td>
<td>TGCAAACTCTTTGATCACGA</td>
<td>A</td>
<td></td>
<td>GTCTACATAATTCTTGAACC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>2421577</td>
<td>AGCTAAATATAAGAAAAAAC</td>
<td></td>
<td>C</td>
<td>GAGAATTATAAGCAAACGTT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>2499159</td>
<td>AAAACATCAATTTTTTTTTT</td>
<td></td>
<td>T</td>
<td>GACAGTAATTATACTAGAAT</td>
<td>No</td>
<td>X</td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="odd">
<td>I</td>
<td>2506043</td>
<td>AAATAGTGTCACATTGTTTT</td>
<td>A</td>
<td>T</td>
<td>ATATTAAGTATTTTTAATCG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>2507463</td>
<td>TTAGGTGTTAAACATAAATA</td>
<td>A</td>
<td></td>
<td>CTTACGTTTTCATACGCGCA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>2588024</td>
<td>AAAGAACAGCAAAAAACAAA</td>
<td>A</td>
<td></td>
<td>GATAATGACAGAAAAAATAT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>2588072</td>
<td>CTGAGAAAGTTGACAAAAAA</td>
<td>A</td>
<td></td>
<td>TTTTTGGCGAAAGAAAGAAA</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>2594264</td>
<td>TAATTTTTAATGAATACTCT</td>
<td>T</td>
<td></td>
<td>AGGCTTCCTTAAATGTTTTT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>2594326</td>
<td>TTAAAGCAGTGTCTTATTAA</td>
<td>A</td>
<td></td>
<td>CATTGTAAACATCAAAAAGC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>2594351</td>
<td>TAAACATCAAAAAGCTTCAT</td>
<td>A</td>
<td></td>
<td>AACAGATGTAAGAAATTAAT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>2594393</td>
<td>AATTAACTTGTGAAGACATT</td>
<td>T</td>
<td></td>
<td>CGATGTGATTAAAGTATGAA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>2605763</td>
<td>TGTAAAAAGCATTTGAAATT</td>
<td>A</td>
<td>T</td>
<td>AAAAAGAAAAAACAAAAAAA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>2605862</td>
<td>AACGTACTGAGGATATCGTT</td>
<td></td>
<td>T</td>
<td>ACCCCTTGCAGAAATATAGA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>2607673</td>
<td>ATCCATGTCTGTAATATTTT</td>
<td>T</td>
<td></td>
<td>GGTTATTAATAAATTGAAAT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>2683903</td>
<td>CAAATATCATCAGTGCGATC</td>
<td></td>
<td>C</td>
<td>AACAGCGGTATTTGTCTGAG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>2759872</td>
<td>TATCTTCCCACCGTCGTTTC</td>
<td>C</td>
<td></td>
<td>TATCGGTAGCATATCCGATT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>3111507</td>
<td>TGCGTTTTCTGTCTGTTGTG</td>
<td>G</td>
<td>A</td>
<td>ATGTCTCGCCTGGACTTCTT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>3115840</td>
<td>ACAGATAAAGAGAGAAAAGT</td>
<td>A</td>
<td>C</td>
<td>CAAGCGTTTAAGCACCGAAG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>3119298</td>
<td>AAAATCTTATTGATTGAAAT</td>
<td>A</td>
<td>G</td>
<td>CAATGCTTCGTAAACAATAC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>3125119</td>
<td>TGCCAAACTCTTTTCTTCAT</td>
<td>T</td>
<td></td>
<td>ATGTTCTGTTGTGCCAAAGA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>3178496</td>
<td>TAACTGGCAAATCGGCTCTT</td>
<td>T</td>
<td></td>
<td>CCCATAAGTAGACGATGATC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>3179237</td>
<td>TGCTAAAATACAGTTAGTTA</td>
<td>C</td>
<td>G</td>
<td>TACTTGCTCACTTATATTTG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>3197530</td>
<td>AATACTGCTGGAAATACAGG</td>
<td>G</td>
<td></td>
<td>TTTGGTTCGCAAGGTACTGG</td>
<td>No</td>
<td></td>
<td><a href="http://getentry.ddbj.nig.ac.jp/getentry/na/LC043101/?format=flatfile&filetype=html&trace=true&show_suppressed=false&limit=10">DDJB:LC043101</a></td>
</tr>
<tr class="odd">
<td>I</td>
<td>3450130</td>
<td>TTATTATTCCCAGGCACGGG</td>
<td></td>
<td>T</td>
<td>AAGTTCCAAAAATCGAATAT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>3460320</td>
<td>TTTACAAAGTCGGACAATCC</td>
<td>C</td>
<td></td>
<td>TGGCGGCGGTGTTCTAAAAA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>3538415</td>
<td>TAAAGTATCGGAATTACATT</td>
<td>T</td>
<td></td>
<td>CGTCAAACCAGCGTTTATGA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>3538428</td>
<td>TTACATTCGTCAAACCAGCG</td>
<td>T</td>
<td></td>
<td>TTTATGATAAAAGATCATCC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>3538436</td>
<td>GTCAAACCAGCGTTTATGAT</td>
<td>A</td>
<td></td>
<td>AAAAGATCATCCTATTCTAA</td>
<td>Yes</td>
<td>-</td>
<td>8</td>
</tr>
<tr class="even">
<td>I</td>
<td>3538448</td>
<td>TTTATGATAAAAGATCATCC</td>
<td>T</td>
<td></td>
<td>TATTCTAATTTTGCTATCGA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>3538450</td>
<td>TATGATAAAAGATCATCCTA</td>
<td>T</td>
<td></td>
<td>TTCTAATTTTGCTATCGATA</td>
<td>Yes</td>
<td>-</td>
<td>2</td>
</tr>
<tr class="even">
<td>I</td>
<td>3538454</td>
<td>ATAAAAGATCATCCTATTCT</td>
<td>A</td>
<td></td>
<td>AATTTTGCTATCGATAGCCC</td>
<td>Yes</td>
<td>-</td>
<td>4</td>
</tr>
<tr class="odd">
<td>I</td>
<td>3538456</td>
<td>AAAAGATCATCCTATTCTAA</td>
<td>T</td>
<td></td>
<td>TTTTGCTATCGATAGCCCTT</td>
<td>Yes</td>
<td>-</td>
<td>2</td>
</tr>
<tr class="even">
<td>I</td>
<td>3583105</td>
<td>TTATAAATTTATAAATGATA</td>
<td>A</td>
<td>G</td>
<td>AAATAAAGTGCAAGAAGAAT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>3665610</td>
<td>CTAGATTTAACTTTGCAACG</td>
<td>C</td>
<td></td>
<td>TTTCGCAATCGGTGAACAGC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>3730377</td>
<td>TGCAACAATTCATTTTTTTT</td>
<td>T</td>
<td></td>
<td>ACCTAATTTGTTTTCGACAA</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>3855791</td>
<td>GGCATTTTTCACGTACAGGT</td>
<td></td>
<td>T</td>
<td>AGTCGAAACATTATTAAATA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>4055068</td>
<td>ATTTAAAAAAAAAAAAAAAA</td>
<td>A</td>
<td></td>
<td>TTATAAGACATACCCTTTCG</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>4276903</td>
<td>TAGAATGTTTTTTTTTTTTT</td>
<td>T</td>
<td></td>
<td>GAGAATATTATTCACACGCC</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>4304172</td>
<td>TCAAACCTATAAACAGGAAA</td>
<td>A</td>
<td></td>
<td>GATAAACGAAAATAGAATTA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>4374343</td>
<td>AGGCTAAGTCTAGGTTGTAG</td>
<td></td>
<td>G</td>
<td>AAAAATCGTTTAGTCTGTAT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>4375420</td>
<td>GTCTGAAGACTTTACCTTCT</td>
<td>T</td>
<td>A</td>
<td>TCTTTTGTGCTCCTATCAAA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>4407495</td>
<td>TCTCGTATAAGTTCATCCTG</td>
<td>G</td>
<td></td>
<td>CGAAGCCGTTGTAGCCTATT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>4410193</td>
<td>ATCGACACAGATCGCGGCGG</td>
<td></td>
<td>G</td>
<td>AAGAAAAAGCAACTGAAGAT</td>
<td>No</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/17035632">PMID:17035632</a></td>
</tr>
<tr class="odd">
<td>I</td>
<td>4431859</td>
<td>ATCGAAACTACGTTGTGAAA</td>
<td>A</td>
<td></td>
<td>TAACTTAGCAAATATATGGT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>4431868</td>
<td>ACGTTGTGAAATAACTTAGC</td>
<td>A</td>
<td></td>
<td>AAATATATGGTAAATAACAC</td>
<td>Yes</td>
<td>-</td>
<td>9</td>
</tr>
<tr class="odd">
<td>I</td>
<td>4442713</td>
<td>TGGAAGCTGATTTTTTGTTA</td>
<td>A</td>
<td>G</td>
<td>TATGTCGTAAGTTACTGTGT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>4455162</td>
<td>TAACTAGAAAAAAAAAAAAA</td>
<td>A</td>
<td></td>
<td>CCAAAGATAAAAAATGAAGT</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>4721884</td>
<td>AGTAGGAGTAGAAAAAAAAA</td>
<td>A</td>
<td></td>
<td>TTTTCAGCGCCTCGTCTCCT</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>4764269</td>
<td>TTTGAAAAAAAAAAAAAAAA</td>
<td>A</td>
<td></td>
<td>TTTCTACATTTCTTTCTTTA</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>4919602</td>
<td>GTTTTTTGGGTAAAGATCTA</td>
<td>G</td>
<td>A</td>
<td>AGGGTATATTGCTTTTTTAA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>5142628</td>
<td>CTGAGGAAGACGTTCCGAAG</td>
<td>G</td>
<td></td>
<td>TAAAGTGGAAAACGTTAGAG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>5148423</td>
<td>AATAAAACAGTAAAAAAAAA</td>
<td></td>
<td>A</td>
<td>GATAAAAAGCTGAACAGTAT</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>5173097</td>
<td>CACAGATAGCTTTTTTTTTT</td>
<td>T</td>
<td></td>
<td>AACAGGTACTTTATACATGG</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="odd">
<td>I</td>
<td>5368264</td>
<td>AGTATAGTTGGTCTCCCTCC</td>
<td></td>
<td>C</td>
<td>ATATACGGTTTGGATAAGTT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>I</td>
<td>5368273</td>
<td>GGTCTCCCTCCCATATACGG</td>
<td>T</td>
<td></td>
<td>TTTGGATAAGTTTCGCCTTG</td>
<td>Yes</td>
<td>-</td>
<td>9</td>
</tr>
<tr class="odd">
<td>I</td>
<td>5400079</td>
<td>TACATTTTAGGATCAGCTCA</td>
<td>T</td>
<td>C</td>
<td>AAGTGTCCCTTTTTACAAGA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>37049</td>
<td>ATTAATAATATATTTGCTCA</td>
<td>T</td>
<td>A</td>
<td>CCTAGATTTAAAGAATTTAG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>128007</td>
<td>TCAGAGCAGTCAATAAAAAA</td>
<td></td>
<td>A</td>
<td>TAAAAAATCGAACAAAGGAA</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>146086</td>
<td>GCCCTCGCTTGTTCCCCCCC</td>
<td></td>
<td>C</td>
<td>TTATCTTACGAGTATATAGC</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>266717</td>
<td>ATTCATGAGATTCGTACACG</td>
<td>C</td>
<td>A</td>
<td>CCTTACCAAGTCTTGCCAGA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>266779</td>
<td>TGTTAGGGGTGATATGGGTT</td>
<td>A</td>
<td>T</td>
<td>CGTTTTATCTGGGATCAAGG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>276538</td>
<td>GACTTTAATTTATTCATGGA</td>
<td>T</td>
<td>A</td>
<td>CCTGAAGCTGCTCAAACTGC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>685727</td>
<td>ATCTCGCTGCTCAAATGTCA</td>
<td>A</td>
<td>G</td>
<td>ATGCTGATTTTGGCTCCAAT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>699950</td>
<td>TTTCTTTGAATTTTTTGTTT</td>
<td>G</td>
<td>T</td>
<td>GTATTTTAAATTTATTATCT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>699951</td>
<td>TTCTTTGAATTTTTTGTTTT</td>
<td>T</td>
<td>G</td>
<td>TATTTTAAATTTATTATCTT</td>
<td>Yes</td>
<td>-</td>
<td>1</td>
</tr>
<tr class="odd">
<td>II</td>
<td>699960</td>
<td>TTTTTTGTTTTGTATTTTAA</td>
<td>T</td>
<td>A</td>
<td>TTTATTATCTTCCTGTCTTT</td>
<td>Yes</td>
<td>-</td>
<td>9</td>
</tr>
<tr class="even">
<td>II</td>
<td>700010</td>
<td>CCTCTATTTTTACAGATTTA</td>
<td></td>
<td>A</td>
<td>CAAGTTTTCAACTTCGAAAC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>700018</td>
<td>TTTACAGATTTAACAAGTTT</td>
<td>A</td>
<td>T</td>
<td>CAACTTCGAAACCCTAGAAA</td>
<td>Yes</td>
<td>-</td>
<td>8</td>
</tr>
<tr class="even">
<td>II</td>
<td>700019</td>
<td>TTACAGATTTAACAAGTTTT</td>
<td>T</td>
<td>C</td>
<td>AACTTCGAAACCCTAGAAAT</td>
<td>Yes</td>
<td>-</td>
<td>1</td>
</tr>
<tr class="odd">
<td>II</td>
<td>700020</td>
<td>TACAGATTTAACAAGTTTTC</td>
<td>C</td>
<td>A</td>
<td>ACTTCGAAACCCTAGAAATT</td>
<td>Yes</td>
<td>-</td>
<td>1</td>
</tr>
<tr class="even">
<td>II</td>
<td>700023</td>
<td>AGATTTAACAAGTTTTCAAC</td>
<td>A</td>
<td>T</td>
<td>TCGAAACCCTAGAAATTTTC</td>
<td>Yes</td>
<td>-</td>
<td>3</td>
</tr>
<tr class="odd">
<td>II</td>
<td>700027</td>
<td>TTAACAAGTTTTCAACTTCG</td>
<td>T</td>
<td>A</td>
<td>AACCCTAGAAATTTTCATAT</td>
<td>Yes</td>
<td>-</td>
<td>4</td>
</tr>
<tr class="even">
<td>II</td>
<td>700112</td>
<td>CTTTATCAAAAGTTATCATT</td>
<td>T</td>
<td>A</td>
<td>AATTGCCCTGCAACCGATAT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>869805</td>
<td>CTCTTCCAAGTCGGGTTCAT</td>
<td>C</td>
<td>G</td>
<td>ATGCACTAATGAATTGTAGA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>879032</td>
<td>ACTAGAGAAAGAAAAAAAGT</td>
<td>A</td>
<td>T</td>
<td>ACAGTGGCTGATATTACAAT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>888600</td>
<td>AAGAAAACTTTTGGTTATAC</td>
<td></td>
<td>C</td>
<td>GAAACTTTGTGTGAAATGTA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>903106</td>
<td>AGCACAAACAATTCGAACTG</td>
<td>T</td>
<td></td>
<td>GTGCTAACATCTAATAAAGA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>923542</td>
<td>TTTAAAAAAATGACAGACGA</td>
<td>A</td>
<td>C</td>
<td>TTAAACTATGTTTTGAAATT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>943553</td>
<td>AGCAAAGAATGAAGCCCTTA</td>
<td>A</td>
<td>C</td>
<td>ATTTTTGGATAGTTTTAAGA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>953200</td>
<td>GTCGTTTCGAAGTTTTAGAA</td>
<td>G</td>
<td>A</td>
<td>ATGAAAAAAATTGGGCGTCT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>967093</td>
<td>GAATTTCCCTTGACTTTTGA</td>
<td>G</td>
<td>A</td>
<td>GGAAGCCATATCTTCGAGTT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>1155902</td>
<td>CTTGCAAAAACAAAAATATA</td>
<td>A</td>
<td>T</td>
<td>ATCCTTTTTTCTGCTGTAAA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>1159677</td>
<td>TGAGTTTATTCATGAATTGC</td>
<td>C</td>
<td>G</td>
<td>CGAAAATGAGGGTCCATCCG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>1159693</td>
<td>TTGCGCGAAAATGAGGGTCC</td>
<td>T</td>
<td>A</td>
<td>TCCGTTCTCTACGATTCTGG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>1163155</td>
<td>GTATATAAGTAAATAAAAAT</td>
<td></td>
<td>T</td>
<td>AGTATGCTTCTGGTCAACTA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>1308626</td>
<td>GTTCGGCGAAGGTTTTTTTT</td>
<td></td>
<td>T</td>
<td>GCATTGATCAACAGGTGATG</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>1672858</td>
<td>GCTTCCGTTTATATACGCGT</td>
<td>T</td>
<td>C</td>
<td>GGACATATAACAATGAACTG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>1673001</td>
<td>GATGCTCCACCACCTGAAAC</td>
<td>C</td>
<td>G</td>
<td>TTGATGCGTTCATTAGAGTT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>1678692</td>
<td>TTTGCATTGGCAGCGACGTT</td>
<td>A</td>
<td>T</td>
<td>GTATTTGCCGGCAAGCCCAA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>1683323</td>
<td>CATTATGGTTGGAAATGATG</td>
<td>G</td>
<td>T</td>
<td>AAACACCGTATACTTAGCTG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>1697123</td>
<td>AGACTTGTTAATCATGTTAG</td>
<td>T</td>
<td>C</td>
<td>ACCTATGCTTTAATATAATT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>1716317</td>
<td>TACCGAGAAAACTTTCGTCA</td>
<td>G</td>
<td>A</td>
<td>ATTTTATGGGCCTAATAATA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>1716319</td>
<td>CCGAGAAAACTTTCGTCAAA</td>
<td>G</td>
<td>T</td>
<td>TTTATGGGCCTAATAATAAT</td>
<td>Yes</td>
<td>-</td>
<td>2</td>
</tr>
<tr class="odd">
<td>II</td>
<td>1716325</td>
<td>AAACTTTCGTCAAATTTTAT</td>
<td>T</td>
<td>G</td>
<td>GGCCTAATAATAATCTTATG</td>
<td>Yes</td>
<td>-</td>
<td>6</td>
</tr>
<tr class="even">
<td>II</td>
<td>1784825</td>
<td>GGCCACATGTCATACATTGA</td>
<td>A</td>
<td>G</td>
<td>TACGACTGCAAAGTTGACCT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>1849776</td>
<td>TGTCAGAATCATTGTCGAGG</td>
<td>A</td>
<td>G</td>
<td>TTTTGCATATCTCGTAAGCA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>1861420</td>
<td>GAAAAGCATAAGAAGTTAAT</td>
<td></td>
<td>G</td>
<td>TACATAAAAGCACTTTAAAT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>1867805</td>
<td>AAAGTACTTATTTTTTTTTT</td>
<td>T</td>
<td></td>
<td>ACGAATAAAATGTAAAAAAA</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>1873781</td>
<td>GTGGTTTGTAATATTCCTAA</td>
<td>A</td>
<td></td>
<td>CCACCCCATTTTTTTCTGGC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>1877305</td>
<td>ACAGCCCGATTTTTTTTTTT</td>
<td>T</td>
<td></td>
<td>GTCTCAGTAATTCAAGAGAC</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>1941966</td>
<td>GATTCACATACCGAAGCGGA</td>
<td>T</td>
<td></td>
<td>GAAAATTAATAAATCAGAGT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>1941982</td>
<td>CGGAGAAAATTAATAAATCA</td>
<td>T</td>
<td></td>
<td>GAGTTTTAACTCAAAGGAAG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>1948954</td>
<td>CGGCGTTGAAGCGATCTTGA</td>
<td></td>
<td>A</td>
<td>GCGGCCTGGAGTGCGCGAGT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>1950052</td>
<td>ATGTCGGCATTGCCAGAAGG</td>
<td>G</td>
<td></td>
<td>TTTTTGCTAGAGGCATAGCT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>1955509</td>
<td>TTCCTTAAAAAAAAAAAAAA</td>
<td>A</td>
<td></td>
<td>GAAAGAAAATGAAAACTCAA</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>1960392</td>
<td>AATTAAATTGTCTAAACGTA</td>
<td>G</td>
<td></td>
<td>TTCGTTGCTCAAGCTTCCTT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>1962418</td>
<td>TAAGAGAGAAAACTCGGTGT</td>
<td>A</td>
<td>T</td>
<td>TATTGGAAAAGTGGTACTAA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>1962438</td>
<td>TTATTGGAAAAGTGGTACTA</td>
<td>G</td>
<td>A</td>
<td>CAAGTGTGGGAAGTATGTTT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>1962457</td>
<td>AACAAGTGTGGGAAGTATGT</td>
<td>A</td>
<td>T</td>
<td>TGACAGATTCATTACATTAG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>1987097</td>
<td>AGCTTCGCATAGACATATAC</td>
<td>A</td>
<td>C</td>
<td>AACGGAACGAATGACACCTG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>1987101</td>
<td>TCGCATAGACATATACCAAC</td>
<td></td>
<td>G</td>
<td>GAACGAATGACACCTGGCAG</td>
<td>Yes</td>
<td>-</td>
<td>4</td>
</tr>
<tr class="odd">
<td>II</td>
<td>1987117</td>
<td>CAACGGAACGAATGACACCT</td>
<td></td>
<td>G</td>
<td>GCAGTGAAAATTCCCTTTAT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>2044928</td>
<td>AATTGACTTATCTATTAATT</td>
<td>T</td>
<td>A</td>
<td>ACTTGATTTTTGTTTGTTAG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>2049893</td>
<td>GAAACTAGCTCATTATGATT</td>
<td></td>
<td>T</td>
<td>GAAGTTTGTGCCAGATAAAA</td>
<td>No</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="even">
<td>II</td>
<td>2053517</td>
<td>AGTGATTTTCTCAAAAAGGC</td>
<td>C</td>
<td></td>
<td>TCTGCTACCCGTCATATTGT</td>
<td>No</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="odd">
<td>II</td>
<td>2108180</td>
<td>GTAAGATAATAAAGCATAAT</td>
<td>A</td>
<td></td>
<td>CCAATCTTCTTTGTTTGAAG</td>
<td>No</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="even">
<td>II</td>
<td>2187407</td>
<td>CATTTTTCTTGGTGGGAAGG</td>
<td>A</td>
<td></td>
<td>CATCAAAAAAATCCCCCTGG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>2219929</td>
<td>AATTGACTGTTTGGCTGTAT</td>
<td>T</td>
<td></td>
<td>GGTGATATGCTGAAAGCGAA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>2228774</td>
<td>AAGACAAGATAAATAATAAG</td>
<td>G</td>
<td>T</td>
<td>TGGAAAAGTATATACCTCTG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>2228776</td>
<td>GACAAGATAAATAATAAGTT</td>
<td>T</td>
<td>G</td>
<td>GAAAAGTATATACCTCTGTT</td>
<td>Yes</td>
<td>-</td>
<td>2</td>
</tr>
<tr class="even">
<td>II</td>
<td>2264919</td>
<td>ATAATAAAAACATTTATTTA</td>
<td>C</td>
<td>A</td>
<td>GTTTTATTTTTGAACTTCTA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>2293304</td>
<td>ATTTATCACAACGAAAAAAA</td>
<td></td>
<td>A</td>
<td>CTCGTATCACTATAATTTTT</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>2436579</td>
<td>ACCTTGCAGCCAACTACTTC</td>
<td>G</td>
<td>C</td>
<td>TCACCAATTACAACTTCTGC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>2444542</td>
<td>GAATAATGCTTCCTACATTG</td>
<td>C</td>
<td>T</td>
<td>TACCGCCTATTTAATTACAA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>2592434</td>
<td>CCTAAACATAAAAAAAAAAA</td>
<td>A</td>
<td></td>
<td>CTTACAATCAATGTATATGG</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>2676335</td>
<td>TTTTACCGGTTCTAAAATAT</td>
<td>T</td>
<td>C</td>
<td>CCAAAGACCTGAAACCCCCA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>2709415</td>
<td>CGTAGCACAAGGCCGAGAGC</td>
<td>C</td>
<td></td>
<td>TCTTCTCATAGTGGCTCTTT</td>
<td>No</td>
<td></td>
<td>pers.</td>
</tr>
<tr class="odd">
<td>II</td>
<td>2788933</td>
<td>ATATTTTGCTGGATTAAAGC</td>
<td>G</td>
<td></td>
<td>AAATAACTTTTGCGTCAAAC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>2798041</td>
<td>ACATTCATGGATGAAGACCT</td>
<td></td>
<td>T</td>
<td>GGGAACATGGGAAGGCCTAG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>2798514</td>
<td>ATTCAGATGTCATGTTCAAA</td>
<td></td>
<td>G</td>
<td>AAAAAGATAGACTATCCTAT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>2811496</td>
<td>CAGAAAGTACAAAGCAATCA</td>
<td></td>
<td>A</td>
<td>TACAAGAAGAAGAGAATATC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>2938969</td>
<td>AGGCAACTTTTTTTTTTTTT</td>
<td>T</td>
<td></td>
<td>AAGGGTTGTAGTATTGTTTA</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>3006136</td>
<td>GTATTCTATACCTTGGGCAG</td>
<td>T</td>
<td>A</td>
<td>TTCTTCAAGCAATTTGGTCA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>3040333</td>
<td>ACTCTAGATCTCCGTCTCCG</td>
<td>G</td>
<td></td>
<td>CTGCACGCCCTATTTCTCGC</td>
<td>No</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="even">
<td>II</td>
<td>3619004</td>
<td>CAACCGAGAAGTCCCGGTAG</td>
<td>G</td>
<td></td>
<td>TTGCGTCTTTGTATATAAAG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>3838114</td>
<td>AGACCTCAAAAAAAAAAAAA</td>
<td></td>
<td>A</td>
<td>GCGAAATGCTAACTTCAAAA</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>4122325</td>
<td>ATCGCTGTTAATGGTGCACA</td>
<td>C</td>
<td>T</td>
<td>ATCCACAACGTGATTGGAAC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>II</td>
<td>4254961</td>
<td>TGGCCGGGTTATTCACAAGA</td>
<td></td>
<td>A</td>
<td>TACCTTTTATCCAGCTTGAC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>II</td>
<td>4400665</td>
<td>CAAATTAAACAATCCAAACC</td>
<td>C</td>
<td>A</td>
<td>TAAATCGTCACCAGTACTAG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>III</td>
<td>227729</td>
<td>TTTCGTGTTCACGAGGAAGA</td>
<td>T</td>
<td>A</td>
<td>CGTTGTTTTTGCACATAATG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>III</td>
<td>240810</td>
<td>TGACCATACCATTGGGATGA</td>
<td>A</td>
<td>G</td>
<td>CCTTACTCTTGATTAGCTCG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>III</td>
<td>284656</td>
<td>AGTATAACTAGCGCAGTAGG</td>
<td>A</td>
<td>C</td>
<td>AAAACAGAAGATTGTATGAG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>III</td>
<td>719956</td>
<td>GTTTAATATATTTAATAAAC</td>
<td></td>
<td>C</td>
<td>TATTTTATTCTATGTCGAGA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>III</td>
<td>719970</td>
<td>ATAAACCTATTTTATTCTAT</td>
<td>T</td>
<td>G</td>
<td>TCGAGAGAAGATTGAGATGA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>III</td>
<td>719973</td>
<td>AACCTATTTTATTCTATGTC</td>
<td></td>
<td>G</td>
<td>AGAGAAGATTGAGATGATTA</td>
<td>Yes</td>
<td>-</td>
<td>3</td>
</tr>
<tr class="odd">
<td>III</td>
<td>719975</td>
<td>CCTATTTTATTCTATGTCGA</td>
<td>A</td>
<td>G</td>
<td>AGAAGATTGAGATGATTAAT</td>
<td>Yes</td>
<td>-</td>
<td>2</td>
</tr>
<tr class="even">
<td>III</td>
<td>719977</td>
<td>TATTTTATTCTATGTCGAGA</td>
<td>A</td>
<td>G</td>
<td>AAGATTGAGATGATTAATCA</td>
<td>Yes</td>
<td>-</td>
<td>2</td>
</tr>
<tr class="odd">
<td>III</td>
<td>719978</td>
<td>ATTTTATTCTATGTCGAGAG</td>
<td>T</td>
<td>A</td>
<td>AGATTGAGATGATTAATCAT</td>
<td>Yes</td>
<td>-</td>
<td>1</td>
</tr>
<tr class="even">
<td>III</td>
<td>719980</td>
<td>TTTATTCTATGTCGAGAGAA</td>
<td>T</td>
<td>G</td>
<td>ATTGAGATGATTAATCATTC</td>
<td>Yes</td>
<td>-</td>
<td>2</td>
</tr>
<tr class="odd">
<td>III</td>
<td>719984</td>
<td>TTCTATGTCGAGAGAAGATT</td>
<td>T</td>
<td>G</td>
<td>AGATGATTAATCATTCCCTC</td>
<td>Yes</td>
<td>-</td>
<td>4</td>
</tr>
<tr class="even">
<td>III</td>
<td>719986</td>
<td>CTATGTCGAGAGAAGATTGA</td>
<td>T</td>
<td>G</td>
<td>ATGATTAATCATTCCCTCTA</td>
<td>Yes</td>
<td>-</td>
<td>2</td>
</tr>
<tr class="odd">
<td>III</td>
<td>719987</td>
<td>TATGTCGAGAGAAGATTGAG</td>
<td>T</td>
<td>A</td>
<td>TGATTAATCATTCCCTCTAT</td>
<td>Yes</td>
<td>-</td>
<td>1</td>
</tr>
<tr class="even">
<td>III</td>
<td>719989</td>
<td>TGTCGAGAGAAGATTGAGAT</td>
<td>T</td>
<td>G</td>
<td>ATTAATCATTCCCTCTATCT</td>
<td>Yes</td>
<td>-</td>
<td>2</td>
</tr>
<tr class="odd">
<td>III</td>
<td>719990</td>
<td>GTCGAGAGAAGATTGAGATG</td>
<td>T</td>
<td>A</td>
<td>TTAATCATTCCCTCTATCTA</td>
<td>Yes</td>
<td>-</td>
<td>1</td>
</tr>
<tr class="even">
<td>III</td>
<td>720000</td>
<td>GATTGAGATGATTAATCATT</td>
<td>T</td>
<td>C</td>
<td>CCTCTATCTATTGATTGTTT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>III</td>
<td>720696</td>
<td>TCCACAATTTTTTTTTTTTT</td>
<td>T</td>
<td></td>
<td>ACCATCTCCTTTAAATCAAA</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="even">
<td>III</td>
<td>867874</td>
<td>CTTTATTTTACTTTTTTTTT</td>
<td></td>
<td>T</td>
<td>ATATTTTTTAAAATATATTC</td>
<td>No</td>
<td>X</td>
<td></td>
</tr>
<tr class="odd">
<td>III</td>
<td>1016312</td>
<td>TAACAATGAAATAACGAAGC</td>
<td>C</td>
<td></td>
<td>TTCAGTATAATTAGTTCTAT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>III</td>
<td>1168327</td>
<td>TCATAGCCTTTTATTACTAT</td>
<td>C</td>
<td>T</td>
<td>TGGGAAGTCAATTTTTAGTA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>III</td>
<td>1374937</td>
<td>AGGTTGTTATGTTTCAGACT</td>
<td>T</td>
<td>A</td>
<td>TGATCATTACGGTGATACTG</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>III</td>
<td>1685491</td>
<td>GGCCACCACCAAGAAGAAAA</td>
<td>A</td>
<td></td>
<td>CCAAAGAAAAAGAACAGTCA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>III</td>
<td>1756401</td>
<td>AAACCTTTGATGAAAGTTTA</td>
<td></td>
<td>A</td>
<td>TAAAGCTGCCAATGTATATA</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>III</td>
<td>1765400</td>
<td>TAAATTGCCGTATTATATTG</td>
<td>T</td>
<td>G</td>
<td>TACTATAACGAAATAATTGC</td>
<td>No</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>III</td>
<td>1774235</td>
<td>AACATCTTCCAAGTTCTCAT</td>
<td>GATC</td>
<td></td>
<td>CTTTAAAGATTCCTCAAGGT</td>
<td>No</td>
<td></td>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372">PMID:16823372</a></td>
</tr>
<tr class="even">
<td>III</td>
<td>2381978</td>
<td>AATTTGATGTCTTAAATTTG</td>
<td>T</td>
<td>G</td>
<td>TGGTTTGCTTGAAGGCATAT</td>
<td>No</td>
<td></td>
<td></td>
</tr>
</tbody>
</table>

  </div>
  <div *ngIf="pageName == 'sequencing-status'">
<h3 id="sequencing-status">Sequencing status</h3>
<p>Note: Many older <em>S. pombe</em> sequence submissions to the DNA databases (International Nucleotide Sequence Database Collaboration databases, i.e. ENA, GenBank, DDBJ) contain one or more errors, and we do not have the resources to maintain past sequences or flag every error in PomBase.</p>
<h4 id="chromosome-1">Chromosome 1</h4>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">Contig Name</th>
<th style="text-align: left;">Region</th>
<th style="text-align: left;">Size</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;">unsequenced to chr1 left telomere</td>
<td style="text-align: left;">10 ± 2 kb*</td>
</tr>
<tr class="even">
<td style="text-align: left;">c212</td>
<td style="text-align: left;">sub-telomeric left arm</td>
<td style="text-align: left;">29,663 bp</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;">Gap</td>
<td style="text-align: left;">&lt;5 kb*</td>
</tr>
<tr class="even">
<td style="text-align: left;">c977</td>
<td style="text-align: left;">left arm and right arm</td>
<td style="text-align: left;">5,549,370 bp</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;">unsequenced to chr1 left telomere</td>
<td style="text-align: left;">18 ± 3 kb*</td>
</tr>
</tbody>
</table>
<h4 id="chromosome-2">Chromosome 2</h4>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">Contig Name</th>
<th style="text-align: left;">Region</th>
<th style="text-align: left;">Size</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">AB325691</td>
<td style="text-align: left;">chr2 left arm gap-filling contig</td>
<td style="text-align: left;">20,000 bp**</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;">Gap</td>
<td style="text-align: left;">5 ± 5 kb</td>
</tr>
<tr class="odd">
<td style="text-align: left;">c1348</td>
<td style="text-align: left;">sub-telomeric left arm</td>
<td style="text-align: left;">80,201 bp</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;">Gap</td>
<td style="text-align: left;">22 ± 5 kb*</td>
</tr>
<tr class="odd">
<td style="text-align: left;">pB10D8</td>
<td style="text-align: left;">left arm to centromeric gap</td>
<td style="text-align: left;">1,536,269 bp</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;">Gap</td>
<td style="text-align: left;">~6 kb</td>
</tr>
<tr class="odd">
<td style="text-align: left;">pJ5566</td>
<td style="text-align: left;">right arm from centromeric gap to telomeric repeats</td>
<td style="text-align: left;">2,923,134 bp</td>
</tr>
</tbody>
</table>
<h4 id="chromosome-3">Chromosome 3</h4>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">Contig Name</th>
<th style="text-align: left;">Region</th>
<th style="text-align: left;">Size</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">p20C8</td>
<td style="text-align: left;">left arm from centromeric gap</td>
<td style="text-align: left;">1,083,348 bp</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;">Gap</td>
<td style="text-align: left;">25.3 ± 6 kb***</td>
</tr>
<tr class="odd">
<td style="text-align: left;">c1676</td>
<td style="text-align: left;">right arm from centromeric gap</td>
<td style="text-align: left;">1,369,435 bp</td>
</tr>
</tbody>
</table>
<p>* pers comm. Randy Hyppa and Gerry Smith.</p>
<p>** Sasaki <em>et al.</em> (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18727152">PMID:18727152</a>)</p>
<p>*** pers comm. Chad Ellermeier and Gerry Smith.</p>
<h4 id="centromere-clone-information"><a routerLink="/status/centromeres">Centromere clone information</a></h4>
<h4 id="telomere-clone-information"><a routerLink="/status/telomeres">Telomere clone information</a></h4>
<h4 id="mating-type-region-information"><a routerLink="/status/mating-type-region">Mating type region information</a></h4>
<h4 id="ribosomal-dna-information-faq"><a routerLink="/faq/there-any-rdna-repeat-sequences-pombase">Ribosomal DNA information (FAQ)</a></h4>

  </div>
  <div *ngIf="pageName == 'sequencing-updates'">
<h3 id="s.-pombe-sequence-updates-since-july-2003"><em>S. pombe</em> Sequence updates since July 2003</h3>
<h4 id="aug-29-2008-chromosome-1-cosmid-c30-or-c29e6-affecting-spac29e6.04cnnf1">Aug 29 2008 Chromosome 1 cosmid C30 OR c29E6 affecting SPAC29E6.04c/Nnf1</h4>
<p><strong>PENDING <span class="no-break">2008-08-29</span></strong></p>
<p>The reference sequence appears to contain an extra ‘G’ at position 4410193 PMID:17035632, also confirmed by Broad sequence; Pers. comm. Aki Hayashi Reported 05/16/07</p>
<h4 id="aug-29-2008-chromosome-1-cosmid-c767-affecting-spac767.01cvps1">Aug 29 2008 Chromosome 1 cosmid c767 affecting SPAC767.01c/Vps1</h4>
<p><strong>PENDING <span class="no-break">2007-08-23</span></strong></p>
<p>At 1999191 in Vps1 TGTGACC[C-&gt;G]TACGAC PMID:18088324, also confirmed by Broad sequence; Pers. comm. Isabelle Jourdain Reported 06/15/05</p>
<h4 id="aug-29-2008-chromosome-2-cosmid-c21d10-affecting-spbc21d10.06cmap4">Aug 29 2008 Chromosome 2 cosmid c21D10 affecting SPBC21D10.06c/Map4</h4>
<p><strong>PENDING <span class="no-break">2008-08-29</span></strong></p>
<p>Map4 in the reference genome contains an array of 5 repeats. In PMID:168571979 repeats are reported. This number is the correct number of repeats and will be updated via an insertion into the contig sequence shortly. Pers. comm. Henar Valdivieso Reported 06/15/05</p>
<h4 id="aug-29-2008-chromosome-1-cosmid-c27d7-misassembly-causing-duplication-of-spac27d7.09c-spac27d7.10c">Aug 29 2008 Chromosome 1 cosmid c27D7 misassembly causing duplication of SPAC27D7.09c (SPAC27D7.10c)</h4>
<p><strong>PENDING <span class="no-break">2008-08-29</span></strong></p>
<p>An apparent repeat region on chromosome 1 coordinates 4526059..4529095 (cosmid c27D7) is caused by a missassembly and will be removed from the genomic sequence shortly. The CDS feature SPAC27D7.10c within this region is an exact duplication of SPAC27D7.09c and will be merged with this CDS. Pers. comm. Klavs Hansen. Reported 09/01/04</p>
<h4 id="jun-2008-gatc-in-overlap-c1450-and-c1442-affects-spcc1442.04c">Jun 2008 GATC in overlap c1450 and c1442 affects SPCC1442.04c</h4>
<p><strong>PENDING <span class="no-break">2008-08-26</span></strong></p>
<p>There are 4 bases missing from the sequence AACATCTTCCAAGTTCTCAT GATC CTTTAAAGATTCCTCAAGGT, which will change the C-terminal region slightly. This will be fixed in the next round of corrections</p>
<h4 id="jun-2008-197-potential-sequence-discrepancies-from-the-broad-institute">Jun 2008 197 potential sequence discrepancies from the Broad Institute</h4>
<p><strong>PENDING <span class="no-break">2008-08-26</span></strong></p>
<p>Generated as part of a technology development project, using Illumina (Solexa) technology the Broad Institute generated &gt;200-fold sequence coverage of the genome of <em>S. pombe</em> strain 972. These data identified 197 potential discrepancies. The discrepancy rate is about 1 in 100,000, which is close to the estimated error rate in published reference sequence. Further information and data <a routerLink="/status/sequence-updates-pending">here</a></p>
<h4 id="jan-2007-genedb-moves-from-contigs-to-chromosomes">Jan 2007 GeneDB moves from Contigs to Chromosomes</h4>
<p>Contigs merged into chromosomes: the 4 sequence gaps represented by 100 Ns<br />
(<strong>Note:</strong> The chromosomes previously made available from the ftp site had 1000 Ns in the gaps)</p>
<h4 id="may-2006-chromosome-1-cosmid-c21e11">May 2006 Chromosome 1 cosmid c21E11:</h4>
<p>Single base insertion at c21E11 from gagtgca to gagtgGca (in cyp8) base 4257404 in new chromsome file (note reverse strand sequence reported)</p>
<h4 id="may-2004-chromosome-2-cosmid-c646">May 2004 Chromosome 2 cosmid c646:</h4>
<p>Single base transition A-&gt;C at c646 from aatggaccAtacccc to aatggaccCtacccc<br />
(<strong>Note:</strong> This change does not affect sequence coordinates)</p>
<h4 id="apr-2004-chromosome-1-cosmid-c6b12">Apr 2004 Chromosome 1 cosmid c6B12:</h4>
<p>Single base deletion at c6B12 13981 from tgttctgcAtttttcc to tgttctgctttttcc<br />
(<strong>Note:</strong> This change was not added to the spreadsheet on the ftp site until Jan. 18, 2006)</p>
<h4 id="jul-2003-chromosome-1-gap-filled">Jul 2003 Chromosome 1 Gap Filled</h4>
<p>Added PCR product SPACPJ113 6900 with overlaps of 504 bp with c1856 and 147bp with p4C9<br />
Joined contigs p4C9 and c977</p>

  </div>
  <div *ngIf="pageName == 'statistics'">
<h2 id="genome-statistics">Genome statistics</h2>
<h3 id="genome-structure-and-organisation">Genome Structure and Organisation</h3>
<table>
<tbody>
<tr class="odd">
<td style="text-align: left;">Genome Size (sequenced portion, excluding rDNA)</td>
<td style="text-align: left;">12.57Mbp</td>
</tr>
<tr class="even">
<td style="text-align: left;">Number of Chromosomes</td>
<td style="text-align: left;">3</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Chromosome Size Range</td>
<td style="text-align: left;">2.45Mbp - 5.58Mbp</td>
</tr>
<tr class="even">
<td style="text-align: left;">Gene Count (protein coding genes on Chr I, II and III)</td>
<td style="text-align: left;">5118</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Gene Count excluding dubious genes (protein coding genes on Chr I, II and III)</td>
<td style="text-align: left;">5064</td>
</tr>
<tr class="even">
<td style="text-align: left;">Gene Density (genes per Mb)</td>
<td style="text-align: left;">552.66</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Genome GC Content</td>
<td style="text-align: left;">36.06%</td>
</tr>
<tr class="even">
<td style="text-align: left;">Intergenic Length (mean for protein coding genes)</td>
<td style="text-align: left;">227.37</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Intergenic GC Content (protein coding genes)</td>
<td style="text-align: left;">28.94%</td>
</tr>
<tr class="even">
<td style="text-align: left;">Mitochondrial Genome Size</td>
<td style="text-align: left;">19.43kbp</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Mitochondrial GC Content</td>
<td style="text-align: left;">30.09%</td>
</tr>
<tr class="even">
<td style="text-align: left;">Telomeric Located Contig Size</td>
<td style="text-align: left;">20.00kbp</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Telomeric Located Contig GC Content</td>
<td style="text-align: left;">33.17%</td>
</tr>
</tbody>
</table>
<h3 id="chromosome-i-ii-and-iii-transcript-breakdown">Chromosome (I, II and III) Transcript Breakdown</h3>
<table>
  <thead>
    <tr>
      <th rowspan="2">Feature Type</th>
      <th colspan="3" style="text-align: center;">Count</th>
      <th>Gene Density</th>
      <th colspan="6" style="text-align: center;">GC Content (%)</th>
      <th colspan="5" style="text-align: center;">Mean Length (bp)</th>
    </tr>
    <tr>
      <th>Gene</th>
      <th>Exon</th>
      <th>Intron</th>
      <th>(genes per Mb)</th>
      <th>Gene</th>
      <th>Exon (All)</th>
      <th>Exon (Coding)</th>
      <th>Intron</th>
      <th>5' UTR</th>
      <th>3' UTR</th>
      <th>Gene</th>
      <th>Exon</th>
      <th>Intron</th>
      <th>5' UTR</th>
      <th>3' UTR</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>rRNA</td>
      <td>47</td>
      <td>47</td>
      <td>0</td>
      <td>3.74</td>
      <td>45.72</td>
      <td>45.72</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td style="text-align:right;">527.53</td>
      <td style="text-align:right;">527.53</td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>snoRNA</td>
      <td>57</td>
      <td>58</td>
      <td>1</td>
      <td>4.53</td>
      <td>36.54</td>
      <td>36.82</td>
      <td></td>
      <td>30.36</td>
      <td></td>
      <td></td>
      <td style="text-align:right;">140.09</td>
      <td style="text-align:right;">131.88</td>
      <td style="text-align:right;">336.00</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>tRNA</td>
      <td>171</td>
      <td>197</td>
      <td>26</td>
      <td>13.60</td>
      <td>53.79</td>
      <td>54.15</td>
      <td></td>
      <td>38.56</td>
      <td></td>
      <td></td>
      <td style="text-align:right;">77.32</td>
      <td style="text-align:right;">65.56</td>
      <td style="text-align:right;">11.77</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>pseudogene</td>
      <td>28</td>
      <td>50</td>
      <td>22</td>
      <td>2.23</td>
      <td>38.95</td>
      <td>39.25</td>
      <td></td>
      <td>30.90</td>
      <td></td>
      <td></td>
      <td style="text-align:right;">1,047.39</td>
      <td style="text-align:right;">565.70</td>
      <td style="text-align:right;">47.36</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>mRNA (protein_coding)</td>
      <td>5118</td>
      <td>10425</td>
      <td>5307</td>
      <td>407.10</td>
      <td>37.08</td>
      <td>37.39</td>
      <td>39.63</td>
      <td>29.72</td>
      <td>33.24</td>
      <td>32.27</td>
      <td style="text-align:right;">2,147.68</td>
      <td style="text-align:right;">1,011.98</td>
      <td style="text-align:right;">83.27</td>
      <td style="text-align:right;">290.05</td>
      <td style="text-align:right;">428.79</td>
    </tr>
    <tr>
      <td>ncRNA</td>
      <td>1522</td>
      <td>1527</td>
      <td>5</td>
      <td>121.06</td>
      <td>35.82</td>
      <td>35.82</td>
      <td></td>
      <td>27.94</td>
      <td></td>
      <td></td>
      <td style="text-align:right;">1,248.46</td>
      <td style="text-align:right;">1,244.21</td>
      <td style="text-align:right;">49.40</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td>snRNA</td>
      <td>5</td>
      <td>6</td>
      <td>1</td>
      <td>0.40</td>
      <td>40.03</td>
      <td>40.47</td>
      <td></td>
      <td>34.00</td>
      <td></td>
      <td></td>
      <td style="text-align:right;">147.40</td>
      <td style="text-align:right;">114.50</td>
      <td style="text-align:right;">50.00</td>
      <td></td>
      <td></td>
    </tr>
  </tbody>
</table>
<h3 id="mitochondrial-transcript-breakdown">Mitochondrial transcript breakdown</h3>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">Feature type</th>
<th style="text-align: left;">Count</th>
<th style="text-align: left;">GC content (%)</th>
<th style="text-align: left;">Mean length (bp)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">rRNA</td>
<td style="text-align: left;">2</td>
<td style="text-align: left;">31.97</td>
<td style="text-align: left;">2,122.00</td>
</tr>
<tr class="even">
<td style="text-align: left;">mRNA (protein_coding)</td>
<td style="text-align: left;">11</td>
<td style="text-align: left;">29.39</td>
<td style="text-align: left;">1,393.00</td>
</tr>
<tr class="odd">
<td style="text-align: left;">tRNA</td>
<td style="text-align: left;">25</td>
<td style="text-align: left;">38.75</td>
<td style="text-align: left;">74.52</td>
</tr>
</tbody>
</table>
<p><br />
### Telomeric located contig transcript breakdown</p>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">Feature type</th>
<th style="text-align: left;">Count</th>
<th style="text-align: left;">GC content (%)</th>
<th style="text-align: left;">Mean length (bp)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">mRNA (protein_coding)</td>
<td style="text-align: left;">5</td>
<td style="text-align: left;">38.46</td>
<td style="text-align: left;">1,428.00</td>
</tr>
</tbody>
</table>
<p><br />
### Mating type region</p>
<table>
<thead>
<tr class="header">
<th style="text-align: left;">Feature type</th>
<th style="text-align: left;">Count</th>
<th style="text-align: left;">GC content (%)</th>
<th style="text-align: left;">Mean length (bp)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">mRNA (protein_coding)</td>
<td style="text-align: left;">4</td>
<td style="text-align: left;">35.94</td>
<td style="text-align: left;">395.75</td>
</tr>
</tbody>
</table>
<p><br />
#### <strong>Release Version:</strong> 30th Jan 2017</p>

  </div>
  <div *ngIf="pageName == 'telomeres'">
<h3 id="telomeres">Telomeres</h3>
<p>The fission yeast complete genome sequence currently stops short of the telomeric repeats. See the <a routerLink="/status/sequencing-status">Sequencing Status</a> page for the current assembly status.</p>
<p>The most proximal anchored cosmids to each telomere are (links to genome browser):</p>
<ul>
<li>Chromosome I left c212 (coordinates <a href="https://www.pombase.org/jbrowse/?loc=I%3A1..29664&amp;tracks=DNA%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features&amp;highlight=">1-29663</a>)</li>
<li>Cromosome I right c750 ([5554844-5579133][tel1lr])</li>
<li>Chromosome II left c1348 (<a href="https://www.pombase.org/jbrowse/?loc=II%3A1..39181&amp;tracks=DNA%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features&amp;highlight=">1-39186</a>)</li>
<li>Chromosome II right pT2R1 (<a href="https://www.pombase.org/jbrowse/?loc=II%3A4500628..4539804&amp;tracks=DNA%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features&amp;highlight=">4500619-4539800</a>)</li>
<li>There are no telomere proximal clones for chromosome III as the unsequenced rDNA blocks occur in between the sequenced portion and the telomeres on both chromosome arms.</li>
</ul>
<p>Details of all clones used for the assembly, and their order, length and overlap details is provided in a set of <a href="ftp://ftp.pombase.org/archive/Cosmid_assembly_data/">spreadsheets</a> available from the PomBase FTP site.</p>
<p>A contig extending the left arm of Chromosome II was sequenced by Sasaki <em>et al.</em> (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18727152">PMID:18727152</a>) and will be attached to the assembly in the next round of sequence changes. In the meantime, the contig can be viewed in the <a href="https://www.pombase.org/jbrowse/?loc=chr_II_telomeric_gap%3A2002..18001&amp;tracks=DNA%2CPomBase%20forward%20strand%20features%2CPomBase%20reverse%20strand%20features&amp;highlight=">PomBase genome browser</a>.</p>
<p>A set of <a href="ftp://ftp.sanger.ac.uk/pub/yeast/sequences/pombe/telomeres/">small insert clones</a> (FTP link; also see table) from a telomere plasmid library has been made available by <a href="mailto:sugawara@hydra.rose.brandeis.edu">Neal Sugawara</a>. None can be assigned to a chromosome at present.</p>
<table>
<thead>
<tr class="header">
<th>Plasmid</th>
<th>Size of insert</th>
<th>Location of telomeric sequence</th>
<th>Comments</th>
<th>Vector</th>
<th>Laboratory</th>
<th>Funded by</th>
<th>Stage</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>pNSU28</td>
<td>approx 1kb</td>
<td></td>
<td>Lies in pNSU21</td>
<td>pUC19</td>
<td>Hinxton</td>
<td>EC</td>
<td>finished</td>
</tr>
<tr class="even">
<td>pNSU31</td>
<td>approx 1kb</td>
<td></td>
<td>Lies in pNSU21</td>
<td>pUC19</td>
<td>Hinxton</td>
<td>EC</td>
<td>finished</td>
</tr>
<tr class="odd">
<td>pNSU68</td>
<td>423 bp</td>
<td>Internal</td>
<td>Contains 195 bp of telomeric DNA and 123 bp from the rDNA</td>
<td>pMLC28</td>
<td>Hinxton</td>
<td>EC</td>
<td>finished</td>
</tr>
<tr class="even">
<td>pNSU71</td>
<td>15 kb</td>
<td>Terminal</td>
<td></td>
<td>pMLC28</td>
<td>Hinxton</td>
<td>EC</td>
<td>finished</td>
</tr>
<tr class="odd">
<td>pNSU64<sup>*</sup></td>
<td>6.9 kb</td>
<td>Terminal</td>
<td></td>
<td>pMLC28</td>
<td>Hinxton</td>
<td>EC</td>
<td>finished</td>
</tr>
<tr class="even">
<td>pNSU70</td>
<td>7.1 kb</td>
<td></td>
<td>pMLC28</td>
<td>Hinxton</td>
<td>EC</td>
<td>finished</td>
<td></td>
</tr>
<tr class="odd">
<td>pNSU77<sup>*</sup></td>
<td>12 kb</td>
<td>Internal</td>
<td>Fusion between telomere sequences (7.1kb) and rDNA sequences (4.9kb)</td>
<td>pMLC28</td>
<td>Hinxton</td>
<td>EC</td>
<td>finished</td>
</tr>
<tr class="even">
<td>pNSU21</td>
<td>7.9 kb</td>
<td>Terminal</td>
<td>pNSU21 and pNSU65 were isolated from the first and second library respectively pMLC28</td>
<td>Hinxton</td>
<td>EC</td>
<td>finished</td>
<td></td>
</tr>
<tr class="odd">
<td>pNSU65</td>
<td>8.1 kb</td>
<td>Terminal</td>
<td>pNSU21 and pNSU65 were isolated from the first and second library respectively pMLC28</td>
<td>Hinxton</td>
<td>EC</td>
<td>finished</td>
<td></td>
</tr>
</tbody>
</table>
<p><strong>Table notes:</strong> “Terminal” refers to the position adjacent to the vector sequences of pMLC12 where the blunt end (SmaI end) ligated to the telomeric sequences. “Internal” means that an <em>S. pombe</em> sequence intervenes between the telomere and vector sequences.</p>

  </div>
  <div *ngIf="pageName == 'unmapped_genes'">
<h3 id="unmapped-genes">Unmapped genes</h3>
<h4 id="genetic-loci-not-cloned-or-physically-mapped">Genetic loci not cloned or physically mapped</h4>
<table>
 <thead>
  <tr>
   <th>Locus name</th>
   <th>Publication(s)</th>
   <th>Allelic with</th>
  </tr>
 </thead>
 <tbody>
 <tr>
  <td>ala1</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>all1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=24177583">PMID:24177583</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>all2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=24177583">PMID:24177583</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ani1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=732808">PMID:732808</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ani2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=732808">PMID:732808</a></td>
  <td>cyh4, tri3</td>
 </tr>
 <tr>
  <td>anr1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1394510">PMID:1394510</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>anr2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1394510">PMID:1394510</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>anr3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1394510">PMID:1394510</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>anr4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1394510">PMID:1394510</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>aro3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=4211223">PMID:4211223</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ban1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8187760">PMID:8187760</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8522609">PMID:8522609</a></td>
  <td>&nbsp;</td>
 <tr>
  <td>ban2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8522609">PMID:8522609</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ban3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8522609">PMID:8522609</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ban4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8522609">PMID:8522609</a></td>
  <td>&nbsp;</td>
 </tr>
  <tr>
  <td>ben2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7003311">PMID:7003311</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ben3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7003311">PMID:7003311</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>can2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=18357653">PMID:18357653</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>chr1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12746851">PMID:12746851</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15114862">PMID:15114862</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1524835">PMID:1524835</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1524835">PMID:1524835</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps6</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1524835">PMID:1524835</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps7</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1524835">PMID:1524835</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps9</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1524835">PMID:1524835</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps10</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1524835">PMID:1524835</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7882425">PMID:7882425</a></td>
  <td>&nbsp;</td>
 <tr>
  <td>cps11</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1524835">PMID:1524835</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps13</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1524835">PMID:1524835</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cps14</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1524835">PMID:1524835</a></td>
  <td>&nbsp;</td>
 <tr>
  <td>crm2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=2647765">PMID:2647765</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cyh1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=607151">PMID:607151</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cyh2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1068110">PMID:1068110</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cyh3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1068110">PMID:1068110</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>cyh4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1068110">PMID:1068110</a></td>
  <td>ani2, tri3</td>
 </tr>
 <tr>
  <td>glu1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7903653">PMID:7903653</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>gps1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7663020">PMID:7663020</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>gps2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7663020">PMID:7663020</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>gps3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7663020">PMID:7663020</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>gps4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7663020">PMID:7663020</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>git4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=2157626">PMID:2157626</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>hsp1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7500954">PMID:7500954</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>hus3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1427071">PMID:1427071</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>hus4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1427071">PMID:1427071</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>hyd1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=23712692">PMID:23712692</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ird5</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16377914">PMID:16377914</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=24301766">PMID:24301766</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=26301056">PMID:26301056</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ird11</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=22095476">PMID:22095476</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=22173629">PMID:22173629</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=25117315">PMID:25117315</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=26301056">PMID:26301056</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ird13</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16377914">PMID:16377914</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=24301766">PMID:24301766</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ird14</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16377914">PMID:16377914</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=24301766">PMID:24301766</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mcs3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=11085271">PMID:11085271</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=2474475">PMID:2474475</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15509866">PMID:15509866</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mcs5</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=2474475">PMID:2474475</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mis2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7865880">PMID:7865880</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mis7</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7865880">PMID:7865880</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mis8</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7865880">PMID:7865880</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mis9</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7865880">PMID:7865880</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mms1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9153313">PMID:9153313</a></td>
  <td>mms35</td>
 </tr>
 <tr>
  <td>mms8</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9153313">PMID:9153313</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mms38</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9153313">PMID:9153313</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mms52</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9153313">PMID:9153313</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mms57</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9153313">PMID:9153313</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>mms60</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9153313">PMID:9153313</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>orb1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8187760">PMID:8187760</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>orb8</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8522609">PMID:8522609</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>orb9</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8522609">PMID:8522609</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>orb10</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8522609">PMID:8522609</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>orb12</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8522609">PMID:8522609</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>pet1g</td>
  <td>&nbsp;</td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>pld6</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=10924454">PMID:10924454</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>pld8</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=10924454">PMID:10924454</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>pld9</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=10924454">PMID:10924454</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>pld10</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=10924454">PMID:10924454</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>pld11</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=10924454">PMID:10924454</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>pld12</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=10924454">PMID:10924454</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ptp1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1644270">PMID:1644270</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8082169">PMID:8082169</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8887563">PMID:8887563</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12058079">PMID:12058079</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=27664110">PMID:27664110</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ptp2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1644270">PMID:1644270</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8082169">PMID:8082169</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=27664110">PMID:27664110</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ral4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=3071741">PMID:3071741</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec50</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec52</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec53</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec54</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec55</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec56</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec58</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec59</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rec60</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1874411">PMID:1874411</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>rzl4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=27664110">PMID:27664110</a></td>
  <td>&nbsp;</td>
 </tr> 
 <tr>
  <td>rzl5</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=27664110">PMID:27664110</a></td>
  <td>&nbsp;</td>
 </tr>  
 <tr>
  <td>rzl6</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=27664110">PMID:27664110</a></td>
  <td>&nbsp;</td>
 </tr> 
 <tr>
  <td>scr1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9535817">PMID:9535817</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>scr2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9535817">PMID:9535817</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SLrsm1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21046351">PMID:21046351</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SLrsm2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21046351">PMID:21046351</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SLrsm3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=21046351">PMID:21046351</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>SP331</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7937765">PMID:7937765</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>srs1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=9651503">PMID:9651503</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ste2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1934121">PMID:1934121</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ste10</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1934121">PMID:1934121</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>ste15</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8575018">PMID:8575018</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts7</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts8</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts9</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts10</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>sts11</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1899230">PMID:1899230</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>tnr1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7961415">PMID:7961415</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1644306">PMID:1644306</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>tnr2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=7961415">PMID:7961415</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1644306">PMID:1644306</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>tri3</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=732808">PMID:732808</a></td>
  <td>ani2, cyh4</td>
 </tr>
 <tr>
  <td>tri4</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=732808">PMID:732808</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>tri5</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=732808">PMID:732808</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>trs1</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1769544">PMID:1769544</a></td>
  <td>&nbsp;</td>
 </tr>
    <tr>
  <td>uvi18</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8048925">PMID:8048925</a></td>
  <td>&nbsp;</td>
 </tr>
 <tr>
  <td>wtr2</td>
  <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=8913338">PMID:8913338</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15338055">PMID:15338055</a></td>
  <td>&nbsp;</td>
 </tr>
 </tbody>
</table>

  </div>
  </div>
</div>
<div *ngIf="section == 'submit-data'">
<app-page-contents-menu title="Submit data" titleRoute="/submit-data">

<div class="left-menu-part left-menu-item"><a routerLink="/submit-data/gene-names">Gene name registration</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/submit-data/gene-naming-guidelines"><i>S. pombe</i> gene naming guidelines</a></div>
<div class="left-menu-part left-sub-menu-item"><a routerLink="/submit-data/gene-naming-committee-members">Gene Naming Committee members</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/submit-data/bulk-annotation">Annotation datasets</a></div>
<div class="left-menu-part left-menu-item"><a routerLink="/documentation/data-submission-form-for-HTP-sequence-linked-data">Sequence-linked HTP data</a></div>


</app-page-contents-menu>
  <div class="docs-content">
  <div *ngIf="pageName == 'bulk-annotation'">
<h2 id="annotation-dataset-submission">Annotation dataset submission</h2>
<p>PomBase accepts batch submissions of certain types of data that appear on PomBase gene pages.</p>
<p>To submit any of these data types, please prepare a file in the format indicated, and send it to the helpdesk:</p>
<ul>
<li>Phenotype <a routerLink="/documentation/phenotype-data-bulk-upload-format">file format</a> and <a href="mailto:helpdesk@pombase.org?subject=phenotype%20data%20submission">submission link</a></li>
<li>Gene Ontology <a href="http://geneontology.org/page/go-annotation-file-gaf-format-21">GAF 2.1 file format</a> at GO site and <a href="mailto:helpdesk@pombase.org?subject=GO%20data%20submission">submission link</a></li>
<li>Modification <a routerLink="/documentation/modification-data-bulk-upload-format">file format</a> and <a href="mailto:helpdesk@pombase.org?subject=modification%20data%20submission">submission link</a></li>
<li>Qualitative gene expression data <a routerLink="/documentation/qualitative-gene-expression-data-bulk-upload-format">file format</a> and <a href="mailto:helpdesk@pombase.org?subject=qualitative%20gene%20expression%20data%20submission">submission link</a></li>
<li>Quantitative gene expression data <a routerLink="/documentation/quantitative-gene-expression-data-bulk-upload-format">file format</a> and <a href="mailto:helpdesk@pombase.org?subject=quantitative%20gene%20expression%20data%20submission">submission link</a></li>
</ul>

  </div>
  <div *ngIf="pageName == 'gene-name-conflicts'">
<h2 id="gene-name-conflicts">Gene name conflicts</h2>
<p>List of gene names that have been used to refer to two (or more) different genes. Resolution of these conflicts is under consideration by the Gene Naming Committee. Upon conflict resolution, the name will become a secondary name for at least one of the genes for which it has been used (a database search will still retrieve all usages). Systematic IDs, and primary names where available, for the conflictingly-named genes are listed in columns 2 and 4.</p>
<table>
<thead>
<tr class="header">
<th>Conflict</th>
<th>Systematic ID</th>
<th>Detail</th>
<th>Systematic ID</th>
<th>Detail</th>
<th>Comments</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>abc1</td>
<td><a routerLink="/gene/SPBC2D10.18">abc1/SPBC2D10.18</a></td>
<td>N Bonnefoy, 28 Aug 96</td>
<td><a routerLink="/gene/SPAC9E9.12c">ybt1/SPAC9E9.12c</a></td>
<td>J Davey, 11 Nov 96</td>
<td></td>
</tr>
<tr class="even">
<td>ade8</td>
<td><a routerLink="/gene/SPBC14F5.09c">ade8/SPBC14F5.09c</a></td>
<td></td>
<td><a routerLink="/gene/SPCC569.08c">ade5/SPCC569.08c</a></td>
<td></td>
<td>mapped ade5 was misnamed as ade8 from its S. cerevisiae ortholog; ade8 is now designated obsolete for SPCC569.08</td>
</tr>
<tr class="odd">
<td>car1</td>
<td><a routerLink="/gene/SPAC17A2.01">bsu1/SPAC17A2.01</a></td>
<td>Jia et al. 1993</td>
<td><a routerLink="/gene/SPBP26C9.02c">car1/SPBP26C9.02c</a></td>
<td>arginase, Van Huffel 1994</td>
<td>sod1 is used for superoxide dismutase (SPAC821.10c) so is a synonym, not a primary name, for SPAC17A2.01</td>
</tr>
<tr class="even">
<td>dim1</td>
<td><a routerLink="/gene/SPBC336.02">SPBC336.02</a></td>
<td>18S rRNA dimethylase, Housen 95</td>
<td><a routerLink="/gene/SPCC16A11.05c">dim1/SPCC16A11.05c</a></td>
<td>U4/U6.U5 snRNP, Berry &amp; Gould 97</td>
<td></td>
</tr>
<tr class="odd">
<td>gpd1</td>
<td><a routerLink="/gene/SPBC215.05">gpd1SPBC215.05</a></td>
<td>glycerol-3-p DH, J. Armstrong Oct 90</td>
<td><a routerLink="/gene/SPBC32F12.11">tdh1/SPBC32F12.11</a></td>
<td>glyceraldehyde-3-p DH, M. Vai Mar 95</td>
<td>implemented suggestion to make tdh1 primary and gpg1 secondary for SPBC32F12.11</td>
</tr>
<tr class="even">
<td>hcs</td>
<td><a routerLink="/gene/SPAC4F8.14c">hcs1/SPAC4F8.14c</a></td>
<td>Katayama, Jul 95</td>
<td>SPCC737.07c</td>
<td></td>
<td>Add “1” for consistency</td>
</tr>
<tr class="odd">
<td>krp1</td>
<td><a routerLink="/gene/SPAC22E12.09c">krp1/SPAC22E12.09c</a></td>
<td></td>
<td>klp3/SPAC1834.07</td>
<td></td>
<td>krp1 is a synonym for klp3</td>
</tr>
<tr class="even">
<td>mip1</td>
<td><a routerLink="/gene/SPCC24B10.22">pog1/SPCC24B10.22</a></td>
<td>DNA pol gamma, Copeland Jan 95</td>
<td><a routerLink="/gene/SPAC57A7.11">mip1/SPAC57A7.11</a></td>
<td>WD repeat, Yoshinori Sep 99</td>
<td></td>
</tr>
<tr class="odd">
<td>pab1</td>
<td><a routerLink="/gene/SPAC227.07c">pab1/SPAC227.07c</a></td>
<td>phosphatase, Kinoshita 95, (Yanagida lab)</td>
<td><a routerLink="/gene/SPAC57A7.pabp/04c">SPAC57A7.04c</a></td>
<td>poly(A) binding, Burd 91</td>
<td></td>
</tr>
<tr class="even">
<td>sod2</td>
<td><a routerLink="/gene/SPAC977.10">sod2/SPAC977.10</a></td>
<td>sodium antiporter, 92</td>
<td><a routerLink="/gene/SPAC1486.01">SPAC1486.01</a></td>
<td>Jeong &amp; Roe, Dec 99</td>
<td></td>
</tr>
<tr class="odd">
<td>ssp1</td>
<td><a routerLink="/gene/SPCC297.03">ssp1/SPCC297.03</a></td>
<td>p kinase, Matsusaka Oct 95</td>
<td><a routerLink="/gene/SPAC664.11">ssc1/SPAC664.11</a></td>
<td>chaperonin, Powell 90</td>
<td></td>
</tr>
<tr class="even">
<td>ppa1</td>
<td><a routerLink="/gene/SPAC823.15">ppa1/SPAC823.15</a></td>
<td>minor serine/threonine protein phosphatase pp2a-1 catalytic subunit, Kinoshita et al. Oct 90</td>
<td><a routerLink="/gene/SPAC23C11.05">SPAC23C11.05</a></td>
<td>inorganic pyrophosphatase, Kawasaki et al. Oct 90</td>
<td>ppa1 not published for SPAC23C11.05 so attached as obsolete gene name</td>
</tr>
<tr class="odd">
<td>ef1a-a, ef1a-b, ef1a-c</td>
<td><a routerLink="/gene/SPCC794.09c">SPCC794.09c</a>, <a routerLink="/gene/SPAC23A1.10">SPAC23A1.10</a>, <a routerLink="/gene/SPBC839.15c">SPBC839.15c</a></td>
<td>Elongation factor 1 alphas</td>
<td></td>
<td></td>
<td>All old, non-standard names obsoleted. These three kept temporarily to distinguish the three genomic copies. Current conventions need checking (Morimyo paper). (I haven’t been able to identify which is which from this as base differences in the sequenced clones or the genome sequence mean they cannot be mapped unambiguously. Needs investigating; our B and C are identical.) SPAC23A1.10 has standard name tef102.</td>
</tr>
<tr class="even">
<td>skp1</td>
<td><a routerLink="/gene/SPBC409.05">skp1/SPBC409.05</a></td>
<td></td>
<td><a routerLink="/gene/SPAC1687.15">gsk3/SPAC1687.15</a></td>
<td></td>
<td>Agreed that skp1 can be used for SPBC409.05</td>
</tr>
<tr class="odd">
<td>spp1</td>
<td><a routerLink="/gene/SPCC594.05c">spf1/SPCC594.05c</a></td>
<td></td>
<td><a routerLink="/gene/SPAC6B12.10c">spp1/SPAC6B12.10c</a></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>

  </div>
  <div *ngIf="pageName == 'gene-names'">
<h2 id="fission-yeast-gene-name-registry">Fission yeast gene name registry</h2>
<p>The Gene Naming Committee was initiated by Amar Klar, Paul Nurse and Mitsuhiro Yanagida to unify fission yeast gene names for newly defined genes, and resolve gene name conflicts.</p>
<!--
### <a routerLink="/registration-form">Gene name reservation form</a>g
-->
<h3 id="gene-name-reservation">Gene name reservation</h3>
<p>The <strong>gene name registration form</strong> will return soon. To reserve a gene name in the meantime, please <a href="mailto:helpdesk@pombase.org?subject=gene%20name%20reservation">email the helpdesk</a> and include:</p>
<ul>
<li>Systematic ID</li>
<li>Gene name</li>
<li>Name description (what the three-letter abbreviation stands for)</li>
<li>PubMed ID (if available) for the paper in which the name is first published</li>
<li>Any other information</li>
</ul>
<h3 id="s.-pombe-gene-naming-guidelines"><a routerLink="/submit-data/gene-naming-guidelines"><em>S. pombe</em> gene naming guidelines</a></h3>
<p>How to choose and reserve a gene name; name conflict resolution</p>
<h3 id="gene-naming-committee-members"><a routerLink="/submit-data/gene-naming-committee-members">Gene Naming Committee members</a></h3>
<hr />
<p>Historical note: Inventories of reserved gene names and <a routerLink="/submit-data/gene-name-conflicts">gene name conflicts</a> were maintained in the past, but have not been updated for several years.</p>

  </div>
  <div *ngIf="pageName == 'gene-naming-committee-members'">
<h2 id="s.-pombe-gene-naming-committee-members"><em>S. pombe</em> Gene Naming Committee members</h2>
<table>
<thead>
<tr class="header">
<th>Member</th>
<th>Affiliation</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Steve Aves</td>
<td>School of Biological Sciences, University of Exeter</td>
</tr>
<tr class="even">
<td>Susan Forsburg</td>
<td>Molecular and Computational Biology Section, University of Southern California</td>
</tr>
<tr class="odd">
<td>Olaf Nielsen</td>
<td>Department of Genetics, University of Copenhagen</td>
</tr>
<tr class="even">
<td>Osami Niwa</td>
<td>Kazusa DNA Research Institute, Kisarazu, Chiba, Japan</td>
</tr>
<tr class="odd">
<td>Matthias Sipiczki</td>
<td>Department of Genetics, University of Debrecen</td>
</tr>
<tr class="even">
<td>Takashi Toda</td>
<td>The Francis Crick Institute, London</td>
</tr>
<tr class="odd">
<td>Nancy Walworth</td>
<td>Robert Wood Johnson Medical School, NJ</td>
</tr>
<tr class="even">
<td>Yoshinori Watanabe</td>
<td>Laboratory of Chromosome Dynamics, Institute of Molecular and Cellular Biosciences, University of Tokyo</td>
</tr>
<tr class="odd">
<td>Valerie Wood</td>
<td>PomBase, University of Cambridge</td>
</tr>
<tr class="even">
<td>Paul Young</td>
<td>Queens University at Kingston, Ontario</td>
</tr>
</tbody>
</table>

  </div>
  <div *ngIf="pageName == 'gene-naming-guidelines'">
<h2 id="fission-yeast-gene-naming-guidelines">Fission yeast gene naming guidelines</h2>
<h3 id="purpose-of-reserving-gene-names">Purpose of Reserving Gene Names</h3>
<p>To avoid gene naming conflicts, the <em>S. pombe</em><a routerLink="/submit-data/gene-naming-committee-members">Gene Naming Committee</a> accepts reservations for gene names that will be published imminently. This does not guarantee that no one else will use your reserved gene name, but the naming committee will actively discourage alternative usage. If we become aware of a nomenclature conflict, we will attempt to notify all parties.</p>
<h3 id="guidelines-for-naming-s.-pombe-genes">Guidelines for naming <em>S. pombe</em> genes</h3>
<ol type="1">
<li>The gene name should consist of three italicized lower case letters (the gene symbol) followed by an integer (e.g. rad26). This will not be italicized in the database. (Note that a few historical gene names with four letters, or without an integer, have been grandfathered into the database, but all new standard gene names must conform to the three-letters-plus-integer pattern.)</li>
<li>The gene name registration form will return soon. To reserve a gene name in the meantime, please <a href="mailto:helpdesk@pombase.org?subject=gene%20name%20reservation">email the helpdesk</a> and include the gene systematic ID, proposed gene name, name description, the relevant PubMed ID (if available), and any other information.</li>
<li>The 3-letter gene symbol should stand for a description of a phenotype, gene product or gene function. In addition, we strongly prefer that a given gene symbol have only one associated description, i.e., all genes which use a given 3 letter symbol should have a related phenotype, gene product or gene function. If a 3-letter symbol is already in use, it should not be used to describe a different class of genes. Before choosing your gene name, search PomBase for any gene name beginning with the 3-letter symbol, by entering the 3-letter name followed by an asterisk, e.g. “cdc*”. Also check the usage of the 3-letter code in <em>S. cerevisiae</em> at <a href="http://www.yeastgenome.org/">SGD</a>.</li>
<li>Please include your gene name(s) in the abstracts of any relevant papers. This simplifies the task of identifying yeast gene names that are already in use.</li>
<li>At the time of registration, the <em>S. pombe</em> curator will check PubMed, GenBank/EMBL/DDBJ and the Gene Name Registry to ensure the gene name is unique. If your chosen gene name is not unique, you will be contacted and asked to select a new name.</li>
<li>Please check that the reserved gene name is still unique, and that the gene still has no other name used for it, before submitting a publication.</li>
<li>If it is discovered that your gene has been previously named, your gene name would instead become an alias (synonym) for the locus, and it and any associated references will remain searchable and linked to the standard locus name.</li>
<li>If you reserve a gene name prior to publication, and your reserved name is published referring to a different gene before your name usage is published, a compromise specific to the situation will be made. In most cases, you will NOT retain the use of the gene name unless you published the name first (see “Resolution of gene name conflicts” below).</li>
<li>If no conflicts arise, your gene name will become the standard name, and will appear on the gene page and elsewhere in PomBase.</li>
</ol>
<!-- restore when form back
2.  Gene names may be reserved using the [Gene Name Registration
    Form](submit-data/gene-registration-form). Please provide requested information and
    an explanation of the 3-letter gene symbol.
-->
<!-- probably won't retain this

### Renewing a reserved gene name

To renew a gene name reservation, you must submit new data demonstrating
continued study of this gene. Please use the [Gene Name Registration
Form](registration-form) to submit a renewal.
-->
<h3 id="resolution-of-gene-name-conflicts">Resolution of gene name conflicts</h3>
<p>Gene name conflicts in which multiple names have been used to describe one gene or, conversely, one name has been applied to multiple genes, will be resolved within 12 months. Whenever possible, all interested parties will be involved in the resolution of the conflict. We recognize that each case is unique, and we will choose the most appropriate solution using the following guidelines:</p>
<ol type="1">
<li><strong>Researcher consensus:</strong> In the first instance, if the researchers involved in the name conflict agree to a resolution that satisfies our Gene Naming requirements, this solution will be implemented.</li>
<li><strong>Literature consensus:</strong> In the absence of researcher consensus, we will examine the literature for the number of name usages and the number of different research groups utilizing a particular name usage. If there is a very obvious imbalance, we will favour the predominant name usage.</li>
<li><strong>Priority:</strong> In the absence of either researcher or literature consensus, we will favour the gene name usage that was first published if this is significantly earlier. However, if papers are published simultaneously (within 3 months), the pre-registered name will become the primary name.</li>
<li><strong>Relevance of the name:</strong> In rare cases where none of the above guidelines apply, we may favour a particular name usage that more accurately describes a phenotype, gene product or gene function.</li>
</ol>

  </div>
  <div *ngIf="pageName == 'index'">
<h2 id="pombase-data-submission">PomBase Data Submission</h2>
<h3 id="gene-names"><a routerLink="/submit-data/gene-names">Gene names</a></h3>
<p>Gene naming guidelines and how to submit a new gene name</p>
<h3 id="annotation-datasets"><a routerLink="/submit-data/bulk-annotation">Annotation datasets</a></h3>
<p>Data formats and instructions for submitting large sets of GO, phenotype, modification, or gene expression data.</p>
<h3 id="sequence-linked-htp-data"><a routerLink="/documentation/data-submission-form-for-HTP-sequence-linked-data">Sequence-linked HTP data</a></h3>
<p>Metadata format and instructions for submitting sequence-linked data to be displayed as a track in the genome browser.</p>
<h3 id="gene-specific-data">Gene-specific data</h3>
<p>To submit information on individual genes, use the Contact Curators link provided on the relevant <a href="http://www.pombase.org/">PomBase</a> gene page</p>

  </div>
  </div>
</div>
</div>
